{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "61854b4f",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/fuzzywuzzy/fuzz.py:11: UserWarning: Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning\n",
      "  warnings.warn('Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning')\n"
     ]
    }
   ],
   "source": [
    "from selenium import webdriver\n",
    "from selenium_stealth import stealth\n",
    "from selenium.webdriver.chrome.service import Service\n",
    "from selectolax.lexbor import LexborHTMLParser\n",
    "from typing import List, Dict, Callable\n",
    "import time, random, re\n",
    "import requests\n",
    "import io\n",
    "from io import BytesIO\n",
    "from io import StringIO\n",
    "from bs4 import BeautifulSoup\n",
    "import glob as glob\n",
    "from fuzzywuzzy import fuzz\n",
    "import numpy as np\n",
    "from pymed import PubMed\n",
    "import os, json\n",
    "from typing import List, Dict, Callable\n",
    "import time, random, re\n",
    "import pandas as pd\n",
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "import time \n",
    "import glob\n",
    "import os\n",
    "import pandas as pd\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "18f3b136",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/bin/bash: brew: command not found\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/bin/bash: brew: command not found\n"
     ]
    }
   ],
   "source": [
    "!brew install chromedriver\n",
    "!brew upgrade chromedriver"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "52f174c8",
   "metadata": {},
   "source": [
    "# TODO\n",
    "- clean up functions and doc strings and parameter typing - especially where it says for you to start onwards. \n",
    "- clean up the resulting files and merge pubmed results (clean duplicates etc.) \n",
    "- we also want to be sure the query is in the paper based on the pubmed url -\n",
    "  - i think there were some false positives from google scholar and most journals easy to hit url grab pdf and search it\n",
    "- when grouping by duplicates mark that multiple neuromabs were marked and str join on groupby so we see all the mabs that showed up in that paper. \n",
    "- query the paper via pubmed in order to confirm the query is in the paper? (we are doing this to confirm something which i cant rembmer its been a while)\n",
    "- follow up when you get the stage above and we will be adding more potential query data to the pipeline from antibodies inc and addgene\n",
    "- make a directory for the dates that were queried and the querieis that were created as files in the directory (unless you have a better idea but we want to be able to abstractly see the queries done and maybe check for files already being there and skip those queries?)   \n",
    "  - for now this is all the data we need need but lets make somethign for this a bit better for the future. \n",
    "\n",
    "\n",
    "\n",
    "## LONG TERM\n",
    "- data story telling and figures\n",
    "- add the add gene counts and do something creative to help us understand the most important research reagents (data science exercise)\n",
    "- antibodies inc sales as well but this and add gene are bottom of the list\n",
    "- add this data to the database/website"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8691d9d5",
   "metadata": {},
   "source": [
    "# A few notes\n",
    "- The code is not very clean but it works. Not sure how much you need of this but would be good excercise to work on and\n",
    "- `brew install chromedriver` to install chromedriver\n",
    "- change the directory above for your outputs \n",
    "- I can give you my login for VPN but you can also run with sleep as a background task - i think it does take a couple hours to reset. \n",
    "- Follow ups is find the papers with pubmed links and then scrape the pubmed links for the abstracts."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bb08afed",
   "metadata": {},
   "source": [
    "# Begin the scrapping of google scholar citations for neuromabs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "fe8aa3f8",
   "metadata": {},
   "outputs": [],
   "source": [
    "query_out = '/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "8a98a095",
   "metadata": {},
   "outputs": [],
   "source": [
    "def parse(parser: Callable, organic_results_data: Callable):\n",
    "    '''\n",
    "    Arugments:\n",
    "    - parser:  Lexbor parser from scrape_google_scholar_organic_results() function.\n",
    "    - organic_results_data: List to append data to. List origin location is scrape_google_scholar_organic_results() function. Line 104.\n",
    "    \n",
    "    This function parses data from Google Scholar Organic results and appends data to a List.\n",
    "    \n",
    "    It's used by scrape_google_scholar_organic_results().\n",
    "    \n",
    "    It returns nothing as it appends data to `organic_results_data`, \n",
    "    which appends it to `organic_results_data` List in the scrape_google_scholar_organic_results() function.\n",
    "    '''\n",
    "    \n",
    "    for result in parser.css('.gs_r.gs_or.gs_scl'):\n",
    "        try:\n",
    "            title: str = result.css_first('.gs_rt').text()\n",
    "        except: title = None\n",
    "\n",
    "        try:\n",
    "            title_link: str = result.css_first('.gs_rt a').attrs['href']\n",
    "        except: title_link = None\n",
    "\n",
    "        try:\n",
    "            publication_info: str = result.css_first('.gs_a').text()\n",
    "        except: publication_info = None\n",
    "\n",
    "        try:\n",
    "            snippet: str = result.css_first('.gs_rs').text()\n",
    "        except: snippet = None\n",
    "\n",
    "        try:\n",
    "            # if Cited by is present in inline links, it will be extracted\n",
    "            cited_by_link = ''.join([link.attrs['href'] for link in result.css('.gs_ri .gs_fl a') if 'Cited by' in link.text()])\n",
    "        except: cited_by_link = None\n",
    "        \n",
    "        try:\n",
    "            # if Cited by is present in inline links, it will be extracted and type cast it to integer\n",
    "            cited_by_count = int(''.join([re.search(r'\\d+', link.text()).group() for link in result.css('.gs_ri .gs_fl a') if 'Cited by' in link.text()]))\n",
    "        except: cited_by_count = None\n",
    "        \n",
    "        try:\n",
    "            pdf_file: str = result.css_first('.gs_or_ggsm a').attrs['href']\n",
    "        except: pdf_file = None\n",
    "\n",
    "        organic_results_data.append({\n",
    "            'title': title,\n",
    "            'title_link': title_link,\n",
    "            'publication_info': publication_info,\n",
    "            'snippet': snippet if snippet else None,\n",
    "            'cited_by_link': f'https://scholar.google.com{cited_by_link}' if cited_by_link else None,\n",
    "            'cited_by_count': cited_by_count if cited_by_count else None,\n",
    "            'pdf_file': pdf_file\n",
    "        })\n",
    "\n",
    "def scrape_google_scholar_organic_results(\n",
    "                                        query: str,\n",
    "                                        pagination: bool = False,\n",
    "                                        operating_system: str = 'Windows' or 'Linux',\n",
    "                                        year_start: int = None,\n",
    "                                        year_end: int = None,\n",
    "                                        save_to_csv: bool = False, \n",
    "                                        save_to_json: bool = False\n",
    "                                        ) -> List[Dict[str, str]]:\n",
    "    '''\n",
    "    Extracts data from Google Scholar Organic resutls page:\n",
    "    - title: str\n",
    "    - title_link: str\n",
    "    - publication_info: str \n",
    "    - snippet: str\n",
    "    - cited_by_link: str \n",
    "    - cited_by_count: int\n",
    "    - pdf_file: str\n",
    "    \n",
    "    Arguments:\n",
    "    - query: str. Search query. \n",
    "    - pagination: bool. Enables or disables pagination.\n",
    "    - operating_system: str. 'Windows' or 'Linux', Checks for operating system to either run Windows or Linux verson of chromedriver\n",
    "    \n",
    "    Usage:\n",
    "    data = scrape_google_scholar_organic_results(query='blizzard', pagination=False, operating_system='win') # pagination defaults to False \n",
    "    \n",
    "    for organic_result in data:\n",
    "        print(organic_result['title'])\n",
    "        print(organic_result['pdf_file'])\n",
    "    '''\n",
    "    assert year_start and year_end\n",
    "    if year_start or year_end:\n",
    "        assert year_start and year_end\n",
    "        assert year_start <= year_end\n",
    "\n",
    "    from selenium import webdriver\n",
    "    from selenium.webdriver.chrome.service import Service\n",
    "\n",
    "    service = Service()\n",
    "    #service = Service(executable_path='/usr/local/bin/chromedriver')\n",
    "\n",
    "\n",
    "    # selenium stealth\n",
    "    options = webdriver.ChromeOptions()\n",
    "    options.add_argument('--headless')\n",
    "    options.add_argument('--no-sandbox')\n",
    "    options.add_argument('--disable-dev-shm-usage')\n",
    "    \n",
    "    options.add_experimental_option('excludeSwitches', ['enable-automation'])\n",
    "    options.add_experimental_option('useAutomationExtension', False)\n",
    "    \n",
    "    # checks for operating system to either run Windows or Linux verson of chromedriver\n",
    "    # expects to have chromedriver near the runnable file\n",
    "    if operating_system is None:\n",
    "        raise Exception('Please provide your OS to `operating_system` argument: \"Windows\" or \"Linux\" for script to operate.')\n",
    "    \n",
    "    # if operating_system.lower() == 'windows' or 'win':\n",
    "    #     driver = webdriver.Chrome(options=options, service=Service(executable_path='chromedriver.exe'))\n",
    "    \n",
    "    #if operating_system.lower() == 'linux': \n",
    "    #driver = webdriver.Chrome(options=options, service=Service(executable_path='/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'))\n",
    "    \n",
    "\n",
    "\n",
    "    driver = webdriver.Chrome(service=service, options=options)\n",
    "    stealth(driver,\n",
    "        languages=['en-US', 'en'],\n",
    "        vendor='Google Inc.',\n",
    "        platform='Win32',\n",
    "        webgl_vendor='Intel Inc.',\n",
    "        renderer='Intel Iris OpenGL Engine',\n",
    "        fix_hairline=True,\n",
    "    )\n",
    "    \n",
    "    page_num: int = 0\n",
    "    organic_results_data: list = []\n",
    "    \n",
    "    if pagination:\n",
    "        while True:\n",
    "            # parse all pages\n",
    "            driver.get(f'https://scholar.google.com/scholar?q={query}&hl=en&gl=us&start={page_num}&as_ylo={year_start}&as_yhi={year_end}')\n",
    "            parser = LexborHTMLParser(driver.page_source)\n",
    "            \n",
    "            parse(parser=parser, organic_results_data=organic_results_data)\n",
    "            \n",
    "            # pagination\n",
    "            if parser.css_first('.gs_ico_nav_next'):  # checks for the \"Next\" page button\n",
    "                page_num += 10                        # paginate to the next page\n",
    "                time.sleep(random.randint(1, 2))      # sleep between paginations\n",
    "            else:\n",
    "                break\n",
    "    else:\n",
    "        # parse single, first page\n",
    "        driver.get(f'https://scholar.google.com/scholar?q={query}&hl=en&gl=us&start={page_num}')\n",
    "        parser = LexborHTMLParser(driver.page_source)\n",
    "    \n",
    "        parse(parser=parser, organic_results_data=organic_results_data)\n",
    "        \n",
    "    if save_to_csv:\n",
    "        pd.DataFrame(data=organic_results_data).to_csv('google_scholar_organic_results_data.csv', \n",
    "                                                        index=False, encoding='utf-8')\n",
    "    if save_to_json:\n",
    "        pd.DataFrame(data=organic_results_data).to_json('google_scholar_organic_results_data.json', \n",
    "                                                        index=False, orient='records')\n",
    "    driver.quit()\n",
    "    return organic_results_data\n",
    "\n",
    "\n",
    "def AddTargetCol(df,final_df):\n",
    "            \n",
    "    merged_df = final_df.merge(df[['Clone', 'Target']], left_on=['Antibody'],right_on=['Clone'], how='left')\n",
    "    merged_df = merged_df.drop('Clone', axis=1)\n",
    "   \n",
    "    return merged_df\n",
    "\n",
    "def CreatURL(query):\n",
    "    \n",
    "    query1 = query[0:8]\n",
    "    end_pos = query.find('/', 9)\n",
    "    query2= query[9:end_pos]\n",
    "    query3= query[end_pos+1:]\n",
    "    \n",
    "    url = 'https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=' + (query1 + '+' + query2 + '%' + '2F' + query3 + '&oq=neuromab+')\n",
    "\n",
    "    return url\n",
    "\n",
    "def CreateDF(query,result_value,data):\n",
    "    \n",
    "    # Record the difference between search result # and data length \n",
    "    # TODO wendon not sure why pop but:\n",
    "        # i bet this whole function can be cleaned quite a bit and might be a good initial cleaning stage\n",
    "        # more info is still better than less info most of the time moving forward more filtering at the final result after pubmed query\n",
    "    count = (int(result_value)) - (int({len(data)}.pop()))\n",
    "    final_df = pd.DataFrame(data)\n",
    "    \n",
    "    # Get rid of extraneous strings from title \n",
    "    final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n",
    "\n",
    "    # Creat file for data \n",
    "    file_query = query.replace(\"/\",\"_\")\n",
    "    file_query = file_query[0:8]+file_query[9:]\n",
    "    filename = file_query + \".csv\"\n",
    "    directory = query_out\n",
    "\n",
    "    # Input Antibody name into table \n",
    "    final_df[\"Antibody\"] = [query[9:]] * len(final_df)\n",
    "    final_df = AddTargetCol(df,final_df)\n",
    "    final_filename = directory + filename\n",
    "    final_df.to_csv(final_filename, index=False)\n",
    "\n",
    "\n",
    "def run_code(new_index, df):\n",
    "    # Initialize a value to count 0 result pages \n",
    "    zero_result_counter = 0\n",
    "    count_list = []\n",
    "    # in the future we will only need to run 2023 - 202X\n",
    "    zero_result_length = 10\n",
    "    start_year = 2023\n",
    "    end_year = 2025\n",
    "\n",
    "\n",
    "    for index, query_name in zip(df.index[new_index:], df['Clone'][new_index:]):\n",
    "        # Construct the filename and directory path\n",
    "        file_query = 'neuromab' + ' ' + query_name.replace(\"/\", \"_\")\n",
    "        filename = file_query[0:8] + file_query[9:] + \".csv\"\n",
    "        directory = query_out\n",
    "        final_filename = directory + filename\n",
    "\n",
    "        # Check if the file already exists\n",
    "        print(final_filename)\n",
    "        if os.path.exists(final_filename):\n",
    "            print(f\"File {final_filename} already exists. Skipping query for {query_name}.\")\n",
    "            count_list = []\n",
    "            continue  # Skip this iteration and move to the next query_name\n",
    "\n",
    "        data = {}\n",
    "        print(index, query_name)\n",
    "        # Set query for google scholar \n",
    "        query = 'neuromab' + ' ' + query_name\n",
    "\n",
    "\n",
    "        # Grab data from google scholar \n",
    "        data = scrape_google_scholar_organic_results(query=query, pagination=True, operating_system=\"Windows\", year_start=start_year, year_end=end_year)\n",
    "        if len(data) == 0:\n",
    "            count_list.append(len(data))\n",
    "            past_0 = 30\n",
    "            # if the last X elements are 0 then break - we got booted and need to reset the vpn\n",
    "            # check if the last past_0 elements are 0\n",
    "            if len(count_list) > past_0 and all([i == 0 for i in count_list[-past_0:]]):\n",
    "                print(\"We got booted, resetting vpn\")\n",
    "                new_index = index - past_0\n",
    "                print(count_list[-20:])\n",
    "                return new_index\n",
    "            continue\n",
    "        else:\n",
    "            print(len(data))\n",
    "            count_list.append(len(data))\n",
    "        # create URL to grab results \n",
    "        url = CreatURL(query)\n",
    "        url_info = requests.get(url)\n",
    "        html = url_info.text\n",
    "\n",
    "        # Create a BeautifulSoup object from the HTML string\n",
    "        soup = BeautifulSoup(html, 'html.parser')\n",
    "        result_elem = soup.select_one('#gs_ab_md .gs_ab_mdw')\n",
    "\n",
    "        if result_elem:\n",
    "            result_text = result_elem.get_text(strip=True)\n",
    "            result_match = re.search(r'(\\d+[\\d,]*)\\s+results', result_text)\n",
    "            if result_match == None:\n",
    "                result_match = re.search(r'(\\d+[\\d,]*)\\s+result', result_text)\n",
    "\n",
    "            if result_match:\n",
    "                result_value = result_match.group(1)\n",
    "                final_df = CreateDF(query,result_value,data)\n",
    "            elif len(data) > 0:\n",
    "                print(query + ' ' + \"Problem with search\")\n",
    "            else:\n",
    "                print(query + ' ' +\"Search - No papers Available\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "0c992473",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN358_68.csv\n",
      "0 N358/68\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN297_59.csv\n",
      "1 N297/59\n",
      "1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN144_14.csv\n",
      "2 N144/14\n",
      "18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN441_35.csv\n",
      "3 N441/35\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN46_30.csv\n",
      "4 N46/30\n",
      "9\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN57_2.csv\n",
      "5 N57/2\n",
      "3\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN444_63.csv\n",
      "6 N444/63\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN103_39.csv\n",
      "7 N103/39\n",
      "19\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN103_31.csv\n",
      "8 N103/31\n",
      "5\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN442_28.csv\n",
      "9 N442/28\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN330A_80.csv\n",
      "10 N330A/80\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN330A_51.csv\n",
      "11 N330A/51\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabL86_33.csv\n",
      "12 L86/33\n",
      "3\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabL86_36.csv\n",
      "13 L86/36\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabL86A_37.csv\n",
      "14 L86A/37\n",
      "11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN105_13.csv\n",
      "15 N105/13\n",
      "9\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN105_17.csv\n",
      "16 N105/17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "14\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN106_20.csv\n",
      "17 N106/20\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN106_36.csv\n",
      "18 N106/36\n",
      "215\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN106_65.csv\n",
      "19 N106/65\n",
      "34\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN106_43.csv\n",
      "20 N106/43\n",
      "3\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN388A_10.csv\n",
      "21 N388A/10\n",
      "4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN388A_27.csv\n",
      "22 N388A/27\n",
      "1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN454_91.csv\n",
      "23 N454/91\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN421A_85.csv\n",
      "24 N421A/85\n",
      "7\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN429_19.csv\n",
      "25 N429/19\n",
      "1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN290B_25.csv\n",
      "26 N290B/25\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN295B_66.csv\n",
      "27 N295B/66\n",
      "99\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN411_51.csv\n",
      "28 N411/51\n",
      "1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN271_44.csv\n",
      "29 N271/44\n",
      "9\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-2332065dedf2>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN347_42.csv\n",
      "30 N347/42\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN76_8.csv\n",
      "31 N76/8\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN76_3.csv\n",
      "32 N76/3\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN65_37.csv\n",
      "33 N65/37\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN360A_24.csv\n",
      "34 N360A/24\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN336B_83.csv\n",
      "35 N336B/83\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN332B_15.csv\n",
      "36 N332B/15\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabL124_59.csv\n",
      "37 L124/59\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN302_10.csv\n",
      "38 N302/10\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN304B_115.csv\n",
      "39 N304B/115\n",
      "We got booted, resetting vpn\n",
      "[3, 4, 1, 0, 7, 1, 0, 99, 1, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]\n",
      "29\n"
     ]
    }
   ],
   "source": [
    "new_index = run_code(0, df)\n",
    "print(new_index)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "a5fee73f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "483"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "d42683c7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN479_107.csv\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN479_107.csv already exists. Skipping query for N479/107.\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN152B_23.csv\n",
      "462 N152B/23\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN351_23.csv\n",
      "463 N351/23\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN351_24.csv\n",
      "464 N351/24\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabL118_80.csv\n",
      "465 L118/80\n",
      "1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-33-c56e2efc4f66>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabL118_66.csv\n",
      "466 L118/66\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN28_9.csv\n",
      "467 N28/9\n",
      "9\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-33-c56e2efc4f66>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN29_29.csv\n",
      "468 N29/29\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN34_34.csv\n",
      "469 N34/34\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabL108_92.csv\n",
      "470 L108/92\n",
      "1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-33-c56e2efc4f66>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN440_21.csv\n",
      "471 N440/21\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN440_61.csv\n",
      "472 N440/61\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN449_73.csv\n",
      "473 N449/73\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN432_21.csv\n",
      "474 N432/21\n",
      "1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-33-c56e2efc4f66>:193: FutureWarning: The default value of regex will change from True to False in a future version.\n",
      "  final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN432_63.csv\n",
      "475 N432/63\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabK91_36.csv\n",
      "476 K91/36\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN391_68.csv\n",
      "477 N391/68\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN428_12.csv\n",
      "478 N428/12\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN286_74.csv\n",
      "479 N286/74\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabK88_64.csv\n",
      "480 K88/64\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN476_9.csv\n",
      "481 N476/9\n",
      "/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3/neuromabN328B_37.csv\n",
      "482 N328B/37\n",
      "None\n"
     ]
    }
   ],
   "source": [
    "new_index = run_code(461, df)\n",
    "print(new_index)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "387f27cd",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "29f5e384",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fd0b9fdb",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "6e96449e",
   "metadata": {},
   "source": [
    "# Add target column from neuromab database\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "0a68a1f4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>TargetType</th>\n",
       "      <th>Target</th>\n",
       "      <th>AccessionNum</th>\n",
       "      <th>Clone</th>\n",
       "      <th>HumanGeneName</th>\n",
       "      <th>DataSheetFileName</th>\n",
       "      <th>IsoType</th>\n",
       "      <th>ValidationT</th>\n",
       "      <th>ValidationBrIB</th>\n",
       "      <th>ValidationBrIHC</th>\n",
       "      <th>ValidationKO</th>\n",
       "      <th>RRIDAB</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Epigenetics</td>\n",
       "      <td>5-formylcytidine</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N358/68</td>\n",
       "      <td>----</td>\n",
       "      <td>N358_68.pdf</td>\n",
       "      <td>IgG1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2877251</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Epigenetics</td>\n",
       "      <td>5-hydroxymethylcytidine</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N297/59</td>\n",
       "      <td>----</td>\n",
       "      <td>N297_59.pdf</td>\n",
       "      <td>IgG2a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2877253</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Tag</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>----</td>\n",
       "      <td>N144_14.pdf</td>\n",
       "      <td>IgG1</td>\n",
       "      <td>Pass</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2877254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Receptors</td>\n",
       "      <td>ADAM11</td>\n",
       "      <td>Q9R1V4</td>\n",
       "      <td>N441/35</td>\n",
       "      <td>ADAM11</td>\n",
       "      <td>N441_35.pdf</td>\n",
       "      <td>IgG2b</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>ND</td>\n",
       "      <td>2877610</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Receptors</td>\n",
       "      <td>ADAM22 (cytoplasmic)</td>\n",
       "      <td>Q9R1V6</td>\n",
       "      <td>N46/30</td>\n",
       "      <td>ADAM22</td>\n",
       "      <td>N46_30.pdf</td>\n",
       "      <td>IgG2b</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>2877292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>478</th>\n",
       "      <td>Rare Disease</td>\n",
       "      <td>WRN</td>\n",
       "      <td>Q14191</td>\n",
       "      <td>N428/12</td>\n",
       "      <td>WRN</td>\n",
       "      <td>N428_12.pdf</td>\n",
       "      <td>IgG1</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>ND</td>\n",
       "      <td>2877600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>Other</td>\n",
       "      <td>Zebrafish PSD Marker</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N286/74</td>\n",
       "      <td>----</td>\n",
       "      <td>N286_74.pdf</td>\n",
       "      <td>IgG1</td>\n",
       "      <td>ND</td>\n",
       "      <td>ND</td>\n",
       "      <td>ND</td>\n",
       "      <td>ND</td>\n",
       "      <td>2877461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>480</th>\n",
       "      <td>Other</td>\n",
       "      <td>Zinedin</td>\n",
       "      <td>Q9NRL3</td>\n",
       "      <td>K88/64</td>\n",
       "      <td>STRN4</td>\n",
       "      <td>K88_64.pdf</td>\n",
       "      <td>IgG2a</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>ND</td>\n",
       "      <td>2877335</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>481</th>\n",
       "      <td>Transporters</td>\n",
       "      <td>ZIP3</td>\n",
       "      <td>Q99K24</td>\n",
       "      <td>N476/9</td>\n",
       "      <td>SLC39A3</td>\n",
       "      <td>N476_9.pdf</td>\n",
       "      <td>IgG1</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>ND</td>\n",
       "      <td>2716255</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>482</th>\n",
       "      <td>Other</td>\n",
       "      <td>ZNF423</td>\n",
       "      <td>Q2M1K9</td>\n",
       "      <td>N328B/37</td>\n",
       "      <td>ZNF423</td>\n",
       "      <td>N328B_37.pdf</td>\n",
       "      <td>IgG1</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>Pass</td>\n",
       "      <td>2877483</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>483 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       TargetType                   Target AccessionNum     Clone  \\\n",
       "0     Epigenetics         5-formylcytidine          NaN   N358/68   \n",
       "1     Epigenetics  5-hydroxymethylcytidine          NaN   N297/59   \n",
       "2             Tag                    6xHis          NaN   N144/14   \n",
       "3       Receptors                   ADAM11       Q9R1V4   N441/35   \n",
       "4       Receptors     ADAM22 (cytoplasmic)       Q9R1V6    N46/30   \n",
       "..            ...                      ...          ...       ...   \n",
       "478  Rare Disease                      WRN       Q14191   N428/12   \n",
       "479         Other     Zebrafish PSD Marker          NaN   N286/74   \n",
       "480         Other                  Zinedin       Q9NRL3    K88/64   \n",
       "481  Transporters                     ZIP3       Q99K24    N476/9   \n",
       "482         Other                   ZNF423       Q2M1K9  N328B/37   \n",
       "\n",
       "    HumanGeneName DataSheetFileName IsoType ValidationT ValidationBrIB  \\\n",
       "0            ----       N358_68.pdf    IgG1         NaN            NaN   \n",
       "1            ----       N297_59.pdf   IgG2a         NaN            NaN   \n",
       "2            ----       N144_14.pdf    IgG1        Pass            NaN   \n",
       "3          ADAM11       N441_35.pdf   IgG2b        Pass           Pass   \n",
       "4          ADAM22        N46_30.pdf   IgG2b        Pass           Pass   \n",
       "..            ...               ...     ...         ...            ...   \n",
       "478           WRN       N428_12.pdf    IgG1        Pass           Pass   \n",
       "479          ----       N286_74.pdf    IgG1          ND             ND   \n",
       "480         STRN4        K88_64.pdf   IgG2a        Pass           Pass   \n",
       "481       SLC39A3        N476_9.pdf    IgG1        Pass           Pass   \n",
       "482        ZNF423      N328B_37.pdf    IgG1        Pass           Pass   \n",
       "\n",
       "    ValidationBrIHC ValidationKO   RRIDAB  \n",
       "0               NaN          NaN  2877251  \n",
       "1               NaN          NaN  2877253  \n",
       "2               NaN          NaN  2877254  \n",
       "3              Pass           ND  2877610  \n",
       "4              Pass         Pass  2877292  \n",
       "..              ...          ...      ...  \n",
       "478            Pass           ND  2877600  \n",
       "479              ND           ND  2877461  \n",
       "480            Pass           ND  2877335  \n",
       "481            Pass           ND  2716255  \n",
       "482            Pass         Pass  2877483  \n",
       "\n",
       "[483 rows x 12 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data = requests.get(\"https://neuromab.ucdavis.edu/catalog-download.cfm\").content\n",
    "df = pd.read_csv(BytesIO(data))\n",
    "df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a919df0b",
   "metadata": {},
   "source": [
    "# Merge all previous runs and the current run into one dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "035655d8",
   "metadata": {},
   "outputs": [],
   "source": [
    "# get data file names - replace with your path for what i send\n",
    "import glob\n",
    "import pandas as pd\n",
    "\n",
    "path = r'/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test'\n",
    "filenames = glob.glob(path + \"/*.csv\")\n",
    "\n",
    "dfs = []\n",
    "for filename in filenames:\n",
    "    dfs.append(pd.read_csv(filename))\n",
    "allnmab_df = pd.concat(dfs, ignore_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "8f226a8a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K Krzystanek, HB Rasmussen, M Grunnet, O Staub...</td>\n",
       "      <td>BACKGROUND: KCNQ1 (Kv7.1), together with its K...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=98014...</td>\n",
       "      <td>35.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N37A/10</td>\n",
       "      <td>Kv7.1/KCNQ1 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Caracterización de los canales de potasio depe...</td>\n",
       "      <td>https://idus.us.es/handle/11441/138533</td>\n",
       "      <td>M Ortego Domínguez - 2022 - idus.us.es</td>\n",
       "      <td>Los adipocitos blancos son células que partici...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://idus.us.es/bitstream/handle/11441/1385...</td>\n",
       "      <td>N375/67</td>\n",
       "      <td>Kv3.3 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Neuroinflammation alters integrative propertie...</td>\n",
       "      <td>https://link.springer.com/article/10.1007/s120...</td>\n",
       "      <td>F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...</td>\n",
       "      <td>… , N212/17, Neuromab, CA, USA) antibodies usi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=19097...</td>\n",
       "      <td>41.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Allostery between two binding sites in the io...</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...</td>\n",
       "      <td>Tetratricopeptide repeat (TPR) domains are ubi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=14199...</td>\n",
       "      <td>12.0</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Distal dendritic enrichment of HCN1 channels i...</td>\n",
       "      <td>https://www.eneuro.org/content/5/5/eneuro.0258...</td>\n",
       "      <td>M Meseke, F Neumüller, B Brunne, X Li, M Anstö...</td>\n",
       "      <td>HCN1 compartmentalization in CA1 pyramidal cel...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=50206...</td>\n",
       "      <td>8.0</td>\n",
       "      <td>https://www.eneuro.org/content/eneuro/5/5/ENEU...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3783</th>\n",
       "      <td>Distinct but overlapping roles of LRRTM1 and L...</td>\n",
       "      <td>https://elifesciences.org/articles/64742</td>\n",
       "      <td>SH Dhume, SA Connor, F Mills, PK Tari… - Elife...</td>\n",
       "      <td>LRRTMs are postsynaptic cell adhesion proteins...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=97818...</td>\n",
       "      <td>5.0</td>\n",
       "      <td>https://elifesciences.org/articles/64742.pdf</td>\n",
       "      <td>N127/31</td>\n",
       "      <td>Pan-SAPAP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3784</th>\n",
       "      <td>Synaptic protein interaction networks encode ...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>JD Lautz, KB Tsegay, Z Zhu, EP Gniffke, JP Wel...</td>\n",
       "      <td>A core network of widely expressed proteins wi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=12974...</td>\n",
       "      <td>6.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>N127/31</td>\n",
       "      <td>Pan-SAPAP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3785</th>\n",
       "      <td>Pannexin 2 protein expression is not restrict...</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>M Le Vasseur, J Lelowski, JF Bechberger… - Fro...</td>\n",
       "      <td>… by UC Davis/NIH NeuroMab Facility (Davis, CA...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=77804...</td>\n",
       "      <td>81.0</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>N121A/1</td>\n",
       "      <td>Pannexin-2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3786</th>\n",
       "      <td>The role of Pannexin 2 in mitochondrial functi...</td>\n",
       "      <td>https://open.library.ubc.ca/soa/cIRcle/collect...</td>\n",
       "      <td>M Le Vasseur - 2016 - open.library.ubc.ca</td>\n",
       "      <td>… Figure 2.1 Specificity of two novel anti–Pan...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://open.library.ubc.ca/media/download/pdf...</td>\n",
       "      <td>N121A/1</td>\n",
       "      <td>Pannexin-2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3787</th>\n",
       "      <td>Pannexin 2 localizes at ER-mitochondria contac...</td>\n",
       "      <td>https://www.mdpi.com/2072-6694/11/3/343</td>\n",
       "      <td>M Le Vasseur, VC Chen, K Huang, WA Vogl, CC Na...</td>\n",
       "      <td>Endomembrane specialization allows functional ...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=79444...</td>\n",
       "      <td>18.0</td>\n",
       "      <td>https://www.mdpi.com/2072-6694/11/3/343/htm</td>\n",
       "      <td>N121A/1</td>\n",
       "      <td>Pannexin-2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3788 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "0     Deubiquitylating enzyme USP2 counteracts Nedd4...   \n",
       "1     Caracterización de los canales de potasio depe...   \n",
       "2     Neuroinflammation alters integrative propertie...   \n",
       "3      Allostery between two binding sites in the io...   \n",
       "4     Distal dendritic enrichment of HCN1 channels i...   \n",
       "...                                                 ...   \n",
       "3783  Distinct but overlapping roles of LRRTM1 and L...   \n",
       "3784   Synaptic protein interaction networks encode ...   \n",
       "3785   Pannexin 2 protein expression is not restrict...   \n",
       "3786  The role of Pannexin 2 in mitochondrial functi...   \n",
       "3787  Pannexin 2 localizes at ER-mitochondria contac...   \n",
       "\n",
       "                                             title_link  \\\n",
       "0     https://www.sciencedirect.com/science/article/...   \n",
       "1                https://idus.us.es/handle/11441/138533   \n",
       "2     https://link.springer.com/article/10.1007/s120...   \n",
       "3     https://www.jbc.org/article/S0021-9258(20)3303...   \n",
       "4     https://www.eneuro.org/content/5/5/eneuro.0258...   \n",
       "...                                                 ...   \n",
       "3783           https://elifesciences.org/articles/64742   \n",
       "3784  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "3785  https://www.frontiersin.org/articles/10.3389/f...   \n",
       "3786  https://open.library.ubc.ca/soa/cIRcle/collect...   \n",
       "3787            https://www.mdpi.com/2072-6694/11/3/343   \n",
       "\n",
       "                                       publication_info  \\\n",
       "0     K Krzystanek, HB Rasmussen, M Grunnet, O Staub...   \n",
       "1                M Ortego Domínguez - 2022 - idus.us.es   \n",
       "2     F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...   \n",
       "3     KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...   \n",
       "4     M Meseke, F Neumüller, B Brunne, X Li, M Anstö...   \n",
       "...                                                 ...   \n",
       "3783  SH Dhume, SA Connor, F Mills, PK Tari… - Elife...   \n",
       "3784  JD Lautz, KB Tsegay, Z Zhu, EP Gniffke, JP Wel...   \n",
       "3785  M Le Vasseur, J Lelowski, JF Bechberger… - Fro...   \n",
       "3786          M Le Vasseur - 2016 - open.library.ubc.ca   \n",
       "3787  M Le Vasseur, VC Chen, K Huang, WA Vogl, CC Na...   \n",
       "\n",
       "                                                snippet  \\\n",
       "0     BACKGROUND: KCNQ1 (Kv7.1), together with its K...   \n",
       "1     Los adipocitos blancos son células que partici...   \n",
       "2     … , N212/17, Neuromab, CA, USA) antibodies usi...   \n",
       "3     Tetratricopeptide repeat (TPR) domains are ubi...   \n",
       "4     HCN1 compartmentalization in CA1 pyramidal cel...   \n",
       "...                                                 ...   \n",
       "3783  LRRTMs are postsynaptic cell adhesion proteins...   \n",
       "3784  A core network of widely expressed proteins wi...   \n",
       "3785  … by UC Davis/NIH NeuroMab Facility (Davis, CA...   \n",
       "3786  … Figure 2.1 Specificity of two novel anti–Pan...   \n",
       "3787  Endomembrane specialization allows functional ...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "0     https://scholar.google.com/scholar?cites=98014...            35.0   \n",
       "1                                                   NaN             NaN   \n",
       "2     https://scholar.google.com/scholar?cites=19097...            41.0   \n",
       "3     https://scholar.google.com/scholar?cites=14199...            12.0   \n",
       "4     https://scholar.google.com/scholar?cites=50206...             8.0   \n",
       "...                                                 ...             ...   \n",
       "3783  https://scholar.google.com/scholar?cites=97818...             5.0   \n",
       "3784  https://scholar.google.com/scholar?cites=12974...             6.0   \n",
       "3785  https://scholar.google.com/scholar?cites=77804...            81.0   \n",
       "3786                                                NaN             NaN   \n",
       "3787  https://scholar.google.com/scholar?cites=79444...            18.0   \n",
       "\n",
       "                                               pdf_file Antibody  \\\n",
       "0                                                   NaN  N37A/10   \n",
       "1     https://idus.us.es/bitstream/handle/11441/1385...  N375/67   \n",
       "2     https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...  N212/17   \n",
       "3     https://www.jbc.org/article/S0021-9258(20)3303...  N212/17   \n",
       "4     https://www.eneuro.org/content/eneuro/5/5/ENEU...  N212/17   \n",
       "...                                                 ...      ...   \n",
       "3783       https://elifesciences.org/articles/64742.pdf  N127/31   \n",
       "3784  https://www.cell.com/cell-reports/pdf/S2211-12...  N127/31   \n",
       "3785  https://www.frontiersin.org/articles/10.3389/f...  N121A/1   \n",
       "3786  https://open.library.ubc.ca/media/download/pdf...  N121A/1   \n",
       "3787        https://www.mdpi.com/2072-6694/11/3/343/htm  N121A/1   \n",
       "\n",
       "                      Target  \n",
       "0     Kv7.1/KCNQ1 K+ channel  \n",
       "1           Kv3.3 K+ channel  \n",
       "2          TRIP8b (constant)  \n",
       "3          TRIP8b (constant)  \n",
       "4          TRIP8b (constant)  \n",
       "...                      ...  \n",
       "3783               Pan-SAPAP  \n",
       "3784               Pan-SAPAP  \n",
       "3785              Pannexin-2  \n",
       "3786              Pannexin-2  \n",
       "3787              Pannexin-2  \n",
       "\n",
       "[3788 rows x 9 columns]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "allnmab_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "04b0efc5",
   "metadata": {},
   "outputs": [],
   "source": [
    "# get data file names - replace with your path for what i send\n",
    "import glob\n",
    "import pandas as pd\n",
    "\n",
    "path = r'/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test2'\n",
    "filenames = glob.glob(path + \"/*.csv\")\n",
    "\n",
    "dfs = []\n",
    "for filename in filenames:\n",
    "    dfs.append(pd.read_csv(filename))\n",
    "allnmab_df2 = pd.concat(dfs, ignore_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "74f1fa2b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K Krzystanek, HB Rasmussen, M Grunnet, O Staub...</td>\n",
       "      <td>BACKGROUND: KCNQ1 (Kv7.1), together with its K...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=98014...</td>\n",
       "      <td>35.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N37A/10</td>\n",
       "      <td>Kv7.1/KCNQ1 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Caracterización de los canales de potasio depe...</td>\n",
       "      <td>https://idus.us.es/handle/11441/138533</td>\n",
       "      <td>M Ortego Domínguez - 2022 - idus.us.es</td>\n",
       "      <td>Los adipocitos blancos son células que partici...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://idus.us.es/bitstream/handle/11441/1385...</td>\n",
       "      <td>N375/67</td>\n",
       "      <td>Kv3.3 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Neuroinflammation alters integrative propertie...</td>\n",
       "      <td>https://link.springer.com/article/10.1007/s120...</td>\n",
       "      <td>F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...</td>\n",
       "      <td>… , N212/17, Neuromab, CA, USA) antibodies usi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=19097...</td>\n",
       "      <td>41.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Allostery between two binding sites in the io...</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...</td>\n",
       "      <td>Tetratricopeptide repeat (TPR) domains are ubi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=14199...</td>\n",
       "      <td>13.0</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Distal dendritic enrichment of HCN1 channels i...</td>\n",
       "      <td>https://www.eneuro.org/content/5/5/eneuro.0258...</td>\n",
       "      <td>M Meseke, F Neumüller, B Brunne, X Li, M Anstö...</td>\n",
       "      <td>HCN1 compartmentalization in CA1 pyramidal cel...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=50206...</td>\n",
       "      <td>9.0</td>\n",
       "      <td>https://www.eneuro.org/content/eneuro/5/5/ENEU...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3483</th>\n",
       "      <td>Protein components of the backbone structure o...</td>\n",
       "      <td>https://soar-ir.repo.nii.ac.jp/?action=reposit...</td>\n",
       "      <td>樋口京一， 竹下敏一 - soar-ir.repo.nii.ac.jp</td>\n",
       "      <td>(論文の内容の要旨)[背景と目的] シナプス後肥厚部 (postsynaptic densi...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://soar-ir.repo.nii.ac.jp/?action=reposit...</td>\n",
       "      <td>N127/31</td>\n",
       "      <td>Pan-SAPAP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3484</th>\n",
       "      <td>Protein components of the backbone structure o...</td>\n",
       "      <td>https://soar-ir.repo.nii.ac.jp/?action=reposit...</td>\n",
       "      <td>郭維恒 - 2017 - soar-ir.repo.nii.ac.jp</td>\n",
       "      <td>(論文の内容の要旨)[背景と目的] シナプス後肥厚部 (postsynaptic densi...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://soar-ir.repo.nii.ac.jp/record/19942/fi...</td>\n",
       "      <td>N127/31</td>\n",
       "      <td>Pan-SAPAP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3485</th>\n",
       "      <td>Pannexin 2 protein expression is not restrict...</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>M Le Vasseur, J Lelowski, JF Bechberger… - Fro...</td>\n",
       "      <td>… by UC Davis/NIH NeuroMab Facility (Davis, CA...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=77804...</td>\n",
       "      <td>84.0</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>N121A/1</td>\n",
       "      <td>Pannexin-2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3486</th>\n",
       "      <td>The role of Pannexin 2 in mitochondrial functi...</td>\n",
       "      <td>https://open.library.ubc.ca/soa/cIRcle/collect...</td>\n",
       "      <td>M Le Vasseur - 2016 - open.library.ubc.ca</td>\n",
       "      <td>… Figure 2.1 Specificity of two novel anti–Pan...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://open.library.ubc.ca/media/download/pdf...</td>\n",
       "      <td>N121A/1</td>\n",
       "      <td>Pannexin-2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3487</th>\n",
       "      <td>Pannexin 2 localizes at ER-mitochondria contac...</td>\n",
       "      <td>https://www.mdpi.com/2072-6694/11/3/343</td>\n",
       "      <td>M Le Vasseur, VC Chen, K Huang, WA Vogl, CC Na...</td>\n",
       "      <td>Endomembrane specialization allows functional ...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=79444...</td>\n",
       "      <td>21.0</td>\n",
       "      <td>https://www.mdpi.com/2072-6694/11/3/343/htm</td>\n",
       "      <td>N121A/1</td>\n",
       "      <td>Pannexin-2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3488 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "0     Deubiquitylating enzyme USP2 counteracts Nedd4...   \n",
       "1     Caracterización de los canales de potasio depe...   \n",
       "2     Neuroinflammation alters integrative propertie...   \n",
       "3      Allostery between two binding sites in the io...   \n",
       "4     Distal dendritic enrichment of HCN1 channels i...   \n",
       "...                                                 ...   \n",
       "3483  Protein components of the backbone structure o...   \n",
       "3484  Protein components of the backbone structure o...   \n",
       "3485   Pannexin 2 protein expression is not restrict...   \n",
       "3486  The role of Pannexin 2 in mitochondrial functi...   \n",
       "3487  Pannexin 2 localizes at ER-mitochondria contac...   \n",
       "\n",
       "                                             title_link  \\\n",
       "0     https://www.sciencedirect.com/science/article/...   \n",
       "1                https://idus.us.es/handle/11441/138533   \n",
       "2     https://link.springer.com/article/10.1007/s120...   \n",
       "3     https://www.jbc.org/article/S0021-9258(20)3303...   \n",
       "4     https://www.eneuro.org/content/5/5/eneuro.0258...   \n",
       "...                                                 ...   \n",
       "3483  https://soar-ir.repo.nii.ac.jp/?action=reposit...   \n",
       "3484  https://soar-ir.repo.nii.ac.jp/?action=reposit...   \n",
       "3485  https://www.frontiersin.org/articles/10.3389/f...   \n",
       "3486  https://open.library.ubc.ca/soa/cIRcle/collect...   \n",
       "3487            https://www.mdpi.com/2072-6694/11/3/343   \n",
       "\n",
       "                                       publication_info  \\\n",
       "0     K Krzystanek, HB Rasmussen, M Grunnet, O Staub...   \n",
       "1                M Ortego Domínguez - 2022 - idus.us.es   \n",
       "2     F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...   \n",
       "3     KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...   \n",
       "4     M Meseke, F Neumüller, B Brunne, X Li, M Anstö...   \n",
       "...                                                 ...   \n",
       "3483                樋口京一， 竹下敏一 - soar-ir.repo.nii.ac.jp   \n",
       "3484                郭維恒 - 2017 - soar-ir.repo.nii.ac.jp   \n",
       "3485  M Le Vasseur, J Lelowski, JF Bechberger… - Fro...   \n",
       "3486          M Le Vasseur - 2016 - open.library.ubc.ca   \n",
       "3487  M Le Vasseur, VC Chen, K Huang, WA Vogl, CC Na...   \n",
       "\n",
       "                                                snippet  \\\n",
       "0     BACKGROUND: KCNQ1 (Kv7.1), together with its K...   \n",
       "1     Los adipocitos blancos son células que partici...   \n",
       "2     … , N212/17, Neuromab, CA, USA) antibodies usi...   \n",
       "3     Tetratricopeptide repeat (TPR) domains are ubi...   \n",
       "4     HCN1 compartmentalization in CA1 pyramidal cel...   \n",
       "...                                                 ...   \n",
       "3483  (論文の内容の要旨)[背景と目的] シナプス後肥厚部 (postsynaptic densi...   \n",
       "3484  (論文の内容の要旨)[背景と目的] シナプス後肥厚部 (postsynaptic densi...   \n",
       "3485  … by UC Davis/NIH NeuroMab Facility (Davis, CA...   \n",
       "3486  … Figure 2.1 Specificity of two novel anti–Pan...   \n",
       "3487  Endomembrane specialization allows functional ...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "0     https://scholar.google.com/scholar?cites=98014...            35.0   \n",
       "1                                                   NaN             NaN   \n",
       "2     https://scholar.google.com/scholar?cites=19097...            41.0   \n",
       "3     https://scholar.google.com/scholar?cites=14199...            13.0   \n",
       "4     https://scholar.google.com/scholar?cites=50206...             9.0   \n",
       "...                                                 ...             ...   \n",
       "3483                                                NaN             NaN   \n",
       "3484                                                NaN             NaN   \n",
       "3485  https://scholar.google.com/scholar?cites=77804...            84.0   \n",
       "3486                                                NaN             NaN   \n",
       "3487  https://scholar.google.com/scholar?cites=79444...            21.0   \n",
       "\n",
       "                                               pdf_file Antibody  \\\n",
       "0                                                   NaN  N37A/10   \n",
       "1     https://idus.us.es/bitstream/handle/11441/1385...  N375/67   \n",
       "2     https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...  N212/17   \n",
       "3     https://www.jbc.org/article/S0021-9258(20)3303...  N212/17   \n",
       "4     https://www.eneuro.org/content/eneuro/5/5/ENEU...  N212/17   \n",
       "...                                                 ...      ...   \n",
       "3483  https://soar-ir.repo.nii.ac.jp/?action=reposit...  N127/31   \n",
       "3484  https://soar-ir.repo.nii.ac.jp/record/19942/fi...  N127/31   \n",
       "3485  https://www.frontiersin.org/articles/10.3389/f...  N121A/1   \n",
       "3486  https://open.library.ubc.ca/media/download/pdf...  N121A/1   \n",
       "3487        https://www.mdpi.com/2072-6694/11/3/343/htm  N121A/1   \n",
       "\n",
       "                      Target  \n",
       "0     Kv7.1/KCNQ1 K+ channel  \n",
       "1           Kv3.3 K+ channel  \n",
       "2          TRIP8b (constant)  \n",
       "3          TRIP8b (constant)  \n",
       "4          TRIP8b (constant)  \n",
       "...                      ...  \n",
       "3483               Pan-SAPAP  \n",
       "3484               Pan-SAPAP  \n",
       "3485              Pannexin-2  \n",
       "3486              Pannexin-2  \n",
       "3487              Pannexin-2  \n",
       "\n",
       "[3488 rows x 9 columns]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "allnmab_df2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "e3041620",
   "metadata": {},
   "outputs": [],
   "source": [
    "# get data file names - replace with your path for what i send\n",
    "import glob\n",
    "import pandas as pd\n",
    "\n",
    "path = r'/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/new_test3'\n",
    "filenames = glob.glob(path + \"/*.csv\")\n",
    "\n",
    "dfs = []\n",
    "for filename in filenames:\n",
    "    dfs.append(pd.read_csv(filename))\n",
    "allnmab_df3 = pd.concat(dfs, ignore_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "7f3b08b4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Characterization of hyperpolarization-activate...</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>KA Lyman, Y Han, AP Robinson… - Frontiers in C...</td>\n",
       "      <td>Mature oligodendrocytes (OLG) are the myelin-f...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A complex of Protocadherin-19 and N-cadherin m...</td>\n",
       "      <td>https://rupress.org/jcb/article-abstract/195/7...</td>\n",
       "      <td>MR Emond, S Biswas, CJ Blevins, JD Jontes - Jo...</td>\n",
       "      <td>… N144/14; University of California Davis/Nati...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=11570...</td>\n",
       "      <td>126.0</td>\n",
       "      <td>https://rupress.org/jcb/article-pdf/195/7/1115...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Cdk5-mediated phosphorylation of CRMP-2 enhan...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>JM Brittain, Y Wang, O Eruvwetere, R Khanna - ...</td>\n",
       "      <td>… Lysates were incubated with CRMP-2-6xHis, fo...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=16563...</td>\n",
       "      <td>84.0</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Reprogramming protein kinase substrate specifi...</td>\n",
       "      <td>https://www.biorxiv.org/content/10.1101/091892...</td>\n",
       "      <td>JM Lubner, GM Church, MF Chou, D Schwartz - bi...</td>\n",
       "      <td>Protein kinase specificity is largely imparted...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=11711...</td>\n",
       "      <td>5.0</td>\n",
       "      <td>https://www.biorxiv.org/content/10.1101/091892...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>α-synuclein promotes neuronal dysfunction and ...</td>\n",
       "      <td>https://www.jneurosci.org/content/43/9/1614.ab...</td>\n",
       "      <td>G Maor, RR Dubreuil, MB Feany - Journal of Neu...</td>\n",
       "      <td>α-Synuclein plays a key role in the pathogenes...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=25347...</td>\n",
       "      <td>7.0</td>\n",
       "      <td>https://www.jneurosci.org/content/jneuro/43/9/...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1440</th>\n",
       "      <td>Dissecting The Tissue-Specific Contributions T...</td>\n",
       "      <td>https://scholar.smu.edu/hum_sci_biologicalscie...</td>\n",
       "      <td>K Paulhus - 2024 - scholar.smu.edu</td>\n",
       "      <td>Sudden unexpected death in epilepsy (SUDEP), t...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://scholar.smu.edu/cgi/viewcontent.cgi?ar...</td>\n",
       "      <td>K20/78</td>\n",
       "      <td>Kv1.1 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1441</th>\n",
       "      <td>Impact of Elevated Brain IL-6 in Transgenic Mi...</td>\n",
       "      <td>https://www.mdpi.com/2073-4409/12/18/2306</td>\n",
       "      <td>DL Gruol, D Calderon, S Huitron-Resendiz… - Ce...</td>\n",
       "      <td>… 500, UC Davis/NIH NeuroMab Facility); a mono...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=45817...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.mdpi.com/2073-4409/12/18/2306/pdf</td>\n",
       "      <td>L118/80</td>\n",
       "      <td>VGAT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1442</th>\n",
       "      <td>Rescue of deficits by Brwd1 copy number restor...</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...</td>\n",
       "      <td>With an incidence of ~1 in 800 births, Down sy...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=44492...</td>\n",
       "      <td>9.0</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1443</th>\n",
       "      <td>Identification of Novel Cell-Type Specific Epi...</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>SL Fulton - 2022 - search.proquest.com</td>\n",
       "      <td>The overarching goal of this thesis work has b...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1444</th>\n",
       "      <td>Domain and cell type-specific immunolocalisat...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>B Otuyemi, T Jackson, R Ma, AR Monteiro… - Jou...</td>\n",
       "      <td>Diverse classes of voltage-gated potassium cha...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=15314...</td>\n",
       "      <td>5.0</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K57/1</td>\n",
       "      <td>Kv4.2 K+ channel (external)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1445 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "0     Characterization of hyperpolarization-activate...   \n",
       "1     A complex of Protocadherin-19 and N-cadherin m...   \n",
       "2      Cdk5-mediated phosphorylation of CRMP-2 enhan...   \n",
       "3     Reprogramming protein kinase substrate specifi...   \n",
       "4     α-synuclein promotes neuronal dysfunction and ...   \n",
       "...                                                 ...   \n",
       "1440  Dissecting The Tissue-Specific Contributions T...   \n",
       "1441  Impact of Elevated Brain IL-6 in Transgenic Mi...   \n",
       "1442  Rescue of deficits by Brwd1 copy number restor...   \n",
       "1443  Identification of Novel Cell-Type Specific Epi...   \n",
       "1444   Domain and cell type-specific immunolocalisat...   \n",
       "\n",
       "                                             title_link  \\\n",
       "0     https://www.frontiersin.org/articles/10.3389/f...   \n",
       "1     https://rupress.org/jcb/article-abstract/195/7...   \n",
       "2     https://www.sciencedirect.com/science/article/...   \n",
       "3     https://www.biorxiv.org/content/10.1101/091892...   \n",
       "4     https://www.jneurosci.org/content/43/9/1614.ab...   \n",
       "...                                                 ...   \n",
       "1440  https://scholar.smu.edu/hum_sci_biologicalscie...   \n",
       "1441          https://www.mdpi.com/2073-4409/12/18/2306   \n",
       "1442  https://www.nature.com/articles/s41467-022-342...   \n",
       "1443  https://search.proquest.com/openview/88710f293...   \n",
       "1444  https://www.sciencedirect.com/science/article/...   \n",
       "\n",
       "                                       publication_info  \\\n",
       "0     KA Lyman, Y Han, AP Robinson… - Frontiers in C...   \n",
       "1     MR Emond, S Biswas, CJ Blevins, JD Jontes - Jo...   \n",
       "2     JM Brittain, Y Wang, O Eruvwetere, R Khanna - ...   \n",
       "3     JM Lubner, GM Church, MF Chou, D Schwartz - bi...   \n",
       "4     G Maor, RR Dubreuil, MB Feany - Journal of Neu...   \n",
       "...                                                 ...   \n",
       "1440                 K Paulhus - 2024 - scholar.smu.edu   \n",
       "1441  DL Gruol, D Calderon, S Huitron-Resendiz… - Ce...   \n",
       "1442  SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...   \n",
       "1443             SL Fulton - 2022 - search.proquest.com   \n",
       "1444  B Otuyemi, T Jackson, R Ma, AR Monteiro… - Jou...   \n",
       "\n",
       "                                                snippet  \\\n",
       "0     Mature oligodendrocytes (OLG) are the myelin-f...   \n",
       "1     … N144/14; University of California Davis/Nati...   \n",
       "2     … Lysates were incubated with CRMP-2-6xHis, fo...   \n",
       "3     Protein kinase specificity is largely imparted...   \n",
       "4     α-Synuclein plays a key role in the pathogenes...   \n",
       "...                                                 ...   \n",
       "1440  Sudden unexpected death in epilepsy (SUDEP), t...   \n",
       "1441  … 500, UC Davis/NIH NeuroMab Facility); a mono...   \n",
       "1442  With an incidence of ~1 in 800 births, Down sy...   \n",
       "1443  The overarching goal of this thesis work has b...   \n",
       "1444  Diverse classes of voltage-gated potassium cha...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "0                                                   NaN             NaN   \n",
       "1     https://scholar.google.com/scholar?cites=11570...           126.0   \n",
       "2     https://scholar.google.com/scholar?cites=16563...            84.0   \n",
       "3     https://scholar.google.com/scholar?cites=11711...             5.0   \n",
       "4     https://scholar.google.com/scholar?cites=25347...             7.0   \n",
       "...                                                 ...             ...   \n",
       "1440                                                NaN             NaN   \n",
       "1441  https://scholar.google.com/scholar?cites=45817...             1.0   \n",
       "1442  https://scholar.google.com/scholar?cites=44492...             9.0   \n",
       "1443                                                NaN             NaN   \n",
       "1444  https://scholar.google.com/scholar?cites=15314...             5.0   \n",
       "\n",
       "                                               pdf_file  Antibody  \\\n",
       "0     https://www.frontiersin.org/articles/10.3389/f...   N212/17   \n",
       "1     https://rupress.org/jcb/article-pdf/195/7/1115...   N144/14   \n",
       "2     https://www.sciencedirect.com/science/article/...   N144/14   \n",
       "3     https://www.biorxiv.org/content/10.1101/091892...   N144/14   \n",
       "4     https://www.jneurosci.org/content/jneuro/43/9/...   N144/14   \n",
       "...                                                 ...       ...   \n",
       "1440  https://scholar.smu.edu/cgi/viewcontent.cgi?ar...    K20/78   \n",
       "1441      https://www.mdpi.com/2073-4409/12/18/2306/pdf   L118/80   \n",
       "1442  https://www.nature.com/articles/s41467-022-342...  N336B/83   \n",
       "1443  https://search.proquest.com/openview/88710f293...  N336B/83   \n",
       "1444  https://www.sciencedirect.com/science/article/...     K57/1   \n",
       "\n",
       "                           Target  \n",
       "0               TRIP8b (constant)  \n",
       "1                           6xHis  \n",
       "2                           6xHis  \n",
       "3                           6xHis  \n",
       "4                           6xHis  \n",
       "...                           ...  \n",
       "1440             Kv1.1 K+ channel  \n",
       "1441                         VGAT  \n",
       "1442                       BAF53a  \n",
       "1443                       BAF53a  \n",
       "1444  Kv4.2 K+ channel (external)  \n",
       "\n",
       "[1445 rows x 9 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "allnmab_df3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "930bdc6d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K Krzystanek, HB Rasmussen, M Grunnet, O Staub...</td>\n",
       "      <td>BACKGROUND: KCNQ1 (Kv7.1), together with its K...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=98014...</td>\n",
       "      <td>35.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N37A/10</td>\n",
       "      <td>Kv7.1/KCNQ1 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Caracterización de los canales de potasio depe...</td>\n",
       "      <td>https://idus.us.es/handle/11441/138533</td>\n",
       "      <td>M Ortego Domínguez - 2022 - idus.us.es</td>\n",
       "      <td>Los adipocitos blancos son células que partici...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://idus.us.es/bitstream/handle/11441/1385...</td>\n",
       "      <td>N375/67</td>\n",
       "      <td>Kv3.3 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Neuroinflammation alters integrative propertie...</td>\n",
       "      <td>https://link.springer.com/article/10.1007/s120...</td>\n",
       "      <td>F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...</td>\n",
       "      <td>… , N212/17, Neuromab, CA, USA) antibodies usi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=19097...</td>\n",
       "      <td>41.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Allostery between two binding sites in the io...</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...</td>\n",
       "      <td>Tetratricopeptide repeat (TPR) domains are ubi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=14199...</td>\n",
       "      <td>12.0</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Distal dendritic enrichment of HCN1 channels i...</td>\n",
       "      <td>https://www.eneuro.org/content/5/5/eneuro.0258...</td>\n",
       "      <td>M Meseke, F Neumüller, B Brunne, X Li, M Anstö...</td>\n",
       "      <td>HCN1 compartmentalization in CA1 pyramidal cel...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=50206...</td>\n",
       "      <td>8.0</td>\n",
       "      <td>https://www.eneuro.org/content/eneuro/5/5/ENEU...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8716</th>\n",
       "      <td>Dissecting The Tissue-Specific Contributions T...</td>\n",
       "      <td>https://scholar.smu.edu/hum_sci_biologicalscie...</td>\n",
       "      <td>K Paulhus - 2024 - scholar.smu.edu</td>\n",
       "      <td>Sudden unexpected death in epilepsy (SUDEP), t...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://scholar.smu.edu/cgi/viewcontent.cgi?ar...</td>\n",
       "      <td>K20/78</td>\n",
       "      <td>Kv1.1 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8717</th>\n",
       "      <td>Impact of Elevated Brain IL-6 in Transgenic Mi...</td>\n",
       "      <td>https://www.mdpi.com/2073-4409/12/18/2306</td>\n",
       "      <td>DL Gruol, D Calderon, S Huitron-Resendiz… - Ce...</td>\n",
       "      <td>… 500, UC Davis/NIH NeuroMab Facility); a mono...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=45817...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.mdpi.com/2073-4409/12/18/2306/pdf</td>\n",
       "      <td>L118/80</td>\n",
       "      <td>VGAT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8718</th>\n",
       "      <td>Rescue of deficits by Brwd1 copy number restor...</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...</td>\n",
       "      <td>With an incidence of ~1 in 800 births, Down sy...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=44492...</td>\n",
       "      <td>9.0</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8719</th>\n",
       "      <td>Identification of Novel Cell-Type Specific Epi...</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>SL Fulton - 2022 - search.proquest.com</td>\n",
       "      <td>The overarching goal of this thesis work has b...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8720</th>\n",
       "      <td>Domain and cell type-specific immunolocalisat...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>B Otuyemi, T Jackson, R Ma, AR Monteiro… - Jou...</td>\n",
       "      <td>Diverse classes of voltage-gated potassium cha...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=15314...</td>\n",
       "      <td>5.0</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K57/1</td>\n",
       "      <td>Kv4.2 K+ channel (external)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>8721 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "0     Deubiquitylating enzyme USP2 counteracts Nedd4...   \n",
       "1     Caracterización de los canales de potasio depe...   \n",
       "2     Neuroinflammation alters integrative propertie...   \n",
       "3      Allostery between two binding sites in the io...   \n",
       "4     Distal dendritic enrichment of HCN1 channels i...   \n",
       "...                                                 ...   \n",
       "8716  Dissecting The Tissue-Specific Contributions T...   \n",
       "8717  Impact of Elevated Brain IL-6 in Transgenic Mi...   \n",
       "8718  Rescue of deficits by Brwd1 copy number restor...   \n",
       "8719  Identification of Novel Cell-Type Specific Epi...   \n",
       "8720   Domain and cell type-specific immunolocalisat...   \n",
       "\n",
       "                                             title_link  \\\n",
       "0     https://www.sciencedirect.com/science/article/...   \n",
       "1                https://idus.us.es/handle/11441/138533   \n",
       "2     https://link.springer.com/article/10.1007/s120...   \n",
       "3     https://www.jbc.org/article/S0021-9258(20)3303...   \n",
       "4     https://www.eneuro.org/content/5/5/eneuro.0258...   \n",
       "...                                                 ...   \n",
       "8716  https://scholar.smu.edu/hum_sci_biologicalscie...   \n",
       "8717          https://www.mdpi.com/2073-4409/12/18/2306   \n",
       "8718  https://www.nature.com/articles/s41467-022-342...   \n",
       "8719  https://search.proquest.com/openview/88710f293...   \n",
       "8720  https://www.sciencedirect.com/science/article/...   \n",
       "\n",
       "                                       publication_info  \\\n",
       "0     K Krzystanek, HB Rasmussen, M Grunnet, O Staub...   \n",
       "1                M Ortego Domínguez - 2022 - idus.us.es   \n",
       "2     F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...   \n",
       "3     KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...   \n",
       "4     M Meseke, F Neumüller, B Brunne, X Li, M Anstö...   \n",
       "...                                                 ...   \n",
       "8716                 K Paulhus - 2024 - scholar.smu.edu   \n",
       "8717  DL Gruol, D Calderon, S Huitron-Resendiz… - Ce...   \n",
       "8718  SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...   \n",
       "8719             SL Fulton - 2022 - search.proquest.com   \n",
       "8720  B Otuyemi, T Jackson, R Ma, AR Monteiro… - Jou...   \n",
       "\n",
       "                                                snippet  \\\n",
       "0     BACKGROUND: KCNQ1 (Kv7.1), together with its K...   \n",
       "1     Los adipocitos blancos son células que partici...   \n",
       "2     … , N212/17, Neuromab, CA, USA) antibodies usi...   \n",
       "3     Tetratricopeptide repeat (TPR) domains are ubi...   \n",
       "4     HCN1 compartmentalization in CA1 pyramidal cel...   \n",
       "...                                                 ...   \n",
       "8716  Sudden unexpected death in epilepsy (SUDEP), t...   \n",
       "8717  … 500, UC Davis/NIH NeuroMab Facility); a mono...   \n",
       "8718  With an incidence of ~1 in 800 births, Down sy...   \n",
       "8719  The overarching goal of this thesis work has b...   \n",
       "8720  Diverse classes of voltage-gated potassium cha...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "0     https://scholar.google.com/scholar?cites=98014...            35.0   \n",
       "1                                                   NaN             NaN   \n",
       "2     https://scholar.google.com/scholar?cites=19097...            41.0   \n",
       "3     https://scholar.google.com/scholar?cites=14199...            12.0   \n",
       "4     https://scholar.google.com/scholar?cites=50206...             8.0   \n",
       "...                                                 ...             ...   \n",
       "8716                                                NaN             NaN   \n",
       "8717  https://scholar.google.com/scholar?cites=45817...             1.0   \n",
       "8718  https://scholar.google.com/scholar?cites=44492...             9.0   \n",
       "8719                                                NaN             NaN   \n",
       "8720  https://scholar.google.com/scholar?cites=15314...             5.0   \n",
       "\n",
       "                                               pdf_file  Antibody  \\\n",
       "0                                                   NaN   N37A/10   \n",
       "1     https://idus.us.es/bitstream/handle/11441/1385...   N375/67   \n",
       "2     https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   N212/17   \n",
       "3     https://www.jbc.org/article/S0021-9258(20)3303...   N212/17   \n",
       "4     https://www.eneuro.org/content/eneuro/5/5/ENEU...   N212/17   \n",
       "...                                                 ...       ...   \n",
       "8716  https://scholar.smu.edu/cgi/viewcontent.cgi?ar...    K20/78   \n",
       "8717      https://www.mdpi.com/2073-4409/12/18/2306/pdf   L118/80   \n",
       "8718  https://www.nature.com/articles/s41467-022-342...  N336B/83   \n",
       "8719  https://search.proquest.com/openview/88710f293...  N336B/83   \n",
       "8720  https://www.sciencedirect.com/science/article/...     K57/1   \n",
       "\n",
       "                           Target  \n",
       "0          Kv7.1/KCNQ1 K+ channel  \n",
       "1                Kv3.3 K+ channel  \n",
       "2               TRIP8b (constant)  \n",
       "3               TRIP8b (constant)  \n",
       "4               TRIP8b (constant)  \n",
       "...                           ...  \n",
       "8716             Kv1.1 K+ channel  \n",
       "8717                         VGAT  \n",
       "8718                       BAF53a  \n",
       "8719                       BAF53a  \n",
       "8720  Kv4.2 K+ channel (external)  \n",
       "\n",
       "[8721 rows x 9 columns]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# merge the dataframes and drop duplicates\n",
    "df_all = pd.concat([allnmab_df, allnmab_df2, allnmab_df3], ignore_index=True)\n",
    "df_all"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "4297a932",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K Krzystanek, HB Rasmussen, M Grunnet, O Staub...</td>\n",
       "      <td>BACKGROUND: KCNQ1 (Kv7.1), together with its K...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=98014...</td>\n",
       "      <td>35.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N37A/10</td>\n",
       "      <td>Kv7.1/KCNQ1 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Caracterización de los canales de potasio depe...</td>\n",
       "      <td>https://idus.us.es/handle/11441/138533</td>\n",
       "      <td>M Ortego Domínguez - 2022 - idus.us.es</td>\n",
       "      <td>Los adipocitos blancos son células que partici...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://idus.us.es/bitstream/handle/11441/1385...</td>\n",
       "      <td>N375/67, L71/5, L23/27, K57/1</td>\n",
       "      <td>Kv1.4 K+ channel (extracellular), Kv1.3 K+ cha...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Neuroinflammation alters integrative propertie...</td>\n",
       "      <td>https://link.springer.com/article/10.1007/s120...</td>\n",
       "      <td>F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...</td>\n",
       "      <td>… , N212/17, Neuromab, CA, USA) antibodies usi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=19097...</td>\n",
       "      <td>41.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Allostery between two binding sites in the io...</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...</td>\n",
       "      <td>Tetratricopeptide repeat (TPR) domains are ubi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=14199...</td>\n",
       "      <td>12.0</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Distal dendritic enrichment of HCN1 channels i...</td>\n",
       "      <td>https://www.eneuro.org/content/5/5/eneuro.0258...</td>\n",
       "      <td>M Meseke, F Neumüller, B Brunne, X Li, M Anstö...</td>\n",
       "      <td>HCN1 compartmentalization in CA1 pyramidal cel...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=50206...</td>\n",
       "      <td>8.0</td>\n",
       "      <td>https://www.eneuro.org/content/eneuro/5/5/ENEU...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8705</th>\n",
       "      <td>Interleukin-6 secretion by astrocytes is dynam...</td>\n",
       "      <td>https://journals.plos.org/plosone/article?id=1...</td>\n",
       "      <td>S Codeluppi, T Fernandez-Zafra, K Sandor, J Kj...</td>\n",
       "      <td>After contusion spinal cord injury (SCI), astr...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=10368...</td>\n",
       "      <td>48.0</td>\n",
       "      <td>https://journals.plos.org/plosone/article/file...</td>\n",
       "      <td>N103/31</td>\n",
       "      <td>Aldh1L1 (staining)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8706</th>\n",
       "      <td>Pathogenic mechanism of the FIG4 mutation resp...</td>\n",
       "      <td>https://journals.plos.org/plosgenetics/article...</td>\n",
       "      <td>GM Lenk, CJ Ferguson, CY Chow, N Jin… - PLoS …...</td>\n",
       "      <td>… The monoclonal antibody was generated using ...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=12174...</td>\n",
       "      <td>129.0</td>\n",
       "      <td>https://journals.plos.org/plosgenetics/article...</td>\n",
       "      <td>N202/7</td>\n",
       "      <td>Fig4/Sac3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8716</th>\n",
       "      <td>Dissecting The Tissue-Specific Contributions T...</td>\n",
       "      <td>https://scholar.smu.edu/hum_sci_biologicalscie...</td>\n",
       "      <td>K Paulhus - 2024 - scholar.smu.edu</td>\n",
       "      <td>Sudden unexpected death in epilepsy (SUDEP), t...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://scholar.smu.edu/cgi/viewcontent.cgi?ar...</td>\n",
       "      <td>K20/78</td>\n",
       "      <td>Kv1.1 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8718</th>\n",
       "      <td>Rescue of deficits by Brwd1 copy number restor...</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...</td>\n",
       "      <td>With an incidence of ~1 in 800 births, Down sy...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=44492...</td>\n",
       "      <td>9.0</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8719</th>\n",
       "      <td>Identification of Novel Cell-Type Specific Epi...</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>SL Fulton - 2022 - search.proquest.com</td>\n",
       "      <td>The overarching goal of this thesis work has b...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2830 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "0     Deubiquitylating enzyme USP2 counteracts Nedd4...   \n",
       "1     Caracterización de los canales de potasio depe...   \n",
       "2     Neuroinflammation alters integrative propertie...   \n",
       "3      Allostery between two binding sites in the io...   \n",
       "4     Distal dendritic enrichment of HCN1 channels i...   \n",
       "...                                                 ...   \n",
       "8705  Interleukin-6 secretion by astrocytes is dynam...   \n",
       "8706  Pathogenic mechanism of the FIG4 mutation resp...   \n",
       "8716  Dissecting The Tissue-Specific Contributions T...   \n",
       "8718  Rescue of deficits by Brwd1 copy number restor...   \n",
       "8719  Identification of Novel Cell-Type Specific Epi...   \n",
       "\n",
       "                                             title_link  \\\n",
       "0     https://www.sciencedirect.com/science/article/...   \n",
       "1                https://idus.us.es/handle/11441/138533   \n",
       "2     https://link.springer.com/article/10.1007/s120...   \n",
       "3     https://www.jbc.org/article/S0021-9258(20)3303...   \n",
       "4     https://www.eneuro.org/content/5/5/eneuro.0258...   \n",
       "...                                                 ...   \n",
       "8705  https://journals.plos.org/plosone/article?id=1...   \n",
       "8706  https://journals.plos.org/plosgenetics/article...   \n",
       "8716  https://scholar.smu.edu/hum_sci_biologicalscie...   \n",
       "8718  https://www.nature.com/articles/s41467-022-342...   \n",
       "8719  https://search.proquest.com/openview/88710f293...   \n",
       "\n",
       "                                       publication_info  \\\n",
       "0     K Krzystanek, HB Rasmussen, M Grunnet, O Staub...   \n",
       "1                M Ortego Domínguez - 2022 - idus.us.es   \n",
       "2     F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...   \n",
       "3     KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...   \n",
       "4     M Meseke, F Neumüller, B Brunne, X Li, M Anstö...   \n",
       "...                                                 ...   \n",
       "8705  S Codeluppi, T Fernandez-Zafra, K Sandor, J Kj...   \n",
       "8706  GM Lenk, CJ Ferguson, CY Chow, N Jin… - PLoS …...   \n",
       "8716                 K Paulhus - 2024 - scholar.smu.edu   \n",
       "8718  SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...   \n",
       "8719             SL Fulton - 2022 - search.proquest.com   \n",
       "\n",
       "                                                snippet  \\\n",
       "0     BACKGROUND: KCNQ1 (Kv7.1), together with its K...   \n",
       "1     Los adipocitos blancos son células que partici...   \n",
       "2     … , N212/17, Neuromab, CA, USA) antibodies usi...   \n",
       "3     Tetratricopeptide repeat (TPR) domains are ubi...   \n",
       "4     HCN1 compartmentalization in CA1 pyramidal cel...   \n",
       "...                                                 ...   \n",
       "8705  After contusion spinal cord injury (SCI), astr...   \n",
       "8706  … The monoclonal antibody was generated using ...   \n",
       "8716  Sudden unexpected death in epilepsy (SUDEP), t...   \n",
       "8718  With an incidence of ~1 in 800 births, Down sy...   \n",
       "8719  The overarching goal of this thesis work has b...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "0     https://scholar.google.com/scholar?cites=98014...            35.0   \n",
       "1                                                   NaN             NaN   \n",
       "2     https://scholar.google.com/scholar?cites=19097...            41.0   \n",
       "3     https://scholar.google.com/scholar?cites=14199...            12.0   \n",
       "4     https://scholar.google.com/scholar?cites=50206...             8.0   \n",
       "...                                                 ...             ...   \n",
       "8705  https://scholar.google.com/scholar?cites=10368...            48.0   \n",
       "8706  https://scholar.google.com/scholar?cites=12174...           129.0   \n",
       "8716                                                NaN             NaN   \n",
       "8718  https://scholar.google.com/scholar?cites=44492...             9.0   \n",
       "8719                                                NaN             NaN   \n",
       "\n",
       "                                               pdf_file  \\\n",
       "0                                                   NaN   \n",
       "1     https://idus.us.es/bitstream/handle/11441/1385...   \n",
       "2     https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "3     https://www.jbc.org/article/S0021-9258(20)3303...   \n",
       "4     https://www.eneuro.org/content/eneuro/5/5/ENEU...   \n",
       "...                                                 ...   \n",
       "8705  https://journals.plos.org/plosone/article/file...   \n",
       "8706  https://journals.plos.org/plosgenetics/article...   \n",
       "8716  https://scholar.smu.edu/cgi/viewcontent.cgi?ar...   \n",
       "8718  https://www.nature.com/articles/s41467-022-342...   \n",
       "8719  https://search.proquest.com/openview/88710f293...   \n",
       "\n",
       "                           Antibody  \\\n",
       "0                           N37A/10   \n",
       "1     N375/67, L71/5, L23/27, K57/1   \n",
       "2                           N212/17   \n",
       "3                           N212/17   \n",
       "4                           N212/17   \n",
       "...                             ...   \n",
       "8705                        N103/31   \n",
       "8706                         N202/7   \n",
       "8716                         K20/78   \n",
       "8718                       N336B/83   \n",
       "8719                       N336B/83   \n",
       "\n",
       "                                                 Target  \n",
       "0                                Kv7.1/KCNQ1 K+ channel  \n",
       "1     Kv1.4 K+ channel (extracellular), Kv1.3 K+ cha...  \n",
       "2                                     TRIP8b (constant)  \n",
       "3                                     TRIP8b (constant)  \n",
       "4                                     TRIP8b (constant)  \n",
       "...                                                 ...  \n",
       "8705                                 Aldh1L1 (staining)  \n",
       "8706                                          Fig4/Sac3  \n",
       "8716                                   Kv1.1 K+ channel  \n",
       "8718                                             BAF53a  \n",
       "8719                                             BAF53a  \n",
       "\n",
       "[2830 rows x 9 columns]"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# groupby the title of the paper and create a new column that merges the antibody names but keep all the other columns\n",
    "df_filter = df_all.groupby('title').agg({'Antibody': lambda x: ', '.join(set(x)), \n",
    "                                         'Target': lambda x: ', '.join(set(x))}).reset_index()\n",
    "# enforce thaat all strings in the Antibody column are unique as well and then merge back to the original df\n",
    "df_final = df_all.merge(df_filter, on='title', how='left').drop_duplicates(subset=['title'])\n",
    "# drop the Antibody_y column and rename Antibody_x to Antibody\n",
    "df_final = df_final.drop(['Antibody_x'], axis=1)\n",
    "# same for the target column\n",
    "df_final = df_final.drop(['Target_x'], axis=1)\n",
    "df_final = df_final.rename(columns={'Target_y': 'Target'})\n",
    "df_final = df_final.rename(columns={'Antibody_y': 'Antibody'})\n",
    "df_final"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "7aa151b8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Chemical corrector treatment ameliorates incre...</td>\n",
       "      <td>https://www.nature.com/articles/nm.3759</td>\n",
       "      <td>N Yokoi, Y Fukata, D Kase, T Miyazaki, M Jaegl...</td>\n",
       "      <td>Epilepsy is one of the most common and intract...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=35472...</td>\n",
       "      <td>74.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>K14/16, N144/14</td>\n",
       "      <td>Kv1.2 K+ channel, 6xHis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1831</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K13/31, L23/27, K67/25, K65/35, K14/16, K36/15...</td>\n",
       "      <td>CASPR2, ADAM22 (cytoplasmic), Kv1.4 K+ channel...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1855</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>N56/21, K74/71, K69/3</td>\n",
       "      <td>Nav1.1 Na+ channel, Nav1.2 Na+ channel, FGF14/...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "15    Chemical corrector treatment ameliorates incre...   \n",
       "1831   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1855  Proteomic analysis of native cerebellar iFGF14...   \n",
       "\n",
       "                                             title_link  \\\n",
       "15              https://www.nature.com/articles/nm.3759   \n",
       "1831  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1855  https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "\n",
       "                                       publication_info  \\\n",
       "15    N Yokoi, Y Fukata, D Kase, T Miyazaki, M Jaegl...   \n",
       "1831  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1855  MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "\n",
       "                                                snippet  \\\n",
       "15    Epilepsy is one of the most common and intract...   \n",
       "1831  Neurodevelopmental disorders, such as intellec...   \n",
       "1855  Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "15    https://scholar.google.com/scholar?cites=35472...            74.0   \n",
       "1831                                                NaN             NaN   \n",
       "1855  https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "\n",
       "                                               pdf_file  \\\n",
       "15                                                  NaN   \n",
       "1831  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1855  https://www.tandfonline.com/doi/pdf/10.1080/19...   \n",
       "\n",
       "                                               Antibody  \\\n",
       "15                                      K14/16, N144/14   \n",
       "1831  K13/31, L23/27, K67/25, K65/35, K14/16, K36/15...   \n",
       "1855                              N56/21, K74/71, K69/3   \n",
       "\n",
       "                                                 Target  \n",
       "15                              Kv1.2 K+ channel, 6xHis  \n",
       "1831  CASPR2, ADAM22 (cytoplasmic), Kv1.4 K+ channel...  \n",
       "1855  Nav1.1 Na+ channel, Nav1.2 Na+ channel, FGF14/...  "
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# filter df_final for Miyazaki\n",
    "df_final[df_final['publication_info'].str.contains('Miyazaki')]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "5a8bea36",
   "metadata": {},
   "outputs": [],
   "source": [
    "# output the final df to the picklequeries folder as googescholar_mabidsearch.csv\n",
    "allnmab_df3.to_csv('/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/googescholar_mabidsearch_5-20-24.csv', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9451d00b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# sum baased on the year\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5ce82499",
   "metadata": {},
   "source": [
    "# Pubmed scraping get PMID for each of the papers\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "db8c30ed",
   "metadata": {},
   "outputs": [],
   "source": [
    "def add_Pubmed_info_to_dataframe(df, developer_mode, pos):\n",
    "    # TODO wendon this function is a bit of a mess and could use some cleaning I \n",
    "        # i would just make sure it is working and make sure the \"significance\" checking is good.\n",
    "        # then i would take a look at the \"try\" and \"except\" blocks and see if there is a better way to do it\n",
    "        # might be a better way then just intermediary files but also a good way to cache and abstract - just dependent on the previous dataframe\n",
    "        # solution could be reading in old outputs titles or something idk - just a thought surely something simple here\n",
    "    '''\n",
    "    Description : \n",
    "    \n",
    "    This function uses the PubMed tool to generate the pubmed ID from the value in the Title \n",
    "    column of each row of the desired dataframe\n",
    "    \n",
    "    Input : \n",
    "    \n",
    "    leftoff = from what row you would like to start adding information to \n",
    "    \n",
    "    autosave = the number of iterations before being saved as csv\n",
    "    \n",
    "    df = the desired dataframe to add the information to \n",
    "    \n",
    "    Returns : \n",
    "    \n",
    "    a dataframe with the PMID, Author, Publication Date, and Journal columns appended\n",
    "    \n",
    "    '''\n",
    "\n",
    "    # initialize the tool being used \n",
    "    pubmed = PubMed(tool=\"PubMedSearcher\", email=\"gmail.com\")\n",
    "\n",
    "    # read in desired dataframe \n",
    "    #df = pd.read_csv(generate_path_to_merged_df())\n",
    "    \n",
    "    # Initialize columns values to be added to NaN\n",
    "    if 'PMID' not in df:\n",
    "        df['PMID'] = np.nan\n",
    "    if 'Author' not in df:\n",
    "        df['Author'] = np.nan\n",
    "    if 'Publication Date' not in df:\n",
    "        df['Publication Date'] = np.nan\n",
    "    if 'Journal' not in df:\n",
    "        df['Journal'] = np.nan\n",
    "\n",
    "    # iterate over the df from the desginated row \n",
    "    for index, row in df[pos:].iterrows():\n",
    "        try:\n",
    "            if pd.isna(row['title']):    # if the value in Title is empty skip over it \n",
    "                continue\n",
    "\n",
    "            if not pd.isna(row['PMID']): # if there is a value in the PMID column skip over it \n",
    "                continue\n",
    "\n",
    "            search_term = row['title']\n",
    "            max_results = 500\n",
    "            results = pubmed.query(search_term, max_results=max_results)\n",
    "            last_pubmedId = None  # Initialize to None\n",
    "\n",
    "            # iterate through each of the paper titles in results, if the title is 80% similar to one of the \n",
    "            # articles - then that is the paper that we will grab information from \n",
    "            for article in results:\n",
    "                title = article.title\n",
    "                pubmedId = article.pubmed_id.partition('\\n')[0]\n",
    "\n",
    "                # Compare the similarity of the title and the search term\n",
    "                similarity = fuzz.token_set_ratio(title.lower(), search_term.lower())\n",
    "                if similarity >= 80:\n",
    "                    last_pubmedId = pubmedId\n",
    "                    break\n",
    "\n",
    "            if last_pubmedId is not None:  # Only assign pubmed_id if a significant title is found\n",
    "                print(row['title'],last_pubmedId)\n",
    "                df.at[index, 'PMID'] = last_pubmedId   # assign the PMID to the row \n",
    "                df.at[index, 'Date'] = article.publication_date # assign the publication date to the row \n",
    "\n",
    "                # Create the Publication information from the first authors name and \n",
    "                # journal the paper is from \n",
    "                authors = article.authors\n",
    "                # Extracting the first author's lastname\n",
    "                first_author_lastname = authors[0]['lastname']\n",
    "\n",
    "                # Creating the \"et al\" part\n",
    "                et_al = \"et al\"\n",
    "\n",
    "                # Creating the formatted string for publication info\n",
    "                publication_info = f\"{first_author_lastname} {et_al}. {article.journal}\"\n",
    "                df.at[index, 'Publication'] = publication_info\n",
    "        \n",
    "        except Exception as e:\n",
    "            import traceback\n",
    "            traceback.print_exc()\n",
    "            print(e)\n",
    "            df[pos:index].to_csv(f\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/merged_df_{pos}-{index}.csv\", index=False)\n",
    "            return df, index\n",
    "\n",
    "        # save the dataframe whenever there are 100 iterations - there is a possibility of being dropped\n",
    "    \n",
    "    # remove the 'unnamed' column from the df \n",
    "    # for col in runresultsdf.columns:\n",
    "    #     if col.startswith(\"Unnamed\"):\n",
    "    #         runresultsdf = runresultsdf.drop(col, axis=1)\n",
    "            \n",
    "    # Delete duplicates in the PMID column \n",
    "    #df = df.drop_duplicates(subset='PMID', keep='first')\n",
    "    \n",
    "    df.to_csv(f\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/merged_df_{pos}_{index}_done_5-20-24.csv\", index=False)\n",
    "    return df\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "6a9ac1b4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Add pubmed info to a dataframe\n",
    "import os\n",
    "import json\n",
    "from Bio import Entrez\n",
    "\n",
    "def fetch_details(pubmed_id):\n",
    "    file_path = f\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/pmidjsons/{pubmed_id}.json\"\n",
    "    \n",
    "    try:\n",
    "        # Check if the file already exists to use cached data\n",
    "        if os.path.exists(file_path):\n",
    "            with open(file_path, 'r') as f:\n",
    "                record = json.load(f)\n",
    "        else:\n",
    "            Entrez.email = \"kgmitchell@ucdavis.edu\"  # Always provide your email\n",
    "            handle = Entrez.efetch(db=\"pubmed\", id=pubmed_id, retmode=\"xml\")\n",
    "            records = Entrez.read(handle)\n",
    "            handle.close()\n",
    "            \n",
    "            if not records or 'PubmedArticle' not in records:\n",
    "                print(f\"PubMed returned no data for {pubmed_id}.\")\n",
    "                return None\n",
    "\n",
    "            record = records\n",
    "            with open(file_path, 'w') as f:\n",
    "                json.dump(record, f)\n",
    "\n",
    "        article = record['PubmedArticle'][0]\n",
    "        article_data = article['MedlineCitation']['Article']\n",
    "        authors = article_data['AuthorList']\n",
    "        \n",
    "        # Use only the first author with \"et al.\" regardless of the number of authors\n",
    "        first_author = authors[0]\n",
    "        author_names = f\"{first_author['LastName']} {first_author['Initials']} et al.\"\n",
    "        \n",
    "        title = article_data['ArticleTitle']\n",
    "        journal = article_data['Journal']['ISOAbbreviation']\n",
    "        year = article_data['Journal']['JournalIssue']['PubDate']['Year']\n",
    "\n",
    "        # Safely check for Pagination and Volume\n",
    "        pages = ''\n",
    "        if 'Pagination' in article_data:\n",
    "            pages = article_data['Pagination'].get('MedlinePgn', '')\n",
    "\n",
    "        volume = article_data['Journal']['JournalIssue'].get('Volume', '')\n",
    "\n",
    "        # Construct citation based on available data\n",
    "        citation_parts = [author_names, year, journal]\n",
    "        if volume:\n",
    "            citation_parts.append(volume)\n",
    "        if pages:\n",
    "            citation_parts.append(pages)\n",
    "\n",
    "        citation = \" \".join(citation_parts) + \".\"\n",
    "\n",
    "        # Return the result as a dictionary\n",
    "        result = {'citation': citation, 'title': title, 'year': year}\n",
    "        return result\n",
    "    except Exception as e:\n",
    "        print(f\"Error fetching data from PubMed for {pubmed_id}.\")\n",
    "        print(e)\n",
    "        return None\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "c9f9e8c9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>Author</th>\n",
       "      <th>Publication Date</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Date</th>\n",
       "      <th>Publication</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Characterization of hyperpolarization-activate...</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>KA Lyman, Y Han, AP Robinson… - Frontiers in C...</td>\n",
       "      <td>Mature oligodendrocytes (OLG) are the myelin-f...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "      <td>38469353</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-03-12</td>\n",
       "      <td>Lyman et al. Frontiers in cellular neuroscience</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A complex of Protocadherin-19 and N-cadherin m...</td>\n",
       "      <td>https://rupress.org/jcb/article-abstract/195/7...</td>\n",
       "      <td>MR Emond, S Biswas, CJ Blevins, JD Jontes - Jo...</td>\n",
       "      <td>… N144/14; University of California Davis/Nati...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=11570...</td>\n",
       "      <td>126.0</td>\n",
       "      <td>https://rupress.org/jcb/article-pdf/195/7/1115...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>22184198</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2011-12-21</td>\n",
       "      <td>Emond et al. The Journal of cell biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Cdk5-mediated phosphorylation of CRMP-2 enhan...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>JM Brittain, Y Wang, O Eruvwetere, R Khanna - ...</td>\n",
       "      <td>… Lysates were incubated with CRMP-2-6xHis, fo...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=16563...</td>\n",
       "      <td>84.0</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>23022559</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2012-10-02</td>\n",
       "      <td>Brittain et al. FEBS letters</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Reprogramming protein kinase substrate specifi...</td>\n",
       "      <td>https://www.biorxiv.org/content/10.1101/091892...</td>\n",
       "      <td>JM Lubner, GM Church, MF Chou, D Schwartz - bi...</td>\n",
       "      <td>Protein kinase specificity is largely imparted...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=11711...</td>\n",
       "      <td>5.0</td>\n",
       "      <td>https://www.biorxiv.org/content/10.1101/091892...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>α-synuclein promotes neuronal dysfunction and ...</td>\n",
       "      <td>https://www.jneurosci.org/content/43/9/1614.ab...</td>\n",
       "      <td>G Maor, RR Dubreuil, MB Feany - Journal of Neu...</td>\n",
       "      <td>α-Synuclein plays a key role in the pathogenes...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=25347...</td>\n",
       "      <td>7.0</td>\n",
       "      <td>https://www.jneurosci.org/content/jneuro/43/9/...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>37333277</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2023-06-19</td>\n",
       "      <td>Maor et al. bioRxiv : the preprint server for ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1440</th>\n",
       "      <td>Dissecting The Tissue-Specific Contributions T...</td>\n",
       "      <td>https://scholar.smu.edu/hum_sci_biologicalscie...</td>\n",
       "      <td>K Paulhus - 2024 - scholar.smu.edu</td>\n",
       "      <td>Sudden unexpected death in epilepsy (SUDEP), t...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://scholar.smu.edu/cgi/viewcontent.cgi?ar...</td>\n",
       "      <td>K20/78</td>\n",
       "      <td>Kv1.1 K+ channel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1441</th>\n",
       "      <td>Impact of Elevated Brain IL-6 in Transgenic Mi...</td>\n",
       "      <td>https://www.mdpi.com/2073-4409/12/18/2306</td>\n",
       "      <td>DL Gruol, D Calderon, S Huitron-Resendiz… - Ce...</td>\n",
       "      <td>… 500, UC Davis/NIH NeuroMab Facility); a mono...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=45817...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.mdpi.com/2073-4409/12/18/2306/pdf</td>\n",
       "      <td>L118/80</td>\n",
       "      <td>VGAT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1442</th>\n",
       "      <td>Rescue of deficits by Brwd1 copy number restor...</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...</td>\n",
       "      <td>With an incidence of ~1 in 800 births, Down sy...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=44492...</td>\n",
       "      <td>9.0</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1443</th>\n",
       "      <td>Identification of Novel Cell-Type Specific Epi...</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>SL Fulton - 2022 - search.proquest.com</td>\n",
       "      <td>The overarching goal of this thesis work has b...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1444</th>\n",
       "      <td>Domain and cell type-specific immunolocalisat...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>B Otuyemi, T Jackson, R Ma, AR Monteiro… - Jou...</td>\n",
       "      <td>Diverse classes of voltage-gated potassium cha...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=15314...</td>\n",
       "      <td>5.0</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K57/1</td>\n",
       "      <td>Kv4.2 K+ channel (external)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1445 rows × 15 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "0     Characterization of hyperpolarization-activate...   \n",
       "1     A complex of Protocadherin-19 and N-cadherin m...   \n",
       "2      Cdk5-mediated phosphorylation of CRMP-2 enhan...   \n",
       "3     Reprogramming protein kinase substrate specifi...   \n",
       "4     α-synuclein promotes neuronal dysfunction and ...   \n",
       "...                                                 ...   \n",
       "1440  Dissecting The Tissue-Specific Contributions T...   \n",
       "1441  Impact of Elevated Brain IL-6 in Transgenic Mi...   \n",
       "1442  Rescue of deficits by Brwd1 copy number restor...   \n",
       "1443  Identification of Novel Cell-Type Specific Epi...   \n",
       "1444   Domain and cell type-specific immunolocalisat...   \n",
       "\n",
       "                                             title_link  \\\n",
       "0     https://www.frontiersin.org/articles/10.3389/f...   \n",
       "1     https://rupress.org/jcb/article-abstract/195/7...   \n",
       "2     https://www.sciencedirect.com/science/article/...   \n",
       "3     https://www.biorxiv.org/content/10.1101/091892...   \n",
       "4     https://www.jneurosci.org/content/43/9/1614.ab...   \n",
       "...                                                 ...   \n",
       "1440  https://scholar.smu.edu/hum_sci_biologicalscie...   \n",
       "1441          https://www.mdpi.com/2073-4409/12/18/2306   \n",
       "1442  https://www.nature.com/articles/s41467-022-342...   \n",
       "1443  https://search.proquest.com/openview/88710f293...   \n",
       "1444  https://www.sciencedirect.com/science/article/...   \n",
       "\n",
       "                                       publication_info  \\\n",
       "0     KA Lyman, Y Han, AP Robinson… - Frontiers in C...   \n",
       "1     MR Emond, S Biswas, CJ Blevins, JD Jontes - Jo...   \n",
       "2     JM Brittain, Y Wang, O Eruvwetere, R Khanna - ...   \n",
       "3     JM Lubner, GM Church, MF Chou, D Schwartz - bi...   \n",
       "4     G Maor, RR Dubreuil, MB Feany - Journal of Neu...   \n",
       "...                                                 ...   \n",
       "1440                 K Paulhus - 2024 - scholar.smu.edu   \n",
       "1441  DL Gruol, D Calderon, S Huitron-Resendiz… - Ce...   \n",
       "1442  SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...   \n",
       "1443             SL Fulton - 2022 - search.proquest.com   \n",
       "1444  B Otuyemi, T Jackson, R Ma, AR Monteiro… - Jou...   \n",
       "\n",
       "                                                snippet  \\\n",
       "0     Mature oligodendrocytes (OLG) are the myelin-f...   \n",
       "1     … N144/14; University of California Davis/Nati...   \n",
       "2     … Lysates were incubated with CRMP-2-6xHis, fo...   \n",
       "3     Protein kinase specificity is largely imparted...   \n",
       "4     α-Synuclein plays a key role in the pathogenes...   \n",
       "...                                                 ...   \n",
       "1440  Sudden unexpected death in epilepsy (SUDEP), t...   \n",
       "1441  … 500, UC Davis/NIH NeuroMab Facility); a mono...   \n",
       "1442  With an incidence of ~1 in 800 births, Down sy...   \n",
       "1443  The overarching goal of this thesis work has b...   \n",
       "1444  Diverse classes of voltage-gated potassium cha...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "0                                                   NaN             NaN   \n",
       "1     https://scholar.google.com/scholar?cites=11570...           126.0   \n",
       "2     https://scholar.google.com/scholar?cites=16563...            84.0   \n",
       "3     https://scholar.google.com/scholar?cites=11711...             5.0   \n",
       "4     https://scholar.google.com/scholar?cites=25347...             7.0   \n",
       "...                                                 ...             ...   \n",
       "1440                                                NaN             NaN   \n",
       "1441  https://scholar.google.com/scholar?cites=45817...             1.0   \n",
       "1442  https://scholar.google.com/scholar?cites=44492...             9.0   \n",
       "1443                                                NaN             NaN   \n",
       "1444  https://scholar.google.com/scholar?cites=15314...             5.0   \n",
       "\n",
       "                                               pdf_file  Antibody  \\\n",
       "0     https://www.frontiersin.org/articles/10.3389/f...   N212/17   \n",
       "1     https://rupress.org/jcb/article-pdf/195/7/1115...   N144/14   \n",
       "2     https://www.sciencedirect.com/science/article/...   N144/14   \n",
       "3     https://www.biorxiv.org/content/10.1101/091892...   N144/14   \n",
       "4     https://www.jneurosci.org/content/jneuro/43/9/...   N144/14   \n",
       "...                                                 ...       ...   \n",
       "1440  https://scholar.smu.edu/cgi/viewcontent.cgi?ar...    K20/78   \n",
       "1441      https://www.mdpi.com/2073-4409/12/18/2306/pdf   L118/80   \n",
       "1442  https://www.nature.com/articles/s41467-022-342...  N336B/83   \n",
       "1443  https://search.proquest.com/openview/88710f293...  N336B/83   \n",
       "1444  https://www.sciencedirect.com/science/article/...     K57/1   \n",
       "\n",
       "                           Target      PMID  Author  Publication Date  \\\n",
       "0               TRIP8b (constant)  38469353     NaN               NaN   \n",
       "1                           6xHis  22184198     NaN               NaN   \n",
       "2                           6xHis  23022559     NaN               NaN   \n",
       "3                           6xHis       NaN     NaN               NaN   \n",
       "4                           6xHis  37333277     NaN               NaN   \n",
       "...                           ...       ...     ...               ...   \n",
       "1440             Kv1.1 K+ channel       NaN     NaN               NaN   \n",
       "1441                         VGAT       NaN     NaN               NaN   \n",
       "1442                       BAF53a       NaN     NaN               NaN   \n",
       "1443                       BAF53a       NaN     NaN               NaN   \n",
       "1444  Kv4.2 K+ channel (external)       NaN     NaN               NaN   \n",
       "\n",
       "      Journal        Date                                        Publication  \n",
       "0         NaN  2024-03-12    Lyman et al. Frontiers in cellular neuroscience  \n",
       "1         NaN  2011-12-21           Emond et al. The Journal of cell biology  \n",
       "2         NaN  2012-10-02                       Brittain et al. FEBS letters  \n",
       "3         NaN         NaN                                                NaN  \n",
       "4         NaN  2023-06-19  Maor et al. bioRxiv : the preprint server for ...  \n",
       "...       ...         ...                                                ...  \n",
       "1440      NaN         NaN                                                NaN  \n",
       "1441      NaN         NaN                                                NaN  \n",
       "1442      NaN         NaN                                                NaN  \n",
       "1443      NaN         NaN                                                NaN  \n",
       "1444      NaN         NaN                                                NaN  \n",
       "\n",
       "[1445 rows x 15 columns]"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_final"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "ca95db89",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0\n",
      "Characterization of hyperpolarization-activated cyclic nucleotide-gated channels in oligodendrocytes 38469353\n",
      "A complex of Protocadherin-19 and N-cadherin mediates a novel mechanism of cell adhesion 22184198\n",
      " Cdk5-mediated phosphorylation of CRMP-2 enhances its interaction with CaV2. 2 23022559\n",
      "α-synuclein promotes neuronal dysfunction and death by disrupting the binding of ankyrin to β-spectrin 37333277\n",
      " Direct binding between pre-S1 and TRP-like domains in TRPP channels mediates gating and functional regulation by PIP2 29425510\n",
      "A cytoskeleton-membrane interaction conserved in fast-spiking neurons controls movement, emotion, and memory 37833406\n",
      "Structural basis for recognition of synaptic vesicle protein 2C by botulinum neurotoxin A 24240280\n",
      " The metalloprotease, Mpr1, engages AnnexinA2 to promote the transcytosis of fungal cells across the blood-brain barrier 28713781\n",
      "Activation of conventional kinesin motors in clusters by Shaw voltage-gated K+ channels 23487040\n",
      "β spectrin-dependent and domain specific mechanisms for Na+ channel clustering 32425157\n",
      " Ankyrin-dependent Na+ channel clustering prevents neuromuscular synapse fatigue 34289389\n",
      "Nodal β spectrins are required to maintain Na+ channel clustering and axon integrity 32052742\n",
      "Rapid sequential clustering of NMDARs, CaMKII, and AMPARs upon activation of NMDARs at developing synapses 38660466\n",
      "VAP spatially stabilizes dendritic mitochondria to locally support synaptic plasticity 38177103\n",
      "Developing high-quality mouse monoclonal antibodies for neuroscience research–approaches, perspectives and opportunities 26644354\n",
      "Subregional differences in GABAA receptor subunit expression in the rostral ventrolateral medulla of sedentary versus physically active rats 31642070\n",
      " Mapping synaptic input fields of neurons with super-resolution imaging 26435106\n",
      "Detection and Quantification of Multiple RNA Sequences Using Emerging Ultrasensitive Fluorescent In Situ Hybridization Techniques 30791216\n",
      "p53 prevents neurodegeneration by regulating synaptic genes 25453105\n",
      "Dual Transgene Expression in Murine Cerebellar Purkinje Neurons by Viral Transduction In Vivo 25093726\n",
      "Gel‐like inclusions of C‐terminal fragments of TDP‐43 sequester stalled proteasomes in neurons 35438230\n",
      "Systematic mapping of fragile X granules in the mouse brain reveals a potential role for presynaptic FMRP in sensorimotor functions 22522693\n",
      "Glial α‐synuclein promotes neurodegeneration characterized by a distinct transcriptional program in vivo 31267577\n",
      "Anastasis drives senescence and non-cell autonomous neurodegeneration in the astrogliopathy Alexander disease 35105675\n",
      "A novel ultrasensitive in situ hybridization approach to detect short sequences and splice variants with cellular resolution 29264769\n",
      " α-synuclein impairs autophagosome maturation through abnormal actin stabilization 33556113\n",
      "A toolbox of IgG subclass-switched recombinant monoclonal antibodies for enhanced multiplex immunolabeling of brain 30667360\n",
      "A cytoskeleton-membrane interaction conserved in fast-spiking neurons controls movement, emotion, and memory 37833406\n",
      " A Drosophila model of chemotherapy-related cognitive impairment 37935827\n",
      " Distinct Activation Properties of the Nuclear Factor of Activated T-cells (NFAT) Isoforms NFATc3 and NFATc4 in Neurons*♦ 22977251\n",
      " Dual regulation of miRNA biogenesis generates target specificity in neurotrophin-induced protein synthesis 22385959\n",
      " Composition and control of a Deg/ENaC channel during presynaptic homeostatic plasticity 28834749\n",
      "Pathways controlling neurotoxicity and proteostasis in mitochondrial complex I deficiency 38324746\n",
      "PSD-95 stabilizes NMDA receptors by inducing the degradation of STEP61 27457929\n",
      " Oligomerization of Lrrk controls actin severing and α-synuclein neurotoxicity in vivo 34030727\n",
      " An ankyrin-G N-terminal gate and protein kinase CK2 dually regulate binding of voltage-gated sodium and KCNQ2/3 potassium channels 25998125\n",
      " Activation and expansion of presynaptic signaling foci drives presynaptic homeostatic plasticity 36087584\n",
      "Distinct roles for the Charcot–Marie–Tooth disease-causing endosomal regulators Mtmr5 and Mtmr13 in axon radial sorting and Schwann cell myelination 34718573\n",
      " HDAC1/2-dependent P0 expression maintains paranodal and nodal integrity independently of myelin stability through interactions with neurofascins 26406915\n",
      "Overexpression of CD133 promotes drug resistance in C6 glioma cells 20663862\n",
      "ER-mitochondria contacts couple mtDNA synthesis with mitochondrial division in human cells 27418514\n",
      "Disease‐linked TDP‐43 hyperphosphorylation suppresses TDP‐43 condensation and aggregation 35112738\n",
      "A requirement for nuclear factor-κB in developmental and plasticity-associated synaptogenesis 21471377\n",
      " Autism and schizophrenia-associated CYFIP1 regulates the balance of synaptic excitation and inhibition 30784587\n",
      " Kalirin12 interacts with dynamin 19534784\n",
      "Functionally clustered mRNAs are distinctly enriched at cortical astroglial processes and are preferentially affected by FMRP deficiency 35701158\n",
      "Cell‐to‐cell transmission of C9orf72 poly‐(Gly‐Ala) triggers key features of ALS/FTD 32175624\n",
      "Insulin signaling controls neurotransmission via the 4eBP-dependent modification of the exocytotic machinery 27525480\n",
      " C9orf72 FTLD/ALS-associated Gly-Ala dipeptide repeat proteins cause neuronal toxicity and Unc119 sequestration 25120191\n",
      " Fluorescence polarization and fluctuation analysis monitors subunit proximity, stoichiometry, and protein complex hydrodynamics 22666486\n",
      "Targeting of NF-κB to dendritic spines is required for synaptic signaling and spine development 29555853\n",
      "Reduced hnRNPA3 increases C9orf72 repeat RNA levels and dipeptide‐repeat protein deposition 27461252\n",
      "Stress-response transcription factors Msn2 and Msn4 couple TORC2-Ypk1 signaling and mitochondrial respiration to ATG8 gene expression and autophagy 29198169\n",
      " DISC1-dependent regulation of mitochondrial dynamics controls the morphogenesis of complex neuronal dendrites 26553875\n",
      " A Drosophila model relevant to chemotherapy-related cognitive impairment 37935827\n",
      "Cytoplasmic poly-GA aggregates impair nuclear import of TDP-43 in C9orf72 ALS/FTLD 28040728\n",
      " An epilepsy-associated GRIN2A rare variant disrupts CaMKIIα phosphorylation of GluN2A and NMDA receptor trafficking 32877683\n",
      "EGFP insertional mutagenesis reveals multiple FXR2P fibrillar states with differing ribosome association in neurons 31434643\n",
      "Antibodies inhibit transmission and aggregation of C9orf72 poly‐GA dipeptide repeat proteins 28351931\n",
      " A novel biotinylated lipid raft reporter for electron microscopic imaging of plasma membrane microdomains 22822037\n",
      " Exosome reporter mice reveal the involvement of exosomes in mediating neuron to astroglia communication in the CNS 31515491\n",
      "Septins modulate the autophagy response after nutrient starvation 37910217\n",
      " Poly-glycine–alanine exacerbates C9orf72 repeat expansion-mediated DNA damage via sequestration of phosphorylated ATM and loss of nuclear hnRNPA3 31642962\n",
      " Lysine 27 ubiquitination of the mitochondrial transport protein Miro is dependent on serine 65 of the Parkin ubiquitin ligase 24671417\n",
      " COPS5 (Jab1) protein increases β site processing of amyloid precursor protein and amyloid β peptide generation by stabilizing RanBP9 protein levels 23926111\n",
      " Integrative analysis reveals a conserved role for the amyloid precursor protein in proteostasis during aging 37923712\n",
      " Ribosome profiling reveals a functional role for autophagy in mRNA translational control 32681145\n",
      "Nestin selectively facilitates the phosphorylation of the lissencephaly-linked protein doublecortin (DCX) by cdk5/p35 to regulate growth cone morphology and Sema3a … 32273484\n",
      "A conserved cytoskeletal signaling cascade mediates neurotoxicity of FTDP-17 tau mutations in vivo 29138281\n",
      "Target of rapamycin signaling mediates vacuolar fission caused by endoplasmic reticulum stress in Saccharomyces cerevisiae 26466677\n",
      "Brain endothelial cells induce astrocytic expression of the glutamate transporter GLT‐1 by a notch‐dependent mechanism 28771710\n",
      " The PICK1 Ca2+ sensor modulates N-methyl-d-aspartate (NMDA) receptor-dependent microRNA-mediated translational repression in neurons 28404816\n",
      " Lrrk promotes tau neurotoxicity through dysregulation of actin and mitochondrial dynamics 30571694\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Input \u001b[0;32mIn [23]\u001b[0m, in \u001b[0;36m<cell line: 3>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m pos \u001b[38;5;241m<\u001b[39m \u001b[38;5;28mlen\u001b[39m(df_final):\n\u001b[1;32m      4\u001b[0m     \u001b[38;5;28mprint\u001b[39m(pos)\n\u001b[0;32m----> 5\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[43madd_Pubmed_info_to_dataframe\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdf_final\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mpos\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m      6\u001b[0m     pos \u001b[38;5;241m=\u001b[39m result[\u001b[38;5;241m1\u001b[39m]\n",
      "Input \u001b[0;32mIn [15]\u001b[0m, in \u001b[0;36madd_Pubmed_info_to_dataframe\u001b[0;34m(df, developer_mode, pos)\u001b[0m\n\u001b[1;32m     52\u001b[0m search_term \u001b[38;5;241m=\u001b[39m row[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mtitle\u001b[39m\u001b[38;5;124m'\u001b[39m]\n\u001b[1;32m     53\u001b[0m max_results \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m500\u001b[39m\n\u001b[0;32m---> 54\u001b[0m results \u001b[38;5;241m=\u001b[39m \u001b[43mpubmed\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mquery\u001b[49m\u001b[43m(\u001b[49m\u001b[43msearch_term\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmax_results\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mmax_results\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     55\u001b[0m last_pubmedId \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mNone\u001b[39;00m  \u001b[38;5;66;03m# Initialize to None\u001b[39;00m\n\u001b[1;32m     57\u001b[0m \u001b[38;5;66;03m# iterate through each of the paper titles in results, if the title is 80% similar to one of the \u001b[39;00m\n\u001b[1;32m     58\u001b[0m \u001b[38;5;66;03m# articles - then that is the paper that we will grab information from \u001b[39;00m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/pymed/api.py:62\u001b[0m, in \u001b[0;36mPubMed.query\u001b[0;34m(self, query, max_results)\u001b[0m\n\u001b[1;32m     50\u001b[0m \u001b[38;5;250m\u001b[39m\u001b[38;5;124;03m\"\"\" Method that executes a query agains the GraphQL schema, automatically\u001b[39;00m\n\u001b[1;32m     51\u001b[0m \u001b[38;5;124;03m    inserting the PubMed data loader.\u001b[39;00m\n\u001b[1;32m     52\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     58\u001b[0m \u001b[38;5;124;03m                    in the \"data\" attribute.\u001b[39;00m\n\u001b[1;32m     59\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m     61\u001b[0m \u001b[38;5;66;03m# Retrieve the article IDs for the query\u001b[39;00m\n\u001b[0;32m---> 62\u001b[0m article_ids \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_getArticleIds\u001b[49m\u001b[43m(\u001b[49m\u001b[43mquery\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mquery\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmax_results\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mmax_results\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     64\u001b[0m \u001b[38;5;66;03m# Get the articles themselves\u001b[39;00m\n\u001b[1;32m     65\u001b[0m articles \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mlist\u001b[39m(\n\u001b[1;32m     66\u001b[0m     [\n\u001b[1;32m     67\u001b[0m         \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_getArticles(article_ids\u001b[38;5;241m=\u001b[39mbatch)\n\u001b[1;32m     68\u001b[0m         \u001b[38;5;28;01mfor\u001b[39;00m batch \u001b[38;5;129;01min\u001b[39;00m batches(article_ids, \u001b[38;5;241m250\u001b[39m)\n\u001b[1;32m     69\u001b[0m     ]\n\u001b[1;32m     70\u001b[0m )\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/pymed/api.py:208\u001b[0m, in \u001b[0;36mPubMed._getArticleIds\u001b[0;34m(self, query, max_results)\u001b[0m\n\u001b[1;32m    205\u001b[0m     parameters[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mretmax\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;241m=\u001b[39m max_results\n\u001b[1;32m    207\u001b[0m \u001b[38;5;66;03m# Make the first request to PubMed\u001b[39;00m\n\u001b[0;32m--> 208\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_get\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43m/entrez/eutils/esearch.fcgi\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mparameters\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mparameters\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    210\u001b[0m \u001b[38;5;66;03m# Add the retrieved IDs to the list\u001b[39;00m\n\u001b[1;32m    211\u001b[0m article_ids \u001b[38;5;241m+\u001b[39m\u001b[38;5;241m=\u001b[39m response\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mesearchresult\u001b[39m\u001b[38;5;124m\"\u001b[39m, {})\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124midlist\u001b[39m\u001b[38;5;124m\"\u001b[39m, [])\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/pymed/api.py:140\u001b[0m, in \u001b[0;36mPubMed._get\u001b[0;34m(self, url, parameters, output)\u001b[0m\n\u001b[1;32m    137\u001b[0m parameters[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mretmode\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;241m=\u001b[39m output\n\u001b[1;32m    139\u001b[0m \u001b[38;5;66;03m# Make the request to PubMed\u001b[39;00m\n\u001b[0;32m--> 140\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[43mrequests\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43mf\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;132;43;01m{\u001b[39;49;00m\u001b[43mBASE_URL\u001b[49m\u001b[38;5;132;43;01m}\u001b[39;49;00m\u001b[38;5;132;43;01m{\u001b[39;49;00m\u001b[43murl\u001b[49m\u001b[38;5;132;43;01m}\u001b[39;49;00m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mparams\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mparameters\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    142\u001b[0m \u001b[38;5;66;03m# Check for any errors\u001b[39;00m\n\u001b[1;32m    143\u001b[0m response\u001b[38;5;241m.\u001b[39mraise_for_status()\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/requests/api.py:73\u001b[0m, in \u001b[0;36mget\u001b[0;34m(url, params, **kwargs)\u001b[0m\n\u001b[1;32m     62\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mget\u001b[39m(url, params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[1;32m     63\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124mr\u001b[39m\u001b[38;5;124;03m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[1;32m     64\u001b[0m \n\u001b[1;32m     65\u001b[0m \u001b[38;5;124;03m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     70\u001b[0m \u001b[38;5;124;03m    :rtype: requests.Response\u001b[39;00m\n\u001b[1;32m     71\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m---> 73\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mrequest\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mget\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mparams\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mparams\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/requests/api.py:59\u001b[0m, in \u001b[0;36mrequest\u001b[0;34m(method, url, **kwargs)\u001b[0m\n\u001b[1;32m     55\u001b[0m \u001b[38;5;66;03m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[1;32m     56\u001b[0m \u001b[38;5;66;03m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[1;32m     57\u001b[0m \u001b[38;5;66;03m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[1;32m     58\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m sessions\u001b[38;5;241m.\u001b[39mSession() \u001b[38;5;28;01mas\u001b[39;00m session:\n\u001b[0;32m---> 59\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43msession\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[43m(\u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/requests/sessions.py:587\u001b[0m, in \u001b[0;36mSession.request\u001b[0;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[1;32m    582\u001b[0m send_kwargs \u001b[38;5;241m=\u001b[39m {\n\u001b[1;32m    583\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtimeout\u001b[39m\u001b[38;5;124m\"\u001b[39m: timeout,\n\u001b[1;32m    584\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mallow_redirects\u001b[39m\u001b[38;5;124m\"\u001b[39m: allow_redirects,\n\u001b[1;32m    585\u001b[0m }\n\u001b[1;32m    586\u001b[0m send_kwargs\u001b[38;5;241m.\u001b[39mupdate(settings)\n\u001b[0;32m--> 587\u001b[0m resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msend\u001b[49m\u001b[43m(\u001b[49m\u001b[43mprep\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43msend_kwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    589\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m resp\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/requests/sessions.py:701\u001b[0m, in \u001b[0;36mSession.send\u001b[0;34m(self, request, **kwargs)\u001b[0m\n\u001b[1;32m    698\u001b[0m start \u001b[38;5;241m=\u001b[39m preferred_clock()\n\u001b[1;32m    700\u001b[0m \u001b[38;5;66;03m# Send the request\u001b[39;00m\n\u001b[0;32m--> 701\u001b[0m r \u001b[38;5;241m=\u001b[39m \u001b[43madapter\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msend\u001b[49m\u001b[43m(\u001b[49m\u001b[43mrequest\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    703\u001b[0m \u001b[38;5;66;03m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[1;32m    704\u001b[0m elapsed \u001b[38;5;241m=\u001b[39m preferred_clock() \u001b[38;5;241m-\u001b[39m start\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/requests/adapters.py:489\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[0;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[1;32m    487\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m    488\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m chunked:\n\u001b[0;32m--> 489\u001b[0m         resp \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    490\u001b[0m \u001b[43m            \u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    491\u001b[0m \u001b[43m            \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    492\u001b[0m \u001b[43m            \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    493\u001b[0m \u001b[43m            \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    494\u001b[0m \u001b[43m            \u001b[49m\u001b[43mredirect\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[1;32m    495\u001b[0m \u001b[43m            \u001b[49m\u001b[43massert_same_host\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[1;32m    496\u001b[0m \u001b[43m            \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[1;32m    497\u001b[0m \u001b[43m            \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[1;32m    498\u001b[0m \u001b[43m            \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmax_retries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    499\u001b[0m \u001b[43m            \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    500\u001b[0m \u001b[43m        \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    502\u001b[0m     \u001b[38;5;66;03m# Send the request.\u001b[39;00m\n\u001b[1;32m    503\u001b[0m     \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    504\u001b[0m         \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mhasattr\u001b[39m(conn, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mproxy_pool\u001b[39m\u001b[38;5;124m\"\u001b[39m):\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/urllib3/connectionpool.py:703\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[1;32m    700\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_prepare_proxy(conn)\n\u001b[1;32m    702\u001b[0m \u001b[38;5;66;03m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[0;32m--> 703\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    704\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    705\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    706\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    707\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    708\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    709\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    710\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    711\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    713\u001b[0m \u001b[38;5;66;03m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[1;32m    714\u001b[0m \u001b[38;5;66;03m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[1;32m    715\u001b[0m \u001b[38;5;66;03m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[1;32m    716\u001b[0m \u001b[38;5;66;03m# mess.\u001b[39;00m\n\u001b[1;32m    717\u001b[0m response_conn \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m release_conn \u001b[38;5;28;01melse\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/urllib3/connectionpool.py:449\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[1;32m    444\u001b[0m             httplib_response \u001b[38;5;241m=\u001b[39m conn\u001b[38;5;241m.\u001b[39mgetresponse()\n\u001b[1;32m    445\u001b[0m         \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    446\u001b[0m             \u001b[38;5;66;03m# Remove the TypeError from the exception chain in\u001b[39;00m\n\u001b[1;32m    447\u001b[0m             \u001b[38;5;66;03m# Python 3 (including for exceptions like SystemExit).\u001b[39;00m\n\u001b[1;32m    448\u001b[0m             \u001b[38;5;66;03m# Otherwise it looks like a bug in the code.\u001b[39;00m\n\u001b[0;32m--> 449\u001b[0m             \u001b[43msix\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mraise_from\u001b[49m\u001b[43m(\u001b[49m\u001b[43me\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mNone\u001b[39;49;00m\u001b[43m)\u001b[49m\n\u001b[1;32m    450\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (SocketTimeout, BaseSSLError, SocketError) \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    451\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_raise_timeout(err\u001b[38;5;241m=\u001b[39me, url\u001b[38;5;241m=\u001b[39murl, timeout_value\u001b[38;5;241m=\u001b[39mread_timeout)\n",
      "File \u001b[0;32m<string>:3\u001b[0m, in \u001b[0;36mraise_from\u001b[0;34m(value, from_value)\u001b[0m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/urllib3/connectionpool.py:444\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[1;32m    441\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mTypeError\u001b[39;00m:\n\u001b[1;32m    442\u001b[0m     \u001b[38;5;66;03m# Python 3\u001b[39;00m\n\u001b[1;32m    443\u001b[0m     \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 444\u001b[0m         httplib_response \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mgetresponse\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    445\u001b[0m     \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    446\u001b[0m         \u001b[38;5;66;03m# Remove the TypeError from the exception chain in\u001b[39;00m\n\u001b[1;32m    447\u001b[0m         \u001b[38;5;66;03m# Python 3 (including for exceptions like SystemExit).\u001b[39;00m\n\u001b[1;32m    448\u001b[0m         \u001b[38;5;66;03m# Otherwise it looks like a bug in the code.\u001b[39;00m\n\u001b[1;32m    449\u001b[0m         six\u001b[38;5;241m.\u001b[39mraise_from(e, \u001b[38;5;28;01mNone\u001b[39;00m)\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/http/client.py:1322\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1320\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m   1321\u001b[0m     \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 1322\u001b[0m         \u001b[43mresponse\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbegin\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1323\u001b[0m     \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m:\n\u001b[1;32m   1324\u001b[0m         \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mclose()\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/http/client.py:303\u001b[0m, in \u001b[0;36mHTTPResponse.begin\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    301\u001b[0m \u001b[38;5;66;03m# read until we get a non-100 response\u001b[39;00m\n\u001b[1;32m    302\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[0;32m--> 303\u001b[0m     version, status, reason \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_read_status\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    304\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m status \u001b[38;5;241m!=\u001b[39m CONTINUE:\n\u001b[1;32m    305\u001b[0m         \u001b[38;5;28;01mbreak\u001b[39;00m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/http/client.py:264\u001b[0m, in \u001b[0;36mHTTPResponse._read_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    263\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m_read_status\u001b[39m(\u001b[38;5;28mself\u001b[39m):\n\u001b[0;32m--> 264\u001b[0m     line \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mstr\u001b[39m(\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfp\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mreadline\u001b[49m\u001b[43m(\u001b[49m\u001b[43m_MAXLINE\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m+\u001b[39;49m\u001b[43m \u001b[49m\u001b[38;5;241;43m1\u001b[39;49m\u001b[43m)\u001b[49m, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124miso-8859-1\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    265\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mlen\u001b[39m(line) \u001b[38;5;241m>\u001b[39m _MAXLINE:\n\u001b[1;32m    266\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m LineTooLong(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mstatus line\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/socket.py:669\u001b[0m, in \u001b[0;36mSocketIO.readinto\u001b[0;34m(self, b)\u001b[0m\n\u001b[1;32m    667\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[1;32m    668\u001b[0m     \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 669\u001b[0m         \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_sock\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrecv_into\u001b[49m\u001b[43m(\u001b[49m\u001b[43mb\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    670\u001b[0m     \u001b[38;5;28;01mexcept\u001b[39;00m timeout:\n\u001b[1;32m    671\u001b[0m         \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_timeout_occurred \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mTrue\u001b[39;00m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/ssl.py:1241\u001b[0m, in \u001b[0;36mSSLSocket.recv_into\u001b[0;34m(self, buffer, nbytes, flags)\u001b[0m\n\u001b[1;32m   1237\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m flags \u001b[38;5;241m!=\u001b[39m \u001b[38;5;241m0\u001b[39m:\n\u001b[1;32m   1238\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mValueError\u001b[39;00m(\n\u001b[1;32m   1239\u001b[0m           \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mnon-zero flags not allowed in calls to recv_into() on \u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m\"\u001b[39m \u001b[38;5;241m%\u001b[39m\n\u001b[1;32m   1240\u001b[0m           \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m\u001b[38;5;18m__class__\u001b[39m)\n\u001b[0;32m-> 1241\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mread\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnbytes\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mbuffer\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1242\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m   1243\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28msuper\u001b[39m()\u001b[38;5;241m.\u001b[39mrecv_into(buffer, nbytes, flags)\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/ssl.py:1099\u001b[0m, in \u001b[0;36mSSLSocket.read\u001b[0;34m(self, len, buffer)\u001b[0m\n\u001b[1;32m   1097\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m   1098\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m buffer \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[0;32m-> 1099\u001b[0m         \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_sslobj\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mread\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mlen\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mbuffer\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1100\u001b[0m     \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m   1101\u001b[0m         \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_sslobj\u001b[38;5;241m.\u001b[39mread(\u001b[38;5;28mlen\u001b[39m)\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "pos = 0\n",
    "df_final = pd.read_csv('/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/googescholar_mabidsearch_5-20-24.csv')\n",
    "while pos < len(df_final):\n",
    "    print(pos)\n",
    "    result = add_Pubmed_info_to_dataframe(df_final, False, pos)\n",
    "    pos = result[1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "3c0b5062",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>Author</th>\n",
       "      <th>Publication Date</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Date</th>\n",
       "      <th>Publication</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Characterization of hyperpolarization-activate...</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>KA Lyman, Y Han, AP Robinson… - Frontiers in C...</td>\n",
       "      <td>Mature oligodendrocytes (OLG) are the myelin-f...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/f...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "      <td>38469353</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-03-12</td>\n",
       "      <td>Lyman et al. Frontiers in cellular neuroscience</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A complex of Protocadherin-19 and N-cadherin m...</td>\n",
       "      <td>https://rupress.org/jcb/article-abstract/195/7...</td>\n",
       "      <td>MR Emond, S Biswas, CJ Blevins, JD Jontes - Jo...</td>\n",
       "      <td>… N144/14; University of California Davis/Nati...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=11570...</td>\n",
       "      <td>126.0</td>\n",
       "      <td>https://rupress.org/jcb/article-pdf/195/7/1115...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>22184198</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2011-12-21</td>\n",
       "      <td>Emond et al. The Journal of cell biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Cdk5-mediated phosphorylation of CRMP-2 enhan...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>JM Brittain, Y Wang, O Eruvwetere, R Khanna - ...</td>\n",
       "      <td>… Lysates were incubated with CRMP-2-6xHis, fo...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=16563...</td>\n",
       "      <td>84.0</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>23022559</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2012-10-02</td>\n",
       "      <td>Brittain et al. FEBS letters</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Reprogramming protein kinase substrate specifi...</td>\n",
       "      <td>https://www.biorxiv.org/content/10.1101/091892...</td>\n",
       "      <td>JM Lubner, GM Church, MF Chou, D Schwartz - bi...</td>\n",
       "      <td>Protein kinase specificity is largely imparted...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=11711...</td>\n",
       "      <td>5.0</td>\n",
       "      <td>https://www.biorxiv.org/content/10.1101/091892...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>α-synuclein promotes neuronal dysfunction and ...</td>\n",
       "      <td>https://www.jneurosci.org/content/43/9/1614.ab...</td>\n",
       "      <td>G Maor, RR Dubreuil, MB Feany - Journal of Neu...</td>\n",
       "      <td>α-Synuclein plays a key role in the pathogenes...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=25347...</td>\n",
       "      <td>7.0</td>\n",
       "      <td>https://www.jneurosci.org/content/jneuro/43/9/...</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>37333277</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2023-06-19</td>\n",
       "      <td>Maor et al. bioRxiv : the preprint server for ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1440</th>\n",
       "      <td>Dissecting The Tissue-Specific Contributions T...</td>\n",
       "      <td>https://scholar.smu.edu/hum_sci_biologicalscie...</td>\n",
       "      <td>K Paulhus - 2024 - scholar.smu.edu</td>\n",
       "      <td>Sudden unexpected death in epilepsy (SUDEP), t...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://scholar.smu.edu/cgi/viewcontent.cgi?ar...</td>\n",
       "      <td>K20/78</td>\n",
       "      <td>Kv1.1 K+ channel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1441</th>\n",
       "      <td>Impact of Elevated Brain IL-6 in Transgenic Mi...</td>\n",
       "      <td>https://www.mdpi.com/2073-4409/12/18/2306</td>\n",
       "      <td>DL Gruol, D Calderon, S Huitron-Resendiz… - Ce...</td>\n",
       "      <td>… 500, UC Davis/NIH NeuroMab Facility); a mono...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=45817...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.mdpi.com/2073-4409/12/18/2306/pdf</td>\n",
       "      <td>L118/80</td>\n",
       "      <td>VGAT</td>\n",
       "      <td>37759527</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2023-09-28</td>\n",
       "      <td>Gruol et al. Cells</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1442</th>\n",
       "      <td>Rescue of deficits by Brwd1 copy number restor...</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...</td>\n",
       "      <td>With an incidence of ~1 in 800 births, Down sy...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=44492...</td>\n",
       "      <td>9.0</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-342...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "      <td>36289231</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2022-10-27</td>\n",
       "      <td>Fulton et al. Nature communications</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1443</th>\n",
       "      <td>Identification of Novel Cell-Type Specific Epi...</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>SL Fulton - 2022 - search.proquest.com</td>\n",
       "      <td>The overarching goal of this thesis work has b...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://search.proquest.com/openview/88710f293...</td>\n",
       "      <td>N336B/83</td>\n",
       "      <td>BAF53a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1444</th>\n",
       "      <td>Domain and cell type-specific immunolocalisat...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>B Otuyemi, T Jackson, R Ma, AR Monteiro… - Jou...</td>\n",
       "      <td>Diverse classes of voltage-gated potassium cha...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=15314...</td>\n",
       "      <td>5.0</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K57/1</td>\n",
       "      <td>Kv4.2 K+ channel (external)</td>\n",
       "      <td>36640913</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2023-01-15</td>\n",
       "      <td>Otuyemi et al. Journal of chemical neuroanatomy</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1445 rows × 15 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "0     Characterization of hyperpolarization-activate...   \n",
       "1     A complex of Protocadherin-19 and N-cadherin m...   \n",
       "2      Cdk5-mediated phosphorylation of CRMP-2 enhan...   \n",
       "3     Reprogramming protein kinase substrate specifi...   \n",
       "4     α-synuclein promotes neuronal dysfunction and ...   \n",
       "...                                                 ...   \n",
       "1440  Dissecting The Tissue-Specific Contributions T...   \n",
       "1441  Impact of Elevated Brain IL-6 in Transgenic Mi...   \n",
       "1442  Rescue of deficits by Brwd1 copy number restor...   \n",
       "1443  Identification of Novel Cell-Type Specific Epi...   \n",
       "1444   Domain and cell type-specific immunolocalisat...   \n",
       "\n",
       "                                             title_link  \\\n",
       "0     https://www.frontiersin.org/articles/10.3389/f...   \n",
       "1     https://rupress.org/jcb/article-abstract/195/7...   \n",
       "2     https://www.sciencedirect.com/science/article/...   \n",
       "3     https://www.biorxiv.org/content/10.1101/091892...   \n",
       "4     https://www.jneurosci.org/content/43/9/1614.ab...   \n",
       "...                                                 ...   \n",
       "1440  https://scholar.smu.edu/hum_sci_biologicalscie...   \n",
       "1441          https://www.mdpi.com/2073-4409/12/18/2306   \n",
       "1442  https://www.nature.com/articles/s41467-022-342...   \n",
       "1443  https://search.proquest.com/openview/88710f293...   \n",
       "1444  https://www.sciencedirect.com/science/article/...   \n",
       "\n",
       "                                       publication_info  \\\n",
       "0     KA Lyman, Y Han, AP Robinson… - Frontiers in C...   \n",
       "1     MR Emond, S Biswas, CJ Blevins, JD Jontes - Jo...   \n",
       "2     JM Brittain, Y Wang, O Eruvwetere, R Khanna - ...   \n",
       "3     JM Lubner, GM Church, MF Chou, D Schwartz - bi...   \n",
       "4     G Maor, RR Dubreuil, MB Feany - Journal of Neu...   \n",
       "...                                                 ...   \n",
       "1440                 K Paulhus - 2024 - scholar.smu.edu   \n",
       "1441  DL Gruol, D Calderon, S Huitron-Resendiz… - Ce...   \n",
       "1442  SL Fulton, W Wenderski, AE Lepack, AL Eagle… -...   \n",
       "1443             SL Fulton - 2022 - search.proquest.com   \n",
       "1444  B Otuyemi, T Jackson, R Ma, AR Monteiro… - Jou...   \n",
       "\n",
       "                                                snippet  \\\n",
       "0     Mature oligodendrocytes (OLG) are the myelin-f...   \n",
       "1     … N144/14; University of California Davis/Nati...   \n",
       "2     … Lysates were incubated with CRMP-2-6xHis, fo...   \n",
       "3     Protein kinase specificity is largely imparted...   \n",
       "4     α-Synuclein plays a key role in the pathogenes...   \n",
       "...                                                 ...   \n",
       "1440  Sudden unexpected death in epilepsy (SUDEP), t...   \n",
       "1441  … 500, UC Davis/NIH NeuroMab Facility); a mono...   \n",
       "1442  With an incidence of ~1 in 800 births, Down sy...   \n",
       "1443  The overarching goal of this thesis work has b...   \n",
       "1444  Diverse classes of voltage-gated potassium cha...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "0                                                   NaN             NaN   \n",
       "1     https://scholar.google.com/scholar?cites=11570...           126.0   \n",
       "2     https://scholar.google.com/scholar?cites=16563...            84.0   \n",
       "3     https://scholar.google.com/scholar?cites=11711...             5.0   \n",
       "4     https://scholar.google.com/scholar?cites=25347...             7.0   \n",
       "...                                                 ...             ...   \n",
       "1440                                                NaN             NaN   \n",
       "1441  https://scholar.google.com/scholar?cites=45817...             1.0   \n",
       "1442  https://scholar.google.com/scholar?cites=44492...             9.0   \n",
       "1443                                                NaN             NaN   \n",
       "1444  https://scholar.google.com/scholar?cites=15314...             5.0   \n",
       "\n",
       "                                               pdf_file  Antibody  \\\n",
       "0     https://www.frontiersin.org/articles/10.3389/f...   N212/17   \n",
       "1     https://rupress.org/jcb/article-pdf/195/7/1115...   N144/14   \n",
       "2     https://www.sciencedirect.com/science/article/...   N144/14   \n",
       "3     https://www.biorxiv.org/content/10.1101/091892...   N144/14   \n",
       "4     https://www.jneurosci.org/content/jneuro/43/9/...   N144/14   \n",
       "...                                                 ...       ...   \n",
       "1440  https://scholar.smu.edu/cgi/viewcontent.cgi?ar...    K20/78   \n",
       "1441      https://www.mdpi.com/2073-4409/12/18/2306/pdf   L118/80   \n",
       "1442  https://www.nature.com/articles/s41467-022-342...  N336B/83   \n",
       "1443  https://search.proquest.com/openview/88710f293...  N336B/83   \n",
       "1444  https://www.sciencedirect.com/science/article/...     K57/1   \n",
       "\n",
       "                           Target      PMID  Author  Publication Date  \\\n",
       "0               TRIP8b (constant)  38469353     NaN               NaN   \n",
       "1                           6xHis  22184198     NaN               NaN   \n",
       "2                           6xHis  23022559     NaN               NaN   \n",
       "3                           6xHis       NaN     NaN               NaN   \n",
       "4                           6xHis  37333277     NaN               NaN   \n",
       "...                           ...       ...     ...               ...   \n",
       "1440             Kv1.1 K+ channel       NaN     NaN               NaN   \n",
       "1441                         VGAT  37759527     NaN               NaN   \n",
       "1442                       BAF53a  36289231     NaN               NaN   \n",
       "1443                       BAF53a       NaN     NaN               NaN   \n",
       "1444  Kv4.2 K+ channel (external)  36640913     NaN               NaN   \n",
       "\n",
       "      Journal        Date                                        Publication  \n",
       "0         NaN  2024-03-12    Lyman et al. Frontiers in cellular neuroscience  \n",
       "1         NaN  2011-12-21           Emond et al. The Journal of cell biology  \n",
       "2         NaN  2012-10-02                       Brittain et al. FEBS letters  \n",
       "3         NaN         NaN                                                NaN  \n",
       "4         NaN  2023-06-19  Maor et al. bioRxiv : the preprint server for ...  \n",
       "...       ...         ...                                                ...  \n",
       "1440      NaN         NaN                                                NaN  \n",
       "1441      NaN  2023-09-28                                 Gruol et al. Cells  \n",
       "1442      NaN  2022-10-27                Fulton et al. Nature communications  \n",
       "1443      NaN         NaN                                                NaN  \n",
       "1444      NaN  2023-01-15    Otuyemi et al. Journal of chemical neuroanatomy  \n",
       "\n",
       "[1445 rows x 15 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "result"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "c3a2142a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>Author</th>\n",
       "      <th>Publication Date</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Date</th>\n",
       "      <th>Publication</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K Krzystanek, HB Rasmussen, M Grunnet, O Staub...</td>\n",
       "      <td>BACKGROUND: KCNQ1 (Kv7.1), together with its K...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=98014...</td>\n",
       "      <td>35.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N37A/10</td>\n",
       "      <td>Kv7.1/KCNQ1 K+ channel</td>\n",
       "      <td>22024150.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2011-10-26</td>\n",
       "      <td>Krzystanek et al. Heart rhythm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Caracterización de los canales de potasio depe...</td>\n",
       "      <td>https://idus.us.es/handle/11441/138533</td>\n",
       "      <td>M Ortego Domínguez - 2022 - idus.us.es</td>\n",
       "      <td>Los adipocitos blancos son células que partici...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://idus.us.es/bitstream/handle/11441/1385...</td>\n",
       "      <td>N375/67</td>\n",
       "      <td>Kv3.3 K+ channel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Neuroinflammation alters integrative propertie...</td>\n",
       "      <td>https://link.springer.com/article/10.1007/s120...</td>\n",
       "      <td>F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...</td>\n",
       "      <td>… , N212/17, Neuromab, CA, USA) antibodies usi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=19097...</td>\n",
       "      <td>41.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "      <td>29427087.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2018-02-11</td>\n",
       "      <td>Frigerio et al. Molecular neurobiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Allostery between two binding sites in the io...</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...</td>\n",
       "      <td>Tetratricopeptide repeat (TPR) domains are ubi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=14199...</td>\n",
       "      <td>12.0</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "      <td>28887304.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2017-09-10</td>\n",
       "      <td>Lyman et al. The Journal of biological chemistry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Distal dendritic enrichment of HCN1 channels i...</td>\n",
       "      <td>https://www.eneuro.org/content/5/5/eneuro.0258...</td>\n",
       "      <td>M Meseke, F Neumüller, B Brunne, X Li, M Anstö...</td>\n",
       "      <td>HCN1 compartmentalization in CA1 pyramidal cel...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=50206...</td>\n",
       "      <td>8.0</td>\n",
       "      <td>https://www.eneuro.org/content/eneuro/5/5/ENEU...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13373</th>\n",
       "      <td>A dominant dendrite phenotype caused by the di...</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3112...</td>\n",
       "      <td>CC Yap, L Digilio, K Kruczek, M Roszkowska… - ...</td>\n",
       "      <td>Doublecortin (DCX) is a protein needed for cor...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=10171...</td>\n",
       "      <td>9.0</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3112...</td>\n",
       "      <td>A12/18</td>\n",
       "      <td>Pan-Neurofascin (extracellular)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13374</th>\n",
       "      <td>RUFY3 and RUFY4 are ARL8 effectors that promot...</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-289...</td>\n",
       "      <td>T Keren-Kaplan, A Sarić, S Ghosh… - Nature …, ...</td>\n",
       "      <td>The small GTPase ARL8 associates with endolyso...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13157...</td>\n",
       "      <td>26.0</td>\n",
       "      <td>https://www.nature.com/articles/s41467-022-289...</td>\n",
       "      <td>A12/18</td>\n",
       "      <td>Pan-Neurofascin (extracellular)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13375</th>\n",
       "      <td>Reversible association with motor proteins (RA...</td>\n",
       "      <td>https://journals.plos.org/plosbiology/article?...</td>\n",
       "      <td>CM Guardia, R De Pace, A Sen, A Saric, M Jarni...</td>\n",
       "      <td>We report the development and characterization...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=72248...</td>\n",
       "      <td>24.0</td>\n",
       "      <td>https://journals.plos.org/plosbiology/article/...</td>\n",
       "      <td>A12/18</td>\n",
       "      <td>Pan-Neurofascin (extracellular)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13377</th>\n",
       "      <td>Differential Alteration in Expression of Stri...</td>\n",
       "      <td>https://www.academia.edu/download/90605829/d09...</td>\n",
       "      <td>Z Du, M Tertrais, G Courtand, T Leste-Lasserre...</td>\n",
       "      <td>Huntington’s disease (HD) is a neurodegenerati...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.academia.edu/download/90605829/d09...</td>\n",
       "      <td>N87/25</td>\n",
       "      <td>GABA(A)R, Beta3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13378</th>\n",
       "      <td>Complete loss of KCNA1 activity causes neonat...</td>\n",
       "      <td>https://jmg.bmj.com/content/jmedgenet/early/20...</td>\n",
       "      <td>E Verdura, C Fons, A Schlüter, M Ruiz… - Journ...</td>\n",
       "      <td>… Cells transfected with p513- KCNA1 vectors w...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40106...</td>\n",
       "      <td>26.0</td>\n",
       "      <td>https://jmg.bmj.com/content/jmedgenet/early/20...</td>\n",
       "      <td>K20/78</td>\n",
       "      <td>Kv1.1 K+ channel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5314 rows × 15 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                   title  \\\n",
       "0      Deubiquitylating enzyme USP2 counteracts Nedd4...   \n",
       "1      Caracterización de los canales de potasio depe...   \n",
       "2      Neuroinflammation alters integrative propertie...   \n",
       "3       Allostery between two binding sites in the io...   \n",
       "4      Distal dendritic enrichment of HCN1 channels i...   \n",
       "...                                                  ...   \n",
       "13373  A dominant dendrite phenotype caused by the di...   \n",
       "13374  RUFY3 and RUFY4 are ARL8 effectors that promot...   \n",
       "13375  Reversible association with motor proteins (RA...   \n",
       "13377   Differential Alteration in Expression of Stri...   \n",
       "13378   Complete loss of KCNA1 activity causes neonat...   \n",
       "\n",
       "                                              title_link  \\\n",
       "0      https://www.sciencedirect.com/science/article/...   \n",
       "1                 https://idus.us.es/handle/11441/138533   \n",
       "2      https://link.springer.com/article/10.1007/s120...   \n",
       "3      https://www.jbc.org/article/S0021-9258(20)3303...   \n",
       "4      https://www.eneuro.org/content/5/5/eneuro.0258...   \n",
       "...                                                  ...   \n",
       "13373  https://www.jbc.org/article/S0021-9258(20)3112...   \n",
       "13374  https://www.nature.com/articles/s41467-022-289...   \n",
       "13375  https://journals.plos.org/plosbiology/article?...   \n",
       "13377  https://www.academia.edu/download/90605829/d09...   \n",
       "13378  https://jmg.bmj.com/content/jmedgenet/early/20...   \n",
       "\n",
       "                                        publication_info  \\\n",
       "0      K Krzystanek, HB Rasmussen, M Grunnet, O Staub...   \n",
       "1                 M Ortego Domínguez - 2022 - idus.us.es   \n",
       "2      F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...   \n",
       "3      KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...   \n",
       "4      M Meseke, F Neumüller, B Brunne, X Li, M Anstö...   \n",
       "...                                                  ...   \n",
       "13373  CC Yap, L Digilio, K Kruczek, M Roszkowska… - ...   \n",
       "13374  T Keren-Kaplan, A Sarić, S Ghosh… - Nature …, ...   \n",
       "13375  CM Guardia, R De Pace, A Sen, A Saric, M Jarni...   \n",
       "13377  Z Du, M Tertrais, G Courtand, T Leste-Lasserre...   \n",
       "13378  E Verdura, C Fons, A Schlüter, M Ruiz… - Journ...   \n",
       "\n",
       "                                                 snippet  \\\n",
       "0      BACKGROUND: KCNQ1 (Kv7.1), together with its K...   \n",
       "1      Los adipocitos blancos son células que partici...   \n",
       "2      … , N212/17, Neuromab, CA, USA) antibodies usi...   \n",
       "3      Tetratricopeptide repeat (TPR) domains are ubi...   \n",
       "4      HCN1 compartmentalization in CA1 pyramidal cel...   \n",
       "...                                                  ...   \n",
       "13373  Doublecortin (DCX) is a protein needed for cor...   \n",
       "13374  The small GTPase ARL8 associates with endolyso...   \n",
       "13375  We report the development and characterization...   \n",
       "13377  Huntington’s disease (HD) is a neurodegenerati...   \n",
       "13378  … Cells transfected with p513- KCNA1 vectors w...   \n",
       "\n",
       "                                           cited_by_link  cited_by_count  \\\n",
       "0      https://scholar.google.com/scholar?cites=98014...            35.0   \n",
       "1                                                    NaN             NaN   \n",
       "2      https://scholar.google.com/scholar?cites=19097...            41.0   \n",
       "3      https://scholar.google.com/scholar?cites=14199...            12.0   \n",
       "4      https://scholar.google.com/scholar?cites=50206...             8.0   \n",
       "...                                                  ...             ...   \n",
       "13373  https://scholar.google.com/scholar?cites=10171...             9.0   \n",
       "13374  https://scholar.google.com/scholar?cites=13157...            26.0   \n",
       "13375  https://scholar.google.com/scholar?cites=72248...            24.0   \n",
       "13377                                                NaN             NaN   \n",
       "13378  https://scholar.google.com/scholar?cites=40106...            26.0   \n",
       "\n",
       "                                                pdf_file Antibody  \\\n",
       "0                                                    NaN  N37A/10   \n",
       "1      https://idus.us.es/bitstream/handle/11441/1385...  N375/67   \n",
       "2      https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...  N212/17   \n",
       "3      https://www.jbc.org/article/S0021-9258(20)3303...  N212/17   \n",
       "4      https://www.eneuro.org/content/eneuro/5/5/ENEU...  N212/17   \n",
       "...                                                  ...      ...   \n",
       "13373  https://www.jbc.org/article/S0021-9258(20)3112...   A12/18   \n",
       "13374  https://www.nature.com/articles/s41467-022-289...   A12/18   \n",
       "13375  https://journals.plos.org/plosbiology/article/...   A12/18   \n",
       "13377  https://www.academia.edu/download/90605829/d09...   N87/25   \n",
       "13378  https://jmg.bmj.com/content/jmedgenet/early/20...   K20/78   \n",
       "\n",
       "                                Target        PMID  Author  Publication Date  \\\n",
       "0               Kv7.1/KCNQ1 K+ channel  22024150.0     NaN               NaN   \n",
       "1                     Kv3.3 K+ channel         NaN     NaN               NaN   \n",
       "2                    TRIP8b (constant)  29427087.0     NaN               NaN   \n",
       "3                    TRIP8b (constant)  28887304.0     NaN               NaN   \n",
       "4                    TRIP8b (constant)         NaN     NaN               NaN   \n",
       "...                                ...         ...     ...               ...   \n",
       "13373  Pan-Neurofascin (extracellular)         NaN     NaN               NaN   \n",
       "13374  Pan-Neurofascin (extracellular)         NaN     NaN               NaN   \n",
       "13375  Pan-Neurofascin (extracellular)         NaN     NaN               NaN   \n",
       "13377                  GABA(A)R, Beta3         NaN     NaN               NaN   \n",
       "13378                 Kv1.1 K+ channel         NaN     NaN               NaN   \n",
       "\n",
       "       Journal        Date                                       Publication  \n",
       "0          NaN  2011-10-26                    Krzystanek et al. Heart rhythm  \n",
       "1          NaN         NaN                                               NaN  \n",
       "2          NaN  2018-02-11            Frigerio et al. Molecular neurobiology  \n",
       "3          NaN  2017-09-10  Lyman et al. The Journal of biological chemistry  \n",
       "4          NaN         NaN                                               NaN  \n",
       "...        ...         ...                                               ...  \n",
       "13373      NaN         NaN                                               NaN  \n",
       "13374      NaN         NaN                                               NaN  \n",
       "13375      NaN         NaN                                               NaN  \n",
       "13377      NaN         NaN                                               NaN  \n",
       "13378      NaN         NaN                                               NaN  \n",
       "\n",
       "[5314 rows x 15 columns]"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# read all the merged_df file sin the directory and concat them together\n",
    "import glob\n",
    "import pandas as pd\n",
    "\n",
    "path = r'/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries'\n",
    "filenames = glob.glob(path + \"/merged_df_*.csv\")\n",
    " \n",
    "dfs = []\n",
    "for file in filenames:\n",
    "    dfs.append(pd.read_csv(file))\n",
    "\n",
    "df_finaln = pd.concat(dfs, ignore_index=True)\n",
    "# drop duplicaates based on the pubmed id, antibody and title\n",
    "df_finaln = df_finaln.drop_duplicates(subset=['PMID', 'Antibody', 'title'])\n",
    "df_finaln"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "4ce0252f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>Author</th>\n",
       "      <th>Publication Date</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Date</th>\n",
       "      <th>Publication</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1797</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K36/15</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1867</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K13/31</td>\n",
       "      <td>Kv1.4 K+ channel</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1909</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>L23/27</td>\n",
       "      <td>Kv1.3 K+ channel</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1939</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K14/16</td>\n",
       "      <td>Kv1.2 K+ channel</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1966</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>N46/30</td>\n",
       "      <td>ADAM22 (cytoplasmic)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2129</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K65/35</td>\n",
       "      <td>CASPR/Neurexin IV</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2164</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K67/25</td>\n",
       "      <td>CASPR2</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12124</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>N46/30, K36/15, K67/25, L23/27, K13/31, K65/35</td>\n",
       "      <td>ADAM22 (cytoplasmic)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                   title  \\\n",
       "1797    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1867    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1909    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1939    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1966    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "2129    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "2164    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "12124   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "\n",
       "                                              title_link  \\\n",
       "1797   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1867   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1909   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1939   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1966   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "2129   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "2164   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "12124  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "\n",
       "                                        publication_info  \\\n",
       "1797   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1867   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1909   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1939   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1966   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "2129   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "2164   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "12124  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "\n",
       "                                                 snippet  \\\n",
       "1797   Neurodevelopmental disorders, such as intellec...   \n",
       "1867   Neurodevelopmental disorders, such as intellec...   \n",
       "1909   Neurodevelopmental disorders, such as intellec...   \n",
       "1939   Neurodevelopmental disorders, such as intellec...   \n",
       "1966   Neurodevelopmental disorders, such as intellec...   \n",
       "2129   Neurodevelopmental disorders, such as intellec...   \n",
       "2164   Neurodevelopmental disorders, such as intellec...   \n",
       "12124  Neurodevelopmental disorders, such as intellec...   \n",
       "\n",
       "                                           cited_by_link  cited_by_count  \\\n",
       "1797   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1867   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1909   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1939   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1966   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "2129   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "2164   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "12124                                                NaN             NaN   \n",
       "\n",
       "                                                pdf_file  \\\n",
       "1797   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1867   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1909   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1939   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1966   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "2129   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "2164   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "12124  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "\n",
       "                                             Antibody  \\\n",
       "1797                                           K36/15   \n",
       "1867                                           K13/31   \n",
       "1909                                           L23/27   \n",
       "1939                                           K14/16   \n",
       "1966                                           N46/30   \n",
       "2129                                           K65/35   \n",
       "2164                                           K67/25   \n",
       "12124  N46/30, K36/15, K67/25, L23/27, K13/31, K65/35   \n",
       "\n",
       "                            Target        PMID  Author  Publication Date  \\\n",
       "1797   Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "1867              Kv1.4 K+ channel  38194969.0     NaN               NaN   \n",
       "1909              Kv1.3 K+ channel  38194969.0     NaN               NaN   \n",
       "1939              Kv1.2 K+ channel  38194969.0     NaN               NaN   \n",
       "1966          ADAM22 (cytoplasmic)  38194969.0     NaN               NaN   \n",
       "2129             CASPR/Neurexin IV  38194969.0     NaN               NaN   \n",
       "2164                        CASPR2  38194969.0     NaN               NaN   \n",
       "12124         ADAM22 (cytoplasmic)  38194969.0     NaN               NaN   \n",
       "\n",
       "       Journal        Date                   Publication  \n",
       "1797       NaN  2024-01-10  Miyazaki et al. Cell reports  \n",
       "1867       NaN  2024-01-10  Miyazaki et al. Cell reports  \n",
       "1909       NaN  2024-01-10  Miyazaki et al. Cell reports  \n",
       "1939       NaN  2024-01-10  Miyazaki et al. Cell reports  \n",
       "1966       NaN  2024-01-10  Miyazaki et al. Cell reports  \n",
       "2129       NaN  2024-01-10  Miyazaki et al. Cell reports  \n",
       "2164       NaN  2024-01-10  Miyazaki et al. Cell reports  \n",
       "12124      NaN  2024-01-10  Miyazaki et al. Cell reports  "
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#filter df_finaln for 38194969\n",
    "df_finaln[df_finaln['PMID'] == 38194969]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "f8c5dc3a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>Author</th>\n",
       "      <th>Publication Date</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Date</th>\n",
       "      <th>Publication</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K Krzystanek, HB Rasmussen, M Grunnet, O Staub...</td>\n",
       "      <td>BACKGROUND: KCNQ1 (Kv7.1), together with its K...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=98014...</td>\n",
       "      <td>35.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N37A/10</td>\n",
       "      <td>Kv7.1/KCNQ1 K+ channel</td>\n",
       "      <td>22024150.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2011-10-26</td>\n",
       "      <td>Krzystanek et al. Heart rhythm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Neuroinflammation alters integrative propertie...</td>\n",
       "      <td>https://link.springer.com/article/10.1007/s120...</td>\n",
       "      <td>F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...</td>\n",
       "      <td>… , N212/17, Neuromab, CA, USA) antibodies usi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=19097...</td>\n",
       "      <td>41.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "      <td>29427087.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2018-02-11</td>\n",
       "      <td>Frigerio et al. Molecular neurobiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Allostery between two binding sites in the io...</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...</td>\n",
       "      <td>Tetratricopeptide repeat (TPR) domains are ubi...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=14199...</td>\n",
       "      <td>12.0</td>\n",
       "      <td>https://www.jbc.org/article/S0021-9258(20)3303...</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "      <td>28887304.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2017-09-10</td>\n",
       "      <td>Lyman et al. The Journal of biological chemistry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>The TRIP8b-HCN Interaction as a Therapeutic Ta...</td>\n",
       "      <td>https://search.proquest.com/openview/493f089f1...</td>\n",
       "      <td>KA Lyman - 2016 - search.proquest.com</td>\n",
       "      <td>Hyperpolarization-activated cyclic nucleotide-...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N212/17</td>\n",
       "      <td>TRIP8b (constant)</td>\n",
       "      <td>28127990.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2017-01-28</td>\n",
       "      <td>Lyman et al. Expert opinion on therapeutic tar...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>A complex of Protocadherin-19 and N-cadherin m...</td>\n",
       "      <td>https://rupress.org/jcb/article-abstract/195/7...</td>\n",
       "      <td>MR Emond, S Biswas, CJ Blevins, JD Jontes - Jo...</td>\n",
       "      <td>… N144/14; University of California Davis/Nati...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=11570...</td>\n",
       "      <td>116.0</td>\n",
       "      <td>https://rupress.org/jcb/article/195/7/1115/50544</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "      <td>22184198.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2011-12-21</td>\n",
       "      <td>Emond et al. The Journal of cell biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13007</th>\n",
       "      <td>Kv4 channels underlie A-currents with highly v...</td>\n",
       "      <td>https://link.springer.com/article/10.1007/s004...</td>\n",
       "      <td>C Strube, L Saliba, E Moubarak, V Penalba… - …...</td>\n",
       "      <td>… Immunodetections of K v 4.2 and K v 4.3 chan...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=66496...</td>\n",
       "      <td>15.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>K57/1, K75/41</td>\n",
       "      <td>Kv4.3 K+ channel</td>\n",
       "      <td>24872163.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2014-05-30</td>\n",
       "      <td>Strube et al. Pflugers Archiv : European journ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13010</th>\n",
       "      <td>Expression, cellular and subcellular localisat...</td>\n",
       "      <td>https://www.mdpi.com/1422-0067/20/2/246</td>\n",
       "      <td>R Alfaro-Ruíz, C Aguado, A Martín-Belmonte… - ...</td>\n",
       "      <td>… -S2 loop, Q9NZV8; NeuroMab, UC Davis/NIH, Da...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=84227...</td>\n",
       "      <td>23.0</td>\n",
       "      <td>https://www.mdpi.com/1422-0067/20/2/246/htm</td>\n",
       "      <td>K57/1, K75/41</td>\n",
       "      <td>Kv4.3 K+ channel</td>\n",
       "      <td>30634540.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-01-13</td>\n",
       "      <td>Alfaro-Ruíz et al. International journal of mo...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13013</th>\n",
       "      <td>DPP6 localization in brain supports function ...</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/n...</td>\n",
       "      <td>BD Clark, E Kwon, J Maffie, HY Jeong… - Fronti...</td>\n",
       "      <td>… K75/41 Neuromab; www.neuromab.org were used ...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=15195...</td>\n",
       "      <td>81.0</td>\n",
       "      <td>https://www.frontiersin.org/articles/10.3389/n...</td>\n",
       "      <td>K57/1, K75/41</td>\n",
       "      <td>Kv4.3 K+ channel</td>\n",
       "      <td>18978958.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2008-11-04</td>\n",
       "      <td>Clark et al. Frontiers in molecular neuroscience</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13020</th>\n",
       "      <td>Erbin interacts with TARP γ-2 for surface expr...</td>\n",
       "      <td>https://www.nature.com/articles/nn.3320</td>\n",
       "      <td>Y Tao, YJ Chen, C Shen, Z Luo, CR Bates, D Lee...</td>\n",
       "      <td>Inhibitory neurons control the firing of gluta...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=11307...</td>\n",
       "      <td>51.0</td>\n",
       "      <td>https://www.researchgate.net/profile/Tian-Ming...</td>\n",
       "      <td>N245/36, N245/1</td>\n",
       "      <td>TARPGamma2/Stargazin</td>\n",
       "      <td>23354328.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2013-01-29</td>\n",
       "      <td>Tao et al. Nature neuroscience</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13037</th>\n",
       "      <td>Carbonic anhydrase-related protein CA10 is an ...</td>\n",
       "      <td>https://www.pnas.org/doi/abs/10.1073/pnas.1621...</td>\n",
       "      <td>FH Sterky, JH Trotter, SJ Lee… - Proceedings o...</td>\n",
       "      <td>Establishment, specification, and validation o...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=17357...</td>\n",
       "      <td>91.0</td>\n",
       "      <td>https://www.pnas.org/doi/full/10.1073/pnas.162...</td>\n",
       "      <td>N95/35, N87/25</td>\n",
       "      <td>GABA(A)R, Alpha1</td>\n",
       "      <td>28320946.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2017-03-23</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3401 rows × 15 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                   title  \\\n",
       "0      Deubiquitylating enzyme USP2 counteracts Nedd4...   \n",
       "2      Neuroinflammation alters integrative propertie...   \n",
       "3       Allostery between two binding sites in the io...   \n",
       "5      The TRIP8b-HCN Interaction as a Therapeutic Ta...   \n",
       "6      A complex of Protocadherin-19 and N-cadherin m...   \n",
       "...                                                  ...   \n",
       "13007  Kv4 channels underlie A-currents with highly v...   \n",
       "13010  Expression, cellular and subcellular localisat...   \n",
       "13013   DPP6 localization in brain supports function ...   \n",
       "13020  Erbin interacts with TARP γ-2 for surface expr...   \n",
       "13037  Carbonic anhydrase-related protein CA10 is an ...   \n",
       "\n",
       "                                              title_link  \\\n",
       "0      https://www.sciencedirect.com/science/article/...   \n",
       "2      https://link.springer.com/article/10.1007/s120...   \n",
       "3      https://www.jbc.org/article/S0021-9258(20)3303...   \n",
       "5      https://search.proquest.com/openview/493f089f1...   \n",
       "6      https://rupress.org/jcb/article-abstract/195/7...   \n",
       "...                                                  ...   \n",
       "13007  https://link.springer.com/article/10.1007/s004...   \n",
       "13010            https://www.mdpi.com/1422-0067/20/2/246   \n",
       "13013  https://www.frontiersin.org/articles/10.3389/n...   \n",
       "13020            https://www.nature.com/articles/nn.3320   \n",
       "13037  https://www.pnas.org/doi/abs/10.1073/pnas.1621...   \n",
       "\n",
       "                                        publication_info  \\\n",
       "0      K Krzystanek, HB Rasmussen, M Grunnet, O Staub...   \n",
       "2      F Frigerio, C Flynn, Y Han, K Lyman, JN Lugo… ...   \n",
       "3      KA Lyman, Y Han, RJ Heuermann, X Cheng… - Jour...   \n",
       "5                  KA Lyman - 2016 - search.proquest.com   \n",
       "6      MR Emond, S Biswas, CJ Blevins, JD Jontes - Jo...   \n",
       "...                                                  ...   \n",
       "13007  C Strube, L Saliba, E Moubarak, V Penalba… - …...   \n",
       "13010  R Alfaro-Ruíz, C Aguado, A Martín-Belmonte… - ...   \n",
       "13013  BD Clark, E Kwon, J Maffie, HY Jeong… - Fronti...   \n",
       "13020  Y Tao, YJ Chen, C Shen, Z Luo, CR Bates, D Lee...   \n",
       "13037  FH Sterky, JH Trotter, SJ Lee… - Proceedings o...   \n",
       "\n",
       "                                                 snippet  \\\n",
       "0      BACKGROUND: KCNQ1 (Kv7.1), together with its K...   \n",
       "2      … , N212/17, Neuromab, CA, USA) antibodies usi...   \n",
       "3      Tetratricopeptide repeat (TPR) domains are ubi...   \n",
       "5      Hyperpolarization-activated cyclic nucleotide-...   \n",
       "6      … N144/14; University of California Davis/Nati...   \n",
       "...                                                  ...   \n",
       "13007  … Immunodetections of K v 4.2 and K v 4.3 chan...   \n",
       "13010  … -S2 loop, Q9NZV8; NeuroMab, UC Davis/NIH, Da...   \n",
       "13013  … K75/41 Neuromab; www.neuromab.org were used ...   \n",
       "13020  Inhibitory neurons control the firing of gluta...   \n",
       "13037  Establishment, specification, and validation o...   \n",
       "\n",
       "                                           cited_by_link  cited_by_count  \\\n",
       "0      https://scholar.google.com/scholar?cites=98014...            35.0   \n",
       "2      https://scholar.google.com/scholar?cites=19097...            41.0   \n",
       "3      https://scholar.google.com/scholar?cites=14199...            12.0   \n",
       "5                                                    NaN             NaN   \n",
       "6      https://scholar.google.com/scholar?cites=11570...           116.0   \n",
       "...                                                  ...             ...   \n",
       "13007  https://scholar.google.com/scholar?cites=66496...            15.0   \n",
       "13010  https://scholar.google.com/scholar?cites=84227...            23.0   \n",
       "13013  https://scholar.google.com/scholar?cites=15195...            81.0   \n",
       "13020  https://scholar.google.com/scholar?cites=11307...            51.0   \n",
       "13037  https://scholar.google.com/scholar?cites=17357...            91.0   \n",
       "\n",
       "                                                pdf_file         Antibody  \\\n",
       "0                                                    NaN          N37A/10   \n",
       "2      https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...          N212/17   \n",
       "3      https://www.jbc.org/article/S0021-9258(20)3303...          N212/17   \n",
       "5                                                    NaN          N212/17   \n",
       "6       https://rupress.org/jcb/article/195/7/1115/50544          N144/14   \n",
       "...                                                  ...              ...   \n",
       "13007                                                NaN    K57/1, K75/41   \n",
       "13010        https://www.mdpi.com/1422-0067/20/2/246/htm    K57/1, K75/41   \n",
       "13013  https://www.frontiersin.org/articles/10.3389/n...    K57/1, K75/41   \n",
       "13020  https://www.researchgate.net/profile/Tian-Ming...  N245/36, N245/1   \n",
       "13037  https://www.pnas.org/doi/full/10.1073/pnas.162...   N95/35, N87/25   \n",
       "\n",
       "                       Target        PMID  Author  Publication Date  Journal  \\\n",
       "0      Kv7.1/KCNQ1 K+ channel  22024150.0     NaN               NaN      NaN   \n",
       "2           TRIP8b (constant)  29427087.0     NaN               NaN      NaN   \n",
       "3           TRIP8b (constant)  28887304.0     NaN               NaN      NaN   \n",
       "5           TRIP8b (constant)  28127990.0     NaN               NaN      NaN   \n",
       "6                       6xHis  22184198.0     NaN               NaN      NaN   \n",
       "...                       ...         ...     ...               ...      ...   \n",
       "13007        Kv4.3 K+ channel  24872163.0     NaN               NaN      NaN   \n",
       "13010        Kv4.3 K+ channel  30634540.0     NaN               NaN      NaN   \n",
       "13013        Kv4.3 K+ channel  18978958.0     NaN               NaN      NaN   \n",
       "13020    TARPGamma2/Stargazin  23354328.0     NaN               NaN      NaN   \n",
       "13037        GABA(A)R, Alpha1  28320946.0     NaN               NaN      NaN   \n",
       "\n",
       "             Date                                        Publication  \n",
       "0      2011-10-26                     Krzystanek et al. Heart rhythm  \n",
       "2      2018-02-11             Frigerio et al. Molecular neurobiology  \n",
       "3      2017-09-10   Lyman et al. The Journal of biological chemistry  \n",
       "5      2017-01-28  Lyman et al. Expert opinion on therapeutic tar...  \n",
       "6      2011-12-21           Emond et al. The Journal of cell biology  \n",
       "...           ...                                                ...  \n",
       "13007  2014-05-30  Strube et al. Pflugers Archiv : European journ...  \n",
       "13010  2019-01-13  Alfaro-Ruíz et al. International journal of mo...  \n",
       "13013  2008-11-04   Clark et al. Frontiers in molecular neuroscience  \n",
       "13020  2013-01-29                     Tao et al. Nature neuroscience  \n",
       "13037  2017-03-23                                                NaN  \n",
       "\n",
       "[3401 rows x 15 columns]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# filte to only things with a pubmed id\n",
    "df_finaln = df_finaln[df_finaln['PMID'].notna()]\n",
    "df_finaln"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "4c6fda3b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# output that to a csv file\n",
    "df_finaln.to_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/googlescholar_mabidsearch_pubmed_5-20-24.csv\", index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5f6972f8",
   "metadata": {},
   "source": [
    "# Scrape the antibodies Inc Pages for all the entities to check."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "8c1bd84c",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-fa80db5e0aff>:21: DeprecationWarning: headless property is deprecated, instead use add_argument('--headless') or add_argument('--headless=new')\n",
      "  options.headless = True\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_1_490.csv already exists. Skipping page 1.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_2_490.csv already exists. Skipping page 2.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_3_490.csv already exists. Skipping page 3.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_4_490.csv already exists. Skipping page 4.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_5_490.csv already exists. Skipping page 5.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_6_490.csv already exists. Skipping page 6.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_7_490.csv already exists. Skipping page 7.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_8_490.csv already exists. Skipping page 8.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_9_490.csv already exists. Skipping page 9.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_10_490.csv already exists. Skipping page 10.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_11_490.csv already exists. Skipping page 11.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_12_490.csv already exists. Skipping page 12.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_13_490.csv already exists. Skipping page 13.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_14_490.csv already exists. Skipping page 14.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_15_490.csv already exists. Skipping page 15.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_16_490.csv already exists. Skipping page 16.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_17_490.csv already exists. Skipping page 17.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_18_490.csv already exists. Skipping page 18.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_19_490.csv already exists. Skipping page 19.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_20_490.csv already exists. Skipping page 20.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_21_490.csv already exists. Skipping page 21.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_22_490.csv already exists. Skipping page 22.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_23_490.csv already exists. Skipping page 23.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_24_490.csv already exists. Skipping page 24.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_25_490.csv already exists. Skipping page 25.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_26_490.csv already exists. Skipping page 26.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_27_490.csv already exists. Skipping page 27.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_28_490.csv already exists. Skipping page 28.\n",
      "Processing page 29...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_29_490.csv\n",
      "Processing page 30...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_30_490.csv\n",
      "Processing page 31...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_31_490.csv\n",
      "Processing page 32...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_32_490.csv\n",
      "Processing page 33...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_33_490.csv\n",
      "Processing page 34...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_34_490.csv\n",
      "Processing page 35...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_35_490.csv\n",
      "Processing page 36...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_36_490.csv\n",
      "Processing page 37...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_37_490.csv\n",
      "Processing page 38...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_38_490.csv\n",
      "Processing page 39...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_39_490.csv\n",
      "Processing page 40...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_40_490.csv\n",
      "Processing page 41...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_41_490.csv\n",
      "Processing page 42...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_42_490.csv\n",
      "Processing page 43...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_43_490.csv\n",
      "Processing page 44...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_44_490.csv\n",
      "Processing page 45...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_45_490.csv\n",
      "Processing page 46...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_46_490.csv\n",
      "Processing page 47...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_47_490.csv\n",
      "Processing page 48...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_48_490.csv\n",
      "Processing page 49...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_49_490.csv\n",
      "Processing page 50...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_50_490.csv\n",
      "Processing page 51...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_51_490.csv\n",
      "Processing page 52...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_52_490.csv\n",
      "Processing page 53...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_53_490.csv\n",
      "Processing page 54...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_54_490.csv\n",
      "Processing page 55...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_55_490.csv\n",
      "Processing page 56...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_56_490.csv\n",
      "Processing page 57...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_57_490.csv\n",
      "Processing page 58...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_58_490.csv\n",
      "Processing page 59...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_59_490.csv\n",
      "Processing page 60...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_60_490.csv\n",
      "Processing page 61...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_61_490.csv\n",
      "Processing page 62...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_62_490.csv\n",
      "Processing page 63...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_63_490.csv\n",
      "Processing page 64...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_64_490.csv\n",
      "Processing page 65...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_65_490.csv\n",
      "Processing page 66...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_66_490.csv\n",
      "Processing page 67...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_67_490.csv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-fa80db5e0aff>:21: DeprecationWarning: headless property is deprecated, instead use add_argument('--headless') or add_argument('--headless=new')\n",
      "  options.headless = True\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_1_Unconjugated.csv already exists. Skipping page 1.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_2_Unconjugated.csv already exists. Skipping page 2.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_3_Unconjugated.csv already exists. Skipping page 3.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_4_Unconjugated.csv already exists. Skipping page 4.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_5_Unconjugated.csv already exists. Skipping page 5.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_6_Unconjugated.csv already exists. Skipping page 6.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_7_Unconjugated.csv already exists. Skipping page 7.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_8_Unconjugated.csv already exists. Skipping page 8.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_9_Unconjugated.csv already exists. Skipping page 9.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_10_Unconjugated.csv already exists. Skipping page 10.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_11_Unconjugated.csv already exists. Skipping page 11.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_12_Unconjugated.csv already exists. Skipping page 12.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_13_Unconjugated.csv already exists. Skipping page 13.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_14_Unconjugated.csv already exists. Skipping page 14.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_15_Unconjugated.csv already exists. Skipping page 15.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_16_Unconjugated.csv already exists. Skipping page 16.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_17_Unconjugated.csv already exists. Skipping page 17.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_18_Unconjugated.csv already exists. Skipping page 18.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_19_Unconjugated.csv already exists. Skipping page 19.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_20_Unconjugated.csv already exists. Skipping page 20.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_21_Unconjugated.csv already exists. Skipping page 21.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_22_Unconjugated.csv already exists. Skipping page 22.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_23_Unconjugated.csv already exists. Skipping page 23.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_24_Unconjugated.csv already exists. Skipping page 24.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_25_Unconjugated.csv already exists. Skipping page 25.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_26_Unconjugated.csv already exists. Skipping page 26.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_27_Unconjugated.csv already exists. Skipping page 27.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_28_Unconjugated.csv already exists. Skipping page 28.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_29_Unconjugated.csv already exists. Skipping page 29.\n",
      "Processing page 30...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_30_Unconjugated.csv\n",
      "Processing page 31...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_31_Unconjugated.csv\n",
      "Processing page 32...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_32_Unconjugated.csv\n",
      "Processing page 33...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_33_Unconjugated.csv\n",
      "Processing page 34...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_34_Unconjugated.csv\n",
      "Processing page 35...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_35_Unconjugated.csv\n",
      "Processing page 36...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_36_Unconjugated.csv\n",
      "Processing page 37...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_37_Unconjugated.csv\n",
      "Processing page 38...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_38_Unconjugated.csv\n",
      "Processing page 39...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_39_Unconjugated.csv\n",
      "Processing page 40...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_40_Unconjugated.csv\n",
      "Processing page 41...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_41_Unconjugated.csv\n",
      "Processing page 42...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_42_Unconjugated.csv\n",
      "Processing page 43...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_43_Unconjugated.csv\n",
      "Processing page 44...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_44_Unconjugated.csv\n",
      "Processing page 45...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_45_Unconjugated.csv\n",
      "Processing page 46...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_46_Unconjugated.csv\n",
      "Processing page 47...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_47_Unconjugated.csv\n",
      "Processing page 48...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_48_Unconjugated.csv\n",
      "Processing page 49...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_49_Unconjugated.csv\n",
      "Processing page 50...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_50_Unconjugated.csv\n",
      "Processing page 51...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_51_Unconjugated.csv\n",
      "Processing page 52...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_52_Unconjugated.csv\n",
      "Processing page 53...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_53_Unconjugated.csv\n",
      "Processing page 54...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_54_Unconjugated.csv\n",
      "Processing page 55...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_55_Unconjugated.csv\n",
      "Processing page 56...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_56_Unconjugated.csv\n",
      "Processing page 57...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_57_Unconjugated.csv\n",
      "Processing page 58...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_58_Unconjugated.csv\n",
      "Processing page 59...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_59_Unconjugated.csv\n",
      "Processing page 60...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_60_Unconjugated.csv\n",
      "Processing page 61...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_61_Unconjugated.csv\n",
      "Processing page 62...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_62_Unconjugated.csv\n",
      "Processing page 63...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_63_Unconjugated.csv\n",
      "Processing page 64...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_64_Unconjugated.csv\n",
      "Processing page 65...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_65_Unconjugated.csv\n",
      "Processing page 66...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_66_Unconjugated.csv\n",
      "Processing page 67...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_67_Unconjugated.csv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-fa80db5e0aff>:21: DeprecationWarning: headless property is deprecated, instead use add_argument('--headless') or add_argument('--headless=new')\n",
      "  options.headless = True\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_1_550.csv already exists. Skipping page 1.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_2_550.csv already exists. Skipping page 2.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_3_550.csv already exists. Skipping page 3.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_4_550.csv already exists. Skipping page 4.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_5_550.csv already exists. Skipping page 5.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_6_550.csv already exists. Skipping page 6.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_7_550.csv already exists. Skipping page 7.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_8_550.csv already exists. Skipping page 8.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_9_550.csv already exists. Skipping page 9.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_10_550.csv already exists. Skipping page 10.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_11_550.csv already exists. Skipping page 11.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_12_550.csv already exists. Skipping page 12.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_13_550.csv already exists. Skipping page 13.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_14_550.csv already exists. Skipping page 14.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_15_550.csv already exists. Skipping page 15.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_16_550.csv already exists. Skipping page 16.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_17_550.csv already exists. Skipping page 17.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_18_550.csv already exists. Skipping page 18.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_19_550.csv already exists. Skipping page 19.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_20_550.csv already exists. Skipping page 20.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_21_550.csv already exists. Skipping page 21.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_22_550.csv already exists. Skipping page 22.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_23_550.csv already exists. Skipping page 23.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_24_550.csv already exists. Skipping page 24.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_25_550.csv already exists. Skipping page 25.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_26_550.csv already exists. Skipping page 26.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_27_550.csv already exists. Skipping page 27.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_28_550.csv already exists. Skipping page 28.\n",
      "Processing page 29...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_29_550.csv\n",
      "Processing page 30...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_30_550.csv\n",
      "Processing page 31...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_31_550.csv\n",
      "Processing page 32...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_32_550.csv\n",
      "Processing page 33...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_33_550.csv\n",
      "Processing page 34...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_34_550.csv\n",
      "Processing page 35...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_35_550.csv\n",
      "Processing page 36...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_36_550.csv\n",
      "Processing page 37...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_37_550.csv\n",
      "Processing page 38...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_38_550.csv\n",
      "Processing page 39...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_39_550.csv\n",
      "Processing page 40...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_40_550.csv\n",
      "Processing page 41...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_41_550.csv\n",
      "Processing page 42...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_42_550.csv\n",
      "Processing page 43...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_43_550.csv\n",
      "Processing page 44...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_44_550.csv\n",
      "Processing page 45...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_45_550.csv\n",
      "Processing page 46...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_46_550.csv\n",
      "Processing page 47...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_47_550.csv\n",
      "Processing page 48...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_48_550.csv\n",
      "Processing page 49...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_49_550.csv\n",
      "Processing page 50...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_50_550.csv\n",
      "Processing page 51...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_51_550.csv\n",
      "Processing page 52...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_52_550.csv\n",
      "Processing page 53...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_53_550.csv\n",
      "Processing page 54...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_54_550.csv\n",
      "Processing page 55...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_55_550.csv\n",
      "Processing page 56...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_56_550.csv\n",
      "Processing page 57...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_57_550.csv\n",
      "Processing page 58...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_58_550.csv\n",
      "Processing page 59...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_59_550.csv\n",
      "Processing page 60...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_60_550.csv\n",
      "Processing page 61...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_61_550.csv\n",
      "Processing page 62...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_62_550.csv\n",
      "Processing page 63...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_63_550.csv\n",
      "Processing page 64...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_64_550.csv\n",
      "Processing page 65...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_65_550.csv\n",
      "Processing page 66...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_66_550.csv\n",
      "Processing page 67...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_67_550.csv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-fa80db5e0aff>:21: DeprecationWarning: headless property is deprecated, instead use add_argument('--headless') or add_argument('--headless=new')\n",
      "  options.headless = True\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_1_594.csv already exists. Skipping page 1.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_2_594.csv already exists. Skipping page 2.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_3_594.csv already exists. Skipping page 3.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_4_594.csv already exists. Skipping page 4.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_5_594.csv already exists. Skipping page 5.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_6_594.csv already exists. Skipping page 6.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_7_594.csv already exists. Skipping page 7.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_8_594.csv already exists. Skipping page 8.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_9_594.csv already exists. Skipping page 9.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_10_594.csv already exists. Skipping page 10.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_11_594.csv already exists. Skipping page 11.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_12_594.csv already exists. Skipping page 12.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_13_594.csv already exists. Skipping page 13.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_14_594.csv already exists. Skipping page 14.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_15_594.csv already exists. Skipping page 15.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_16_594.csv already exists. Skipping page 16.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_17_594.csv already exists. Skipping page 17.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_18_594.csv already exists. Skipping page 18.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_19_594.csv already exists. Skipping page 19.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_20_594.csv already exists. Skipping page 20.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_21_594.csv already exists. Skipping page 21.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_22_594.csv already exists. Skipping page 22.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_23_594.csv already exists. Skipping page 23.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_24_594.csv already exists. Skipping page 24.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_25_594.csv already exists. Skipping page 25.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_26_594.csv already exists. Skipping page 26.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_27_594.csv already exists. Skipping page 27.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_28_594.csv already exists. Skipping page 28.\n",
      "Processing page 29...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_29_594.csv\n",
      "Processing page 30...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_30_594.csv\n",
      "Processing page 31...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_31_594.csv\n",
      "Processing page 32...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_32_594.csv\n",
      "Processing page 33...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_33_594.csv\n",
      "Processing page 34...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_34_594.csv\n",
      "Processing page 35...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_35_594.csv\n",
      "Processing page 36...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_36_594.csv\n",
      "Processing page 37...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_37_594.csv\n",
      "Processing page 38...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_38_594.csv\n",
      "Processing page 39...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_39_594.csv\n",
      "Processing page 40...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_40_594.csv\n",
      "Processing page 41...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_41_594.csv\n",
      "Processing page 42...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_42_594.csv\n",
      "Processing page 43...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_43_594.csv\n",
      "Processing page 44...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_44_594.csv\n",
      "Processing page 45...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_45_594.csv\n",
      "Processing page 46...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_46_594.csv\n",
      "Processing page 47...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_47_594.csv\n",
      "Processing page 48...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_48_594.csv\n",
      "Processing page 49...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_49_594.csv\n",
      "Processing page 50...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_50_594.csv\n",
      "Processing page 51...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_51_594.csv\n",
      "Processing page 52...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_52_594.csv\n",
      "Processing page 53...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_53_594.csv\n",
      "Processing page 54...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_54_594.csv\n",
      "Processing page 55...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_55_594.csv\n",
      "Processing page 56...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_56_594.csv\n",
      "Processing page 57...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_57_594.csv\n",
      "Processing page 58...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_58_594.csv\n",
      "Processing page 59...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_59_594.csv\n",
      "Processing page 60...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_60_594.csv\n",
      "Processing page 61...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_61_594.csv\n",
      "Processing page 62...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_62_594.csv\n",
      "Processing page 63...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_63_594.csv\n",
      "Processing page 64...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_64_594.csv\n",
      "Processing page 65...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_65_594.csv\n",
      "Processing page 66...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_66_594.csv\n",
      "Processing page 67...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_67_594.csv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-fa80db5e0aff>:21: DeprecationWarning: headless property is deprecated, instead use add_argument('--headless') or add_argument('--headless=new')\n",
      "  options.headless = True\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_1_650.csv already exists. Skipping page 1.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_2_650.csv already exists. Skipping page 2.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_3_650.csv already exists. Skipping page 3.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_4_650.csv already exists. Skipping page 4.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_5_650.csv already exists. Skipping page 5.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_6_650.csv already exists. Skipping page 6.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_7_650.csv already exists. Skipping page 7.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_8_650.csv already exists. Skipping page 8.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_9_650.csv already exists. Skipping page 9.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_10_650.csv already exists. Skipping page 10.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_11_650.csv already exists. Skipping page 11.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_12_650.csv already exists. Skipping page 12.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_13_650.csv already exists. Skipping page 13.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_14_650.csv already exists. Skipping page 14.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_15_650.csv already exists. Skipping page 15.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_16_650.csv already exists. Skipping page 16.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_17_650.csv already exists. Skipping page 17.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_18_650.csv already exists. Skipping page 18.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_19_650.csv already exists. Skipping page 19.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_20_650.csv already exists. Skipping page 20.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_21_650.csv already exists. Skipping page 21.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_22_650.csv already exists. Skipping page 22.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_23_650.csv already exists. Skipping page 23.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_24_650.csv already exists. Skipping page 24.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_25_650.csv already exists. Skipping page 25.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_26_650.csv already exists. Skipping page 26.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_27_650.csv already exists. Skipping page 27.\n",
      "File /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_28_650.csv already exists. Skipping page 28.\n",
      "Processing page 29...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_29_650.csv\n",
      "Processing page 30...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_30_650.csv\n",
      "Processing page 31...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_31_650.csv\n",
      "Processing page 32...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_32_650.csv\n",
      "Processing page 33...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_33_650.csv\n",
      "Processing page 34...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_34_650.csv\n",
      "Processing page 35...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_35_650.csv\n",
      "Processing page 36...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_36_650.csv\n",
      "Processing page 37...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_37_650.csv\n",
      "Processing page 38...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_38_650.csv\n",
      "Processing page 39...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_39_650.csv\n",
      "Processing page 40...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_40_650.csv\n",
      "Processing page 41...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_41_650.csv\n",
      "Processing page 42...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_42_650.csv\n",
      "Processing page 43...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_43_650.csv\n",
      "Processing page 44...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_44_650.csv\n",
      "Processing page 45...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_45_650.csv\n",
      "Processing page 46...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_46_650.csv\n",
      "Processing page 47...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_47_650.csv\n",
      "Processing page 48...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_48_650.csv\n",
      "Processing page 49...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_49_650.csv\n",
      "Processing page 50...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_50_650.csv\n",
      "Processing page 51...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_51_650.csv\n",
      "Processing page 52...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_52_650.csv\n",
      "Processing page 53...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_53_650.csv\n",
      "Processing page 54...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_54_650.csv\n",
      "Processing page 55...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_55_650.csv\n",
      "Processing page 56...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_56_650.csv\n",
      "Processing page 57...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_57_650.csv\n",
      "Processing page 58...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_58_650.csv\n",
      "Processing page 59...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_59_650.csv\n",
      "Processing page 60...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_60_650.csv\n",
      "Processing page 61...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_61_650.csv\n",
      "Processing page 62...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_62_650.csv\n",
      "Processing page 63...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_63_650.csv\n",
      "Processing page 64...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_64_650.csv\n",
      "Processing page 65...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_65_650.csv\n",
      "Processing page 66...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_66_650.csv\n",
      "Processing page 67...\n",
      "Saved /Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages//page_67_650.csv\n"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "import pandas as pd\n",
    "import os\n",
    "from selenium import webdriver\n",
    "from selenium.webdriver.chrome.options import Options\n",
    "from bs4 import BeautifulSoup\n",
    "from selenium import webdriver\n",
    "import pandas as pd\n",
    "from selenium import webdriver\n",
    "from selenium.webdriver.chrome.options import Options\n",
    "import re\n",
    "import os\n",
    "\n",
    "def scrape_matching_links(base_url, num_items, items_per_page, url_pattern, output_dir, conj):\n",
    "    if not os.path.exists(output_dir):\n",
    "        os.makedirs(output_dir)\n",
    "\n",
    "    total_pages = (num_items // items_per_page) + (1 if num_items % items_per_page else 0)\n",
    "    options = Options()\n",
    "    options.headless = True\n",
    "    browser = webdriver.Chrome(options=options)\n",
    "\n",
    "    for page in range(1, total_pages + 1):\n",
    "        output_file = f\"{output_dir}/page_{page}_{conj}.csv\"\n",
    "        if os.path.exists(output_file):\n",
    "            print(f\"File {output_file} already exists. Skipping page {page}.\")\n",
    "            continue\n",
    "\n",
    "        print(f\"Processing page {page}...\")\n",
    "        url = f\"{base_url}&page={page}\"\n",
    "        browser.get(url)\n",
    "        html = browser.page_source\n",
    "\n",
    "        if not html or \"No results found\" in html:\n",
    "            print(\"Empty response or no results found. Terminating script.\")\n",
    "            browser.quit()\n",
    "            return\n",
    "\n",
    "        soup = BeautifulSoup(html, 'html.parser')\n",
    "        links = [f\"https://www.antibodiesinc.com{link['href']}\" for link in soup.find_all('a', href=True) if url_pattern in link['href']]\n",
    "\n",
    "        if not links:\n",
    "            print(\"No links found on this page. Terminating script.\")\n",
    "            browser.quit()\n",
    "            return\n",
    "\n",
    "        df = pd.DataFrame(links, columns=['Link'])\n",
    "        df.to_csv(output_file, index=False)\n",
    "        print(f\"Saved {output_file}\")\n",
    "\n",
    "    browser.quit()\n",
    "\n",
    "# Modify these parameters according to your needs\n",
    "#conj_list = ['490', 'Unconjugated', \"550\", \"594\", \"650\"]\n",
    "conj_list = ['490', 'Unconjugated', \"550\", \"594\", \"650\"]\n",
    "\n",
    "for conj in conj_list:\n",
    "    base_url = f\"https://www.antibodiesinc.com/search?refinementList%5Bvendor%5D%5B0%5D=NeuroMab&refinementList%5Bnamed_tags.conj%5D%5B0%5D={conj}\"\n",
    "    num_items = 1000\n",
    "    items_per_page = 15\n",
    "    url_pattern = \"products\"\n",
    "    output_dir = \"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages/\"\n",
    "\n",
    "    scrape_matching_links(base_url, num_items, items_per_page, url_pattern, output_dir, conj)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "15b4fa12",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n",
      "<ipython-input-24-68d128074f2d>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append(dfi)\n"
     ]
    }
   ],
   "source": [
    "# create a master dataframe of all the pmids so we can track the index\n",
    "df = pd.DataFrame()\n",
    "for file in glob.glob(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pages/*.csv\"):\n",
    "    dfi = pd.read_csv(file)\n",
    "    df = df.append(dfi)\n",
    "    df = df.reset_index(drop=True)\n",
    "# output this file\n",
    "df.to_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/antibodiesinc_master-5-20-24.csv\", index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "79ff283a",
   "metadata": {},
   "source": [
    "# Now we need to get the citations from antibodies inc for each of these."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "b628fbc4",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "from selenium import webdriver\n",
    "from selenium.webdriver.chrome.options import Options\n",
    "import re\n",
    "import os\n",
    "import time\n",
    "import random\n",
    "\n",
    "def extract_pmids(urls, base_output_dir, start_index, overwrite):\n",
    "    output_dir = os.path.join(base_output_dir, 'pmid_output')\n",
    "    if not os.path.exists(output_dir):\n",
    "        os.makedirs(output_dir)\n",
    "\n",
    "    options = Options()\n",
    "    options.headless = True\n",
    "    options.add_argument(\"window-size=1920x1080\")\n",
    "\n",
    "    user_agents = [\n",
    "        'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3',\n",
    "        # Add more user agents here\n",
    "    ]\n",
    "\n",
    "    browser = webdriver.Chrome(options=options)\n",
    "    number_of_0s = 170\n",
    "    number_of_0s_count = 0\n",
    "    # start at the index provided \n",
    "    for index, url in enumerate(urls[start_index:]):\n",
    "        print(f\"Processing URL {url}...\")\n",
    "        # check if file already exists\n",
    "        output_file = os.path.join(output_dir, f\"{url.split('/')[-1]}.csv\")\n",
    "        if os.path.exists(output_file) and not overwrite:\n",
    "            print(f\"File {output_file} already exists. Skipping URL {url}.\")\n",
    "            continue\n",
    "\n",
    "        # Rotate user agent\n",
    "        user_agent = random.choice(user_agents)\n",
    "        browser.execute_cdp_cmd('Network.setUserAgentOverride', {\"userAgent\": user_agent})\n",
    "        browser.get(url)\n",
    "        html = browser.page_source\n",
    "\n",
    "        pmid_pattern = re.compile(r'https://pubmed.ncbi.nlm.nih.gov/\\d+')\n",
    "        pmid_patter2 = re.compile(r'https://www.ncbi.nlm.nih.gov/pubmed/\\d+')\n",
    "        pmids = pmid_pattern.findall(html)\n",
    "        pmids += pmid_patter2.findall(html)\n",
    "        # also grab the Citations count  and the number in the () following it from the html\n",
    "        citations_pattern = re.compile(r'Citations \\(\\d+\\)')\n",
    "        citations = citations_pattern.findall(html)\n",
    "        \n",
    "        if len(citations) >= 1:\n",
    "            print(citations, len(pmids))\n",
    "            print(pmids)\n",
    "            assert len(citations) == 1\n",
    "            #assert len(pmids) == int(citations[0].split('(')[1].split(')')[0])\n",
    "        # if not pmids:\n",
    "        #     print(f\"No PMIDs found for {url}. Skipping.\")\n",
    "        #     continue\n",
    "        if len(pmids) == 0:\n",
    "            number_of_0s_count += 1\n",
    "            if number_of_0s_count >= number_of_0s:\n",
    "                print(f\"Too many 0s in a row, resetting vpn\")\n",
    "                return (index - number_of_0s) + start_index\n",
    "        else:\n",
    "            number_of_0s_count = 0\n",
    "        pmid_data = {'URL': [], 'PMID': []}\n",
    "        for pmid in pmids:\n",
    "            pmid_data['URL'].append(url)\n",
    "            pmid_data['PMID'].append(pmid)\n",
    "        # make this long when updating the daatafraame\n",
    "        df = pd.DataFrame(pmid_data)\n",
    "        df.to_csv(output_file, index=False)\n",
    "\n",
    "        print(f\"Saved {output_file}\")\n",
    "\n",
    "        # Delay to prevent triggering anti-bot mechanisms\n",
    "        time.sleep(random.uniform(5, 10))\n",
    "\n",
    "    browser.quit()\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "1aad5997",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-25-9ece17db1bde>:15: DeprecationWarning: headless property is deprecated, instead use add_argument('--headless') or add_argument('--headless=new')\n",
      "  options.headless = True\n"
     ]
    },
    {
     "ename": "WebDriverException",
     "evalue": "Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home\n",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mSeleniumManagerException\u001b[0m                  Traceback (most recent call last)",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/selenium/webdriver/common/service.py:95\u001b[0m, in \u001b[0;36mService.start\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     94\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m---> 95\u001b[0m     path \u001b[38;5;241m=\u001b[39m \u001b[43mSeleniumManager\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mdriver_location\u001b[49m\u001b[43m(\u001b[49m\u001b[43mbrowser\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     96\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m WebDriverException \u001b[38;5;28;01mas\u001b[39;00m new_err:\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/selenium/webdriver/common/selenium_manager.py:74\u001b[0m, in \u001b[0;36mSeleniumManager.driver_location\u001b[0;34m(self, browser)\u001b[0m\n\u001b[1;32m     73\u001b[0m binary, flag, browser \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mstr\u001b[39m(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mget_binary()), \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m--browser\u001b[39m\u001b[38;5;124m\"\u001b[39m, browser\n\u001b[0;32m---> 74\u001b[0m result \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrun\u001b[49m\u001b[43m(\u001b[49m\u001b[43m(\u001b[49m\u001b[43mbinary\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mflag\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mbrowser\u001b[49m\u001b[43m)\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     75\u001b[0m executable \u001b[38;5;241m=\u001b[39m result\u001b[38;5;241m.\u001b[39msplit(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;130;01m\\t\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)[\u001b[38;5;241m-\u001b[39m\u001b[38;5;241m1\u001b[39m]\u001b[38;5;241m.\u001b[39mstrip()\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/selenium/webdriver/common/selenium_manager.py:93\u001b[0m, in \u001b[0;36mSeleniumManager.run\u001b[0;34m(args)\u001b[0m\n\u001b[1;32m     92\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m completed_proc\u001b[38;5;241m.\u001b[39mreturncode:\n\u001b[0;32m---> 93\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m SeleniumManagerException(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mSelenium manager failed for: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00mcommand\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m.\u001b[39m\u001b[38;5;130;01m\\n\u001b[39;00m\u001b[38;5;132;01m{\u001b[39;00mstdout\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;132;01m{\u001b[39;00mstderr\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m     94\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m     95\u001b[0m     \u001b[38;5;66;03m# selenium manager exited 0 successfully, parse the executable path from stdout.\u001b[39;00m\n",
      "\u001b[0;31mSeleniumManagerException\u001b[0m: Message: Selenium manager failed for: /opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/selenium/webdriver/common/macos/selenium-manager --browser chrome.\nERROR\tWrong browser/driver version\n",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mWebDriverException\u001b[0m                        Traceback (most recent call last)",
      "Input \u001b[0;32mIn [30]\u001b[0m, in \u001b[0;36m<cell line: 7>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      5\u001b[0m output_dir \u001b[38;5;241m=\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m      6\u001b[0m start_index \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m0\u001b[39m\n\u001b[0;32m----> 7\u001b[0m results_index \u001b[38;5;241m=\u001b[39m \u001b[43mextract_pmids\u001b[49m\u001b[43m(\u001b[49m\u001b[43murls\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43moutput_dir\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mstart_index\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43moverwrite\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mTrue\u001b[39;49;00m\u001b[43m)\u001b[49m\n\u001b[1;32m      8\u001b[0m \u001b[38;5;28mprint\u001b[39m(results_index)\n",
      "Input \u001b[0;32mIn [25]\u001b[0m, in \u001b[0;36mextract_pmids\u001b[0;34m(urls, base_output_dir, start_index, overwrite)\u001b[0m\n\u001b[1;32m     16\u001b[0m options\u001b[38;5;241m.\u001b[39madd_argument(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mwindow-size=1920x1080\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m     18\u001b[0m user_agents \u001b[38;5;241m=\u001b[39m [\n\u001b[1;32m     19\u001b[0m     \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mMozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3\u001b[39m\u001b[38;5;124m'\u001b[39m,\n\u001b[1;32m     20\u001b[0m     \u001b[38;5;66;03m# Add more user agents here\u001b[39;00m\n\u001b[1;32m     21\u001b[0m ]\n\u001b[0;32m---> 23\u001b[0m browser \u001b[38;5;241m=\u001b[39m \u001b[43mwebdriver\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mChrome\u001b[49m\u001b[43m(\u001b[49m\u001b[43moptions\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43moptions\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     24\u001b[0m number_of_0s \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m170\u001b[39m\n\u001b[1;32m     25\u001b[0m number_of_0s_count \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m0\u001b[39m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/selenium/webdriver/chrome/webdriver.py:80\u001b[0m, in \u001b[0;36mWebDriver.__init__\u001b[0;34m(self, executable_path, port, options, service_args, desired_capabilities, service_log_path, chrome_options, service, keep_alive)\u001b[0m\n\u001b[1;32m     77\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m service:\n\u001b[1;32m     78\u001b[0m     service \u001b[38;5;241m=\u001b[39m Service(executable_path, port, service_args, service_log_path)\n\u001b[0;32m---> 80\u001b[0m \u001b[38;5;28;43msuper\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[38;5;21;43m__init__\u001b[39;49m\u001b[43m(\u001b[49m\n\u001b[1;32m     81\u001b[0m \u001b[43m    \u001b[49m\u001b[43mDesiredCapabilities\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mCHROME\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mbrowserName\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     82\u001b[0m \u001b[43m    \u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mgoog\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m,\u001b[49m\n\u001b[1;32m     83\u001b[0m \u001b[43m    \u001b[49m\u001b[43mport\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     84\u001b[0m \u001b[43m    \u001b[49m\u001b[43moptions\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     85\u001b[0m \u001b[43m    \u001b[49m\u001b[43mservice_args\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     86\u001b[0m \u001b[43m    \u001b[49m\u001b[43mdesired_capabilities\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     87\u001b[0m \u001b[43m    \u001b[49m\u001b[43mservice_log_path\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     88\u001b[0m \u001b[43m    \u001b[49m\u001b[43mservice\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     89\u001b[0m \u001b[43m    \u001b[49m\u001b[43mkeep_alive\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     90\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/selenium/webdriver/chromium/webdriver.py:101\u001b[0m, in \u001b[0;36mChromiumDriver.__init__\u001b[0;34m(self, browser_name, vendor_prefix, port, options, service_args, desired_capabilities, service_log_path, service, keep_alive)\u001b[0m\n\u001b[1;32m     98\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mAttributeError\u001b[39;00m(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mservice cannot be None\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    100\u001b[0m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mservice \u001b[38;5;241m=\u001b[39m service\n\u001b[0;32m--> 101\u001b[0m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mservice\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mstart\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    103\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m    104\u001b[0m     \u001b[38;5;28msuper\u001b[39m()\u001b[38;5;241m.\u001b[39m\u001b[38;5;21m__init__\u001b[39m(\n\u001b[1;32m    105\u001b[0m         command_executor\u001b[38;5;241m=\u001b[39mChromiumRemoteConnection(\n\u001b[1;32m    106\u001b[0m             remote_server_addr\u001b[38;5;241m=\u001b[39m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mservice\u001b[38;5;241m.\u001b[39mservice_url,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    112\u001b[0m         options\u001b[38;5;241m=\u001b[39moptions,\n\u001b[1;32m    113\u001b[0m     )\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/selenium/webdriver/common/service.py:98\u001b[0m, in \u001b[0;36mService.start\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     96\u001b[0m         \u001b[38;5;28;01mexcept\u001b[39;00m WebDriverException \u001b[38;5;28;01mas\u001b[39;00m new_err:\n\u001b[1;32m     97\u001b[0m             logger\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mUnable to obtain driver using Selenium Manager: \u001b[39m\u001b[38;5;124m\"\u001b[39m \u001b[38;5;241m+\u001b[39m new_err\u001b[38;5;241m.\u001b[39mmsg)\n\u001b[0;32m---> 98\u001b[0m             \u001b[38;5;28;01mraise\u001b[39;00m err\n\u001b[1;32m    100\u001b[0m         \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_start_process(path)\n\u001b[1;32m    102\u001b[0m count \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m0\u001b[39m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/selenium/webdriver/common/service.py:88\u001b[0m, in \u001b[0;36mService.start\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     81\u001b[0m \u001b[38;5;250m\u001b[39m\u001b[38;5;124;03m\"\"\"Starts the Service.\u001b[39;00m\n\u001b[1;32m     82\u001b[0m \n\u001b[1;32m     83\u001b[0m \u001b[38;5;124;03m:Exceptions:\u001b[39;00m\n\u001b[1;32m     84\u001b[0m \u001b[38;5;124;03m - WebDriverException : Raised either when it can't start the service\u001b[39;00m\n\u001b[1;32m     85\u001b[0m \u001b[38;5;124;03m   or when it can't connect to the service\u001b[39;00m\n\u001b[1;32m     86\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m     87\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m---> 88\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_start_process\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mpath\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     89\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m WebDriverException \u001b[38;5;28;01mas\u001b[39;00m err:\n\u001b[1;32m     90\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mexecutable needs to be in PATH\u001b[39m\u001b[38;5;124m\"\u001b[39m \u001b[38;5;129;01min\u001b[39;00m err\u001b[38;5;241m.\u001b[39mmsg:\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/selenium/webdriver/common/service.py:209\u001b[0m, in \u001b[0;36mService._start_process\u001b[0;34m(self, path)\u001b[0m\n\u001b[1;32m    207\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mOSError\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m err:\n\u001b[1;32m    208\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m err\u001b[38;5;241m.\u001b[39merrno \u001b[38;5;241m==\u001b[39m errno\u001b[38;5;241m.\u001b[39mENOENT:\n\u001b[0;32m--> 209\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m WebDriverException(\n\u001b[1;32m    210\u001b[0m             \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mos\u001b[38;5;241m.\u001b[39mpath\u001b[38;5;241m.\u001b[39mbasename(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mpath)\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m executable needs to be in PATH. \u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mstart_error_message\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    211\u001b[0m         )\n\u001b[1;32m    212\u001b[0m     \u001b[38;5;28;01melif\u001b[39;00m err\u001b[38;5;241m.\u001b[39merrno \u001b[38;5;241m==\u001b[39m errno\u001b[38;5;241m.\u001b[39mEACCES:\n\u001b[1;32m    213\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m WebDriverException(\n\u001b[1;32m    214\u001b[0m             \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mos\u001b[38;5;241m.\u001b[39mpath\u001b[38;5;241m.\u001b[39mbasename(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mpath)\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m executable may have wrong permissions. \u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mstart_error_message\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    215\u001b[0m         )\n",
      "\u001b[0;31mWebDriverException\u001b[0m: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home\n"
     ]
    }
   ],
   "source": [
    "# Read URLs from CSV\n",
    "file = \"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/antibodiesinc_master-5-20-24.csv\"\n",
    "urls = pd.read_csv(file)['Link'].tolist()[1200:]\n",
    "# Extract PMIDs and save to CSV files\n",
    "output_dir = \"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks\"\n",
    "start_index = 0\n",
    "results_index = extract_pmids(urls, output_dir, start_index, overwrite=True)\n",
    "print(results_index)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b71cab5b",
   "metadata": {},
   "source": [
    "# Create the master Antibodies inc dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "e59787bf",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n",
      "<ipython-input-31-8f56fb73441f>:5: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  dfantiinc = dfantiinc.append(dfi)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>URL</th>\n",
       "      <th>PMID</th>\n",
       "      <th>MABID1</th>\n",
       "      <th>MABID2</th>\n",
       "      <th>ANTIBODYINC1</th>\n",
       "      <th>ANTIBODYINC2</th>\n",
       "      <th>CONJUGATE</th>\n",
       "      <th>URLsplit</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-ne...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pubmed/25429153</td>\n",
       "      <td>n126b</td>\n",
       "      <td>31</td>\n",
       "      <td>75</td>\n",
       "      <td>226</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, n126b, 31, 75, 226]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-ne...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pubmed/24229388</td>\n",
       "      <td>n126b</td>\n",
       "      <td>31</td>\n",
       "      <td>75</td>\n",
       "      <td>226</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, n126b, 31, 75, 226]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-ne...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pubmed/22249266</td>\n",
       "      <td>n126b</td>\n",
       "      <td>31</td>\n",
       "      <td>75</td>\n",
       "      <td>226</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, n126b, 31, 75, 226]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-kv...</td>\n",
       "      <td>https://pubmed.ncbi.nlm.nih.gov/28115470</td>\n",
       "      <td>k19</td>\n",
       "      <td>36</td>\n",
       "      <td>75</td>\n",
       "      <td>012</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, k19, 36, 75, 012]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-kv...</td>\n",
       "      <td>https://pubmed.ncbi.nlm.nih.gov/27872049</td>\n",
       "      <td>k19</td>\n",
       "      <td>36</td>\n",
       "      <td>75</td>\n",
       "      <td>012</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, k19, 36, 75, 012]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5326</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-ca...</td>\n",
       "      <td>https://pubmed.ncbi.nlm.nih.gov/24703904</td>\n",
       "      <td>n263</td>\n",
       "      <td>31</td>\n",
       "      <td>75</td>\n",
       "      <td>257</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, n263, 31, 75, 257]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5327</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-ca...</td>\n",
       "      <td>https://pubmed.ncbi.nlm.nih.gov/23858011</td>\n",
       "      <td>n263</td>\n",
       "      <td>31</td>\n",
       "      <td>75</td>\n",
       "      <td>257</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, n263, 31, 75, 257]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5328</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-ca...</td>\n",
       "      <td>https://pubmed.ncbi.nlm.nih.gov/23568894</td>\n",
       "      <td>n263</td>\n",
       "      <td>31</td>\n",
       "      <td>75</td>\n",
       "      <td>257</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, n263, 31, 75, 257]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5329</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-lr...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pubmed/25981034</td>\n",
       "      <td>n209c</td>\n",
       "      <td>35</td>\n",
       "      <td>75</td>\n",
       "      <td>364</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, n209c, 35, 75, 364]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5330</th>\n",
       "      <td>https://www.antibodiesinc.com/products/anti-sy...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pubmed/25155179</td>\n",
       "      <td>n277</td>\n",
       "      <td>7</td>\n",
       "      <td>75</td>\n",
       "      <td>304</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[antibody, n277, 7, 75, 304]</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5331 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                    URL  \\\n",
       "0     https://www.antibodiesinc.com/products/anti-ne...   \n",
       "1     https://www.antibodiesinc.com/products/anti-ne...   \n",
       "2     https://www.antibodiesinc.com/products/anti-ne...   \n",
       "3     https://www.antibodiesinc.com/products/anti-kv...   \n",
       "4     https://www.antibodiesinc.com/products/anti-kv...   \n",
       "...                                                 ...   \n",
       "5326  https://www.antibodiesinc.com/products/anti-ca...   \n",
       "5327  https://www.antibodiesinc.com/products/anti-ca...   \n",
       "5328  https://www.antibodiesinc.com/products/anti-ca...   \n",
       "5329  https://www.antibodiesinc.com/products/anti-lr...   \n",
       "5330  https://www.antibodiesinc.com/products/anti-sy...   \n",
       "\n",
       "                                              PMID MABID1 MABID2 ANTIBODYINC1  \\\n",
       "0     https://www.ncbi.nlm.nih.gov/pubmed/25429153  n126b     31           75   \n",
       "1     https://www.ncbi.nlm.nih.gov/pubmed/24229388  n126b     31           75   \n",
       "2     https://www.ncbi.nlm.nih.gov/pubmed/22249266  n126b     31           75   \n",
       "3         https://pubmed.ncbi.nlm.nih.gov/28115470    k19     36           75   \n",
       "4         https://pubmed.ncbi.nlm.nih.gov/27872049    k19     36           75   \n",
       "...                                            ...    ...    ...          ...   \n",
       "5326      https://pubmed.ncbi.nlm.nih.gov/24703904   n263     31           75   \n",
       "5327      https://pubmed.ncbi.nlm.nih.gov/23858011   n263     31           75   \n",
       "5328      https://pubmed.ncbi.nlm.nih.gov/23568894   n263     31           75   \n",
       "5329  https://www.ncbi.nlm.nih.gov/pubmed/25981034  n209c     35           75   \n",
       "5330  https://www.ncbi.nlm.nih.gov/pubmed/25155179   n277      7           75   \n",
       "\n",
       "     ANTIBODYINC2  CONJUGATE                        URLsplit  \n",
       "0             226        NaN  [antibody, n126b, 31, 75, 226]  \n",
       "1             226        NaN  [antibody, n126b, 31, 75, 226]  \n",
       "2             226        NaN  [antibody, n126b, 31, 75, 226]  \n",
       "3             012        NaN    [antibody, k19, 36, 75, 012]  \n",
       "4             012        NaN    [antibody, k19, 36, 75, 012]  \n",
       "...           ...        ...                             ...  \n",
       "5326          257        NaN   [antibody, n263, 31, 75, 257]  \n",
       "5327          257        NaN   [antibody, n263, 31, 75, 257]  \n",
       "5328          257        NaN   [antibody, n263, 31, 75, 257]  \n",
       "5329          364        NaN  [antibody, n209c, 35, 75, 364]  \n",
       "5330          304        NaN    [antibody, n277, 7, 75, 304]  \n",
       "\n",
       "[5331 rows x 8 columns]"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# create a master dataframe of the outputs from the code aboe\n",
    "dfantiinc = pd.DataFrame()\n",
    "for file in glob.glob(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinclinks/pmid_output/*.csv\"):\n",
    "    dfi = pd.read_csv(file)\n",
    "    dfantiinc = dfantiinc.append(dfi)\n",
    "    dfantiinc = dfantiinc.reset_index(drop=True)\n",
    "    \n",
    "for index, row in dfantiinc.iterrows():\n",
    "    # look for -f with a number after it with regex\n",
    "    dfantiinc.at[index, 'MABID1'] = row[\"URL\"].split('-')[-4]\n",
    "    dfantiinc.at[index, 'MABID2'] = row[\"URL\"].split('-')[-3]\n",
    "    dfantiinc.at[index, 'ANTIBODYINC1'] = row[\"URL\"].split('-')[-2]\n",
    "    dfantiinc.at[index, 'ANTIBODYINC2'] = row[\"URL\"].split('-')[-1]\n",
    "    dfantiinc.at[index, 'CONJUGATE'] = None\n",
    "    # creaate a list column splitting on the last 5 objects of the url\n",
    "dfantiinc['URLsplit'] = dfantiinc['URL'].str.split('-').str[-5:]\n",
    "dfantiinc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "1b9df2c7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PMID_ID</th>\n",
       "      <th>ANTIBODYINC_ID</th>\n",
       "      <th>MABID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>16525047</td>\n",
       "      <td>75-003, 75-017</td>\n",
       "      <td>K75/41, K55/7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>16566008</td>\n",
       "      <td>75-022</td>\n",
       "      <td>L6/60</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>16687515</td>\n",
       "      <td>75-038</td>\n",
       "      <td>K73/20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>16817858</td>\n",
       "      <td>75-026, 75-001</td>\n",
       "      <td>K87A/10, K65/35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>16870743</td>\n",
       "      <td>75-026</td>\n",
       "      <td>K87A/10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3298</th>\n",
       "      <td>38294129</td>\n",
       "      <td>75-029</td>\n",
       "      <td>K28/86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3299</th>\n",
       "      <td>38313055</td>\n",
       "      <td>75-188</td>\n",
       "      <td>N138/6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3300</th>\n",
       "      <td>38360747</td>\n",
       "      <td>75-029</td>\n",
       "      <td>K28/86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3301</th>\n",
       "      <td>38367666</td>\n",
       "      <td>75-010</td>\n",
       "      <td>K13/31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3302</th>\n",
       "      <td>38472649</td>\n",
       "      <td>75-160, 75-000</td>\n",
       "      <td>N97A/31, K56A/50</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3303 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       PMID_ID  ANTIBODYINC_ID             MABID\n",
       "0     16525047  75-003, 75-017     K75/41, K55/7\n",
       "1     16566008          75-022             L6/60\n",
       "2     16687515          75-038            K73/20\n",
       "3     16817858  75-026, 75-001   K87A/10, K65/35\n",
       "4     16870743          75-026           K87A/10\n",
       "...        ...             ...               ...\n",
       "3298  38294129          75-029            K28/86\n",
       "3299  38313055          75-188            N138/6\n",
       "3300  38360747          75-029            K28/86\n",
       "3301  38367666          75-010            K13/31\n",
       "3302  38472649  75-160, 75-000  N97A/31, K56A/50\n",
       "\n",
       "[3303 rows x 3 columns]"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# create column antibody inc id that joins the antibodyinc1 and antibodyinc2 columns by -\n",
    "dfantiinc['ANTIBODYINC_ID'] = dfantiinc['ANTIBODYINC1'] + '-' + dfantiinc['ANTIBODYINC2']\n",
    "# same for MABID but make all caps and split on /\n",
    "dfantiinc['MABID'] = dfantiinc['MABID1'].str.upper() + '/' + dfantiinc['MABID2'].str.upper()\n",
    "# extract the column pmid_id from the url column\n",
    "dfantiinc['PMID_ID'] = dfantiinc['PMID'].str.split('/').str[-1]\n",
    "# remove duplicates again\n",
    "# do a groupby and add the antibody ids together in a list and the mabids \n",
    "# join the list with a comma\n",
    "dfantiinc = dfantiinc.groupby('PMID_ID').agg({'ANTIBODYINC_ID': lambda x: ', '.join(set(x)), \n",
    "                                         'MABID': lambda x: ', '.join(set(x))}).reset_index()\n",
    "\n",
    "dfantiinc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "93948f82",
   "metadata": {},
   "outputs": [],
   "source": [
    "# output the antibodies INC final \n",
    "dfantiinc.to_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinc_final-5-20-24.csv\", index=False)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d0e6903a",
   "metadata": {},
   "source": [
    "## Run the antibodies inc dataa"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "f2663a72",
   "metadata": {},
   "outputs": [],
   "source": [
    "dfantiinc = pd.read_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinc_final.csv\")\n",
    "# get all the pmids from the PMID column\n",
    "pubmed_ids = dfantiinc['PMID_ID'].tolist()\n",
    "for pid in pubmed_ids:\n",
    "    record = fetch_details(pid)\n",
    "    if record:\n",
    "        continue\n",
    "    else:\n",
    "        print(f\"For {pid}, no record was found.\")\n",
    "        break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "69f73606",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Error fetching data from PubMed for nan.\n",
      "'float' object has no attribute 'split'\n",
      "For nan, no record was found.\n"
     ]
    }
   ],
   "source": [
    "# run the legacy data from the pubmed column\n",
    "df = pd.read_excel(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/publications 041922.xlsx\")\n",
    "# get all the pmids from the PMID column\n",
    "pubmed_ids = df['Pubmed'].tolist()\n",
    "for pid in pubmed_ids:\n",
    "    record = fetch_details(pid)\n",
    "    if record:\n",
    "        continue\n",
    "    else:\n",
    "        print(f\"For {pid}, no record was found.\")\n",
    "        break\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "1490df83",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4540"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(os.listdir(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/pmidjsons/\"))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "856ff5f3",
   "metadata": {},
   "source": [
    "### These seem to mostly be caused by the neuromab showing up in proximity to a target - we could search for Neuromab and the target but does not guarantee that the neuromab was the one used (though extremely likely). \n",
    "- Unfortunately scraping all papers to check proximity is not feasible as UC davis does not have access to all the papers.\n",
    "- The snipped could also be used as well to see if proximal to the target. I think 99% of the time it will be relevant. \n",
    "- The one hard part is that targets and the way they are shown have weird characters and stuff so search for those with google scholar wont be super easy. \n",
    "- Will certainly merge all this previous manual work into the pipeline so the current count will add 676\n",
    "- Overall my code retrieves 1185 more though (without adding just the general Neuromab search) and ensuring it is a review paper. \n",
    "- Need to remove the <sub> and </sub> from the titles and abstracts.\n",
    "- Need to remove the <sup> and </sup> from the titles and abstracts.\n",
    "- Need to remove the <i> and </i> from the titles and abstracts.\n",
    "- Need to remove the <b> and </b> from the titles and abstracts.\n",
    "- Need to remove the <u> and </u> from the titles and abstracts.\n",
    "- Be more clear where the source is from for the information\n",
    "- Need to fix a small subset that failed in the annotations of Pubmed info should be an easy fix\n",
    "- Possible new queries:\n",
    "  - Antibodies inc and target\n",
    "  - Antibodies inc and target and neuromab\n",
    "  - Antibodies inc and davis and target\n",
    "  - Davis and target and neuromab\n",
    "  - neuromab and (antibody or antibodies or mab or monoclonal) == review (if no target)\n",
    "  - antibodies inc and neuromab id\n",
    "     - how is antibodies inc doing this\n",
    "     - search for antibodies inc and \"antibody or monoclonal or mab\" and \"target\" and each part of the catalog number. \n",
    "- Double check the merging process to include all targets and mabs for the merge with old data\n",
    "- Mark which ones were exclusive to the aantibodies inc scraping\n",
    "- "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 267,
   "id": "0b257499",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "676\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'16511870',\n",
       " '16982420',\n",
       " '17379638',\n",
       " '17537961',\n",
       " '17656725',\n",
       " '17855588',\n",
       " '18083105',\n",
       " '18344983',\n",
       " '18458091',\n",
       " '18480286',\n",
       " '18511484',\n",
       " '18680201',\n",
       " '18716211',\n",
       " '18753128',\n",
       " '18753382',\n",
       " '18832009',\n",
       " '18995817',\n",
       " '19038221',\n",
       " '19118598',\n",
       " '19162133',\n",
       " '19167097',\n",
       " '19185209',\n",
       " '19224364',\n",
       " '19370153',\n",
       " '19558451',\n",
       " '19627441',\n",
       " '19657031',\n",
       " '19697962',\n",
       " '19717445',\n",
       " '19723784',\n",
       " '19746427',\n",
       " '19756181',\n",
       " '19795496',\n",
       " '19796685',\n",
       " '19841742',\n",
       " '19912469',\n",
       " '19945389',\n",
       " '19955355',\n",
       " '20008515',\n",
       " '20037628',\n",
       " '20075334',\n",
       " '20124353',\n",
       " '20139976',\n",
       " '20147652',\n",
       " '20178748',\n",
       " '20304926',\n",
       " '20335472',\n",
       " '20385823',\n",
       " '20451588',\n",
       " '20511282',\n",
       " '20573888',\n",
       " '20628354',\n",
       " '20805102',\n",
       " '20823221',\n",
       " '20826790',\n",
       " '20837535',\n",
       " '20850541',\n",
       " '20876567',\n",
       " '20926677',\n",
       " '20943764',\n",
       " '20946956',\n",
       " '20950357',\n",
       " '21041398',\n",
       " '21046559',\n",
       " '21115101',\n",
       " '21115966',\n",
       " '21116650',\n",
       " '21123556',\n",
       " '21145482',\n",
       " '21179406',\n",
       " '21245341',\n",
       " '21307237',\n",
       " '21307309',\n",
       " '21315255',\n",
       " '21325018',\n",
       " '21358642',\n",
       " '21362533',\n",
       " '21371776',\n",
       " '21390300',\n",
       " '21411656',\n",
       " '21490598',\n",
       " '21555557',\n",
       " '21565394',\n",
       " '21602806',\n",
       " '21606927',\n",
       " '21700703',\n",
       " '21710546',\n",
       " '21762014',\n",
       " '21799757',\n",
       " '21800318',\n",
       " '21810783',\n",
       " '21832194',\n",
       " '21849563',\n",
       " '21874017',\n",
       " '21909089',\n",
       " '21940706',\n",
       " '22003227',\n",
       " '22049414',\n",
       " '22102216',\n",
       " '22114295',\n",
       " '22120148',\n",
       " '22120753',\n",
       " '22171032',\n",
       " '22179605',\n",
       " '22198502',\n",
       " '22225533',\n",
       " '22289077',\n",
       " '22337871',\n",
       " '22340949',\n",
       " '22407911',\n",
       " '22438991',\n",
       " '22458534',\n",
       " '22493505',\n",
       " '22579606',\n",
       " '22581473',\n",
       " '22606307',\n",
       " '22606319',\n",
       " '22623657',\n",
       " '22745509',\n",
       " '22749475',\n",
       " '22763697',\n",
       " '22802050',\n",
       " '22843680',\n",
       " '22902719',\n",
       " '22912588',\n",
       " '22948147',\n",
       " '23015759',\n",
       " '23041629',\n",
       " '23071766',\n",
       " '23090950',\n",
       " '23100425',\n",
       " '23123677',\n",
       " '23145349',\n",
       " '23175819',\n",
       " '23197389',\n",
       " '23201445',\n",
       " '23213356',\n",
       " '23226425',\n",
       " '23231809',\n",
       " '23269831',\n",
       " '23283174',\n",
       " '23300088',\n",
       " '23303958',\n",
       " '23329067',\n",
       " '23359859',\n",
       " '23365224',\n",
       " '23365230',\n",
       " '23371365',\n",
       " '23382180',\n",
       " '23382246',\n",
       " '23403072',\n",
       " '23423560',\n",
       " '23469015',\n",
       " '23479737',\n",
       " '23486945',\n",
       " '23519441',\n",
       " '23533599',\n",
       " '23553633',\n",
       " '23595754',\n",
       " '23602553',\n",
       " '23602570',\n",
       " '23606560',\n",
       " '23630944',\n",
       " '23637329',\n",
       " '23638386',\n",
       " '23685357',\n",
       " '23691031',\n",
       " '23719799',\n",
       " '23740188',\n",
       " '23761910',\n",
       " '23765921',\n",
       " '23782671',\n",
       " '23784547',\n",
       " '23799030',\n",
       " '23813837',\n",
       " '23825423',\n",
       " '23826333',\n",
       " '23829864',\n",
       " '23855983',\n",
       " '23861403',\n",
       " '23862016',\n",
       " '23881099',\n",
       " '23911793',\n",
       " '23912628',\n",
       " '23918451',\n",
       " '23926118',\n",
       " '23934334',\n",
       " '23940033',\n",
       " '23991227',\n",
       " '24035918',\n",
       " '24036351',\n",
       " '24036914',\n",
       " '24057349',\n",
       " '24068831',\n",
       " '24077800',\n",
       " '24089485',\n",
       " '24094109',\n",
       " '24098357',\n",
       " '24101510',\n",
       " '24123853',\n",
       " '24174661',\n",
       " '24205342',\n",
       " '24312498',\n",
       " '24316432',\n",
       " '24326163',\n",
       " '24332449',\n",
       " '24336048',\n",
       " '24360808',\n",
       " '24363122',\n",
       " '24394417',\n",
       " '24444358',\n",
       " '24477985',\n",
       " '24492841',\n",
       " '24498314',\n",
       " '24568373',\n",
       " '24600045',\n",
       " '24618276',\n",
       " '24631250',\n",
       " '24637809',\n",
       " '24653680',\n",
       " '24658113',\n",
       " '24694595',\n",
       " '24698977',\n",
       " '24704313',\n",
       " '24725412',\n",
       " '24728190',\n",
       " '24763057',\n",
       " '24782410',\n",
       " '24784230',\n",
       " '24790208',\n",
       " '24792162',\n",
       " '24806686',\n",
       " '24824260',\n",
       " '24838000',\n",
       " '24855944',\n",
       " '24990929',\n",
       " '24991957',\n",
       " '25023288',\n",
       " '25045015',\n",
       " '25071925',\n",
       " '25128699',\n",
       " '25130439',\n",
       " '25152393',\n",
       " '25189211',\n",
       " '25223359',\n",
       " '25237136',\n",
       " '25273269',\n",
       " '25355208',\n",
       " '25374361',\n",
       " '25378149',\n",
       " '25401692',\n",
       " '25424568',\n",
       " '25428619',\n",
       " '25450465',\n",
       " '25462158',\n",
       " '25472739',\n",
       " '25484285',\n",
       " '25517933',\n",
       " '25540577',\n",
       " '25575489',\n",
       " '25582292',\n",
       " '25589740',\n",
       " '25616162',\n",
       " '25632041',\n",
       " '25637662',\n",
       " '25647156',\n",
       " '25675530',\n",
       " '25694803',\n",
       " '25709106',\n",
       " '25750616',\n",
       " '25755278',\n",
       " '25771685',\n",
       " '25816842',\n",
       " '25836421',\n",
       " '25843643',\n",
       " '25849048',\n",
       " '25855178',\n",
       " '25941471',\n",
       " '25964214',\n",
       " '25966954',\n",
       " '25997006',\n",
       " '26003447',\n",
       " '26023097',\n",
       " '26053317',\n",
       " '26071364',\n",
       " '26074404',\n",
       " '26077708',\n",
       " '26078055',\n",
       " '26094487',\n",
       " '26146087',\n",
       " '26198926',\n",
       " '26214256',\n",
       " '26255006',\n",
       " '26298330',\n",
       " '26303499',\n",
       " '26305592',\n",
       " '26347316',\n",
       " '26352593',\n",
       " '26393286',\n",
       " '26398945',\n",
       " '26408000',\n",
       " '26419955',\n",
       " '26446216',\n",
       " '26446227',\n",
       " '26453550',\n",
       " '26483560',\n",
       " '26493400',\n",
       " '26496209',\n",
       " '26538089',\n",
       " '26558789',\n",
       " '26565907',\n",
       " '26586132',\n",
       " '26616331',\n",
       " '26658865',\n",
       " '26701851',\n",
       " '26721934',\n",
       " '26748701',\n",
       " '26755726',\n",
       " '26755826',\n",
       " '26814127',\n",
       " '26847545',\n",
       " '26859289',\n",
       " '26865625',\n",
       " '26884322',\n",
       " '26903620',\n",
       " '26908602',\n",
       " '26911699',\n",
       " '26921771',\n",
       " '26931375',\n",
       " '26956396',\n",
       " '26961175',\n",
       " '26975851',\n",
       " '26980613',\n",
       " '26988269',\n",
       " '27002170',\n",
       " '27009068',\n",
       " '27012630',\n",
       " '27017927',\n",
       " '27025876',\n",
       " '27035958',\n",
       " '27046536',\n",
       " '27060558',\n",
       " '27090093',\n",
       " '27094867',\n",
       " '27098703',\n",
       " '27098968',\n",
       " '27104747',\n",
       " '27147662',\n",
       " '27157711',\n",
       " '27160897',\n",
       " '27161034',\n",
       " '27167970',\n",
       " '27194325',\n",
       " '27221754',\n",
       " '27226355',\n",
       " '27229916',\n",
       " '27239447',\n",
       " '27257627',\n",
       " '27273464',\n",
       " '27274066',\n",
       " '27321162',\n",
       " '27342122',\n",
       " '27353765',\n",
       " '27372643',\n",
       " '27383597',\n",
       " '27445135',\n",
       " '27450559',\n",
       " '27457813',\n",
       " '27458189',\n",
       " '27462810',\n",
       " '27466368',\n",
       " '27472419',\n",
       " '27488240',\n",
       " '27493724',\n",
       " '27514924',\n",
       " '27521566',\n",
       " '27529348',\n",
       " '27535911',\n",
       " '27537483',\n",
       " '27537937',\n",
       " '27538456',\n",
       " '27557559',\n",
       " '27567313',\n",
       " '27581454',\n",
       " '27594413',\n",
       " '27594585',\n",
       " '27594586',\n",
       " '27604541',\n",
       " '27620175',\n",
       " '27621269',\n",
       " '27626669',\n",
       " '27653801',\n",
       " '27666021',\n",
       " '27667007',\n",
       " '27688126',\n",
       " '27748445',\n",
       " '27762395',\n",
       " '27771510',\n",
       " '27794462',\n",
       " '27796754',\n",
       " '27797341',\n",
       " '27799542',\n",
       " '27815839',\n",
       " '27856360',\n",
       " '27857176',\n",
       " '27863209',\n",
       " '27877708',\n",
       " '27884724',\n",
       " '27884958',\n",
       " '27903723',\n",
       " '27905566',\n",
       " '27943180',\n",
       " '27956620',\n",
       " '27956638',\n",
       " '27998989',\n",
       " '28007900',\n",
       " '28011637',\n",
       " '28017328',\n",
       " '28029095',\n",
       " '28040436',\n",
       " '28041852',\n",
       " '28069946',\n",
       " '28087224',\n",
       " '28089772',\n",
       " '28098200',\n",
       " '28108258',\n",
       " '28122242',\n",
       " '28130356',\n",
       " '28139716',\n",
       " '28143929',\n",
       " '28180185',\n",
       " '28193459',\n",
       " '28241839',\n",
       " '28270570',\n",
       " '28284898',\n",
       " '28285821',\n",
       " '28297715',\n",
       " '28321181',\n",
       " '28347403',\n",
       " '28355134',\n",
       " '28388438',\n",
       " '28391401',\n",
       " '28394324',\n",
       " '28396597',\n",
       " '28397972',\n",
       " '28419087',\n",
       " '28455369',\n",
       " '28455787',\n",
       " '28457970',\n",
       " '28461216',\n",
       " '28522732',\n",
       " '28552585',\n",
       " '28598327',\n",
       " '28607108',\n",
       " '28628102',\n",
       " '28641111',\n",
       " '28654047',\n",
       " '28677303',\n",
       " '28683263',\n",
       " '28695613',\n",
       " '28696294',\n",
       " '28701345',\n",
       " '28740101',\n",
       " '28742985',\n",
       " '28787594',\n",
       " '28835433',\n",
       " '28856239',\n",
       " '28860087',\n",
       " '28871036',\n",
       " '28874824',\n",
       " '28878098',\n",
       " '28880079',\n",
       " '28905188',\n",
       " '28921568',\n",
       " '28924012',\n",
       " '28928015',\n",
       " '28934977',\n",
       " '28939041',\n",
       " '28973398',\n",
       " '29028947',\n",
       " '29042434',\n",
       " '29045836',\n",
       " '29046483',\n",
       " '29062015',\n",
       " '29066292',\n",
       " '29107844',\n",
       " '29114861',\n",
       " '29128334',\n",
       " '29130579',\n",
       " '29137457',\n",
       " '29158396',\n",
       " '29168010',\n",
       " '29196321',\n",
       " '29217690',\n",
       " '29229294',\n",
       " '29237760',\n",
       " '29242167',\n",
       " '29244098',\n",
       " '29250591',\n",
       " '29263233',\n",
       " '29273760',\n",
       " '29282697',\n",
       " '29293918',\n",
       " '29302779',\n",
       " '29314192',\n",
       " '29320741',\n",
       " '29337350',\n",
       " '29351997',\n",
       " '29362376',\n",
       " '29382353',\n",
       " '29390050',\n",
       " '29425353',\n",
       " '29429933',\n",
       " '29439165',\n",
       " '29439712',\n",
       " '29444299',\n",
       " '29449608',\n",
       " '29458199',\n",
       " '29501530',\n",
       " '29514831',\n",
       " '29518270',\n",
       " '29524612',\n",
       " '29534062',\n",
       " '29581259',\n",
       " '29606348',\n",
       " '29606581',\n",
       " '29606615',\n",
       " '29610302',\n",
       " '29619614',\n",
       " '29621484',\n",
       " '29625451',\n",
       " '29662202',\n",
       " '29704886',\n",
       " '29712777',\n",
       " '29730824',\n",
       " '29731308',\n",
       " '29785489',\n",
       " '29804286',\n",
       " '29853627',\n",
       " '29920554',\n",
       " '29949735',\n",
       " '29960115',\n",
       " '29974865',\n",
       " '29981480',\n",
       " '29996097',\n",
       " '30017522',\n",
       " '30021165',\n",
       " '30052626',\n",
       " '30065633',\n",
       " '30066827',\n",
       " '30071854',\n",
       " '30086768',\n",
       " '30089262',\n",
       " '30103769',\n",
       " '30126943',\n",
       " '30190379',\n",
       " '30228185',\n",
       " '30237867',\n",
       " '30257928',\n",
       " '30263951',\n",
       " '30277607',\n",
       " '30312396',\n",
       " '30318412',\n",
       " '30323233',\n",
       " '30325063',\n",
       " '30342961',\n",
       " '30355501',\n",
       " '30391323',\n",
       " '30392800',\n",
       " '30396995',\n",
       " '30406178',\n",
       " '30439288',\n",
       " '30446707',\n",
       " '30497839',\n",
       " '30551997',\n",
       " '30573647',\n",
       " '30591411',\n",
       " '30612588',\n",
       " '30642393',\n",
       " '30686507',\n",
       " '30707832',\n",
       " '30745530',\n",
       " '30789343',\n",
       " '30792833',\n",
       " '30836092',\n",
       " '30902970',\n",
       " '30904951',\n",
       " '30926797',\n",
       " '30957013',\n",
       " '30967682',\n",
       " '31010741',\n",
       " '31028116',\n",
       " '31028242',\n",
       " '31044365',\n",
       " '31145471',\n",
       " '31169885',\n",
       " '31182635',\n",
       " '31223136',\n",
       " '31237944',\n",
       " '31257558',\n",
       " '31283873',\n",
       " '31291571',\n",
       " '31306446',\n",
       " '31313076',\n",
       " '31313987',\n",
       " '31321035',\n",
       " '31348683',\n",
       " '31356041',\n",
       " '31388001',\n",
       " '31399533',\n",
       " '31401396',\n",
       " '31433295',\n",
       " '31461551',\n",
       " '31492750',\n",
       " '31504267',\n",
       " '31535451',\n",
       " '31571216',\n",
       " '31597090',\n",
       " '31601621',\n",
       " '31685865',\n",
       " '31712561',\n",
       " '31800046',\n",
       " '31838437',\n",
       " '31878108',\n",
       " '31914378',\n",
       " '31940498',\n",
       " '32004315',\n",
       " '32030809',\n",
       " '32033274',\n",
       " '32047423',\n",
       " '32069266',\n",
       " '32087004',\n",
       " '32116557',\n",
       " '32133675',\n",
       " '32138754',\n",
       " '32146679',\n",
       " '32164788',\n",
       " '32196746',\n",
       " '32203820',\n",
       " '32217627',\n",
       " '32220329',\n",
       " '32229587',\n",
       " '32256029',\n",
       " '32265258',\n",
       " '32270035',\n",
       " '32286126',\n",
       " '32294447',\n",
       " '32296133',\n",
       " '32327735',\n",
       " '32329240',\n",
       " '32341026',\n",
       " '32461616',\n",
       " '32521269',\n",
       " '32521280',\n",
       " '32548211',\n",
       " '32670741',\n",
       " '32706374',\n",
       " '32708917',\n",
       " '32795422',\n",
       " '32858836',\n",
       " '32948689',\n",
       " '32950876',\n",
       " '33001981',\n",
       " '33046105',\n",
       " '33117519',\n",
       " '33237428',\n",
       " '33381867',\n",
       " '33416459',\n",
       " '33483523',\n",
       " '33501759',\n",
       " '33529173',\n",
       " '33536249',\n",
       " '33547341',\n",
       " '33556372',\n",
       " '33615226',\n",
       " '33663561'}"
      ]
     },
     "execution_count": 267,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# compare all the pmids to this file and see which we dont have\n",
    "#/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/publications 041922.xlsx\n",
    "df = pd.read_excel(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/publications 041922.xlsx\")\n",
    "# filter out anything that saays review in the clone\n",
    "# remove all Pubmed that are empty\n",
    "df = df[df['Pubmed'].notna()]\n",
    "# set the Clone to a string column and remove any that are na\n",
    "df['Clone'] = df['Clone'].astype(str)\n",
    "df = df[df['Clone'] != 'nan']\n",
    "df = df[~df['Clone'].str.contains('review', case=False)]\n",
    "print(len(set([str(i) for i in df[\"Pubmed\"].tolist()])- set([str(i.split('.')[0]) for i in os.listdir(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/pmidjsons/\")])))\n",
    "set([str(i) for i in df[\"Pubmed\"].tolist()])- set([str(i.split('.')[0]) for i in os.listdir(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/pmidjsons/\")])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 269,
   "id": "5bd5264d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1185\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'28119464',\n",
       " '34718573',\n",
       " '25691569',\n",
       " '36220889',\n",
       " '35176221',\n",
       " '37051140',\n",
       " '31142543',\n",
       " '35441593',\n",
       " '36034774',\n",
       " '31816021',\n",
       " '36097791',\n",
       " '36429074',\n",
       " '34063992',\n",
       " '31831638',\n",
       " '35701597',\n",
       " '34003113',\n",
       " '36833753',\n",
       " '28496181',\n",
       " '28158844',\n",
       " '30154505',\n",
       " '32917588',\n",
       " '26027926',\n",
       " '36358921',\n",
       " '23323617',\n",
       " '36472910',\n",
       " '37333281',\n",
       " '29398366',\n",
       " '27568555',\n",
       " '35760104',\n",
       " '35456979',\n",
       " '23404451',\n",
       " '32697988',\n",
       " '30773633',\n",
       " '37528484',\n",
       " '34705506',\n",
       " '32156701',\n",
       " '29692709',\n",
       " '26435106',\n",
       " '32801157',\n",
       " '33550628',\n",
       " '26527003',\n",
       " '33817875',\n",
       " '37848409',\n",
       " '34880118',\n",
       " '37114864',\n",
       " '34985913',\n",
       " '37704727',\n",
       " '36982541',\n",
       " '29872123',\n",
       " '33727250',\n",
       " '33854181',\n",
       " '36257688',\n",
       " '30456368',\n",
       " '34000300',\n",
       " '37425915',\n",
       " '34492310',\n",
       " '35789851',\n",
       " '36658457',\n",
       " '37048156',\n",
       " '33837049',\n",
       " '28432142',\n",
       " '37819632',\n",
       " '34030727',\n",
       " '34152608',\n",
       " '34610637',\n",
       " '35966009',\n",
       " '27134175',\n",
       " '29221132',\n",
       " '29134655',\n",
       " '33606320',\n",
       " '19409269',\n",
       " '35318302',\n",
       " '36732641',\n",
       " '36609445',\n",
       " '34500285',\n",
       " '31186137',\n",
       " '23894271',\n",
       " '34459572',\n",
       " '34921780',\n",
       " '34727523',\n",
       " '37182096',\n",
       " '37502980',\n",
       " '32634532',\n",
       " '36847487',\n",
       " '37099425',\n",
       " '34789895',\n",
       " '36693320',\n",
       " '34999186',\n",
       " '34408636',\n",
       " '30904180',\n",
       " '27083044',\n",
       " '35073946',\n",
       " '37308047',\n",
       " '34658338',\n",
       " '35760142',\n",
       " '35943566',\n",
       " '36263376',\n",
       " '30951495',\n",
       " '38019651',\n",
       " '31835170',\n",
       " '30783093',\n",
       " '37709161',\n",
       " '32976982',\n",
       " '35105675',\n",
       " '34268301',\n",
       " '27022192',\n",
       " '32822706',\n",
       " '36253083',\n",
       " '32754280',\n",
       " '25775468',\n",
       " '36573349',\n",
       " '26646078',\n",
       " '35180391',\n",
       " '37300770',\n",
       " '35662277',\n",
       " '33932338',\n",
       " '29678909',\n",
       " '32335198',\n",
       " '32209430',\n",
       " '23473316',\n",
       " '35269482',\n",
       " '35952672',\n",
       " '29951000',\n",
       " '31248728',\n",
       " '35843530',\n",
       " '37315917',\n",
       " '35626730',\n",
       " '35680881',\n",
       " '34957478',\n",
       " '37010860',\n",
       " '37192625',\n",
       " '36261284',\n",
       " '29464191',\n",
       " '36427690',\n",
       " '34731636',\n",
       " '31408664',\n",
       " '35733986',\n",
       " '34261648',\n",
       " '37146610',\n",
       " '24598713',\n",
       " '36211980',\n",
       " '36653193',\n",
       " '36997734',\n",
       " '35899529',\n",
       " '34485605',\n",
       " '29440485',\n",
       " '37378339',\n",
       " '37303235',\n",
       " '35625761',\n",
       " '34531281',\n",
       " '35867797',\n",
       " '34968236',\n",
       " '28945249',\n",
       " '31164339',\n",
       " '33191931',\n",
       " '34261657',\n",
       " '29950669',\n",
       " '35840571',\n",
       " '37717828',\n",
       " '32783930',\n",
       " '32702291',\n",
       " '20573181',\n",
       " '35101451',\n",
       " '37722569',\n",
       " '36088390',\n",
       " '27034755',\n",
       " '34413725',\n",
       " '28916793',\n",
       " '32066698',\n",
       " '20139089',\n",
       " '34417170',\n",
       " '34963485',\n",
       " '34008015',\n",
       " '21575722',\n",
       " '33539918',\n",
       " '31805436',\n",
       " '31429499',\n",
       " '36417513',\n",
       " '26595655',\n",
       " '33613257',\n",
       " '31091456',\n",
       " '28710481',\n",
       " '29074619',\n",
       " '36919335',\n",
       " '37797581',\n",
       " '37916792',\n",
       " '32019829',\n",
       " '37159095',\n",
       " '36334265',\n",
       " '34055805',\n",
       " '30334255',\n",
       " '33199366',\n",
       " '32227113',\n",
       " '36640913',\n",
       " '32205379',\n",
       " '33113832',\n",
       " '35219855',\n",
       " '33994959',\n",
       " '30949397',\n",
       " '35217690',\n",
       " '32119167',\n",
       " '37211572',\n",
       " '37694138',\n",
       " '36482968',\n",
       " '20720179',\n",
       " '36517241',\n",
       " '34131060',\n",
       " '33097637',\n",
       " '30352064',\n",
       " '27377235',\n",
       " '36631269',\n",
       " '35440632',\n",
       " '31626293',\n",
       " '26980779',\n",
       " '30858560',\n",
       " '21262851',\n",
       " '30333730',\n",
       " '36876503',\n",
       " '37862170',\n",
       " '30723396',\n",
       " '36543363',\n",
       " '34871934',\n",
       " '19219384',\n",
       " '35396942',\n",
       " '34363623',\n",
       " '30524232',\n",
       " '27047338',\n",
       " '34458961',\n",
       " '36465182',\n",
       " '35121636',\n",
       " '34260930',\n",
       " '37270156',\n",
       " '37355147',\n",
       " '36149401',\n",
       " '36584686',\n",
       " '29246927',\n",
       " '37379838',\n",
       " '27932504',\n",
       " '37002718',\n",
       " '34852231',\n",
       " '35045307',\n",
       " '36762588',\n",
       " '25732511',\n",
       " '32277041',\n",
       " '36274588',\n",
       " '30050412',\n",
       " '36139754',\n",
       " '27005990',\n",
       " '36040482',\n",
       " '35987203',\n",
       " '36458527',\n",
       " '31059774',\n",
       " '31391239',\n",
       " '34799447',\n",
       " '35104503',\n",
       " '29357897',\n",
       " '28611083',\n",
       " '34818559',\n",
       " '30617260',\n",
       " '35248787',\n",
       " '27161151',\n",
       " '35752364',\n",
       " '36971428',\n",
       " '23377220',\n",
       " '34791217',\n",
       " '29937715',\n",
       " '37321992',\n",
       " '32639590',\n",
       " '34534742',\n",
       " '28860483',\n",
       " '30089263',\n",
       " '30390433',\n",
       " '35602097',\n",
       " '34673136',\n",
       " '34604446',\n",
       " '28365780',\n",
       " '35016003',\n",
       " '30504823',\n",
       " '34967003',\n",
       " '31256144',\n",
       " '36087584',\n",
       " '27999940',\n",
       " '37383228',\n",
       " '26766587',\n",
       " '35417693',\n",
       " '36421705',\n",
       " '31623918',\n",
       " '34995515',\n",
       " '33248694',\n",
       " '33057002',\n",
       " '33082319',\n",
       " '31430518',\n",
       " '27134173',\n",
       " '34716617',\n",
       " '30048714',\n",
       " '28566000',\n",
       " '37108794',\n",
       " '37595869',\n",
       " '34081780',\n",
       " '33333928',\n",
       " '33987657',\n",
       " '25172903',\n",
       " '36395884',\n",
       " '29535609',\n",
       " '24362252',\n",
       " '36398458',\n",
       " '35328381',\n",
       " '35953295',\n",
       " '34090516',\n",
       " '32853409',\n",
       " '36496200',\n",
       " '33828089',\n",
       " '34576197',\n",
       " '37356721',\n",
       " '35112738',\n",
       " '35862456',\n",
       " '22251557',\n",
       " '34785580',\n",
       " '34582787',\n",
       " '36085162',\n",
       " '36007011',\n",
       " '35402277',\n",
       " '31046837',\n",
       " '31212067',\n",
       " '28675162',\n",
       " '35437278',\n",
       " '35438230',\n",
       " '23843614',\n",
       " '37650832',\n",
       " '35966206',\n",
       " '26711344',\n",
       " '23630453',\n",
       " '27458797',\n",
       " '24835999',\n",
       " '33656439',\n",
       " '35990894',\n",
       " '37838947',\n",
       " '29726811',\n",
       " '36494423',\n",
       " '21281707',\n",
       " '33818761',\n",
       " '29665166',\n",
       " '34783461',\n",
       " '28469556',\n",
       " '37171958',\n",
       " '32249816',\n",
       " '31522381',\n",
       " '37134150',\n",
       " '35231583',\n",
       " '28334922',\n",
       " '28490751',\n",
       " '34546336',\n",
       " '32540363',\n",
       " '26381166',\n",
       " '26492527',\n",
       " '37833406',\n",
       " '37401985',\n",
       " '36612049',\n",
       " '37594721',\n",
       " '32183972',\n",
       " '38260608',\n",
       " '33619909',\n",
       " '36905628',\n",
       " '37957716',\n",
       " '35347144',\n",
       " '35789858',\n",
       " '37935827',\n",
       " '27199666',\n",
       " '34551307',\n",
       " '32250352',\n",
       " '35346959',\n",
       " '36795458',\n",
       " '37945352',\n",
       " '33650969',\n",
       " '33986647',\n",
       " '34086081',\n",
       " '37283058',\n",
       " '34665127',\n",
       " '35978188',\n",
       " '36090255',\n",
       " '36309963',\n",
       " '34250948',\n",
       " '36765071',\n",
       " '33582225',\n",
       " '34114604',\n",
       " '36311018',\n",
       " '32639632',\n",
       " '27986928',\n",
       " '34530852',\n",
       " '36997609',\n",
       " '34302896',\n",
       " '32438059',\n",
       " '36494348',\n",
       " '35145085',\n",
       " '31503067',\n",
       " '34180393',\n",
       " '37419688',\n",
       " '36323817',\n",
       " '29354033',\n",
       " '32450347',\n",
       " '29760652',\n",
       " '29615496',\n",
       " '32938721',\n",
       " '37793046',\n",
       " '36848492',\n",
       " '16885215',\n",
       " '22565157',\n",
       " '33410399',\n",
       " '36623761',\n",
       " '26095359',\n",
       " '34463328',\n",
       " '31331967',\n",
       " '36552754',\n",
       " '35671105',\n",
       " '26553645',\n",
       " '37883842',\n",
       " '37333277',\n",
       " '34156984',\n",
       " '35758537',\n",
       " '37591201',\n",
       " '33090569',\n",
       " '36653191',\n",
       " '35661858',\n",
       " '36658491',\n",
       " '36083912',\n",
       " '33859248',\n",
       " '29138281',\n",
       " '23980198',\n",
       " '37438479',\n",
       " '26985008',\n",
       " '29507146',\n",
       " '37408217',\n",
       " '33106078',\n",
       " '30233305',\n",
       " '29382710',\n",
       " '36007521',\n",
       " '30952813',\n",
       " '36217300',\n",
       " '33593425',\n",
       " '32251970',\n",
       " '29467616',\n",
       " '35417922',\n",
       " '37516919',\n",
       " '32824677',\n",
       " '36859689',\n",
       " '29429934',\n",
       " '27935475',\n",
       " '33958989',\n",
       " '34471138',\n",
       " '36056393',\n",
       " '31472117',\n",
       " '35662394',\n",
       " '33789707',\n",
       " '28719464',\n",
       " '35123326',\n",
       " '36322399',\n",
       " '36672247',\n",
       " '27929705',\n",
       " '36550622',\n",
       " '30699097',\n",
       " '35201268',\n",
       " '30687002',\n",
       " '33831386',\n",
       " '29199957',\n",
       " '37253603',\n",
       " '36459557',\n",
       " '35326469',\n",
       " '37910217',\n",
       " '36077460',\n",
       " '37898313',\n",
       " '28614186',\n",
       " '36206767',\n",
       " '37874635',\n",
       " '34759048',\n",
       " '35420982',\n",
       " '31063821',\n",
       " '33083769',\n",
       " '31877124',\n",
       " '35931052',\n",
       " '28387307',\n",
       " '25926772',\n",
       " '37425759',\n",
       " '34288120',\n",
       " '34730290',\n",
       " '33739978',\n",
       " '36042811',\n",
       " '37059474',\n",
       " '27482820',\n",
       " '34113010',\n",
       " '32908139',\n",
       " '34306319',\n",
       " '28634442',\n",
       " '33863995',\n",
       " '32007961',\n",
       " '36123538',\n",
       " '37164542',\n",
       " '28243192',\n",
       " '34284042',\n",
       " '34393723',\n",
       " '31176559',\n",
       " '20333733',\n",
       " '21813652',\n",
       " '37305696',\n",
       " '30224722',\n",
       " '30670570',\n",
       " '30517687',\n",
       " '25863253',\n",
       " '37172641',\n",
       " '37777094',\n",
       " '36803960',\n",
       " '35972937',\n",
       " '37391270',\n",
       " '35701825',\n",
       " '28902836',\n",
       " '37891825',\n",
       " '37277177',\n",
       " '30771034',\n",
       " '32523511',\n",
       " '33994940',\n",
       " '34481267',\n",
       " '27798188',\n",
       " '30002060',\n",
       " '32458389',\n",
       " '31859031',\n",
       " '32062456',\n",
       " '38239897',\n",
       " '37474640',\n",
       " '36672216',\n",
       " '35202614',\n",
       " '30067977',\n",
       " '36849716',\n",
       " '37461609',\n",
       " '37415299',\n",
       " '25310985',\n",
       " '34348157',\n",
       " '18063376',\n",
       " '34429357',\n",
       " '30914445',\n",
       " '28662109',\n",
       " '25918844',\n",
       " '31682945',\n",
       " '28373550',\n",
       " '35440129',\n",
       " '32435793',\n",
       " '35759110',\n",
       " '37303064',\n",
       " '33197196',\n",
       " '34269178',\n",
       " '31554841',\n",
       " '35877103',\n",
       " '34428287',\n",
       " '31071333',\n",
       " '35544276',\n",
       " '35131865',\n",
       " '37491316',\n",
       " '26217189',\n",
       " '35022233',\n",
       " '33251215',\n",
       " '34280489',\n",
       " '34137443',\n",
       " '35672149',\n",
       " '35803737',\n",
       " '36161458',\n",
       " '31663853',\n",
       " '28184187',\n",
       " '28935671',\n",
       " '33890333',\n",
       " '29198169',\n",
       " '35303056',\n",
       " '35726031',\n",
       " '31379510',\n",
       " '36775097',\n",
       " '36830639',\n",
       " '34163037',\n",
       " '32899153',\n",
       " '27595347',\n",
       " '34289389',\n",
       " '33594532',\n",
       " '25172509',\n",
       " '31974009',\n",
       " '26969737',\n",
       " '34930944',\n",
       " '37388725',\n",
       " '37507375',\n",
       " '23022559',\n",
       " '30758043',\n",
       " '16793899',\n",
       " '36729040',\n",
       " '30946007',\n",
       " '31589838',\n",
       " '33472075',\n",
       " '37650457',\n",
       " '28576936',\n",
       " '31518901',\n",
       " '27373832',\n",
       " '28701918',\n",
       " '28459465',\n",
       " '37884508',\n",
       " '32347567',\n",
       " '35051614',\n",
       " '34125248',\n",
       " '32275651',\n",
       " '34751579',\n",
       " '34652420',\n",
       " '37428632',\n",
       " '33989524',\n",
       " '26714926',\n",
       " '34728675',\n",
       " '36661357',\n",
       " '36465112',\n",
       " '30804752',\n",
       " '34057457',\n",
       " '37577625',\n",
       " '34061161',\n",
       " '27623382',\n",
       " '29970989',\n",
       " '37833914',\n",
       " '33192297',\n",
       " '32616470',\n",
       " '35184043',\n",
       " '37620442',\n",
       " '35368073',\n",
       " '34253863',\n",
       " '37323648',\n",
       " '32321836',\n",
       " '36991001',\n",
       " '32938678',\n",
       " '36090254',\n",
       " '36395772',\n",
       " '30647193',\n",
       " '36824888',\n",
       " '35147977',\n",
       " '31337710',\n",
       " '35119496',\n",
       " '36652335',\n",
       " '29168928',\n",
       " '35970564',\n",
       " '37884493',\n",
       " '37693426',\n",
       " '36292621',\n",
       " '34366878',\n",
       " '28127990',\n",
       " '22153079',\n",
       " '36414721',\n",
       " '29386106',\n",
       " '29146773',\n",
       " '31849613',\n",
       " '37479688',\n",
       " '36163283',\n",
       " '36189803',\n",
       " '26945093',\n",
       " '33938021',\n",
       " '37669448',\n",
       " '36100034',\n",
       " '34496239',\n",
       " '33909305',\n",
       " '36060074',\n",
       " '28578430',\n",
       " '34648330',\n",
       " '31524157',\n",
       " '37624696',\n",
       " '32675279',\n",
       " '35987207',\n",
       " '31943018',\n",
       " '29369354',\n",
       " '34226493',\n",
       " '29214211',\n",
       " '34936876',\n",
       " '31437534',\n",
       " '37543038',\n",
       " '37925173',\n",
       " '34673031',\n",
       " '38112323',\n",
       " '34135003',\n",
       " '35445918',\n",
       " '32616667',\n",
       " '35780977',\n",
       " '22325201',\n",
       " '33852912',\n",
       " '31761534',\n",
       " '31636080',\n",
       " '35666232',\n",
       " '9383641',\n",
       " '33556457',\n",
       " '28522250',\n",
       " '36173683',\n",
       " '34227061',\n",
       " '37056784',\n",
       " '30403198',\n",
       " '25176525',\n",
       " '35245800',\n",
       " '33801205',\n",
       " '22209788',\n",
       " '31740674',\n",
       " '31813652',\n",
       " '35721319',\n",
       " '26644354',\n",
       " '36273507',\n",
       " '31992990',\n",
       " '22710833',\n",
       " '34431981',\n",
       " '32097818',\n",
       " '34132778',\n",
       " '36243751',\n",
       " '30588702',\n",
       " '31216424',\n",
       " '25174890',\n",
       " '30347190',\n",
       " '32516721',\n",
       " '22982711',\n",
       " '28283559',\n",
       " '33568460',\n",
       " '21712833',\n",
       " '37758766',\n",
       " '31639389',\n",
       " '37878364',\n",
       " '33185190',\n",
       " '37260747',\n",
       " '27033551',\n",
       " '31091457',\n",
       " '33372037',\n",
       " '35739402',\n",
       " '28751743',\n",
       " '37384525',\n",
       " '34592633',\n",
       " '36327326',\n",
       " '33157003',\n",
       " '35064871',\n",
       " '30224655',\n",
       " '30905605',\n",
       " '34911751',\n",
       " '31474508',\n",
       " '34932577',\n",
       " '32840212',\n",
       " '28864826',\n",
       " '35322202',\n",
       " '37425946',\n",
       " '31566427',\n",
       " '32169553',\n",
       " '37009450',\n",
       " '30302388',\n",
       " '34346313',\n",
       " '34910912',\n",
       " '29397366',\n",
       " '21804529',\n",
       " '26257610',\n",
       " '38137092',\n",
       " '27616982',\n",
       " '35966008',\n",
       " '28320946',\n",
       " '31268420',\n",
       " '36546837',\n",
       " '31862971',\n",
       " '36969554',\n",
       " '29490268',\n",
       " '32111696',\n",
       " '36018183',\n",
       " '26572867',\n",
       " '37758930',\n",
       " '27364017',\n",
       " '29440991',\n",
       " '34038184',\n",
       " '36050224',\n",
       " '27869801',\n",
       " '34234241',\n",
       " '36220815',\n",
       " '37261953',\n",
       " '36634053',\n",
       " '31282864',\n",
       " '36301206',\n",
       " '22037195',\n",
       " '33361086',\n",
       " '34931070',\n",
       " '30856389',\n",
       " '33441399',\n",
       " '34455393',\n",
       " '33987909',\n",
       " '36030255',\n",
       " '35276083',\n",
       " '35732272',\n",
       " '35948005',\n",
       " '34825141',\n",
       " '37994700',\n",
       " '37843188',\n",
       " '34167580',\n",
       " '33306870',\n",
       " '33013747',\n",
       " '34781056',\n",
       " '27841474',\n",
       " '35177616',\n",
       " '29057936',\n",
       " '36681359',\n",
       " '33459343',\n",
       " '35701158',\n",
       " '36774519',\n",
       " '38187582',\n",
       " '34375626',\n",
       " '38091401',\n",
       " '34331901',\n",
       " '27157611',\n",
       " '30451289',\n",
       " '28515230',\n",
       " '32437657',\n",
       " '33945785',\n",
       " '34065662',\n",
       " '37490545',\n",
       " '33730581',\n",
       " '36251395',\n",
       " '36710500',\n",
       " '34433078',\n",
       " '30463852',\n",
       " '30104343',\n",
       " '33767611',\n",
       " '34613779',\n",
       " '34580980',\n",
       " '36606424',\n",
       " '33877363',\n",
       " '36608123',\n",
       " '35238051',\n",
       " '34392086',\n",
       " '32123821',\n",
       " '32237183',\n",
       " '28794445',\n",
       " '26258683',\n",
       " '30927072',\n",
       " '27117416',\n",
       " '36367592',\n",
       " '37953841',\n",
       " '36596304',\n",
       " '32273484',\n",
       " '34658337',\n",
       " '32477061',\n",
       " '37315729',\n",
       " '33838110',\n",
       " '38231269',\n",
       " '34990802',\n",
       " '34110073',\n",
       " '29697531',\n",
       " '36604600',\n",
       " '35508186',\n",
       " '26859820',\n",
       " '32265664',\n",
       " '29173281',\n",
       " '29555853',\n",
       " '35806143',\n",
       " '36545239',\n",
       " '35016853',\n",
       " '32490811',\n",
       " '37856017',\n",
       " '28066910',\n",
       " '37689317',\n",
       " '36580912',\n",
       " '34223875',\n",
       " '34375627',\n",
       " '33887226',\n",
       " '35434559',\n",
       " '36842733',\n",
       " '29294243',\n",
       " '36289231',\n",
       " '35862168',\n",
       " '35905928',\n",
       " '34962957',\n",
       " '36945454',\n",
       " '28814779',\n",
       " '31607869',\n",
       " '37532012',\n",
       " '22249457',\n",
       " '37168495',\n",
       " '31354436',\n",
       " '30598502',\n",
       " '36336067',\n",
       " '36369219',\n",
       " '29592971',\n",
       " '30825330',\n",
       " '19923600',\n",
       " '32531277',\n",
       " '37756587',\n",
       " '30592011',\n",
       " '26834556',\n",
       " '34844642',\n",
       " '37773316',\n",
       " '34289359',\n",
       " '28432329',\n",
       " '26808347',\n",
       " '35289477',\n",
       " '27593178',\n",
       " '38014238',\n",
       " '31806492',\n",
       " '35346832',\n",
       " '33364977',\n",
       " '34231259',\n",
       " '36949148',\n",
       " '37607607',\n",
       " '36967142',\n",
       " '32364262',\n",
       " '28235805',\n",
       " '31964774',\n",
       " '26764482',\n",
       " '38194969',\n",
       " '36697254',\n",
       " '36216607',\n",
       " '36014775',\n",
       " '37533653',\n",
       " '27943591',\n",
       " '31685650',\n",
       " '36878501',\n",
       " '24276495',\n",
       " '37587147',\n",
       " '37310860',\n",
       " '34803580',\n",
       " '36417873',\n",
       " '31597963',\n",
       " '30022868',\n",
       " '35501931',\n",
       " '35577065',\n",
       " '37105442',\n",
       " '35940118',\n",
       " '36181963',\n",
       " '28677679',\n",
       " '28973664',\n",
       " '28430183',\n",
       " '31350430',\n",
       " '29487522',\n",
       " '26845140',\n",
       " '35599495',\n",
       " '26218245',\n",
       " '28103480',\n",
       " '37858465',\n",
       " '35246535',\n",
       " '36709330',\n",
       " '32634531',\n",
       " '37075948',\n",
       " '34450161',\n",
       " '37847372',\n",
       " '32248813',\n",
       " '35263136',\n",
       " '34974536',\n",
       " '38075021',\n",
       " '31554851',\n",
       " '37141099',\n",
       " '34461722',\n",
       " '24872163',\n",
       " '31107446',\n",
       " '35359569',\n",
       " '26186538',\n",
       " '31604597',\n",
       " '28447247',\n",
       " '27621318',\n",
       " '29091766',\n",
       " '38038892',\n",
       " '26291161',\n",
       " '25193658',\n",
       " '28040728',\n",
       " '36451656',\n",
       " '26590557',\n",
       " '36395691',\n",
       " '31212013',\n",
       " '37003353',\n",
       " '37702787',\n",
       " '29706865',\n",
       " '33132203',\n",
       " '36693856',\n",
       " '21745462',\n",
       " '28522792',\n",
       " '25939886',\n",
       " '27418514',\n",
       " '33836114',\n",
       " '29976627',\n",
       " '34766907',\n",
       " '29472595',\n",
       " '37837904',\n",
       " '37963972',\n",
       " '35841648',\n",
       " '24744440',\n",
       " '29222469',\n",
       " '35803734',\n",
       " '37923712',\n",
       " '36262316',\n",
       " '35726573',\n",
       " '34496232',\n",
       " '35952494',\n",
       " '34144124',\n",
       " '36684387',\n",
       " '36142639',\n",
       " '36351843',\n",
       " '31249512',\n",
       " '31591156',\n",
       " '36793543',\n",
       " '33989527',\n",
       " '35678782',\n",
       " '37289222',\n",
       " '37759527',\n",
       " '34522029',\n",
       " '31591393',\n",
       " '31403402',\n",
       " '29934346',\n",
       " '35012605',\n",
       " '35857479',\n",
       " '36656118',\n",
       " '34335184',\n",
       " ...}"
      ]
     },
     "execution_count": 269,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# check which we have and the other doesnt \n",
    "print(len(set([str(i.split('.')[0]) for i in os.listdir(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/pmidjsons/\")])- set([str(i) for i in df[\"Pubmed\"].tolist()])))\n",
    "set([str(i.split('.')[0]) for i in os.listdir(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/pmidjsons/\")]) - set([str(i) for i in df[\"Pubmed\"].tolist()])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 270,
   "id": "df145405",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BACKGROUND: KCNQ1 (Kv7.1), together with its KCNE β subunits, plays a pivotal role \n",
      "both in the repolarization of cardiac tissue and in water and salt transport across epithelial …\n",
      "\n",
      "… , N212/17, Neuromab, CA, USA) antibodies using a previously described protocol [36, 37]. \n",
      "Imaging analysis was performed using a Nikon A1 confocal microscope at the Northwestern …\n",
      "\n",
      "Tetratricopeptide repeat (TPR) domains are ubiquitous structural motifs that mediate protein–protein \n",
      "interactions. For example, the TPR domains in the peroxisomal import receptor …\n",
      "\n",
      "Hyperpolarization-activated cyclic nucleotide-gated (HCN) channels are expressed in the \n",
      "heart and central nervous system where they regulate membrane excitability. Recently, …\n",
      "\n",
      "… N144/14; University of California Davis/National Institutes of Health … (N144/14) was \n",
      "obtained form the University of California Davis/National Institutes of Health NeuroMab Facility. …\n",
      "\n",
      "… Lysates were incubated with CRMP-2-6xHis, following phosphorylation, and immunoprecipitated \n",
      "(IP) with a 6xHis antibody (UC Davis/NIH NeuroMab Facility, clone N144/14) overnight …\n",
      "\n",
      "… Mouse antibody against His (N144/14) for western blotting was from NeuroMab (Davis, \n",
      "CA). Mouse antibody against His (27E8) and rabbit antibody against His (no. 2365) were …\n",
      "\n",
      "α-Synuclein plays a key role in the pathogenesis of Parkinson’s disease and related disorders, \n",
      "but critical interacting partners and molecular mechanisms mediating neurotoxicity are …\n",
      "\n",
      "Eukaryotic pathogens display multiple mechanisms for breaching the blood-brain barrier (BBB) \n",
      "and invading the central nervous system (CNS). Of the fungal spp., that cause disease in …\n",
      "\n",
      "The conventional kinesin motor transports many different cargos to specific locations in neurons. \n",
      "How cargos regulate motor function remains unclear. Here we focus on KIF5, the heavy …\n",
      "\n",
      "Botulinum neurotoxin A (BoNT/A) belongs to the most dangerous class of bioweapons 1 . \n",
      "Despite this, BoNT/A is used to treat a wide range of common medical conditions such as …\n",
      "\n",
      "… the UC Davis/NIH NeuroMab facility: AnkG (106/36; RRID:AB_10673030), AnkG (106/65; \n",
      "RRID:AB_10675130), PanNav (N419/78; RRID:AB_2493099), b1 spectrin (N385/21; RRID:…\n",
      "\n",
      "Skeletal muscle contraction depends on activation of clustered acetylcholine receptors (AchRs) \n",
      "and muscle-specific Na + channels (Nav1.4). Some Nav1.4 channels are highly …\n",
      "\n",
      "Clustered ion channels at nodes of Ranvier are critical for fast action potential propagation \n",
      "in myelinated axons. Axon-glia interactions converge on ankyrin and spectrin cytoskeletal …\n",
      "\n",
      "… Mouse monoclonal antibody against GKAP (clone N1427/31, used at 1:100), pan Shank (clone \n",
      "N23B/49, which recognizes all three members of the Shank family: Shank 1, 2 and 3, …\n",
      "\n",
      "… 1:1000 for Western), and pan Shank (clone N23B/49, used at 1:200 for microscopy and 1:1000 \n",
      "for Western) were from NeuroMab, Davis, CA, USA. Rabbit polyclonal antibody against …\n",
      "\n",
      "… The monoclonal mouse antibodies against Shank (N23B/49) and PSD-95 (K28/43) were \n",
      "from NeuroMAB. ECL and ECL plus reagents were from Amersham. Other reagents were from …\n",
      "\n",
      "IRSp53 (BAIAP2) is an abundant protein at the postsynaptic density (PSD) that binds to major \n",
      "PSD scaffolds, PSD-95 and Shanks, as well as to F-actin. The distribution of IRSp53 at the …\n",
      "\n",
      "Early in development spontaneous activity modulates survival and connectivity of neurons \n",
      "and thus plays a crucial role in the formation of neural networks. The emergence of …\n",
      "\n",
      "Regulated exocytosis is essential for many biological processes and many components of \n",
      "the protein trafficking machinery are ubiquitous. However, there are also exceptions, such as …\n",
      "\n",
      "The family of Shank scaffolding molecules (comprising Shank1, 2 and 3) are core components \n",
      "of the postsynaptic density ( PSD ) in neuronal synapses. Shanks link surface receptors to …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "In vitro studies suggest that the intracellular C terminus of Neuroligin1 (NL1) could play a \n",
      "central role in the maturation of excitatory synapses. However, it is unknown how this activity …\n",
      "\n",
      "… ), mouse monoclonal anti-Pan-Shank (clone N23B/49; 1:400, Neuromab, N23B/49), mouse \n",
      "monoclonal anti-NR2A (clone N327/95; 1:400, Neuromab, Q00959), mouse monoclonal anti-…\n",
      "\n",
      "… for Western), and pan-Shank (clone N23B/49, which recognizes all three members of the \n",
      "Shank family: Shank1, 2 and 3, used at 1:250 for EM) were from NeuroMab (Davis, CA). Mouse …\n",
      "\n",
      "… (clone N23B/49, which recognizes all three members of the Shank family: Shank 1, 2 and 3, \n",
      "used at 1:250), and Shank 2 (clone N23B/6, used at 1:200) were from NeuroMab (Davis, CA)…\n",
      "\n",
      "Excitatory glutamatergic synapses facilitate important aspects of communication between \n",
      "the neurons that govern complex forms of behavior. Accordingly, small differences in the …\n",
      "\n",
      "Synapses are specialized adhesion sites between neurons that are connected by protein \n",
      "complexes spanning the synaptic cleft. These trans‐synaptic interactions can organize …\n",
      "\n",
      "Despite the central role PSD-95 plays in anchoring postsynaptic AMPARs, how PSD-95 \n",
      "itself is tethered to postsynaptic sites is not well understood. Here we show that the F-actin …\n",
      "\n",
      "Mutations or altered protein levels of SHANK3 are implicated in neurodevelopmental disorders \n",
      "such as Phelan–McDermid syndrome, autism spectrum disorders, and schizophrenia ( …\n",
      "\n",
      "Variants in the ANK3 gene encoding ankyrin-G are associated with neurodevelopmental \n",
      "disorders, including intellectual disability, autism, schizophrenia, and bipolar disorder. However…\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "Scribble (Scrib) is a key regulator of apicobasal polarity, presynaptic architecture, and short-term \n",
      "synaptic plasticity in Drosophila. In mammals, its homolog Scrib1 has been implicated …\n",
      "\n",
      "LRRTMs are postsynaptic cell adhesion proteins that have region-restricted expression in the \n",
      "brain. To determine their role in the molecular organization of synapses in vivo, we studied …\n",
      "\n",
      "… Knockouts were confirmed by Sanger sequencing and Western blotting using anti- ATAD1 \n",
      "antibody (N125/10, NeuroMab) and anti- Histone H3 (ab1791, Abcam) as a loading control. …\n",
      "\n",
      "The spatial and temporal balance of spinal α‐motoneuron (αMN) intrinsic membrane \n",
      "conductances underlies the neural output of the final common pathway for motor commands. …\n",
      "\n",
      "Shank3 is a postsynaptic density (PSD) scaffold protein of the Shank family. Here we use \n",
      "pre-embedding immunogold electron microscopy to investigate factors influencing the …\n",
      "\n",
      "… subjected to immunoblotting with antibody N69/46 to Shank3. The … N69/46 anti-Shank3 \n",
      "antibody directed against an epitope downstream of the PDZ domain (UC Davis/NIH NeuroMab …\n",
      "\n",
      "Haploinsufficiency of the Shank3 gene, which encodes a scaffolding protein at glutamatergic \n",
      "synapses, is a highly prevalent and penetrant risk factor for autism. Using combined …\n",
      "\n",
      "Rho GTPases activated by GDP/GTP exchange factors (GEFs) play key roles in the \n",
      "developing and adult nervous system. Kalirin-7 (Kal7), the predominant adult splice form of the …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "… of NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs \n",
      "… for specific labeling (N355/1 in a1 and N399/19 in b1) and two other candidates from these …\n",
      "\n",
      "Neurons in the rostral ventrolateral medulla (RVLM) regulate blood pressure through direct \n",
      "projections to spinal sympathetic preganglionic neurons. Only some RVLM neurons are …\n",
      "\n",
      "As a basic functional unit in neural circuits, each neuron integrates input signals from hundreds \n",
      "to thousands of synapses. Knowledge of the synaptic input fields of individual neurons, …\n",
      "\n",
      "… Here, we describe a BaseScope/RNAscope-immunohistochemical protocol using a mouse \n",
      "monoclonal anti-GFP antibody from NeuroMab (clone N86/8; University of California, Davis). …\n",
      "\n",
      "… the auspices of the National Institute of Child Health and Human Development and \n",
      "maintained by the University of Iowa; and from the University of California Davis/NIH NeuroMab …\n",
      "\n",
      "… The primary antibodies used for western blotting were mouse monoclonal anti-GFP (1∶1000, \n",
      "NeuroMab, N86/8) and rabbit polyclonal anti-FGF14 (1∶1000). Secondary antibodies …\n",
      "\n",
      "Loss of Fragile X mental retardation protein (FMRP) leads to Fragile X syndrome (FXS), the \n",
      "most common form of inherited intellectual disability and autism. Although the functions of …\n",
      "\n",
      "Aggregation of the multifunctional RNA‐binding protein TDP‐43 defines large subgroups of \n",
      "amyotrophic lateral sclerosis and frontotemporal dementia and correlates with …\n",
      "\n",
      "… The GFP N86/8 antibody was from the University of California, Davis/NIH NeuroMab \n",
      "Facility. Drosophila stocks obtained from the Bloomington Drosophila Stock Center (NIH …\n",
      "\n",
      "Anastasis is a recently described process in which cells recover after late-stage apoptosis \n",
      "activation. The functional consequences of anastasis for cells and tissues are not clearly …\n",
      "\n",
      "The homeostatic control of presynaptic neurotransmitter release stabilizes information transfer \n",
      "at synaptic connections in the nervous system of organisms ranging from insect to human. …\n",
      "\n",
      "Chemotherapy-related cognitive impairment (CRCI) is a common adverse effect of treatment \n",
      "and is characterized by deficits involving multiple cognitive domains including memory. …\n",
      "\n",
      "Vesicular trafficking defects, particularly those in the autophagolysosomal system, have \n",
      "been strongly implicated in the pathogenesis of Parkinson’s disease and related α-…\n",
      "\n",
      "INTRODUCTION Mitochondria are endosymbiotic organelles that have their own genome and \n",
      "perform many essential functions in eukaryotic cells, including ATP synthesis via oxidative …\n",
      "\n",
      "Investigating the expression of RNAs that differ by short or single nucleotide sequences at a \n",
      "single-cell level in tissue has been limited by the sensitivity and specificity of in situ …\n",
      "\n",
      "Control of translation is a fundamental source of regulation in gene expression. The induction \n",
      "of protein synthesis by brain-derived neurotrophic factor (BDNF) critically contributes to …\n",
      "\n",
      "Mutations in LRRK2 are the most common cause of familial Parkinson’s disease and typically \n",
      "cause disease in the context of abnormal aggregation and deposition of α-synuclein within …\n",
      "\n",
      "The Ca 2+ /calcineurin-dependent transcription factor NFAT (nuclear factor of activated T-cells) \n",
      "is implicated in regulating dendritic and axonal development, synaptogenesis, and …\n",
      "\n",
      "… Citation 49 Membranes were probed with anti-GFP N86/8 mouse monoclonal antibody (1 \n",
      "μg/mL; UC Davis/NeuroMab Facility, 73–131:RRID AB_10671444) and rabbit anti-Zwf1/…\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… ) prior to immuno-detection with mouse anti-GFP (UC Davis/NIH Neuromab Facility, Cat. \n",
      "No.: N86/8) and rabbit anti Calnexin antibody (Enzo Life Sciences, Cat. No.: ADI-SPA-860). …\n",
      "\n",
      "Förster resonance energy transfer (FRET) microscopy is frequently used to study protein \n",
      "interactions and conformational changes in living cells. The utility of FRET is limited by false …\n",
      "\n",
      "Background Guanine nucleotide exchange factors (GEFs) and their target Rho GTPases \n",
      "regulate cytoskeletal changes and membrane trafficking. Dynamin, a large force-generating …\n",
      "\n",
      "… The blot was blocked with 5% milk for 1 hour, and then developed with 1 μg/mL anti-GFP \n",
      "mouse monoclonal clone N86/8 (catalogue no. 75-131; nonprofit University of California Davis/…\n",
      "\n",
      "Autophagy promotes protein degradation, and therefore has been proposed to maintain \n",
      "amino acid pools to sustain protein synthesis during metabolic stress. To date, how autophagy …\n",
      "\n",
      "Altered excitatory/inhibitory (E/I) balance is implicated in neuropsychiatric and neurodevelopmental \n",
      "disorders, but the underlying genetic etiology remains poorly understood. Copy …\n",
      "\n",
      "… (NeuroMab clone N86/8) were obtained commercially. Following incubation with the primary \n",
      "antibody, the membrane was exposed to the HRP-conjugated goat anti-rabbit secondary …\n",
      "\n",
      "10.7554/eLife.16807.001 Altered insulin signaling has been linked to widespread nervous \n",
      "system dysfunction including cognitive dysfunction, neuropathy and susceptibility to …\n",
      "\n",
      "… Antibodies were purchased: mouse anti-GFP (clone N86/8, antibody registry #AB_10671444, \n",
      "UC Davis/NIH NeuroMab Facility), rabbit anti-GFP (Invitrogen), rat monoclonal anti-HA (…\n",
      "\n",
      "… Mouse anti-GFP antibody (N86/8) was from NeuroMab. Mouse anti-β tubulin (E7) and anti-GAPDH \n",
      "(MAB374) were from DSHB and MilliporeSigma (Burlington, Massachusetts, USA), …\n",
      "\n",
      "… Louis, MO), anti–glucose-6-phosphate dehydrogenase (G6PDH; Zwf1; 1:100,000; Sigma-Aldrich), \n",
      "or anti-GFP (1 μg/ml; N86/8; Neuromab, Davis, CA) primary antibodies and visualized …\n",
      "\n",
      "Phosphorylation regulates surface and synaptic expression of NMDA receptors (NMDARs). \n",
      "Both the tyrosine kinase Fyn and the tyrosine phosphatase striatal-enriched protein tyrosine …\n",
      "\n",
      "Presynaptic homeostatic plasticity (PHP) adaptively regulates synaptic transmission in health \n",
      "and disease. Despite identification of numerous genes that are essential for PHP, we lack …\n",
      "\n",
      "The submicroscopic spatial organization of cell surface receptors and plasma membrane \n",
      "signaling molecules is readily characterized by electron microscopy (EM) via immunogold …\n",
      "\n",
      "Proper brain development and function requires finely controlled mechanisms for protein \n",
      "turnover, and disruption of genes involved in proteostasis is a common cause of …\n",
      "\n",
      "The C9orf72 repeat expansion causes amyotrophic lateral sclerosis and frontotemporal \n",
      "dementia, but the poor correlation between C9orf72‐specific pathology and TDP ‐43 pathology …\n",
      "\n",
      "The pathogenesis of peripheral neuropathies in adults is linked to maintenance mechanisms \n",
      "that are not well understood. Here, we elucidate a novel critical maintenance mechanism …\n",
      "\n",
      "Structural plasticity of dendritic spines and synapses is a fundamental mechanism governing \n",
      "neuronal circuits and may form an enduring basis for information storage in the brain. We …\n",
      "\n",
      "… Membranes were probed with α-GFP N86/8 monoclonal antibody (1 µg/ml; NeuroMab), α-G6PDH \n",
      "(Zwf1; 1:1,000; Sigma-Aldrich), α-Atg13 (1:1,000; polyclonal, gift of D. Klionsky, …\n",
      "\n",
      "RNA-binding proteins (RBPs) function in higher-order assemblages such as RNA granules \n",
      "to regulate RNA localization and translation. The Fragile X homolog FXR2P is an RBP …\n",
      "\n",
      "Intronic hexanucleotide (G 4 C 2 ) repeat expansions in C9orf72 are genetically associated \n",
      "with frontotemporal lobar degeneration ( FTLD ) and amyotrophic lateral sclerosis ( ALS ). …\n",
      "\n",
      "… The following antibodies were used: Anti-GFP (mouse N86/8, Neuromab, Davis, CA, USA \n",
      "and rabbit, Clontech, Mountainview, CA, USA), anti-β-actin (Sigma Aldrich, St. Louis, MO, USA)…\n",
      "\n",
      "Long-term forms of brain plasticity share a requirement for changes in gene expression \n",
      "induced by neuronal activity. Mechanisms that determine how the distinct and overlapping …\n",
      "\n",
      "A repeat expansion in the non-coding region of C9orf72 gene is the most common mutation \n",
      "causing frontotemporal lobar degeneration (FTLD) and amyotrophic lateral sclerosis (ALS). …\n",
      "\n",
      "… Mouse monoclonal anti-gfp, clone N86/8, was obtained from Antibodies Inc. (University \n",
      "of California Davis, National Institutes of Health NeuroMab, Davis, CA). Mouse monoclonal …\n",
      "\n",
      "The DISC1 protein is implicated in major mental illnesses including schizophrenia, depression, \n",
      "bipolar disorder, and autism. Aberrant mitochondrial dynamics are also associated with …\n",
      "\n",
      "MicroRNAs (miRNAs) are important regulators of localized mRNA translation in neuronal \n",
      "dendrites. The presence of RNA-induced silencing complex proteins in these compartments …\n",
      "\n",
      "… Mouse monoclonal anti-gfp, clone N86/8 was obtained from Antibodies Incorporated (UC \n",
      "Davis, NIH NeuroMab, Davis, CA, USA).Mouse monoclonal antibody against beta-actin (cat # …\n",
      "\n",
      "nan\n",
      "\n",
      "Astroglia play active and diverse roles in modulating neuronal/synaptic functions in the CNS. \n",
      "How these astroglial functions are regulated, especially by neuronal signals, remains …\n",
      "\n",
      "Mitochondrial transport plays an important role in matching mitochondrial distribution to \n",
      "localized energy production and calcium buffering requirements. Here, we demonstrate that …\n",
      "\n",
      "Repeat expansion in C9orf72 causes amyotrophic lateral sclerosis and frontotemporal lobar \n",
      "degeneration. Expanded sense and antisense repeat RNA transcripts in C9orf72 are …\n",
      "\n",
      "Mutations in leucine-rich repeat kinase 2 (LRRK2) are the most common cause of familial \n",
      "Parkinson disease. Genetics and neuropathology link Parkinson disease with the microtubule-…\n",
      "\n",
      "Cell‐to‐cell transmission of protein aggregates is an emerging theme in neurodegenerative \n",
      "disease. Here, we analyze the dipeptide repeat ( DPR ) proteins that form neuronal …\n",
      "\n",
      "The microtubule binding protein tau is strongly implicated in multiple neurodegenerative \n",
      "disorders, including frontotemporal dementia and parkinsonism linked to chromosome 17 (FTDP…\n",
      "\n",
      "Rare variants in GRIN genes, which encode NMDAR subunits, are strongly associated with \n",
      "neurodevelopmental disorders. Among these, GRIN2A, which encodes the GluN2A subunit …\n",
      "\n",
      "Neuron‐secreted factors induce astrocytic expression of the glutamate transporter, GLT ‐1 (excitatory \n",
      "amino acid transporter 2). In addition to their elaborate anatomic relationships with …\n",
      "\n",
      "Synaptic plasticity requires remodeling of the actin cytoskeleton. Although two actin isoforms, \n",
      "β- and γ-actin, are expressed in dendritic spines, the specific contribution of γ-actin in the …\n",
      "\n",
      "The dystrophin complex is a multimolecular membrane-associated protein complex whose \n",
      "defects underlie many forms of muscular dystrophy. The dystrophin complex is postulated to …\n",
      "\n",
      "Mutations in the X-linked MECP2 cause Rett syndrome, a devastating neurological disorder \n",
      "typified by a period of apparently normal development followed by loss of cognitive and …\n",
      "\n",
      "GGGGCC (G 4 C 2 ) repeat expansion in the C9orf72 gene has been shown to cause \n",
      "frontotemporal lobar degeneration and amyotrophic lateral sclerosis. Dipeptide repeat proteins …\n",
      "\n",
      "The process by which future behavioral responses are shaped by past experiences is one of \n",
      "the central questions in neuroscience. To gain insight into this process at the molecular and …\n",
      "\n",
      "10.7554/eLife.31181.001 Treatment for medulloblastoma, the most common malignant \n",
      "brain tumor in children, remains limited to surgical resection, radiation, and traditional …\n",
      "\n",
      "… Mouse IgG2a monoclonal antibody (NeuroMab, clone N86/8), raised against fusion \n",
      "protein amino acid 1-238 (full length) of jellyfish green fluorescent protein. …\n",
      "\n",
      "Protein phosphatase 2A (PP2A) is a large enzyme family responsible for most cellular Ser/Thr \n",
      "dephosphorylation events. PP2A substrate specificity, localization, and regulation by …\n",
      "\n",
      "Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) are severe \n",
      "neurodegenerative diseases with overlapping clinical and pathological features. Recently, a …\n",
      "\n",
      "Nestin, an intermediate filament protein widely used as a marker of neural progenitors, was \n",
      "recently found to be expressed transiently in developing cortical neurons in culture and in …\n",
      "\n",
      "10.7554/eLife.07739.001 The conserved MICOS complex functions as a primary determinant \n",
      "of mitochondrial inner membrane structure. We address the organization and functional …\n",
      "\n",
      "Mitochondrial transport plays an important role in matching mitochondrial distribution to \n",
      "localised energy production and calcium buffering requirements. Miro is an outer mitochondrial …\n",
      "\n",
      "Protein translation is necessary for cell function, but it is an incredibly energy demanding \n",
      "process, and is therefore tightly regulated by the metabolic state of the cell. There are a …\n",
      "\n",
      "Schizophrenia is associated with cognitive and behavioral dysfunctions thought to reflect \n",
      "imbalances in neurotransmission systems. Recent screenings suggested that lack of (functional…\n",
      "\n",
      "… antibody to PSD-95 (clone K28/43, NeuroMab), rabbit antibody to GRIP1 (Millipore), mouse \n",
      "antibody to PICK1 (clone L20/8, NeuroMab) and rabbit antibody to phospho-Ser880 GluA2 (…\n",
      "\n",
      "Protein interacting with C-kinase 1 (PICK1) is a 415-aa multidomain scaffold protein \n",
      "encoded by the PICK1 gene. Accumulating evidence suggests that PICK1 is involved in the …\n",
      "\n",
      "… ), and NR2B (1:300, mouse IgG2a N59/20; NeuroMab). Sections were washed three times \n",
      "for 5 min in 1× PBS at room temperature with gentle agitation before applying Alexa Fluor dye-…\n",
      "\n",
      "MicroRNAs (miRNAs) are important regulators of localized mRNA translation in neuronal \n",
      "dendrites. The presence of RNA-induced silencing complex proteins in these compartments …\n",
      "\n",
      "… Mouse anti-PICK1 antibody (#L20-8) was from NeuroMab (Davis, CA) and rabbit \n",
      "polycolonal anti-PICK1 antibody was from Jun Xia (Hong Kong). Horseradish peroxidase-conjugated …\n",
      "\n",
      "Excitatory glutamatergic synapses facilitate important aspects of communication between \n",
      "the neurons that govern complex forms of behavior. Accordingly, small differences in the …\n",
      "\n",
      "The authors regret an error in section 2.3 of the Methods section of the manuscript. We \n",
      "inadvertently listed the Arl13b antibody lot number (455–8JD-29) in place of the antibody clone …\n",
      "\n",
      "Primary cilia are microtubule-based organelles that protrude from the cell surface. Primary \n",
      "cilia play a critical role in development and disease through regulation of signaling pathways …\n",
      "\n",
      "… NeuroMab Facility, clone N295B/66) to verify the marking of cilia with the antibody against \n",
      "acetylated tubulin. The slides were then washed with PBS for 10 minutes (3 x 10 min). The …\n",
      "\n",
      "Genetic diseases involving overactivation of the mechanistic target of rapamycin (mTOR) \n",
      "pathway, so‐called “mTORopathies,” often manifest with malformations of cortical development …\n",
      "\n",
      "The neural mechanisms that underlie responses to drugs of abuse are complex, and \n",
      "impacted by a number of neuromodulatory peptides. Within the past 10 years it has been …\n",
      "\n",
      "Background Ciliary defects cause heterogenous phenotypes related to mutation burden which \n",
      "lead to impaired development. A previously reported homozygous deletion in the Man1a2 …\n",
      "\n",
      "The primary cilium is a nexus for cell signaling and relies on specific protein trafficking for \n",
      "function. The tubby family protein TULP3 transports integral membrane proteins into cilia …\n",
      "\n",
      "… on knockout tissue by NeuroMab (documented on NeuroMab Clone N295B/66 datasheet). In \n",
      "WB … detects an ≈60 kDa band that is absent in the Arl13b knockout (datasheet NeuroMab). …\n",
      "\n",
      "… The following antibodies and dilutions were used in this study: mouse anti-ARL13B (NeuroMabs \n",
      "N295B/66, 1:500), rabbit anti-ARL13B (gift from Tamara Caspary, 1:500, and …\n",
      "\n",
      "Cilia are highly conserved organelles, which serve critical roles in development and physiology. \n",
      "Motile cilia are expressed in a limited range of tissues, where they principally regulate …\n",
      "\n",
      "… ), rabbit anti-Olig2 (AB9610, 1:500, Millipore Sigma), and mouse anti-Arl13b (N295B/66, 1:1500, \n",
      "NeuroMab). Multiple sections from three embryos of each genotype were examined. …\n",
      "\n",
      "Activating LRRK2 mutations cause Parkinson’s disease, and pathogenic LRRK2 kinase \n",
      "interferes with ciliogenesis. Previously, we showed that cholinergic interneurons of the dorsal …\n",
      "\n",
      "Retinitis pigmentosa (RP) is an inherited retinal degenerative disease with severe vision \n",
      "impairment leading to blindness. About 10–15% of RP cases are caused by mutations in the …\n",
      "\n",
      "… (C) Confocal images of cilia marker IFT88 (green) and ARL13B (magenta; NeuroMab) \n",
      "staining in neural tube of E10.5 somite-matched embryos. ARL13B-positive cilia are visible in …\n",
      "\n",
      "Cellular primary cilia crucially sense and transduce extracellular physicochemical stimuli. \n",
      "Cilium-mediated developmental signaling is tissue and cell type specific. Primary cilia are …\n",
      "\n",
      "The primary cilium is a paradigmatic organelle for studying compartmentalized signaling; \n",
      "however, unlike soluble protein trafficking, processes targeting integral membrane proteins to …\n",
      "\n",
      "The primary cilium is fundamentally important for the proliferation of neural stem/progenitor \n",
      "cells and for neuronal differentiation during embryonic, postnatal, and adult life. In addition, …\n",
      "\n",
      "Background: Cilia are important for Hedgehog signaling in vertebrates and many genes that \n",
      "encode proteins involved in ciliogenesis have been studied for their roles in embryonic …\n",
      "\n",
      "… NeuroMab Facility clone N295B/66), rabbit anti-centrosome and spindle pole associated \n",
      "protein 1 (CSPP1, 1:200, Proteintech, 11931-1-AP), rabbit anti-intraflagellar transport protein …\n",
      "\n",
      "During the development of the cerebral cortex, neurons are generated directly from radial \n",
      "glial cells or indirectly via basal progenitors. The balance between these division modes …\n",
      "\n",
      "We have previously described a forward genetic screen in mice for abnormalities of brain \n",
      "development. Characterization of two hydrocephalus mutants by whole-exome sequencing …\n",
      "\n",
      "We tested whether cannabinoids (CBs) potentiate alcohol-induced birth defects in mice and \n",
      "zebrafish, and explored the underlying pathogenic mechanisms on Sonic Hedgehog (Shh) …\n",
      "\n",
      "Histone H4 lysine 16 acetylation (H4K16ac), governed by the histone acetyltransferase MOF, \n",
      "orchestrates gene expression regulation and chromatin interaction. However, the roles of …\n",
      "\n",
      "Growing evidence shows that alterations occurring at early developmental stages contribute \n",
      "to symptoms manifested in adulthood in the setting of neurodegenerative diseases. Here, …\n",
      "\n",
      "The primary cilium is a microtubule-enriched cell-communication organelle that participates \n",
      "in mechanisms controlling tissue development and maintenance, including cerebellar …\n",
      "\n",
      "Four distinct subgroups of cerebellar medulloblastomas (MBs) differ in their histopathology, \n",
      "molecular profiles, and prognosis. c-Myc (Myc) or MycN overexpression in granule neuron …\n",
      "\n",
      "… They were incubated overnight in mouse anti-ADP-ribosylation factor-like protein 13B (Arl13b \n",
      "clone N295B/66, NeuroMab) diluted 1:100 in blocking buffer at 4 ​C overnight. The LROs …\n",
      "\n",
      "Cilia perform essential signalling functions during development and tissue homeostasis. A \n",
      "key event in ciliogenesis occurs when the distal appendages of the mother centriole form a …\n",
      "\n",
      "As essential components of the eukaryotic cytoskeleton, microtubules fulfill a variety of functions \n",
      "that can be temporally and spatially controlled by tubulin posttranslational modifications. …\n",
      "\n",
      "Primary cilia are complex organelles, usually singularly located on cell surfaces that are \n",
      "now known to be important for signaling and whose defect is implicated in a category of …\n",
      "\n",
      "… The following primary antibodies were used: mouse anti-ARL13B and rabbit-anti-ARL13B \n",
      "(NeuroMab, clone N295B/66; 1:300 dilution for immunofluorescence (IF) and ProteinTech, …\n",
      "\n",
      "Primary cilia are compartmentalized sensory organelles present on the majority of neurons \n",
      "in the mammalian brain throughout adulthood. Recent evidence suggests that cilia regulate …\n",
      "\n",
      "Polycystic kidney disease proteins, polycystin-1 and polycystin-2, localize to primary cilia. \n",
      "Polycystin knockouts have severe cystogenesis compared to ciliary disruption, whereas …\n",
      "\n",
      "Evidence is lacking as to how developing neurons integrate mitogenic signals with \n",
      "microenvironment cues to control proliferation and differentiation. We determine that the Siah2 E3 …\n",
      "\n",
      "The microtubule cytoskeleton is one of the principal components of eukaryotic cells. Microtubules \n",
      "are involved in the processes of cell division, transport, signaling and differentiation. …\n",
      "\n",
      "The regulation of proliferation is a primary function of Hedgehog (Hh) signaling in development. \n",
      "Hh signal transduction requires the primary cilium for several steps in the pathway [1–5]. …\n",
      "\n",
      "Primary cilia are sensory organelles present on most vertebrate cells and are critical for \n",
      "development and health. Ciliary dysfunction is associated with a large class of human …\n",
      "\n",
      "Aberrant Notch signalling has been linked to many cancers including choroid plexus (CP) \n",
      "tumours, a group of rare and predominantly paediatric brain neoplasms. We developed …\n",
      "\n",
      "Primary Cilia (PC) are a very likely place for signal integration where multiple signaling \n",
      "pathways converge. Two major signaling pathways clearly shown to signal through the PC, …\n",
      "\n",
      "Background The mechanisms by which primary cilia affect glioma pathogenesis are unclear. \n",
      "Depending on the glioma cell line, primary cilia can promote or inhibit tumor development. …\n",
      "\n",
      "The health and maintenance of rod photoreceptors depends on the high-fidelity vectorial \n",
      "transport of rhodopsin from its site of synthesis in the inner segment to its final resting place in …\n",
      "\n",
      "The intracellular ciliogenesis pathway requires membrane trafficking, fusion, and reorganization. \n",
      "Here, we demonstrate in human cells and zebrafish that the F-BAR domain containing …\n",
      "\n",
      "Cilia are near ubiquitous small, cellular appendages critical for cell-to-cell communication. \n",
      "As such, they are involved in diverse developmental and homeostatic processes, including …\n",
      "\n",
      "Sonic hedgehog (Shh) signal transduction specifies ventral cell fates in the neural tube and \n",
      "is mediated by the Gli transcription factors that play both activator (GliA) and repressor (GliR) …\n",
      "\n",
      "Mutations in the hepatocyte nuclear factor (HNF)1β gene (HNF1B) cause autosomal dominant \n",
      "tubulointerstitial kidney disease, a rare and heterogeneous disease characterized by …\n",
      "\n",
      "Centriolar satellites are dynamic, membraneless granules composed of over 200 proteins. \n",
      "They store, modify, and traffic centrosome and primary cilium proteins, and help to regulate …\n",
      "\n",
      "Primary cilia are a small microtubule based signaling organelle. Primary cilia can be found \n",
      "on almost every mammalian cell, including neurons and glia. Tau Tubulin Kinase 2 (TTBK2) …\n",
      "\n",
      "The small GTPase Arl13b is enriched in primary cilia and regulates Sonic hedgehog (Shh) \n",
      "signaling. During neural development, Shh controls patterning and proliferation through a …\n",
      "\n",
      "Multiciliated cells (MCCs) in the brain reside in the ependyma and the choroid plexus (CP) \n",
      "epithelia. The CP secretes cerebrospinal fluid that circulates within the ventricular system, …\n",
      "\n",
      "Primary cilia are chemosensors that play a dual role to either activate or repress Hedgehog \n",
      "signaling, depending on presence or absence of ligand, respectively. While inhibition of …\n",
      "\n",
      "Arl13b is a gene known to regulate ciliogenesis. Functional alterations in this gene's activity \n",
      "have been associated with Joubert syndrome. We found that in Arl13 null mouse embryos …\n",
      "\n",
      "… Rabbit polyclonal anti-WDR5 (Bethyl, A302-430A; 1:150), mouse monoclonal anti-Arl13b \n",
      "(NeuroMab, N295B/66; 1:100) and mouse monoclonal anti-γ tubulin (Sigma-Aldrich, T6557; 1:…\n",
      "\n",
      "Cilia-mediated signal transduction involves precise targeting and localization of selected \n",
      "molecules along the ciliary membrane. However, the molecular mechanism underlying these …\n",
      "\n",
      "Trisomy 21 (T21) is the most prevalent human chromosomal disorder, causing a range of \n",
      "cardiovascular, musculoskeletal, and neurological abnormalities. However, the cellular …\n",
      "\n",
      "The primary cilium in neural stem cells plays distinct roles in different stages during cortical \n",
      "development. Ciliary dysfunctions in human (ie, ciliopathy) cause developmental defects in …\n",
      "\n",
      "In the absence of Hedgehog ligand, patched-1 (Ptch1) localizes to cilia and prevents ciliary \n",
      "accumulation and activation of smoothened (Smo). Upon ligand binding, Ptch1 is removed …\n",
      "\n",
      "Sonic hedgehog (Shh) determines cerebellar granule cell (GC) progenitor proliferation and \n",
      "medulloblastoma pathogenesis. However, the pathways regulating GC progenitors during …\n",
      "\n",
      "The mechanisms underlying subcellular targeting of cAMP-generating adenylyl cyclases \n",
      "and processes regulated by their compartmentalization are poorly understood. Here, we …\n",
      "\n",
      "Primary cilia are ubiquitous sensory organelles that mediate diverse signaling pathways. \n",
      "Cilia position on the cell surface is determined by the location of the basal body (BB) that …\n",
      "\n",
      "Defects in primary cilia, cellular antennas that control multiple intracellular signaling pathways, \n",
      "underlie several neurodevelopmental disorders, but it remains unknown how cilia control …\n",
      "\n",
      "Primary cilia perceive the extracellular environment through receptors localized in the ciliary \n",
      "membrane, but mechanisms directing specific proteins to this domain are poorly understood…\n",
      "\n",
      "Cilia are organelles specialized for movement and signaling. To infer when during evolution \n",
      "signaling pathways became associated with cilia, we characterized the proteomes of cilia …\n",
      "\n",
      "Mutations in the Abelson-helper integration site 1 (AHI1) gene are associated with \n",
      "neurological/neuropsychiatric disorders, and cause the neurodevelopmental ciliopathy Joubert …\n",
      "\n",
      "Trisomy 21, the genetic cause of Down syndrome, disrupts primary cilia formation and function, \n",
      "in part through elevated Pericentrin, a centrosome protein encoded on chromosome 21. …\n",
      "\n",
      "… NeuroMab, a joint facility of the University of California-Davis and the NIH (http://www.neuromab.… \n",
      "Two mAbs were produced from the A3 VCD fusion: N144/17 (anti-A3), which is specific …\n",
      "\n",
      "Neuregulin 1 (NRG1) is a neuron-derived trophic molecule that supports axoglial and \n",
      "neuromuscular development through several alternatively spliced isoforms; its possible role in the …\n",
      "\n",
      "… Use of the type III (cysteine-rich domain) specific antibody (clone N126B/31, UC Davis/NIH \n",
      "NeuroMab facility) was as described previously (Citation6). …\n",
      "\n",
      "Classical accommodation is a form of spike frequency adaptation in neurons whereby \n",
      "excitatory drive results in action potential output of gradually decreasing frequency. Here we …\n",
      "\n",
      "C-type synaptic boutons (C-boutons) provide cholinergic afferent input to spinal cord motor \n",
      "neurons (MNs), which display an endoplasmic reticulum (ER)–related subsurface cistern (…\n",
      "\n",
      "… were purchased from the UC Davis/NIH NeuroMab facility: AnkG (106/36; RRID:AB_10673030), \n",
      "AnkG (106/65; RRID:AB_10675130), PanNav (N419/78; RRID:AB_2493099), b1 …\n",
      "\n",
      "Skeletal muscle contraction depends on activation of clustered acetylcholine receptors (AchRs) \n",
      "and muscle-specific Na + channels (Nav1.4). Some Nav1.4 channels are highly …\n",
      "\n",
      "… The following monoclonal primary antibodies were purchased from the UC Davis/NIH \n",
      "NeuroMab facility: AnkG (106/36; RRID: AB_10673030), Pan-Nav1 (N419/78; RRID: AB_2493099)…\n",
      "\n",
      "Neuronal ankyrins cluster and link membrane proteins to the actin and spectrin-based \n",
      "cytoskeleton. Among the three vertebrate ankyrins, little is known about neuronal Ankyrin-R (AnkR…\n",
      "\n",
      "The axon initial segment (AIS) is a highly specialized axonal compartment where the action \n",
      "potential is initiated. The heterogeneity of AISs has been suggested to occur between …\n",
      "\n",
      "Mutations of the cyclin-dependent kinase-like 5 (CDKL5) and netrin-G1 (NTNG1) genes \n",
      "cause a severe neurodevelopmental disorder with clinical features that are closely related to …\n",
      "\n",
      "… Mouse monoclonal anti-PSD-93 (N18/30) and anti-ADAM22 (cytoplasmic, N46/30; extracellular, \n",
      "N57/2) were obtained from the University of California Davis/NIH NeuroMab Facility and …\n",
      "\n",
      "Disruption of axon‐glia interactions in the peripheral nervous system has emerged as a major \n",
      "cause of arthrogryposis multiplex congenita (AMC), a condition characterized by multiple …\n",
      "\n",
      "… A mouse monoclonal anti-SNAT1 antibody (N104/37; NeuroMabs, UC Davis, like the rabbit \n",
      "… background staining (Figure S1) and therefore N104/37 was used. Free-floating sections …\n",
      "\n",
      "… The LRRTM4 monoclonal antibody N205B/22 was developed with the UC Davis/NIH \n",
      "NeuroMab Facility. Mono- and disaccharide analysis of GPC4-Fc was performed by the UCSD …\n",
      "\n",
      "Synaptic cell-adhesion molecules (CAMs) organize the architecture and properties of neural \n",
      "circuits. However, whether synaptic CAMs are involved in activity-dependent remodeling of …\n",
      "\n",
      "Presynaptic neurexins (Nrxs) and type IIa receptor-type protein tyrosine phosphatases (RPTPs) \n",
      "organize synapses through a network of postsynaptic ligands. We show that leucine-rich-…\n",
      "\n",
      "LRRTMs are postsynaptic cell adhesion proteins that have region-restricted expression in the \n",
      "brain. To determine their role in the molecular organization of synapses in vivo, we studied …\n",
      "\n",
      "LRRTM4 is a transsynaptic adhesion protein regulating glutamatergic synapse assembly on \n",
      "dendrites of central neurons. In the mouse retina, we find that LRRTM4 is enriched at …\n",
      "\n",
      "The four members of the LRRTM family (LRRTM1-4) are postsynaptic adhesion molecules \n",
      "essential for excitatory synapse development. They have also been implicated in …\n",
      "\n",
      "… The monoclonal antibody vGluT3 was obtained from the UC Davis/NIH NeuroMab Facility, \n",
      "supported by NIH grant U24NS050606 and maintained by the Department of Neurobiology, …\n",
      "\n",
      "… (N29/29), VGLUT3 (N34/34), as well as the mouse monoclonal antibody for VAChT (N6/38) \n",
      "were purchased from Antibodies Incorporated (UC Davis/NIH NeuroMab Facility, Davis, CA). …\n",
      "\n",
      "Glutamate and its receptor N-methyl-D-aspartate receptor (NMDAR) have been associated \n",
      "with cancer, although their functions are not fully understood. Herein, we implicate glutamate-…\n",
      "\n",
      "The Foxi3 transcription factor, expressed in the neural plate border at the end of gastrulation, \n",
      "is necessary for the formation of posterior placodes and is thus important for ectodermal …\n",
      "\n",
      "… The mouse monoclonal antibody K88-64 that we used was … Unexpectedly, we showed that \n",
      "the Mab K88-64 has a very … the UC Davis/NINDS/NIMH NeuroMab Facility, supported by NIH …\n",
      "\n",
      "Cortactin-binding protein 2 (CTTNBP2) interacts with cortactin to regulate cortactin mobility \n",
      "and control dendritic spine formation. CTTNBP2 has also been associated with autistic …\n",
      "\n",
      "… Syt3 NB is a monoclonal antibody from Neuromab, and Syt3 NT is a polyclonal antibody \n",
      "developed by Synaptic Systems. Tubulin is a loading control. (B) Syt3 immunoreactivity is …\n",
      "\n",
      "A hallmark of inflammatory diseases is the excessive recruitment and influx of monocytes to \n",
      "sites of tissue damage and their ensuing differentiation into macrophages. Numerous stimuli …\n",
      "\n",
      "… monoclonal antibodies that specifically target QKI-5 (N195A/16), QKI-6 (N182/17), or QKI-7 \n",
      "(N183/15.1; UC Davis/NIH NeuroMab Facility). Mouse monoclonal antibodies were used to …\n",
      "\n",
      "Alternative splicing is critical for animal ontogeny; however, its role in the earliest developmental \n",
      "decision, the specification of the three embryonic germ layers, is poorly understood. By …\n",
      "\n",
      "… (APC-021; d) and a mouse monoclonal (L6/23; e) antibody against the BK channel. f, The … \n",
      "(APC-021 recognizes an epitope between amino acid residues 1089 and 1196, and L6/23 …\n",
      "\n",
      "… (APC-021; d) and a mouse monoclonal (L6/23; e) antibody against the BK channel. f, The … \n",
      "(APC-021 recognizes an epitope between amino acid residues 1089 and 1196, and L6/23 …\n",
      "\n",
      "α-Synucleinopathies are spreading neurodegenerative disorders characterized by the \n",
      "intracellular accumulation of insoluble aggregates populated by α-Synuclein (α-Syn) fibrils. In …\n",
      "\n",
      "A hallmark of inflammatory diseases is the excessive recruitment and influx of monocytes to \n",
      "sites of tissue damage and their ensuing differentiation into macrophages. Numerous stimuli …\n",
      "\n",
      "Alternative splicing is critical for animal ontogeny; however, its role in the earliest developmental \n",
      "decision, the specification of the three embryonic germ layers, is poorly understood. By …\n",
      "\n",
      "… N121A/1 and N121A/31) were generated by UC Davis/NIH NeuroMab Facility (Davis, CA, … \n",
      "Panx2 monoclonal antibodies (N121A/1 and N121A/31) by showing that both clones identified …\n",
      "\n",
      "Pannexins (PANX) are a family of three channel-forming membrane glycoproteins expressed \n",
      "in the skin. Previous studies have focused on the role of PANX1 and PANX3 in the …\n",
      "\n",
      "Traumatic brain injury (TBI) can result in the development of posttraumatic epilepsy (PTE). \n",
      "Recently, we reported differential alterations in tonic and phasic GABA A receptor (GABA A R) …\n",
      "\n",
      "… NeuroMab N423/75. In all cases duplicate immunoblots were probed the anti-Mortalin/GRP75 \n",
      "NeuroMab N52A/42 … in which the anti-pan-KChIP NeuroMab K55/82 was used to show the …\n",
      "\n",
      "Developmental and epileptic encephalopathies (DEE) are a group of severe epilepsies that \n",
      "usually present with intractable seizures, developmental delay, and often have elevated risk …\n",
      "\n",
      "… (NeuroMab N52A/42; RRID:2120479; 1 μg/mL), which served as a normalization control. \n",
      "Anti-rabbit Alexa Fluor 790 and anti-mouse 680 antibodies (Jackson ImmunoResearch, 1:…\n",
      "\n",
      "… (1:500; KC; RRID: AB_2315767) and as loading controls mouse anti-mortalin (1:2, N52A/42 \n",
      "tissue culture supernatant; RRID: AB_10674108, from the UC Davis/NIH NeuroMab Facility) …\n",
      "\n",
      "Transient receptor potential melastatin-like 4 (TRPM4) is a Ca 2+ -activated non-selective \n",
      "cation channel expressed in a wide range of human tissues. TRPM4 participates in a variety of …\n",
      "\n",
      "Neuroinflammation plays a fundamental role in the pathogenesis of Alzheimer's disease (AD), \n",
      "resulting in the extensive activation of microglial and astroglial cells. Here we describe the …\n",
      "\n",
      "Voltage-gated K + (Kv) channels play important roles in regulating neuronal excitability. Kv \n",
      "channels comprise four principal α subunits, and transmembrane and/or cytoplasmic …\n",
      "\n",
      "The plasma membrane (PM) comprises distinct subcellular domains with diverse functions \n",
      "that need to be dynamically coordinated with intracellular events, one of the most impactful …\n",
      "\n",
      "TRPM4 is a Ca2+-activated non-selective cation channel expressed in a wide range of \n",
      "human tissues. TRPM4 participates in a variety of physiological processes such as T cell …\n",
      "\n",
      "… (K7/45) and Mortalin/Grp75 (N52A/42) using monoclonal antibodies (clone number in \n",
      "parentheses) obtained from the UC Davis/NIH NeuroMab Facility. Alexa-conjugated fluorescent …\n",
      "\n",
      "… N52A/42; NeuroMab), and subsequently with goat anti-mouse or anti-rabbit IgG–HRP \n",
      "secondary antibodies (1:10,000; Antibodies Inc.). Immunoreactive proteins on membranes were …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… The plasmid backbone was generated by EcoRI/HindIII restriction digestion of the N52A/42 …\n",
      "\n",
      "… For immunoblots we used anti-Grp75/mortalin mAb N52A/42 (UC Davis/NIH NeuroMab \n",
      "Facility, Davis, CA, USA) or otubulin (T5168, Sigma, St Louis, MO, USA) as a loading control. …\n",
      "\n",
      "… subunits (1:1000-500, Clone N52A/42) were obtained from Sigma-Aldrich and the UC \n",
      "Davis/NIH NeuroMab facility, respectively. Control immunoprecipitations were performed using a …\n",
      "\n",
      "The neurodevelopmental factor dysbindin is required for synapse function and GABA \n",
      "interneuron development. Dysbindin protein levels are reduced in the hippocampus of …\n",
      "\n",
      "Errors in replication or segregation lead to DNA damage, mutations, and aneuploidies. \n",
      "Consequently, cells monitor these events and delay progression through the cell cycle so repair …\n",
      "\n",
      "Charcot-Marie-Tooth (CMT) disease 4A is an autosomal-recessive polyneuropathy caused \n",
      "by mutations of ganglioside-induced differentiation-associated protein 1 (GDAP1), a putative …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… The plasmid backbone was generated by EcoRI/HindIII restriction digestion of the N52A/42 …\n",
      "\n",
      "… 1b (N16B/8, NeuroMab, San Diego, CA), rabbit polyclonal anti-Kv3.1b (laboratory-generated), \n",
      "mouse monoclonal anti-mortalin (N52A/42, NeuroMab), mouse monoclonal anti-HA tag (…\n",
      "\n",
      "… For immunoblots we used anti-Grp75/mortalin mAb N52A/42 (UC Davis/NIH NeuroMab \n",
      "Facility, Davis, CA, USA) or otubulin (T5168, Sigma, St Louis, MO, USA) as a loading control. …\n",
      "\n",
      "Recent studies have underscored the importance of C5a and other complement components \n",
      "in inflammatory and neuropathic pain, although the underlying mechanisms are largely …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "Genetic variants in SCN2A, encoding the Na V 1.2 voltage-gated sodium channel, are \n",
      "associated with a range of neurodevelopmental disorders with overlapping phenotypes. Some …\n",
      "\n",
      "… GRP75 was used as a loading control and probed with a monoclonal anti-GRP75 antibody \n",
      "(1:2000; clone N52A/42; Neuromab). The membranes were then washed with TBS and …\n",
      "\n",
      "Dynamic modulation of ion channel expression, localization, and/or function drives plasticity \n",
      "in intrinsic neuronal excitability. Voltage-gated Kv2.1 potassium channels are constitutively …\n",
      "\n",
      "… The mouse mAbs N52A/42, K13/31, K89/34, L83/11, L86/33, and L86A/37 are available \n",
      "from the UC Davis/NIH NeuroMab Facility (http://neuromab.ucdavis.edu/), a not-for profit …\n",
      "\n",
      "… (N372B/60) and Grp75/Mortalin (N52A/42) using mAbs (clone number in parentheses) \n",
      "obtained from the UC-Davis/NIH NeuroMab Facility (http://neuromab.ucdavis.edu/), a not-for profit …\n",
      "\n",
      "Skeletal muscle contraction depends on activation of clustered acetylcholine receptors (AchRs) \n",
      "and muscle-specific Na + channels (Nav1.4). Some Nav1.4 channels are highly …\n",
      "\n",
      "… Mouse Mono K89/34 Amino acids 837–853 ratHMLPGGGAHGSTRDQSI NeuroMab 75-014 … \n",
      "Ankyrin-G Mouse Mono N106/65 full length human NeuroMab 75–147 Purified IgG 10 μg/…\n",
      "\n",
      "… encoding functional recombinant K89/34 mAb (IgG2a). Recombinant K89/34 was produced \n",
      "… Sections were labeled free floating and probed with K89/34 IgG1 antibody diluted in vehicle …\n",
      "\n",
      "… Briefly, Kv2.1 and Kv2.2 expression was characterized using the K89/34 (for Kv2.1) and \n",
      "N372B/1 or N372B/60 (for Kv2.2) antibodies (both from NeuroMab) in 4 Thy1 mice (P26–P42) (…\n",
      "\n",
      "Objective: To perform functional characterization of a potentially pathogenic KCNB1 variant \n",
      "identified by clinical exome sequencing of a proband with a neurodevelopmental disorder …\n",
      "\n",
      "… Immunocytochemistry We stained organotypic slices with the Neuromab Kv2.1 monoclonal \n",
      "antibody (K89/34). Slices contained non-transfected cells and cells with GFP fluorescence, …\n",
      "\n",
      "… (K89/34). Sections were then incubated with fluorescein isothiocyanate-conjugated or cyanine \n",
      "3-conjugated secondary antibodies (Jackson ImmunoResearch, West Grove, PA). Images …\n",
      "\n",
      "… with an anti-K v 2.1 (K89/34) mAb in order to detect the … with an anti-K v 2.1 (K89/34) \n",
      "mAb. The DTDP-induced tyrosine … with anti-K v 2.1 (K89/34, NeuroMab, CA, USA) and in-house …\n",
      "\n",
      "… (K89/34) mAb. Tyrosine phosphorylation of the Kv2.1 channel was detected using an anti-… \n",
      "(K89/34) mAb. PV treatments induced tyrosine phosphorylation of Kv2.1-WT, Kv2.1-…\n",
      "\n",
      "… K89/34 (green) antibodies in cerebral cortex of CO 2 -treated (2 min) mouse… (K89/34 and \n",
      "K39/25), Kv1.2 (K14/16) and PSD-95 (K28/43) were obtained from the UC Davis/NIH NeuroMab …\n",
      "\n",
      "… Neurons were then incubated with the Kv2.1 primary antibody K89/34 (NeuroMab) and a \n",
      "GFP primary antibody (A10262, Invitrogen) overnight at 4 C. Cells were washed three times …\n",
      "\n",
      "… K89/34 can be obtained from NeuroMab (www.neuromab.org… S563P or S603P together \n",
      "with K89/34. The sections were then … with S563P or S603P together with K89/34, and Alexa-…\n",
      "\n",
      "… at the UC Davis/NIH NeuroMab Facility, we have generated a … These monoclonal K89/34 \n",
      "cell lines have been expanded … of the various K89/34 subclones archived at different times. …\n",
      "\n",
      "The microtubule-associated axonal specification collapsin response mediator protein 2 (CRMP2) \n",
      "is a novel target for neuroprotection. A CRMP2 peptide (TAT-CBD3) conjugated to the …\n",
      "\n",
      "Objective Numerous studies have demonstrated increased load of de novo copy number \n",
      "variants or single nucleotide variants in individuals with neurodevelopmental disorders, …\n",
      "\n",
      "… (K7/45) and Mortalin/Grp75 (N52A/42) using monoclonal antibodies (clone number in \n",
      "parentheses) obtained from the UC Davis/NIH NeuroMab Facility. Alexa-conjugated fluorescent …\n",
      "\n",
      "Lysosomes communicate through cholesterol transfer at endoplasmic reticulum (ER) contact \n",
      "sites. At these sites, the Niemann Pick C1 cholesterol transporter (NPC1) facilitates the …\n",
      "\n",
      "… with K89/34 mAb plus D3/71 R-mAb and the immunolabeling detected with a cocktail of \n",
      "anti-mouse IgG1 (green, for the K89/34 … of the UC Davis/NIH NeuroMab Facility and the Trimmer …\n",
      "\n",
      "… The membranes were blocked with 1% BSA in PBS with 0.05% Tween 20 before probing \n",
      "with either an anti-Kv2.1 mouse monoclonal (clone K89/34; NeuroMab, Davis, CA, USA) or an …\n",
      "\n",
      "… at the UC Davis/NIH NeuroMab Facility, we have generated a … These monoclonal K89/34 \n",
      "cell lines have been expanded … of the various K89/34 subclones archived at different times. …\n",
      "\n",
      "… K89/34 mouse mAb (diluted to 10 μg/mL in vehicle: 2% NGS in TBS) or anti-AMIGO-1 L86/33 \n",
      "mouse mAb tissue culture supernatant (diluted 1:2 in vehicle). After three washes in 25 mM …\n",
      "\n",
      "… B) Representative K V 2.1 immunoblot of Kcnb1 G379R brain membrane protein using the \n",
      "anti-K V 2.1 K89/34 mAb. The anti-GRP75 mAb is used as a loading control. C) Quantification …\n",
      "\n",
      "The sigma-1 receptor (σ1-R) and sigma-2 receptor (σ2-R) are potential drug targets for \n",
      "treatment of cancer, pain, depression, retinal degeneration and other neuronal diseases. …\n",
      "\n",
      "… The expression of Kv2.1 was checked using immunofluorescence microscopy and \n",
      "Western blotting using mouse anti-Kv2.1 α subunit antibodies (clone K89/34, NeuroMab). …\n",
      "\n",
      "… For some experiments an anti-Kv2.1 antibody (Neuromab, CA) was added to the intracellular \n",
      "solution (final concentration of 0.5 μg/ml). For HEK293 cells, two voltage protocols were …\n",
      "\n",
      "… K89/34 from NeuroMab). After washing 3 times with the blocking solution, blots were \n",
      "incubated with horseradish peroxidase (HRP)-conjugated anti-mouse IgG antibody (1:10,000, …\n",
      "\n",
      "… then incubated for 1 h with the K89/34 antibody (1:400 in blocking solution; University of \n",
      "California at Davis/National Institutes of Health Neuromab Facility). After repeated washes with …\n",
      "\n",
      "Voltage-gated potassium (K + ) channel sub-family B member 1 (KCNB1, Kv2.1) is known to \n",
      "undergo oxidation-induced oligomerization during aging but whether this process affects …\n",
      "\n",
      "… (1:1000; clone K89/34; NeuroMab) or rabbit polyclonal anti-p38 MAPK or rabbit monoclonal \n",
      "anti-phospho-p38 MAPK (1:800; both from Cell Signaling Technology)]. After washing three …\n",
      "\n",
      "… Similarly, an anti-Kv2.1 mouse mAb (IgG1 mAb K89/34 in green) only labels the Kv2.1 pool \n",
      "in RBM, and the sample from heterologous cells expressing recombinant Kv2.1. Note that …\n",
      "\n",
      "The plasma membrane (PM) comprises distinct subcellular domains with diverse functions \n",
      "that need to be dynamically coordinated with intracellular events, one of the most impactful …\n",
      "\n",
      "Altered neuronal excitability is emerging as an important feature in Alzheimer’s disease (AD). \n",
      "Kv2. 1 potassium channels are important modulators of neuronal excitability and synaptic …\n",
      "\n",
      "… Anti-Kv2.1 antibody (mouse monoclonal clone K89/34) was obtained from NeuroMab. \n",
      "Rabbit anti-phospho Kv2.1 (S800) antibody was provided by Elias Aizenman (University of …\n",
      "\n",
      "… We found the Kv8.2 antibody (Clone N448/88 from NeuroMab) generated a non-specific \n",
      "signal in immunohistochemistry (Supplementary Material, Fig. S2). Courtesy of James Trimmer, …\n",
      "\n",
      "… (clone K89/34; NeuroMab), used at 1 μg/ml; V5: mouse monoclonal (clone SV5-Pk1; Bio-Rad), \n",
      "used at 1–2 μg/ml; GFP: mouse monoclonal antibody (clone N86/6; NeuroMab), used at 1 …\n",
      "\n",
      "… As a positive control, cells colabeled with two distinct Ca V 1.2 antibodies [rabbit FP1 (28) \n",
      "and monoclonal NeuroMab clone N263/31] showed surface membrane staining (fig. S1A) and …\n",
      "\n",
      "… These mAbs are available from the UC Davis/ NIH NeuroMab Facility (neuromab.ucdavis.… \n",
      "(determined by staining with the mouse mAb K89/34), and the Slo1 large conductance calcium…\n",
      "\n",
      "… Kv2.1 antibody (clone K89/34) was purchased from NeuroMab, UC Davis/NIH. GFAP \n",
      "antibody (clone MAB3402) and actin antibody (clone MAB1501) were purchased from Millipore, …\n",
      "\n",
      "Voltage-gated sodium (Nav) channels initiate action potentials in brain neurons and are \n",
      "primary therapeutic targets for anti-epileptic drugs controlling neuronal hyperexcitability in …\n",
      "\n",
      "Nursing has important physiological and psychological consequences on mothers during the \n",
      "postpartum period. Tuberoinfundibular peptide of 39 residues (TIP39) may contribute to its …\n",
      "\n",
      "In arterial myocytes, the canonical function of voltage-gated Ca V 1.2 and KV 2.1 channels \n",
      "is to induce myocyte contraction and relaxation through their responses to membrane …\n",
      "\n",
      "The Kv2 channels encode delayed rectifier currents that regulate membrane potential in \n",
      "many tissues. They also have a non-conducting function to form stable junctions between the …\n",
      "\n",
      "Kv2.1 channels mediate cell death–enabling loss of cytosolic potassium in neurons following \n",
      "plasma membrane insertion at somatodendritic clusters. Overexpression of the carboxyl …\n",
      "\n",
      "… Kv2.1 protein was detected using a mouse monoclonal antiobody (Clone K89/34) \n",
      "obtained from UC Davis/NIH Neuromab facility (1:1000, Davis, CA). …\n",
      "\n",
      "… a mouse monoclonal antiKv2.1 antibody (UC Davis/NIH NeuroMab Facility; see below). … \n",
      "[from NeuroMab clone K89/34, no cross-reactivity with Kv2.2, which was recently reported to …\n",
      "\n",
      "NMDA receptors (NMDARs) are Ca 2+ -permeant, ligand-gated ion channels activated by \n",
      "the excitatory neurotransmitter glutamate and have well-characterized roles in the nervous …\n",
      "\n",
      "… (University of California at Davis/National Institutes of Health NeuroMab Facility; see below). \n",
      "… (from NeuroMab clone K89/34, no cross-reactivity with Kv2.2, which was recently reported …\n",
      "\n",
      "… K89/34 and Kv1.2 K14/16 were obtained from UC Davis/NIH NeuroMab Facility (Western \n",
      "blotting). Kv2.1 (APC-012) was obtained from Alomone Labs (immunohistochemistry). …\n",
      "\n",
      "… following sources: mouse anti-Kv2.1 monoclonal antibody (clone K89/34) and rabbit anti-Kv2.1 \n",
      "antibody were obtained from NeuroMab (Davis, CA) and Alomone Lab (Jerusalem, Israel)…\n",
      "\n",
      "… available from the UC Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). 27 … RRID:AB_2571516 \n",
      "1:3 dilution of Tissue Culture Supernatant, concentration unknown K89/34, antiKv2.…\n",
      "\n",
      "… (mouse; 1:250, clone K89/34, NeuroMab, USA), anti m2 muscarinic receptor (rat; 1:500, \n",
      "MAB367, Merck Millipore), anti-GFAP (mouse; 1:500, G3893, Sigma-Aldrich, USA), anti-Iba1 (…\n",
      "\n",
      "… Neurons were permeabilized and blocked with 0.1% Triton X-100, 3% (w/v) nonfat dry milk \n",
      "in DPBS for 45 min at room temperature and stained with anti-Kv2.1 mouse K89/34 mAb and …\n",
      "\n",
      "… Concatemers were detected by overnight incubation of mouse Kv2.1 antibodies (K89/34, \n",
      "University of California Davis/NIH NeuroMab Facility), followed by incubation with horseradish …\n",
      "\n",
      "… (clone K89/34, #75−014) obtained from the University of California, Davis NIH NeuroMab \n",
      "Facility (RRID:AB_2877280) was used at a dilution of 1:500. This antibody was raised against …\n",
      "\n",
      "The sigma-1 receptor (Sig-1R), an endoplasmic reticulum (ER) chaperone protein, is an \n",
      "interorganelle signaling modulator that potentially plays a role in drug-seeking behaviors. …\n",
      "\n",
      "… ) using a monoclonal antibody specific for K V 2.1 subunits (NeuroMab; 75-159; clone \n",
      "K39/25, University of California, Davis). The secondary antibody was an Alexa Fluor 488-…\n",
      "\n",
      "… experiments, we used the K89/34 mouse IgG1 antibody (NeuroMab; mAb binds within aa … \n",
      ", we used the N372B/1 mouse IgG1 antibody (NeuroMab; mAb binds within aa 764 −907, cat#…\n",
      "\n",
      "In arterial myocytes, the canonical function of voltage-gated Ca V 1.2 and K V 2.1 channels \n",
      "is to induce myocyte contraction and relaxation through their responses to membrane …\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "… K89/34 IgG1: 1:20 dilution of tissue culture supernatant, Research Resource Identifier [RRID]: \n",
      "AB_10672253; anti-GFP N86/38 IgG2a: 1:5 dilution of tissue culture supernatant, RRID: …\n",
      "\n",
      "Potassium channels regulate numerous aspects of neuronal excitability, and several voltage-gated \n",
      "K + channel subunits have been identified in pyramidal neurons of rat neocortex. …\n",
      "\n",
      "Junctions between cortical endoplasmic reticulum (cER) and the plasma membrane are a \n",
      "subtle but ubiquitous feature in mammalian cells; however, very little is known about the …\n",
      "\n",
      "Oxidative modification of the voltage-gated potassium (K+) channel KCNB1 promotes apoptosis \n",
      "in the neurons of cortex and hippocampus through a signaling pathway mediated by Src …\n",
      "\n",
      "The Kv2 family of voltage-gated potassium channel α subunits, comprising Kv2.1 and Kv2.2, \n",
      "mediate the bulk of the neuronal delayed rectifier K + current in many mammalian central …\n",
      "\n",
      "Intracellular mislocalization of TAR DNA-binding protein 43 (TDP-43), a nuclear DNA/RNA-binding \n",
      "protein involved in RNA metabolism, is a pathological hallmark of amyotrophic lateral …\n",
      "\n",
      "Sub-lethal preconditioning stimuli can confer neuronal tolerance by triggering the activation \n",
      "of endogenous survival pathways that limit or resist subsequent injury. Recent evidence has …\n",
      "\n",
      "The neuregulin receptor ErbB4 is an important modulator of GABAergic interneurons and \n",
      "neural network synchronization. However, little is known about the endogenous ligands that …\n",
      "\n",
      "The axon initial segment (AIS) is the site of action potential generation and a locus of activity-dependent \n",
      "homeostatic plasticity. A multimeric complex of sodium channels, linked via a …\n",
      "\n",
      "… Acknowledgements: Most of the antibodies used in this study are available from the UC \n",
      "Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). Correspondence should be addressed to …\n",
      "\n",
      "Neurological disabilities following traumatic brain injury (TBI) may be due to excitotoxic neuronal \n",
      "loss. The excitotoxic loss of neurons following TBI occurs largely due to hyperactivation …\n",
      "\n",
      "… Cultured neocortical neurons (14–16 DIV) were fixed and stained with monoclonal \n",
      "antibodies (NeuroMAB) against Kv2.1, Cav1.3, and Cav1.2, and Alexa-conjugated secondary …\n",
      "\n",
      "By studying healthy women who do not request analgesia during their first delivery, we \n",
      "investigate genetic effects on labor pain. Such women have normal sensory and psychometric …\n",
      "\n",
      "… (clone K89/34) also called anti-KCNB1 here, was purchased from NeuroMab (University \n",
      "of California–Davis, Davis, Davis, CA; NIH, Bethesda, MD, USA). Anti-integrin-α5 (clone sc-…\n",
      "\n",
      "… (clone K89/34; NeuroMab); rabbit polyclonal against NRG1 (SC-348; Santa Cruz \n",
      "Biotechnology); mouse monoclonal against V5 tag (clone SV-Pk1; Bio-Rad); mouse monoclonal …\n",
      "\n",
      "nan\n",
      "\n",
      "The endoplasmic reticulum (ER) and plasma membrane (PM) form junctions crucial to ion \n",
      "and lipid signaling and homeostasis. The Kv2.1 ion channel is localized at ER–PM junctions …\n",
      "\n",
      "… The mouse mAbs N52A/42, K13/31, K89/34, L83/11, L86/33, and L86A/37 are available \n",
      "from the UC Davis/NIH NeuroMab Facility (http://neuromab.ucdavis.edu/), a not-for profit …\n",
      "\n",
      "Spinal muscular atrophy (SMA) is a neurodegenerative disease caused by reduced survival \n",
      "motor neuron (SMN) protein. Recently, SMN dysfunction has been linked to individual …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "… For co-immunoprecipitation experiments, 50 μl of protein G-coated magnetic beads (Life \n",
      "Technologies) were incubated (4 C, overnight with nutation) with 10 μg of anti-Kvβ2 (NeuroMab…\n",
      "\n",
      "Substantia nigra pars compacta (SNc) dopaminergic (DA) neurons display a peculiar \n",
      "electrical phenotype characterized in vitro by a spontaneous tonic regular activity (pacemaking …\n",
      "\n",
      "… The commercial antibody used was from NeuroMab, Davis, CA, USA, GABA A α1 antibody \n",
      "(N95/35, … Gabra6-a antibody and Gabrg2 antibody (Neuromab, Davis, CA, USA, N452/69, …\n",
      "\n",
      "Latrophilin-2 (Lphn2) and latrophilin-3 (Lphn3) are adhesion GPCRs that serve as postsynaptic \n",
      "recognition molecules in CA1 pyramidal neurons of the hippocampus, where they are …\n",
      "\n",
      "Although synaptogenesis within the retina is obviously essential for vision, mechanisms \n",
      "responsible for the initiation and maintenance of retinal synapses are poorly understood. In …\n",
      "\n",
      "… using the antibodies mainly from Neuromab for IP from extracts … Recently, the Neuromab \n",
      "facility started to provide a unique … use of antibodies mainly from Neuromab to start to examine …\n",
      "\n",
      "The cellular and molecular mechanisms underlying photoreceptor synaptogenesis are poorly \n",
      "understood. Furthermore, a detailed picture of the molecular composition of photoreceptor …\n",
      "\n",
      "… A mouse monoclonal antibody against Cask was purchased from NeuroMAB as cell culture \n",
      "supernatant (Clone number K56A/50; catalog number 73-000). The antibody was raised …\n",
      "\n",
      "Leukocyte common antigen-related receptor tyrosine phosphatases (LAR-RPTPs) are \n",
      "evolutionarily conserved presynaptic organizers. The synaptic role of vertebrate LAR-RPTPs in …\n",
      "\n",
      "… Immunostaining for CASK was done with a CASK antibody from Neuromab (clone \n",
      "number K56A/50). Tbr1 immunostaining was done with Anti-Tbr1 antibody from Millipore (AB2261). …\n",
      "\n",
      "BIN1, a member of the BAR adaptor protein family, is a significant late-onset Alzheimer \n",
      "disease risk factor. Here, we investigate BIN1 function in the brain using conditional knockout (…\n",
      "\n",
      "… ); mouse monoclonal anti-CASK (clone K56A/50; NeuroMab, RRID:AB_2068730), mouse \n",
      "monoclonal anti-PSD-95 (clone K28/43; NeuroMab, RRID:AB_2307331), and mouse …\n",
      "\n",
      "… monoclonal anti-PSD-95 (clone K28/43; NeuroMab); mouse monoclonal anti-β-actin (clone … \n",
      "mouse monoclonal anti-CASK (clone K56A/50; NeuroMab). The following antibodies have …\n",
      "\n",
      "LRRTMs are postsynaptic cell adhesion proteins that have region-restricted expression in the \n",
      "brain. To determine their role in the molecular organization of synapses in vivo, we studied …\n",
      "\n",
      "Microcephaly with pontine and cerebellar hypoplasia (MICPCH) syndrome is a neurodevelopmental \n",
      "disorder caused by the deficiency of the X-chromosomal gene CASK. However, the …\n",
      "\n",
      "… Davis/NIH NeuroMab Facility; K14/16), mouse anti-CASK (University of California Davis/NIH \n",
      "NeuroMab Facility; K56A/50), mouse anti-Ankyrin G (University of California Davis/NIH …\n",
      "\n",
      "Presynaptic neurexins (Nrxs) and type IIa receptor-type protein tyrosine phosphatases (RPTPs) \n",
      "organize synapses through a network of postsynaptic ligands. We show that leucine-rich-…\n",
      "\n",
      "Leukocyte common antigen-related receptor tyrosine phosphatases (LAR-RPTPs) are \n",
      "evolutionarily conserved presynaptic organizers. The synaptic role of vertebrate LAR-RPTPs in …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "Presenilin mutations that alter γ-secretase activity cause familial Alzheimer's disease (AD), \n",
      "whereas ApoE4, an apolipoprotein for cholesterol transport, predisposes to sporadic AD. …\n",
      "\n",
      "RTN4-binding proteins were widely studied as \"NoGo\" receptors, but their physiological \n",
      "interactors and roles remain elusive. Similarly, BAI adhesion-GPCRs were associated with …\n",
      "\n",
      "The cellular and molecular mechanisms underlying photoreceptor synaptogenesis are \n",
      "poorly understood. In fact, there is no clear picture of the molecular composition of a typical …\n",
      "\n",
      "Neurexins (Nrxns) and LAR-RPTPs (leukocyte common antigen-related protein tyrosine \n",
      "phosphatases) are presynaptic adhesion proteins responsible for organizing presynaptic …\n",
      "\n",
      "Myelinated neurons are organized into distinct molecular domains that are essential for \n",
      "electrical conduction. These domains include the axon initial segment, nodes of Ranvier, …\n",
      "\n",
      "Developmental and epileptic encephalopathies (DEE) are a group of severe epilepsies that \n",
      "usually present with intractable seizures, developmental delay, and often have elevated risk …\n",
      "\n",
      "… The was followed by incubation with one of the following antibodies: anti-Ca v 1.3α 1 \n",
      "NeuroMab clone N38/8 or anti-GAPDH mouse monoclonal antibody (MA5-15738, Pierce) in 2.5% …\n",
      "\n",
      "… 2+ channel (NeuroMab clone L48A/9) directed against amino acids 859–875 in the N terminus \n",
      "of rat Ca v 1.3, and 10) monoclonal anti-Ca v 1.3 Ca 2+ channel (NeuroMab clone N38/8) …\n",
      "\n",
      "… Cells were incubated with blocking buffer (3% BSA in PBS) for 1-h at room temperature \n",
      "and overnight with the Ca V 1.3 antibody, clone N38/8 (UC Davis/NIH NeuroMab Facility; 1:500 …\n",
      "\n",
      "… L57/46,) or commercially available anti-Ca v 1.3 (Alo: Alomone Labs, ACC-005; NM: Neuromab \n",
      "Antibodies Inc. N38/8) antibodies. Blots were developed using Amersham ECL Plus and …\n",
      "\n",
      "… The primary antibodies used included antibody to Ca v 1.3 (mouse monoclonal, Neuromab; \n",
      "3μg/ml) and antibody to GAPDH (mouse monoclonal; Ambion, MAB 5718; 1:25,000 dilution), …\n",
      "\n",
      "… Specificity of the Ca V 1 subtype antibodies was determined by the supplier (NeuroMab) \n",
      "by immunohistochemistry in brain tissue and by western blot. They have been used (http://…\n",
      "\n",
      "Exceptional mechanical and electrical properties of carbon nanotubes (CNT) have attracted \n",
      "neuroscientists and neural tissue engineers aiming to develop novel devices that interface …\n",
      "\n",
      "… a mouse monoclonal primary antibody (N38/8) to a conserved region of the A1D subunit of \n",
      "the L-type calcium channel, purchased from Neuromab http://www.neuromab.org. Blots were …\n",
      "\n",
      "… Anti-Ca V 1.3 antibody, obtained from NeuroMab (UC Davis/NIH), showed specific localization \n",
      "to ∼250 kD with Western blot analysis. Although we measured relatively high Ca V 3.2 …\n",
      "\n",
      "Mouse lemur ( Microcebus murinus ) is one of the smallest and more ancestral primates \n",
      "known so far. It is an emerging model of senescence and neurodegeneration, thanks to its …\n",
      "\n",
      "… , 1∶4 dilution) or mouse anti-α1D (clone N38/8, #73-080, lot 437-4VA-10, 1∶4 dilution) \n",
      "obtained from the UC Davis/NIH NeuroMab Facility – mixed into TST containing 5% w/v nonfat …\n",
      "\n",
      "… Cultured neocortical neurons (14–16 DIV) were fixed and stained with monoclonal \n",
      "antibodies (NeuroMAB) against Kv2.1, Cav1.3, and Cav1.2, and Alexa-conjugated secondary …\n",
      "\n",
      "… Furthermore, immunostaining of Ca v 1.3α 1 using our and commercial (Alomone, Millipore, \n",
      "Neuromab) antibodies was conducted under two different fixation conditions, brief 2% …\n",
      "\n",
      "Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) are severe \n",
      "neurodegenerative diseases with overlapping clinical and pathological features. Recently, a …\n",
      "\n",
      "Intracellular chloride concentration ([Cl − ] i ) in pancreatic β-cells is kept above electrochemical \n",
      "equilibrium due to the predominant functional presence of Cl − loaders such as the Na + …\n",
      "\n",
      "… The sections were incubated with mouse primary anti-GluN2A/NR2A clone N32/95 and \n",
      "anti-GluN2B/NR2B clone N59/20 (1:100; NeuroMab), antibodies in blocking solution overnight (…\n",
      "\n",
      "Proteins of the major histocompatibility complex class I (MHCI) are known for their role in \n",
      "immunity and have recently been implicated in long-term plasticity of excitatory synaptic …\n",
      "\n",
      "… , N308-48), rabbit anti-GluN2A (Millipore, A12W), mouse anti-GluN2B (NeuroMab N59/20 \n",
      "and N59/36), mouse anti-GluA2 (NeuroMab, L21/32), mouse anti-homer 1 (NeuroMab, L113/…\n",
      "\n",
      "Copper (Cu), an essential trace element present throughout the mammalian nervous system, \n",
      "is crucial for normal synaptic function. Neuronal handling of Cu is poorly understood. We …\n",
      "\n",
      "The supraoptic nucleus (SON) of the hypothalamus contains magnocellular neurosecretory \n",
      "neurons (MNC) which synthesize and release the peptide hormones vasopressin and …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "… ; NeuroMab) diluted 1:10; a mouse monoclonal supernatant NR2B glutamate receptor antibody \n",
      "(clone N59/20; NeuroMab) … receptor antibody (clone N308/48; NeuroMab) diluted 1:10; a …\n",
      "\n",
      "… ), Synapsin1 (1:400, rabbit polyclonal; Millipore), panSHANK (1:400, mouse IgG1 N23B/49; \n",
      "NeuroMab), and NR2B (1:300, mouse IgG2a N59/20; NeuroMab). Sections were washed …\n",
      "\n",
      "Brain radiotherapy is frequently used successfully to treat brain tumors. However, radiotherapy \n",
      "is often associated with declines in short-term and long-term memory, learning ability, …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… N59/20, #75-097, 1:1000, NeuroMab, Davis, CA, USA) and chicken anti-Synapsin 1–2 in \n",
      "1% donkey and 1% goat serum in PBS with 0.1% TritonX-100 were applied for 48 h at 4 C. In …\n",
      "\n",
      "… The monoclonal anti-NR2B glutamate receptor antibody (NeuroMab clone N59/20) was \n",
      "obtained from the University of California, Davis/National Institutes of Health (NIH) NeuroMab …\n",
      "\n",
      "… /NR2B glutamate receptor (clone N59/20, NeuroMab) and IRDye 800CW Goat anti-Mouse \n",
      "IgG Secondary Antibody (LI-COR). Mortalin (clone N52A/9, NeuroMab) expression was used …\n",
      "\n",
      "Background The long-term effects of cocaine on behavior are accompanied by structural \n",
      "changes in excitatory glutamatergic synapses onto the medium spiny neurons of the striatum. …\n",
      "\n",
      "Drug-evoked synaptic plasticity in the mesolimbic dopamine (DA) system reorganizes neural \n",
      "circuits that may lead to addictive behavior. The first cocaine exposure potentiates AMPAR …\n",
      "\n",
      "Postsynaptic density (PSD) fractions were isolated from rat forebrain and sonicated. Pellets \n",
      "from sonicated samples examined by electron microscopy revealed particles with an electron …\n",
      "\n",
      "The ability to attend to relevant stimuli and to adapt dynamically as demands change is a \n",
      "core aspect of cognition, and one that is impaired in several neuropsychiatric diseases, …\n",
      "\n",
      "Estradiol treatment of ovariectomized rodents is known to affect the morphology of dendritic \n",
      "spines and produce behavioral and cognitive effects. Kalirin-7 (Kal7), a postsynaptic density (…\n",
      "\n",
      "… N59/20; Neuromab), anti-GABA-B1 antibody (1:500; rabbit monoclonal antibody raised \n",
      "against the GABA-B1 CTD; generous gift from Julie Kniazeff, Montpellier, France), anti-αTubuline …\n",
      "\n",
      "Estradiol enhances the formation of dendritic spines and excitatory synapses in hippocampal \n",
      "neurons in vitro and in vivo, but the underlying mechanisms are not fully understood. Kalirin…\n",
      "\n",
      "Microwave (MW) radiation induced oxidative stress reduces dendritic arborization, spine \n",
      "density and number of hippocampal pyramidal neurons and hence, impair learning and spatial …\n",
      "\n",
      "Reciprocal interactions between neurons and oligodendrocytes are not only crucial for \n",
      "myelination, but also for long-term survival of axons. Degeneration of axons occurs in several …\n",
      "\n",
      "Rho GTPases activated by GDP/GTP exchange factors (GEFs) play key roles in the \n",
      "developing and adult nervous system. Kalirin-7 (Kal7), the predominant adult splice form of the …\n",
      "\n",
      "Huntington's disease is caused by an expanded polyglutamine repeat in the huntingtin \n",
      "protein (HTT), but the pathophysiological sequence of events that trigger synaptic failure and …\n",
      "\n",
      "MicroRNAs (miRNAs) are small RNAs that modulate gene expression by binding target \n",
      "mRNAs. The hundreds of miRNAs expressed in the brain are critical for synaptic development …\n",
      "\n",
      "The proper function of neural circuits requires spatially and temporally balanced development \n",
      "of excitatory and inhibitory synapses. However, the molecular mechanisms coordinating …\n",
      "\n",
      "Hyperactivating mutations in the non-receptor tyrosine phosphatase SHP2 cause Noonan \n",
      "syndrome (NS). NS is associated with cognitive deficits, but how hyperactivation of SHP2 in …\n",
      "\n",
      "Autism spectrum disorder (ASD) is a heritable, common neurodevelopmental disorder with \n",
      "diverse genetic causes. Several studies have implicated protein synthesis as one among …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "… selected as being as target-specific, driving the eventual selection of certain of these as \n",
      "NeuroMabs (labeled with ‘N's above the lanes): N419/40 (Fig. 2a) and N431/64 (Fig. 2b). …\n",
      "\n",
      "The axon initial segment (AIS) is a specialized region in neurons that encompasses two \n",
      "essential functions, the generation of action potentials and the regulation of the axodendritic …\n",
      "\n",
      "… After 3 washes with PBS 0.05% Triton X-100, cells were incubated with PanNav antibody \n",
      "(Neuromab, catalog #N419/40, RRID:AB_2750795; dilution 1:1000) overnight at 4C. Then, …\n",
      "\n",
      "Amyloid-β (Aβ) plaques are one of the central pathologies of Alzheimer's disease (AD). \n",
      "Plaque formation in animal models of AD coincides with the appearance of synaptic …\n",
      "\n",
      "In the hypothalamus, insulin takes on many roles involved in energy homoeostasis. Therefore, \n",
      "the aim of this study was to examine hypothalamic insulin expression during the initial …\n",
      "\n",
      "… anti-NEUN, clone A60 (1:500, Neuromab); mouse monoclonal anti-glial fibrillary acidic \n",
      "protein (GFAP), clone N206A/8 (1:300, Neuromab, Davis, CA); and goat polyclonal anti-IBA (1:…\n",
      "\n",
      "Neonatal hypoxic injury (NHI) is a devastating cause of disease that affects >60% of babies \n",
      "born with a very low birth weight, resulting in significant morbidity and mortality, including life-…\n",
      "\n",
      "Human brain organoids differentiated from embryonic stem cells offer the unique opportunity \n",
      "to study complicated interactions of multiple cell types in a three-dimensional system. Here …\n",
      "\n",
      "This study examined possible mechanisms by which Substance P (Sub P) assumes a \n",
      "pronociceptive role in the rostral ventromedial medulla (RVM) under conditions of peripheral …\n",
      "\n",
      "… and proximal dendrites of neurons, and anti-GFAP mAb N206A/8 (IgG1, green) and anti-pan-… \n",
      "We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and …\n",
      "\n",
      "Interleukin-6 (IL-6) is an important neuroimmune factor that is increased in the brain by alcohol \n",
      "exposure/withdrawal and is thought to play a role in the actions of alcohol on the brain. …\n",
      "\n",
      "Corticothalamic interactions between associative cortices and higher order thalamic nuclei \n",
      "are involved in high‐cognitive functions such as decision‐making and working memory. …\n",
      "\n",
      "Dynamin is a large GTPase crucial for endocytosis and sustained neurotransmission, but its \n",
      "role in synapse development in the mammalian brain has received little attention. We …\n",
      "\n",
      "… Slides were then incubated overnight in mouse anti-GFAP (N206A/8; NeuroMab) and chicken \n",
      "anti-peripherin (CPCA-Peri; Encor Biotechnology) at 1:1000 in blocking buffer overnight at …\n",
      "\n",
      "The development of therapies for retinal disorders is hampered by a lack of appropriate animal \n",
      "models. Higher nonhuman primates are the only animals with retinal structure similar to …\n",
      "\n",
      "Pathophysiological mechanisms underlying pain associated with cancer are poorly \n",
      "understood. microRNAs (miRNAs) are a class of noncoding RNAs with emerging functional …\n",
      "\n",
      "… (NeuroMab N206A/8) at 1:1000 dilution in TBST for 2 h, washed three times with TBST, \n",
      "reacted with Goat anti-mouse IgG Alexa 488 conjugated antibody (Invitrogen) at 1:2000 dilution …\n",
      "\n",
      "Chemotherapy-induced peripheral neuropathy (CIPN) is a primary dose-limiting side effect \n",
      "caused by antineoplastic agents, such as paclitaxel. This causes damage to peripheral …\n",
      "\n",
      "… anti-Caspr, clone K65/35, NeuroMab, 1:2000 dilution; rabbit polyclonal antimyelin … clone \n",
      "N206A/8, NeuroMab, 1:2000 dilution; mouse monoclonal anti-PSD95, clone K28/43, NeuroMab, …\n",
      "\n",
      "Simultaneous labeling of multiple targets in a single sample, or multiplexing, is a powerful \n",
      "approach to directly compare the amount, localization and/or molecular properties of different …\n",
      "\n",
      "Alzheimer’s disease (AD) is characterized by beta-amyloid accumulation, phosphorylated \n",
      "tau formation, hyperactivation of glial cells, and neuronal loss. The mechanisms of AD …\n",
      "\n",
      "… ), glial fibrillary acidic protein (GFAP, N206A/8), and Nav1.6 (K87/10A) antibodies were \n",
      "purchased from the UC Davis/NIH funded Neuromab monoclonal antibody resource. Mouse pan-…\n",
      "\n",
      "Background Fragile XS yndrome, the most common single gene cause of autism, results from \n",
      "loss of the RNA ‐binding protein FMRP . Although FMRP is highly expressed in neurons, it …\n",
      "\n",
      "SLC7A10 (Asc-1) is a sodium-independent amino acid transporter known to facilitate transport \n",
      "of a number of amino acids including glycine, L-serine, L-alanine, and L-cysteine, as well …\n",
      "\n",
      "Long-lasting forms of synaptic plasticity such as synaptic scaling are critically dependent on \n",
      "transcription. Activity-dependent transcriptional dynamics in neurons, however, remain …\n",
      "\n",
      "The mechanisms by which prenatal immune activation increase the risk for neuropsychiatric \n",
      "disorders are unclear. Here, we generated developmental cortical interneurons (cINs)—…\n",
      "\n",
      "Oligodendrocytes associate with axons to establish myelin and provide metabolic support to \n",
      "neurons. In the spinal cord of amyotrophic lateral sclerosis (ALS) mice, oligodendrocytes …\n",
      "\n",
      "Parvalbumin (PV)-expressing basket interneurons in the prefrontal cortex (PFC) regulate \n",
      "pyramidal cell firing, synchrony, and network oscillations. Yet, it is unclear how their perisomatic …\n",
      "\n",
      "The region surrounding the central canal (CC) of the spinal cord is a highly plastic area, \n",
      "defined as a postnatal neurogenic niche. Within this region are ependymal cells that can …\n",
      "\n",
      "Autism spectrum disorder (ASD) is a neurodevelopmental disorder characterized by \n",
      "impairments in social interaction and communication as well as ritualistic repetitive behaviors. …\n",
      "\n",
      "10.7554/eLife.11324.001 During the development, tight regulation of the expansion of neural \n",
      "progenitor cells (NPCs) and their differentiation into neurons is crucial for normal cortical …\n",
      "\n",
      "Chronic pain, broadly classified as inflammatory (caused by tissue damage) or neuropathic (caused \n",
      "by nerve damage), affects approximately 100 million Americans and only a minority …\n",
      "\n",
      "… incubated in NeuroMab monoclonal antibodies to Kv2.1 (K89/34; 10 μg/mL; NeuroMab #73-… \n",
      ".1) and N372B/1 or N372B/60 (for Kv2.2) antibodies (both from NeuroMab) in 4 Thy1 mice (…\n",
      "\n",
      "Voltage-gated K + (Kv) channels play important roles in regulating neuronal excitability. Kv \n",
      "channels comprise four principal α subunits, and transmembrane and/or cytoplasmic …\n",
      "\n",
      "… For Ab-induced blockade of Kv2.2, we used mAb N372B/60 at 10 μg/ml or ∼100 nm, \n",
      "again with 1% BSA. As a control, both Abs were heated to 65C for 3 h to inactivate them. …\n",
      "\n",
      "… available from the UC Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). 27 … \n",
      "NeuroMab Cat# 73-014 RRID:AB_10672253 … N372B/60, anti-Kv2.2 mouse IgG2b mAb …\n",
      "\n",
      "… ) obtained from the UC Davis/NIH NeuroMab Facility. Alexa-conjugated fluorescent … (N372B/60)]. \n",
      "Sections were exposed to Alexa-conjugated fluorescent secondary antibodies …\n",
      "\n",
      "Among all voltage-gated potassium (Kv) channels, Kv2 channels are the most widely \n",
      "expressed in the mammalian brain. However, studying Kv2 in neurons has been challenging …\n",
      "\n",
      "… Acknowledgements: Most of the antibodies used in this study are available from the UC \n",
      "Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). Correspondence should be addressed to …\n",
      "\n",
      "The endoplasmic reticulum (ER) and plasma membrane (PM) form junctions crucial to ion \n",
      "and lipid signaling and homeostasis. The Kv2.1 ion channel is localized at ER–PM junctions …\n",
      "\n",
      "… Anti-Kv1.2 antibody was obtained from NeuroMab. Kv1.2 has a calculated MW of 56.7 kDa \n",
      "but may run at 66–88 kDa depending on the degree of posttranslational modification. On …\n",
      "\n",
      "Numerous studies have documented the effects of chronic N-methyl-D-aspartate receptor (NMDAR) \n",
      "blockade on excitatory circuits, but the effects on inhibitory circuitry are not well …\n",
      "\n",
      "10.7554/eLife.12661.001 Neuropathic pain following peripheral nerve injury is associated \n",
      "with hyperexcitability in damaged myelinated sensory axons, which begins to normalise over …\n",
      "\n",
      "Neuropathic pain following peripheral nerve injury is associated with hyperexcitability in \n",
      "damaged myelinated sensory axons, which begins to normalise over time. We investigated the …\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "Neuropathic pain (NP) embraces a broad range of conditions linked with a disease or \n",
      "lesion of the peripheral or central somatosensory system and its prevalence in the general …\n",
      "\n",
      "Neuropathic pain (NP) embraces a broad range of conditions linked with a disease or \n",
      "lesion of the peripheral or central somatosensory system and its prevalence in the general …\n",
      "\n",
      "… Additional commercial available mouse monoclonal antibodies from NeuroMab (Anti-Kv1.1: … \n",
      "K19/36) were used. Whereas monoclonal antibodies (mAntiKv1.x) mAnti-Kv1.1, mAnti-Kv1.6 …\n",
      "\n",
      "Autosomal dominant lateral temporal epilepsy (ADLTE) is an inherited syndrome caused by \n",
      "mutations in the leucine-rich glioma inactivated 1 (LGI1) gene. It is known that glutamatergic …\n",
      "\n",
      "… The primary cortical neurons were then incubated with the primary GluN1 antibodies (C-terminal \n",
      "rabbit antibody, Millipore, AB9864 and N-terminal mouse antibody, NeuroMab, N308/48…\n",
      "\n",
      "… primary antibodies anti-GluN1 1:1000 (NeuroMab clone N308/48), anti-GluN2A 1:500 (NeuroMab \n",
      "clone N327/95) or anti-GluN2B 1:1000 (NeuroMab clone N59/36), and then washed …\n",
      "\n",
      "… Supernatant fractions were incubated with anti-HA (Santa Cruz Biotechnology, sc-805) or \n",
      "anti-NR1 (NeuroMab clone N308/48) for overnight at 4 C, followed by incubation with 50 μl of …\n",
      "\n",
      "The efficient knock-in of large DNA fragments to label endogenous proteins remains especially \n",
      "challenging in non-dividing cells such as neurons. We developed Targeted Knock-In with …\n",
      "\n",
      "NMDARs, the Ca 2+ permeable channels, play central roles in synaptic plasticity, brain \n",
      "development, learning, and memory. NMDAR binding partners and associated signaling has …\n",
      "\n",
      "Excessive stimulation of NMDA receptors with glutamate or other potent agonists such as \n",
      "NMDA leads to excitotoxicity and neural injury. In this study, we aimed to provide insight into …\n",
      "\n",
      "Purpose Intrinsically photosensitive retinal ganglion cells (ipRGCs) express the photopigment \n",
      "melanopsin (OPN4) and are mainly responsible for non-image-forming visual tasks such …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "Behaviors that rely on the hippocampus are particularly susceptible to chronological aging, \n",
      "with many aged animals (including humans) maintaining cognition at a young adult-like level…\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "Mutations affecting the levels of microRNA miR-137 are associated with intellectual disability \n",
      "and schizophrenia. However, the pathophysiological role of miR-137 remains poorly …\n",
      "\n",
      "… /20; NeuroMab) diluted 1:10; a purified mouse monoclonal NR1 glutamate receptor antibody \n",
      "(clone N308/48; NeuroMab) … /GluR2 antibody (clone L21/32; NeuroMab) diluted 1:10; and a …\n",
      "\n",
      "Membrane-associated guanylate kinases (MAGUKs), including SAP102, PSD-95, PSD-93, \n",
      "and SAP97, are scaffolding proteins for ionotropic glutamate receptors at excitatory synapses…\n",
      "\n",
      "Proteins derived from the Kalrn gene, encoding 2 Rho guanine nucleotide exchange factor (GEF) \n",
      "domains, affect dendritic and axonal morphogenesis. The roles of endogenous Kalirin-…\n",
      "\n",
      "10.7554/eLife.04287.001 Lrp4, the muscle receptor for neuronal Agrin, is expressed in the \n",
      "hippocampus and areas involved in cognition. The function of Lrp4 in the brain, however, is …\n",
      "\n",
      "Microwave (MW) radiation induced oxidative stress reduces dendritic arborization, spine \n",
      "density and number of hippocampal pyramidal neurons and hence, impair learning and spatial …\n",
      "\n",
      "Although loss-of-function mutations in the PARK2 gene, the gene that encodes the protein \n",
      "parkin, cause autosomal recessive juvenile parkinsonism, the responsible molecular …\n",
      "\n",
      "… The anti-GluN1 (clone N308/48) and anti-GluA1 (N355/1) monoclonal antibodies were \n",
      "obtained from the UC Davis/NIH NeuroMab Facility. …\n",
      "\n",
      "… antibody (#N312/10, NeuroMab) was similarly diluted in Duolink Antibody diluent at the \n",
      "concentration 1:1000, and NUP98 (N-term) antibody (#M1-26400, antibodies-online.com) at the …\n",
      "\n",
      "Human papillomavirus (HPV)-negative head and neck squamous cell carcinomas (HNSCCs) \n",
      "are deadly and common cancers. Recent genomic studies implicate multiple genetic …\n",
      "\n",
      "Histone H3K27M is a driving mutation in diffuse intrinsic pontine glioma (DIPG), a deadly \n",
      "pediatric brain tumor. H3K27M reshapes the epigenome through a global inhibition of PRC2 …\n",
      "\n",
      "Chromatin dysregulation has emerged as an important mechanism of oncogenesis. To \n",
      "develop targeted treatments, it is important to understand the transcriptomic consequences of …\n",
      "\n",
      "The discovery of H3K27M mutations in pediatric gliomas marked a new chapter in cancer \n",
      "epigenomics. Numerous studies have investigated the effect of this mutation on H3K27 …\n",
      "\n",
      "How cancer-associated chromatin abnormalities shape tumor-immune interaction remains \n",
      "incompletely understood. Recent studies have linked DNA hypomethylation and de-…\n",
      "\n",
      "… sections (ie, specific labeling of nodes of Ranvier and the AIS) as clone N106/36. N106/36 \n",
      "yields a single band of the appropriate size on immunoblots of rat and mouse brain samples, …\n",
      "\n",
      "… F/F ) spinal cords (AnkG, N106/36). Two mice per genotype and more than … (N106/36 and \n",
      "N106/65, NeuroMab) (1:200), Caspr (K65/35, NeuroMab) (1:200), contactin (K73/20, NeuroMab…\n",
      "\n",
      "… Coverslips were then placed in primary antibody solution, in PBS plus 2 % GS, at relevant \n",
      "concentrations: mouse monoclonal anti-ankyrin-G (Neuromab clone N106/36; 1 : 500) and …\n",
      "\n",
      "The node of Ranvier is a functionally important site on the myelinated axon where sodium \n",
      "channels are clustered and regeneration of action potentials occurs, allowing fast saltatory …\n",
      "\n",
      "… Mouse monoclonal anti-AnkG (N106/36; RRID: AB_10697718) and anti-GST (N100/13; \n",
      "RRID:AB_10671817) antibodies were purchased from UC Davis and the NIH NeuroMab facility. …\n",
      "\n",
      "This protocol allows repeated whole-cell patch-clamp recordings from individual rodent CA1 \n",
      "hippocampal neurons, followed by immunohistological labeling of the axon initial segment. …\n",
      "\n",
      "… We used the following primary antibodies: mouse monoclonal anti-ankyrin G (clone N106/36; \n",
      "1:500; NeuroMab), chicken polyclonal anti-βIV spectrin (1:500; a gift from M. Komada), …\n",
      "\n",
      "… We have long encouraged our end users to provide the specific NeuroMab clone number \n",
      "(eg, N106/36, N106/65, etc.) as an unambiguous transparent identifier in all publications, a …\n",
      "\n",
      "Structural plasticity of the axon initial segment (AIS), the site of action potential initiation, is \n",
      "observed as part of the normal early development of the cortex, as well as in association with …\n",
      "\n",
      "… ankyrin-G (AnkG; NeuroMab, Davis, CA; N106/36, 1:200) and the urn:x-wiley:08941491:media:glia22991:glia22991-math-0003 \n",
      "channel isoform 1.6 (Na v 1.6; NeuroMab; K87A/10, 1:…\n",
      "\n",
      "… Primary antibodies used were rabbit anti-Na v 1.2, anti-Na v 1.6, anti-Na v 1.3, and anti-Na \n",
      "v 1.1 (1:250 Alomone Laboratories) and ankyrin G (1:250 NeuroMab N106/36). Secondary …\n",
      "\n",
      "… Teased nerve fibers were immunostained for anti-Caspr antibody (rabbit polyclonal; 1:200; \n",
      "Abcam) and anti-Ankyrin-G antibody (mouse monoclonal; 1:200; clone N106/36; NeuroMab). …\n",
      "\n",
      "… The primary antibodies used were: chicken anti-MAP2 (1:10,000, Abcam, ab5392), mouse \n",
      "anti-ankyrinG (1:100) from NeuroMab (clone N106/36), mouse anti-pan sodium channel (1:…\n",
      "\n",
      "… Primary antibodies against GFP (host: chicken, 1:5000, Abcam, ab13970) and AnkG (mouse \n",
      "monoclonal, 1:500, Neuromab, clone N106/36) were incubated in blocking solution at 4 C …\n",
      "\n",
      "The precise and remarkable subdivision of myelinated axons into molecularly and functionally \n",
      "distinct membrane domains depends on axoglial junctions that function as barriers. …\n",
      "\n",
      "Heterozygous missense variants in KCNQ2, which encodes the potassium channel subunit \n",
      "Kv7.2, are among the most common genetic causes of severe neonatal-onset epileptic …\n",
      "\n",
      "Axonal ion channels control spike initiation and propagation along the axon and determine \n",
      "action potential waveform. We show here that functional suppression of axonal Kv1 channels …\n",
      "\n",
      "Axons and dendrites are long extensions of neurons that contain arrays of noncentrosomal \n",
      "microtubules. Calmodulin-regulated spectrin-associated proteins (CAMSAPs) bind to and …\n",
      "\n",
      "… N106/36 Neuromab, Davis, USA) and rabbit anti-synaptopodin (1 : 250; Sigma, St Louis, \n",
      "Missouri, USA), to visualize the AIS and the cisternal organelle, respectively; rabbit anti-GAT-1 (…\n",
      "\n",
      "The axon initial segment (AIS) cytoskeleton undergoes rapid and irreversible disruption prior \n",
      "to cell death after injury, and loss of AIS integrity can produce profound neurological effects …\n",
      "\n",
      "Introduction Peripheral nerves are often exposed to mechanical stress leading to compression \n",
      "neuropathies. The pathophysiology underlying nerve dysfunction by chronic compression …\n",
      "\n",
      "Mammalian hibernation is a natural process in which the brain undergoes profound \n",
      "adaptive changes that appear to protect the brain from extreme hypoxia and hypothermia. In …\n",
      "\n",
      "… Antibodies for Immunostaining The primary antibody that was used is mouse anti-ankyrin-G \n",
      "(NeuroMab, clone N106/36). Alexa Fluor 594-conjugated goat anti-mouse secondary anti…\n",
      "\n",
      "… The following other primary antibodies were used: mouse monoclonal anti-ankG for \n",
      "immunostaining (N106/36, UC Davis/NIH NeuroMab facility, Davis, CA, USA); rabbit anti-GAPDH (…\n",
      "\n",
      "The axon initial segment (AIS) is a highly specialized neuronal compartment that plays a key \n",
      "role in neuronal development and excitability. It concentrates multiple membrane proteins …\n",
      "\n",
      "RNA binding proteins required for the maintenance of myelin and axoglial junctions are \n",
      "unknown. Herein, we report that deletion of the Quaking (QKI) RNA binding proteins in …\n",
      "\n",
      "… (1:100, Neuromab K36/15) and mouse IgG2a anti-ankyrinG (1:100, Neuromab N106/36). \n",
      "After extensive washing in incubation buffer, the secondary antibodies, Alexa 568 goat anti-…\n",
      "\n",
      "… The monoclonal ankG, (N106/36 and N106/20), glial fibrillary acidic protein (GFAP, \n",
      "N206A/8), and Nav1.6 (K87/10A) antibodies were purchased from the UC Davis/NIH funded …\n",
      "\n",
      "CRISPR/Cas-based technologies have revolutionized genetic approaches to addressing a \n",
      "wide range of neurobiological questions. The ability of CRISPR/Cas to introduce mutations …\n",
      "\n",
      "… I) Representative images of ANKG and TRIM46 expression in SH-SY5Y-derived neurons \n",
      "(A–C) and mouse primary neurons (G–I), immunostained with monoclonal anti-ANKG (N106/36…\n",
      "\n",
      "Neuropathic pain caused by nerve injury presents with severe spontaneous pain and a \n",
      "variety of comorbidities, including deficits in higher executive functions. None of these clinical …\n",
      "\n",
      "The axon initial segment (AIS) is critical for the initiation and propagation of action potentials. \n",
      "Assembly of the AIS requires interactions between scaffolding molecules and voltage-…\n",
      "\n",
      "Extracellular voltage fields produced by a neuron’s action potentials provide a primary means \n",
      "for studying neuron function, yet their biophysical sources remain ambiguous. The neuron’…\n",
      "\n",
      "All-optical electrophysiology—spatially resolved simultaneous optical perturbation and \n",
      "measurement of membrane voltage—would open new vistas in neuroscience research. We …\n",
      "\n",
      "… ; NeuroMab, N106/36) in PBS with 5% bovine albumin serum and 0.3% Triton X-100 overnight \n",
      "at 4C. Subsequently, after slices were rinsed with 0.1M PBS 3 times each for 5 minutes, …\n",
      "\n",
      "Angelman syndrome (AS) is a neurodevelopmental disorder caused by the loss of function \n",
      "of the maternal UBE3A gene. The hippocampus is one of the most prominently affected brain …\n",
      "\n",
      "… (1∶1000, NeuroMab, clone N56/21), mouse monoclonal anti-AnkyrinG (1∶1000, NeuroMab, \n",
      "clone N106/36), and mouse monoclonal anti-GFP (1∶1000, NeuroMab, clone N86/38). …\n",
      "\n",
      "The most prominent structural hallmark of the mammalian neocortical circuitry is the layer-based \n",
      "organization of specific cell types and synaptic inputs. Accordingly, cortical inhibitory …\n",
      "\n",
      "TDP-43 is extensively studied in neurons in physiological and pathological contexts. \n",
      "However, emerging evidence indicates that glial cells are also reliant on TDP-43 function. We …\n",
      "\n",
      "The shape of the presynaptic action potential (AP) has a strong impact on neurotransmitter \n",
      "release. Because of the small size of most terminals in the central nervous system, little is …\n",
      "\n",
      "… Axon initial segments were visualized using mouse monoclonal antibodies directed \n",
      "against AnkG (NeuroMab, Davis, CA; N106/36, 1:200) or βIV spectrin (a generous gift from Dr. …\n",
      "\n",
      "Axon formation, the initial step in establishing neuronal polarity, critically depends on local \n",
      "microtubule reorganization and is characterized by the formation of parallel microtubule …\n",
      "\n",
      "… N18/28, University of California Davis/NIH NeuroMab Facility), mouse anti-Ankyrin G (clone \n",
      "N106/36, University of California Davis/NIH NeuroMab Facility), guinea pig anti-VGlut2 (…\n",
      "\n",
      "The axon initial segment (AIS) is a critical locus of control of action potential (AP) generation \n",
      "and neuronal information synthesis. Concussive traumatic brain injury gives rise to diffuse …\n",
      "\n",
      "Alzheimer’s disease (AD) is a devastating neurodegenerative disorder characterized by \n",
      "pathological brain lesions and a decline in cognitive function. β-Amyloid peptides (Aβ), derived …\n",
      "\n",
      "… Mouse monoclonal antibody against ankyrin G (clones N106/36 and 106/65) was purchased \n",
      "from NeuroMAb. Rabbit antibody against NrCAM (ab24344) was purchased from Abcam. A …\n",
      "\n",
      "Core planar cell polarity (PCP) genes, which are involved in various neurodevelopmental \n",
      "disorders such as neural tube closure, epilepsy, and autism spectrum disorder, have poorly …\n",
      "\n",
      "Identification of early adaptive and maladaptive neuronal stress responses is an important \n",
      "step in developing targeted neuroprotective therapies for degenerative disease. In glaucoma, …\n",
      "\n",
      "… The primary antibodies used were: mouse IgG2b anti-K v 1.1 (1:100, Neuromab K36/15) \n",
      "and mouse IgG2a anti-ankyrinG (1:100, Neuromab N106/36). After extensive washing in …\n",
      "\n",
      "Establishment of neuronal polarity depends on local microtubule (MT) reorganization. The \n",
      "endoplasmic reticulum (ER) consists of cisternae and tubules and, like MTs, forms an …\n",
      "\n",
      "… Mouse monoclonal antibodies against AnkG (1:1000, N106/36 NeuroMab), rabbit polyclonal \n",
      "antibodies against Neurofascin (1:500, Abcam ab31457), rabbit polyclonal antibodies …\n",
      "\n",
      "… Coverslips were then placed in primary antibody, made in 2% GS, to relevant concentrations: \n",
      "mouse monoclonal anti-ankyrin-G (clone N106/36, 1:500; catalog #75-146; NeuroMab); …\n",
      "\n",
      "… Coverslips were then placed in primary antibody, made in 2% GS, to relevant concentrations: \n",
      "mouse monoclonal anti-ankryin-G (clone N106/36; 1:500; NeuroMab); rabbit polyclonal anti…\n",
      "\n",
      "… mouse monoclonal anti-ankyrin G antibody (clone N106/36) and Caspr antibody (clone \n",
      "K65/35) were from University of California at Davis/NIH NeuroMab Facility (Davis, CA). Rabbit …\n",
      "\n",
      "Brain enriched voltage-gated sodium channel (VGSC) Na v 1.2 and Na v 1.6 are critical for \n",
      "electrical signaling in the CNS. Previous studies have extensively characterized cell-type-…\n",
      "\n",
      "ANK3, encoding the adaptor protein Ankyrin-G (AnkG), has been implicated in bipolar \n",
      "disorder by genome-wide association studies. ANK3 has multiple alternative first exons, and a …\n",
      "\n",
      "10.7554/eLife.43322.001 Generating recombinant monoclonal antibodies (R-mAbs) from \n",
      "mAb-producing hybridomas offers numerous advantages that increase the effectiveness, …\n",
      "\n",
      "Simultaneous labeling of multiple targets in a single sample, or multiplexing, is a powerful \n",
      "approach to directly compare the amount, localization and/or molecular properties of different …\n",
      "\n",
      "Previously, we showed that a hierarchy of spectrin cytoskeletal proteins maintains nodal Na \n",
      "+ channels (Liu et al., 2020). Here, using mice lacking β1, β4, or β1/β4 spectrins, we show …\n",
      "\n",
      "The spatial and temporal balance of spinal α‐motoneuron (αMN) intrinsic membrane \n",
      "conductances underlies the neural output of the final common pathway for motor commands. …\n",
      "\n",
      "The axon initial segment ( AIS ) is a specialized domain essential for neuronal function, the \n",
      "formation of which begins with localization of an ankyrin‐G (AnkG) scaffold. However, the …\n",
      "\n",
      "Diverse types of cortical interneurons (INs) mediate various kinds of inhibitory control \n",
      "mechanisms to balance and shape network activity. Distinct IN subtypes develop uniquely …\n",
      "\n",
      "… The following primary antibodies were used: mouse monoclonal anti-iFGF14 (1:1000; \n",
      "NeuroMab), mouse monoclonal anti-Ankyrin G clone N106/36 (1:1000; NeuroMab), and mouse …\n",
      "\n",
      "P2X 7 receptors (P2X 7 Rs) are associated with numerous pathophysiological mechanisms, \n",
      "and this promotes them as therapeutic targets for certain neurodegenerative conditions. …\n",
      "\n",
      "… Other mouse monoclonal antibodies used were sourced as follows: AnkG (N106/36, UC \n",
      "Davis/NIH NeuroMab facility, 1:250), AnkB (N105/17 and N105/13, UC Davis/NIH NeuroMab …\n",
      "\n",
      "… were used: mouse monoclonal ankyrinG (N106/36, UC Davis/NIH NeuroMab Facility Cat# \n",
      "75-146 RRID:AB_10673030), Kv1.2 (K14/16, UC Davis/NIH NeuroMab Facility Cat# 75-008 …\n",
      "\n",
      "… Chicken anti-GFP was from Abcam (ab13970); mouse anti-Ankyrin-G was from NeuroMab \n",
      "(clone N106/36); rabbit anti-MAP2 was from Synaptic Systems (188 003); mouse anti-CD4 …\n",
      "\n",
      "… The following primary antibodies were used: mouse ankyrin-G (Neuromab; N106/36), \n",
      "rabbit ankyrin-G (SantaCruz; sc-28561), rabbit Homer1b/c (Antibodies-online; ABIN-1742339), …\n",
      "\n",
      "Genetically defined subgroups of inhibitory interneurons are thought to play distinct roles in \n",
      "learning, but heterogeneity within these subgroups has limited our understanding of the …\n",
      "\n",
      "… Mouse mAbs included anti-AnkG (clone N106/36; 1:50) and Nfasc (pan, external; cloneA12/18; \n",
      "1:100) from Antibodies Incorporated (Neuromab). Anti-Nav (pan; clone K58/35; 1:300) …\n",
      "\n",
      "The epithelial sodium channel (ENaC) is the limiting entry point for Na + reabsorption in the \n",
      "distal kidney nephron and is regulated by numerous hormones, including the …\n",
      "\n",
      "The axon initial segment (AIS) is the site of action potential (AP) initiation, thus a crucial \n",
      "regulator of neuronal activity. In excitatory pyramidal neurons, the high density of voltage-gated …\n",
      "\n",
      "… clone N106/20 (western blot) and clone N106/36 (immunocytochemistry) from Neuromab \n",
      "(Davis, CA, United States), PSD-95 (Neuromab), turboGFP mAb (to identify RNAi expression, …\n",
      "\n",
      "… Chicken anti-GFP was from Abcam (ab13970); mouse anti-Ankyrin-G was from NeuroMab \n",
      "(clone N106/36); rabbit anti-MAP2 was from Synaptic Systems (188 003); mouse anti-CD4 …\n",
      "\n",
      "… Cells were then blocked with 3% bovine serum albumin and 0.1% Triton X-100 in PBS \n",
      "and incubated with mouse anti-AnkG (N106/36, NeuroMab), rat anti-EB3 (KT36, Abcam), and …\n",
      "\n",
      "… Monoclonal antibodies for ankyrin-G (clone N106/36) and K ir 2.1 (clone N112B/14) were \n",
      "obtained from the University of California Davis/National Institutes of Health NeuroMab Facility. …\n",
      "\n",
      "We recently described new pathogenic variants in VRK1, in patients affected with distal \n",
      "Hereditary Motor Neuropathy associated with upper motor neurons signs. Specifically, we …\n",
      "\n",
      "The axon initial segment (AIS) has characteristically dense clustering of voltage-gated sodium \n",
      "channels (Nav), cell adhesion molecule Neurofascin 186 (Nfasc), and neuronal scaffold …\n",
      "\n",
      "… Ankyrin G (AnkG; 1:3000; Clone N106/36 catalog #75–187) were all obtained from the \n",
      "University of California, Davis (UC Davis)/NIH NeuroMab Facility. A goat anti-Ankyrin G antibody (…\n",
      "\n",
      "Neuronal GABAergic responses switch from excitatory to inhibitory at an early postnatal \n",
      "period in rodents. The timing of this switch is controlled by intracellular Cl− concentrations, but …\n",
      "\n",
      "The tetrodotoxin‐resistant ( TTX ‐R) voltage‐gated sodium channel Na v 1.8 is predominantly \n",
      "expressed in peripheral afferent neurons, but in case of neuronal injury an ectopic and …\n",
      "\n",
      "… Retinas were then incubated with a solution of 0.1 M PBS containing the mouse monoclonal \n",
      "anti-ankyrin-G (clone N106/36; 1:250; NeuroMab; ID: AB_2749806) and 1% normal donkey …\n",
      "\n",
      "… (1:200, Neuromab #75-105, clone K36/15, external), and mouse anti-ankyrin G (1:200, \n",
      "Neuromab #75-146, clone N106/36). The following secondary antibodies were used for …\n",
      "\n",
      "The lack of therapies for progressive multiple sclerosis highlights the need to understand the \n",
      "regenerative process of remyelination that can follow CNS demyelination. This involves an …\n",
      "\n",
      "α-Synuclein plays a key role in the pathogenesis of Parkinson’s disease and related disorders, \n",
      "but critical interacting partners and molecular mechanisms mediating neurotoxicity are …\n",
      "\n",
      "… Anti-AnkG (75-146, RRID:AB_10673030) mouse monoclonal antibody N106/36 was obtained \n",
      "from the UC Davis/NIH NeuroMab Facility (United States), and rabbit polyclonal anti-FLAG …\n",
      "\n",
      "Microglia, the resident immune cells of the central nervous system, are key players in healthy \n",
      "brain homeostasis and plasticity. In neurological diseases, such as Multiple Sclerosis, …\n",
      "\n",
      "… Teased nerve fibers were immunostained for anti-Caspr antibody (rabbit polyclonal; 1:200; \n",
      "Abcam) and anti-Ankyrin-G antibody (mouse monoclonal; 1:200; clone N106/36; NeuroMab). …\n",
      "\n",
      "Alternative splicing (AS) is an important source of proteome diversity in eukaryotes. However, \n",
      "how this affects protein repertoires at a single-cell level remains an open question. Here, …\n",
      "\n",
      "… 1 h with these mAbs and mouse anti-Ank-G mAb (N106/36 or N106/65), or mouse anti-PSD-93 \n",
      "(N18/30) all from the UC Davis/NIH NeuroMab Facility; rat anti-EB1 mAb (KT-51; AbCam)…\n",
      "\n",
      "… N106/36, NeuroMab). After washing, sections were treated with Alexa Fluor 647-labeled \n",
      "goat anti-rat IgG secondary antibody (1:500 in 10% NGS; A21247 Invitrogen) or donkey-anti-…\n",
      "\n",
      "Integrins are adhesion and survival molecules involved in axon growth during CNS development, \n",
      "as well as axon regeneration after injury in the peripheral nervous system (PNS). Adult …\n",
      "\n",
      "… Antibodies The following primary antibodies were used: mouse anti ankyrin-G clone N106/36 \n",
      "(1:500; NeuroMab, Davis), mouse anti panNav (1:500; Sigma, Saint Louis), and rabbit anti …\n",
      "\n",
      "Axon initial segments (AISs) generate action potentials and regulate the polarized distribution \n",
      "of proteins, lipids, and organelles in neurons. While the mechanisms of AIS Na + and K + …\n",
      "\n",
      "In vertebrates, fast saltatory conduction along myelinated axons relies on the node of \n",
      "Ranvier. How nodes assemble on CNS neurons is not yet fully understood. We previously …\n",
      "\n",
      "The axon initial segment (AIS) is essential for maintaining neuronal polarity, modulating \n",
      "protein transport into the axon, and action potential generation. These functions are supported …\n",
      "\n",
      "… N106/36, NeuroMab) for 24 hours at 4C. The slices then were washed three times with \n",
      "0.1M PBS and reacted with a solution containing the secondary antibody (Alexa Fluor 405; 1:500…\n",
      "\n",
      "Grey matter damage has been established as a key contributor to disability progression in \n",
      "multiple sclerosis. Aside from neuronal loss and axonal transections, which predominate in …\n",
      "\n",
      "We have previously shown that transplantation of immature DCX+/NeuN+/Prox1+ neurons (found \n",
      "in the neonatal DG), but not undifferentiated neuronal progenitor cells (NPCs) from …\n",
      "\n",
      "… et al., 2004), mouse α-K v 1.2 (K14/16; NeuroMab), chicken α-MAP2 (EnCor Biotechnology), \n",
      "mouse α-ankyrin G (ankG) (N106/36; NeuroMab), mouse α-pan Na v α (K58/35; Sigma), …\n",
      "\n",
      "… N106/36, NeuroMab). After washing, sections were treated with Alexa Fluor 647-labeled \n",
      "goat anti-rat IgG secondary antibody (1:500 in 10% NGS; A21247 Invitrogen) or donkeyanti-…\n",
      "\n",
      "Cell cycle reentry followed by neuronal hyperploidy and synaptic failure are two early hallmarks \n",
      "of Alzheimer’s disease (AD), however their functional connection remains unexplored. …\n",
      "\n",
      "Loss of neuronal polarity and missorting of the axonal microtubule-associated-protein TAU \n",
      "are hallmarks of Alzheimer’s disease (AD) and related tauopathies. Impairment of …\n",
      "\n",
      "… Mouse anti-AnkG (clone N106/36 and clone N106/20) and anti-Pan-Neurofascin (cloneA12/18) \n",
      "were from Neuromab. Mouse anti-EB1 (clone 5/EB1), and mouse anti-Rab11 (clone 47/…\n",
      "\n",
      "… The primary antibodies used were the following: anti-AnkG (mouse IgG2a, Neuromab, clone \n",
      "N106/36, 1:200), anti-PLP (hybridoma of polyclonal rat IgGs, clone AA3, gift from K. Ikenaka, …\n",
      "\n",
      "Axon initial segment (AIS) cell surface proteins mediate key biological processes in neurons \n",
      "including action potential initiation and axo-axonic synapse formation. However, few AIS …\n",
      "\n",
      "Ventricular ATP-sensitive potassium (K ATP ) channels link intracellular energy metabolism \n",
      "to membrane excitability and contractility. Our recent proteomics experiments identified …\n",
      "\n",
      "Differentiated neurons can undergo cell cycle re-entry during pathological conditions, but it \n",
      "remains largely accepted that M-phase is prohibited in these cells. Here we show that …\n",
      "\n",
      "During cortical development, neurons undergo polarization, oriented migration and layer-type \n",
      "differentiation. The biological and biochemical mechanisms underlying these processes …\n",
      "\n",
      "The axon initial segment (AIS) is a specialized region in neurons that encompasses two \n",
      "essential functions, the generation of action potentials and the regulation of the axodendritic …\n",
      "\n",
      "Copy number variations (CNVs) are associated with psychiatric and neurodevelopmental \n",
      "disorders (NDDs), and most, including the recurrent 15q13.3 microdeletion disorder, have …\n",
      "\n",
      "The precise control of the nervous system function under the vitality of synapses is \n",
      "extremely critical. Efforts have been taken to explore the underlying cellular and molecular …\n",
      "\n",
      "Neuronal type-specific physiologic heterogeneity can be driven by both extrinsic and intrinsic \n",
      "mechanisms. In retinal ganglion cells (RGCs), which carry visual information from the …\n",
      "\n",
      "Grey matter damage has been established as a key contributor to disability progression in \n",
      "multiple sclerosis. Aside from neuronal loss and axonal transections, which predominate in …\n",
      "\n",
      "Interregional neuronal communication is pivotal to instructing and adjusting cortical circuit \n",
      "assembly. Subcortical neuromodulatory systems project long-range axons to the cortex and …\n",
      "\n",
      "The pathogenesis of peripheral neuropathies in adults is linked to maintenance mechanisms \n",
      "that are not well understood. Here, we elucidate a novel critical maintenance mechanism …\n",
      "\n",
      "G-protein-coupled receptors (GPCRs) initiate a variety of signaling cascades, depending on \n",
      "effector coupling. β-arrestins, which were initially characterized by their ability to \"arrest\" …\n",
      "\n",
      "… Anti-AnkG (N106/36), anti-FGF14 (N56/21), and anti-Nav1.6 (K87A/10) mouse monoclonal \n",
      "antibodies were purchased from NeuroMab. Affinity-purified rabbit polyclonal antibodies …\n",
      "\n",
      "Axon initial segments (AISs) initiate action potentials and regulate the trafficking of vesicles \n",
      "between somatodendritic and axonal compartments. However, the mechanisms controlling …\n",
      "\n",
      "Action potential conduction along myelinated axons depends on high densities of voltage-gated \n",
      "Na + channels at the nodes of Ranvier. Flanking each node, paranodal junctions (…\n",
      "\n",
      "… (clone K13/31), anti-ADAM22 (clone N46/30), and anti-ankyrinG (clone N106/36) mAbs \n",
      "were obtained from NeuroMab (UC Davis/NIH NeuroMab Facility). Mouse anti-TAG-1 (1C12) …\n",
      "\n",
      "… anti-ankyrin-G was from NeuroMab (clone N106/36; cat. no. 75-146; RRID: AB_10673030. … \n",
      "anti-JNK1 (Pharmingen) at 1:100, ankyrin-G (NeuroMab) at 1:800, or P-JNK (Biosource) at 1:…\n",
      "\n",
      "The gene regulation underlying axon formation and its exclusiveness to neurons remains \n",
      "elusive. TRIM46 is postulated to determine axonal fate. We show Trim46 mRNA is expressed …\n",
      "\n",
      "The axon initial segment (AIS) is a unique neuronal compartment that plays a crucial role in \n",
      "the generation of action potential and neuronal polarity. The assembly of the AIS requires …\n",
      "\n",
      "Myelination is essential for central nervous system (CNS) formation, health, and function. \n",
      "Emerging evidence of oligodendrocyte heterogeneity in health and disease and divergent CNS …\n",
      "\n",
      "Differentiated neurons can undergo cell cycle re-entry during pathological conditions, but it \n",
      "remains largely accepted that M-phase is prohibited in these cells. Here we show that …\n",
      "\n",
      "Traumatic brain injury (TBI) is the largest non-genetic, non-aging related risk factor for \n",
      "Alzheimer's disease (AD). We report here that TBI induces tau acetylation (ac-tau) at sites …\n",
      "\n",
      "… /NIH NeuroMab Facility, clone N106/36). Primer oligonucleotides were obtained from \n",
      "Integrated DNA Technologies (Coralville, IA). A mouse Ank3 cDNA clone was obtained (BC021657…\n",
      "\n",
      "The axon initial segment (AIS) is the site of action potential generation and a locus of activity-dependent \n",
      "homeostatic plasticity. A multimeric complex of sodium channels, linked via a …\n",
      "\n",
      "The axon initial segment (AIS) is a highly specialized axonal compartment where the action \n",
      "potential is initiated. The heterogeneity of AISs has been suggested to occur between …\n",
      "\n",
      "The endoplasmic reticulum (ER) and plasma membrane (PM) form junctions crucial to ion \n",
      "and lipid signaling and homeostasis. The Kv2.1 ion channel is localized at ER–PM junctions …\n",
      "\n",
      "Supplementary Figure 2 (continued) Ten electron micrographs taken from serial sections of \n",
      "neurobiotin-labeled (HRP end-product) boutons (B) of AAC B45a making type II synapses (…\n",
      "\n",
      "Background Myelinated axons are organized into distinct subcellular and molecular regions. \n",
      "Without proper organization, electrical nerve conduction is delayed, resulting in detrimental …\n",
      "\n",
      "Neurons implement a variety of plasticity mechanisms to alter their function over timescales \n",
      "ranging from seconds to days. One powerful means of controlling excitability is to directly …\n",
      "\n",
      "Axons must withstand mechanical forces, including tension, torsion, and compression. \n",
      "Spectrins and actin form a periodic cytoskeleton proposed to protect axons against these forces. …\n",
      "\n",
      "Dyrk1a triplication in Down's syndrome and its overexpression in Alzheimer's disease \n",
      "suggest a role for increased DYRK1A activity in the abnormal metabolism of APP. Transport …\n",
      "\n",
      "During development of the peripheral nervous system (PNS), Schwann-cell-secreted \n",
      "gliomedin induces the clustering of Na + channels at the edges of each myelin segment to form …\n",
      "\n",
      "AnkyrinG (ankG) is highly enriched in neurons at axon initial segments (AISs) where it \n",
      "clusters Na + and K + channels and maintains neuronal polarity. How ankG becomes …\n",
      "\n",
      "The axon initial segment (AIS) is a highly specialized neuronal compartment that regulates \n",
      "the generation of action potentials and maintenance of neuronal polarity. Live imaging of the …\n",
      "\n",
      "… The following primary antibodies were used: mouse monoclonal ankyrinG (N106/36, UC \n",
      "Davis/NIH NeuroMab Facility Cat# 75-146 RRID:AB_10673030), NeuN (Millipore Cat# …\n",
      "\n",
      "Clustered ion channels at nodes of Ranvier are critical for fast action potential propagation \n",
      "in myelinated axons. Axon-glia interactions converge on ankyrin and spectrin cytoskeletal …\n",
      "\n",
      "Fast-spiking (FS) neurons can fire action potentials (APs) up to 1,000 Hz and play key roles \n",
      "in vital functions such as sound location, motor coordination, and cognition. Here we report …\n",
      "\n",
      "10.7554/eLife.21392.001 A high density of Na + channels at nodes of Ranvier is necessary \n",
      "for rapid and efficient action potential propagation in myelinated axons. Na+ channel …\n",
      "\n",
      "In neurons, most microtubules are not associated with a central microtubule-organizing \n",
      "center (MTOC), and therefore, both the minus and plus-ends of these noncentrosomal …\n",
      "\n",
      "The fidelity of inhibitory neurotransmission is dependent on the accumulation of γ-aminobutyric \n",
      "acid type A receptors (GABA A Rs) at the appropriate synaptic sites. Synaptic GABA A Rs …\n",
      "\n",
      "The formation, function, and plasticity of synapses require dynamic changes in synaptic \n",
      "receptor composition. Here, we identify the sorting receptor SorCS1 as a key regulator of …\n",
      "\n",
      "Myelinated neurons are organized into distinct molecular domains that are essential for \n",
      "electrical conduction. These domains include the axon initial segment, nodes of Ranvier, …\n",
      "\n",
      "… We provide examples from NeuroMab screens, and from the subsequent specialized … of \n",
      "NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs at the …\n",
      "\n",
      "… Membranes were blocked in 5%BSA/TBS for 1 h at RT and incubated overnight at 4C with \n",
      "a primary monoclonal antibody (anti-SCN4b; UC Davis/NeuroMab facility, clone N168/6, #75-…\n",
      "\n",
      "… IgG 1 ; clone N168/6), Kv1.2 (subtype IgG 2b ; clone K14/16) and Caspr (subtype IgG 1 ; \n",
      "clone K65/35) were obtained from the UC Davis/NIH NeuroMab Facility. Mouse monoclonal …\n",
      "\n",
      "… The following primary antibodies were used: mouse anti-α-Na v β4 (N168/6 generated \n",
      "against a fusion protein including amino acids 184–228; NeuroMab), rabbit α-Na v β4 F3888 (…\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… -G hybridoma-generated N106/20 mAb (left section) and the N106/20 scFv (right section). (c) …\n",
      "\n",
      "… We used the following primary antibodies: mouse monoclonal anti-AnkG (1:500; clone \n",
      "N106/20, NeuroMab), guinea pig polyclonal GABA A α2 (1:500; 224 104; Synaptic Systems) and …\n",
      "\n",
      "… N106/36 and N106/20), glial fibrillary acidic protein (GFAP, N206A/8), and Nav1.6 (K87/10A) \n",
      "antibodies were purchased from the UC Davis/NIH funded Neuromab monoclonal antibody …\n",
      "\n",
      "Axon formation, the initial step in establishing neuronal polarity, critically depends on local \n",
      "microtubule reorganization and is characterized by the formation of parallel microtubule …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… -G hybridoma-generated N106/20 mAb (left section) and the N106/20 scFv (right section). (c) …\n",
      "\n",
      "Ankyrin-G is an adaptor protein that links membrane proteins to the underlying cytoskeletal \n",
      "network. Alternative splicing of the Ank3 gene gives rise to multiple ankyrin-G isoforms in …\n",
      "\n",
      "… After a 2 194 hour block step using 10% normal goat serum and 0.25% Triton X-100 (Sigma), \n",
      "the mouse 195 monoclonal anti-ankyrinG antibody (1:500, clone N106/20, NeuroMab) and …\n",
      "\n",
      "… ) were used: AnkG mAb clone N106/20 (western blot) and clone N106/36 (immunocytochemistry) \n",
      "from Neuromab (Davis, CA, United States), PSD-95 (Neuromab), turboGFP mAb (to …\n",
      "\n",
      "… using monoclonal antibodies from the UC Davis/NIH NeuroMab Facility (University of California, \n",
      "Davis, CA 95616, USA), clones N105/13 and N106/20, respectively. F-actin was labeled …\n",
      "\n",
      "AnkyrinG (ankG) is highly enriched in neurons at axon initial segments (AISs) where it \n",
      "clusters Na + and K + channels and maintains neuronal polarity. How ankG becomes …\n",
      "\n",
      "Neuronal GABAergic responses switch from excitatory to inhibitory at an early postnatal \n",
      "period in rodents. The timing of this switch is controlled by intracellular Cl− concentrations, but …\n",
      "\n",
      "… Mouse anti-AnkG (clone N106/36 and clone N106/20) and anti-Pan-Neurofascin (cloneA12/18) \n",
      "were from Neuromab. Mouse anti-EB1 (clone 5/EB1), and mouse anti-Rab11 (clone 47/…\n",
      "\n",
      "The axon initial segment (AIS) is a unique neuronal compartment that plays a crucial role in \n",
      "the generation of action potential and neuronal polarity. The assembly of the AIS requires …\n",
      "\n",
      "Peripheral sensory neurons are characterized by their size, molecular profiles, and physiological \n",
      "responses to specific stimuli. In mouse, the peptidergic and non‐peptidergic subsets of …\n",
      "\n",
      "… the following antibodies: mouse monoclonal anti-Na v 1.7 clone N68/6 (UC Davis/National \n",
      "Institute of Health (NIH) NeuroMab Facility, University of California), mouse monoclonal clone …\n",
      "\n",
      "Nedd4-2 (NEDD4L in humans) is a ubiquitin protein ligase best known for its role in regulating \n",
      "ion channel internalization and turnover. Nedd4-2 deletion in mice causes perinatal …\n",
      "\n",
      "… or 10 μg of mouse anti-Na v 1.7 monoclonal antibody N68/6 (UC Davis/NIH NeuroMab \n",
      "Facility). Anti-Na v 1.7 antibody N68/6 does not cross-react with other Na + channel isoforms or …\n",
      "\n",
      "Key points Mouse chromaffin cells (MCCs) of the adrenal medulla possess fast‐inactivating \n",
      "Nav channels whose availability alters spontaneous action potential firing patterns and the …\n",
      "\n",
      "… We used an anti-NaV1.7 antibody [N68/6] (NeuroMab) that binds to both human and mouse \n",
      "NaV1.7 (0.5 μg/mL). Paraffin-embedded formalin-fixed 4 μm sections were deparaffinized …\n",
      "\n",
      "Schizophrenia is associated with cognitive and behavioral dysfunctions thought to reflect \n",
      "imbalances in neurotransmission systems. Recent screenings suggested that lack of (functional…\n",
      "\n",
      "… primary antibodies anti-GluN1 1:1000 (NeuroMab clone N308/48), anti-GluN2A 1:500 (NeuroMab \n",
      "clone N327/95) or anti-GluN2B 1:1000 (NeuroMab clone N59/36), and then washed …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "… glutamate receptor antibody (clone N327/95; NeuroMab) diluted 1:… ; NeuroMab) diluted \n",
      "1:10; a purified mouse monoclonal NR1 glutamate receptor antibody (clone N308/48; NeuroMab) …\n",
      "\n",
      "Painful peripheral neuropathy is a severe adverse effect of chemotherapeutic drugs such as \n",
      "paclitaxel (Taxol). The glutamate N-methyl-d-aspartate receptors (NMDARs) are critically …\n",
      "\n",
      "Dendritic spines (DS) are tiny protrusions implicated in excitatory postsynaptic responses in \n",
      "the CNS. To achieve their function, DS concentrate a high density of ion channels and …\n",
      "\n",
      "Mutations of the cyclin-dependent kinase-like 5 (CDKL5) and netrin-G1 (NTNG1) genes \n",
      "cause a severe neurodevelopmental disorder with clinical features that are closely related to …\n",
      "\n",
      "Microwave (MW) radiation induced oxidative stress reduces dendritic arborization, spine \n",
      "density and number of hippocampal pyramidal neurons and hence, impair learning and spatial …\n",
      "\n",
      "Heterotetrameric glutamate receptors are essential for the development, function, and \n",
      "plasticity of spine synapses but how they are organized to achieve this is not known. Here we …\n",
      "\n",
      "Exo70 is a subunit of the greater exocyst complex, a collection of proteins that oversees \n",
      "cellular membrane addition and polarized exocytosis by acting as a tethering intermediate …\n",
      "\n",
      "Background Exo70 is a subunit of the greater exocyst complex, a collection of proteins that \n",
      "oversees cellular membrane addition and polarized exocytosis by acting as a tethering …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "Autism spectrum disorder (ASD) is a heritable, common neurodevelopmental disorder with \n",
      "diverse genetic causes. Several studies have implicated protein synthesis as one among …\n",
      "\n",
      "The complex functions of neuronal synapses depend on their tightly interconnected protein \n",
      "network, and their dysregulation is implicated in the pathogenesis of autism spectrum …\n",
      "\n",
      "Intracellular trafficking of AMPA receptors is a tightly regulated process which involves \n",
      "several adaptor proteins, and is crucial for the activity of excitatory synapses both in basal …\n",
      "\n",
      "… (2012) in analyses of the same mice, and the mouse antibody to mGluR1/5 (Neuromab clone \n",
      "N75/3) that was also used in the other immunoblots of this study (panels A and C). Proteins …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "… Anti-GABA-A β1 receptor N96/55: yields a single band of expected size (55 kDa) on \n",
      "immunoblots of rat brain samples, does not cross-react with GABA-A β2 receptor or GABA-A β3 …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… 1:50 for EM), clone N367/62] and Shank3 ab4 [mouse monoclonal against Shank3 (aa \n",
      "840–857 of rat Shank3; 1:50 for EM), clone N69/46] were from NeuroMab (Davis, CA). Specificity …\n",
      "\n",
      "… , Sigma, 1153630500), including: mouse anti-shank3 (1:500, NeuroMab, Davis, CA, Cat. #: \n",
      "75-344, clone N367/62), mouse anti-tubulin (1:10 000, Sigma, St. Louis, MO, Cat. #: T9026). …\n",
      "\n",
      "… fluoride membrane and immunoblotted with either the N367/62 anti-Shank3 antibody directed \n",
      "against an epitope in the SH3 domain (UC Davis/NIH NeuroMab Facility) or the H160 anti-…\n",
      "\n",
      "During development, pyramidal neurons undergo dynamic regulation of AMPA receptor (AMPAR) \n",
      "subunit composition and density to help drive synaptic plasticity and maturation. …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "Perineuronal nets (PNNs) are structures that contain extracellular matrix chondroitin sulfate \n",
      "proteoglycan and surround the soma and dendrites of various neuronal cell types. They are …\n",
      "\n",
      "Mutation or disruption of the SH3 and ankyrin repeat domains 3 (SHANK3) gene represents \n",
      "a highly penetrant, monogenic risk factor for autism spectrum disorder, and is a cause of …\n",
      "\n",
      "The family of Shank scaffolding molecules (comprising Shank1, 2 and 3) are core components \n",
      "of the postsynaptic density ( PSD ) in neuronal synapses. Shanks link surface receptors to …\n",
      "\n",
      "… In contrast, when probed with an antibody raised against the N terminus of SHANK3 (Neuromab \n",
      "367/62), we detected truncated SHANK3 bands in striatal PSD preparations from the …\n",
      "\n",
      "The Par3 polarity protein is critical for subcellular compartmentalization in different developmental \n",
      "processes. Variants of PARD3, which encodes PAR3, are associated with intelligence …\n",
      "\n",
      "Haploinsufficiency of the SHANK3 gene is causally linked to autism spectrum disorder (ASD), \n",
      "and ASD-associated genes are also enriched for chromatin remodelers. Here we found …\n",
      "\n",
      "Haploinsufficiency of the Shank3 gene, which encodes a scaffolding protein at glutamatergic \n",
      "synapses, is a highly prevalent and penetrant risk factor for autism. Using combined …\n",
      "\n",
      "Autism spectrum disorder (ASD) is a heritable, common neurodevelopmental disorder with \n",
      "diverse genetic causes. Several studies have implicated protein synthesis as one among …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "… KCC2 NeuroMab N1/12) that exhibit comparable immunoreactivity across rodent and human \n",
      "brain samples, and two (anti-Kv3.1b NeuroMab N16B/8 and anti-HCN2 NeuroMab N71/37) …\n",
      "\n",
      "Thalamocortical neurons (TCNs) play a critical role in the maintenance of thalamocortical \n",
      "oscillations, dysregulation of which can result in certain types of seizures. Precise control over …\n",
      "\n",
      "… Proteins were transferred to a nitrocellulose membrane and subjected to immunoblot \n",
      "analysis with mouse monoclonal antibodies against HCN2 (N71/37; UC Davis/NIH NeuroMab …\n",
      "\n",
      "… Antibodies used were as follows: rat monoclonal anti-HCN1 (clone RQT 7C3, EMD \n",
      "Millipore); mouse monoclonal anti-HCN2 (clone N71/37, NeuroMab); and rabbit polyclonal anti-…\n",
      "\n",
      "The superior paraolivary nucleus (SPON) is a prominent structure in the auditory brainstem. \n",
      "In contrast to the principal superior olivary nuclei with identified roles in processing binaural …\n",
      "\n",
      "… Proteins were transferred to a nitrocellulose membrane and subjected to immunoblot \n",
      "analysis with mouse monoclonal antibodies against HCN2 (N71/37; UC Davis/NIH NeuroMab …\n",
      "\n",
      "Properties, developmental regulation, and cAMP modulation of the hyperpolarization-activated \n",
      "current (I h ) were investigated by the whole cell patch-clamp technique in vestibular …\n",
      "\n",
      "The chemosensory glomus cells of the carotid body (CB) detect changes in O 2 tension. \n",
      "Carotid sinus nerve fibers, which originate from peripheral sensory neurons located within the …\n",
      "\n",
      "Schizophrenia is associated with cognitive and behavioral dysfunctions thought to reflect \n",
      "imbalances in neurotransmission systems. Recent screenings suggested that lack of (functional…\n",
      "\n",
      "… antibodies anti-GluN1 1:1000 (NeuroMab clone N308/48), anti-GluN2A 1:500 (NeuroMab \n",
      "clone N327/95) or anti-GluN2B 1:1000 (NeuroMab clone N59/36), and then washed 3x 10 min …\n",
      "\n",
      "Excessive stimulation of NMDA receptors with glutamate or other potent agonists such as \n",
      "NMDA leads to excitotoxicity and neural injury. In this study, we aimed to provide insight into …\n",
      "\n",
      "Mutations in leucine-rich repeat kinase 2 (LRRK2), which are associated with autosomal \n",
      "dominant Parkinson's disease, elicit progressive dendrite degeneration in neurons. We …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "AB. Diagram of the structure of the human SHANK2 gene and protein.(A) The nonsense \n",
      "mutation R841X is found in an individual with ASD and ID.(B) SHANK2 protein contains 5 …\n",
      "\n",
      "Cue-induced drug craving progressively intensifies after withdrawal from self-administration \n",
      "of cocaine, methamphetamine, and other drugs of abuse, a phenomenon termed incubation …\n",
      "\n",
      "… receptor antibody (clone N59/20; NeuroMab) diluted 1:10; a purified mouse … ; NeuroMab) \n",
      "diluted 1:10; a mouse monoclonal supernatant GluA2/GluR2 antibody (clone L21/32; NeuroMab…\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "The majority of γ-aminobutyric acid (GABA)ergic interneurons have smooth dendrites with no \n",
      "or only few dendritic spines, but certain types of spiny GABAergic interneurons do actually …\n",
      "\n",
      "Alzheimer's disease (AD) is characterized by memory impairments and age-dependent \n",
      "synapse loss. Experimental and clinical studies have shown decreased expression of the …\n",
      "\n",
      "… Twenty micrograms of GluA2 antibody (NeuroMab, clone L21/32) was covalently conjugated \n",
      "onto the magnetic beads with Protein A and Protein G as follows. Equal volume (80 μl each…\n",
      "\n",
      "… The monoclonal antibodies PSD-95 (Clone K28/43) and GluA2 (Clone L21/32) were \n",
      "developed by and/or obtained from the UC Davis/NIH NeuroMab Facility and maintained by the …\n",
      "\n",
      "Diverse lines of evidence suggest that amyloid-β (Aβ) peptides causally contribute to the \n",
      "pathogenesis of Alzheimer disease (AD), the most frequent neurodegenerative disorder. …\n",
      "\n",
      "We sought to examine AMPA receptor (AMPAR) function in the medial posterodorsal amygdala \n",
      "(MePD), as glutamate neurotransmission is critical for the neural response to vaginal–…\n",
      "\n",
      "A subset of amyotrophic lateral sclerosis (ALS) and frontotemporal lobar degeneration (FTLD) \n",
      "patients present pathological redistribution and aggregation of the nuclear protein fused in …\n",
      "\n",
      "L-methionine, the principal sulfur-containing amino acid in proteins, plays critical roles in \n",
      "cell physiology as an antioxidant and in the breakdown of fats and heavy metals. Previous …\n",
      "\n",
      "The locus coeruleus (LC) provides the principal supply of noradrenaline (NA) to the brain, \n",
      "thereby modulating an array of brain functions. The release of NA, and therefore its impact on …\n",
      "\n",
      "Glutamate AMPA receptors (AMPARs) lacking GluA2 subunit are calcium permeable (CP‐AMPARs), \n",
      "which are increased in the hypothalamic paraventricular nucleus (PVN) and …\n",
      "\n",
      "In the rodent nucleus accumbens (NAc), cocaine elevates levels of brain‐derived neurotrophic \n",
      "factor (BDNF). Conversely, BDNF can augment cocaine‐related behavioral responses. …\n",
      "\n",
      "Changes in dendritic spine morphology, a hallmark of synaptic plasticity, involve remodeling \n",
      "of the actin cytoskeleton, a process that is regulated by Rho GTPases. RhoA, a member of …\n",
      "\n",
      "Glutamate in the peripheral nervous system is involved in neuropathic pain, yet we know little \n",
      "how nerve injury alters responses to this neurotransmitter in primary sensory neurons. We …\n",
      "\n",
      "Alzheimer’s Disease (AD) is the primary cause of dementia among the elderly population. \n",
      "Elevated plasma levels of homocysteine (HCy), an amino acid derived from methionine …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "10.7554/eLife.24192.001 Glutamatergic synapses rely on AMPA receptors (AMPARs) for fast \n",
      "synaptic transmission and plasticity. AMPAR auxiliary proteins regulate receptor trafficking, …\n",
      "\n",
      "The alpha-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptors are important \n",
      "glutamatergic receptors mediating fast excitatory synaptic transmission in the brain. The …\n",
      "\n",
      "Glutamate in the peripheral nervous system is involved in neuropathic pain, yet we know little \n",
      "how nerve injury alters responses to this neurotransmitter in primary sensory neurons. We …\n",
      "\n",
      "Alzheimer's disease (AD) is a neurodegenerative disorder characterized by a progressive \n",
      "deterioration of cognitive abilities, accumulation of the amyloid-β peptide (Aβ), increase of …\n",
      "\n",
      "Neuronal activity is highly dependent on energy metabolism. Nuclear respiratory factor 2 (NRF-2) \n",
      "tightly couples neuronal activity and energy metabolism by transcriptionally co-…\n",
      "\n",
      "Brain-derived neurotrophic factor (BDNF) contributes to diverse types of plasticity, including \n",
      "cocaine addiction. We investigated the role of BDNF in the rat nucleus accumbens (NAc) in …\n",
      "\n",
      "Proteins derived from the Kalrn gene, encoding 2 Rho guanine nucleotide exchange factor (GEF) \n",
      "domains, affect dendritic and axonal morphogenesis. The roles of endogenous Kalirin-…\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "Dendrite branching and spine formation determines the function of morphologically distinct \n",
      "and specialized neuronal subclasses. However, little is known about the programs instructing …\n",
      "\n",
      "… ), mouse anti-GluA2 monoclonal antibody (1:250, Neuromab, L21/32), mouse anti-GluN2B \n",
      "monoclonal antibody (1:250, Neuromab, N59/36), mouse anti-GluN2A monoclonal antibody (…\n",
      "\n",
      "The forebrain specific AMPA receptor antagonist, LY3130481/CERC-611, which selectively \n",
      "antagonizes the AMPA receptors associated with TARP γ-8, an auxiliary subunit enriched in …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "… L86A/37 does not detect other isoforms of AMIGO in immunoblots using cell lysates from \n",
      "heterologous cells expressing either AMIGO-1, -2, or -3 (manufacturer's specifications). …\n",
      "\n",
      "… Other lanes include antiserum from one of the immunized mice (HB), a mouse serum \n",
      "negative control (NMS) and a positive control NeuroMab mAb against a different target (L86A/37, …\n",
      "\n",
      "… (red, N372B/1 mAb), and AMIGO-1 (blue, L86A/37 mAb). Images were adjusted linearly \n",
      "for … distinct cytoplasmic domain binding sites (L86/33 and L86A/37), and a rabbit pAb (AMIGO-1) …\n",
      "\n",
      "… Purified Kv2 and AMIGO mouse monoclonal antibodies, knockout verified, were from \n",
      "NeuroMab (Kv2.1, K89/34; Kv2.2, N37B/1; and AMIGO, L86A/37) were used in respective …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… ) in red and the anti-Amigo-1 L86A/37 mAb (IgG2b) in purple. Hoechst nuclear labelling is …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… (IgG2a) in red and the anti-Amigo-1 L86A/37 (IgG2b) in purple. Hoechst nuclear labelling is …\n",
      "\n",
      "The Kv2 channels encode delayed rectifier currents that regulate membrane potential in \n",
      "many tissues. They also have a non-conducting function to form stable junctions between the …\n",
      "\n",
      "… CMSSID[pS]FISSA; anti-AMIGO mAb L86/33, generated against amino acids 395-493 (cytoplasmic \n",
      "C-terminus) of human AMIGO-1 (RRID:AB_2315798); anti-AMIGO mAb L86A/37, …\n",
      "\n",
      "Ring finger protein 146 (Iduna) facilitates DNA repair and protects against cell death \n",
      "induced by NMDA receptor-mediated glutamate excitotoxicity or by cerebral ischemia. …\n",
      "\n",
      "Traumatic brain injury (TBI) is common and often fatal in current times. The role of poly(adenosine \n",
      "diphosphate-ribose) polymerase (PARP)-induced cell death (parthanatos) in TBI has …\n",
      "\n",
      "… The following primary antibodies were used: mouse antibody to RNF146 (N201/35, 1:5000, \n",
      "NeuroMab, Davis, CA, USA), mouse antibody to PAR (cat# 4335-MC-100, 1:3000, Trevigen, …\n",
      "\n",
      "Progressive degeneration of dopaminergic neurons characterizes Parkinson’s disease (PD). \n",
      "This neuronal loss occurs through diverse mechanisms, including a form of programmed …\n",
      "\n",
      "… The following primary antibodies were used: mouse antibody to RNF146 (N201/35, 1:5000, \n",
      "NeuroMab), rabbit antibody to GFP (cat# 2956, 1:5000, Cell Signaling Technology), mouse …\n",
      "\n",
      "… We also tested another Syt9 antibody (Neuromab N276A/15) but labeling with this \n",
      "antibody was unchanged in Syt9 global knockout retina. …\n",
      "\n",
      "Clearance of mitochondria following damage is critical for neuronal homeostasis. Here, we \n",
      "investigate the role of Miro proteins in mitochondrial turnover by the PINK1/Parkin …\n",
      "\n",
      "… (Neuromab N372B/1: right). Note that there is very little Kv2.2 expression on the EYFP-positive \n",
      "(green) PNs. Most of the Kv2.2 staining is on EYFP-negative (non-Thy1) PNs. …\n",
      "\n",
      "Voltage-gated K + (Kv) channels play important roles in regulating neuronal excitability. Kv \n",
      "channels comprise four principal α subunits, and transmembrane and/or cytoplasmic …\n",
      "\n",
      "… available from the UC Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). 27 … RRID:AB_2532100 \n",
      "1:5 dilution of Tissue Culture Supernatant, concentration unknown N372B/1, anti…\n",
      "\n",
      "… Acknowledgements: Most of the antibodies used in this study are available from the UC \n",
      "Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). Correspondence should be addressed to …\n",
      "\n",
      "… (NeuroMab; mAb binds within aa 837–853; cat #73-014 RRID:AB10672253). For the Kv2.2 \n",
      "immunoreactivity experiments, we used the N372B/1 mouse IgG1 antibody (NeuroMab; mAb …\n",
      "\n",
      "The axon initial segment (AIS) is the site of action potential generation and a locus of activity-dependent \n",
      "homeostatic plasticity. A multimeric complex of sodium channels, linked via a …\n",
      "\n",
      "… This probe was based on the well-characterized mouse mAb L109/57 (UC Davis/NIH NeuroMab \n",
      "… This is the same penetration depth as seen for the L109/57 mAb from which the scFv is …\n",
      "\n",
      "Lysosomes communicate through cholesterol transfer at endoplasmic reticulum (ER) contact \n",
      "sites. At these sites, the Niemann Pick C1 cholesterol transporter (NPC1) facilitates the …\n",
      "\n",
      "… Note that the NeuroMab and Santa Cruz antibodies do not detect all the identical species, \n",
      "but … SC3H3 and NeuroMab antibodies also recognize a 27 kDa T1-LIR species in adult brain (…\n",
      "\n",
      "… (L113/13, L113/27, L113/130 and) that were eventually selected as NeuroMabs, and the … \n",
      "mAb L113/27 does not selectively recognize the target expressing cells (GI) and is similar in …\n",
      "\n",
      "The hyperpolarization-activated cyclic-nucleotide-gated non-selective cation (HCN) channels \n",
      "play a vital role in the neurological basis underlying nervous system diseases. However, …\n",
      "\n",
      "Hyperpolarization-activated cyclic-nucleotide-gated cation nonselective (HCN) channels \n",
      "are involved in the pathology of nervous system diseases. HCN channels and γ-aminobutyric …\n",
      "\n",
      "Dysregulation of hyperpolarization-activated cyclic nucleotide-gated cation (HCN) channels \n",
      "alters neuronal excitability. However, the role of HCN channels in status epilepticus is not …\n",
      "\n",
      "Neuronal ankyrins cluster and link membrane proteins to the actin and spectrin-based \n",
      "cytoskeleton. Among the three vertebrate ankyrins, little is known about neuronal Ankyrin-R (AnkR…\n",
      "\n",
      "… For co-immunoprecipitation experiments, 50 μl of protein G-coated magnetic beads (Life \n",
      "Technologies) were incubated (4 C, overnight with nutation) with 10 μg of anti-Kvβ2 (NeuroMab…\n",
      "\n",
      "… (K7/45) were purchased from Neuromab, and anti–β-actin Abs were purchased from Bioeasy. \n",
      "Digital images were captured using the Amersham Image 600 system (GE Healthcare). …\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "… (NeuroMab) recognized a single protein band at 70 kDa on western blot of lysates from \n",
      "Kv1.5-transfected COS cells. No band of equivalent size is observed from rat brain samples and …\n",
      "\n",
      "… (K7/45) and Mortalin/Grp75 (N52A/42) using monoclonal antibodies (clone number in \n",
      "parentheses) obtained from the UC Davis/NIH NeuroMab Facility. Alexa-conjugated fluorescent …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… ) using a monoclonal antibody specific for K V 2.1 subunits (NeuroMab; 75-159; clone \n",
      "K39/25, University of California, Davis). The secondary antibody was an Alexa Fluor 488-…\n",
      "\n",
      "… Although both the Alomone anti-BKbeta2 Ab and the NeuroMab N53/32 β2 Ab were of use \n",
      "for IP and identification of β2 protein in our Western blot procedures, in our hands these Abs …\n",
      "\n",
      "Background: Inflammation-induced sensitization of primary afferents is associated with a \n",
      "decrease in K + current. However, the type of K + current and basis for the decrease varies as a …\n",
      "\n",
      "… The membranes were incubated with primary antibodies including anti-BK α (Neuromab, \n",
      "L6/60, 1:1000), anti-BK β2 (Neuromab, N53/32 1:1000), anti-BK β4 (Neuromab, L18A.3, 1:1000…\n",
      "\n",
      "Inactivation is an intrinsic property of several voltage-dependent ion channels, closing the \n",
      "conduction pathway during membrane depolarization and dynamically regulating neuronal …\n",
      "\n",
      "… For co-immunoprecipitation experiments, 50 μl of protein G-coated magnetic beads (Life \n",
      "Technologies) were incubated (4 C, overnight with nutation) with 10 μg of anti-Kvβ2 (NeuroMab…\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… polyclonal KCNE1 antibody (1:100; Alomone Labs, Jerusalem, Israel), and Kv1.4 was also \n",
      "labeled with a mouse monoclonal Kv1.4 antibody (1:250, clone K13/31, 1:300, NeuroMab). A …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "… /13, L133/27, L113/130) that were ultimately selected as NeuroMab mAbs. B. Exemplar results \n",
      "… , L113/27, L113/130 and) that were eventually selected as NeuroMabs, and the fourth row …\n",
      "\n",
      "… -CD98 (Abcam, 2 µg mL −1 ), anti-KCC2 transporter NeuroMab clone N1/12 (1:1,000), \n",
      "anti-NKCC1 (NeuroMab clone N161/20, 1:1,000) at 4C overnight. Membranes were washed in …\n",
      "\n",
      "… (K7/45) and Mortalin/Grp75 (N52A/42) using monoclonal antibodies (clone number in \n",
      "parentheses) obtained from the UC Davis/NIH NeuroMab Facility. Alexa-conjugated fluorescent …\n",
      "\n",
      "… (K13/31) mouse monoclonal antibodies used in this study are available from NeuroMab (http://www.neuromab.org), \n",
      "a not-for-profit supplier of monoclonal antibodies administered …\n",
      "\n",
      "… (K13/31) were obtained from NeuroMab. Rabbit pAbs to mitogen-activated protein kinase \n",
      "(MAPK) and to phospho-MAPK were from Promega. A mouse mAb to Src as well as to PP2 …\n",
      "\n",
      "… These mAbs are available from the UC Davis/ NIH NeuroMab Facility (neuromab.ucdavis.… \n",
      "(determined by staining with mouse mAb K13/31) in the middle molecular of the dentate gyrus, …\n",
      "\n",
      "10.7554/eLife.12661.001 Neuropathic pain following peripheral nerve injury is associated \n",
      "with hyperexcitability in damaged myelinated sensory axons, which begins to normalise over …\n",
      "\n",
      "… K13/31, 1:300, NeuroMab). A negative control was also performed for the colocalization \n",
      "correlation, under the same conditions as for the positive control but with one image rotated 90 …\n",
      "\n",
      "… #: K14/16, K17/70, and K13/31, respectively; UC Davis/NIH Neuromab Facility, Davis, CA, \n",
      "USA), … We thank University of California, Davis-NIH NeuroMab facility (a nonprofit supplier) for …\n",
      "\n",
      "… The total cellular pool was detected by cytoplasmic-directed mAb K14/16 (Kv1.2) or K13/31 \n",
      "(Kv1.4) after detergent permeabilization. Alternatively, surface staining of intact cells with Kv1.…\n",
      "\n",
      "The plasma membrane (PM) comprises distinct subcellular domains with diverse functions \n",
      "that need to be dynamically coordinated with intracellular events, one of the most impactful …\n",
      "\n",
      "Cortical dysplasia (CD) is a common cause for intractable epilepsy. Hyperactivation of the \n",
      "mechanistic target of rapamycin (mTOR) pathway has been implicated in CD; however, the …\n",
      "\n",
      "… as for K V 1.4 but is sustained mostly by the specificity of the monoclonal antibody 75–112, \n",
      "which has been also demonstrated not to cross react with other K V channels (Neuromab). As …\n",
      "\n",
      "A specialized axonal ending, the basket cell “pinceau,” encapsulates the Purkinje cell axon \n",
      "initial segment (AIS), exerting final inhibitory control over the integrated outflow of the …\n",
      "\n",
      "Neuropathic pain following peripheral nerve injury is associated with hyperexcitability in \n",
      "damaged myelinated sensory axons, which begins to normalise over time. We investigated the …\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "… the supplier's (NeuroMab) datasheets online. The protein bands detected match with the \n",
      "expected sizes from endogenous and heterologous expression as reported by NeuroMab. We …\n",
      "\n",
      "miR-17-92 is a microRNA cluster with six distinct members. Here, we show that the miR-17-92 \n",
      "cluster and its individual members modulate chronic neuropathic pain. All cluster …\n",
      "\n",
      "Modulation of voltage-gated potassium (Kv) channel surface expression can profoundly \n",
      "affect neuronal excitability. Some but not all mammalian Shaker or Kv1 α subunits contain a …\n",
      "\n",
      "Over the last years extensive kinase-mediated regulation of a number of voltage-gated \n",
      "potassium (Kv) channels important in cardiac electrophysiology has been reported. This includes …\n",
      "\n",
      "The chemosensory glomus cells of the carotid body (CB) detect changes in O 2 tension. \n",
      "Carotid sinus nerve fibers, which originate from peripheral sensory neurons located within the …\n",
      "\n",
      "Potassium channels regulate numerous aspects of neuronal excitability, and several voltage-gated \n",
      "K + channel subunits have been identified in pyramidal neurons of rat neocortex. …\n",
      "\n",
      "In this study, we investigated the tissue expression levels, alpha subunit composition and \n",
      "distribution of Shaker-related voltage-dependent potassium Kv1 channels in human …\n",
      "\n",
      "Neuropathic pain (NP) embraces a broad range of conditions linked with a disease or \n",
      "lesion of the peripheral or central somatosensory system and its prevalence in the general …\n",
      "\n",
      "Neuropathic pain (NP) embraces a broad range of conditions linked with a disease or \n",
      "lesion of the peripheral or central somatosensory system and its prevalence in the general …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "In myelinated fibers, the voltage-gated sodium channels Nav1 are concentrated at the nodal \n",
      "gap to ensure the saltatory propagation of action potentials. The voltage-gated potassium …\n",
      "\n",
      "Autosomal dominant lateral temporal epilepsy (ADLTE) is an inherited syndrome caused by \n",
      "mutations in the leucine-rich glioma inactivated 1 (LGI1) gene. It is known that glutamatergic …\n",
      "\n",
      "… The mouse mAbs N52A/42, K13/31, K89/34, L83/11, L86/33, and L86A/37 are available \n",
      "from the UC Davis/NIH NeuroMab Facility (http://neuromab.ucdavis.edu/), a not-for profit …\n",
      "\n",
      "Oligodendrocytes (OLs) are glial cells that form myelin sheaths surrounding the axons in the \n",
      "central nervous system (CNS). Jimpy (jp) mutant mice are dysmyelinating disease models …\n",
      "\n",
      "Copy number variations encompassing the gene encoding Cyfip1 have been associated \n",
      "with a variety of human diseases, including autism and schizophrenia. Here we show that …\n",
      "\n",
      "Axonal branching and synapse formation are tightly linked developmental events during the \n",
      "establishment of synaptic circuits. Newly formed synapses promote branch initiation and …\n",
      "\n",
      "… We found that unlike K28/43, mAbs K28/37, K28/74 and K28/77 labeled synapses in both … \n",
      "negative control (NMS) and a positive control NeuroMab mAb against a different target (L86A/…\n",
      "\n",
      "Substantial data has shown that the lectican group of chondroitin sulfate proteoglycans are \n",
      "involved in inhibition of axonal plasticity in response to injury in the central nervous system. …\n",
      "\n",
      "… For quantitation studies, immunolabeling of the various sample preparations was conducted \n",
      "with two different monoclonal antibodies, namely K28/74 (PSD-95 MAGUK scaffold protein, …\n",
      "\n",
      "10.7554/eLife.40316.001 Protein palmitoylation and depalmitoylation alter protein function. \n",
      "This post-translational modification is critical for synaptic transmission and plasticity. …\n",
      "\n",
      "Synapses are found in vast numbers in the brain and contain complex proteomes. We \n",
      "developed genetic labeling and imaging methods to examine synaptic proteins in individual …\n",
      "\n",
      "… experiments were: mouse monoclonal anti-PSD95 (clone K28/74; 1 mg mL −1 , dilution 1:… \n",
      "immunohistochemistry: mouse monoclonal anti-PSD95 (clone K28/74; 1 mg mL −1 , dilution 1:…\n",
      "\n",
      "Visual signals are segregated into parallel pathways at the first synapse in the retina between \n",
      "cones and bipolar cells. Within the OFF pathways of mammals, the selective expression of …\n",
      "\n",
      "Although considerable evidence suggests that the chemical synapse is a lynchpin underlying \n",
      "affective disorders, how molecular insults differentially affect specific synaptic connections …\n",
      "\n",
      "Synapses in the brain exhibit cell-type-specific differences in basal synaptic transmission \n",
      "and plasticity. Here, we evaluated cell-type-specific specializations in the composition of …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "… Among the examples shown, immunolabeling in panel a1 fits these criteria, such that N355/1, \n",
      "which ultimately became a NeuroMab, was judged as target-specific. Immunolabeling for …\n",
      "\n",
      "… Neuronal homogenates were then western blotted against GluA1 (clone N355/1; NeuroMab), \n",
      "GluA1 Phospho Ser 831 (Cat. #04-823; Millipore), CamKII G-1 (Cat. #sc-5306; Santa Cruz)…\n",
      "\n",
      "Gray matter degeneration contributes to progressive disability in multiple sclerosis (MS) and \n",
      "can occur out of proportion to measures of white matter disease. Although white matter …\n",
      "\n",
      "… of endogenously expressed BK channels and NMDARs with in situ PLA, we examined \n",
      "the antigen specificities of a mouse monoclonal anti-BKα antibody (L6/60; NeuroMab) and a …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "… /ml) (K28/43, UC Davis/NIH NeuroMab Facility), anti-GluA1 mouse monoclonal antibody (0.5 \n",
      "μg/ml, N355/1, UC Davis/NIH NeuroMab Facility), anti-β-actin mouse monoclonal antibody (…\n",
      "\n",
      "… Each membrane was incubated with mouse anti-N terminus GluA1 (NeuroMab clone \n",
      "N355/1) and either anti-rabbit GluA1 phospho-Ser 831 (Millipore cat. no. 04-823), or anti-rabbit …\n",
      "\n",
      "10.7554/eLife.30640.001 AKT is a kinase regulating numerous cellular processes in the \n",
      "brain, and mutations in AKT are known to affect brain function. AKT is indirectly implicated in …\n",
      "\n",
      "BIN1, a member of the BAR adaptor protein family, is a significant late-onset Alzheimer \n",
      "disease risk factor. Here, we investigate BIN1 function in the brain using conditional knockout (…\n",
      "\n",
      "Tau-mediated toxicity is associated with cognitive decline and Alzheimer's disease (AD) \n",
      "progression. In particular, tau post-translational modifications (PTMs) are thought to generate …\n",
      "\n",
      "… a mouse anti-GluA1 antibody (#N355/1, RRID:AB2877405, 1:1,000; NeuroMab), a mouse \n",
      "anti-GluA2 antibody (#L21/32, RRID:AB2877267, 1:1,000; NeuroMab), a mouse anti-GluA3 (#…\n",
      "\n",
      "LRRK2 is a kinase expressed in striatal spiny projection neurons (SPNs), cells which lose \n",
      "dopaminergic input in Parkinson’s disease (PD). R1441C and G2019S are the most common …\n",
      "\n",
      "Exo70 is a subunit of the greater exocyst complex, a collection of proteins that oversees \n",
      "cellular membrane addition and polarized exocytosis by acting as a tethering intermediate …\n",
      "\n",
      "… The anti-GluN1 (clone N308/48) and anti-GluA1 (N355/1) monoclonal antibodies were \n",
      "obtained from the UC Davis/NIH NeuroMab Facility. …\n",
      "\n",
      "The Par3 polarity protein is critical for subcellular compartmentalization in different developmental \n",
      "processes. Variants of PARD3, which encodes PAR3, are associated with intelligence …\n",
      "\n",
      "Background Exo70 is a subunit of the greater exocyst complex, a collection of proteins that \n",
      "oversees cellular membrane addition and polarized exocytosis by acting as a tethering …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "… A monoclonal antibody raised against a peptide corresponding to the predicted N terminus \n",
      "of FHF2A (NeuroMab N235/22) could be shown to react with FHF1A, FHF2A, and FHF4A …\n",
      "\n",
      "… FHF (N235/22, Catalogue 75-246, NeuroMab) and pan-sodium channel (K58/35, Catalogue \n",
      "S8809, Sigma Aldrich) followed by secondary peroxidase-conjugated antibodies (1:10,000, …\n",
      "\n",
      "Nav1.6 and Nav1.6-mediated resurgent currents have been implicated in several pain \n",
      "pathologies. However, our knowledge of how fast resurgent currents are modulated in neurons is …\n",
      "\n",
      "… -iFGF13 mouse monoclonal antibody (NeuroMab clone N91/27, 1:300; Antibodies Incorporated), \n",
      "and the anti-Pan-iFGF-A mouse monoclonal antibody (NeuroMab clone N235/22, 1:300…\n",
      "\n",
      "Hyperactive Notch signalling is frequently observed in breast cancer and correlates with \n",
      "poor prognosis. However, relatively few mutations in the core Notch signalling pathway have …\n",
      "\n",
      "Scribble (Scrib) is a key regulator of apicobasal polarity, presynaptic architecture, and short-term \n",
      "synaptic plasticity in Drosophila. In mammals, its homolog Scrib1 has been implicated …\n",
      "\n",
      "We report a signaling pathway linking two fundamental functions of the ER, oxidative protein \n",
      "folding, and intracellular calcium regulation. Cells sense ER oxidative protein folding …\n",
      "\n",
      "Canonical transient receptor potential (TRPC) channels are plasma membrane cation \n",
      "channels included in the TRP superfamily. TRPC1 is expressed widely in the central nervous …\n",
      "\n",
      "Background: The intracellular ion channel type 1 inositol 1, 4, 5-trisphosphate receptor (IP 3 \n",
      "R1) releases Ca 2+ from the endoplasmic reticulum upon stimulation with IP 3. Perturbation …\n",
      "\n",
      "… (K39/25), anti-AMIGO or control antibodies from the same species. (A) AMIGO antibody \n",
      "coimmunoprecipitates Kv2.1 from brain samples of non-treated and CO 2 -treated mice. Notice …\n",
      "\n",
      "… F) Representative K V 2.1 immunoblot of Kcnb1 fs brain membrane protein using the anti-K \n",
      "V 2.1 K39/25 mAb. The anti-GRP75 mAb is used as a loading control. G) Quantification of the …\n",
      "\n",
      "… K39/25 dilution 1:500, NeuroMab, UC Davis). Membranes were then incubated with \n",
      "horseradish peroxidase-conjugated anti-IgG. Signal was detected by chemiluminescence (Pierce …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… ) using a monoclonal antibody specific for K V 2.1 subunits (NeuroMab; 75-159; clone \n",
      "K39/25, University of California, Davis). The secondary antibody was an Alexa Fluor 488-…\n",
      "\n",
      "… 1e or K39/25 antibodies after fixation and before detergent permeabilization. Our previous \n",
      "studies (Manganas and Trimmer, 2000; Tiffany et al., 2000; Manganas et al., 2001a,b; Shibata …\n",
      "\n",
      "The Amigo protein family consists of three transmembrane proteins characterized by six \n",
      "leucine-rich repeat domains and one immunoglobulin-like domain in their extracellular moieties. …\n",
      "\n",
      "KCNE5 is an X-linked gene encoding KCNE5, an ancillary subunit to voltage-gated \n",
      "potassium (KV) channels. Human KCNE5 mutations are associated with atrial fibrillation (AF)–and …\n",
      "\n",
      "The diversity of the voltage-gated K + (Kv) channel subfamily Kv2 is increased by interactions \n",
      "with auxiliary β-subunits and by assembly with members of the modulatory so-called silent …\n",
      "\n",
      "Light as a trigger is ideal to perturb biological function due to its high spatial and temporal \n",
      "precision. This is evidenced by the ever-expanding arsenal of genetically-encoded optical …\n",
      "\n",
      "… 1, we used mouse anti-RCOR1 (NeuroMab clone K72/8, diluted 1:1,000), rabbit anti-RCOR2 \n",
      "(in house-generated rabbit serum recognizing the N terminus of human RCOR2; 1:10,000), …\n",
      "\n",
      "… from NeuroMab recognizes a single band at 55 kD by western blot of the OE (Fig. 4h) and \n",
      "has been verified in CoREST transfected Cos-1 cells (see NeuroMab datasheet, clone K72/8). …\n",
      "\n",
      "The development of the cerebral cortex is a dynamic and coordinated process in which cell \n",
      "division, cell death, migration, and differentiation must be highly regulated to acquire the final …\n",
      "\n",
      "… Two monoclonal anti-CoREST1 antibodies were used: K72/8 from Neuromab and 612146 \n",
      "from BD Transduction Laboratories. Anti-HDAC1 (10E2), anti-HDAC2 (3F3), polyclonal anti…\n",
      "\n",
      "… anti-CoREST1 antibodies K72/8 and 612146 were obtained from Neuromab and BD \n",
      "Transduction Laboratories, respectively. Anti-HDAC1 (10E2), anti-HDAC2 (3F3), anti-CoREST3 (…\n",
      "\n",
      "Mammalian genomes harbor three CoREST genes. rcor1 encodes CoREST (CoREST1), \n",
      "and the paralogues rcor2 and rcor3 encode CoREST2 and CoREST3, respectively. Here, we …\n",
      "\n",
      "Latrophilin-2 (Lphn2) and latrophilin-3 (Lphn3) are adhesion GPCRs that serve as postsynaptic \n",
      "recognition molecules in CA1 pyramidal neurons of the hippocampus, where they are …\n",
      "\n",
      "… We provide examples from NeuroMab screens, and from the subsequent specialized … of \n",
      "NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs at the …\n",
      "\n",
      "… We provide examples from NeuroMab screens, and from the subsequent specialized … of \n",
      "NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs at the …\n",
      "\n",
      "Previously, we showed that a hierarchy of spectrin cytoskeletal proteins maintains nodal Na \n",
      "+ channels (Liu et al., 2020). Here, using mice lacking β1, β4, or β1/β4 spectrins, we show …\n",
      "\n",
      "… Mouse monoclonal antibodies anti-ankyrin R (clone N388A/10) and anti-AQP1 (clone 1/A5F6) \n",
      "were from Neuromab (UC Davis/NIH, USA) and Bio-Rad (Marnes-la-Coquette, France) …\n",
      "\n",
      "… Mouse monoclonal antibodies anti-ankyrin R (clone N388A/10) and anti-AQP1 (clone 1/A5F6) \n",
      "were from Neuromab (UC Davis/NIH, USA) and Bio-Rad (Marnesla-Coquette, France) …\n",
      "\n",
      "… (clone N65/37 obtained from NeuroMab Facility, University of California, Davis) followed by \n",
      "FITC-conjugated secondary antibodies. After washing with PBS, slides were mounted using …\n",
      "\n",
      "… Using a different TRPC 7 antibody (N64A/36; 100% homology to the rat sequence) we \n",
      "detected no staining of pre-glomerular vessels. A double staining of rat brain and renal sections …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… -2 and double immunolabeled with the progenitor N468/37 IgG1 mAb (green) and the N468/…\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… -2 and double immunolabeled with the progenitor N468/37 IgG1 mAb (green) and the N468/…\n",
      "\n",
      "Hyperpolarization-activated cyclic nucleotide–gated (HCN) channels are dually gated \n",
      "channels that are operated by voltage and by neurotransmitters via the cAMP system. cAMP-…\n",
      "\n",
      "… were all based on mAbs from NeuroMab (UC Davis/NIH NeuroMab Facility, nd). This facility \n",
      "… 2013), our current efforts to systematically convert NeuroMab mAbs into scFvs that retain the …\n",
      "\n",
      "Crucial in the pathogenesis of neurodegenerative diseases is the process of neuroinflammation \n",
      "that is often linked to the pro-inflammatory cytokines Tumor necrosis factor alpha (TNFα) …\n",
      "\n",
      "… The mouse anti-pan-neurofascin antibody (extNF; A12/18, Neuromab) was generated \n",
      "against the rat-specific extracellular domain common to NF155 and NF186 (amino acids 25-1110)…\n",
      "\n",
      "… A mouse monoclonal antibody against ankyrin-G was purchased from the UC Davis/NIH \n",
      "NeuroMab Facility (clone N106/43). Specificity was previously demonstrated by preadsorption …\n",
      "\n",
      "Spike timing-dependent synaptic plasticity (STDP) serves as a key cellular correlate of \n",
      "associative learning, which is facilitated by elevated attentional and emotional states involving …\n",
      "\n",
      "… clone N70/28 Fusion protein amino acids 778-910 at C-terminus of rat HCN1 NeuroMab, 75-… \n",
      "-1108 at C-terminus of rat HCN4 NeuroMab, 75-150 Mouse monoclonal AB_2248534 1:200 …\n",
      "\n",
      "… N70/28; NeuroMab), the densities of both bands were pooled for analysis as described \n",
      "elsewhere [25]. The density of HCN1 bands were normalized to β-actin, a prototypical loading …\n",
      "\n",
      "… The anti-HCN1 monoclonal antibody N70/28 was developed by and obtained from the \n",
      "University of California at Davis/NINDS/NIMH NeuroMab Facility, supported by NIH Grant …\n",
      "\n",
      "… Sections were then immunostained as described 51 using mouse antibodies against HCN1 \n",
      "(clone N70/28, NeuroMab), PanNav 52 , or Caspr (clone K65/35, NeuroMab), and rabbit …\n",
      "\n",
      "The development of the cerebral cortex relies on the controlled division of neural stem and \n",
      "progenitor cells. The requirement for precise spatiotemporal control of proliferation and cell …\n",
      "\n",
      "A specialized axonal ending, the basket cell “pinceau,” encapsulates the Purkinje cell axon \n",
      "initial segment (AIS), exerting final inhibitory control over the integrated outflow of the …\n",
      "\n",
      "Properties, developmental regulation, and cAMP modulation of the hyperpolarization-activated \n",
      "current (I h ) were investigated by the whole cell patch-clamp technique in vestibular …\n",
      "\n",
      "… Sections were then incubated for 48 h in anti-HCN1 antibodies (Neuromab) at a final protein \n",
      "… A monoclonal antibody against HCN1 (Clone N70/28) was obtained from the NeuroMab …\n",
      "\n",
      "Retinal bipolar cells are second-order neurons in the visual system, which initiate multiple \n",
      "image feature-based neural streams. Among more than ten types of bipolar cells, type-5 cells …\n",
      "\n",
      "Hypersensitivity in response to sensory stimuli and neocortical hyperexcitability are prominent \n",
      "features of Fragile X Syndrome (FXS) and autism spectrum disorders, but little is known …\n",
      "\n",
      "… The HCN1 results were obtained with a mouse monoclonal antibody directed against amino \n",
      "acids 778–910 (clone N70/28; NeuroMab) and an affinity-purified rabbit polyclonal antibody …\n",
      "\n",
      "Background: Ketamine is a commonly used anesthetic, but the mechanistic basis for its \n",
      "clinically relevant actions remains to be determined. The authors previously showed that HCN1 …\n",
      "\n",
      "Parallel visual pathways originate at the first synapse in the retina, where cones make \n",
      "connections with cone bipolar cells that in turn contact ganglion cells. There are more ganglion …\n",
      "\n",
      "… to HCN1 obtained from NeuroMab, a polyclonal antibody to HCN1 obtained from Alomone \n",
      "Labs and a monoclonal antibody to Ca V 3.2 obtained from Neuromab. To examine the …\n",
      "\n",
      "De novo mutations in voltage-and ligand-gated channels have been associated with an \n",
      "increasing number of cases of developmental and epileptic encephalopathies, which often fail to …\n",
      "\n",
      "In the primate visual system, the ganglion cells of the magnocellular pathway underlie motion \n",
      "and flicker detection and are relatively transient, while the more sustained ganglion cells of …\n",
      "\n",
      "… (88) The immunoblotting antibodies were mouse HCN1 (UC Davis/NIH NeuroMab Facility, \n",
      "clone N70/28), anti-ERK2 (mouse scbt #sc-1647), and anti-Akt (mouse scbt #sc-5298). The …\n",
      "\n",
      "Visual (and probably also magnetic) signal processing starts at the first synapse, at which \n",
      "photoreceptors contact different types of bipolar cells, thereby feeding information into different …\n",
      "\n",
      "… The monoclonal antibodies HCN1 (Clone N70/28) were developed by and obtained from \n",
      "the UC Davis/NIH NeuroMab Facility, supported by NIH Grant U24NS050606, and maintained …\n",
      "\n",
      "… The HCN subtypes were detected using the following antibodies: HCN1 (mouse monoclonal \n",
      "antibody, dilution 1:100, Cat# N70-28, UC Davis/NIH NeuroMab Facility), HCN2 (rabbit …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… The Kv1.2 antibody is knockout verified by the company NeuroMab. The specificity of this \n",
      "antibody in gerbils can be derived from its precise labeling in the hemi-nodes of MSO axons (…\n",
      "\n",
      "… cells by clone L55/54, with antibody L36/12 serving as a positive control for CNGA3 … L36/12, \n",
      "which detects both CNGA1 and CNGA3, was obtained from the UC Davis/NIH NeuroMab …\n",
      "\n",
      "… Neuromab N231B/34, which was raised against a synthetic peptide corresponding to amino \n",
      "acids 841–960 did not detect recombinant LRRK2-(883–1300), suggesting that the epitope …\n",
      "\n",
      "… (a) Cells were stained with specific antibodies against FAK and LRRK2 (N231B/34). \n",
      "Magnified images of boxed areas in 4–6 and 13–15 were shown in 7–9 and 16–18, respectively. …\n",
      "\n",
      "… Interestingly, a different picture was observed using anti-LRRK2 N231B/34, a high affinity \n",
      "antibody against human LRRK2 [26]. This latter antibody showed a 3–4 fold increase in human …\n",
      "\n",
      "Mutations in the Leucine-rich repeat kinase 2 gene (LRRK2) are associated with familial \n",
      "Parkinson’s disease (PD). LRRK2 protein contains several functional domains, including protein…\n",
      "\n",
      "Lrrk2, a gene linked to Parkinson’s disease, encodes a large scaffolding protein with kinase \n",
      "and GTPase activities implicated in vesicle and cytoskeletal-related processes. At the …\n",
      "\n",
      "… Snap frozen-cryosections were prepared and stained with a mouse monoclonal Kv1.3 \n",
      "antibody (1:50 dilution, clone: L23/27, NeuroMab) and a mouse monoclonal CD3 antibody (1:100 …\n",
      "\n",
      "Introduction K + channels are central to vascular pathophysiology. Previous results demonstrated \n",
      "that phenotypic modulation associates with a change in Kv1.3 to Kv1.5 expression, …\n",
      "\n",
      "… h) and treated with antibodies: Anti-Kv1.3 clone L23/27 mouse monoclonal primary antibody \n",
      "(1:500; cat no. 75-009, NeuroMab, USA); Anti-COX IV mouse monoclonal primary antibody (…\n",
      "\n",
      "K+ channels are central to vascular pathophysiology. Previous results demonstrated that \n",
      "phenotypic modulation associates with a change in Kv1.3 to Kv1.5 expression, and that Kv1.3 …\n",
      "\n",
      "Microglia and CNS-infiltrating monocytes/macrophages (CNS-MPs) perform pro-inflammatory \n",
      "and protective anti-inflammatory functions following ischemic stroke. Selective inhibition of …\n",
      "\n",
      "… (NeuroMab) recognized a single protein band at 70 kDa on western blot of lysates from \n",
      "Kv1.5-transfected COS cells. No band of equivalent size is observed from rat brain samples and …\n",
      "\n",
      "Kv1.3 channels are known to modulate many aspects of neuronal function. We tested the \n",
      "hypothesis that Kv1.3 modulates the function of postganglionic sympathetic neurons. RT-PCR, …\n",
      "\n",
      "There is a great need for pharmacological approaches to enhance neural progenitor cell (NPC) \n",
      "function particularly in neuroinflammatory diseases with failed neuroregeneration. In …\n",
      "\n",
      "… Anti-Kv1.2 antibody was obtained from NeuroMab. Kv1.2 has a calculated MW of 56.7 kDa \n",
      "but may run at 66–88 kDa depending on the degree of posttranslational modification. On …\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "… (L23/27), and Kv1.5 (K7/45) were purchased from Neuromab, and anti–β-actin Abs were \n",
      "purchased from Bioeasy. Digital images were captured using the Amersham Image 600 system …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Numerous studies have documented the effects of chronic N-methyl-D-aspartate receptor (NMDAR) \n",
      "blockade on excitatory circuits, but the effects on inhibitory circuitry are not well …\n",
      "\n",
      "The mechanism underlying a hypercholinergic state in Parkinson's disease (PD) remains \n",
      "uncertain. Here, we show that disruption of the K v 1 channel-mediated function causes …\n",
      "\n",
      "… mice for specificity 105 (NeuroMab). In addition, we used a monoclonal anti-vesicular \n",
      "glutamate transporter 106 antibody (VGlut2, NeuroMab clone N29/29, fusion protein amino acids …\n",
      "\n",
      "… mice for specificity (NeuroMab). In addition, we used a monoclonal anti-vesicular glutamate \n",
      "transporter antibody (clone N29/29, fusion protein amino acids 501–582, NeuroMab), a goat …\n",
      "\n",
      "nan\n",
      "\n",
      "The type of neurotransmitter secreted by a neuron is a product of the vesicular transporters \n",
      "present on its synaptic vesicle membranes and the available transmitters in the local cytosolic …\n",
      "\n",
      "Tonotopic differentiations of ion channels ensure sound processing across frequencies. \n",
      "Afferent input plays a critical role in differentiations. We demonstrate here in organotypic culture …\n",
      "\n",
      "Glutamate and its receptor N-methyl-D-aspartate receptor (NMDAR) have been associated \n",
      "with cancer, although their functions are not fully understood. Herein, we implicate glutamate-…\n",
      "\n",
      "… We found the Kv8.2 antibody (Clone N448/88 from NeuroMab) generated a non-specific … \n",
      "monoclonal antibody project and identified clone N448/50 as specific in both western blots and …\n",
      "\n",
      "… of NeuroMabs as a method to archive all NeuroMabs, to … K28/43 and anti-KCC2 NeuroMab \n",
      "N1/12) that exhibit … 1b NeuroMab N16B/8 and anti-HCN2 NeuroMab N71/37) that exhibit …\n",
      "\n",
      "Epileptiform neuronal activity during seizures is observed in many brain areas, but its origins \n",
      "following status epilepticus (SE) are unclear. We have used the Li low‐dose pilocarpine rat …\n",
      "\n",
      "… NeuroMab clone N1/12 (1:1,000), anti-NKCC1 (NeuroMab clone N161/20, 1:1,000) at 4C \n",
      "overnight. Membranes were washed in TBST and transferred into a solution containing …\n",
      "\n",
      "Songbirds learn to produce vocalizations early in life by listening to, then copying the songs \n",
      "of conspecific males. The anterior forebrain pathway, homologous to a basal ganglia‐…\n",
      "\n",
      "Purpose WNK3 kinase (PRKWNK3) has been implicated in the development and function of \n",
      "the brain via its regulation of the cation-chloride cotransporters, but the role of WNK3 in …\n",
      "\n",
      "γ-Aminobutyric acid (GABA) is the main inhibitory neurotransmitter in the mature brain but \n",
      "has the paradoxical property of depolarizing neurons during early development. …\n",
      "\n",
      "… After incubation of ∼1 mg total proteins with 5 μl of undiluted monoclonal mouse anti-KCC2 \n",
      "(clone N1/12) antibodies (Neuromab) on a rotator (overnight; 4 C), KCC2 protein was …\n",
      "\n",
      "… Cells were then incubated with primary antibody N1/12 (NeuroMab, Davis, USA), diluted \n",
      "1:500 in carrier solution (0.3% Triton X-100, 1% bovine serum albumin, 1% goat serum in PBS) …\n",
      "\n",
      "… The specificity of the mouse monoclonal KCC2pan antibody (clone N1/12) was established \n",
      "by double staining together with the rabbit KCC2pan antibody, which produced identical …\n",
      "\n",
      "… Antibodies—Monoclonal mouse anti-KCC2 antibody clone N1/12 was purchased from \n",
      "the University of California Davis/NINDS/NIMH NeuroMab facility. Polyclonal rabbit anti-KCC2 …\n",
      "\n",
      "During neuronal maturation, the neuron‐specific K–Cl co‐transporter KCC2 lowers the \n",
      "intracellular chloride and thereby renders GABAergic transmission hyperpolarizing. …\n",
      "\n",
      "… Primary antibody solution (anti-KCC2 N1-12; 1:1,000; Neuromab, Carlifornia, USA) was \n",
      "added in carrier solution (0.3% Triton X-100, 1% bovine serum albumin, 1% goat serum in PBS) …\n",
      "\n",
      "Intracellular chloride concentration ([Cl − ] i ) in pancreatic β-cells is kept above electrochemical \n",
      "equilibrium due to the predominant functional presence of Cl − loaders such as the Na + …\n",
      "\n",
      "Huntington Disease (HD) is an inherited neurodegenerative disease caused by a CAG \n",
      "repeat expansion in the gene encoding the huntingtin protein. The resultant mutant huntingtin …\n",
      "\n",
      "… The pan-KCC2 antibody (NeuroMab clone N1/12) was from NeuroMab. The FLAG M2 \n",
      "antibody (1 µg/ml, F3165) was from Sigma. The KCC4 antibody was from Novus Biologicals (0.4 …\n",
      "\n",
      "Key points With daily electrophysiological recordings and neurochemical analysis, we uncovered \n",
      "a transient period of synaptic imbalance between enhanced inhibition and suppressed …\n",
      "\n",
      "Synaptic inhibition depends on a transmembrane gradient of chloride, which is set by the \n",
      "neuron-specific K + -Cl − co-transporter KCC2. Reduced KCC2 levels in the neuronal …\n",
      "\n",
      "… The antibodies described in the paper are available from NeuroMab (antiKCC2) or … Mouse \n",
      "anti-KCC2 was purchased from NeuroMab and used at 1/500. Mouse anti actin was from …\n",
      "\n",
      "10.7554/eLife.28270.001 KCC2 is a neuron-specific K + -Cl – cotransporter essential for \n",
      "establishing the Cl - gradient required for hyperpolarizing inhibition in the central nervous …\n",
      "\n",
      "10.7554/eLife.20770.001 Neocortical GABAergic interneuron migration and thalamo-cortical \n",
      "axon (TCA) pathfinding follow similar trajectories and timing, suggesting they may be …\n",
      "\n",
      "There is increasing appreciation for the role of the neurovascular unit in neurodegenerative \n",
      "diseases. We showed previously that the angiogenic and neurotrophic cytokine, vascular …\n",
      "\n",
      "Spinocerebellar ataxia type 1 (SCA1) is an adult-onset neurodegenerative disease caused \n",
      "by a polyglutamine expansion in the protein ATXN1, which is involved in transcriptional …\n",
      "\n",
      "… from a P7 mouse spinal cord immunostained for AnkB (N105/13), AnkG (N106/36) and βIV \n",
      "… membrane-binding domain (N105/13 and N105/17, UC Davis/NIH NeuroMab Facility) (1:200 …\n",
      "\n",
      "Na + /K + ‐ ATPase ( NKA ) participates in setting electrochemical gradients, cardiotonic \n",
      "steroid signaling and cellular adhesion. Distinct isoforms of NKA are found in different tissues …\n",
      "\n",
      "… conducted using monoclonal antibodies from the UC Davis/NIH NeuroMab Facility (University \n",
      "of California, Davis, CA 95616, USA), clones N105/13 and N106/20, respectively. F-actin …\n",
      "\n",
      "The scaffolding protein ankyrin-G is required for Na + channel clustering at axon initial \n",
      "segments. It is also considered essential for Na + channel clustering at nodes of Ranvier to …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Spike timing-dependent synaptic plasticity (STDP) serves as a key cellular correlate of \n",
      "associative learning, which is facilitated by elevated attentional and emotional states involving …\n",
      "\n",
      "Lysosomal positioning and mTOR (mammalian target of rapamycin) signaling coordinate \n",
      "cellular responses to nutrient levels. Inadequate nutrient sensing can result in growth delays, a …\n",
      "\n",
      "The axon initial segment (AIS) plays a key role in initiation of action potentials and neuronal \n",
      "output. The plasma membrane of the AIS contains high densities of voltage‐gated ion …\n",
      "\n",
      "… .2 amino acids 428–499) used to generate K14/16, and found to bind within the same \n",
      "peptide (rat Kv1.2 amino acids 463–480) as K14/16. The generation and validation of anti-PSD-95 …\n",
      "\n",
      "… 2P and the phosphoindependent mouse mAb K14/16 showed strong staining of … K14/16-associated \n",
      "pool present in those cells. Cells that exhibited only intracellular/ perinuclear K14/16 …\n",
      "\n",
      "… ; NeuroMab, K14/16), and standardized using β-Actin (1:2000; Sigma, A5316) or α/β-Tubulin \n",
      "(1:1000, Cell Signaling Technology, #2148). After washing in TBS-T, blots were exposed to …\n",
      "\n",
      "Peripheral myelin protein 2 (Pmp2, P2 or Fabp8), a member of the fatty acid binding protein \n",
      "family, was originally described together with myelin basic protein (Mbp or P1) and myelin …\n",
      "\n",
      "… K14/16, mouse monoclonal anti-K v 1.2 is described in detail in Bekele-Arcuri et al. (… \n",
      "K14/16 can be obtained from the UC Davis/NINDS/NIMH NeuroMab facility (http://www.neuromab…\n",
      "\n",
      "… (clone K14/16, 1:500; NeuroMab), antigen retrieval was performed as described previously \n",
      "[65]. Briefly, sections were treated with 0.8% sodium borohydride, washed with PBS then …\n",
      "\n",
      "Nodes of Ranvier and associated paranodal and juxtaparanodal domains along myelinated \n",
      "axons are essential for normal function of the peripheral and central nervous systems. …\n",
      "\n",
      "… (extracellular epitope, 1/100, NeuroMab K36/15, IgG2b) and mouse anti-Kv1.2 (1/100, \n",
      "NeuroMab K14/16, IgG2b). The secondary antibodies used were goat anti-mouse IgG2a-Alexa-…\n",
      "\n",
      "… (K14/16) and anti-Kv1.4 (K13/31) mouse monoclonal antibodies used in this study are \n",
      "available from NeuroMab (http://www.neuromab.org), a not-for-profit supplier of monoclonal …\n",
      "\n",
      "… Jose, California), ankyrinG (N106/36, UC Davis/NIH NeuroMab Facility, Davis, California), \n",
      "Kv1.2 (K14/16, NeuroMab), rabbit antibodies against contactin-associated protein (Caspr), βIV …\n",
      "\n",
      "… (K7/45) and Mortalin/Grp75 (N52A/42) using monoclonal antibodies (clone number in \n",
      "parentheses) obtained from the UC Davis/NIH NeuroMab Facility. Alexa-conjugated fluorescent …\n",
      "\n",
      "… Ms 75-008 Clone K14/16 413-7RR-30 Antibody produced against bacterially expressed … \n",
      "Anti-Kv1.2 antibody was obtained from NeuroMab. Kv1.2 has a calculated MW of 56.7 kDa but …\n",
      "\n",
      "Kv1.2 is a prominent voltage-gated potassium channel that influences action potential \n",
      "generation and propagation in the central nervous system. We explored multi-protein complexes …\n",
      "\n",
      "… Sections were immunolabelled with 1:1000 anti-Kv1.2 potassium channel subunit antibody \n",
      "(NeuroMab clone K14/16) in PBS with a standard block solution for at least 6 h at room …\n",
      "\n",
      "Recent studies have reported that delayed-rectifier Kv channels regulate apoptosis in the \n",
      "nervous system. Herein, we investigated changes in the expression of the delayed-rectifier Kv …\n",
      "\n",
      "… The mouse K v 1.2 (clone K14/16, 1: 500) and Na v 1.6 (clone K87A/10, 1:500) antibodies \n",
      "were purchased from the University of California, Davis NINDS/NIMH NeuroMab Facility. The …\n",
      "\n",
      "… 1e mAb (K36/15; UC Davis/NIH NeuroMab Facility) before detergent permeabilization to … \n",
      "mAbs K14/16 (anti-Kv1.2) or K20/78 (anti-Kv1.1), all from the UC Davis/NIH NeuroMab Facility, …\n",
      "\n",
      "… (K14/16) and PSD-95 (K28/43) were obtained from the UC Davis/NIH NeuroMab Facility. \n",
      "The rabbit polyclonal antibody to Kv2.1 was obtained from Sigma-Aldrich (P9607). …\n",
      "\n",
      "The development of simple molecular assays with membrane protein receptors in a native \n",
      "conformation still represents a challenging task. Exosomes are extracellular vesicles which, …\n",
      "\n",
      "… For co-localization experiments, we used mouse monoclonal anti-Kv1.2 antibodies (clone \n",
      "K14/16; 1:1000 dilution; UC Davis/NIH NeuroMab Facility) and rabbit polyclonal anti-KCNQ2 …\n",
      "\n",
      "… blotting was performed using a mouse antibody to K V 1.2 (NeuroMab clone K14/16). \n",
      "Water-injected oocytes and untransfected and water-transfected CHO cells were used as controls. …\n",
      "\n",
      "Every cell expresses a molecularly diverse surface glycan coat (glycocalyx) comprising its \n",
      "interface with its cellular environment. In vertebrates, the terminal sugars of the glycocalyx are …\n",
      "\n",
      "Fluorescent dextran tracers of varying sizes have been used to assess paranodal permeability \n",
      "in myelinated sciatic nerve fibers from control and three “myelin mutant” mice, Caspr‐null, …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… 2K + channel hybridoma-generated K14/16 mAb (IgG2b) (left section) and the K14/16 scFv (…\n",
      "\n",
      "… These mAbs are available from the UC Davis/ NIH NeuroMab Facility (neuromab.ucdavis.… \n",
      "(determined by staining with mouse mAb K14/16) and Kv1.4 (determined by staining with …\n",
      "\n",
      "… Sections were then incubated in primary antibodies from NeuroMab (UC Davis/NIH \n",
      "NeuroMab facility): mouse monoclonal anti-Kv1.1 (K36/15) and anti-Kv1.2 (K14/16), both IgG 2b …\n",
      "\n",
      "Precise timing of synaptic inputs is a fundamental principle of neural circuit processing. The \n",
      "temporal precision of postsynaptic input integration is known to vary with the computational …\n",
      "\n",
      "Cerebrospinal fluid samples collected from children during initial presentation of central \n",
      "nervous system inflammation, who may or may not subsequently be diagnosed as having …\n",
      "\n",
      "Action potential timing is fundamental to information processing; however, its determinants \n",
      "are not fully understood. Here we report unexpected structural specializations in the Ranvier …\n",
      "\n",
      "… Kv1.2 was detected using a monoclonal mouse Kv1.2 antibody (NeuroMAb, clone #K14/16 \n",
      "75–008), HRP-conjugated goat anti-mouse antibody (Applied Biological Materials, …\n",
      "\n",
      "Neuron-glia interactions establish functional membrane domains along myelinated axons. \n",
      "These include nodes of Ranvier, paranodal axoglial junctions and juxtaparanodes. …\n",
      "\n",
      "Changes to the structure of nodes of Ranvier in the normal-appearing white matter (NAWM) \n",
      "of multiple sclerosis (MS) brains are associated with chronic inflammation. We show that the …\n",
      "\n",
      "… years, including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection \n",
      "… K + channel hybridoma-generated K14/16 mAb (IgG2b) (left section) and the K14/16 scFv (…\n",
      "\n",
      "Contactin-associated protein-like 2 (Caspr2), also known as CNTNAP2, is a cell adhesion \n",
      "molecule that clusters voltage-gated potassium channels (Kv1.1/1.2) at the juxtaparanodes of …\n",
      "\n",
      "The consequences of dysmyelination are poorly understood and vary widely in severity. \n",
      "The shaking mouse, a quaking allele, is characterized by severe central nervous system (CNS) …\n",
      "\n",
      "The enormous variability in electrical properties of neurons is largely affected by a multitude \n",
      "of potassium channel subunits. Kv2.1 is a widely expressed voltage-dependent potassium …\n",
      "\n",
      "Actin, spectrin, and associated molecules form a membrane-associated periodic skeleton (MPS) \n",
      "in neurons. The molecular composition and functions of the MPS remain incompletely …\n",
      "\n",
      "De novo mutations in voltage-and ligand-gated channels have been associated with an \n",
      "increasing number of cases of developmental and epileptic encephalopathies, which often fail to …\n",
      "\n",
      "… , and anti-Kv1.4 antibodies (clone #: K14/16, K17/70, and K13/31, respectively; UC Davis/NIH … \n",
      "We thank University of California, Davis-NIH NeuroMab facility (a nonprofit supplier) for …\n",
      "\n",
      "Autoantibodies against CASPR2 (contactin-associated protein-like 2) have been linked to \n",
      "autoimmune limbic encephalitis that manifests with memory disorders and temporal lobe …\n",
      "\n",
      "Protein kinase M zeta (PKM-ζ), a constitutively active N-terminal truncated form of PKC-ζ, \n",
      "has long been implicated in a cellular correlate of learning, long-term potentiation (LTP). …\n",
      "\n",
      "… Kv1.2 was detected using a mouse monoclonal Kv1.2 antibody (1:2,000 dilution, clone \n",
      "K14/16 75-008; NeuroMab) and HRP-conjugated goat anti-mouse antibody (1:20,000 dilution, …\n",
      "\n",
      "Cortical dysplasia (CD) is a common cause for intractable epilepsy. Hyperactivation of the \n",
      "mechanistic target of rapamycin (mTOR) pathway has been implicated in CD; however, the …\n",
      "\n",
      "… K+ channel, clone K14/16 and anti-Caspr, clone K65/35 were obtained from the UC Davis/NINDS/NIMH \n",
      "NeuroMab Facility, supported by NIH grant U24NS050606 and maintained by …\n",
      "\n",
      "A specialized axonal ending, the basket cell “pinceau,” encapsulates the Purkinje cell axon \n",
      "initial segment (AIS), exerting final inhibitory control over the integrated outflow of the …\n",
      "\n",
      "Abstract Changes to the structure of nodes of Ranvier in the normal-appearing white matter (NAWM) \n",
      "of MS brains are associated with chronic inflammation. We show that the paranodal …\n",
      "\n",
      "Damage to long axons in white matter tracts is a major pathology in closed head traumatic \n",
      "brain injury (TBI). Acute TBI treatments are needed that protect against axon damage and …\n",
      "\n",
      "… the supplier's (NeuroMab) datasheets online. The protein bands detected match with the \n",
      "expected sizes from endogenous and heterologous expression as reported by NeuroMab. We …\n",
      "\n",
      "… Additional commercial available mouse monoclonal antibodies from NeuroMab (Anti-Kv1.1: \n",
      "K20/78, Anti-Kv1.2: K14/16, Anti-Kv1.4: K13/31 and Anti-Kv1.6: K19/36) were used. …\n",
      "\n",
      "… Other commercial antibodies used were as follows: goat anti-human contactin-1 polyclonal \n",
      "antibodies (R&D Systems), mouse monoclonal antibody K14/16 against the K v 1.2 α-subunit …\n",
      "\n",
      "… (K14/16; Neuromab) and rabbit polyclonal α-glyceraldehyde-3-phosphate dehydrogenase \n",
      "(… Parallel experiments compared α-Kvβ2 (Neuromab 75–021) labeling in slices fixed under …\n",
      "\n",
      "… were all based on mAbs from NeuroMab (UC Davis/NIH NeuroMab Facility, nd). This facility \n",
      "… 2013), our current efforts to systematically convert NeuroMab mAbs into scFvs that retain the …\n",
      "\n",
      "… (subtype IgG 2b ; clone K14/16) and Caspr (subtype IgG 1 ; clone K65/35) were obtained \n",
      "from the UC Davis/NIH NeuroMab Facility. Mouse monoclonal antibodies were detected by the …\n",
      "\n",
      "… In collaboration with the NIH-sponsored monoclonal antibody resource at University of \n",
      "California Davis, NeuroMab, we also generated mouse monoclonal antibodies targeting Na v β4. …\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "The exact site of initiation and shape of action potentials vary among different neuronal types. \n",
      "The reason for this variability is largely unknown, but the subunit composition, density and …\n",
      "\n",
      "Background Myelinated axons are organized into distinct subcellular and molecular regions. \n",
      "Without proper organization, electrical nerve conduction is delayed, resulting in detrimental …\n",
      "\n",
      "Action potential conduction along myelinated axons depends on high densities of voltage-gated \n",
      "Na + channels at the nodes of Ranvier. Flanking each node, paranodal junctions (…\n",
      "\n",
      "Spiral ganglion neurons (SGNs) relay acoustic code from cochlear hair cells to the brainstem, \n",
      "and their stimulation enables electrical hearing via cochlear implants. Rapid adaptation, a …\n",
      "\n",
      "The leucine-rich glioma-inactivated (LGI) family consists of four highly conserved paralogous \n",
      "genes, LGI1-4, that are highly expressed in mammalian central and/or peripheral nervous …\n",
      "\n",
      "The pathogenesis of peripheral neuropathies in adults is linked to maintenance mechanisms \n",
      "that are not well understood. Here, we elucidate a novel critical maintenance mechanism …\n",
      "\n",
      "The vast complexity of native heteromeric K + channels is largely unexplored. Defining the \n",
      "composition and subunit arrangement of individual subunits in native heteromeric K + …\n",
      "\n",
      "Numerous studies have documented the effects of chronic N-methyl-D-aspartate receptor (NMDAR) \n",
      "blockade on excitatory circuits, but the effects on inhibitory circuitry are not well …\n",
      "\n",
      "… The specific recognition of K v 1.1/K v 1.2 proteins by K36/15 or K14/16 antibody was \n",
      "confirmed by using K v 1.1 −/− or K v 1.2 −/− mice which serve as negative controls (Lorincz and …\n",
      "\n",
      "… (clone K14/16), anti-Kv1.4 (clone K13/31), anti-ADAM22 (clone N46/30), and anti-ankyrinG \n",
      "(clone N106/36) mAbs were obtained from NeuroMab (UC Davis/NIH NeuroMab Facility). …\n",
      "\n",
      "The interaction between myelinating Schwann cells and the axons they ensheath is mediated \n",
      "by cell adhesion molecules of the Cadm/Necl/SynCAM family. This family consists of four …\n",
      "\n",
      "… (K14/16; RRID:AB_2296313) antibodies were purchased from the University of California \n",
      "at Davis/National Institutes of Health NeuroMab Facility (Davis, CA). The pan-Na + channel …\n",
      "\n",
      "… (K14/16; RRID:AB_2296313) antibodies were purchased from the University of California–Davis/NIH \n",
      "NeuroMab Facility. Rabbit polycolonal antibodies were as follows: βI-spectrin (…\n",
      "\n",
      "What percentage of the protein function is required to prevent disease symptoms is a \n",
      "fundamental question in genetic disorders. Decreased transsynaptic LGI1-ADAM22 protein …\n",
      "\n",
      "Autosomal dominant lateral temporal epilepsy (ADLTE) is an inherited syndrome caused by \n",
      "mutations in the leucine-rich glioma inactivated 1 (LGI1) gene. It is known that glutamatergic …\n",
      "\n",
      "Myelinated neurons are organized into distinct molecular domains that are essential for \n",
      "electrical conduction. These domains include the axon initial segment, nodes of Ranvier, …\n",
      "\n",
      "… V 1.1 (K20/78), mouse anti-K V 1.2 (K14/16), and mouse anti-PSD95 (K28/43) were obtained \n",
      "from University of California, Davis/NIH NeuroMab Facility. Secondary antibodies used for …\n",
      "\n",
      "The axon initial segment (AIS) is a highly specialized axonal compartment where the action \n",
      "potential is initiated. The heterogeneity of AISs has been suggested to occur between …\n",
      "\n",
      "Cell adhesion proteins of the Cadm (SynCAM/Necl) family regulate myelination and the \n",
      "organization of myelinated axons. In the peripheral nervous system (PNS), intercellular contact …\n",
      "\n",
      "The evolution of myelin was a major key event in vertebrates which aimed to enhance \n",
      "conduction velocity of electrical impulses in axons. Distinct myelination patterns along axons can …\n",
      "\n",
      "At high-conductance state, a neuron is exposed to increased synaptic bombardment. We \n",
      "analyzed the ability of hippocampal pyramidal neurons, as well as the principle cell type of the …\n",
      "\n",
      "Schizophrenia is associated with cognitive and behavioral dysfunctions thought to reflect \n",
      "imbalances in neurotransmission systems. Recent screenings suggested that lack of (functional…\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "The family of Shank scaffolding molecules (comprising Shank1, 2 and 3) are core components \n",
      "of the postsynaptic density ( PSD ) in neuronal synapses. Shanks link surface receptors to …\n",
      "\n",
      "Shank and GKAP are scaffold proteins and binding partners at the postsynaptic density (PSD). \n",
      "The distribution and dynamics of Shank and GKAP were studied in dissociated …\n",
      "\n",
      "… Mouse monoclonal antibodies against Shank1 (clone N22/21, used at 1:50-100 for microscopy \n",
      "and 1:500 for Western), against Shank2 (clone N23B/6, used at 1:200 for microscopy and …\n",
      "\n",
      "(論文の内容の要旨)[背景と目的] シナプス後肥厚部 (postsynaptic density, PSD) とはシナプス膜\n",
      "直下にある, 神経細胞に高度に分化した特殊な細胞骨格系で, 極めて精巧なタンパク質複合体である…\n",
      "\n",
      "It is essential to study the molecular architecture of post‐synaptic density ( PSD ) to understand \n",
      "the molecular mechanism underlying the dynamic nature of PSD , one of the bases of …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "Shank3 is a postsynaptic density (PSD) scaffold protein of the Shank family. Here we use \n",
      "pre-embedding immunogold electron microscopy to investigate factors influencing the …\n",
      "\n",
      "Synaptic plasticity requires remodeling of the actin cytoskeleton. Although two actin isoforms, \n",
      "β- and γ-actin, are expressed in dendritic spines, the specific contribution of γ-actin in the …\n",
      "\n",
      "At most synapses in the brain, short-term plasticity dynamically modulates synaptic strength. \n",
      "Rapid frequency-dependent changes in synaptic strength have key roles in sensory …\n",
      "\n",
      "When an action potential invades a presynaptic terminal it evokes large, brief Ca 2+ signals \n",
      "that trigger vesicle fusion within milliseconds that is followed by a small residual Ca 2+ (Ca …\n",
      "\n",
      "Autapses are self-synapses of a neuron. Inhibitory autapses in the neocortex release GABA \n",
      "in 2 modes, synchronous release and asynchronous release (AR), providing precise and …\n",
      "\n",
      "Neurotransmitter release can be synchronous and occur within milliseconds of action \n",
      "potential invasion, or asynchronous and persist for tens of milliseconds. The molecular …\n",
      "\n",
      "The timing and size of inhibition are crucial for dynamic excitation-inhibition balance and \n",
      "information processing in the neocortex. The underlying mechanism for temporal control of …\n",
      "\n",
      "Synaptotagmin 7 (Syt7) is a high-affinity calcium sensor that is implicated in multiple aspects \n",
      "of synaptic transmission. Here, we study the influence of Syt7 on the climbing fiber (CF) to …\n",
      "\n",
      "… The membrane was incubated with the primary antibodies: Mmachc monoclonal antibody \n",
      "(NeuroMab clone N230/21), (1:2500); and β-actin antibody (Cell Signaling), (1:5000); and with …\n",
      "\n",
      "… ; NeuroMab) or anti-Slack (1:3,000, clone N3/26; NeuroMab) antibody for 2.5 hours at \n",
      "room temperature. Membranes were incubated with HRP-labeled goat anti-mouse IgG1 (1:150,000…\n",
      "\n",
      "… monoclonal Slack antibody (clone N3/26, dilution 1 : 50; NeuroMab, Davis, CA, USA) and \n",
      "mouse monoclonal Slick antibody (clone N11/33, dilution 1 : 50; NeuroMab). We also used …\n",
      "\n",
      "… For each experiment, 40 µg of either anti-Slick (clone N11/33, Neuromab) or anti-Slack (clone \n",
      "N3/26, Neuromab) channel antibody was used and 2 mg of synaptic plasma membranes …\n",
      "\n",
      "The role of the Slack (also known as Slo2.2, KNa1.1, or KCNT1) channel in pain-sensing is \n",
      "still in debate on which kind of pain it regulates. In the present study, we found that the Slack-/…\n",
      "\n",
      "… The anti-Slack antibody we used for immunostaining (N3/26 NeuroMab produced by \n",
      "University of California Davis/NIH NeuroMab Facility, Davis, CA) was raised against an antigen …\n",
      "\n",
      "… Middle, brain membrane proteins were separated and probed with N3/26 anti-Slo2.2 \n",
      "mAb (… N3/26, were developed by and obtained from the UC Davis/NIH NeuroMab Facility, …\n",
      "\n",
      "… N11/33, UC Davis/NIH NeuroMab Facility) and Slack (Anti-Slo2.2 sodium-activated potassium \n",
      "channel, clone N3/26, UC Davis/NIH NeuroMab Facility). For the immunocytochemical …\n",
      "\n",
      "… TMEM16C polyclonal antibodies (1:1,000), mouse anti-tubulin (T9026, Sigma, 1:2,000) and \n",
      "mouse anti-Slack (1:1,000, Neuromab) were diluted in blocking buffer and incubated for 2 h …\n",
      "\n",
      "… The PVDF membrane was incubated with an antibody against Kcnt1 protein (anti-KCNT1 \n",
      "mse monoclonal [N3/26] 8 μg/mL, NeuroMab catalog 75-051), anti–β-actin (Mse IgG1 …\n",
      "\n",
      "KCNT1 encodes the sodium-activated potassium channel Slack (KCNT1, KNa1.1), an \n",
      "important mediator of neuronal membrane excitability. Gain-of-function (GOF) mutations in …\n",
      "\n",
      "Anxiety disorders are a series of mental disorders characterized by anxiety and fear, but the \n",
      "molecular basis of these disorders remains unclear. In the present study, we find that the …\n",
      "\n",
      "… anti-iFGF13 mouse monoclonal antibody (NeuroMab clone N91/27, 1:300; Antibodies \n",
      "Incorporated), and the anti-Pan-iFGF-A mouse monoclonal antibody (NeuroMab clone N235/22, 1:…\n",
      "\n",
      "… primary ANO5 mAb N421A/85 (UC Davis/NIH NeuroMab, Davis, CA… With mAb clone N421A/85, \n",
      "COS-1 cells expressing the … The mAb N421A/85 binding site may be located downstream …\n",
      "\n",
      "… A Western blot study using monoclonal antibody clone N421A/85 (UC Davis/NIH NeuroMab, \n",
      "… Using the same monoclonal antibody clone N421A/85, Xu and colleagues showed the …\n",
      "\n",
      "The pathogenic role of intronic variants is generally difficult to assess, except for those near \n",
      "known splice sites for which aberrant splicing is suspected, although deeper intronic variants …\n",
      "\n",
      "… The membrane was blocked for 1 h in 5% non-fat dry milk and stained over night at 4C with \n",
      "Anti-Ano5 antibody (NeuroMab clone N421A/85) at a 1:100 dilution in PBST + 1% milk. HRP …\n",
      "\n",
      "Endothelial cells (ECs) normally form an anticoagulant surface under physiological conditions, \n",
      "but switch to support coagulation following pathogenic stimuli. This switch promotes …\n",
      "\n",
      "Anoctamin 5 (ANO5) is a membrane protein belonging to the TMEM16/Anoctamin family \n",
      "and its deficiency leads to the development of limb girdle muscular dystrophy R12 (LGMDR12). …\n",
      "\n",
      "… Lysate was collected, precleared for 30 min with Protein-G Dynabeads (Invitrogen) at 4 C, \n",
      "and then incubated with mouse anti-DLK antibody (Neuromab clone N377/20; 1:100) or an …\n",
      "\n",
      "Dual leucine zipper kinase (DLK, Map3k12) is an axonal protein that governs the balance \n",
      "between degeneration and regeneration through its downstream effectors c-jun N-terminal …\n",
      "\n",
      "… NeuroMab identified multiple clones and clone N345/51 was … To demonstrate specificity of \n",
      "REEP1 mAb clone N345/51 and a … monoclonal antibody (NeuroMab clone N345/51) to assess …\n",
      "\n",
      "… (1:5, clone K75/41; NeuroMab, UC Davis/NIH NeuroMab Facility) and mouse monoclonal \n",
      "anti-K V 7.4 (1:50, clone N43/6; NeuroMab) were used to detect various potassium channel α- …\n",
      "\n",
      "Objective— In renal arteries, inhibitors of G protein βγ subunits (Gβγ) reduce Kv7 activity \n",
      "and inhibit Kv7-dependent receptor-mediated vasorelaxations. However, the mechanisms …\n",
      "\n",
      "… Primary antibodies used were Neuromab monoclonal antibodies against KCNQ1 (N37/A10, \n",
      "0.6 μg/ml), KCNQ2 (N26A/23, 0.6 μg/ml), and KCNQ4 (N43/6, 0.6 μg/ml) and a non-…\n",
      "\n",
      "Within the vasculature Kv7 channels are key regulators of basal tone and contribute to a variety \n",
      "of receptor mediated vasorelaxants. The Kv7.4 isoform, abundant within the vasculature, …\n",
      "\n",
      "… The antibody (clone N43/6) was developed by and/or obtained from the UC Davis/NIH \n",
      "NeuroMab Facility, supported by NIH grant U24NS050606 and maintained by the Department of …\n",
      "\n",
      "Simultaneous labeling of multiple targets in a single sample, or multiplexing, is a powerful \n",
      "approach to directly compare the amount, localization and/or molecular properties of different …\n",
      "\n",
      "… Mouse monoclonal anti-AnkG (N106/36; RRID: AB_10697718) and anti-GST (N100/13; \n",
      "RRID:AB_10671817) antibodies were purchased from UC Davis and the NIH NeuroMab facility. …\n",
      "\n",
      "… The plates were then washed with PBST buffer (PBS with 0.05% Tween 20) and incubated \n",
      "with mouse anti-GST antibody (clone N100/13, NeuroMab) for 1 hour at room temperature. …\n",
      "\n",
      "The scaffolding protein ankyrin-G is required for Na + channel clustering at axon initial \n",
      "segments. It is also considered essential for Na + channel clustering at nodes of Ranvier to …\n",
      "\n",
      "GABA A receptors mediate fast inhibitory transmission in the brain, and their number can be \n",
      "rapidly up- or downregulated to alter synaptic strength. Neuroligin-2 plays a critical role in …\n",
      "\n",
      "Despite the central role PSD-95 plays in anchoring postsynaptic AMPARs, how PSD-95 \n",
      "itself is tethered to postsynaptic sites is not well understood. Here we show that the F-actin …\n",
      "\n",
      "… , AB_2282826), and anti-GST (1:2000; N100/13, AB_10671818) antibodies were purchased \n",
      "from the UC Davis/NIH NeuroMab Facility (Davis, CA). Rabbit polyclonal antibody anti-GFP (…\n",
      "\n",
      "The conventional kinesin motor transports many different cargos to specific locations in neurons. \n",
      "How cargos regulate motor function remains unclear. Here we focus on KIF5, the heavy …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Astrocytic GLT-1 is the main glutamate transporter involved in glutamate buffering in the brain, \n",
      "pivotal for glutamate removal at excitatory synapses to terminate neurotransmission and …\n",
      "\n",
      "… Mouse monoclonal anti-PSD-93 (N18/30) and anti-ADAM22 (cytoplasmic, N46/30; extracellular, \n",
      "N57/2) were obtained from the University of California Davis/NIH NeuroMab Facility and …\n",
      "\n",
      "A specialized axonal ending, the basket cell “pinceau,” encapsulates the Purkinje cell axon \n",
      "initial segment (AIS), exerting final inhibitory control over the integrated outflow of the …\n",
      "\n",
      "Cerebrospinal fluid samples collected from children during initial presentation of central \n",
      "nervous system inflammation, who may or may not subsequently be diagnosed as having …\n",
      "\n",
      "In myelinated fibers, the voltage-gated sodium channels Nav1 are concentrated at the nodal \n",
      "gap to ensure the saltatory propagation of action potentials. The voltage-gated potassium …\n",
      "\n",
      "What percentage of the protein function is required to prevent disease symptoms is a \n",
      "fundamental question in genetic disorders. Decreased transsynaptic LGI1-ADAM22 protein …\n",
      "\n",
      "Lateral diffusion on the neuronal plasma membrane of the AMPA-type glutamate receptor (AMPAR) \n",
      "serves an important role in synaptic plasticity. We investigated the role of the …\n",
      "\n",
      "Neuronal precursor cell migration in the developing mammalian brain is a complex process \n",
      "requiring the coordinated interaction of numerous proteins. We have recently shown that …\n",
      "\n",
      "… Antibodies were used as follows: Monoclonal mouse anti-human Olfm-1 clone K96/7 (against \n",
      "amino acids 28–48 of the common region of human Olfm-1 splice variants (…\n",
      "\n",
      "… For the TRPC 5 and 7 images shown we used antibodies N67/15 and N64A/36 respectively \n",
      "(both … Using the N67/15 antibody from Neuromab, Davis, CA, USA raised against the human …\n",
      "\n",
      "… N67/15 (NIH/NeuroMab) targeting the cytosolic carboxyl tail (827-845 amino acids) of \n",
      "TRPC5 and a commercial secondary antibody and fluorescein-based visualization kit (Vector …\n",
      "\n",
      "… C-terminus of rat TRPC4 and TRPC5 antibody N67/15 against amino acids 827–845 at the \n",
      "cytoplasmic C-terminus of human TRPC5 were from NIH NeuroMab Facility (Davis, CA, USA; …\n",
      "\n",
      "Insulin receptors in podocytes are essential for normal kidney function. Here, we show that \n",
      "insulin evokes a rapid increase in the surface expression of canonical transient receptor …\n",
      "\n",
      "Various magnetic deep brain stimulation (DBS) methods have been developing rapidly in \n",
      "the last decade for minimizing the invasiveness of DBS. However, current magnetic DBS …\n",
      "\n",
      "… For blotting, the primary antibody was either mouse monoclonal to TRPC5 (1:500; clone: \n",
      "N67/15; NeuroMab), mouse monoclonal to HA (1:1,000; Cell Signaling), or mouse monoclonal …\n",
      "\n",
      "… : N67/15, NeuroMab, Davis, CA) and diluted in DaVinci Green diluent (Biocare) applied for \n",
      "1 h. For double labeling, a cocktail of secondary antibodies (Alexa 555-conjugated goat anti…\n",
      "\n",
      "Rare neurological diseases shed light onto universal neurobiological processes. However, \n",
      "molecular mechanisms connecting genetic defects to their disease phenotypes are elusive. …\n",
      "\n",
      "… 29) and mouse anti-FHF4 (N56/21, Catalogue 75–096, NeuroMab), both of which have \n",
      "been prior validated for immunoprecipitation competence. Streptavidin-agarose-captured, …\n",
      "\n",
      "… (1∶1000, NeuroMab, clone N56/21), mouse monoclonal … NeuroMab antibody and in our \n",
      "experience it gives very specific staining of the AIS. The GFP antibody is a validated NeuroMab …\n",
      "\n",
      "Intracellular Fibroblast Growth Factor 14 (iFGF14) and the other intracellular FGFs (iFGF11-13) \n",
      "regulate the properties and densities of voltage-gated neuronal and cardiac Na + (Nav) …\n",
      "\n",
      "The axon initial segment (AIS) is a specialized region in neurons that encompasses two \n",
      "essential functions, the generation of action potentials and the regulation of the axodendritic …\n",
      "\n",
      "… Cells were then incubated with monoclonal FHF4 antibody (1:200, N56/21, UC Davis/NIH \n",
      "NeuroMab Facility) or polyclonal anti- Navβ4 antibody (1:500, #Ab80539, Abcam) diluted in …\n",
      "\n",
      "Persistent alterations in neuronal activity elicit homeostatic plastic changes in synaptic \n",
      "transmission and/or intrinsic excitability. However, it is unknown whether these homeostatic …\n",
      "\n",
      "Protein–protein interactions (PPI) offer unexploited opportunities for CNS drug discovery \n",
      "and neurochemical probe development. Here, we present ZL181, a novel peptidomimetic …\n",
      "\n",
      "Mitochondrial Ca 2+ efflux by NCLX is a critical rate-limiting step in mitochondria signaling. \n",
      "We previously showed that NCLX is phosphorylated at a putative Casein Kinase 2 (CKII) site, …\n",
      "\n",
      "In the primate visual system, the ganglion cells of the magnocellular pathway underlie motion \n",
      "and flicker detection and are relatively transient, while the more sustained ganglion cells of …\n",
      "\n",
      "… Anti-AnkG (N106/36), anti-FGF14 (N56/21), and anti-Nav1.6 (K87A/10) mouse monoclonal \n",
      "antibodies were purchased from NeuroMab. Affinity-purified rabbit polyclonal antibodies …\n",
      "\n",
      "… A monoclonal antibody anti-GABAB1 (Clone N93A/49; NeuroMab, Davis, CA, USA) and \n",
      "anti-GABAB2 (Clone N81/37; NeuroMab) were used. A guinea-pig polyclonal antibody anti-…\n",
      "\n",
      "Cognitive dysfunction is a common symptom in many neuropsychiatric disorders and directly \n",
      "correlates with poor patient outcomes. The majority of prolonged inhibitory signaling in the …\n",
      "\n",
      "Polysialic acid is a unique carbohydrate polymer specifically attached to a limited number of \n",
      "glycoproteins. Among them is synaptic cell adhesion molecule 1 (SynCAM 1), a member of …\n",
      "\n",
      "Previously, we showed that a hierarchy of spectrin cytoskeletal proteins maintains nodal Na \n",
      "+ channels (Liu et al., 2020). Here, using mice lacking β1, β4, or β1/β4 spectrins, we show …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection \n",
      "of … The sections were also labelled with the mouse mAbs anti-Parvalbumin L114/3 mAb (IgG2a…\n",
      "\n",
      "Genes with multiple co-active promoters appear common in brain, yet little is known about \n",
      "functional requirements for these potentially redundant genomic regulatory elements. SCN1A, …\n",
      "\n",
      "… years, including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection \n",
      "… The sections were also labelled with the mouse mAbs anti-Parvalbumin L114/3 (IgG2a) in …\n",
      "\n",
      "Developmental and epileptic encephalopathies (DEE) are a group of severe epilepsies that \n",
      "usually present with intractable seizures, developmental delay, and often have elevated risk …\n",
      "\n",
      "… -reactivity validated by NeuroMab. Antibodies N257/25 and … The subclone N262/16 raised \n",
      "against the hapten 4 showed … Given these results, N257/25.1.1 and N262/16 represent …\n",
      "\n",
      "Cardiac Kir2.1 and Nav1.5 channels generate the inward rectifier K + (I K1 ) and the Na + (I \n",
      "Na ) currents, respectively. There is a mutual interplay between the ventricular I Na and I K1 …\n",
      "\n",
      "Key points This paper provides a comprehensive electrophysiological characterization of the \n",
      "external [K + ] dependence and inward rectifying properties of Kir channels in fast skeletal …\n",
      "\n",
      "… Monoclonal antibodies for ankyrin-G (clone N106/36) and K ir 2.1 (clone N112B/14) were \n",
      "obtained from the University of California Davis/National Institutes of Health NeuroMab Facility. …\n",
      "\n",
      "… (Clone N112B/14, Neuromab, catalog #73-210, tissue culture supernatant, RRID: AB_11001668, \n",
      "dilution 1:15), ms anti-Ca V 3.1 (Clone N178A/9; catalog #Neuromab, 73-206, tissue …\n",
      "\n",
      "… We found the Kv8.2 antibody (Clone N448/88 from NeuroMab) generated a non-specific \n",
      "signal in immunohistochemistry (Supplementary Material, Fig. S2). Courtesy of James Trimmer, …\n",
      "\n",
      "Control of translation is a fundamental source of regulation in gene expression. The induction \n",
      "of protein synthesis by brain-derived neurotrophic factor (BDNF) critically contributes to …\n",
      "\n",
      "Variation in gene expression is an important mechanism underlying susceptibility to complex \n",
      "disease and traits. Single nucleotide polymorphisms (SNPs) account for a substantial …\n",
      "\n",
      "… 2+ channel (NeuroMab clone L48A/9) directed against amino acids 859–875 in the N terminus \n",
      "of rat Ca v 1.3, and 10) monoclonal anti-Ca v 1.3 Ca 2+ channel (NeuroMab clone N38/8) …\n",
      "\n",
      "… Furthermore, immunostaining of Ca v 1.3α 1 using our and commercial (Alomone, Millipore, \n",
      "Neuromab) antibodies was conducted under two different fixation conditions, brief 2% …\n",
      "\n",
      "Recent studies have shown that zinc ion (Zn) can behave as an intracellular signaling \n",
      "molecule. We previously demonstrated that mast cells stimulated through the high-affinity IgE …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… α4 hybridoma-generated N398A/34 mAb (top section) and the N398A/34 scFv (bottom section…\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… α4 hybridoma-generated N398A/34 mAb (top section) and the N398A/34 scFv (bottom section…\n",
      "\n",
      "Gamma aminobutyric acid (GABA) subtype A receptors (GABA A Rs) are integral membrane \n",
      "ion channels composed of five individual proteins or subunits. Up to 19 different GABA A R …\n",
      "\n",
      "Impaired inhibitory signaling underlies the pathophysiology of many neuropsychiatric and \n",
      "neurodevelopmental disorders including autism spectrum disorders and epilepsy. Neuronal …\n",
      "\n",
      "Alzheimer's disease (AD) is characterized by memory impairments and age-dependent \n",
      "synapse loss. Experimental and clinical studies have shown decreased expression of the …\n",
      "\n",
      "Retromer is an evolutionarily conserved endosomal trafficking complex that mediates the \n",
      "retrieval of cargo proteins from a degradative pathway for sorting back to the cell surface. To …\n",
      "\n",
      "Coordinated development of excitatory and inhibitory synapses is essential for higher brain \n",
      "function, and impairment in this development is associated with neuropsychiatric disorders. …\n",
      "\n",
      "Altered excitatory/inhibitory (E/I) balance is implicated in neuropsychiatric and neurodevelopmental \n",
      "disorders, but the underlying genetic etiology remains poorly understood. Copy …\n",
      "\n",
      "Multiple trans-synaptic complexes organize synapse development, yet their roles in the \n",
      "mature brain and cooperation remain unclear. We analyzed the postsynaptic adhesion protein …\n",
      "\n",
      "LRRTMs are postsynaptic cell adhesion proteins that have region-restricted expression in the \n",
      "brain. To determine their role in the molecular organization of synapses in vivo, we studied …\n",
      "\n",
      "… α-SAP102 antibodies, ie a mouse monoclonal (NeuroMab clone N19/2; left lane) and a \n",
      "rabbit … with α-Nbea and α-SAP102 antibody (NeuroMab clone N19/2). (C) Dendritic Nbea …\n",
      "\n",
      "… PSD-95 (K28/43) and SAP97 (N19/2) antibodies were obtained from Neuromab. ERK 1/2 \n",
      "(extracellular signal-regulated kinase 1/2) antibody was purchased from Cell Signaling …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "The mechanisms controlling synapse growth and maintenance are of critical importance for \n",
      "learning and memory. The MAGUK family of synaptic scaffolding proteins is abundantly …\n",
      "\n",
      "… 75-058; RRID:AB_2261666) recognized a single band of 105 kDa in Western blots of rat \n",
      "and mouse brain membranes (UC Davis/NIH NeuroMab Facility). SAP102 labeling was …\n",
      "\n",
      "Neurotransmission and synaptic strength depend on expression of post-synaptic receptors \n",
      "on the cell surface. Post-translational modification of receptors, trafficking to the synapse …\n",
      "\n",
      "… ), and NR2B (1:300, mouse IgG2a N59/20; NeuroMab). Sections were washed three times \n",
      "for 5 min in 1× PBS at room temperature with gentle agitation before applying Alexa Fluor dye-…\n",
      "\n",
      "10.7554/eLife.40316.001 Protein palmitoylation and depalmitoylation alter protein function. \n",
      "This post-translational modification is critical for synaptic transmission and plasticity. …\n",
      "\n",
      "… Anti-PSD95/SAP90 antibody (K28/43) and SAP102 (N19/2) was purchased from \n",
      "University of California Davis/National Institutes of Health (NIH) NeuroMab facility. Anti-α-tubulin …\n",
      "\n",
      "Synapses are found in vast numbers in the brain and contain complex proteomes. We \n",
      "developed genetic labeling and imaging methods to examine synaptic proteins in individual …\n",
      "\n",
      "Phosphorylation regulates surface and synaptic expression of NMDA receptors (NMDARs). \n",
      "Both the tyrosine kinase Fyn and the tyrosine phosphatase striatal-enriched protein tyrosine …\n",
      "\n",
      "(論文の内容の要旨)[背景と目的] シナプス後肥厚部 (postsynaptic density, PSD) とはシナプス膜\n",
      "直下にある, 神経細胞に高度に分化した特殊な細胞骨格系で, 極めて精巧なタンパク質複合体である…\n",
      "\n",
      "… N11/33, Neuromab) or anti-Slack (clone N3/26, Neuromab) … , IgG1, clone N3/26, NeuroMab), \n",
      "mouse polyclonal anti-beta-… (1:2000, IgG1, clone N19/2, Biolegend) antibody for 2.5 h at …\n",
      "\n",
      "Depolarization of neurons in 3-week-old rat hippocampal cultures promotes a rapid increase \n",
      "in the density of surface NMDA receptors (NRs), accompanied by transient formation of …\n",
      "\n",
      "We have recently shown that membrane association of neuronal nitric oxide synthase-α (nNOSα) \n",
      "is critical in the regulation of synthesis of NO during nitrergic neurotransmission. The …\n",
      "\n",
      "… /43), synapse-associated protein 102 (SAP102) (clone N19/2), PSD-93 (clone N18/30), and \n",
      "SAP97 (clone K64/15) from NeuroMab (Davis, CA); neuronal nitric oxide synthase (nNOS) (…\n",
      "\n",
      "It is essential to study the molecular architecture of post‐synaptic density ( PSD ) to understand \n",
      "the molecular mechanism underlying the dynamic nature of PSD , one of the bases of …\n",
      "\n",
      "Despite the central role PSD-95 plays in anchoring postsynaptic AMPARs, how PSD-95 \n",
      "itself is tethered to postsynaptic sites is not well understood. Here we show that the F-actin …\n",
      "\n",
      "Excitatory glutamatergic synapses facilitate important aspects of communication between \n",
      "the neurons that govern complex forms of behavior. Accordingly, small differences in the …\n",
      "\n",
      "Neurodevelopmental disorders such as autism spectrum disorders and schizophrenia \n",
      "differentially impact males and females and are highly heritable. The ways in which sex and …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "… Control antibody used for the outer membrane was voltage-dependent anion channel (Neuromab \n",
      "N152B/23), and cytochrome c oxidase 5 (Abcam 110263) was used to examine …\n",
      "\n",
      "Mitochondria are composed of many small proteins that control protein synthesis, complex \n",
      "assembly, metabolism, and ion and reactive oxygen species (ROS) handling. We show that a …\n",
      "\n",
      "Mutations in the mitochondrial protein CHCHD2 cause autosomal dominant Parkinson’s \n",
      "disease characterized by the preferential loss of substantia nigra dopamine (DA) neurons. …\n",
      "\n",
      "α-synuclein, β-synuclein, and γ-synuclein are abundantly expressed proteins in the vertebrate \n",
      "nervous system. α-synuclein functions in neurotransmitter release by binding to and …\n",
      "\n",
      "SIRT3 is a member of the Sirtuin family of NAD + -dependent deacylases and plays a critical \n",
      "role in metabolic regulation. Organism-wide SIRT3 loss manifests in metabolic alterations; …\n",
      "\n",
      "Point mutations in cysteine string protein-α (CSPα) cause dominantly inherited adult-onset \n",
      "neuronal ceroid lipofuscinosis (ANCL), a rapidly progressing and lethal neurodegenerative …\n",
      "\n",
      "α-Synuclein (αSyn), β-synuclein (βSyn), and γ-synuclein (γSyn) are abundantly expressed \n",
      "in the vertebrate nervous system. αSyn functions in neurotransmitter release via binding to …\n",
      "\n",
      "Background: Inflammation-induced sensitization of primary afferents is associated with a \n",
      "decrease in K + current. However, the type of K + current and basis for the decrease varies as a …\n",
      "\n",
      "… , L133/27, L113/130) that were ultimately selected as NeuroMab mAbs. B. Exemplar results \n",
      "… candidates (L113/13, L113/27, L113/130 and) that were eventually selected as NeuroMabs, …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… Shown are sequence entries returned for a search for “L113/13”. This includes the mAb ID, …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "Synucleinopathies form a group of neurodegenerative diseases defined by the misfolding \n",
      "and aggregation of α-synuclein (α-syn). Abnormal accumulation and spreading of α-syn …\n",
      "\n",
      "Classical cadherins, a set of ~20 related recognition and signaling molecules, have been \n",
      "implicated in many aspects of neural development, including the formation and remodeling of …\n",
      "\n",
      "… -Fc protein as lacking the HRD but retaining NLGN1 binding ability by Western blotting using \n",
      "an anti-NRX1β antibody that recognizes an epitope in the HRD (clone N170A/1; Neuromab…\n",
      "\n",
      "Sensitive and specific antibodies are essential for detecting molecules in cells and tissues. \n",
      "However, currently used polyclonal and monoclonal antibodies are often less sensitive than …\n",
      "\n",
      "The formation, function, and plasticity of synapses require dynamic changes in synaptic \n",
      "receptor composition. Here, we identify the sorting receptor SorCS1 as a key regulator of …\n",
      "\n",
      "… Here we show that the anti-RGS14 mouse monoclonal antibody (Clone N133/21, \n",
      "NeuroMabs) used in our study is very specific and sensitive for RGS14, and that this antibody …\n",
      "\n",
      "… In the case of the NeuroMab target RGS14, a priori knowledge that within the brain, RGS14 \n",
      "… the relatively straightforward identification of the N133/21 NeuroMab, that has subsequently …\n",
      "\n",
      "… The tissue sections were subsequently incubated with mouse primary antibodies for RGS14 \n",
      "(NeuroMab; N133/21; 1:500) or STEP (Cell Signaling Technology; 4396S; 1:500) for 16 hr …\n",
      "\n",
      "Since 1959, the Russian Farm‐Fox study has bred foxes to be either tame or, more recently, \n",
      "aggressive, and scientists have used them to gain insight into the brain structures …\n",
      "\n",
      "The hippocampus must be capable of sorting and integrating multiple sensory inputs \n",
      "separately but simultaneously. However, it remains to be elucidated how the hippocampus …\n",
      "\n",
      "… available from the UC Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). 27 … dilution \n",
      "of Tissue Culture Supernatant, concentration unknown N133/21, antiRGS14 mouse IgG2a mAb …\n",
      "\n",
      "Rationale In rodents, exposure to novel environments or psychostimulants promotes \n",
      "locomotion. Indeed, locomotor reactivity to novelty strongly predicts behavioral responses to …\n",
      "\n",
      "… , 1:2000, Atlas Antibodies; rat anti-GFP, 1:2000, Nacalai; rabbit anti-NECAB1, 1:2000, Atlas \n",
      "Antibodies; mouse anti-PKC-δ, 1;500, BD Biosciences; mouse anti-RGS14, 1:500, NeuroMab…\n",
      "\n",
      "… Acknowledgements: Most of the antibodies used in this study are available from the UC \n",
      "Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). Correspondence should be addressed to …\n",
      "\n",
      "… NeuroMab mouse monoclonal antibodies were validated as follows: Anti-Ankyrin-G N106/65: … \n",
      "N106/65 does not recognize denatured Ankyrin-G on immunoblots, but has an identical …\n",
      "\n",
      "The axon initial segment (AIS) is a specialized neuronal compartment involved in the \n",
      "maintenance of axo-dendritic polarity and in the generation of action potentials. It is also a site of …\n",
      "\n",
      "… S1 a, b); (ii) the guinea pig, rabbit and mouse monoclonal anti-AnkG (N106/65; IgG2b; \n",
      "Neuromab) antibodies were shown to strictly colocalize at the AISs (Fig. S1 d); (iii) western blots …\n",
      "\n",
      "… (N106/65 NeuroMab), anti-Nav mouse monoclonal (K58/35, Sigma–Aldrich), goat polyclonal \n",
      "antibody to CK2α (Santa Cruz Biotechnology, Inc.). Secondary antibodies conjugated to …\n",
      "\n",
      "… end users to provide the specific NeuroMab clone number (eg, N106/36, N106/65, etc.) as … \n",
      "‘RRID:AB_’ identifier has been assigned to each NeuroMab (and extended to the level of TCS …\n",
      "\n",
      "Non‐Technical Summary To be effective, synaptic transmission requires precise alignment \n",
      "of the presynaptic terminal, releasing the neurotransmitter, with the postsynaptic density, …\n",
      "\n",
      "Previously, we showed that a hierarchy of spectrin cytoskeletal proteins maintains nodal Na \n",
      "+ channels (Liu et al., 2020). Here, using mice lacking β1, β4, or β1/β4 spectrins, we show …\n",
      "\n",
      "Neuron-glia interactions establish functional membrane domains along myelinated axons. \n",
      "These include nodes of Ranvier, paranodal axoglial junctions and juxtaparanodes. …\n",
      "\n",
      "… 1e mAb (K36/15; UC Davis/NIH NeuroMab Facility) before detergent permeabilization to \n",
      "detect the cell surface pool. The total cellular pools of the respective proteins were detected by …\n",
      "\n",
      "In tauopathic conditions, such as Alzheimer's disease (AD), highly soluble and natively \n",
      "unfolded tau polymerizes into an insoluble filament; however, the mechanistic details of this …\n",
      "\n",
      "… in panel C1) is shown together with labeling for N106/65 (green), an IgG1 mAb specific for \n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and …\n",
      "\n",
      "TDP-43 is extensively studied in neurons in physiological and pathological contexts. \n",
      "However, emerging evidence indicates that glial cells are also reliant on TDP-43 function. We …\n",
      "\n",
      "Glycoprotein M6B and the closely related proteolipid protein regulate oligodendrocyte \n",
      "myelination in the central nervous system, but their role in the peripheral nervous system is less …\n",
      "\n",
      "Voltage-gated sodium channels (Nav1s) are responsible for the initiation and propagation of \n",
      "action potentials in neurons, muscle, and endocrine cells. Many clinically used drugs such …\n",
      "\n",
      "Myelin is a highly specialized structure, organized into short periodical segments, which are \n",
      "extensions of the Schwann cell membrane in the peripheral nervous system (PNS). Myelin …\n",
      "\n",
      "… Primary antibodies: mouse IgG2a anti-AnkyrinG (clone N106/36; 1:100), mouse IgG2b \n",
      "anti-AnkyrinG (clone N106/65; 1:75), and mouse IgG1 anti-Caspr (1:100), all from Neuromab; …\n",
      "\n",
      "Multicolor three-dimensional (3D) superresolution techniques allow important insight into \n",
      "the relative organization of cellular structures. While a number of innovative solutions have …\n",
      "\n",
      "The endoplasmic reticulum (ER) and plasma membrane (PM) form junctions crucial to ion \n",
      "and lipid signaling and homeostasis. The Kv2.1 ion channel is localized at ER–PM junctions …\n",
      "\n",
      "… : AB_10673030, and N106/65; RRID: AB_10675130) and neurofascin (A12/18; AB_2282826) \n",
      "antibodies were purchased from the University of California, Davis/NIH NeuroMab Facility. …\n",
      "\n",
      "The activity-dependent rules that govern the wiring of GABAergic interneurons are not well \n",
      "understood. Chandelier cells (ChCs) are a type of GABAergic interneuron that control …\n",
      "\n",
      "The survival of complex organisms depends on their ability to interact with their environment \n",
      "through precise control of motor behaviours. This is achieved via specialised synapses …\n",
      "\n",
      "Axon initial segments (AISs) generate action potentials and regulate the polarized distribution \n",
      "of proteins, lipids, and organelles in neurons. While the mechanisms of AIS Na + and K + …\n",
      "\n",
      "… Antibodies included the following: rabbit monoclonal mAB11 against NRG2 (Vullhorst et al., \n",
      "2015); mouse monoclonal against Ankyrin G (clone N106/65; NeuroMab); rabbit polyclonal …\n",
      "\n",
      "… We provide examples from NeuroMab screens, and from the subsequent specialized … of \n",
      "NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs at the …\n",
      "\n",
      "… K28/43 gives dense non-specific label, but mAb K28/86 from the same project performs \n",
      "well … control (NMS) and a positive control NeuroMab mAb against a different target (L86A/37, …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Membrane-associated guanylate kinases (MAGUKs), including SAP102, PSD-95, PSD-93, \n",
      "and SAP97, are scaffolding proteins for ionotropic glutamate receptors at excitatory synapses…\n",
      "\n",
      "Noise exposure is particularly stressful to hair-cell mitochondria, which must produce \n",
      "enough energy to meet high metabolic demands as well as regulate local intracellular Ca2+ …\n",
      "\n",
      "Sympathetic neurons require NGF from their target fields for survival, axonal target innervation, \n",
      "dendritic growth and formation, and maintenance of synaptic inputs from preganglionic …\n",
      "\n",
      "Background Trichoplax adhaerens is the best-known member of the phylum Placozoa, one \n",
      "of the earliest-diverging metazoan phyla. It is a small disk-shaped animal that glides on …\n",
      "\n",
      "… The following commercial antibodies were used in this study: MAGUK (K28/86; 1:500; \n",
      "NeuroMab, UC Davis), acetylated Tubulin (1:1000; Sigma-Aldrich), Parvalbumin (1:2000; Thermo …\n",
      "\n",
      "10.7554/eLife.35796.001 The orientation of hair bundles on top of sensory hair cells (HCs) \n",
      "in neuromasts of the lateral line system allows fish to detect direction of water flow. Each …\n",
      "\n",
      "The ability of dendritic spines to change size and shape rapidly is critical in modulating synaptic \n",
      "strength; these morphological changes are dependent upon rearrangements of the actin …\n",
      "\n",
      "… A mouse monoclonal anti-pan-MAGUK scaffolding protein family antibody (clone K28/86, \n",
      "final concentration: 3.46 μg/ml) was purchased from the NIH NeuroMab Facility. This antibody …\n",
      "\n",
      "L-type calcium channels (Ca V 1) are involved in diverse processes, such as neurotransmission, \n",
      "hormone secretion, muscle contraction, and gene expression. In this study, we uncover …\n",
      "\n",
      "… Anti–PAN MAGUK antibody (clone K28/86) and anti-SAP97 antibody (clone K64/15) were \n",
      "purchased from the University of California Davis/NIH NeuroMab Facility. Anti–human SAP97 …\n",
      "\n",
      "Usher Syndrome (USH) is the most common cause of human deaf/blindness. The zebrafish \n",
      "myo7aa -/- mutant, faithfully models USH1; homozygous zebrafish are deaf and exhibit …\n",
      "\n",
      "The role of the cannabinoid receptor 2 (CNR2) is still poorly described in sensory epithelia. \n",
      "We found strong cnr2 expression in hair cells (HCs) of the inner ear and the lateral line (LL), …\n",
      "\n",
      "The mariner (myo7aa −/− ) mutant is a zebrafish model for Usher syndrome type 1 (USH1). \n",
      "To further characterize hair cell synaptic elements in myo7aa −/− mutants, we focused on the …\n",
      "\n",
      "… density (anti-pan-MAGUK clone K28/86, 1:100, mouse IgG1; University of California, \n",
      "Davis/National Institutes of Health NeuroMab Facility), inhibitory postsynaptic density (anti-gephyrin…\n",
      "\n",
      "The synthetic glucocorticoid dexamethasone is commonly used to treat inner ear disorders. \n",
      "Previous work in larval zebrafish has shown that dexamethasone treatment enhances hair …\n",
      "\n",
      "Excess noise damages sensory hair cells, resulting in loss of synaptic connections with \n",
      "auditory nerves and, in some cases, hair-cell death. The cellular mechanisms underlying …\n",
      "\n",
      "How synaptic neuropil is formed within the CNS is poorly understood. The retinal inner \n",
      "plexiform layer (IPL) is positioned between the cell bodies of amacrine cells (ACs) and retinal …\n",
      "\n",
      "Abstract Analysis of mechanotransduction among ensembles of sensory hair cells in vivo is \n",
      "challenging in many species. To overcome this challenge, we used optical indicators to …\n",
      "\n",
      "Sensory hair cells in the ear utilize specialized ribbon synapses. These synapses are \n",
      "defined by electron-dense presynaptic structures called ribbons, composed primarily of the …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… -Mortalin/GRP75 NeuroMab N52A/42 as a control to show comparable loading of all samples, \n",
      "with the exception of (h) in which the anti-pan-KChIP NeuroMab K55/82 was used to show …\n",
      "\n",
      "Recent human exome-sequencing studies have implicated polymorphic Brg1-associated \n",
      "factor (BAF) complexes (mammalian SWI/SNF chromatin remodeling complexes) in several …\n",
      "\n",
      "A truncating mutation in the mouse Srrm4 gene, which encodes a neuronal splicing factor, \n",
      "causes alternative splicing defects selectively in the ear. The mechanism by which splicing is …\n",
      "\n",
      "… (NeuroMab; K17/70) primary antibodies in 200 μl of PBS with 0.01% Tween-20 (PBS-t). \n",
      "Antibodies were cross-linked to magnetic beads by washing with PBS containing 50 mM sodium …\n",
      "\n",
      "… NP_034728), conserved in zebra finch Neuromab (Davis, CA) K17/70 Mouse \n",
      "monoclonal … no bands in lysates from knockout mice (http://neuromab.ucdavis.edu/datasheet/K17_70.pdf). …\n",
      "\n",
      "… Mouse mAbs to Kvβ2 (K17/70) and to Kv1.4 (K13/31) were obtained from NeuroMab. \n",
      "Rabbit pAbs to mitogen-activated protein kinase (MAPK) and to phospho-MAPK were from …\n",
      "\n",
      "A specialized axonal ending, the basket cell “pinceau,” encapsulates the Purkinje cell axon \n",
      "initial segment (AIS), exerting final inhibitory control over the integrated outflow of the …\n",
      "\n",
      "… (clone #: K14/16, K17/70, and K13/31, respectively; UC Davis/NIH Neuromab Facility, Davis, \n",
      "CA, … We thank University of California, Davis-NIH NeuroMab facility (a nonprofit supplier) for …\n",
      "\n",
      "Axons must withstand mechanical forces, including tension, torsion, and compression. \n",
      "Spectrins and actin form a periodic cytoskeleton proposed to protect axons against these forces. …\n",
      "\n",
      "… Monoclonal anti-Kvβ2 antibody (clone K17/70) was purchased from NIH Neuromab. \n",
      "Monoclonal anti-c-myc (clone 9E10) was from Sigma-Aldrich. Monoclonal anti-GAPDH-hrp (clone …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Cortical dysplasia (CD) is a common cause for intractable epilepsy. Hyperactivation of the \n",
      "mechanistic target of rapamycin (mTOR) pathway has been implicated in CD; however, the …\n",
      "\n",
      "… presented as typical of a negative candidate (N244/3), and three represented as positive \n",
      "candidates (N244/5, N244/19 and N244/20), of which N244/5 ultimately became a NeuroMab. …\n",
      "\n",
      "Nectin‐like molecule 4 ( N ecl‐4)/ CADM 4, a transmembrane cell–cell adhesion molecule \n",
      "with three Ig‐like domains, was shown to serve as a tumor suppressor, but its mode of action …\n",
      "\n",
      "Neuron-glia interactions establish functional membrane domains along myelinated axons. \n",
      "These include nodes of Ranvier, paranodal axoglial junctions and juxtaparanodes. …\n",
      "\n",
      "RNA localization is a regulatory mechanism that is conserved from bacteria to mammals. Yet, \n",
      "little is known about the mechanism and the logic that govern the distribution of RNA …\n",
      "\n",
      "Glycoprotein M6B and the closely related proteolipid protein regulate oligodendrocyte \n",
      "myelination in the central nervous system, but their role in the peripheral nervous system is less …\n",
      "\n",
      "Myelin is a highly specialized structure, organized into short periodical segments, which are \n",
      "extensions of the Schwann cell membrane in the peripheral nervous system (PNS). Myelin …\n",
      "\n",
      "Myelinating cells, eg oligodendrocytes (OLs) and Schwann cells (SCs), have long been \n",
      "subject to extensive study. Although many of the factors regulating myelination have been …\n",
      "\n",
      "… The immunogens used to produce the antibodies (Abs) PINK1175–250 and N4/15 are \n",
      "indicated by the red and blue lines, respectively. (B) SH-SY5Y cells expressing PINK1-IRESeGFP …\n",
      "\n",
      "Dysfunction of PINK1, a mitochondrial Ser/Thr kinase, causes familial Parkinson's disease (PD). \n",
      "Recent studies have revealed that PINK1 is rapidly degraded in healthy mitochondria …\n",
      "\n",
      "… Membranes were then incubated overnight with primary antibodies, including mouse \n",
      "monoclonal Dlx1 (clone L43/40, NeuroMab, Davis, CA, USA), mouse monoclonal Gad67 (clone …\n",
      "\n",
      "… /Antibodies, Inc.); mouse anti-human QKI-6 (10 μg ml −1 , clone N182/17; NeuroMab/Antibodies \n",
      "Inc.) or mouse anti-human QKI-7 (10 μg ml −1 , clone N183/15; NeuroMab/Antibodies Inc…\n",
      "\n",
      "Rationale: RNA-binding proteins are critical post-transcriptional regulators of RNA and can \n",
      "influence pre-mRNA splicing, RNA localization, and stability. The RNA-binding protein …\n",
      "\n",
      "Dysfunction of endothelial cells (ECs) and vascular smooth muscle cells (VSMCs) leads to \n",
      "ischaemia, the central pathology of cardiovascular disease. Stem cell technology will …\n",
      "\n",
      "Alternative splicing is critical for animal ontogeny; however, its role in the earliest developmental \n",
      "decision, the specification of the three embryonic germ layers, is poorly understood. By …\n",
      "\n",
      "… L24/18; NeuroMab) and either rabbit polyclonal anti-BK Ca α (Abcam) or rabbit polyclonal \n",
      "anti-TRPM4 (Thermo Fisher Scientific), each at a dilution of 1:100. After washing, cells were …\n",
      "\n",
      "… 407140), and 3) monoclonal mouse anti-IP3 receptor clone L24/18, obtained from the \n",
      "UC Davis/NIH NeuroMab Facility, University of California, Davis (catalog No. 75-035). …\n",
      "\n",
      "… The monoclonal anti-InsP 3 R1 antibody L24/18 was developed and obtained from the \n",
      "University of California at Davis/NINDS/National Institute of Mental Health NeuroMab Facility …\n",
      "\n",
      "Key points • Feed arteries and arterioles, respectively, control the magnitude and distribution \n",
      "of blood flow to skeletal muscle but regional differences in the regulation of vasomotor tone …\n",
      "\n",
      "… ); epitope corresponding to AA 1894-1973 in the cytosolic modulatory domain • InsP3R1, \n",
      "UC Davis/NIH NeuroMab Facility, clone L24/18; epitope against AA 2680-2749 in C-terminus …\n",
      "\n",
      "… -trisphosphate receptor 1 (ITPR1), clone L24/18, NeuroMab, 1:1000 dilution; mouse monoclonal \n",
      "anti- Kv3.1b, KCNC1, clone N16B/8, NeuroMab, 1:2000 dilution; mouse monoclonal anti…\n",
      "\n",
      "… The PVDF membrane was incubated in mouse anti-IP3R1 antibody (1:3, mouse monoclonal \n",
      "tissue culture supernatant, clone L24/18; UC Davis/NINDS/NIMH NeuroMab Facility, Davis, …\n",
      "\n",
      "Intrauterine growth restriction (IUGR) affects vascular reactivity in older rats, but at present \n",
      "the causative factors for this change are unknown. Therefore, we investigated downstream …\n",
      "\n",
      "Axon degeneration is a critical pathological feature of many neurodegenerative diseases. \n",
      "Misregulation of local axonal ion homeostasis has been recognized as an important yet …\n",
      "\n",
      "… sc-575; 1:1,000), rabbit antibody against Calbindin d-28 (Swant CB38; 1:1,000), mouse \n",
      "monoclonal antibody against IP3R1 (NeuroMab, clone L24-18; 1:200) (34), mouse monoclonal …\n",
      "\n",
      "The classical pathway linking stimulation by neurotransmitters and hormones to changes in \n",
      "phosphoinositide metabolism and the subsequent InsP 3 induced Ca 2+ release was in …\n",
      "\n",
      "The classical pathway linking stimulation by neurotransmitters and hormones to changes in \n",
      "phosphoinositide metabolism and the subsequent InsP 3 induced Ca 2+ release was in …\n",
      "\n",
      "… : anti-Slick antibody 1:500 (clone N11/33, IgG1 subtype; NeuroMab) or anti-Slack antibody \n",
      "1:400 (clone N3/26, IgG1 subtype; NeuroMab). After being washed three times for 10 minutes …\n",
      "\n",
      "… For each experiment, 40 µg of either anti-Slick (clone N11/33, Neuromab) or anti-Slack (clone \n",
      "N3/26, Neuromab) channel antibody was used and 2 mg of synaptic plasma membranes …\n",
      "\n",
      "… N3/26, dilution 1 : 50; NeuroMab, Davis, CA, USA) and mouse monoclonal Slick antibody \n",
      "(clone N11/33, dilution 1 : 50; NeuroMab). We also used antibodies against neurophysin II (…\n",
      "\n",
      "… KO mice were probed with N11/33 anti-Slo2.1 antibody (… should be recognized by the \n",
      "NeuroMab antibody, but is not … contain the sequence recognized by the NeuroMab anti-Slo2.2 …\n",
      "\n",
      "… potassium channel, clone N11/33, UC Davis/NIH NeuroMab Facility) and Slack (Anti-Slo2.2 \n",
      "sodium-activated potassium channel, clone N3/26, UC Davis/NIH NeuroMab Facility). For …\n",
      "\n",
      "Background Slick, a sodium-activated potassium channel, has been recently identified in \n",
      "somatosensory pathways, but its functional role is poorly understood. The authors of this study …\n",
      "\n",
      "… (NeuroMab, 1:50, clone N11/33). F-actin was labeled with Alexa Fluor 633 Phalloidin (Molecular \n",
      "Probes, 1:200). To determine the clonogenic potential of MB cells, cells were plated at …\n",
      "\n",
      "The GABA-B receptor is densely expressed throughout the brain and has been implicated in \n",
      "many CNS functions and disorders, including addiction, epilepsy, spasticity, schizophrenia, …\n",
      "\n",
      "… , N6/38) from NeuroMab; mouse anti-SATB2 (clone, 4B19) from Abcam; rabbit anti-fluorescein \n",
      "(Life Technologies); and rabbit anti-protein kinase C alpha (P4334) from Sigma. Rabbit …\n",
      "\n",
      "Central nervous system GABAergic and glycinergic synaptic activity switches from postsynaptic \n",
      "excitation to inhibition during the stage when motor neuron numbers are being reduced, …\n",
      "\n",
      "… /34), as well as the mouse monoclonal antibody for VAChT (N6/38) were purchased from \n",
      "Antibodies Incorporated (UC Davis/NIH NeuroMab Facility, Davis, CA). Additional antibodies …\n",
      "\n",
      "… , N6/38) from NeuroMab (Davis, CA); mouse anti-protein kinase C- α (PKCα) (clone MC5) \n",
      "and rabbit anti-PKCα (P4334) from Sigma; rabbit anti-Opn4 from Thermo Fisher (PA1-780), …\n",
      "\n",
      "… Mouse monoclonal antibodies against vesicular acetylcholine transporter (VAChT; clone \n",
      "N6/38) were from UC Davis/NIH NeuroMab Facility. Rabbit polyclonal antibodies against …\n",
      "\n",
      "Expansion of CAG trinucleotide repeats in ATXN1 causes spinocerebellar ataxia type 1 (SCA1), \n",
      "a neurodegenerative disease that impairs coordination and cognition. While ATXN1 is …\n",
      "\n",
      "… with 0.1% Triton X-100 in Tris-buffered saline (TBS), blocked in 5% horse serum in TBS (vehicle), \n",
      "and incubated overnight in vehicle containing K55/7 (anti-KChIP1) or K75/30 (anti-Kv4.…\n",
      "\n",
      "… Following the blocking step, membranes were washed with Tris-buffered saline/Tween 20 \n",
      "and then incubated with an anti-Gα s monoclonal antibody (NeuroMab clone N192/12, Davis, …\n",
      "\n",
      "… Following the blocking step, membranes were washed with Tris-buffered saline/Tween 20 \n",
      "and then incubated with an anti-Gsα monoclonal antibody (NeuroMab clone N192/12, Davis, …\n",
      "\n",
      "… N192/12; NeuroMab) and subsequently with goat anti-mouse or anti-rabbit IgG–HRP \n",
      "secondary antibodies (1:10,000; Antibodies Inc.). Immunoreactive proteins on membranes were …\n",
      "\n",
      "It is uncertain if downregulation of β-adrenoceptor signaling pathway is promoted by an \n",
      "enhanced adrenergic tone at an early stage of cardiac disease, or it develops secondary to …\n",
      "\n",
      "… Antibodies used were: anti-Gα s monoclonal antibody (NeuroMab clone N192/12, Davis, \n",
      "CA, USA, catalog #75-211), caveolin-1 (BD Biosci #610059), β-actin (SIGMA Clone AC-74), p-…\n",
      "\n",
      "… Mouse monoclonal anti-G s protein α subunit (clone N192/12) antibody was from NeuroMab \n",
      "(Davis, CA). Rhodamine-conjugated goat anti-mouse antibody (10 mg/ml) was purchased …\n",
      "\n",
      "A brainstem circuit for encoding the spatial location of sounds involves neurons in the cochlear \n",
      "nucleus that project to medial superior olivary (MSO) neurons on both sides of the brain …\n",
      "\n",
      "… 36, UC Davis/NIH NeuroMab facility, Davis, CA, USA); rabbit anti-GAPDH (Sigma-Aldrich, \n",
      "St. Louis, MO, USA); mouse monoclonal anti-Caspr (K65/35, UC Davis/NIH NeuroMab facility); …\n",
      "\n",
      "… After further rinsing in PBS, the sections were immunostained with 1:500 NeuroMab clone \n",
      "K65/35 antibody in PBS with a standard block solution [0.3% Triton X-100 (Sigma), 5% normal …\n",
      "\n",
      "The peripheral auditory and vestibular systems rely on sensorineural structures that are \n",
      "vulnerable to ototoxic agents that cause hearing loss and/or equilibrium deficits. Although …\n",
      "\n",
      "… was obtained from the UC Davis/NIH NeuroMab Facility. It was generated using a fusion … \n",
      "antibody was obtained from the UC Davis/NIH NeuroMab Facility and was used in Figures 4-7. …\n",
      "\n",
      "… We used the mouse monoclonal anti-Caspr1 (clon K65/35; used at 1:400 dilution) antibody \n",
      "from the UC Davis/NIH NeuroMab Facility. Other mouse monoclonal antibodies used were …\n",
      "\n",
      "A rapidly progressing aging society has raised attention to white matter lesions in Alzheimer’s \n",
      "disease. In the present study, we applied an AD plus cerebral hypoperfusion (HP) mouse …\n",
      "\n",
      "… Sections were then immunostained as described 51 using mouse antibodies against HCN1 \n",
      "(clone N70/28, NeuroMab), PanNav 52 , or Caspr (clone K65/35, NeuroMab), and rabbit …\n",
      "\n",
      "Myelinated axons have a distinct protein architecture essential for action potential propagation, \n",
      "neuronal communication, and maintaining cognitive function. Damage to myelinated …\n",
      "\n",
      "10.7554/eLife.23329.001 Myelination speeds conduction of the nerve impulse, enhancing \n",
      "cognitive power. Changes of white matter structure contribute to learning, and are often …\n",
      "\n",
      "Leukemia/lymphoma‐related factor (LRF), a zinc‐finger transcription factor encoded by Zbtb7a, \n",
      "is a protooncogene that regulates differentiation in diverse cell lineages, and in the CNS, …\n",
      "\n",
      "The ENU-induced neurological mutant ataxia3 was mapped to distal mouse chromosome \n",
      "15. Sequencing of the positional candidate gene Scn8a encoding the sodium channel Na v …\n",
      "\n",
      "… K+ channel, clone K14/16 and anti-Caspr, clone K65/35 were obtained from the UC Davis/NINDS/NIMH \n",
      "NeuroMab Facility, supported by NIH grant U24NS050606 and maintained by …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… (A) Representative images of the sensory epithelia showing cristae immunolabeled with the \n",
      "K65/35 anti-CASPR1 antibody (green) and the AB19013 anti-tenascin antibody (red). In the …\n",
      "\n",
      "… Trimmer, University of California, Davis, CA) and/or mouse monoclonal anti-Caspr (K65/35) \n",
      "antibody (1:500 dilution in vehicle; UC Davis/NIH NeuroMab Facility). Subsequently, …\n",
      "\n",
      "… (Alomone labs ASC-009) or mouse anti-Caspr (NeuroMab K65/35). Sections were then \n",
      "incubated for 2 h with their respective fluorescent secondary antibodies (ThermoFisher highly …\n",
      "\n",
      "… K65/35 (obtained from the UC Davis/NIH NeuroMab Facility, University of California), \n",
      "rabbit anti-human myelin basic protein (MBP) polyclonal antibodies (Dako), mouse anti-human …\n",
      "\n",
      "Congenital peripheral nerve hyperexcitability (PNH) is usually associated with impaired \n",
      "function of voltage-gated K + channels (VGKCs) in neuromyotonia and demyelination in …\n",
      "\n",
      "… /paranodin clone K65/35 (1:200; NeuroMab). Primary antibodies were detected with Alexa \n",
      "Fluor dye-conjugated (1:1000; Invitrogen) or Rhodamine Red-X-conjugated (1:1000; Jackson …\n",
      "\n",
      "… The monoclonal antibody mouse anti-caspr (clone K65/35) was developed by and obtained \n",
      "from the UC Davis/National Institutes of Health Neuromab Facility. Sections were washed in …\n",
      "\n",
      "Background: ATP-sensitive potassium (K ATP ) channels in neurons mediate neuroprotection, \n",
      "they regulate membrane excitability, and they control neurotransmitter release. Because …\n",
      "\n",
      "Brain enriched voltage-gated sodium channel (VGSC) Na v 1.2 and Na v 1.6 are critical for \n",
      "electrical signaling in the CNS. Previous studies have extensively characterized cell-type-…\n",
      "\n",
      "Changes to the structure of nodes of Ranvier in the normal-appearing white matter (NAWM) \n",
      "of multiple sclerosis (MS) brains are associated with chronic inflammation. We show that the …\n",
      "\n",
      "… -ankyrin G antibody (clone N106/36) and Caspr antibody (clone K65/35) were from University \n",
      "of California at Davis/NIH NeuroMab Facility (Davis, CA). Rabbit anti-GFP was purchased …\n",
      "\n",
      "Excessive activation of mTOR in microglia impairs CNS homeostasis and causes severe \n",
      "epilepsy. Autophagy constitutes an important part of mTOR signaling. The contribution of …\n",
      "\n",
      "… ; mouse monoclonal (IgG 2A ) anti-ankyrin-G, NeuroMab N106/36, 1:100; mouse monoclonal \n",
      "(IgG 1 ) anti-Caspr, NeuroMab K65/35, 1:100; chicken polyclonal anti-GFP, ThermoFisher …\n",
      "\n",
      "… The mouse anti-Kv 1.2 antibodies were obtained from Neuromab and have both been \n",
      "knock-out validated. The main Caspr antibody used was rabbit anti-Caspr from Abcam. The …\n",
      "\n",
      "Background: Modulation of M-type currents has been proposed as a new strategy for the \n",
      "treatment of neuropathic pain due to their role in regulating neuronal excitability. Using …\n",
      "\n",
      "Neuron-glia interactions establish functional membrane domains along myelinated axons. \n",
      "These include nodes of Ranvier, paranodal axoglial junctions and juxtaparanodes. …\n",
      "\n",
      "Damage to long axons in white matter tracts is a major pathology in closed head traumatic \n",
      "brain injury (TBI). Acute TBI treatments are needed that protect against axon damage and …\n",
      "\n",
      "The velocity of nerve conduction is moderately enhanced by larger axonal diameters and \n",
      "potently sped up by myelination of axons. Myelination thus allows rapid impulse propagation …\n",
      "\n",
      "Mesenchymal stem cell (MSC)-secreted factors have been shown to significantly promote \n",
      "oligodendrogenesis from cultured primary adult neural stem cells (aNSCs) and …\n",
      "\n",
      "Abstract Changes to the structure of nodes of Ranvier in the normal-appearing white matter (NAWM) \n",
      "of MS brains are associated with chronic inflammation. We show that the paranodal …\n",
      "\n",
      "… (subtype IgG 2b ; clone K14/16) and Caspr (subtype IgG 1 ; clone K65/35) were obtained \n",
      "from the UC Davis/NIH NeuroMab Facility. Mouse monoclonal antibodies were detected by the …\n",
      "\n",
      "… K65/35, NeuroMab, 1:2000 dilution; rabbit polyclonal antimyelin oligodendrocyte glycoprotein \n",
      "(MOG), ab32760, 1:300 dilution; mouse monoclonal anti-GFAP, clone N206A/8, NeuroMab…\n",
      "\n",
      "To examine the role of gray and white matter niches for oligodendrocyte differentiation, we \n",
      "used homo- and heterotopic transplantations into the adult mouse cerebral cortex. White …\n",
      "\n",
      "The scaffolding protein ankyrin-G is required for Na + channel clustering at axon initial \n",
      "segments. It is also considered essential for Na + channel clustering at nodes of Ranvier to …\n",
      "\n",
      "Action potential initiation and propagation in myelinated axons require ion channel clustering \n",
      "at axon initial segments (AIS) and nodes of Ranvier. Disruption of these domains after …\n",
      "\n",
      "… (K65/35 mouse monoclonal IgG 1 , 1:100; NeuroMab, Davis, CA). Images were acquired \n",
      "with a Zeiss LSM 880, using a Plan-Apochromat 63 × /1.40 Oil DIC objective (Carl Zeiss, Jena, …\n",
      "\n",
      "Simultaneous labeling of multiple targets in a single sample, or multiplexing, is a powerful \n",
      "approach to directly compare the amount, localization and/or molecular properties of different …\n",
      "\n",
      "… Rabbit anti-Na v 1.6 (1:500 Alomone Labs), mouse anti-CASPR (Clone K65/35; 1:200 \n",
      "NeuroMab) primary antibodies were diluted in PBS blocking solution [0.1% (v/v) Triton X-100 and …\n",
      "\n",
      "Myelin is essential for rapid and efficient action potential propagation in vertebrates. However, \n",
      "the molecular mechanisms regulating myelination remain incompletely characterized. For …\n",
      "\n",
      "Neuronal GABAergic responses switch from excitatory to inhibitory at an early postnatal \n",
      "period in rodents. The timing of this switch is controlled by intracellular Cl− concentrations, but …\n",
      "\n",
      "… Peter Brophy), type Ⅵ sodium channel (Nav1.6, 1:200, AB5580, Millipore), contactin-associated \n",
      "protein (Caspr, 1:100, clone K65/35, UC Davies NIH NeuroMab), ionized calcium …\n",
      "\n",
      "Central nervous system myelination increases action potential conduction velocity. However, \n",
      "it is unclear how myelination is coordinated to ensure the temporally precise arrival of …\n",
      "\n",
      "Myelin, the electrically insulating sheath on axons, undergoes dynamic changes over time. \n",
      "However, it is composed of proteins with long lifetimes. This raises the question how such a …\n",
      "\n",
      "Cerebral white matter injury (WMI) persistently disrupts myelin regeneration by oligodendrocyte \n",
      "progenitor cells (OPCs). We identified a specific bioactive hyaluronan fragment (bHAf) …\n",
      "\n",
      "Neuregulin 1 type III is processed following regulated intramembrane proteolysis, which allows \n",
      "communication from the plasma membrane to the nucleus. We found that the intracellular …\n",
      "\n",
      "We combined translating ribosome affinity purification (TRAP) with in utero electroporation (IUE), \n",
      "called iTRAP to identify the molecular profile of specific neuronal populations during …\n",
      "\n",
      "The pathogenesis of peripheral neuropathies in adults is linked to maintenance mechanisms \n",
      "that are not well understood. Here, we elucidate a novel critical maintenance mechanism …\n",
      "\n",
      "Translational readthrough-inducing agents have been developed for the treatment of nonsense \n",
      "mutations in hereditary diseases. The clinical effectiveness of readthrough agents has …\n",
      "\n",
      "Background Multiple sclerosis is characterised by inflammation, oligodendrocyte loss and \n",
      "axonal demyelination and shows an additional impact on astrocytes, and their polarization. …\n",
      "\n",
      "Charcot–Marie–Tooth (CMT) disease is a hereditary neuropathy mainly caused by gene \n",
      "mutation of peripheral myelin proteins including myelin protein zero (P0, MPZ). Large myelin …\n",
      "\n",
      "10.7554/eLife.23332.001 Impairment of peripheral nerve function is frequent in neurometabolic \n",
      "diseases, but mechanistically not well understood. Here, we report a novel disease …\n",
      "\n",
      "Oligodendrocytes, the myelinating cells of the CNS, promote rapid action potential conduction \n",
      "along axons. Changes in the geometry of gaps between myelin segments, known as …\n",
      "\n",
      "Axons must withstand mechanical forces, including tension, torsion, and compression. \n",
      "Spectrins and actin form a periodic cytoskeleton proposed to protect axons against these forces. …\n",
      "\n",
      "Microglia are critical in damage/repair processes during ischemic white matter injury (WMI). \n",
      "Voltage‐gated proton channel (Hv1) is expressed in microglia and contributes to …\n",
      "\n",
      "Movement and posture depend on sensory feedback that is regulated by specialized \n",
      "GABAergic neurons (GAD2 + ) that form axo-axonic contacts onto myelinated proprioceptive …\n",
      "\n",
      "… anti-CASPR1 (clone K65/35, RRID: AB_2083496, used at 1:400 dilution) and anti-PSD95 \n",
      "(clone K28/43, RRID: AB_2292909, 1:100) from the UC Davis/NIH NeuroMab Facility; mouse …\n",
      "\n",
      "… -associated protein (Caspr1) (clone K65/35, IgG1, cat. # 75–001) and anti-PSD95 (clone \n",
      "K28/43, IgG2a, cat# 75–028) from Neuromab. We used the following secondary antibodies …\n",
      "\n",
      "Astrocytes, once considered passive support cells, are increasingly appreciated as dynamic \n",
      "regulators of neuronal development and function, in part via secreted factors. The extent to …\n",
      "\n",
      "Movement and posture depend on sensory feedback that is regulated by specialized GABAergic \n",
      "neurons (GAD2+) that form axo-axonic contacts onto myelinated proprioceptive sensory …\n",
      "\n",
      "Excitatory and inhibitory neurons are connected into microcircuits that generate circuit output. \n",
      "Central in the hippocampal CA3 microcircuit is the mossy fiber (MF) synapse, which …\n",
      "\n",
      "Tau is a major driver of neurodegeneration and is implicated in over 20 diseases. Tauopathies \n",
      "are characterized by synaptic loss and neuroinflammation, but it is unclear if these …\n",
      "\n",
      "GABA A receptors mediate fast inhibitory transmission in the brain, and their number can be \n",
      "rapidly up- or downregulated to alter synaptic strength. Neuroligin-2 plays a critical role in …\n",
      "\n",
      "… 1b NeuroMab N16B/8 and anti-HCN2 NeuroMab N71/37) that exhibit preferential \n",
      "immunoreactivity against rodent samples. We also perform IB analyses against samples from KO or …\n",
      "\n",
      "… 1b (N16b/8), and anti-HA revealed strong staining of a … N16b/8-associated pool. By contrast, \n",
      "cells expressing Kv3.1b-N229Q-HA or Kv3.1b-N220Q/N229Q-HA exhibited only N16b/8-…\n",
      "\n",
      "The Kv3.1 channel plays a crucial role in regulating the high-frequency firing properties of \n",
      "neurons. Here, we determined whether Src regulates the subcellular distributions of the Kv3.…\n",
      "\n",
      "Despite major advances in neuroscience, a comprehensive understanding of the structural \n",
      "and functional components of the adult brain compartments remains to be fully elucidated at …\n",
      "\n",
      "… 1b (subtype IgG 1 ; clone N16b/8), and PSD95 (subtype IgG 2a ; clone K28/43) were \n",
      "obtained from the University of California Davis/National Institutes of Health NeuroMab Facility. …\n",
      "\n",
      "… 1b (clone N16B/8) a single band of 110 kDa. For calretinin, a 29 kDa single band was detected \n",
      "and for α-tubulin a single band of ~55 kDa. The specific pattern of cellular labeling for the …\n",
      "\n",
      "The conventional kinesin motor transports many different cargos to specific locations in neurons. \n",
      "How cargos regulate motor function remains unclear. Here we focus on KIF5, the heavy …\n",
      "\n",
      "Neuronal ankyrins cluster and link membrane proteins to the actin and spectrin-based \n",
      "cytoskeleton. Among the three vertebrate ankyrins, little is known about neuronal Ankyrin-R (AnkR…\n",
      "\n",
      "… Labeling for both proteins is in contrast to that for mAb N147/6 (IgG2b) which recognizes all \n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and …\n",
      "\n",
      "A hallmark of inflammatory diseases is the excessive recruitment and influx of monocytes to \n",
      "sites of tissue damage and their ensuing differentiation into macrophages. Numerous stimuli …\n",
      "\n",
      "Mammalian Quaking (QKI) and its Caenorhabditis elegans homolog, GLD-1 (defective in \n",
      "germ line development), are evolutionarily conserved RNA-binding proteins, which post-…\n",
      "\n",
      "Simultaneous labeling of multiple targets in a single sample, or multiplexing, is a powerful \n",
      "approach to directly compare the amount, localization and/or molecular properties of different …\n",
      "\n",
      "Quaking (QKI) is an RNA-binding protein (RBP) involved in multiple aspects of RNA \n",
      "metabolism and many biological processes. Despite a known immune function in regulating …\n",
      "\n",
      "… (clone N147/6; UC Davis/NIH NeuroMab Facility) and smooth muscle α-actin (ACTA2; \n",
      "clone 1A4; Sigma-Aldrich, Zwijndrecht, the Netherlands). Antibody binding was visualized with …\n",
      "\n",
      "Quaking (QKI) controls RNA metabolism in many biological processes including innate \n",
      "immunity, where its roles remain incompletely understood. To illuminate these roles, we …\n",
      "\n",
      "Mutations in human tuberous sclerosis complex (TSC) genes TSC1 and TSC2 are the leading \n",
      "causes of developmental brain abnormalities and large tumors in other tissues. Murine …\n",
      "\n",
      "Angiogenesis is critical to cancer development and metastasis. However, anti-angiogenic \n",
      "agents have only had modest therapeutic success, partly due to an incomplete understanding …\n",
      "\n",
      "Herpesvirus type 1 (HSV-1) based oncolytic vectors arise as a promising therapeutic alternative \n",
      "for neoplastic diseases including hepatocellular carcinoma. However, the mechanisms …\n",
      "\n",
      "The region surrounding the central canal (CC) of the spinal cord is a highly plastic area, \n",
      "defined as a postnatal neurogenic niche. Within this region are ependymal cells that can …\n",
      "\n",
      "Alternative splicing (AS) is a key component of gene expression programs that drive cellular \n",
      "differentiation. Smooth muscle cells (SMCs) are important in the function of a number of …\n",
      "\n",
      "… Health (NIMH) NeuroMab facility/Antibodies, Inc., clone N147/6). Total RNA of transfected \n",
      "cells was extracted using TRIzol reagent according to the instructions of the manufacturer. …\n",
      "\n",
      "Alternative splicing is critical for animal ontogeny; however, its role in the earliest developmental \n",
      "decision, the specification of the three embryonic germ layers, is poorly understood. By …\n",
      "\n",
      "Oligodendrocytes secrete vesicles into the extracellular space, where they might play a role \n",
      "in neuron–glia communication. These exosomes are small vesicles with a diameter of 50–…\n",
      "\n",
      "Over the last decade, a large variety of alterations of the CNTNAP2 gene, encoding Caspr2, \n",
      "have been identified in several neuronal disorders, including neurodevelopmental disorders …\n",
      "\n",
      "… anti-GFP (1:2000, clone N86/38; NeuroMab) overnight at 4C followed by donkey antimouse-… \n",
      "the following antibodies: 1) mouse anti-GFP (NeuroMab clone N86/38, 1:2000, 12–16 h at 4…\n",
      "\n",
      "The mammalian retina consists of five major classes of neuronal cells, as well as glial cells, \n",
      "and it contains more than 50 cell types. The ability to manipulate gene expression in specific …\n",
      "\n",
      "… 16 μg of mouse monoclonal anti-AnkB N105/17 and anti-GFP N86/38 antibodies (for \n",
      "sciatic nerves) and 16 μg of mouse monoclonal anti-AnkG N106/36 and anti-GFP N86/38 …\n",
      "\n",
      "… Lysate was centrifuged to separate insoluble material and protein extract was added to \n",
      "Protein A Sepharose beads conjugated to mouse anti-GFP (NeuroMab N86/38). Lysate-bead …\n",
      "\n",
      "Aggregation of the multifunctional RNA‐binding protein TDP‐43 defines large subgroups of \n",
      "amyotrophic lateral sclerosis and frontotemporal dementia and correlates with …\n",
      "\n",
      "PSD-95, a membrane-associated guanylate kinase, is the major scaffolding protein in the \n",
      "excitatory postsynaptic density (PSD) and a potent regulator of synaptic strength. Here we …\n",
      "\n",
      "… Mouse anti-GFP (University of California at Davis/NIH NeuroMab Facility, Davis, CA; \n",
      "abbreviated NeuroMab; Cat. no. 75-132, clone N86/38, RRID:AB_10672529) was generated …\n",
      "\n",
      "… , NeuroMab, clone N106/36), and mouse monoclonal anti-GFP (1∶1000, NeuroMab, clone \n",
      "N86/38). … The AnkyrinG antibody is a validated NeuroMab antibody and in our experience it …\n",
      "\n",
      "Sensory hair cells in the zebrafish lateral line regenerate rapidly and completely after \n",
      "damage. Previous studies have used a variety of ototoxins to kill lateral line hair cells to study …\n",
      "\n",
      "Oligodendrocytes produce multi-lamellar myelin membranes that surround axons in the \n",
      "central nervous system (CNS). Preservation and generation of myelin are potential therapeutic …\n",
      "\n",
      "… Staining for tau utilized mouse monoclonal Tau5 antibody [25] and mouse monoclonal \n",
      "anti-GFP (NeuroMab clone N86/38; also recognizes Venus) with secondary Licor antibodies: …\n",
      "\n",
      "Transport of LDL-derived cholesterol from lysosomes into the cytoplasm requires NPC1 \n",
      "protein; NPC1L1 mediates uptake of dietary cholesterol. We introduced single disulfide bonds …\n",
      "\n",
      "Measuring levels of mRNAs in the process of translation in individual cells provides \n",
      "information on the proteins involved in cellular functions at a given point in time. The protocol …\n",
      "\n",
      "… primary antibody (specific to full GFP, monoclonal, NeuroMab, clone N86/38, 1:200), anti-m \n",
      "Alexa 488 (molecular probes, 1:250). To generate heat shock flip clones, we crossed the …\n",
      "\n",
      "The Kv2.1 voltage‐gated K + channel is widely expressed throughout mammalian brain, \n",
      "where it contributes to dynamic activity‐dependent regulation of intrinsic neuronal excitability. …\n",
      "\n",
      "… ), mouse (IgG2a) anti-GFP clone N86/38 (MitoP line CFP reporter amplification in type 1 \n",
      "BC, UC Davis/ NIH NeuroMab; P42212). Secondary Antibodies were acquired from Jackson …\n",
      "\n",
      "Slo2 channels are large-conductance potassium channels abundantly expressed in the \n",
      "nervous system. However, it is unclear how their expression level in neurons is regulated. Here …\n",
      "\n",
      "… Primary antibodies used include the following: mouse monoclonal anti-GFP (N86/38, \n",
      "NeuroMab) and anti-beta-Actin HRP conjugate (13E5, Cell Signaling). …\n",
      "\n",
      "… (NeuroMab αGFP, clone N86/38). Lysate-bead slurry was then incubated overnight at 4 C \n",
      "followed by washing in Wash Buffer (150 mM NaCl, 0.05% Triton X-100, 50 mM Tris, 5 mM …\n",
      "\n",
      "… N86/38 (UC Davis/NIH NeuroMab Facility, nd). This mAb binds to both GFP and the GFP \n",
      "derivative YFP. The sequence (derived by PCR cloning from hybridoma cells (Andrews et al., …\n",
      "\n",
      "Amyloid precursor protein (APP) plays a pivotal role in Alzheimer’s disease (AD) pathogenesis, \n",
      "but its normal physiological functions are less clear. Combined deletion of the APP and …\n",
      "\n",
      "The voltage-gated Na + channel β1 subunit, encoded by SCN1B, regulates cell surface \n",
      "expression and gating of α subunits, and participates in cell adhesion. β1 is cleaved by α/β and γ-…\n",
      "\n",
      "… Dtr1p-GFP was detected with N86/38 (NeuroMab, Davis, CA) followed by horseradish \n",
      "peroxidase-conjugated goat anti-rabbit secondary antibody at a 1:1,000 dilution (Antibody …\n",
      "\n",
      "Frontotemporal dementia and amyotrophic lateral sclerosis patients with C9orf72 mutation \n",
      "show cytoplasmic poly-GR and poly-PR aggregates. Short poly-(Gly-Arg) and poly-(Pro-Arg) (…\n",
      "\n",
      "The scaffolding protein ankyrin-G is required for Na + channel clustering at axon initial \n",
      "segments. It is also considered essential for Na + channel clustering at nodes of Ranvier to …\n",
      "\n",
      "Lysine-specific demethylase 1 (LSD1) has been reported to repress and activate transcription \n",
      "by mediating histone H3K4me1/2 and H3K9me1/2 demethylation, respectively. The …\n",
      "\n",
      "… ; anti-GFP N86/38 IgG2a: 1:5 dilution of tissue culture supernatant, RRID: AB_10671955; \n",
      "both from the UC Davis/National Institutes of Health [NIH] NeuroMab Facility; anti-HA, RRID: …\n",
      "\n",
      "Stem cells reside in a niche, a local environment whose cellular and molecular complexity is \n",
      "still being elucidated. In Drosophila ovaries, germline stem cells depend on cap cells for self…\n",
      "\n",
      "10.7554/eLife.40202.001 Parkinson’s disease-associated LRRK2 kinase phosphorylates \n",
      "multiple Rab GTPases, including Rab8A and Rab10. We show here that LRRK2 kinase …\n",
      "\n",
      "Background Machado–Joseph Disease (MJD), a form of dominantly inherited ataxia belonging \n",
      "to the group of polyQ expansion neurodegenerative disorders, occurs when a threshold …\n",
      "\n",
      "… Primary antibodies used were: mouse anti-GFP (clone N86/38, 1:25; UC Davis/National \n",
      "Institutes of Health NeuroMab Facility), rat anti-HA (3F10, 1:500; Roche), mouse anti-Wg (4D4, 1:…\n",
      "\n",
      "Memory is the process by which acquired information is encoded, stored and retrieved if \n",
      "needed. It is absolutely essential for managing our everyday lives and chores and, even more …\n",
      "\n",
      "The olfactory pathway integrates the odor information required to generate correct behavioral \n",
      "responses. To address how changes of serotonin signaling in two contralaterally projecting…\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… For visualization of reconstituted GFP (Figure 3I–J), we used mouse anti-GFP AB (Clone \n",
      "N86/38, Neuro-Mab, Antibodies Inc., 1:100) followed by goat anti-mouse Alexa 488 (Invitrogen, …\n",
      "\n",
      "BACKGROUND: Machado-Joseph Disease (MJD), a form of dominantly inherited ataxia \n",
      "belonging to the group of polyQ expansion neurodegenerative disorders, occurs when a …\n",
      "\n",
      "The strength of synaptic inhibition can be controlled by the stability and endocytosis of \n",
      "surface and synaptic GABA A receptors (GABA A Rs), but the surface receptor dynamics that …\n",
      "\n",
      "The C9orf72 repeat expansion causes amyotrophic lateral sclerosis and frontotemporal \n",
      "dementia, but the poor correlation between C9orf72‐specific pathology and TDP ‐43 pathology …\n",
      "\n",
      "The voltage-gated Na+ channel β1 subunit, encoded by SCN1B, regulates cell surface \n",
      "expression and gating of α subunits, and participates in cell adhesion. β1 is cleaved by α/β and γ-…\n",
      "\n",
      "… Studies Hybridoma Bank, 1:200), Living Colors® rabbit anti-DsRed (Clontech, 1:200) and \n",
      "75–132 anti-GFP primary antibody (specific to full GFP, monoclonal, NeuroMab, clone N86/38, …\n",
      "\n",
      "Fibroblast growth factors (FGF) are expressed in many non–small cell lung carcinoma (NSCLC) \n",
      "primary tumors and derived cell lines, and mutations in FGF receptor 3 (FGFR3) have …\n",
      "\n",
      "A single injection of N-methyl-D-aspartate receptor (NMDAR) antagonists produces a rapid \n",
      "antidepressant response. Lasting changes in the synapse structure and composition …\n",
      "\n",
      "… /49) and GABA B R2 (clone N81/2) antibodies were obtained from the University of California, \n",
      "Davis, CA/National Institutes of Health NeuroMab Facility, which is supported by National …\n",
      "\n",
      "… The monoclonal anti-PSD95 (clone K28/43), GABA B R1 (clone NR3A/49) and GABA B R2 \n",
      "(clone N81/2) antibodies were obtained from the UC Davis/NIH NeuroMab Facility, supported …\n",
      "\n",
      "G-protein-gated inwardly rectifying potassium (GIRK) channels, which help control neuronal \n",
      "excitability, are important for the response to drugs of abuse. Here, we describe a novel …\n",
      "\n",
      "The behavioral and anatomical deficits seen in fragile X syndrome (FXS) are widely \n",
      "believed to result from imbalances in the relative strengths of excitatory and inhibitory …\n",
      "\n",
      "… Mouse monoclonal antibodies were developed by UC Davis/NIH NeuroMab Facility in mouse \n",
      "generating hybridomas. The anti-GABAAR α1 mouse monoclonal antibody (clone N95/35 …\n",
      "\n",
      "Microglia, the resident immune cells of the brain, have emerged as crucial regulators of synaptic \n",
      "refinement and brain wiring. However, whether the remodeling of distinct synapse types …\n",
      "\n",
      "… L51/82 was primarily in a 100-kilodalton molecular weight form, corresponding to pro-MMP-9 \n",
      "(Figure 2A). L51/82 … A mouse monoclonal antibody (L51/82; UC Davis/NJMH NeuroMab …\n",
      "\n",
      "Microribonucleic acids (miRNAs) work with exquisite specificity and are able to distinguish a \n",
      "target from a non-target based on a single nucleotide mismatch in the core nucleotide …\n",
      "\n",
      "… A Western blot of culture supernatant with the anti-MMP-9 (1:1000 in 1% milk TBST, Clone \n",
      "L51/82, NeuroMab, Davis, CA) antibody followed by densitometric quantification documented …\n",
      "\n",
      "… The MMP-9 antisera from Torrey Pines (B) and the antibody from Neuro Mab (C) both \n",
      "recognized one recombinant pro band at 92 kDa and two recombinant active bands at 82 and 68 …\n",
      "\n",
      "Aberrant activation of matrix metalloproteinases (MMPs) is a common feature of pathological \n",
      "cascades observed in diverse disorders, such as cancer, fibrosis, immune dysregulation, …\n",
      "\n",
      "… incubated with K74/71 alone, with K74/71 preincubated with … as a nonspecific blocker, \n",
      "or with K74/71 preincubated with 10× … /NINDS/NIMH NeuroMab facility (http://www.neuromab.org). …\n",
      "\n",
      "The voltage-sensitive sodium channel Na V 1.1 plays a critical role in regulating excitability \n",
      "of GABAergic neurons and mutations in the corresponding gene are associated to Dravet …\n",
      "\n",
      "The axon initial segment (AIS) contains the site of action potential initiation and plays a \n",
      "major role in neuronal excitability. AIS function relies on high concentrations of different ion …\n",
      "\n",
      "Dravet syndrome, an epileptic encephalopathy affecting children, largely results from \n",
      "heterozygous loss-of-function mutations in the brain voltage-gated sodium channel gene SCN1A. …\n",
      "\n",
      "The exact site of initiation and shape of action potentials vary among different neuronal types. \n",
      "The reason for this variability is largely unknown, but the subunit composition, density and …\n",
      "\n",
      "Purpose Intracellular free calcium ions (Ca 2+) are an important element in retinal ganglion \n",
      "cell response. Two major EF-hand (E-helix-loop-F-helix-hand) calcium binding proteins in …\n",
      "\n",
      "… antibodies directed against Na V 1.1 (K74/71, mouse, IgG1, 1:100, NeuroMab) and \n",
      "parvalbumin (L114/3, 75-455, mouse, IgG2a, 1:100, NeuroMab). Subclass-specific secondary …\n",
      "\n",
      "Proteins that interact with voltage-gated sodium (Na v ) channels are important in channel \n",
      "sorting and modulation. In this study, we identified the transcriptional regulator, Sin3B, as a …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Purpose: Recent studies indicate the presence of functional voltage-gated sodium channels \n",
      "(Na v channels) in the distal retina in several species. This study examined the distribution …\n",
      "\n",
      "Purpose: Recent studies indicate the presence of functional voltage-gated sodium channels \n",
      "(Nav channels) in the distal retina in several species. This study examined the distribution of …\n",
      "\n",
      "GABAergic inhibitory interneurons of the cerebral cortex expressing vasoactive intestinal \n",
      "peptide (VIP-INs) are rapidly emerging as important regulators of network dynamics and normal …\n",
      "\n",
      "In neurons, voltage-gated sodium channel β subunits regulate the expression levels, subcellular \n",
      "localization, and electrophysiological properties of sodium channel α subunits. However…\n",
      "\n",
      "Autosomal dominant lateral temporal epilepsy (ADLTE) is an inherited syndrome caused by \n",
      "mutations in the leucine-rich glioma inactivated 1 (LGI1) gene. It is known that glutamatergic …\n",
      "\n",
      "Intracellular Fibroblast Growth Factor 14 (iFGF14) and the other intracellular FGFs (iFGF11-13) \n",
      "regulate the properties and densities of voltage-gated neuronal and cardiac Na + (Nav) …\n",
      "\n",
      "In the primate visual system, the ganglion cells of the magnocellular pathway underlie motion \n",
      "and flicker detection and are relatively transient, while the more sustained ganglion cells of …\n",
      "\n",
      "Hyperphosphorylated microtubule-associated protein tau has been implicated in dementia, \n",
      "epilepsy, and other neurological disorders. In contrast, site-specific phosphorylation of tau at …\n",
      "\n",
      "… ) and mouse monoclonal (clone K74/71, NeuroMab) antibodies raised respectively against \n",
      "… monoclonal (clone K87A/10, NeuroMab) antibodies raised respectively against amino acids …\n",
      "\n",
      "… The first employed mouse mAb K69/3, raised against a recombinant fusion protein corresponding \n",
      "to amino acids 1882−2005 of the cytoplasmic C-terminus of rat Nav1.2. The specificity …\n",
      "\n",
      "The axon initial segment (AIS) contains the site of action potential initiation and plays a \n",
      "major role in neuronal excitability. AIS function relies on high concentrations of different ion …\n",
      "\n",
      "… Na v 1.2 and ankyrin-G were labeled using monoclonal anti-Na v 1.2 sodium channel (clone \n",
      "K69/3; NeuroMab, Davis, CA) and rabbit polyclonal antiankyrin-G (Cat. No. SC-28561, …\n",
      "\n",
      "Voltage-gated sodium channels (Nav1.1–Nav1.9) are responsible for the initiation and \n",
      "propagation of action potentials in neurons, controlling firing patterns, synaptic transmission and …\n",
      "\n",
      "Alzheimer's disease (AD) is the most frequent neurodegenerative disease and a common \n",
      "dementia in elderly individuals. Previous studies found a strong correlation between axon …\n",
      "\n",
      "The axon initial segment (AIS) plays an important role in maintaining neuronal polarity and \n",
      "initiating action potentials (APs). The AIS adapts to its environment by changing its length and …\n",
      "\n",
      "Cerebellar granule (CG) cells generate high‐frequency action potentials that have been \n",
      "proposed to depend on the unique properties of their voltage‐gated ion channels. To address …\n",
      "\n",
      "Precise axon pathfinding is crucial for establishment of the initial neuronal network during \n",
      "development. Pioneer axons navigate without the help of preexisting axons and pave the way …\n",
      "\n",
      "… An affinity-purified monoclonal antibody against Na v 1.2 (clone K69/3) was obtained \n",
      "from NeuroMab (UC Davis/NIH NeuroMab facility, Davis, CA, USA). The characteristics and …\n",
      "\n",
      "… Monoclonal antibodies K74/71, K69/3, and K87A/10 were developed by and obtained from \n",
      "the University of California Davis/NINDS/NIMH NeuroMab Facility, supported by NIH grant …\n",
      "\n",
      "… -soluble RBM fraction for 13 h at 4 C with K69/3 mAb cross-linked to protein G-agarose beads… \n",
      "For sequential IP experiments, Nav1.2 was first purified with K69/3 mAb beads, and Nav1.2 …\n",
      "\n",
      "Proteins that interact with voltage-gated sodium (Na v ) channels are important in channel \n",
      "sorting and modulation. In this study, we identified the transcriptional regulator, Sin3B, as a …\n",
      "\n",
      "Maternal immune activation (MIA) and juvenile social isolation (SI) are two most prevalent \n",
      "and widely accepted environmental insults that could increase the propensity of psychiatric …\n",
      "\n",
      "Cerebrospinal fluid samples collected from children during initial presentation of central \n",
      "nervous system inflammation, who may or may not subsequently be diagnosed as having …\n",
      "\n",
      "People with epilepsy have greatly increased probability of premature mortality due to \n",
      "sudden unexpected death in epilepsy (SUDEP). Identifying which patients are most at risk of …\n",
      "\n",
      "… The anti-neurofascin antibody (extNF; A12/18, 75-172) and anti-Nav1.2 antibody (K69/3) \n",
      "were obtained from NeuroMab (USA). The anti-ankyrin G antibody (33-8800) was purchased …\n",
      "\n",
      "… The blocked sections were incubated overnight with primary antibodies: mouse monoclonal \n",
      "Nav1.2 clone K69/3, 1:300 (NeuroMab) and rabbit anti-Ankyrin G sc28561, 1:500 (Santa …\n",
      "\n",
      "Intrinsic excitability is a key feature dictating neuronal response to synaptic input. Here we \n",
      "investigate the recent observation that dentate granule neurons exhibit a more depolarized …\n",
      "\n",
      "Neurons implement a variety of plasticity mechanisms to alter their function over timescales \n",
      "ranging from seconds to days. One powerful means of controlling excitability is to directly …\n",
      "\n",
      "Key points The laserspritzer we developed activates axo‐axonic synapses (AASs) with \n",
      "spatial resolutions of less than 40 μm. AASs innervate the entire length of the axon initial …\n",
      "\n",
      "Intracellular Fibroblast Growth Factor 14 (iFGF14) and the other intracellular FGFs (iFGF11-13) \n",
      "regulate the properties and densities of voltage-gated neuronal and cardiac Na + (Nav) …\n",
      "\n",
      "… Monoclonal antibodies K74/71, K69/3, and K87A/10 were developed by as well as obtained \n",
      "from the University of California Davis/NINDS/NIMH NeuroMab Facility, supported by NIH …\n",
      "\n",
      "… Monoclonal antibodies K74/71, K69/3, and K87A/10 were developed by as well as obtained \n",
      "from the University of California Davis/NINDS/NIMH NeuroMab Facility, supported by NIH …\n",
      "\n",
      "Oligodendrocyte (OL) maturation and axon-glial communication are required for proper \n",
      "myelination in the developing brain. However, physiological properties of OLs remain largely …\n",
      "\n",
      "Spiking immature oligodendrocytes (OLs), referred to as spiking OLs, express voltage-activated \n",
      "Na + channels (Na v ) and K + (K v ) channels, endowing a subpopulation of OLs with …\n",
      "\n",
      "Autosomal dominant lateral temporal epilepsy (ADLTE) is an inherited syndrome caused by \n",
      "mutations in the leucine-rich glioma inactivated 1 (LGI1) gene. It is known that glutamatergic …\n",
      "\n",
      "Fast-spiking (FS) neurons can fire action potentials (APs) up to 1,000 Hz and play key roles \n",
      "in vital functions such as sound location, motor coordination, and cognition. Here we report …\n",
      "\n",
      "J. Neurochem. (2010) 10.1111/j.1471‐4159.2010.06582.x Abstract EphA4, a receptor tyrosine \n",
      "kinase, is expressed in various pre‐, post‐ and peri‐synaptic organelles and implicated in …\n",
      "\n",
      "… Membranes were blotted with a monoclonal mouse antibody specific to a conserved epitope \n",
      "of Slo1 (1:1000 dilution; NeuroMab L6/60 clone) or with a polyclonal rabbit antihuman PKG-…\n",
      "\n",
      "… For immunoblots, BKα and LRRC26 proteins were probed by a mouse monoclonal anti-BKα \n",
      "antibody (L6/60, NeuroMab), recognizing the C-terminal domain of BKα around the S9 …\n",
      "\n",
      "… The anti-BK monoclonal IgG2a antibody L6/60 used in this study was raised against a … \n",
      "We used a monospecific monoclonal antibody (L6/60, NeuroMab) against the α-subunit of the …\n",
      "\n",
      "Endothelial cells (EC) control vascular smooth muscle cell (VSMC) tone by release of \n",
      "paracrine factors. VSMC may also influence the EC layer, and therefore, the present study …\n",
      "\n",
      "A limited number of studies have shown functional changes in mitochondrial ion channels \n",
      "in aging and senescent cells. We have identified, for the first time, mitochondrial large-…\n",
      "\n",
      "… Protein isolated from the membrane fraction of distal colonic epithelial cells was immunoblotted \n",
      "with an antibody against the K Ca 1.1 protein (Kcnma1) NeuroMab clone L6/60. An …\n",
      "\n",
      "… (L6/60; NeuroMab) and a rabbit monoclonal anti-GluN1 antibody (AB9864R; EMD Millipore). \n",
      "We observed that these two antibodies produced strong point-like PLA signals only when …\n",
      "\n",
      "Most physiological systems show daily variations in functional output, entrained to the day–night \n",
      "cycle. Humans exhibit a daily rhythm in urinary voiding (micturition), and disruption of …\n",
      "\n",
      "… The following primary antibodies were used: mouse anti-Slo1/BKAlpha potassium \n",
      "channel (clone L6/60; 1:400; UC Davis/NIH NeuroMab Facility), rabbit anti-calcitonin gene-related …\n",
      "\n",
      "… NeuroMab clone L6/60; 1:4000 dilution), rabbit anti-KCNN2 (Kca2.2, SK2) antibody (Alomone \n",
      "lab, APC-028; 1:4000 dilution), rabbit anti-Na V 1.6 (SCN8A) antibody (Alomone lab, ASC-…\n",
      "\n",
      "… antibody (clone L6/60, NeuroMab, Antibodies Inc.). The polyclonal anti-rabbit LRRC26 \n",
      "antibody was from Santa Cruz Biotechnology (sc-102015 V19) and used at a dilution of 1:200. …\n",
      "\n",
      "… Slo1 (clone L6/60) was obtained from the University of California Davis/National Institute \n",
      "of Neurological Disorders and Stroke/National Institute of Mental Health NeuroMab Facility. …\n",
      "\n",
      "… L6/60, 1∶500, NeuroMab, Davis, CA, USA) or an anti-BK Ca β subunit antibody (ab3587, \n",
      "1∶500, Abcam, Cambridge, MA, USA). Bound antibody was detected using SuperSignal …\n",
      "\n",
      "… These included: the α subunit of the BK Ca channel (AKA slo1, NeuroMab clone L6/60, \n",
      "NeuroMab, Davis CA: 1:200), BK Ca β2 (NeuroMab clone N53/32: 1:200), BK Ca β3 (NeuroMab …\n",
      "\n",
      "An important event during apoptosis is regulated cell condensation known as apoptotic \n",
      "volume decrease (AVD). Ion channels have emerged as essential regulators of this process …\n",
      "\n",
      "Key points At the end of pregnancy, the uterus transitions from a quiescent state to a highly \n",
      "contractile state. This transition requires that the uterine (myometrial) smooth muscle cells …\n",
      "\n",
      "… These acini were probed with antisera raised against BK (α-MSlo clone L6/60 monoclonal \n",
      "antibody; UC Davis, NeuroMab facility), α-K Ca 1.1 (Alomone Labs), α-type–3 InsP 3 receptor (…\n",
      "\n",
      "Myoblasts are mononucleated precursors of myofibers; they persist in mature skeletal \n",
      "muscles for growth and regeneration post injury. During myotonic dystrophy type 1 (DM1), a …\n",
      "\n",
      "… Anti-MaxiKα monoclonal and polyclonal antibodies were from NeuroMab (clone L6/60, \n",
      "catalogue no. 75-022; UC Davis/NIH NeuroMab Facility) and Alomone Labs (catalogue no. APC-…\n",
      "\n",
      "… After protein transfer, the membranes were exposed to an anti-BK α antibody (NeuroMab, \n",
      "clone L6/60, diluted 1:200) and an anti-mHSP75 antibody (Abcam, 1:1000). The blots were …\n",
      "\n",
      "… The membranes were incubated with primary antibodies including anti-BK α (Neuromab, \n",
      "L6/60, 1:1000), anti-BK β2 (Neuromab, N53/32 1:1000), anti-BK β4 (Neuromab, L18A.3, 1:1000…\n",
      "\n",
      "… , and a mouse monoclonal anti-Mslo (KCa1.1, clone L6/60) produced against fusion \n",
      "protein amino acids 690–1196 (NeuroMab, UC Davis). Secondary antibodies were goat anti-mouse …\n",
      "\n",
      "10.7554/eLife.01009.001 Mammalian spermatozoa gain competence to fertilize an oocyte \n",
      "as they travel through the female reproductive tract. This process is accompanied by an …\n",
      "\n",
      "… fiber pathway and whether BK channels are expressed at mossy fiber axon and terminals, \n",
      "double immunofluorescence assays were developed using anti-BK 690–1196 (2 μg/ml; L6/60…\n",
      "\n",
      "… Densitometry of BK doublet band (1 μg ml −1 L6/60 mouse monoclonal α-BK antibody, \n",
      "Neuromab; University of California, Davis, CA, USA) to DM1α anti-tubulin (1:75,000; Sigma T-…\n",
      "\n",
      "… These mAbs are available from the UC Davis/ NIH NeuroMab Facility (neuromab.ucdavis.… \n",
      "BK potassium channel (determined by staining with the mouse mAb L6/60). As shown in Fig. 1, …\n",
      "\n",
      "… against Slo1 (clone L6/60) obtained from the University of California Davis/National Institute \n",
      "of Neurological Disorders and Stroke/National Institute of Mental Health NeuroMab Facility, …\n",
      "\n",
      "… C5, www.abcam.com, Cambridge, MA), Anti-BKα (clone L6/60) monoclonal antibodies \n",
      "were acquired from NeuroMab (http://neuromab.ucdavis.edu/). Anti-HA was used for surface/total …\n",
      "\n",
      "The chemosensory glomus cells of the carotid body (CB) detect changes in O 2 tension. \n",
      "Carotid sinus nerve fibers, which originate from peripheral sensory neurons located within the …\n",
      "\n",
      "… Specimens were incubated at 4C overnight in mouse monoclonal antibody against BK \n",
      "alpha subunit (clone L6/60, UC Davis/NIH NeuroMab Facility, Davis, CA) at 1:200 dilution. After …\n",
      "\n",
      "… The blots were probed with a mouse anti-BK Ca antibody (clone L6/60, 1:500; NeuroMab, \n",
      "Davis, CA) and a mouse monoclonal anti-α smooth muscle actin primary antibody (Sigma-…\n",
      "\n",
      "… A monoclonal anti-BK Ca channel antibody was obtained from the UC Davis/NIH NeuroMab \n",
      "… 690–1196 of mouse Slo1 (clone L6/60) and recognized a single band of approximately 125 …\n",
      "\n",
      "… L6/60, UC Davis/NIH NeuroMab Facility, Davis, CA) and rabbit polyclonal antibody against \n",
      "BK beta 1 subunit (PA 1–924, Invitrogen, Carlsbad, CA). After primary antibody washout, …\n",
      "\n",
      "… A monoclonal anti-BK channel antibody was obtained from the UC Davis/NIH NeuroMab … \n",
      "acids 690–1196 of mouse Slo1 (clone L6/60) and recognized a single band of approximately …\n",
      "\n",
      "… The monoclonal antibody against Slo1 (clone L6/60) was obtained from the UC Davis/NINDS/NIMH \n",
      "NeuroMab Facility, supported by National Institutes of Health (NIH) Grant U24-…\n",
      "\n",
      "… (NeuroMabs, USA, clone L6/60, diluted 1:200), an anti-β-actin antibody (Abcam, UK, 1:1000, \n",
      "no. 8227), anti-SDHA (1:1000, Abcam, no. ab137040), anti-COXIV (1:1000, Cell Signaling, …\n",
      "\n",
      "… Davis/NIH NeuroMab clone L6/60; 1:4,000 dilution), rabbit anti-KCNN2 (KCa2.2, SK2) \n",
      "antibody (Alomone lab, APC-028; 1:4,000 dilution), and rabbit anti-glyceraldehyde-3-phosphate …\n",
      "\n",
      "Large conductance voltage and calcium-activated potassium (MaxiK) channels are activated \n",
      "by membrane depolarization and elevated cytosolic Ca 2+ . In the brain, they localize to …\n",
      "\n",
      "… Ca channel primary antibody (1:200 in 5% milk in TBST; NeuroMab Anti-Slo1/BK alpha \n",
      "potassium channel, clone L6/60), followed by incubation with anti-mouse secondary antibody (1:…\n",
      "\n",
      "Simultaneous labeling of multiple targets in a single sample, or multiplexing, is a powerful \n",
      "approach to directly compare the amount, localization and/or molecular properties of different …\n",
      "\n",
      "… The specific primary antibodies used were anti-BKCa (monoclonal, Neuromab, clone L6/60), \n",
      "anti-PKG-1 (polyclonal, Enzo LifeSciences), anti-PSer92-PDE5 (polyclonal, FabGennix), …\n",
      "\n",
      "… (clone L6/60) was obtained from the University of California Davis/National Institute of \n",
      "Neurological Disorders and Stroke (NINDS)/National Institute of Mental Health NeuroMab Facility, …\n",
      "\n",
      "Mitochondrial potassium channels have been implicated in cytoprotective mechanisms. \n",
      "Activation of the mitochondrial large-conductance Ca 2+ -regulated potassium (mitoBK Ca ) …\n",
      "\n",
      "… and BK channel (1:500, Clone L6/60, mouse monoclonal antibodies were obtained from the \n",
      "UC Davis/NIH NeuroMab facility) immunostaining was performed. Measurement of staining …\n",
      "\n",
      "As high-conductance calcium- and voltage-dependent potassium channels, BK channels \n",
      "consist of pore-forming, voltage-, and Ca 2+ -sensing α and auxiliary subunits. The leucine-rich …\n",
      "\n",
      "… 238 The blots were probed with a mouse anti-BKCa antibody (clone L6/60, 1:500, \n",
      "NeuroMab, 239 Davis, CA, USA), and a mouse monoclonal anti-α smooth muscle actin primary …\n",
      "\n",
      "… After centrifugation at 17,000 g for 10 min, the solubilized BKα–γ1 complex in the supernatant \n",
      "was incubated with immobilized mouse monoclonal anti-BKα antibody (L6/60; NeuroMab) …\n",
      "\n",
      "… immunoprecipitated with NeuroMab anti-Slo1 Ab or the NeuroMab N53/32 BKβ2 Ab (nAB) \n",
      "and then visualized either with NeuroMab N33 Slo1 Ab (top) or with NeuroMab BKβ2 N53/32. …\n",
      "\n",
      "… Lung lysates were prepared in RIPA buffer and submitted to immunoblot analysis with \n",
      "specific anti-BKCa (monoclonal, Neuromab, Davis, CA, USA, clone L6/60), anti-PKG-1 (polyclonal…\n",
      "\n",
      "The present study tested the hypotheses that nephrotic syndrome (NS) leads to renal K + \n",
      "loss because of augmented epithelial Na + channel (ENaC) activity followed by …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… Lung lysates were prepared in RIPA buffer and submitted to immunoblot analysis with \n",
      "specific anti-BKCa (monoclonal, Neuromab, clone L6/60), anti-PKG-1 (polyclonal, Enzo …\n",
      "\n",
      "… The mouse antibodies were as follows: anti-BKα(L6/60) (University of California, Davis/NINDS/NIMH \n",
      "NeuroMab Facility, Davis, CA); anti-Na + /K + -ATPase α1 (sc-21712) and anti-β-…\n",
      "\n",
      "Neonatal pulmonary hypertension (NPHT) is produced by sustained pulmonary vasoconstriction \n",
      "and increased vascular remodeling. Soluble guanylyl cyclase (sGC) participates in …\n",
      "\n",
      "Neonatal pulmonary hypertension (NPHT) is produced by sustained pulmonary vasoconstriction \n",
      "and increased vascular remodeling. Soluble guanylyl cyclase (sGC) participates in …\n",
      "\n",
      "(Pro)renin receptor (PRR) is highly expressed in the distal nephron, but it has an unclear \n",
      "functional implication. The present study was conducted to explore a potential role of renal …\n",
      "\n",
      "Action potentials clustered into high-frequency bursts play distinct roles in neural computations. \n",
      "However, little is known about ionic currents that control the duration and probability of …\n",
      "\n",
      "DYT1/DYT-TOR1A dystonia is an early-onset, generalized dystonia. Most DYT1 dystonia \n",
      "patients have a heterozygous trinucleotide GAG deletion in the DYT1 gene, resulting in a loss …\n",
      "\n",
      "The modulation of AMPA receptor (AMPAR) content at synapses is thought to be an underlying \n",
      "molecular mechanism of memory and learning. AMPAR content at synapses is highly …\n",
      "\n",
      "During development of the central nervous system, precise synaptic connections between \n",
      "presynaptic and postsynaptic neurons are formed. While significant progress has been made …\n",
      "\n",
      "… Blots were blocked with bovine serum albumin and probed with an ASIC1-specific antibody \n",
      "(NeuroMab clone N271/44) overnight at a 1:1,000 dilution. Blots were washed with Tris-…\n",
      "\n",
      "… (N271/44; 1:300; NeuroMab) and a monoclonal antibody against actin (AC-40; 1:1,000; \n",
      "Sigma) as a loading control. siRNAs were intrathecally injected into mice (2 µg per mouse at a …\n",
      "\n",
      "… using an anti-ASIC1 antibody (NeuroMab clone N271/44 from UC Davis, Merck) and … \n",
      "dilution in 5% milk in TBST; NeuroMab clone N271/44 from UC Davis, Merck) in concert …\n",
      "\n",
      "Key points Chronic limb ischaemia, characterized by inflammatory mediator release and a \n",
      "low extracellular pH, leads to acid‐sensing ion channel (ASIC) activation and reflexively …\n",
      "\n",
      "The exercise pressor reflex is initiated by the contraction-induced activation of group III and \n",
      "IV muscle afferents. The reflex is manifested by increases in arterial blood pressure and …\n",
      "\n",
      "Lysosome membranes contain diverse phosphoinositide (PtdIns) lipids that coordinate \n",
      "lysosome function and dynamics. The PtdIns repertoire on lysosomes is tightly regulated by the …\n",
      "\n",
      "Lysosome membranes contain diverse phosphoinositide (PtdIns) lipids that co-ordinate \n",
      "lysosome function and dynamics. The PtdIns repertoire on lysosomes is tightly regulated by the …\n",
      "\n",
      "… Kvβ1.2 was precipitated with a mouse anti-Kvβ1.2 primary antibody (clone K47/42; Neuromab). \n",
      "GAPDH was precipitated with a rabbit anti-GAPDH primary antibody (ab9485; Abcam) …\n",
      "\n",
      "… L102/45 … L102/45 and NG5.73, we assayed whether the two antibodies compete with each \n",
      "other for the same binding site. Rat brain membrane homogenates were probed with L102/45…\n",
      "\n",
      "Glutamate plays a role in hair cell afferent transmission, but the receptors that mediate \n",
      "neurotransmission between outer hair cells (OHCs) and type II ganglion neurons are not well …\n",
      "\n",
      "Dopamine‐ and tyrosine hydroxylase–immunopositive cells (TH cells) modulate visually \n",
      "driven signals as they flow through retinal photoreceptor, bipolar, and ganglion cells. Previous …\n",
      "\n",
      "The structural microtubule-associated proteins (MAPs) are critical for the organization of \n",
      "neuronal microtubules (MTs). Microtubule-associated protein 1A (MAP1A) is one of the most …\n",
      "\n",
      "… (NeuroMab, Davis, CA, USA, 75-210/ clone N112B/14) or anti-Kir2.2 (NeuroMab 75-203/ \n",
      "clone N124B/38). Both antibodies were diluted at 1:500. The secondary antibody used was an …\n",
      "\n",
      "… In contrast, immunolabeling with the anti-Ankyrin-R/G NeuroMab N388A/27 TCS (green) \n",
      "does not differ between WT and KO. Dye staining with Hoechst 33342 (blue) labels cellular …\n",
      "\n",
      "Growth of a properly complex dendrite arbor is a key step in neuronal differentiation and a \n",
      "prerequisite for neural circuit formation. Diverse cell surface molecules, such as the clustered …\n",
      "\n",
      "… :500–1:1000) were generated by NeuroMab in collaboration with the Weiner laboratory [55] … \n",
      "); N148/30 (specific for γB2); N174B/27 (specific for γC3). A rabbit polyclonal antibody raised …\n",
      "\n",
      "Developmental and epileptic encephalopathies (DEE) are a group of severe epilepsies that \n",
      "usually present with intractable seizures, developmental delay, and often have elevated risk …\n",
      "\n",
      "Complex spike bursts (CSBs) represent a characteristic firing pattern of hippocampal pyramidal \n",
      "cells (PCs). In CA1PCs, CSBs are driven by regenerative dendritic plateau potentials, …\n",
      "\n",
      "Among all voltage-gated potassium (Kv) channels, Kv2 channels are the most widely \n",
      "expressed in the mammalian brain. However, studying Kv2 in neurons has been challenging …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… immunolabeled with the progenitor N93A/49 mouse mAb (green) and the N93A/49 scFv (red)…\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… immunolabeled with the progenitor N93A/49 mouse mAb (green) and the N93A/49 scFv (red)…\n",
      "\n",
      "Addictive drugs usurp the brain9s intrinsic mechanism for reward, leading to compulsive and \n",
      "destructive behaviors. In the ventral tegmental area (VTA), the center of the brain9s reward …\n",
      "\n",
      "Dendritic spines can be directly connected to both inhibitory and excitatory presynaptic \n",
      "terminals, resulting in nanometer-scale proximity of opposing synaptic functions. While dually …\n",
      "\n",
      "The GABA-B receptor is densely expressed throughout the brain and has been implicated in \n",
      "many CNS functions and disorders, including addiction, epilepsy, spasticity, schizophrenia, …\n",
      "\n",
      "The functions of different G-protein αβγ subunit combinations are traditionally ascribed to \n",
      "their various α components. However, the discovery of similarly diverse γ subtypes raises the …\n",
      "\n",
      "Heterotrimeric guanine nucleotide-binding proteins (G proteins) transduce signals from G \n",
      "protein-coupled receptors (GPCRs) to effector ion channels and enzymes G αo , a member of …\n",
      "\n",
      "GABA B receptors are G-protein-coupled receptors that mediate inhibitory synaptic actions \n",
      "through a series of downstream target proteins. It is increasingly appreciated that the GABA B …\n",
      "\n",
      "Psychostimulants induce neuroadaptations in excitatory and fast inhibitory transmission in \n",
      "the ventral tegmental area (VTA). Mechanisms underlying drug-evoked synaptic plasticity of …\n",
      "\n",
      "The lateral habenula (LHb) encodes aversive signals, and its aberrant activity contributes to \n",
      "depression-like symptoms. However, a limited understanding of the cellular mechanisms …\n",
      "\n",
      "GABA B receptors are heterodimeric G protein-coupled receptors, which control neuronal \n",
      "excitability by mediating prolonged inhibition. The magnitude of GABA B receptor-mediated …\n",
      "\n",
      "… 135 302), one mouse anti-VGLUT1 (UC Davis/NIH NeuroMab Facility; clone N28/9), and \n",
      "a VGAT antibody raised in guinea pig (Synaptic Systems; cat. no. 131 004) were used. …\n",
      "\n",
      "Autism Spectrum Disorders (ASDs) are neurodevelopmental disorders characterised by \n",
      "deficits in social interactions and repetitive behaviours. ASDs have a strong genetic basis with …\n",
      "\n",
      "… ; N28/9 (Neuromab)) primary antibodies in PBS with 5% BSA, washed, and stained with \n",
      "polyclonal anti-complexin and monocolonal anti-vGlut1 and visualized using Alexa Fluor 488 …\n",
      "\n",
      "… ; N28/9 (Neuromab)) primary antibodies in PBS with 5% BSA, washed, and stained with \n",
      "polyclonal anti-complexin and monocolonal anti-vGlut1 and visualized using Alexa Fluor 488 …\n",
      "\n",
      "Increased relative brain size characterizes the evolution of primates, suggesting that enhanced \n",
      "cognition plays an important part in the behavioral adaptations of this mammalian order. …\n",
      "\n",
      "Elevated levels of the inflammatory cytokine interleukin-6 (IL-6) occur in a number of CNS \n",
      "disorders. However, little is known about how this condition affects CNS neuronal function. …\n",
      "\n",
      "… The monoclonal antibody against VGluT1 (clone N28/9) was developed by and/or obtained \n",
      "from the UC Davis/NIH NeuroMab Facility, supported by NIH grant U24NS050606 and …\n",
      "\n",
      "Use of microfluidic devices to compartmentalize cultured neurons has become a standard \n",
      "method in neuroscience. This protocol shows how to use a pre-assembled multi-compartment …\n",
      "\n",
      "… The hibridoma clone N28/9 used in this study recognizes a single band of approximately \n",
      "52 kDa on Western blots of rat adult brain extracts (manufacturer's data sheet). The labeling …\n",
      "\n",
      "Copper (Cu), an essential trace element present throughout the mammalian nervous system, \n",
      "is crucial for normal synaptic function. Neuronal handling of Cu is poorly understood. We …\n",
      "\n",
      "… C-terminus of rat VGluT1 (amino acids 493–560; NeuroMab, UC Davis, CA; catalog No. \n",
      "75-066; clone N28/9). Immunoblotting recognizes a single band of approximately 52 kDa with …\n",
      "\n",
      "Microfabricated methods to compartmentalize neurons have become essential tools for \n",
      "many neuroscientists. This protocol describes the use of a commercially available pre-…\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection \n",
      "of … anti-VGlut1 hybridoma-generated N28/9 mAb (IgG1) (left section) and the N28/9 scFv (right …\n",
      "\n",
      "… ; vesicular glutamate transporter 1 (VGlut1; clone N28/9, code 73-066, University of California, \n",
      "Davis/National Institutes of Health NeuroMab Facility), a presynaptic marker of excitatory …\n",
      "\n",
      "… containing 0.1 M PBS, 5% normal goat serum, 5% BSA, and 0.25% Triton X-100, then overnight \n",
      "at 4 C with anti-Bassoon (1:1000, Stressgen) and antiVGLUT1 (1:300; NeuroMab N28/9) …\n",
      "\n",
      "Neurotrophin receptor tyrosine kinases (Trks) have well-defined trophic roles in nervous \n",
      "system development through kinase activation by neurotrophins. Yet Trks have typical cell-…\n",
      "\n",
      "Axon damage may cause axon regeneration, retrograde synapse loss, and hyper-excitability, \n",
      "all of which affect recovery following acquired brain injury. While axon regeneration is …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection \n",
      "of … anti-VGlut1 hybridoma-generated N28/9 mAb (IgG1) (left section) and the N28/9 scFv (right …\n",
      "\n",
      "Autism Spectrum Disorders (ASD) and schizophrenia are distinct neurodevelopmental \n",
      "disorders that share certain symptoms and genetic components. Both disorders show …\n",
      "\n",
      "SLITRK1 is an obsessive-compulsive disorder spectrum-disorders-associated gene that \n",
      "encodes a neuronal transmembrane protein. Here we show that SLITRK1 suppresses …\n",
      "\n",
      "Exaggerated synaptic elimination in the prefrontal cortex (PFC) during adolescence has been \n",
      "suggested to contribute to the neuropathological changes of schizophrenia. Recent data …\n",
      "\n",
      "… Anti-vesicular glutamate transporter 1 (VGLUT1; NeuroMAB, University of California, \n",
      "Davis, Davis, CA; clone N28/9), a mouse monoclonal antibody, raised against fusion protein …\n",
      "\n",
      "In presynaptic nerve terminals, complexin regulates spontaneous “mini” neurotransmitter \n",
      "release and activates Ca 2+ -triggered synchronized neurotransmitter release. We studied the …\n",
      "\n",
      "In the mammalian retina, processes of approximately 70 types of interneurons form specific \n",
      "synapses on roughly 30 types of retinal ganglion cells (RGCs) in a neuropil called the inner …\n",
      "\n",
      "… ; 1:150) (37), mouse anti-vGlut1 (NeuroMab, clone N28/9; 1:1000), guinea pig anti-vGlut1 \n",
      "(… The monoclonal antibody anti-vGlut1 N28/9 was obtained by the UC Davis/NIH NeuroMab …\n",
      "\n",
      "… presynaptic terminal marker VGluT1, labeled with mAb N28/9 (IgG1, blue), which exhibits … \n",
      "We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and …\n",
      "\n",
      "Select adhesion proteins control the development of synapses and modulate their structural \n",
      "and functional properties. Despite these important roles, the extent to which different …\n",
      "\n",
      "… (clone, N28/9), mouse anti-pan-MAGUK (clone, N28/86), mouse anti-HCN4 (clone, N114.10), \n",
      "and mouse anti-Vesicular acetylcholine transporter (clone, N6/38) from NeuroMab (Davis, …\n",
      "\n",
      "Synapses are found in vast numbers in the brain and contain complex proteomes. We \n",
      "developed genetic labeling and imaging methods to examine synaptic proteins in individual …\n",
      "\n",
      "The transgenic P23H line 1 (P23H-1) rat expresses a variant of rhodopsin with a mutation \n",
      "that leads to loss of visual function. This rat strain is an experimental model usually employed …\n",
      "\n",
      "… N28/9; neuroMAB). Primary antibodies were diluted with PBS containing 30 ml/L of normal \n",
      "donkey serum, 30 ml/L of normal rat serum and 0.02 g/L NaN 3 while the relevant species-…\n",
      "\n",
      "The type of neurotransmitter secreted by a neuron is a product of the vesicular transporters \n",
      "present on its synaptic vesicle membranes and the available transmitters in the local cytosolic …\n",
      "\n",
      "The locus coeruleus (LC) is the major source for norepinephrine (NE) in the brain and \n",
      "projects to areas involved in learning and memory, reward, arousal, attention, and autonomic …\n",
      "\n",
      "Alzheimer's disease (AD) is a neurodegenerative disorder with cognitive deficits. Amyloidogenic \n",
      "processing of amyloid precursor protein (APP) produces amyloid β (Aβ), the major …\n",
      "\n",
      "… were all based on mAbs from NeuroMab (UC Davis/NIH NeuroMab Facility, nd). This facility \n",
      "… 2013), our current efforts to systematically convert NeuroMab mAbs into scFvs that retain the …\n",
      "\n",
      "Background Phosphatidylinositol (3,4,5)-trisphosphate-dependent Rac Exchanger 2 (P-Rex2) \n",
      "is a guanine nucleotide exchange factor (GEF) that specifically activates Rac GTPases, …\n",
      "\n",
      "The proper formation and maintenance of functional synapses in the central nervous system \n",
      "(CNS) requires communication between neurons and astrocytes and the ability of …\n",
      "\n",
      "LRRTMs are postsynaptic cell adhesion proteins that have region-restricted expression in the \n",
      "brain. To determine their role in the molecular organization of synapses in vivo, we studied …\n",
      "\n",
      "Synapses are fundamental units of communication in the brain. The prototypical synapse-organizing \n",
      "complex neurexin-neuroligin mediates synapse development and function and is …\n",
      "\n",
      "During development, pyramidal neurons undergo dynamic regulation of AMPA receptor (AMPAR) \n",
      "subunit composition and density to help drive synaptic plasticity and maturation. …\n",
      "\n",
      "Brain-derived neurotrophic factor (BDNF) signals through its high affinity receptor Tropomyosin \n",
      "receptor kinase-B (TrkB) to regulate neuronal development, synapse formation and …\n",
      "\n",
      "In addition to its role in Alzheimer's disease, amyloid precursor protein (APP) has physiological \n",
      "roles in synapse development and function. APP induces presynaptic differentiation …\n",
      "\n",
      "Glutamate and its receptor N-methyl-D-aspartate receptor (NMDAR) have been associated \n",
      "with cancer, although their functions are not fully understood. Herein, we implicate glutamate-…\n",
      "\n",
      "Post‐synaptic dendritic spines are structurally composed of actin cytoskeleton, which undergoes \n",
      "dynamic morphological changes to accommodate incoming synaptic activity. Drebrin is …\n",
      "\n",
      "The accumulation of insoluble amyloid-beta (Aβ) peptides is associated with neurodegenerative \n",
      "disorders, such as Alzheimer's disease (AD). As essential tremor (ET) could involve …\n",
      "\n",
      "Injury of CNS nerve tracts remodels circuitry through dendritic spine loss and hyper-excitability, \n",
      "thus influencing recovery. Due to the complexity of the CNS, a mechanistic …\n",
      "\n",
      "Multiple trans-synaptic complexes organize synapse development, yet their roles in the \n",
      "mature brain and cooperation remain unclear. We analyzed the postsynaptic adhesion protein …\n",
      "\n",
      "KCNQ5/Kv7.5 is a low-threshold non-inactivating voltage-gated potassium channel \n",
      "preferentially targeted to excitatory endings in brain neurons. The M-type current is mediated by …\n",
      "\n",
      "Parvalbumin (PV)-expressing basket interneurons in the prefrontal cortex (PFC) regulate \n",
      "pyramidal cell firing, synchrony, and network oscillations. Yet, it is unclear how their perisomatic …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "Long-term, activity-dependent changes in synaptic efficacy at excitatory and inhibitory \n",
      "neurons contribute to hippocampal-dependent forms of learning and memory. While much is …\n",
      "\n",
      "… To test the hypothesis that N241A/34 could isolate the … ), we conjugated Neuromab N241A/34 \n",
      "(Antibodies Inc, Davis CA) … Ten μl of N241A/34 bead solution (ie, 10 μg of N241A/34) was …\n",
      "\n",
      "… but results demonstrate that the mouse monoclonal N241A/34 is suitable for most applications, \n",
      "… Figures S2G–S2J), Neuromab N138/6 and N241A/34 both produced a single band in the …\n",
      "\n",
      "… The LRRK2 N241A/34 mouse monoclonal antibody (73-253, NeuroMab, … The N241A/34 \n",
      "mouse monoclonal antibody was not … in neurons observed in rats using the N241A/34 antibody. …\n",
      "\n",
      "… mouse monoclonal anti-LRRK2 (N241A/34; UC Davies/NIH NeuroMab, clone N241A/34, \n",
      "75–253), … mouse monoclonal anti-LRRK2 N241A/34 (1:200, UC Davies/NIH NeuroMab, clone …\n",
      "\n",
      "… Membranes were probed overnight at 4 C for LRRK2 (N241A/34, Neuromab) and \n",
      "phosphorylated LRRK2 P-Ser935 (UDD2, Abcam), P-Ser955 (MJF-R11, Abcam), P-Ser973 (MJF-R12, …\n",
      "\n",
      "… We found that 2 antibodies, the UDD2 P-Ser935 LRRK2 antibody from Abcam (Cambridge, \n",
      "UK) and the N241A/34 LRRK2 antibody from Neuromab, stained neurons in the cerebellum …\n",
      "\n",
      "… Where indicated, membranes were probed for total LRRK2 (N241A/34, Neuromab), \n",
      "phosphorylated LRRK2 pSer935 (UDD2, Abcam), total Rab10 (D36C4, Cell Signaling Technology), …\n",
      "\n",
      "… (EG) GS LRRK2-expressing BV2 cells were analyzed by immunofluorescence using anti-LRRK2 \n",
      "[N241A/34] (Alexa-488, green) and anti-Tom20 (Texas Red, red). The morphology of …\n",
      "\n",
      "Background Leucine‐rich repeat kinase 2 is a potential therapeutic target for the treatment \n",
      "of Parkinson's disease, and clinical trials of leucine‐rich repeat kinase 2 inhibitors are in …\n",
      "\n",
      "… Total and pS935 LRRK2 assays developed on the Quanterix Simoa platform [16] used \n",
      "Neuromab N241A/34 for capture and biotinylated detection antibodies (Abcam clone UDD2 for …\n",
      "\n",
      "… Brain extracts (20 mg protein) were combined with 50 µl Protein G-Dynabeads (Invitrogen) \n",
      "pre-incubated with mouse monoclonal anti-LRRK2 (5 µg; N241A/34; NeuroMab) antibody …\n",
      "\n",
      "… Immunocytochemistry was performed on 3–5-day-old flies expressing G2019S as described \n",
      "recently [14], using the LRRK2 mouse antibody (Neuromab, clone N241A/34). In some flies, …\n",
      "\n",
      "In response to brain injury, microglia rapidly extend processes that isolate lesion sites and \n",
      "protect the brain from further injury. Here we report that microglia carrying a pathogenic …\n",
      "\n",
      "… The increased levels of LRRK2 in R26-LRRK2 +/+ /DAT-Cre mice could also be confirmed \n",
      "using an alternative LRRK2 antibody (N241A/34) (Fig. 1e). The detection of human LRRK2 …\n",
      "\n",
      "Missense mutations in Leucine-Rich Repeat Kinase 2 (LRRK2) cause the majority of familial \n",
      "and some sporadic forms of Parkinson’s disease (PD). The hyperactivity of LRRK2 kinase …\n",
      "\n",
      "… Similar results were obtained using the anti-LRRK2 UDD3 and N241A/34 antibodies, all \n",
      "of which recognize both mouse and human LRRK2 in similar extraction conditions (Figs. 1B …\n",
      "\n",
      "… Primary antibodies used were anti-LRRK2 (N241A/34, UC Davis/NIH NeuroMab Facility), \n",
      "a previously-described custom antibody against pSer1292 LRRK2 (49) and anti-beta actin (AC…\n",
      "\n",
      "Activating mutations in the leucine-rich repeat kinase 2 (LRRK2) cause Parkinson’s disease, \n",
      "and previously we showed that activated LRRK2 phosphorylates a subset of Rab GTPases (…\n",
      "\n",
      "… Where indicated, membranes were probed for total LRRK2 (N241A/34, Neuromab), \n",
      "phosphorylated LRRK2 pSer935 (UDD2, Abcam), total Rab10 (D36C4, Cell Signaling Technology), …\n",
      "\n",
      "Mutations in the leucine-rich repeat kinase-2 (LRRK2) gene cause late-onset Parkinson’s \n",
      "disease, but its physiological function has remained largely unknown. Here we report that …\n",
      "\n",
      "The Leucine Rich Repeat Kinase 2 (LRRK2) gene is a major genetic determinant of Parkinson’s \n",
      "disease (PD), encoding a homonymous multi-domain protein with two catalytic activities…\n",
      "\n",
      "Objective The aim of this study was to search for genes/variants that modify the effect of LRRK2 \n",
      "mutations in terms of penetrance and age‐at‐onset of Parkinson's disease. Methods We …\n",
      "\n",
      "Mutations in leucine-rich repeated kinase 2 (LRRK2) cause autosomal dominant late-onset \n",
      "Parkinson's disease (PD), and the G2019S mutation in the kinase domain of LRRK2 is the …\n",
      "\n",
      "Mutations in leucine-rich repeat kinase 2 (LRRK2) are associated with increased risk for \n",
      "developing Parkinson’s disease (PD). Previously, we found that LRRK2 G2019S mutation …\n",
      "\n",
      "Leucine-rich repeat kinase 2 (LRRK2) is a promising therapeutic target for the treatment of \n",
      "Parkinson's disease (PD), and orally bioavailable, brain penetrant and highly potent LRRK2 …\n",
      "\n",
      "… D) Western blot analysis of peripheral tissues (lung, kidney and spleen) or PBMCs indicating \n",
      "levels of phosphorylated (pSer935) and total (c41-2 or N241A/34) endogenous LRRK2 in …\n",
      "\n",
      "… L15XA-3, MSD, Gaithersburg, MD, USA) were first coated with LRRK2 antibody (LRRK2-N241A/34, \n",
      "NeuroMab) at a concentration of 0.5 μg/mL in 30 μL of Tris-base saline (TBS). The …\n",
      "\n",
      "… Antibodies were purchased from as follows: LRRK2 N241A/34, Antibodies Inc./UC Davis \n",
      "NeuroMab; Rab5, Cell Signaling (2143S); TfR, Invitrogen (136800); p62, Abcam (56416); Actin, …\n",
      "\n",
      "Parkinson's disease (PD) is a fatal neurodegenerative disorder that is primarily caused by \n",
      "the degeneration and loss of dopaminergic neurons of the substantia nigra in the ventral …\n",
      "\n",
      "… (E) Western blot analysis of Triton-soluble kidney, spleen, or PBMC extracts indicating levels \n",
      "of endogenous pSer935 and total (c41-2 or N241A/34) LRRK2 in rats fed with vehicle or PF-…\n",
      "\n",
      "Loss of function mutations in the PTEN-induced kinase 1 (PINK1) kinase are causal for \n",
      "autosomal recessive Parkinson's disease (PD) whilst gain of function mutations in the LRRK2 …\n",
      "\n",
      "Mutations and variations in the leucine-rich repeat kinase 2 (LRRK2) gene are strongly \n",
      "associated with an increased risk to develop Parkinson’s disease (PD). Most pathogenic LRRK2 …\n",
      "\n",
      "Leucine-rich repeat kinase 2 (LRRK2) mutations are the most common genetic cause of \n",
      "Parkinson's disease (PD). LRRK2 contains a functional kinase domain and G2019S, the most …\n",
      "\n",
      "… The panel shows a TEM image of uEVs and dual immunogold labeling with anti-LRRK2 \n",
      "antibodies (RabMab UDD3 and Neuromab N241, revealed by large and small gold particles, …\n",
      "\n",
      "Evidence suggests that crosstalk occurs between microglial leucine-rich repeat kinase 2 (LRRK2)—a \n",
      "regulator of neuroinflammation—and neuron-released α-synuclein (αSyn)—a …\n",
      "\n",
      "… Mouse monoclonal antibody against total LRRK2 (C-terminus) was from NeuroMab (clone \n",
      "N241A/34). Rabbit monoclonal antibodies detecting phospho-Ser1292 LRRK2 [MJFR-19-7-8] …\n",
      "\n",
      "Parkinson’s disease (PD) is a progressive neurodegenerative disorder for which there is no \n",
      "existing therapeutic approach to delay or stop progression. Genetic, biochemical and pre-…\n",
      "\n",
      "Parkinson disease (PD) is the most common neurodegenerative movement disorder for which \n",
      "no treatments exist capable of slowing progression of the disease. One obstacle slowing …\n",
      "\n",
      "Parkinson's disease (PD) is a currently incurable disease and the number of patients is expected \n",
      "to increase due to the extended human lifespan. α-Synuclein is a pathological hallmark …\n",
      "\n",
      "Parkinson's-disease-associated LRRK2 is a multidomain Ser/Thr kinase that phosphorylates \n",
      "a subset of Rab GTPases to control their effector functions. Rab GTPases are the prime …\n",
      "\n",
      "LRRK2 is a highly phosphorylated multidomain protein and mutations in the gene encoding \n",
      "LRRK2 are a major genetic determinant of Parkinson's disease (PD). Dephosphorylation at …\n",
      "\n",
      "… The mouse monoclonal antibody against total LRRK2 (C-terminus) was purchased from \n",
      "NeuroMab (clone N241A/34, #75-253). Rabbit monoclonal anti-Rab10 (#8127), mouse …\n",
      "\n",
      "… The mouse monoclonal antibody against total LRRK2 (C-terminus) was purchased from \n",
      "NeuroMab (clone N241A/34, #75-253). Rabbit monoclonal anti-Rab10 (#8127), mouse …\n",
      "\n",
      "Rab8a is associated with the dynamic regulation of membrane protrusions in polarized cells. \n",
      "Rab8a is one of several Rab GTPases that are substrates of leucine-rich repeat kinase 2 (…\n",
      "\n",
      "Variants in the leucine-rich repeat kinase 2 (LRRK2) gene are associated with increased \n",
      "risk for familial and sporadic Parkinson’s disease (PD). Pathogenic variants in LRRK2, …\n",
      "\n",
      "Mutations in the leucine-rich repeat kinase 2 (LRRK2) gene cause late-onset, autosomal \n",
      "dominant Parkinson’s disease (PD). LRRK2 mutations typically give rise to Lewy pathology in …\n",
      "\n",
      "… Membranes were incubated with primary antibodies overnight at 4 C: mouse monoclonal \n",
      "[N241A/34] anti-LRRK2 antibody (1:2,000; NeuroMab, California, US), rabbit monoclonal […\n",
      "\n",
      "Biomarkers suitable for early diagnosis and monitoring disease progression are the cornerstone \n",
      "of developing disease-modifying treatments for neurodegenerative diseases such as …\n",
      "\n",
      "Mutations in leucine-rich repeat kinase 2 (LRRK2) are the most-common genetic determinants \n",
      "of Parkinson's disease (PD). The G2019S mutation is detected most frequently and is …\n",
      "\n",
      "The Leucine Rich Repeat Kinase 2 (LRRK2) gene is a major genetic determinant of Parkinson’s \n",
      "disease (PD), encoding a homonymous multi-domain protein with two catalytic activities…\n",
      "\n",
      "… The C-terminal total LRRK2 mouse monoclonal antibody was from Neuromab (clone \n",
      "N241A/34, #75-253). Rabbit monoclonal antibodies for LRRK2 phospho-Ser1292 (#ab203181), …\n",
      "\n",
      "Parkinson's disease-causing leucine-rich repeat kinase 2 (LRRK2) mutations lead to \n",
      "varying degrees of Rab GTPase hyperphosphorylation. Puzzlingly, LRRK2 GTPase-inactivating …\n",
      "\n",
      "Rab8a GTPase is associated with the dynamic regulation of membrane protrusions in \n",
      "polarized cells. Rab8a is one of several Rab-family GTPases that are substrates of leucine-rich …\n",
      "\n",
      "Mutations in LRRK2 cause autosomal dominant and sporadic Parkinson’s disease, but the \n",
      "mechanisms involved in LRRK2 toxicity in PD are yet to be fully understood. We found that …\n",
      "\n",
      "Leucine-rich repeat kinase 2 (LRRK2) is involved in Parkinson’s disease (PD) pathology. A \n",
      "previous study showed that rotenone treatment induced apoptosis, mitochondrial damage, …\n",
      "\n",
      "Missense mutations in leucine-rich repeat kinase 2 (LRRK2) are the most common cause of \n",
      "familial Parkinson’s disease (PD); however, pathways regulating LRRK2 subcellular …\n",
      "\n",
      "SHANK3 is a large scaffolding protein in the postsynaptic density (PSD) that organizes \n",
      "protein networks, which are critical for synaptic structure and function. The strong genetic …\n",
      "\n",
      "… of NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs \n",
      "at … N52A/42 as a loading control, showing two NeuroMabs (anti-PSD-95 NeuroMab K28/43 …\n",
      "\n",
      "… Cultures were blocked with 9% BSA (1 h), and incubated with an antibody for PSD-95 (Neuromab \n",
      "#K28/43, 1:1000) at room temperature overnight. After washing with PBS (3x), cultures …\n",
      "\n",
      "… Mouse anti-PSD-95 monoclonal antibody was purchased from NeuroMab (clone K28/43, \n",
      "UC Davis, CA). Rabbit anti-Aβ polyclonal antibody was from Cell Signaling (Cat# 2454, …\n",
      "\n",
      "… (NeuroMab K28/43, 1:1000), and repeated washing, secondary antibody application, and \n",
      "development as above. We scanned films on a transparency scanner (Hewlett Packard) and …\n",
      "\n",
      "… 500, NeuroMab UC Davis/NIH NeuroMab Facility, cat. no. 75-028, NeuroMab clone K28/43, \n",
      "predicted band size 95-110 kDa, observed band ∼95 kDa). β-actin (1:5,000, Abcam cat. no. …\n",
      "\n",
      "Ubiquitin Proteasome System (UPS) has been shown to regulate neuronal development and \n",
      "synapse formation. Activity-dependent regulation of E3 ligase, a component of the UPS that …\n",
      "\n",
      "… , representing proteolytic fragments that presumably retain the K28/43 epitope. The GST-Kv1.… \n",
      "As representative examples, the catalog of the UC Davis/NIH NeuroMab facility (as of 12/17/…\n",
      "\n",
      "Copper (Cu), an essential trace element present throughout the mammalian nervous system, \n",
      "is crucial for normal synaptic function. Neuronal handling of Cu is poorly understood. We …\n",
      "\n",
      "The modulation of AMPA receptor (AMPAR) content at synapses is thought to be an underlying \n",
      "molecular mechanism of memory and learning. AMPAR content at synapses is highly …\n",
      "\n",
      "The role of adhesion molecules in the assembly of synapses in the nervous system is an \n",
      "important issue. To characterize the role of neurexin/neuroligin adhesion complexes in synapse …\n",
      "\n",
      "Dopamine‐ and tyrosine hydroxylase–immunopositive cells (TH cells) modulate visually \n",
      "driven signals as they flow through retinal photoreceptor, bipolar, and ganglion cells. Previous …\n",
      "\n",
      "… NeuroMab Facility), and mouse IgG 2a anti-PSD95 clone K28/43 (1:500; UC Davis/NIH \n",
      "NeuroMab … mouse brain membranes (UC Davis/NIH NeuroMab Facility). SAP102 labeling was …\n",
      "\n",
      "The transient receptor potential channel TRPM1 is required for synaptic transmission between \n",
      "photoreceptors and the ON subtype of bipolar cells (ON-BPC), mediating depolarization …\n",
      "\n",
      "Apolipoprotein E (apoE) plays a critical role in maintaining synaptic integrity by transporting \n",
      "cholesterol to neurons through the low-density lipoprotein receptor related protein-1 (LRP1). …\n",
      "\n",
      "Induced pluripotent stem cell (iPSC) techniques have had considerable breakthroughs in \n",
      "modeling human neurological diseases. Multiple protocols inducing neurons, astrocytes, …\n",
      "\n",
      "Investigating the roles of synaptogenic adhesion molecules during synapse formation has \n",
      "proven challenging, often due to compensatory functions between additional family members. …\n",
      "\n",
      "… The monoclonal mouse antibodies against Shank (N23B/49) and PSD-95 (K28/43) were \n",
      "from NeuroMAB. ECL and ECL plus reagents were from Amersham. Other reagents were from …\n",
      "\n",
      "… Mouse anti-PSD-95 was raised to amino acids 77–299 of human PSD-95/SAP-90 and \n",
      "used at 1:50 (K28/43; NeuroMab). The specificity of the monoclonal to PSD-95 has been …\n",
      "\n",
      "Mastoparan-7 (Mas-7), an analogue of the peptide mastoparan, which is derived from wasp \n",
      "venom, is a direct activator of Pertussis toxin- (PTX-) sensitive G proteins. Mas-7 produces …\n",
      "\n",
      "The cleft is an integral part of synapses, yet its macromolecular organization remains unclear. \n",
      "We show here that the cleft of excitatory synapses exhibits a distinct density profile as …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… the K28/43 hybridoma cell line 9 . We initially used this plasmid to express the K28/43 mAb, …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "The hexanucleotide repeat in the first intron of the C9orf72 gene is the most significant cause \n",
      "of amyotropic lateral sclerosis as well as some forms of fronto‐temporal dementia. The …\n",
      "\n",
      "Sensitization to inflammatory pain is a pathological form of neuronal plasticity that is poorly \n",
      "understood and treated. Here we examine the role of the SH3 domain of postsynaptic density …\n",
      "\n",
      "… mouse anti-PSD-95 (NeuroMab clone K28/43); mouse anti-MAP2 (… NeuroMab antibodies \n",
      "were purchased from the University of California, Davis/National Institutes of Health NeuroMab …\n",
      "\n",
      "… The mouse monoclonal anti-PSD95 antibody was produced against a fusion protein (77–299 \n",
      "aa) of human PSD95/SAP90 (clone K28/43). The antibody recognizes an ∼95 kDa …\n",
      "\n",
      "Dendritic spine loss is observed in many psychiatric disorders, including schizophrenia, and \n",
      "likely contributes to the altered sense of reality, disruption of working memory, and attention …\n",
      "\n",
      "Mutations in leucine-rich repeat kinase 2 (LRRK2), which are associated with autosomal \n",
      "dominant Parkinson's disease, elicit progressive dendrite degeneration in neurons. We …\n",
      "\n",
      "… (NeuroMab, Davis, California, clone K28/43) antibodies were used. For visualization of \n",
      "auditory nerve fibers innervating inner and outer hair cells, rabbit anti-Myo7a and mouse anti-NF-…\n",
      "\n",
      "Purpose In this study, we aimed to characterize the clinical phenotype of a SHANK1-related \n",
      "disorder and define the functional consequences of SHANK1 truncating variants. Methods …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… the K28/43 hybridoma cell line 9 . We initially used this plasmid to express the K28/43 mAb, …\n",
      "\n",
      "Although synaptogenesis within the retina is obviously essential for vision, mechanisms \n",
      "responsible for the initiation and maintenance of retinal synapses are poorly understood. In …\n",
      "\n",
      "Mutations of the cyclin-dependent kinase-like 5 (CDKL5) and netrin-G1 (NTNG1) genes \n",
      "cause a severe neurodevelopmental disorder with clinical features that are closely related to …\n",
      "\n",
      "… We screened eight commercially available antibodies and found best results with anti-PSD95 \n",
      "monoclonal IgG2a (NeuroMab) at 16 ng/ml (a 250-fold dilution of the 4 μg/ml stock solution…\n",
      "\n",
      "The ubiquitously expressed SUMO proteins regulate a plethora of cellular pathways and \n",
      "processes. While they have a predominantly nuclear localization, extranuclear roles of SUMO …\n",
      "\n",
      "Postsynaptic density 95 (PSD-95), the major scaffold protein at excitatory synapses, plays a \n",
      "major role in mediating intracellular signaling by synaptic N-methyl-d-aspartate (NMDA) type …\n",
      "\n",
      "… The monoclonal anti-PSD95 (clone K28/43), GABA B R1 (clone NR3A/49) and GABA B R2 \n",
      "(clone N81/2) antibodies were obtained from the UC Davis/NIH NeuroMab Facility, supported …\n",
      "\n",
      "Kalirin (Kal), a dual Rho GDP / GTP exchange factor ( GEF ), plays essential roles within and \n",
      "outside the nervous system. Tissue‐specific, developmentally regulated alternative splicing …\n",
      "\n",
      "Wnt signaling regulates synaptic development and function and contributes to the fine-tuning \n",
      "of the molecular and morphological differentiation of synapses. We have shown previously …\n",
      "\n",
      "… K28/43, 5 μg/ml; NeuroMab, Davis, CA), synaptophysin, a presynaptic terminal marker (guinea \n",
      "pig, 1:500; SySy, Göttingen, Germany), and tryptophan hydroxylase (TPH, mouse, WH-3 …\n",
      "\n",
      "… Protein (2 mg) was used to immunoprecipitate PSD-95 with 5 μg anti-ms-PSD-95 (K28/43, \n",
      "NeuroMab) and 50 μl Protein G magnetic beads (GE Healthcare). Five micrograms of normal …\n",
      "\n",
      "The mechanisms controlling synapse growth and maintenance are of critical importance for \n",
      "learning and memory. The MAGUK family of synaptic scaffolding proteins is abundantly …\n",
      "\n",
      "… Cultures were blocked with 9% BSA (1 h), and incubated with an antibody for PSD-95 (Neuromab# \n",
      "K28/43, 1: 1000) at room temperature overnight. After washing with PBS (3x), …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "The transsynaptic complex of neuroligin (NLGN) and neurexin forms a physical connection \n",
      "between pre- and postsynaptic neurons that occurs early in the course of new synapse …\n",
      "\n",
      "… To stain for PSD95, the fixed cultures were incubated overnight at 4C with mouse anti-PSD95 \n",
      "(NeuroMab K28/43; 1:500) diluted in KPBS with 0.1% Triton-X 100 (0.1% KPBS) and 5% …\n",
      "\n",
      "During retinal development, the cell-fate of photoreceptors is committed long before \n",
      "maturation, which entails the expression of opsins and functional transduction of light. The …\n",
      "\n",
      "… Synaptoneurosome fractions were then analyzed by western blot analysis using \n",
      "antibodies against PSD95 (NeuroMab, clone K28/43), synapsin I (Chemicon), GFAP (Chemicon), …\n",
      "\n",
      "… (K14/16) and PSD-95 (K28/43) were obtained from the UC Davis/NIH NeuroMab Facility. \n",
      "The rabbit polyclonal antibody to Kv2.1 was obtained from Sigma-Aldrich (P9607). …\n",
      "\n",
      "Platelet activating factor (PAF) is an inflammatory phospholipid signaling molecule implicated \n",
      "in synaptic plasticity, learning and memory and neurotoxicity during neuroinflammation. …\n",
      "\n",
      "… , mouse antibody to PSD-95 (clone K28/43, NeuroMab), rabbit antibody to GRIP1 (Millipore), \n",
      "mouse antibody to PICK1 (clone L20/8, NeuroMab) and rabbit antibody to phospho-Ser880 …\n",
      "\n",
      "Proteins of the major histocompatibility complex class I (MHCI) are known for their role in \n",
      "immunity and have recently been implicated in long-term plasticity of excitatory synaptic …\n",
      "\n",
      "Pyramidal neurons in the mammalian forebrain receive their synaptic inputs through their \n",
      "dendritic trees, and dendritic spines are the sites of most excitatory synapses. Dendritic spine …\n",
      "\n",
      "Guanine nucleotide-binding protein β3 (GNB3) is an isoform of the β subunit of the \n",
      "heterotrimeric G protein second messenger complex that is commonly associated with …\n",
      "\n",
      "Proper neuronal function requires tight control of gene dosage, and failure of this process \n",
      "underlies the pathogenesis of multiple neuropsychiatric disorders. The SHANK3 gene …\n",
      "\n",
      "The cellular and molecular mechanisms underlying photoreceptor synaptogenesis are poorly \n",
      "understood. Furthermore, a detailed picture of the molecular composition of photoreceptor …\n",
      "\n",
      "Synapses are the functional elements of neurons and their defects or losses are at the basis \n",
      "of several neurodegenerative and neurological disorders. Imaging studies are widely used …\n",
      "\n",
      "… PSD-95 (K28/43) and SAP97 (N19/2) antibodies were obtained from Neuromab. ERK 1/2 \n",
      "(extracellular signal-regulated kinase 1/2) antibody was purchased from Cell Signaling …\n",
      "\n",
      "… After 24 h, either control normal rat IgG or PSD-95 antibody (5 μg; UC Davis/NIH NeuroMab \n",
      "Facility clone K28/43) was added to the lysate for co-immunoprecipitation analysis. …\n",
      "\n",
      "… NeuroMab Facility), anti-PSD95 mouse monoclonal antibody (0.5 μg/ml) (K28/43, UC \n",
      "Davis/NIH NeuroMab … (0.5 μg/ml, N355/1, UC Davis/NIH NeuroMab Facility), anti-β-actin mouse …\n",
      "\n",
      "Identifying causative gene(s) within disease-associated large genomic regions of copy-number \n",
      "variants (CNVs) is challenging. Here, by targeted sequencing of genes within …\n",
      "\n",
      "Glioblastoma is a universally lethal form of brain cancer that exhibits an array of \n",
      "pathophysiological phenotypes, many of which are mediated by interactions with the neuronal …\n",
      "\n",
      "Introduction Synapse loss is the structural correlate of the cognitive decline indicative of \n",
      "dementia. In the brains of Alzheimer's disease sufferers, amyloid β (Aβ) peptides aggregate to …\n",
      "\n",
      "A subset of amyotrophic lateral sclerosis (ALS) and frontotemporal lobar degeneration (FTLD) \n",
      "patients present pathological redistribution and aggregation of the nuclear protein fused in …\n",
      "\n",
      "Lateral diffusion on the neuronal plasma membrane of the AMPA-type glutamate receptor (AMPAR) \n",
      "serves an important role in synaptic plasticity. We investigated the role of the …\n",
      "\n",
      "TrkB agonist drugs are shown here to have a significant effect on the regeneration of afferent \n",
      "cochlear synapses after noise-induced synaptopathy. The effects were consistent with …\n",
      "\n",
      "Brain radiotherapy is frequently used successfully to treat brain tumors. However, radiotherapy \n",
      "is often associated with declines in short-term and long-term memory, learning ability, …\n",
      "\n",
      "… (Neuromab, Davis/NIH NeuroMab Facility, Davis, CA), rabbit anti-vesicle-associated \n",
      "membrane protein (VAMP), mouse anti-β-catenin, goat anti-synaptophysin, mouse anti-dishevelled-…\n",
      "\n",
      "… To label intracellular antigens, appropriate dilutions of anti-PSD 95 (clone K28/43) or anti-PDZ \n",
      "mAb (clone K28/86.2; UC Davis/NINDS/NIMH NeuroMab Facility) were incubated …\n",
      "\n",
      "Emerging evidence indicates that microtubule-associated proteins (MAPs) are implicated in \n",
      "synaptic function; in particular, mice deficient for MAP6 exhibit striking deficits in plasticity …\n",
      "\n",
      "… Synapses were defined as puncta with overlapping signal between vGluT1 (Millipore \n",
      "catalogue number AB5905) and PSD95 (K28/43 clone from Neuromab) or vGluT1 and GluA (…\n",
      "\n",
      "It has been shown that vitamin C (VC) is transported at synaptic boutons, but how this occurs \n",
      "has not been elucidated. This study investigates the role of the sodium‐dependent vitamin …\n",
      "\n",
      "… nonimmune goat and rabbit IgG (Sigma); K28/43 (PSD-95 specific) monoclonal \n",
      "antibodies (NeuroMabs); and Enigma/PDLIM7 monoclonal (Abnova) and rabbit polyclonal (Imgenex). …\n",
      "\n",
      "… peptide (aa 659–677), rabbit anti-HA-tag (Santa Cruz Biotechnology), mouse antiHA-tag (Roche \n",
      "Applied Science), mouse synaptophysin (Sigma), mouse PSD-95 K28/43 (NeuroMab), …\n",
      "\n",
      "Tau-mediated toxicity is associated with cognitive decline and Alzheimer's disease (AD) \n",
      "progression. In particular, tau post-translational modifications (PTMs) are thought to generate …\n",
      "\n",
      "Cerebrospinal fluid samples collected from children during initial presentation of central \n",
      "nervous system inflammation, who may or may not subsequently be diagnosed as having …\n",
      "\n",
      "Neurons encode information by rapidly modifying synaptic protein complexes, which \n",
      "changes the strength of specific synaptic connections. Homer1 is abundantly expressed at …\n",
      "\n",
      "Genetic screens in invertebrates have discovered many synaptogenic genes and pathways. \n",
      "However, similar genetic studies have not been possible in mammals. We have optimized …\n",
      "\n",
      "Activity of glycogen synthase kinase-3β (GSK-3β) is required for long-term depression (LTD) \n",
      "via molecular mechanisms that are incompletely understood. Here, we report that PSD-95, …\n",
      "\n",
      "Here, we uncover a mechanism for regulating the number of active presynaptic GABA B \n",
      "receptors (GABA B Rs) at nerve terminals, an important determinant of neurotransmitter release. …\n",
      "\n",
      "… monoclonal anti-CASK (clone K56A/50; NeuroMab, RRID:AB_2068730), mouse monoclonal \n",
      "anti-PSD-95 (clone K28/43; NeuroMab, RRID:AB_2307331), and mouse monoclonal anti-…\n",
      "\n",
      "… Primary antibodies used were mouse monoclonal anti-PSD95 (clone K28/43; Neuromab), \n",
      "mouse monoclonal anti-AU1 (Covance MMS-130R), rat monoclonal anti-HA (clone 3F10; …\n",
      "\n",
      "Photoreceptor ribbon synapses are continuously active synapses with large active zones \n",
      "that contain synaptic ribbons. Synaptic ribbons are anchored to the active zones and are …\n",
      "\n",
      "Wnt ligands play crucial roles in the development and regulation of synapse structure and \n",
      "function. Specifically, Wnt-5a acts as a secreted growth factor that regulates dendritic spine …\n",
      "\n",
      "The efficient knock-in of large DNA fragments to label endogenous proteins remains especially \n",
      "challenging in non-dividing cells such as neurons. We developed Targeted Knock-In with …\n",
      "\n",
      "… 95 clone K28/43 (UC Davis/NIH NeuroMab Facility; AB_444362) antibodies. Five areas of \n",
      "the retina were scanned from at least three sections per animal; 4 animals were used for each …\n",
      "\n",
      "… K28/43, 1:1,000; UC Davis/NIH NeuroMab Facility). Appropriate secondary antibodies \n",
      "were used (anti-rabbit 488, anti-mouse 555; Invitrogen) and the coverslips were mounted onto …\n",
      "\n",
      "Accumulating genetic evidence suggests that schizophrenia (SZ) is associated with individually \n",
      "rare copy number variations (CNVs) of diverse genes, often specific to single cases. …\n",
      "\n",
      "Synaptic degeneration and accumulation of the neurotoxic amyloid β‐peptide (Aβ) in the \n",
      "brain are hallmarks of Alzheimer disease. Aβ is produced by sequential cleavage of the …\n",
      "\n",
      "… Endogenous PSD-95 or gephyrin were labeled using a 1:500 dilution of mouse anti-PSD-95 \n",
      "antibody (NeuroMab, clone K28/43) or mouse anti-gephyrin antibody (Synaptic Systems, …\n",
      "\n",
      "Myosin Va (MyoVa) mediates F-actin-based vesicular transport toward the plasma membrane \n",
      "and is found at neuronal postsynaptic densities (PSDs), but the role of MyoVa in synaptic …\n",
      "\n",
      "… The whole brain crude membrane fractions from WT (2 μg total protein) and KI (20 μg total \n",
      "protein) mice were analyzed with an anti-PSD-95 K28/43 antibody. F, Immunohistochemical …\n",
      "\n",
      "Recent evidence suggests that nano-organization of proteins within synapses may control \n",
      "the strength of communication between neurons in the brain. The unique subsynaptic …\n",
      "\n",
      "Hippocampal CA1 pyramidal cells, which receive γ‐aminobutyric acid (GABA)ergic input \n",
      "from at least 18 types of presynaptic neuron, express 14 subunits of the pentameric GABA A …\n",
      "\n",
      "Phosphorylation regulates surface and synaptic expression of NMDA receptors (NMDARs). \n",
      "Both the tyrosine kinase Fyn and the tyrosine phosphatase striatal-enriched protein tyrosine …\n",
      "\n",
      "Hexanucleotide repeat expansion in C9orf72 is the most common pathogenic mutation in \n",
      "patients with amyotrophic lateral sclerosis (ALS) and frontotemporal lobar degeneration (FTLD)…\n",
      "\n",
      "… anti-Caspr, clone K65/35, NeuroMab, 1:2000 dilution; rabbit polyclonal antimyelin … N206A/8, \n",
      "NeuroMab, 1:2000 dilution; mouse monoclonal anti-PSD95, clone K28/43, NeuroMab, 1:…\n",
      "\n",
      "… , K28/43, NeuroMab, Davis, CA). Following incubation in the primary antibody, the samples \n",
      "were rinsed in 0.1 M PBS (1X; 137 mM NaCl, 26.8 mM KCl, 10.1 mM Na 2 HPO 4 , 17.6 mM …\n",
      "\n",
      "At the post-synaptic density (PSD), large protein complexes dynamically form and dissociate \n",
      "in response to synaptic activity, comprising the biophysical basis for learning and memory. …\n",
      "\n",
      "… Other antibodies used include mouse monoclonal antibody against PSD-95 (clone K28/43; \n",
      "NeuroMab; http://www.neuromab.org) and guinea pig α-green fluorescent protein (GFP; El-…\n",
      "\n",
      "… (University of California-Davis/National Institutes of Health Neuromab Facility), mouse \n",
      "anti-PSD-95 monoclonal clone K28/43 (University of California-Davis/National Institutes of Health …\n",
      "\n",
      "A hippocampal mossy fiber synapse implicated in learning and memory is a complex \n",
      "structure in which a presynaptic bouton attaches to the dendritic trunk by puncta adherentia …\n",
      "\n",
      "BIN1, a member of the BAR adaptor protein family, is a significant late-onset Alzheimer \n",
      "disease risk factor. Here, we investigate BIN1 function in the brain using conditional knockout (…\n",
      "\n",
      "… -95 mAb (1:500, clone K28/43) or a mAb that recognizes all four PSD-95 family members \n",
      "(1:500, clone K28/86.2; UC Davis/NINDS/NIMH NeuroMab Facility Antibodies Inc.). The cells …\n",
      "\n",
      "The purpose of this study was to develop ELISAs for key neural proteins, three synaptic and \n",
      "one glial, that exist in different intracellular compartments, which would be used as a …\n",
      "\n",
      "Neuronal gene expression is modulated by activity via calcium-permeable receptors such \n",
      "as NMDA receptors (NMDARs). While gene expression changes downstream of evoked …\n",
      "\n",
      "A specialized axonal ending, the basket cell “pinceau,” encapsulates the Purkinje cell axon \n",
      "initial segment (AIS), exerting final inhibitory control over the integrated outflow of the …\n",
      "\n",
      "The persistent activity of protein kinase Mzeta (PKMζ), a brain‐specific, constitutively active \n",
      "protein kinase C isoform, maintains synaptic long‐term potentiation (LTP). Structural …\n",
      "\n",
      "Background Nitric oxide synthase 1 adaptor protein (NOS1AP; previously named CAPON) \n",
      "is linked to the glutamatergic postsynaptic density through interaction with neuronal nitric …\n",
      "\n",
      "… ); mouse monoclonal anti-PSD-95 (clone K28/43; NeuroMab); mouse monoclonal anti-β-actin \n",
      "(… anti-CASK (clone K56A/50; NeuroMab). The following antibodies have been previously …\n",
      "\n",
      "… density-95 (PSD95) (clone K28/43 at 0.1 μg/ml; University of California, Davis/National \n",
      "Institutes of Health NeuroMab Facility) and detected using chemiluminescence. Bands on …\n",
      "\n",
      "… Primary antibodies used were as follows: PSD-95 (clone K28/43 UC Davis/NIH Neuromab \n",
      "Facility), vGlut1 (clone N59/36, UC Davis/NIH Neuromab Facility), Syp (sc-7568, Santa Cruz …\n",
      "\n",
      "In the spinal nerve ligation (SNL) model of neuropathic pain, synaptic plasticity shifts the \n",
      "excitation/inhibition balance toward excitation in the spinal dorsal horn. We investigated the …\n",
      "\n",
      "Introduction: Although neurofilaments are mainly expressed in large caliber myelinated axons, \n",
      "recent evidence supports the existence of a specific synaptic pool, where neurofilament …\n",
      "\n",
      "… (K13/31) and anti-PSD95 (K28/43) have been described previously (21-24). Mouse \n",
      "monoclonal antibodies against ectodomains of Kv α subunits, anti-Kv4.2 (K57/1), anti-Kv1.1 …\n",
      "\n",
      "… The mouse monoclonal antibodies PSD95 (Clone K28/43) for immunoblotting, and NR2B \n",
      "(Clone N59-36) were developed by and obtained from the UC Davis/NIH Neuromab facility. …\n",
      "\n",
      "A hippocampal mossy fiber synapse, which is implicated in learning and memory, has a \n",
      "complex structure in which mossy fiber boutons attach to the dendritic shaft by puncta …\n",
      "\n",
      "Synapses are found in vast numbers in the brain and contain complex proteomes. We \n",
      "developed genetic labeling and imaging methods to examine synaptic proteins in individual …\n",
      "\n",
      "… were all based on mAbs from NeuroMab (UC Davis/NIH NeuroMab Facility, nd). This facility \n",
      "… 2013), our current efforts to systematically convert NeuroMab mAbs into scFvs that retain the …\n",
      "\n",
      "Activity-dependent changes in the number of AMPA-type glutamate receptors (AMPARs) at \n",
      "the synapse underpin the expression of LTP and LTD, cellular correlates of learning and …\n",
      "\n",
      "Compact myelin, the paranode, and the juxtaparanode are discrete domains that are formed \n",
      "on myelinated axons. In humans, neurological disorders associated with loss of myelin, …\n",
      "\n",
      "Ionotropic neurotransmitter receptors mediate fast synaptic transmission by functioning as \n",
      "ligand-gated ion channels. Fast inhibitory transmission in the brain is mediated mostly by …\n",
      "\n",
      "The associations between cognitive function and neuropathological markers in patients with \n",
      "mild cognitive impairment (MCI) and Alzheimer disease (AD) remain only partly defined. We …\n",
      "\n",
      "… The following primary antibodies were used: mouse ankyrin-G (Neuromab; N106/36), \n",
      "rabbit ankyrin-G (SantaCruz; sc-28561), rabbit Homer1b/c (Antibodies-online; ABIN-1742339), …\n",
      "\n",
      "… The monoclonal antibodies PSD-95 (Clone K28/43) and GluA2 (Clone L21/32) were \n",
      "developed by and/or obtained from the UC Davis/NIH NeuroMab Facility and maintained by the …\n",
      "\n",
      "Alzheimer’s disease (AD) is a devastating neurodegenerative disorder characterized by \n",
      "pathological brain lesions and a decline in cognitive function. β-Amyloid peptides (Aβ), derived …\n",
      "\n",
      "Autism spectrum disorder (ASD) is a neurological condition characterized by difficulties in \n",
      "social 22 interaction, communication, and behavior. The classical type II cadherins cadherin-8 (…\n",
      "\n",
      "… Immunocytochemistry experiments were performed using mouse monoclonal PSD95 (K28/43 \n",
      "from NeuroMab) antibodies, rabbit vGLUT1 antibody ( …\n",
      "\n",
      "Despite the potential of rodent models of maternal immune activation (MIA) to identify new \n",
      "biomarkers and therapeutic interventions for a range of psychiatric disorders, current …\n",
      "\n",
      "In Alzheimer's disease, fibrillar tau pathology accumulates and spreads through the brain \n",
      "and synapses are lost. Evidence from mouse models indicates that tau spreads trans-…\n",
      "\n",
      "Angelman syndrome (AS) is a neurodevelopmental disorder that results from deletions or \n",
      "mutations in chromosome 15, which usually includes the UBE3A gene. Ube3A protein is an E3 …\n",
      "\n",
      "… ) at 1:5000, PSD95 with mouse antibody clone K28/43 at 0.1 µg/ml (University of California, \n",
      "Davis/National Institutes of Health NeuroMab Facility) and actin with mouse c4 monoclonal …\n",
      "\n",
      "… (clone K28/43; Neuromab), mouse monoclonal anti-gephyrin (clone 3B11; Synaptic Systems), \n",
      "and rabbit polyclonal anti-ARF6 (Abcam). Rabbit polyclonal anti-IQSEC3 (JK079) [29] and …\n",
      "\n",
      "Leukocyte common antigen-related receptor tyrosine phosphatases (LAR-RPTPs) are \n",
      "evolutionarily conserved presynaptic organizers. The synaptic role of vertebrate LAR-RPTPs in …\n",
      "\n",
      "Hebbian, or associative, forms of synaptic plasticity are considered the molecular basis of \n",
      "learning and memory. However, associative synaptic modifications, including long-term …\n",
      "\n",
      "Despite the central role PSD-95 plays in anchoring postsynaptic AMPARs, how PSD-95 \n",
      "itself is tethered to postsynaptic sites is not well understood. Here we show that the F-actin …\n",
      "\n",
      "The Ca 2+ /calmodulin-dependent protein kinase II (CaMKII) is critically required for the synaptic \n",
      "recruitment of AMPA-type glutamate receptors (AMPARs) during both development and …\n",
      "\n",
      "Synaptic plasticity, the cellular basis of learning and memory, involves the dynamic \n",
      "trafficking of AMPA receptors (AMPARs) into and out of synapses. One of the remaining key …\n",
      "\n",
      "Synapses are specialized adhesion sites between neurons that are connected by protein \n",
      "complexes spanning the synaptic cleft. These trans‐synaptic interactions can organize …\n",
      "\n",
      "Heterotetrameric glutamate receptors are essential for the development, function, and \n",
      "plasticity of spine synapses but how they are organized to achieve this is not known. Here we …\n",
      "\n",
      "During NMDA receptor-mediated long-term potentiation (LTP), synapses are strengthened \n",
      "by trafficking AMPA receptors to the synapse through a calcium-dependent kinase cascade …\n",
      "\n",
      "Introduction. Oligomeric and fibrillar species of the synaptic protein α-synuclein are established \n",
      "key players in the pathophysiology of Parkinson’s disease and other synucleinopathies. …\n",
      "\n",
      "Mutations of the Interleukin-1-receptor accessory protein like 1 (IL1RAPL1) gene are associated \n",
      "with cognitive impairment ranging from non-syndromic X-linked mental retardation to …\n",
      "\n",
      "Brain derived neurotrophic factor (BDNF) and its receptor tropomyosin kinase receptor B (TRKB) \n",
      "are key regulators of activity‐dependent plasticity in the brain. TRKB is the target for …\n",
      "\n",
      "Mutations or altered protein levels of SHANK3 are implicated in neurodevelopmental disorders \n",
      "such as Phelan–McDermid syndrome, autism spectrum disorders, and schizophrenia ( …\n",
      "\n",
      "Excitatory neurotransmission and its activity-dependent plasticity are largely determined by \n",
      "AMPA-receptors (AMPARs), ion channel complexes whose cell physiology is encoded by …\n",
      "\n",
      "Objectives Normal cellular function requires a rate of ATP production sufficient to meet \n",
      "demand. In most neurodegenerative diseases (including Amyotrophic Lateral Sclerosis [ALS]), …\n",
      "\n",
      "The cysteine protease caspase-3, best known as an executioner of cell death in apoptosis, \n",
      "also plays a non-apoptotic role in N-methyl-d-aspartate receptor-dependent long-term …\n",
      "\n",
      "Debilitating chronic pain resulting from genetic predisposition, injury, or acquired neuropathy \n",
      "is becoming increasingly pervasive. Opioid analgesics remain the gold standard for …\n",
      "\n",
      "… Primary antibodies used were as follows: PSD-95 (clone K28/43 UC Davis/NIH Neuromab \n",
      "Facility), vGlut1 (clone N59/36, UC Davis/NIH Neuromab Facility), Syp (sc-7568, Santa Cruz …\n",
      "\n",
      "CDKL5 disorder is a devastating neurodevelopmental disorder associated with epilepsy, \n",
      "developmental retardation, autism, and related phenotypes. Mutations in the CDKL5 gene, …\n",
      "\n",
      "10.7554/eLife.04287.001 Lrp4, the muscle receptor for neuronal Agrin, is expressed in the \n",
      "hippocampus and areas involved in cognition. The function of Lrp4 in the brain, however, is …\n",
      "\n",
      "Muscleblind‐like (MBNL) is a CCCH zinc finger‐containing RNA‐binding protein required \n",
      "for the development of both muscle and photoreceptors in Drosophila; it is conserved …\n",
      "\n",
      "10.7554/eLife.24192.001 Glutamatergic synapses rely on AMPA receptors (AMPARs) for fast \n",
      "synaptic transmission and plasticity. AMPAR auxiliary proteins regulate receptor trafficking, …\n",
      "\n",
      "Parallel visual pathways originate at the first synapse in the retina, where cones make \n",
      "connections with cone bipolar cells that in turn contact ganglion cells. There are more ganglion …\n",
      "\n",
      "Altered excitatory/inhibitory (E/I) balance is implicated in neuropsychiatric and neurodevelopmental \n",
      "disorders, but the underlying genetic etiology remains poorly understood. Copy …\n",
      "\n",
      "The association between cognitive function and neuropathological markers remains only \n",
      "partly defined in mild cognitive impairment (MCI) and Alzheimer’s disease (AD). We …\n",
      "\n",
      "The triggering receptor expressed on myeloid cells 2 (TREM2) is a microglial innate immune \n",
      "receptor associated with a lethal form of early, progressive dementia, Nasu-Hakola disease…\n",
      "\n",
      "Organization of signaling complexes at excitatory synapses by membrane-associated guanylate \n",
      "kinase (MAGUK) proteins regulates synapse development, plasticity, senescence and …\n",
      "\n",
      "The formation and remodeling of mossy fiber-CA3 pyramidal cell synapses in the stratum \n",
      "lucidum of the hippocampus are implicated in the cellular basis of learning and memory. Afadin …\n",
      "\n",
      "During development, pyramidal neurons undergo dynamic regulation of AMPA receptor (AMPAR) \n",
      "subunit composition and density to help drive synaptic plasticity and maturation. …\n",
      "\n",
      "The ability of dendritic spines to change size and shape rapidly is critical in modulating synaptic \n",
      "strength; these morphological changes are dependent upon rearrangements of the actin …\n",
      "\n",
      "PSD-95 is a scaffolding protein that regulates the synaptic localization of many receptors, \n",
      "channels, and signaling proteins. The NLGN gene family encodes single-pass transmembrane …\n",
      "\n",
      "Information processing and storage in the brain rely on AMPA-receptors (AMPARs) and \n",
      "their context-dependent dynamics in synapses and extra-synaptic sites. We found that …\n",
      "\n",
      "Elucidating how synaptic molecules such as AMPA receptors mediate neuronal communication \n",
      "and tracking their dynamic expression during behavior is crucial to understand cognition …\n",
      "\n",
      "The accumulation of insoluble amyloid-beta (Aβ) peptides is associated with neurodegenerative \n",
      "disorders, such as Alzheimer's disease (AD). As essential tremor (ET) could involve …\n",
      "\n",
      "Spatially heterogeneous synapse loss is a characteristic of many psychiatric and neurological \n",
      "disorders, but the underlying mechanisms are unclear. Here, we show that spatially-…\n",
      "\n",
      "Robust approaches for chemogenetic control of protein function would have many biological \n",
      "applications. We developed stabilizable polypeptide linkages (StaPLs) based on hepatitis …\n",
      "\n",
      "AMPA receptors and interacting proteins are importantly involved in mediating stress-dependent \n",
      "plasticity. Previously we reported that GluA1-containing AMPA receptors and their …\n",
      "\n",
      "Structural plasticity of dendritic spines and synapses is a fundamental mechanism governing \n",
      "neuronal circuits and may form an enduring basis for information storage in the brain. We …\n",
      "\n",
      "When chronically silenced, cortical and hippocampal neurons homeostatically upregulate \n",
      "excitatory synaptic function. However, the subcellular position of such changes on the …\n",
      "\n",
      "The ubiquitination/proteasome system is important for the spatiotemporal control of protein \n",
      "synthesis and degradation at synapses, while dysregulation may underlie autism spectrum …\n",
      "\n",
      "Bipolar disorder (BD) is a heritable neuropsychiatric disorder with largely unknown \n",
      "pathogenesis. Given their prominent role in brain function and disease, we hypothesized that …\n",
      "\n",
      "The neural cell recognition molecule NB‐3, also referred to as contactin‐6, is expressed \n",
      "prominently in the developing nervous system after birth and its deficiency has been shown to …\n",
      "\n",
      "NMDA receptors (NMDARs) are key mediators of glutamatergic transmission and synaptic \n",
      "plasticity, and their dysregulation has been linked to diverse neuropsychiatric and …\n",
      "\n",
      "The proper formation and maintenance of functional synapses in the central nervous system \n",
      "(CNS) requires communication between neurons and astrocytes and the ability of …\n",
      "\n",
      "Coding of itch versus pain has been heatedly debated for decades. However, the current \n",
      "coding theories (labeled line, intensity, and selectivity theory) cannot accommodate all …\n",
      "\n",
      "The leucine-rich glioma-inactivated (LGI) family consists of four highly conserved paralogous \n",
      "genes, LGI1-4, that are highly expressed in mammalian central and/or peripheral nervous …\n",
      "\n",
      "Paraformaldehyde ( PFA ) is the most commonly used fixative for immunostaining of cells, but \n",
      "has been associated with various problems, ranging from loss of antigenicity to changes in …\n",
      "\n",
      "Synapses in the brain exhibit cell-type-specific differences in basal synaptic transmission \n",
      "and plasticity. Here, we evaluated cell-type-specific specializations in the composition of …\n",
      "\n",
      "Alzheimer’s Disease (AD) is the primary cause of dementia among the elderly population. \n",
      "Elevated plasma levels of homocysteine (HCy), an amino acid derived from methionine …\n",
      "\n",
      "Gap junctions in retinal photoreceptors suppress voltage noise and facilitate input of rod \n",
      "signals into the cone pathway during mesopic vision. These synapses are highly plastic and …\n",
      "\n",
      "Parkinson's disease (PD) and dementia with Lewy bodies (DLB) are common neurodegenerative \n",
      "disorders of the aging population, characterized by progressive and abnormal …\n",
      "\n",
      "This protocol establishes a tri-culture of hiPSC-derived neurons, astrocytes, and microglia \n",
      "for the study of cellular interactions during homeostasis, injury, and disease. This system …\n",
      "\n",
      "Microwave (MW) radiation induced oxidative stress reduces dendritic arborization, spine \n",
      "density and number of hippocampal pyramidal neurons and hence, impair learning and spatial …\n",
      "\n",
      "How does chronic activity modulation lead to global remodeling of proteins at synapses and \n",
      "synaptic scaling? Here we report that guanylate kinase–associated protein (GKAP; also …\n",
      "\n",
      "Dendritic spines are fine neuronal processes where spatially restricted input can induce \n",
      "activity-dependent changes in one spine, while leaving neighboring spines unmodified. …\n",
      "\n",
      "NMDA receptor (NMDAR) subunit composition plays a pivotal role in synaptic plasticity at \n",
      "excitatory synapses. Still, the mechanisms responsible for the synaptic retention of NMDARs …\n",
      "\n",
      "… K14/16), and mouse anti-PSD95 (K28/43) were obtained from University of California, \n",
      "Davis/NIH NeuroMab Facility. Secondary antibodies used for immunofluorescence were as follows…\n",
      "\n",
      "The Par3 polarity protein is critical for subcellular compartmentalization in different developmental \n",
      "processes. Variants of PARD3, which encodes PAR3, are associated with intelligence …\n",
      "\n",
      "Mafb and c-Maf transcription factor (TF) expression is enriched in medial ganglionic eminence \n",
      "(MGE) lineages, beginning in late-secondary progenitors and continuing into mature …\n",
      "\n",
      "… 1b (subtype IgG 1 ; clone N16b/8), and PSD95 (subtype IgG 2a ; clone K28/43) were \n",
      "obtained from the University of California Davis/National Institutes of Health NeuroMab Facility. …\n",
      "\n",
      "… GluN1 NMDA receptor subunit mouse monoclonal antibody (556308; BD Biosciences), \n",
      "PSD-95 mouse monoclonal antibody (clone K28/43; 73-028; NeuroMab) and neuroligin1 rabbit …\n",
      "\n",
      "Multiple lines of evidence suggest altered synaptic plasticity/connectivity as a pathophysiologic \n",
      "mechanism for various symptom domains of schizophrenia. Olfactory dysfunction, an …\n",
      "\n",
      "Tuberous sclerosis complex (TSC) is a genetic disorder caused by mutations in TSC1 or TSC2. \n",
      "Patients frequently have epilepsy, autism spectrum disorder, and/or intellectual disability, …\n",
      "\n",
      "SLITRK2 is a single-pass transmembrane protein expressed at postsynaptic neurons that \n",
      "regulates neurite outgrowth and excitatory synapse maintenance. In the present study, we …\n",
      "\n",
      "p21-activated kinase 1 (PAK1) and PAK3 belong to group I of the PAK family and control cell \n",
      "movement and division. They also regulate dendritic spine formation and maturation in the …\n",
      "\n",
      "… , CA); PSD-95 (clone K28/43), synapse-associated protein 102 (SAP102) (clone N19/2), \n",
      "PSD-93 (clone N18/30), and SAP97 (clone K64/15) from NeuroMab (Davis, CA); neuronal nitric …\n",
      "\n",
      "Regulation of AMPA-type glutamate receptor (AMPAR) number at synapses is a major \n",
      "mechanism for controlling synaptic strength during homeostatic scaling in response to global …\n",
      "\n",
      "Dendritic spines are postsynaptic structures in neurons often having a mushroom-like shape. \n",
      "Physiological significance and cytoskeletal mechanisms that maintain this shape are …\n",
      "\n",
      "Alzheimer's disease (AD) is a neurodegenerative disorder characterized by a progressive \n",
      "deterioration of cognitive abilities, accumulation of the amyloid-β peptide (Aβ), increase of …\n",
      "\n",
      "Cadherins function in the adhesion of presynaptic and postsynaptic membranes at \n",
      "excitatory synapses. Here we show that the cadherin-associated protein neural plakophilin-related …\n",
      "\n",
      "… clone B-5-1-2 (Sigma), antib-actin clone C4 (Millipore), and anti-PSD-95 clone K28/43 (NeuroMab). \n",
      "All drugs and chemicals were obtained from Tocris Biosciences or Ascent Scientific. …\n",
      "\n",
      "Abnormalities in neocortical and synaptic development are linked to neurodevelopmental \n",
      "disorders. However, the molecular and cellular mechanisms governing initial synapse …\n",
      "\n",
      "Microtubule‐associated protein (MAP) 2 has been perceived as a static cytoskeletal protein \n",
      "enriched in neuronal dendritic shafts. Emerging evidence indicates dynamic functions for …\n",
      "\n",
      "Microtubule organization and dynamics are essential during axon and dendrite formation \n",
      "and maintenance in neurons. However, little is known about the regulation of microtubule …\n",
      "\n",
      "The transport and translation of dendritic mRNAs by RNA-binding proteins (RBPs) allows \n",
      "for spatially restricted gene expression in neuronal processes. Although local translation in …\n",
      "\n",
      "… 75–001) and anti-PSD95 (clone K28/43, IgG2a, cat# 75–028) from Neuromab. We used \n",
      "the following secondary antibodies conjugated with Alexa Fluor fluorochromes: 488 goat anti-…\n",
      "\n",
      "ADNP syndrome, involving the ADNP transcription factor of the SWI/SNF chromatin-remodeling \n",
      "complex, is characterized by developmental delay, intellectual disability, and autism …\n",
      "\n",
      "Alternative splicing in the 3’ untranslated region (3’UTR) of mammalian genes plays a crucial \n",
      "role in diverse biological processes, including cell differentiation and development. SAM68 …\n",
      "\n",
      "The proteolytic machinery comprising metalloproteases and γ-secretase, an intramembrane \n",
      "aspartyl protease involved in Alzheimer's disease, cleaves several substrates in addition to …\n",
      "\n",
      "Postsynaptic density 95 (PSD-95) is a major synaptic scaffolding protein that plays a key role \n",
      "in bidirectional synaptic plasticity, which is a process important for learning and memory. It …\n",
      "\n",
      "Fragile X syndrome (FXS) is the most frequent form of inherited intellectual disability and the \n",
      "best-described monogenic cause of autism. CGG-repeat expansion in the FMR1 gene …\n",
      "\n",
      "… ) anti-PSD-95 (clone K28/43, 1:1000; Upstate biochemical), anti-GluR1 (extracellular, 1:1000; \n",
      "AGC-004, AbCAM), anti-vGlut1, (1:1000, Clone N28/9; NeuroMAB). The following day the …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "Synaptic plasticity requires remodeling of the actin cytoskeleton. Although two actin isoforms, \n",
      "β- and γ-actin, are expressed in dendritic spines, the specific contribution of γ-actin in the …\n",
      "\n",
      "Postsynaptic density 95 (PSD-95), the major scaffold at excitatory synapses, is critical for \n",
      "synapse maturation and learning. In rodents, eye opening, the onset of pattern vision, triggers a …\n",
      "\n",
      "Experience results in long-lasting changes in dendritic spine size, yet how the molecular \n",
      "architecture of the synapse responds to plasticity remains poorly understood. Here a combined …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "Experience can alter anatomical connectivity in the brain, but such plasticity is strongly \n",
      "suppressed in adulthood. We examined the turnover of dendritic spines and axonal varicosities in …\n",
      "\n",
      "L-methionine, the principal sulfur-containing amino acid in proteins, plays critical roles in \n",
      "cell physiology as an antioxidant and in the breakdown of fats and heavy metals. Previous …\n",
      "\n",
      "Hair cell (HC) loss by epithelial extrusion has been described to occur in the rodent vestibular \n",
      "system during chronic 3, 3′-iminodipropionitrile (IDPN) ototoxicity. This is preceded by …\n",
      "\n",
      "Most disease variants lie within noncoding genomic regions, making their functional interpretation \n",
      "challenging. Because chromatin openness strongly influences transcriptional activity, …\n",
      "\n",
      "Palmitoylation, a key regulatory mechanism controlling protein targeting, is catalyzed by \n",
      "DHHC-family palmitoyl acyltransferases (PATs). Impaired PAT activity is linked to …\n",
      "\n",
      "Neurodevelopmental disorders such as autism spectrum disorders and schizophrenia \n",
      "differentially impact males and females and are highly heritable. The ways in which sex and …\n",
      "\n",
      "Toxic aggregates of α-synuclein (αsyn) are considered key drivers of Parkinson’s disease (PD) \n",
      "pathology. In early PD, αsyn induces synaptic dysfunction also modulating the …\n",
      "\n",
      "Major histocompatibility complex class I (MHCI) molecules negatively regulate cortical \n",
      "connections and are implicated in neurodevelopmental disorders, including autism spectrum …\n",
      "\n",
      "The four members of the LRRTM family (LRRTM1-4) are postsynaptic adhesion molecules \n",
      "essential for excitatory synapse development. They have also been implicated in …\n",
      "\n",
      "Neuronal activity elicits changes in synaptic composition that play an important role in \n",
      "experience-dependent plasticity ( Choquet and Triller, 2003 ; Lisman and Raghavachari, 2006 ; …\n",
      "\n",
      "… Mouse monoclonal anti-CASPR1 (clone K65/35, 1:400) and anti-PSD95 (clone K28/43; 1:200) \n",
      "antibodies were obtained from the UC Davis/NIH NeuroMab Facility. Other antibodies …\n",
      "\n",
      "Protein ubiquitination has a significant influence on diverse aspects of neuronal development \n",
      "and function. Dorfin, also known as Rnf19a, is a RING finger E3 ubiquitin ligase implicated …\n",
      "\n",
      "The proper function of neural circuits requires spatially and temporally balanced development \n",
      "of excitatory and inhibitory synapses. However, the molecular mechanisms coordinating …\n",
      "\n",
      "At hippocampal synapses, activation of group I metabotropic glutamate receptors (mGluRs) \n",
      "induces long-term depression (LTD), which requires new protein synthesis. However, the …\n",
      "\n",
      "BEACH domain proteins are involved in membrane protein traffic and human diseases, but \n",
      "their molecular mechanisms are not understood. The BEACH protein LRBA has been …\n",
      "\n",
      "Syntaphilin (SNPH) is a major mitochondrial anchoring protein targeted to axons and excluded \n",
      "from dendrites. In this study, we provide in vivo evidence that this spatial specificity is lost …\n",
      "\n",
      "Pain and itch are two distinct yet related sensations. Here we present a series of experiments \n",
      "elucidating mechanisms of pain and itch in central and peripheral nervous systems. …\n",
      "\n",
      "Specific formation of excitatory and inhibitory synapses is crucial for proper functioning of \n",
      "the brain. Fibroblast growth factor 22 (FGF22) and FGF7 are postsynaptic-cell-derived …\n",
      "\n",
      "… The Ms mAb to postsynaptic density 95 (PSD-95) was from Millipore (clone 6G6-1C9; \n",
      "catalog #MAB1596; used with rat cultures) or from NeuroMab (clone K28/43; catalog #73-028; …\n",
      "\n",
      "10.7554/eLife.49953.001 The voltage-gated K + channel Kv2.1 serves a major structural \n",
      "role in the soma and proximal dendrites of mammalian brain neurons, tethering the plasma …\n",
      "\n",
      "Global changes of activity in neuronal networks induce homeostatic adaptations of synaptic \n",
      "strengths, which involve functional remodeling of both presynaptic and postsynaptic …\n",
      "\n",
      "The neural circuit mechanisms underlying the integration and functions of adult-born dentate \n",
      "granule cell (DGCs) are poorly understood. Adult-born DGCs are thought to compete with …\n",
      "\n",
      "… Sudhof provided by NeuroMab). Its specificity in chick was evaluated by western blotting \n",
      "using tissue extracts from ED15 or ED21 retinas or COS7 cells transfected with HA-tagged …\n",
      "\n",
      "Dendritic spines are dynamic, actin-rich protrusions in neurons that undergo remodeling during \n",
      "neuronal development and activity-dependent plasticity within the central nervous system…\n",
      "\n",
      "Dendritic filopodia select synaptic partner axons by interviewing the cell surface of potential \n",
      "targets, but how filopodia decipher the complex pattern of adhesive and repulsive molecular …\n",
      "\n",
      "Calsyntenin-3 (Clstn3) is a postsynaptic adhesion molecule that induces presynaptic \n",
      "differentiation via presynaptic neurexins (Nrxns), but whether Nrxns directly bind to Clstn3 has …\n",
      "\n",
      "Kaufman oculocerebrofacial syndrome (KOS) is a severe autosomal recessive disorder \n",
      "characterized by intellectual disability, developmental delays, microcephaly and characteristic …\n",
      "\n",
      "10.7554/eLife.06327.001 Precise regulation of the dendritic spine actin cytoskeleton is \n",
      "critical for neurodevelopment and neuronal plasticity, but how neurons spatially control actin …\n",
      "\n",
      "Synaptic ribbons are large proteinaceous scaffolds at the active zone of ribbon synapses \n",
      "that are specialized for rapid sustained synaptic vesicles exocytosis. A single ribbon‐specific …\n",
      "\n",
      "The etiology of many complex neurological disorders, including schizophrenia (SZ) and HIV-Associated \n",
      "Neurocognitive Disorders (HAND), are still unknown, partly due to a lack of …\n",
      "\n",
      "Functional cochlear hair cells (HCs) innervated by spiral ganglion neurons (SGNs) are essential \n",
      "for hearing, whereas robust models that recapitulate the peripheral auditory circuity are …\n",
      "\n",
      "The process by which excitatory neurons are generated and mature during the development \n",
      "of the cerebral cortex occurs in a stereotyped manner; coordinated neuronal birth, migration…\n",
      "\n",
      "Neuronal alternative splicing is a core mechanism for functional diversification. We \n",
      "previously found that STAR family proteins (SAM68, SLM1, SLM2) regulate spatiotemporal …\n",
      "\n",
      "The beneficial effects of physical activity on brain ageing are well recognised, with exerkines, \n",
      "factors that are secreted into the circulation in response to exercise, emerging as likely …\n",
      "\n",
      "… clone K28/43 (University of California Davis/National Institutes of Health Neuromab Facility). \n",
      "The HA-Epac2-WT construct was a kind gift from Lawrence Quilliam (Indiana University). …\n",
      "\n",
      "Leukocyte common antigen-related receptor tyrosine phosphatases (LAR-RPTPs) are \n",
      "evolutionarily conserved presynaptic organizers. The synaptic role of vertebrate LAR-RPTPs in …\n",
      "\n",
      "Gephyrin is a central scaffold protein that mediates development, function, and plasticity of \n",
      "mammalian inhibitory synapses by interacting with various inhibitory synaptic proteins. Here, …\n",
      "\n",
      "The mitochondrial ATP synthase emerges as key hub of cellular functions controlling the \n",
      "production of ATP, cellular signaling, and fate. It is regulated by the ATPase inhibitory factor 1 (…\n",
      "\n",
      "Progestins bind to the progestin receptor (PR) isoforms, PR-A and PR-B, in brain to influence \n",
      "development, female reproduction, anxiety, and stress. Hormone-activated PRs associate …\n",
      "\n",
      "Genetic variation within the transcription factor TCF4 locus can cause the intellectual \n",
      "disability and developmental disorder Pitt-Hopkins syndrome (PTHS), whereas single-nucleotide …\n",
      "\n",
      "… The monoclonal antibody PSD-95 (Clone K28/43) was obtained from the UC Davis/NIH \n",
      "NeuroMab Facility supported by NIH Grant U24NS050606 and maintained by the Department of …\n",
      "\n",
      "Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) are severe \n",
      "neurodegenerative diseases with overlapping clinical and pathological features. Recently, a …\n",
      "\n",
      "… For β4-subunits lacking an epitope tag, we used a mouse monoclonal antibody targeted \n",
      "to an extracellular epitope of β4 (NeuroMab clone L18A/3). In nonpermeabilized and …\n",
      "\n",
      "Background: Inflammation-induced sensitization of primary afferents is associated with a \n",
      "decrease in K + current. However, the type of K + current and basis for the decrease varies as a …\n",
      "\n",
      "… The membranes were incubated with primary antibodies including anti-BK α (Neuromab, \n",
      "L6/60, 1:1000), anti-BK β2 (Neuromab, N53/32 1:1000), anti-BK β4 (Neuromab, L18A.3, 1:1000…\n",
      "\n",
      "… (hereafter referred to as rbTRPM4), and the mouse monoclonal L88/86 antibody, which was \n",
      "generated for this project. The L88/86 mAb (hereafter referred to as msTRPM4) was raised …\n",
      "\n",
      "… Monoclonal anti-TRPM4 L88/86 antibody was obtained from hybridoma tissue cultures \n",
      "and used as non-diluted supernatant (RRID: AB_2716758). The antibody was validated by …\n",
      "\n",
      "Voltage-gated potassium (Kv) channels are increasingly recognised as key regulators of \n",
      "nociceptive excitability. Kcns1 is one of the first potassium channels to be associated with …\n",
      "\n",
      "… , NeuroMab, clone N134/12) was applied overnight at 4C. According to the product datasheet, \n",
      "the Na v 1.8 antibody used in these experiments was developed against the C terminus of …\n",
      "\n",
      "Maturation of nociceptive neurons depends on changes in transcription factors, ion channels \n",
      "and neuropeptides. Mature nociceptors initiate pain in part by drastically reducing the …\n",
      "\n",
      "Migraine is believed to be initiated by neuronal activity in the CNS, that triggers excitation of \n",
      "nociceptive trigeminal ganglion (TG) nerve fibers innervating the meninges and thus causes …\n",
      "\n",
      "Finding causative genetic mutations is important in the diagnosis and treatment of hereditary \n",
      "peripheral neuropathies. This study was conducted to find new genes involved in the …\n",
      "\n",
      "… on behalf of NeuroMab, the top quality depository of monoclonal antibodies used in … /anti-nav1-8-na-channel-antibody-antibody-n134-12-75-166). \n",
      "Both the anti-ASIC3 and anti-P2X3 …\n",
      "\n",
      "The tetrodotoxin‐resistant ( TTX ‐R) voltage‐gated sodium channel Na v 1.8 is predominantly \n",
      "expressed in peripheral afferent neurons, but in case of neuronal injury an ectopic and …\n",
      "\n",
      "Bacterial products can act on neurons to alter signaling and function. In the present study, we \n",
      "found that dorsal root ganglion (DRG) sensory neurons are enriched for ANTXR2, the high-…\n",
      "\n",
      "… (1:500; UC Davis/NIH NeuroMab Facility, clone N134/12), goat anti-transient receptor \n",
      "potential cation channel subfamily V member 1 (TRPV1, 1:500, P-19, Santa Cruz), rabbit anti-…\n",
      "\n",
      "Depolarization drives neuronal plasticity. However, whether depolarization drives \n",
      "sensitization of peripheral nociceptive neurons remains elusive. By high-content screening (HCS) …\n",
      "\n",
      "Rationale Structural differences between ventricular regions may not be the sole determinant \n",
      "of local ventricular fibrillation (VF) dynamics and molecular remodeling may play a role. …\n",
      "\n",
      "Rationale: Structural differences between ventricular regions may not be the sole determinant \n",
      "of local ventricular fibrillation (VF) dynamics and molecular remodeling may play a role. …\n",
      "\n",
      "… The lysates were purified with anti-TRPC4β monoclonal antibody (N77/15, UC Davis/NIH \n",
      "NeuroMab Facility) at 4 C, followed by the addition of protein G (GE Healthcare) and …\n",
      "\n",
      "… was immunoprecipitated with 5 μg of anti-TRPC4 antibody (NeuroMab, University of California \n",
      "(UC) Davis) or 5 μg of anti-TRPC5 antibody (NeuroMab, UC Davis) and 10 μg of protein A …\n",
      "\n",
      "The neural cell adhesion molecule (NCAM) plays important functional roles in the developing \n",
      "and mature nervous systems. Here, we show that the transient receptor potential canonical …\n",
      "\n",
      "… Similar results were obtained using multiple Abs against AMIGO-1 (data not shown), including \n",
      "two different mouse mAbs with distinct cytoplasmic domain binding sites (L86/33 and L86A…\n",
      "\n",
      "Developmental and epileptic encephalopathies (DEE) are a group of severe epilepsies that \n",
      "usually present with intractable seizures, developmental delay, and often have elevated risk …\n",
      "\n",
      "… CMSSID[pS]FISSA; anti-AMIGO mAb L86/33, generated against amino acids 395-493 (… \n",
      "L86/33, and L86A/37 are available from the UC Davis/NIH NeuroMab Facility (http://neuromab…\n",
      "\n",
      "… Anti-snapin antibody (clone L8/15) was purchased from NeuroMab Inc. (Davis, CA, USA). \n",
      "Antibodies to ANXA1 (sc-12740), AKAP150 (sc-10765), dynein intermediate chain (DIC, sc-…\n",
      "\n",
      "Neuron-glia interactions establish functional membrane domains along myelinated axons. \n",
      "These include nodes of Ranvier, paranodal axoglial junctions and juxtaparanodes. …\n",
      "\n",
      "… We had generated two anti-KChIP1 mouse monoclonal antibodies (K55/7 and K55/29) of \n",
      "distinct … Control double-labeling experiments using K55/7 and K55/29 revealed the expected …\n",
      "\n",
      "… The NeuroMab KChIP1 antibody stained a double band with … The NeuroMab KChIP2 antibody \n",
      "stained a single band with a … The NeuroMab KChIP3 antibody stained a single band with …\n",
      "\n",
      "… As representative examples, the catalog of the UC Davis/NIH NeuroMab facility (as of 12/17/11) \n",
      "contains 278 mAbs, and a representative part of the commercial catalog of EMD …\n",
      "\n",
      "… The tissue was transferred to primary antibody solution (NeuroMab, Anti-KChIP1, clone \n",
      "K55/7, 1:200) for overnight incubation at 4 C. The next day, sections were again serially rinsed in …\n",
      "\n",
      "… The specificity of each of the primary antibodies has been verified using Western blots \n",
      "from brain tissue by the UC, Davis/NeuroMab Facility and also in our laboratory in BLA tissue …\n",
      "\n",
      "… , K55/7 (available from the University of California, Davis NeuroMab Facility; www.neuromab.… \n",
      "In immunofluorescence and immunoblot assays, K55/7 specifically recognizes recombinant …\n",
      "\n",
      "… Mouse anti-KChIP1 (University of California at Davis / NIH NeuroMab Facility, Davis, CA; \n",
      "abbreviated NeuroMab; Cat. no. 75-003, clone K55/7, RRID:AB_10673162) was generated …\n",
      "\n",
      "… Mouse anti-KChIP1 (NeuroMab, Cat. no. 75-003, clone K55/7, RRID:AB_10673162) was \n",
      "generated against a recombinant fusion protein containing full-length (amino acids 1–216) of …\n",
      "\n",
      "Water diffusion magnetic resonance imaging (dMRI) is a powerful tool for studying biological \n",
      "tissue microarchitectures in vivo. Recently, there has been increased effort to develop …\n",
      "\n",
      "Potassium channels comprise the most diverse family of ion channels. In nerve cells, their \n",
      "critical roles in synaptic integration and output generation have been demonstrated. Here, we …\n",
      "\n",
      "… We used the following primary antibodies from NeuroMab (UC Davis/NIH, Davis, CA, \n",
      "USA): (1) monoclonal anti-Kv4.2 (K57/1; amino acid. 209–225 of human Kv4.2, Q9NZV8), (2) …\n",
      "\n",
      "… NeuroMab, a joint facility of the University of California-Davis and the NIH (http://www.neuromab.… \n",
      "The mAb produced using the constant domain (N159/5, anti-γC) detected all of the over-…\n",
      "\n",
      "… F, anti-GCP (upper blots) detects a portion of endogenous γ-Pcdhs (all γ-Pcdhs were detected \n",
      "with mAb N159/5, lower blots) in embryonic brain and postnatal cerebral cortex. The lack …\n",
      "\n",
      "… The Ms mAb to pan-Pcdh-γ (clone N159/5, to amino acids 808–931 of the constant \n",
      "cytoplasmic domain of mouse Pcdh-γA1 that is shared by all 22 Pcdh-γs) and the Ms mAb to pan-…\n",
      "\n",
      "The 22 γ-Protocadherin (γ-Pcdh) cell adhesion molecules are critical for the elaboration of \n",
      "complex dendritic arbors in the cerebral cortex. Here, we provide evidence that the γ-Pcdhs …\n",
      "\n",
      "Growth of a properly complex dendrite arbor is a key step in neuronal differentiation and a \n",
      "prerequisite for neural circuit formation. Diverse cell surface molecules, such as the clustered …\n",
      "\n",
      "It has been proposed that the combinatorial expression of γ‐protocadherins (Pcdh‐γs) and \n",
      "other clustered protocadherins (Pcdhs) provides a code of molecular identity and individuality …\n",
      "\n",
      "… (1:500–1:1000) were generated by NeuroMab in collaboration with the Weiner laboratory \n",
      "[55] and obtained from Antibodies, Inc.: N159/5 (detecting an epitope in constant exon 1 or 2 …\n",
      "\n",
      "… The mouse anti-γ-Pcdh constant domain (N159/5) and mouse anti-γ-Pcdh-B2 (N148/30) \n",
      "were generated by the UC Davis/NIH NeuroMab Facility using antigens produced in our …\n",
      "\n",
      "Strictly controlled dendrite patterning underlies precise neural connection. Dendrite self-avoidance \n",
      "is a crucial system preventing self-crossing and clumping of dendrites. Although …\n",
      "\n",
      "… A monoclonal antibody to the Pcdh-γ constant domain (clone N159/5) was obtained from \n",
      "the University of California, Davis/National Institutes of Health NeuroMab Facility. Monoclonal …\n",
      "\n",
      "Thalamocortical neurons (TCNs) play a critical role in the maintenance of thalamocortical \n",
      "oscillations, dysregulation of which can result in certain types of seizures. Precise control over …\n",
      "\n",
      "In sensory circuits of the brain, developmental changes in the expression and modulation of \n",
      "voltage-gated ion channels are a common occurrence, but such changes are often difficult …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… immunolabeling with the anti-HCN4 hybridoma-generated N114/10 mAb to the N114/10R R-…\n",
      "\n",
      "… terminus of rat HCN1 NeuroMab, 75-110, Mouse monoclonal AB_2115181 1:200 HCN4 \n",
      "clone N114/10 Fusion protein amino acids 1019-1108 at C-terminus of rat HCN4 NeuroMab, 75-…\n",
      "\n",
      "… The HCN4 results of this study were obtained with a mouse monoclonal antibody directed \n",
      "against amino acids 1019–1108 of the rat HCN4 sequence (clone N114/10; NeuroMab) and …\n",
      "\n",
      "Immunohistochemical and ex vivo anatomical studies have provided many glimpses of the \n",
      "variety, distribution, and signaling components of vertebrate retinal neurons. The beauty of …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… immunolabeling with the anti-HCN4 hybridoma-generated N114/10 mAb to the N114/10R R-…\n",
      "\n",
      "Large-conductance Ca 2+ - and voltage-activated K + (BK) channels play prominent roles in \n",
      "shaping muscle and neuronal excitability. In the cardiovascular system, BK channels …\n",
      "\n",
      "… , and HCN4 were used (all from UC Davis/NIH Neuromab Facility, clones N71/37, N70/28, \n",
      "N141/28, N114/10, respectively). Tissue sections and vestibular ganglion neuron cultures …\n",
      "\n",
      "Purpose.: The membrane expression and gene promoter of the glycosylphosphatidylinositol \n",
      "(GPI)-anchored protein Thy1 have been widely used to examine the morphology and …\n",
      "\n",
      "Hyperpolarisation-activated (I h ) currents are considered important for dendritic integration, \n",
      "synaptic transmission, setting membrane potential and rhythmic action potential (AP) …\n",
      "\n",
      "Processes of >100 types of interneurons (bipolar and amacrine cells) and projection neurons \n",
      "(retinal ganglion cells, RGCs) form specific and stereotyped patterns of connections in the …\n",
      "\n",
      "Supplementary Figure 2 (continued) Ten electron micrographs taken from serial sections of \n",
      "neurobiotin-labeled (HRP end-product) boutons (B) of AAC B45a making type II synapses (…\n",
      "\n",
      "Malignant gliomas are highly invasive brain cancers that carry a dismal prognosis. Recent \n",
      "studies indicate that Cl − channels facilitate glioma cell invasion by promoting hydrodynamic …\n",
      "\n",
      "… The monoclonal antibody to ClC-3 (cat#75-259, clone N258/5) was developed and/or \n",
      "obtained from the UC Davis/NIH NeuroMab Facility, supported by NIH grant U24NS050606 and …\n",
      "\n",
      "Malignant gliomas are the most common and deadly form of primary brain cancer afflicting \n",
      "adults. Current treatment regimens, including surgical debulking, radiotherapy, and …\n",
      "\n",
      "… with 50 ng/ml of K75/41 at which concentration this monoclonal antibody is not detected by \n",
      "Alexa-conjugated secondary antibodies (not shown). The K75/41 signal was detected with a …\n",
      "\n",
      "Precise axon pathfinding is crucial for establishment of the initial neuronal network during \n",
      "development. Pioneer axons navigate without the help of preexisting axons and pave the way …\n",
      "\n",
      "Kv4 channels mediate most of the somatodendritic subthreshold operating A‐type current (I \n",
      "SA ) in neurons. This current plays essential roles in the regulation of spike timing, repetitive …\n",
      "\n",
      "… (K57/41.1, 1:1000; NeuroMab) or the mouse anti-Kv4.3 (K75/41, 1:1000; NeuroMab) antibody \n",
      "with a rabbit anti-GFAP primary antibody (G-9269, 1:2000; Sigma, St. Louis, MO), a rabbit …\n",
      "\n",
      "KCND3 has been identified as the disease gene for spinocerebellar ataxia type (SCA) 19/22. \n",
      "De novo variants were subsequently identified in patients with childhood-onset cerebellar …\n",
      "\n",
      "… The NeuroMab KChIP1 antibody stained a double band with … The NeuroMab KChIP2 antibody \n",
      "stained a single band with a … The NeuroMab KChIP3 antibody stained a single band with …\n",
      "\n",
      "Spectrins are cytoskeletal proteins essential for membrane biogenesis and regulation and \n",
      "serve critical roles in protein targeting and cellular signaling. αII spectrin (SPTAN1) is one of …\n",
      "\n",
      "… anti-K V 4.3 (1:5, clone K75/41; NeuroMab, UC Davis/NIH NeuroMab Facility) and mouse \n",
      "monoclonal anti-K V 7.4 (1:50, clone N43/6; NeuroMab) were used to detect various potassium …\n",
      "\n",
      "… (NeuroMab, Cat. no. 75-017, clone K75/41, RRID:AB_2314723) was raised against a fusion \n",
      "protein containing amino acids 415–636 of rat Kv4.3. Western blot identified a single band …\n",
      "\n",
      "… K75/41 (IgG1), antiKChIP2 K60/73 (IgG1), and anti-KChIP3 K66/38 (IgG2a). These mAbs \n",
      "are available from the UC Davis/ NIH NeuroMab Facility (neuromab.ucdavis.edu). Rabbit …\n",
      "\n",
      "Potassium channels comprise the most diverse family of ion channels. In nerve cells, their \n",
      "critical roles in synaptic integration and output generation have been demonstrated. Here, we …\n",
      "\n",
      "Spectrins are essential for vertebrate membrane biogenesis and regulation and serve critical \n",
      "roles in cellular signaling. Expressed in most vertebrate cells, αII spectrin is one of two …\n",
      "\n",
      "… (NeuroMab, Cat. no. 75-017, clone K75/41, RRID:AB_2314723) was raised against a \n",
      "recombinant protein containing amino acids 415–636 of rat Kv4.3 (full-length 655 residues). This …\n",
      "\n",
      "… Immunodetections of K v 4.2 and K v 4.3 channels were performed using monoclonal \n",
      "antibodies from Neuro Mab (clones K57/1 and K75/41, respectively). These antibodies have been …\n",
      "\n",
      "… The specificity of each of the primary antibodies has been verified using Western blots \n",
      "from brain tissue by the UC, Davis/NeuroMab Facility and also in our laboratory in BLA tissue …\n",
      "\n",
      "… (clone K75/41; University of California Davis-National Institute of Neurological Disorders \n",
      "and Stroke-National Institute of Mental Health Neuromab Facility, Davis, CA). A similar …\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "miR-17-92 is a microRNA cluster with six distinct members. Here, we show that the miR-17-92 \n",
      "cluster and its individual members modulate chronic neuropathic pain. All cluster …\n",
      "\n",
      "Potassium channels regulate numerous aspects of neuronal excitability, and several voltage-gated \n",
      "K + channel subunits have been identified in pyramidal neurons of rat neocortex. …\n",
      "\n",
      "… -S2 loop, Q9NZV8; NeuroMab, UC Davis/NIH, Davis, CA, USA); monoclonal anti-Kv4.3 (K75/41; \n",
      "aa. 415–636 of rat Kv4.3, cytoplasmic C-terminus, Q62897; NeuroMab, UC Davis/NIH, …\n",
      "\n",
      "In cerebellar Purkinje cell dendrites, heterosynaptic calcium signaling induced by the \n",
      "proximal climbing fiber (CF) input controls plasticity at distal parallel fiber (PF) synapses. The …\n",
      "\n",
      "Voltage-gated potassium (Kv) channels comprise four transmembrane α subunits, often \n",
      "associated with cytoplasmic β subunits that impact channel expression and function. Here, we …\n",
      "\n",
      "… K75/41 Neuromab; www.neuromab.org were used for immunolocalization experiments in \n",
      "brain. Immunoperoxidase [DPP6 antibodies diluted (1:1000); Kv4.2 (1:2000)] and …\n",
      "\n",
      "… We used the following primary antibodies from NeuroMab (UC Davis/NIH, Davis, CA, \n",
      "USA): (1) monoclonal anti-Kv4.2 (K57/1; amino acid. 209–225 of human Kv4.2, Q9NZV8), (2) …\n",
      "\n",
      "… Here, identical patterns were found in the spinal cord and DRG sections immunostained \n",
      "with NeuroMab anti-Kv4.3 and Alomone Labs anti-Kv4.3 (the AN01–04 but not AN05–07 …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Dopaminergic neurons modulate neural circuits and behaviors via dopamine (DA) release \n",
      "from expansive, long range axonal projections. The elaborate cytoarchitecture of these …\n",
      "\n",
      "Mice in which netrin‐G2 has been genetically inhibited do not startle to an acoustic stimulus, \n",
      "but otherwise perform normally through a behavioral test battery. Light microscopic …\n",
      "\n",
      "… Antibody 1 was a mouse monoclonal antibody (Clone L117/1) raised against a peptide \n",
      "corresponding to AAs 1–250 of human IRSp53 from Neuromab (1:4 for Western, undiluted for …\n",
      "\n",
      "Postsynaptic density (PSD) fractions were isolated from rat forebrain and sonicated. Pellets \n",
      "from sonicated samples examined by electron microscopy revealed particles with an electron …\n",
      "\n",
      "… were all based on mAbs from NeuroMab (UC Davis/NIH NeuroMab Facility, nd). This facility \n",
      "… 2013), our current efforts to systematically convert NeuroMab mAbs into scFvs that retain the …\n",
      "\n",
      "Objective Leucine‐rich glioma‐inactivated 1 (LGI1) encephalitis is the second most common \n",
      "antibody‐mediated encephalopathy, but insight into the intrathecal B‐cell autoimmune …\n",
      "\n",
      "Inhibitory neurons control the firing of glutamatergic neurons and synchronize brain activity. \n",
      "However, little is known about mechanisms of excitatory synapse formation in inhibitory …\n",
      "\n",
      "Lafora disease (LD) is an autosomal recessive form of a fatal disorder characterized by the \n",
      "myoclonus epilepsy, ataxia, psychosis, dementia, and dysarthria. A hallmark of LD is the …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Arsenic, a metalloid with cytotoxic and carcinogenic effects related to the disruption of \n",
      "glutathione homeostasis, induces the expression of ATF4, a central transcription factor in the …\n",
      "\n",
      "… Anti-GABA-A α1 receptor N95/35: yields a single band of expected size (52 kDa) on \n",
      "immunoblots of rat and mouse brain samples, a band not seen in a similar sample prepared from …\n",
      "\n",
      "The GABA A receptor (GABA A R) α1 subunit mutation, A322D, causes autosomal dominant \n",
      "juvenile myoclonic epilepsy (JME). Previous in vitro studies demonstrated that A322D elicits …\n",
      "\n",
      "Genetic epilepsy is a common disorder with phenotypic variation, but the basis for the variation \n",
      "is unknown. Comparing the molecular pathophysiology of mutations in the same epilepsy …\n",
      "\n",
      "Patients that suffer mild traumatic brain injuries (mTBI) often develop cognitive impairments, \n",
      "including memory and learning deficits. The hippocampus shows a high susceptibility to …\n",
      "\n",
      "Latrophilin-2 (Lphn2) and latrophilin-3 (Lphn3) are adhesion GPCRs that serve as postsynaptic \n",
      "recognition molecules in CA1 pyramidal neurons of the hippocampus, where they are …\n",
      "\n",
      "… A R alpha-1 (clone N95/35, 1–500, UC Davis/NIH NeuroMab Facility); a purified mouse … \n",
      "(#75-240, 1–10,000, UC Davis/NIH NeuroMab Facility); a purified rabbit polyclonal antibody …\n",
      "\n",
      "Tuberous sclerosis complex (TSC) is a neurogenetic disorder associated with epilepsy, \n",
      "intellectual disabilities, and autistic behaviors. These neurological symptoms result from synaptic …\n",
      "\n",
      "Brain radiotherapy is frequently used successfully to treat brain tumors. However, radiotherapy \n",
      "is often associated with declines in short-term and long-term memory, learning ability, …\n",
      "\n",
      "Mutations in MECP2 gene have been identified in more than 95% of patients with classic \n",
      "Rett syndrome, one of the most common neurodevelopmental disorders in females. Taking …\n",
      "\n",
      "Neuronal circuitries in the hippocampus are involved in navigation and memory and are \n",
      "controlled by major networks of GABAergic interneurons. Parvalbumin (PV)-expressing …\n",
      "\n",
      "… Mouse monoclonal anti α1(N95/35) (1:1,000) was from the University of California Davis/National \n",
      "Institutes of Health NeuroMab Facility. Rabbit anti-cleaved caspase 3 (9664S, 1:300) …\n",
      "\n",
      "The pentameric γ-Aminobutyric acid type A receptors (GABA A Rs) are ligand-gated ion \n",
      "channels that mediate the majority of inhibitory neurotransmission in the brain. In the cerebellum…\n",
      "\n",
      "Brain metastases are devastating complications of cancer. The blood–brain barrier (BBB), \n",
      "which protects the normal brain, morphs into an inadequately characterized blood–tumor …\n",
      "\n",
      "Coordinated development of excitatory and inhibitory synapses is essential for higher brain \n",
      "function, and impairment in this development is associated with neuropsychiatric disorders. …\n",
      "\n",
      "… The mouse monoclonal anti-α1 subunit antibody N95/35 was from the University of California, \n",
      "Davis/National Institutes of Health NeuroMab Facility. The mouse monoclonal anti-human …\n",
      "\n",
      "Although considerable evidence suggests that the chemical synapse is a lynchpin underlying \n",
      "affective disorders, how molecular insults differentially affect specific synaptic connections …\n",
      "\n",
      "The strength of synaptic inhibition can be controlled by the stability and endocytosis of \n",
      "surface and synaptic GABA A receptors (GABA A Rs), but the surface receptor dynamics that …\n",
      "\n",
      "… Mouse monoclonal antibodies were developed by UC Davis/NIH NeuroMab Facility in mouse \n",
      "generating hybridomas. The anti-GABAAR α1 mouse monoclonal antibody (clone N95/35 …\n",
      "\n",
      "Ionotropic neurotransmitter receptors mediate fast synaptic transmission by functioning as \n",
      "ligand-gated ion channels. Fast inhibitory transmission in the brain is mediated mostly by …\n",
      "\n",
      "Establishment, specification, and validation of synaptic connections are thought to be mediated \n",
      "by interactions between pre- and postsynaptic cell-adhesion molecules. Arguably, the …\n",
      "\n",
      "We have found that the large intracellular loop of the γ2 GABA A receptor (R) subunit (γ2IL) \n",
      "interacts with RNF34 (an E3 ubiquitin ligase), as shown by yeast two-hybrid and in vitro …\n",
      "\n",
      "Gamma aminobutyric acid (GABA) subtype A receptors (GABA A Rs) are integral membrane \n",
      "ion channels composed of five individual proteins or subunits. Up to 19 different GABA A R …\n",
      "\n",
      "… Neuromab … Primary antibodies utilized were anti-GABAAa1 (clone N95/35 from \n",
      "NeuroMab at 1:5000) and anti-Actin (mouse monoclonal 1:5000 Chemicon). …\n",
      "\n",
      "Impaired inhibitory signaling underlies the pathophysiology of many neuropsychiatric and \n",
      "neurodevelopmental disorders including autism spectrum disorders and epilepsy. Neuronal …\n",
      "\n",
      "Cadherins contribute to the organization of nearly all tissues, but the functions of several \n",
      "evolutionarily conserved cadherins, including those of calsyntenins, remain enigmatic. …\n",
      "\n",
      "Our preliminary studies demonstrate that Fmr1 KO mice at p21 have a decrease in phosphorylation \n",
      "of S443in the 4 subunit compared to WT and an increase in the phosphorylation of 3 …\n",
      "\n",
      "LRRTM4 is a transsynaptic adhesion protein regulating glutamatergic synapse assembly on \n",
      "dendrites of central neurons. In the mouse retina, we find that LRRTM4 is enriched at …\n",
      "\n",
      "Autism spectrum disorder (ASD) refers to a large set of neurodevelopmental disorders, \n",
      "which have in common both repetitive behavior and abnormalities in social interactions and …\n",
      "\n",
      "… ), which were then immunostained with anti-KCNQ2 (N26A/23, 0.22 μg ml −1 ) antibody. … \n",
      "3 μg of anti-Kv7.2 antibody (NeuroMab) added for immunoprecipitation. Extracts were used to …\n",
      "\n",
      "The ion channels responsible for the pattern and frequency of discharge in arterial baroreceptor \n",
      "terminals are, with few exceptions, unknown. In this study we examined the contribution …\n",
      "\n",
      "K v 7 channels are enriched at the axonal plasma membrane where their voltage-dependent \n",
      "potassium currents suppress neuronal excitability. Mutations in K v 7.2 and K v 7.3 subunits …\n",
      "\n",
      "The chemosensory glomus cells of the carotid body (CB) detect changes in O 2 tension. \n",
      "Carotid sinus nerve fibers, which originate from peripheral sensory neurons located within the …\n",
      "\n",
      "… using the antibodies mainly from Neuromab for IP from extracts … Recently, the Neuromab \n",
      "facility started to provide a unique … use of antibodies mainly from Neuromab to start to examine …\n",
      "\n",
      "… Primary antibodies used were Neuromab monoclonal antibodies against KCNQ1 (N37/A10, \n",
      "0.6 μg/ml), KCNQ2 (N26A/23, 0.6 μg/ml), and KCNQ4 (N43/6, 0.6 μg/ml) and a non-…\n",
      "\n",
      "Mutations in the KCNQ2 gene, encoding for voltage-gated Kv7.2 K + channel subunits, are \n",
      "responsible for early-onset epileptic diseases with widely-diverging phenotypic presentation, …\n",
      "\n",
      "Neuronal K v 7/KCNQ channels are voltage-gated potassium channels composed of K v 7.2/KCNQ2 \n",
      "and K v 7.3/KCNQ3 subunits. Enriched at the axonal membrane, they potently …\n",
      "\n",
      "… KCNQ2 protein was detected with a mouse monoclonal antibody (N26A/23, NeuroMAb) \n",
      "and HRP-conjugated rabbit anti-mouse secondary antibody (ABM) and visualized using ECL …\n",
      "\n",
      "… Antibodies used in immunocytochemistry include mouse anti-ankyrin G (Neuromab, 75–146), \n",
      "rabbit anti-phospho IκBα Ser32 (14D4) (Cell Signaling, 2859), rabbit anti-MAP2 antibody (…\n",
      "\n",
      "Neuronal ankyrins cluster and link membrane proteins to the actin and spectrin-based \n",
      "cytoskeleton. Among the three vertebrate ankyrins, little is known about neuronal Ankyrin-R (AnkR…\n",
      "\n",
      "Background The neuromuscular junction (NMJ) is a peripheral synapse critical to muscle \n",
      "contraction. Like acetylcholine receptors (AChRs), many essential proteins of NMJ are …\n",
      "\n",
      "… (clone N170A/26, NeuroMab, CA) and anti-actin (sc-1615, Santa Cruz, CA) overnight in \n",
      "blocking buffer at 4C, and with secondary antibodies (goat anti-mouse conjugate to horseradish …\n",
      "\n",
      "… 1e mAb (K36/15; UC Davis/NIH NeuroMab Facility) before detergent permeabilization to … \n",
      "(anti-Kv1.1), all from the UC Davis/NIH NeuroMab Facility, or K25/73 (anti-Kvβ2) after detergent …\n",
      "\n",
      "… Two monoclonal antibodies raised against the diethoxyphosphonyl-serine 3 (N257/25) \n",
      "and … by NeuroMab. Antibodies N257/25 and N262/16 are available (neuromab.ucdavis.edu). …\n",
      "\n",
      "… We provide examples from NeuroMab screens, and from the subsequent specialized … of \n",
      "NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs at the …\n",
      "\n",
      "Synaptic plasticity requires remodeling of the actin cytoskeleton. Although two actin isoforms, \n",
      "β- and γ-actin, are expressed in dendritic spines, the specific contribution of γ-actin in the …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "… Dr-Vsp protein expression was examined using an anti-Dr-Vsp antibody (NeuroMab clone \n",
      "N432/21). The immunostainings detected abundant expression of Dr-Vsp protein in the …\n",
      "\n",
      "… Other lanes include antiserum from one of the immunized mice (HB), a mouse serum \n",
      "negative control (NMS) and a positive control NeuroMab mAb against a different target (L86A/37, …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "… Similar cone staining was observed with another NCKX4 antibody (N414/25, NeuroMab, \n",
      "UC Davis/NIH NeuroMab Facility) recently shown to specifically recognize NCKX4 (Bronckers …\n",
      "\n",
      "… The specificity of the used antibody (Neuromab -clone N97A/31) for NLGN1 was verified \n",
      "on brains of NLGN1 and NLGN2 -null mice (Supplementary Fig. S1). The analysis on TMA (Fig. …\n",
      "\n",
      "Alzheimer's disease (AD) is characterized by memory impairments and age-dependent \n",
      "synapse loss. Experimental and clinical studies have shown decreased expression of the …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "Retromer is an evolutionarily conserved endosomal trafficking complex that mediates the \n",
      "retrieval of cargo proteins from a degradative pathway for sorting back to the cell surface. To …\n",
      "\n",
      "In vitro studies suggest that the intracellular C terminus of Neuroligin1 (NL1) could play a \n",
      "central role in the maturation of excitatory synapses. However, it is unknown how this activity …\n",
      "\n",
      "Autism spectrum disorder (ASD) is a neurological condition characterized by difficulties in \n",
      "social 22 interaction, communication, and behavior. The classical type II cadherins cadherin-8 (…\n",
      "\n",
      "The four members of the LRRTM family (LRRTM1-4) are postsynaptic adhesion molecules \n",
      "essential for excitatory synapse development. They have also been implicated in …\n",
      "\n",
      "Coordinated development of excitatory and inhibitory synapses is essential for higher brain \n",
      "function, and impairment in this development is associated with neuropsychiatric disorders. …\n",
      "\n",
      "Classical cadherins, a set of ~20 related recognition and signaling molecules, have been \n",
      "implicated in many aspects of neural development, including the formation and remodeling of …\n",
      "\n",
      "… , MN), mouse monoclonal anti-neuroligin-1 (clone# N97A/31, NeuroMab, Davis, CA) and \n",
      "neurexin-1β (clone# N170A-1, NeuroMab, Davis, CA). Alexa Fluor 555- and DyLight 488 …\n",
      "\n",
      "… N97A/31; NeuroMab); rabbit polyclonal anti-NL-2 (Synaptic Systems); guinea pig polyclonal \n",
      "anti-VGLUT1 (Millipore); mouse monoclonal anti-GAD67 (clone 1G10.2; Millipore); mouse …\n",
      "\n",
      "Excitatory glutamatergic synapses facilitate important aspects of communication between \n",
      "the neurons that govern complex forms of behavior. Accordingly, small differences in the …\n",
      "\n",
      "Calsyntenin-3 (Clstn3) is a postsynaptic adhesion molecule that induces presynaptic \n",
      "differentiation via presynaptic neurexins (Nrxns), but whether Nrxns directly bind to Clstn3 has …\n",
      "\n",
      "Excitatory synapses are formed and matured by the cooperative actions of synaptic organizers, \n",
      "such as neurexins (Nrxns), neuroligins (Nlgns), LRRTMs, and Cbln1. Recent super-…\n",
      "\n",
      "Synaptogenesis is a critical process in the initial formation of the central nervous system (CNS) \n",
      "and in the ability of the CNS to continually reshape itself in response to stimuli throughout …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… The commercial antibody used was from NeuroMab, Davis, CA, USA, GABA A α1 antibody \n",
      "(N95/35, AB_106697873) and GABA A α6 antibody (N229A/32, AB_2491091, labeled as …\n",
      "\n",
      "… a mouse monoclonal antibody (NeuroMab N138/6) that is also capable of immunoprecipitating \n",
      "endogenous LRRK2, Fig. 3a. We have found that use of NeuroMab N138/6 antibody in …\n",
      "\n",
      "… With the exception of N138/6, all of the antibodies precipitated both active human and … For \n",
      "the mouse monoclonal antibodies (Supplementary Figures S2G–S2J), Neuromab N138/6 and …\n",
      "\n",
      "… Anti-LRRK2 IP confirmed by western blotting with a second anti-LRRK2 antibody (NeuroMab \n",
      "N138/6). LRRK2 and co-complexed proteins are present in MJFF2 eluates and cell lysate, …\n",
      "\n",
      "… commercial mouse monoclonal LRRK2 antibody (N138/6) that also recognizes both mouse \n",
      "and human LRRK2 proteins. The specificity of the N138/6 was tested using LRRK2 knockout …\n",
      "\n",
      "The G2019S mutation in leucine-rich repeat kinase 2 (LRRK2) causes familial Parkinson’s \n",
      "disease (PD) and is also found in a subset of idiopathic cases. Prior studies in Drosophila …\n",
      "\n",
      "A common genetic form of Parkinson's disease (PD) is caused by mutations in LRRK2. We \n",
      "identify WSB1 as a LRRK2 interacting protein. WSB1 ubiquitinates LRRK2 through K27 and …\n",
      "\n",
      "Mutations in the leucine-rich repeat kinase-2 (LRRK2) gene cause late-onset Parkinson’s \n",
      "disease, but its physiological function has remained largely unknown. Here we report that …\n",
      "\n",
      "… C, Whole-brain lysates prepared from WT and LRRK2 KO mice were subjected to \n",
      "coimmunoprecipitation with anti-LRRK2 (Neuromab), followed by anti-LRRK2 and anti-ArfGAP1 …\n",
      "\n",
      "… N138/6 (Neuromab) or flag-M2 antibody followed by incubation with appropriate secondary \n",
      "horseradish peroxidase coupled antibodies and chemiluminescent detection using ECL …\n",
      "\n",
      "Background LRRK2 mutations and risk variants increase susceptibility to inherited and \n",
      "idiopathic Parkinson’s disease, while recent studies have identified potential protective variants. …\n",
      "\n",
      "… Mouse monoclonal antibody to LRRK2 (clone N138/6) was from the UC Davis/NIH \n",
      "NeuroMab Facility. Rabbit monoclonal antibody to clathrin heavy chain (4796) and mouse …\n",
      "\n",
      "… The following antibodies were used: anti-LRRK2 [MJFF2 (c41–2)] (Abcam), anti-LRRK2 [N138/6] \n",
      "(NeuroMab, … Anti-LRRK2 antibody [N138/6] and anti-mouse IgG antibody were used for …\n",
      "\n",
      "Mutations and variations in the leucine-rich repeat kinase 2 (LRRK2) gene are strongly \n",
      "associated with an increased risk to develop Parkinson’s disease (PD). Most pathogenic LRRK2 …\n",
      "\n",
      "… , with two total LRRK2 antibodies showing weak detection (N138/6 and MC.028.83.76.242), … \n",
      "anti-LRRK2 antibodies (RabMab UDD3 and Neuromab N241, revealed by large and small …\n",
      "\n",
      "Leucine Rich Repeat Kinase 2 (LRRK2) is one of the most important genetic contributors to \n",
      "Parkinson's disease. LRRK2 has been implicated in a number of cellular processes, …\n",
      "\n",
      "Mutations in LRRK2 are the most common genetic causes of Parkinson's disease (PD). While \n",
      "the enzymatic activity of LRRK2 has been linked to PD, previous work has also provided …\n",
      "\n",
      "43 Leucine Rich Repeat Kinase 2 (LRRK2) is a multidomain protein of 44 unknown function \n",
      "containing two enzymatic domains, a GTPase (Ras 45 of Complex Proteins, ROC) and a …\n",
      "\n",
      "By studying healthy women who do not request analgesia during their first delivery, we \n",
      "investigate genetic effects on labor pain. Such women have normal sensory and psychometric …\n",
      "\n",
      "… for this study included mouse monoclonals anti-GluR2 (NeuroMab); NR1, NR2B, PSD-95, … \n",
      "neurons was examined using monoclonal N135/37 antibody generated against a mouse IB2 …\n",
      "\n",
      "Phelan–McDermid syndrome (also known as 22q13.3 deletion syndrome) is a syndromic \n",
      "form of autism spectrum disorder and currently thought to be caused by heterozygous loss of …\n",
      "\n",
      "… Labs, ACC-003; NM: Neuromab Antibodies Inc. L57/46,) or commercially available anti-Ca \n",
      "v 1.3 (Alo: Alomone Labs, ACC-005; NM: Neuromab Antibodies Inc. N38/8) antibodies. Blots …\n",
      "\n",
      "Aims On the basis of its ability to inhibit fibrosis, pirfenidone has drawn the attention as an \n",
      "intriguing candidate for treating cardiac disease. However, its precise electrophysiological …\n",
      "\n",
      "… (L57/46; 1:50; NeuroMab), rabbit polyclonal anti-Ca V 1.2 channel (ACC-003; 1:50; Alomone \n",
      "Labs), and rabbit polyclonal anti-Ca V 1.3 channel (ACC-005; 1:50; Alomone Labs). The …\n",
      "\n",
      "… Specificity of the Ca V 1 subtype antibodies was determined by the supplier (NeuroMab) \n",
      "by immunohistochemistry in brain tissue and by western blot. They have been used (http://…\n",
      "\n",
      "… Cav1.2 was detected using a mouse monoclonal Cav1.2 antibody (1:1000 dilution, clone \n",
      "L57/46; NeuroMab) and HRP-conjugated goat antimouse antibody (1:20,000 dilution, SH023; …\n",
      "\n",
      "Ca 2+ entry through L-type calcium channels (Ca V 1.2) is critical in shaping the cardiac action \n",
      "potential and initiating cardiac contraction. Modulation of Ca V 1.2 channel gating directly …\n",
      "\n",
      "The maintenance of chronic pain states requires the regulation of gene expression, which \n",
      "relies on an influx of calcium. Calcium influx through neuronal L-type voltage-gated calcium …\n",
      "\n",
      "A recent study identified Nipsnap1 as an auxiliary protein inhibiting TRPV6 ion channel \n",
      "activity. Based upon this finding, we investigated the role of Nipsnap1, and the closely related …\n",
      "\n",
      "… For mouse LTCC western blots an Anti-Cav1.2 calcium channel, NeuroMab clone L57/46 \n",
      "antibody was used (1:200). For rat heart material, we used the Anti-CACNA 1 C Antibody from …\n",
      "\n",
      "Background— Recently, we reported that sarcoplasmic reticulum Ca 2+ ATPase 2a (SERCA2a), \n",
      "the pump responsible for reuptake of cytosolic calcium during diastole, plays a central …\n",
      "\n",
      "… antibody (clone L57/46) used in the present study was developed by and/or obtained from \n",
      "the University of California Davis/NINDS/NIMH NeuroMab Facility and maintained by the …\n",
      "\n",
      "… (L57/46; 1:50; NeuroMab), rabbit polyclonal anti-Ca V 1.2 channel (ACC-003; 1:50; Alomone \n",
      "Labs), and rabbit polyclonal anti-Ca V 1.3 channel (ACC-005; 1:50; Alomone Labs). The …\n",
      "\n",
      "… Mouse monoclonal antibody (Ca V 1.2m, clone L57/46) against the proximal C terminus \n",
      "of Ca V 1.2 channels was purchased from UC Davis/NIH NeuroMab Facility. Polyclonal anti-Ca …\n",
      "\n",
      "Aims Atrial fibrillation (AF) produces rapid changes in the electrical properties of the atria (electrical \n",
      "remodelling) that promote its own recurrence. In chronic AF (CAF) patients, up-…\n",
      "\n",
      "Long QT syndrome (LQTS) exhibits great phenotype variability among family members \n",
      "carrying the same mutation, which can be partially attributed to genetic factors. We functionally …\n",
      "\n",
      "… S3 shows immunocytochemistry results of sensory neurons from conditional Ca V 1.2 \n",
      "knockout mice and after viral knockdown of Ca V 1.2 using a pan-Ca V 1 antibody (clone L57/46). …\n",
      "\n",
      "Smooth muscle of the gastrointestinal tract (visceral smooth muscle) propels and mixes food \n",
      "to facilitate digestion and absorption of nutrients. Similar to skeletal muscle, the contraction …\n",
      "\n",
      "… (clone L57/46, #73-053, lot 437-4VA-62, 1∶4 dilution) or mouse anti-α1D (clone N38/8, #73-080, \n",
      "lot 437-4VA-10, 1∶4 dilution) obtained from the UC Davis/NIH NeuroMab Facility – …\n",
      "\n",
      "In neurons, L-type voltage-gated Ca2+ channels (LTCC) couple electrical activity to gene \n",
      "expression that is necessary for long lasting changes in synaptic strength. A-kinase anchoring …\n",
      "\n",
      "… DIV) were fixed and stained with monoclonal antibodies (NeuroMAB) against Kv2.1, Cav1.3, … \n",
      "(L57/46). Anti Kv2.1 has been validated using tissue from transgenic mouse lines in which …\n",
      "\n",
      "… (Clone N55/10) was developed by and/or obtained from the UC Davis/NIH NeuroMab \n",
      "Facility (Department of Neurobiology, Physiology and Behavior, College of Biological Sciences, …\n",
      "\n",
      "Acetylcholine critically influences hippocampal-dependent learning. Cholinergic fibers \n",
      "innervate hippocampal neuron axons, dendrites, and somata. The effects of acetylcholine on …\n",
      "\n",
      "Epithelial tumor cells express T-type Ca 2+ channels, which are thought to promote cell \n",
      "proliferation. This study investigated the cellular response to T-type Ca 2+ channel inhibition …\n",
      "\n",
      "… ) and Ca V 3.2 (Clone N55/10) were developed by and obtained from the University of \n",
      "California at Davis/US National Institutes of Health NeuroMab Facility, supported by US National …\n",
      "\n",
      "Parvalbumin interneurons (PVIs) are affected in many psychiatric disorders including \n",
      "schizophrenia (SCZ), however the mechanism remains unclear. FXR1, a high confident risk gene …\n",
      "\n",
      "… (N55/10) antibodies were from the University of California, Davis/National Institutes of Health \n",
      "NeuroMab facility. Polyclonal Gephyrin (147 002 for Western blotting), GABA A receptor α2 (…\n",
      "\n",
      "Regulation of inwardly rectifying potassium channels by intracellular ligands couples cell \n",
      "membrane excitability to important signaling cascades and metabolic pathways. We …\n",
      "\n",
      "… Total proteins were loaded on a 7.5% SDS gel and transferred to nitrocellulose membrane \n",
      "and blotted with anti-SUR1 antibody (NeuroMAb, clone N289/16), goat anti-mouse secondary …\n",
      "\n",
      "The hERG potassium channel is critical for repolarisation of the cardiac action potential. \n",
      "Reduced expression of hERG at the plasma membrane, whether caused by hereditary …\n",
      "\n",
      "… ; BD), monoclonal mouse AnkG antibody (clone 65; Neuromab), and monoclonal \n",
      "neurofascin antibody (clone A12/18; Neuromab) conjugated in-house to Cy3 by N-hydroxysuccinimidyl …\n",
      "\n",
      "… Axon initial segment was stained for 5 min at RT with anti–pan-neurofascin antibody (UC \n",
      "Davis/NIH NeuroMab Facility, clone A12/18) diluted 1:100 in artificial cerebrospinal fluid (ACSF …\n",
      "\n",
      "… To localize the AIS we detached the Aclar from the well plate and incubated the neurons \n",
      "upside-down with anti-NF186 (1:200; NeuroMab, A12/18) in 60 μl of its own growth medium for …\n",
      "\n",
      "… To identify axons for subcellular stimulation, we stained neurons using an anti–pan-neurofascin \n",
      "primary antibody (clone A12/18; NeuroMab) that labels the axon initial segment, …\n",
      "\n",
      "Superresolution fluorescence microscopy of multiple fluorophores still requires development. \n",
      "Here we present simultaneous three-colour stimulated emission depletion (STED) …\n",
      "\n",
      "… The mouse anti-pan-neurofascin antibody (extNF; A12/18, Neuromab) was generated \n",
      "against the rat-specific extracellular domain common to NF155 and NF186 (amino acids 25-1110)…\n",
      "\n",
      "Although actin at neuronal growth cones is well-studied, much less is known about actin \n",
      "organization and dynamics along axon shafts and presynaptic boutons. Using probes that …\n",
      "\n",
      "… (NeuroMab, clone number A12/18, 1:100) in conditioned Neurobasal medium for 10 min \n",
      "at 37 C. After this, neurons were washed three times in warm Neurobasal medium, and anti-…\n",
      "\n",
      "Cerebrospinal fluid samples collected from children during initial presentation of central \n",
      "nervous system inflammation, who may or may not subsequently be diagnosed as having …\n",
      "\n",
      "… To identify axons, neurons were live labeled with an Alexa Fluor 594-labeled neurofascin \n",
      "antibody (clone A12/18, 1:200; NeuroMab) for 45 min at 37C. For antibody labeling, the Tris-…\n",
      "\n",
      "… As previously described [21], mouse monoclonal antibodies (A12/18; NeuroMab) that \n",
      "recognize an extracellular epitope of neurofascin were used for live labeling of the AIS. …\n",
      "\n",
      "… The AIS was stained with anti-pan-neurofascin primary antibody (UC Davis/NIH NeuroMab \n",
      "Facilitym clone A12/18) and AlexaFluor488 secondary antibody (ab150113) following the …\n",
      "\n",
      "In vertebrates, fast saltatory conduction along myelinated axons relies on the node of \n",
      "Ranvier. How nodes assemble on CNS neurons is not yet fully understood. We previously …\n",
      "\n",
      "… ), and anti-pan-Neurofascin mouse IgG1 (L11A/41, 1:1 dilution; Neuromab). The secondary \n",
      "antibodies included the following: Alexa Fluor 568-conjugated goat anti-mouse IgG2b or …\n",
      "\n",
      "… The mouse anti-pan-neurofascin antibody (extNF; A12/18, NeuroMab, USA) was generated \n",
      "against the rat-specific extracellular domain common to NF155 and NF-186 (amino acids 25…\n",
      "\n",
      "SLITRK1 is an obsessive-compulsive disorder spectrum-disorders-associated gene that \n",
      "encodes a neuronal transmembrane protein. Here we show that SLITRK1 suppresses …\n",
      "\n",
      "… For extracellular pan-neurofascin label, we used a live incubation of primary antibody (clone \n",
      "A12/18, 1:500; catalog #75-172; NeuroMab) in Neurobasal media at 37C for 30 min before …\n",
      "\n",
      "… For live AIS imaging prior to all other electrophysiological recordings, we utilized the \n",
      "mouse-anti-pan-neurofascin antibody (NF-ext, A12/18, Neuromab). Pre-labeling, neurons were …\n",
      "\n",
      "Neurons are highly polarized cells having distinct somatodendritic and axonal domains. \n",
      "Here we report that polarized sorting of the Cu 2+ transporter ATP7B and the vesicle-SNARE …\n",
      "\n",
      "Actin, spectrin, and associated molecules form a membrane-associated periodic skeleton (MPS) \n",
      "in neurons. The molecular composition and functions of the MPS remain incompletely …\n",
      "\n",
      "Core planar cell polarity (PCP) genes, which are involved in various neurodevelopmental \n",
      "disorders such as neural tube closure, epilepsy, and autism spectrum disorder, have poorly …\n",
      "\n",
      "… For live-cell imaging of proteasome transport across the AIS region, we preincubated \n",
      "neurons with pan-neurofascin antibody (A12/18) for 30 min to label the neurofascin extracellular …\n",
      "\n",
      "Axon formation, the initial step in establishing neuronal polarity, critically depends on local \n",
      "microtubule reorganization and is characterized by the formation of parallel microtubule …\n",
      "\n",
      "Establishment of neuronal polarity depends on local microtubule (MT) reorganization. The \n",
      "endoplasmic reticulum (ER) consists of cisternae and tubules and, like MTs, forms an …\n",
      "\n",
      "The study of polarized protein trafficking in live neurons is critical for understanding neuronal \n",
      "structure and function. Given the complex anatomy of neurons and the numerous trafficking …\n",
      "\n",
      "Soluble cytosolic proteins vital to axonal and presynaptic function are synthesized in the \n",
      "neuronal soma and conveyed via slow axonal transport. Our previous studies suggest that the …\n",
      "\n",
      "… Mouse anti–pan-neurofascin A12/18 was obtained from the University of California, Davis/NIH \n",
      "NeuroMab facility and was conjugated to Mix-n-Stain CF640 (Biotium). Rabbit antisera to …\n",
      "\n",
      "The DISC1 protein is implicated in major mental illnesses including schizophrenia, depression, \n",
      "bipolar disorder, and autism. Aberrant mitochondrial dynamics are also associated with …\n",
      "\n",
      "Neurotransmitter-filled synaptic vesicles (SVs) mediate synaptic transmission and are a \n",
      "hallmark specialization in neuronal axons. Yet, how SV proteins are sorted to presynaptic nerve …\n",
      "\n",
      "… , mouse clone A12/18, and anti-Pan-neurofascin against an intracellular epitope, mouse \n",
      "clone L11A/41, were obtained from University of California, Davis, CA (NIH NeuroMab facility). …\n",
      "\n",
      "… At DIV11, live primary cultured neurons were treated with an antibody to extNF [anti-pan-neurofascin \n",
      "(external), NeuroMab clone A12/18] as previously described (63) to label the AIS. …\n",
      "\n",
      "Compact myelin, the paranode, and the juxtaparanode are discrete domains that are formed \n",
      "on myelinated axons. In humans, neurological disorders associated with loss of myelin, …\n",
      "\n",
      "The axon initial segment (AIS) is the site of action potential initiation and serves as a cargo \n",
      "transport filter and diffusion barrier that helps maintain neuronal polarity. The AIS actin …\n",
      "\n",
      "… together with 480AnkG-GFP or 480AnkG-NN-GFP and extracellularly labeled with antiNF186 \n",
      "(1: 200; NeuroMab, A12/18). Cells were washed in culture medium and fixed with 2% …\n",
      "\n",
      "The small GTPase ARL8 associates with endolysosomes, leading to the recruitment of \n",
      "several effectors that couple endolysosomes to kinesins for anterograde transport along …\n",
      "\n",
      "The axon initial segment (AIS), located within the first 30 μm of the axon, has two essential \n",
      "roles in generating action potentials and maintaining axonal identity. AIS assembly depends …\n",
      "\n",
      "… Other primary antibodies were: mouse anti-neurofascin clone A12/18, NeuroMab (RRID:AB_10671311), \n",
      "1:200; rabbit anti-Rab11 71-5300, Thermo, 1:50; rabbit anti-ARF6 ab77581, …\n",
      "\n",
      "The axon initial segment (AIS) is a unique neuronal compartment that plays a crucial role in \n",
      "the generation of action potential and neuronal polarity. The assembly of the AIS requires …\n",
      "\n",
      "… ), anti-Neurofascin (1:500; A12/18, AB_2282826), and anti-GST (1:2000; N100/13, \n",
      "AB_10671818) antibodies were purchased from the UC Davis/NIH NeuroMab Facility (Davis, CA). …\n",
      "\n",
      "… (A12/18; AB_2282826) antibodies were purchased from the University of California, Davis/NIH \n",
      "NeuroMab … Next, mouse monoclonal NF186 primary antibody (A12/18), which specifically …\n",
      "\n",
      "The axon initial segment (AIS) is the site of action potential generation and a locus of activity-dependent \n",
      "homeostatic plasticity. A multimeric complex of sodium channels, linked via a …\n",
      "\n",
      "Polarized sorting of newly synthesized proteins to the somatodendritic and axonal domains \n",
      "of neurons occurs by selective incorporation into distinct populations of vesicular transport …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "The identification of cancer-associated mutations in the tricarboxylic acid (TCA) cycle enzymes \n",
      "isocitrate dehydrogenases 1 and 2 (IDH1/2) highlights the prevailing notion that aberrant …\n",
      "\n",
      "Activating K-Ras mutations occurs frequently in pancreatic cancers and is implicated in their \n",
      "development. Cancer-initiating events, such as oncogenic Ras activation, lead to the …\n",
      "\n",
      "Chromatin-modifying enzymes play a fundamental role in regulating chromatin structure so \n",
      "that DNA replication is spatially and temporally coordinated. For example, the lysine …\n",
      "\n",
      "… (b) A sagittal section from rat brain immunolabeled for KChIP4 using NeuroMab N423/75 \n",
      "TCS … (c) IF labeling for KChIP4 using NeuroMab N423/75 TCS (magenta) in a sagittal section …\n",
      "\n",
      "… (1:4,000; NeuroMab), a concentration 10 times lower than that for single antigen \n",
      "immunohistochemistry. On the second day, with intervening washes with LTBS, the anti-Kv4.3 signal …\n",
      "\n",
      "… To purify the TRPM7 channel proteins, we used a monoclonal anti-TRPM7 antibody (N74/25, \n",
      "Neuromab, CA) that binds to the cytoplasmic C-terminus of the channel. The TRPM7 …\n",
      "\n",
      "The mixed lineage kinase domain-like protein (MLKL) has recently been identified as a key \n",
      "RIP3 (receptor interacting protein 3) downstream component of tumour necrosis factor (TNF)-…\n",
      "\n",
      "… Anti-TRPM7 C-terminus antibody was purchased from NeuroMab (N74/25, USA), and anti-β \n",
      "actin (8H10D10), -CREB (86B10), -pCREB (1B6; Ser133), -Myosin IIA (3403), -pMyosin IIA (…\n",
      "\n",
      "… TRPM7 (N74/25) was purchased from NeuroMab, and antibodies against Poly (ADP-ribose) \n",
      "polymerase (PARP; 9542), caspase-3 (9662), caspase-8 (9746), and c-FLIP (56343) were …\n",
      "\n",
      "The transient receptor potential melastatin-subfamily member 7 (TRPM7) is a ubiquitously \n",
      "expressed membrane protein consisting of ion channel and protein kinase domains. TRPM7 …\n",
      "\n",
      "The tetracycline repressor (TetR)-regulated system is a widely used tool to study gene \n",
      "functions through control of its expression. Various effectors such as tetracycline (Tc) and …\n",
      "\n",
      "… The primary antibodies used were: anti-AnkG (33–8800, Invitrogen), mouse anti-βIV-spectrin \n",
      "(N393/76, NeuroMab) and mouse anti-Na v 1.2 (K69/3, NeuroMab). …\n",
      "\n",
      "… , Abcam), mouse anti-βIV-spectrin (1:500, N393/76, NeuroMab) and mouse anti-Na v 1.2 (1:500, \n",
      "K69/3, NeuroMab) primary antibodies. Next, neurons were incubated for 60 min at room …\n",
      "\n",
      "In tauopathic conditions, such as Alzheimer's disease (AD), highly soluble and natively \n",
      "unfolded tau polymerizes into an insoluble filament; however, the mechanistic details of this …\n",
      "\n",
      "Neuronal GABAergic responses switch from excitatory to inhibitory at an early postnatal \n",
      "period in rodents. The timing of this switch is controlled by intracellular Cl− concentrations, but …\n",
      "\n",
      "Neuronal maturation requires dramatic morphological and functional changes, but the molecular \n",
      "mechanisms governing this process are not well understood. Here, we studied the role …\n",
      "\n",
      "Core planar cell polarity (PCP) genes, which are involved in various neurodevelopmental \n",
      "disorders such as neural tube closure, epilepsy, and autism spectrum disorder, have poorly …\n",
      "\n",
      "Astrocytes, once considered passive support cells, are increasingly appreciated as dynamic \n",
      "regulators of neuronal development and function, in part via secreted factors. The extent to …\n",
      "\n",
      "… Abs against Dlg1 (K64/15), pan-Kvβ potassium channel subunit (K40/17), Kv1.3 (L23/27), \n",
      "and Kv1.5 (K7/45) were purchased from Neuromab, and anti–β-actin Abs were purchased …\n",
      "\n",
      "Nav1.6 and Nav1.6-mediated resurgent currents have been implicated in several pain \n",
      "pathologies. However, our knowledge of how fast resurgent currents are modulated in neurons is …\n",
      "\n",
      "Shank and GKAP are scaffold proteins and binding partners at the postsynaptic density (PSD). \n",
      "The distribution and dynamics of Shank and GKAP were studied in dissociated …\n",
      "\n",
      "… (clone N23B/6, used at 1:200 for microscopy and 1:1000 for Western), and pan Shank (clone \n",
      "N23B/49, used at 1:200 for microscopy and 1:1000 for Western) were from NeuroMab, …\n",
      "\n",
      "The family of Shank scaffolding molecules (comprising Shank1, 2 and 3) are core components \n",
      "of the postsynaptic density ( PSD ) in neuronal synapses. Shanks link surface receptors to …\n",
      "\n",
      "… N23B/6 at 1:200) were from NeuroMab; mouse monoclonal antibody against NR1 (clone \n",
      "R1JHL at 1:100) was from Calbiochem; rabbit polyclonal antibody against NR2A/B (catalog #…\n",
      "\n",
      "(論文の内容の要旨)[背景と目的] シナプス後肥厚部 (postsynaptic density, PSD) とはシナプス膜\n",
      "直下にある, 神経細胞に高度に分化した特殊な細胞骨格系で, 極めて精巧なタンパク質複合体である…\n",
      "\n",
      "Mutations or altered protein levels of SHANK3 are implicated in neurodevelopmental disorders \n",
      "such as Phelan–McDermid syndrome, autism spectrum disorders, and schizophrenia ( …\n",
      "\n",
      "… N23B/6, 1:1000 for Western), and pan-Shank (clone N23B/49, which recognizes all three \n",
      "members of the Shank family: Shank1, 2 and 3, used at 1:250 for EM) were from NeuroMab (…\n",
      "\n",
      "… N23B/6, used at 1:200) were from NeuroMab (Davis, CA). Rabbit polyclonal antibody against \n",
      "Shank 3 (used at 1:200–800) was from Synaptic Systems (Goettingen, Germany). Rabbit …\n",
      "\n",
      "… Monoclonal anti–ProSAP1 (N23B/6) antibodies were from NeuroMab. Aliquots of rabbit \n",
      "and guinea pig anti-ProSAP1 antibodies were provided by T. Boeckers. …\n",
      "\n",
      "Compartmentalization of calcium-dependent plasticity allows for rapid actin remodeling in \n",
      "dendritic spines. However, molecular mechanisms for the spatio-temporal regulation of …\n",
      "\n",
      "During development, pyramidal neurons undergo dynamic regulation of AMPA receptor (AMPAR) \n",
      "subunit composition and density to help drive synaptic plasticity and maturation. …\n",
      "\n",
      "… (K7/45) and Mortalin/Grp75 (N52A/42) using monoclonal antibodies (clone number in \n",
      "parentheses) obtained from the UC Davis/NIH NeuroMab Facility. Alexa-conjugated fluorescent …\n",
      "\n",
      "The Kv2 channels encode delayed rectifier currents that regulate membrane potential in \n",
      "many tissues. They also have a non-conducting function to form stable junctions between the …\n",
      "\n",
      "The plasma membrane (PM) comprises distinct subcellular domains with diverse functions \n",
      "that need to be dynamically coordinated with intracellular events, one of the most impactful …\n",
      "\n",
      "… The mouse mAbs N52A/42, K13/31, K89/34, L83/11, L86/33, and L86A/37 are available \n",
      "from the UC Davis/NIH NeuroMab Facility (http://neuromab.ucdavis.edu/), a not-for profit …\n",
      "\n",
      "Latrophilin-2 (Lphn2) and latrophilin-3 (Lphn3) are adhesion GPCRs that serve as postsynaptic \n",
      "recognition molecules in CA1 pyramidal neurons of the hippocampus, where they are …\n",
      "\n",
      "… UC Davis/NIH NeuroMab Facility); an antibody raised against a fusion protein from the N-terminus \n",
      "of human gephyrin (anti-gephyrin clone L106/83, UC Davis/NIH NeuroMab Facility); a …\n",
      "\n",
      "… NeuroMab Facility); an antibody raised against a fusion protein from the N-terminus of human \n",
      "gephyrin (anti-gephyrin clone L106/83, UC Davis/NIH NeuroMab … -GFAP, NeuroMab clone …\n",
      "\n",
      "… NeuroMab Facility); an antibody raised against a fusion protein from the N-terminus of human \n",
      "gephyrin (anti-gephyrin clone L106/83, UC Davis/NIH NeuroMab … /NIH NeuroMab Facility); …\n",
      "\n",
      "… UC Davis/NIH NeuroMab Facility); an antibody raised against a fusion protein from the N-terminus \n",
      "of human gephyrin (anti-gephyrin clone L106/83, UC Davis/NIH NeuroMab Facility); a …\n",
      "\n",
      "… Monoclonal antibody anti-mSlo3 N2/16 was obtained from the University of California, \n",
      "Davis/NIH NeuroMab Facility, which is supported by NIH Grant U24NS050606 and maintained by …\n",
      "\n",
      "… The monoclonal antibody N2/16 was developed by or obtained from the University of \n",
      "California Davis/NIH NeuroMab Facility, supported by NIH Grant U24NS050606, and maintained …\n",
      "\n",
      "10.7554/eLife.49253.001 Inhibiting high-voltage-activated calcium channels (HVACCs; Ca V \n",
      "1/Ca V 2) is therapeutic for myriad cardiovascular and neurological diseases. For particular …\n",
      "\n",
      "… As a positive control, cells colabeled with two distinct Ca V 1.2 antibodies [rabbit FP1 (28) \n",
      "and monoclonal NeuroMab clone N263/31] showed surface membrane staining (fig. S1A) and …\n",
      "\n",
      "Lysosomes communicate through cholesterol transfer at endoplasmic reticulum (ER) contact \n",
      "sites. At these sites, the Niemann Pick C1 cholesterol transporter (NPC1) facilitates the …\n",
      "\n",
      "… 1); anti-Ca v 1.2 monoclonal antibody (NeuroMab Cav1.2 N263/31, Davis, CA, USA) 30 ; or \n",
      "anti-RyR2 monoclonal antibody (Thermo Fisher Scientific, Clone C3-33, Rockford, IL, USA)) …\n",
      "\n",
      "Depolarization drives neuronal plasticity. However, whether depolarization drives \n",
      "sensitization of peripheral nociceptive neurons remains elusive. By high-content screening (HCS) …\n",
      "\n",
      "Benzodiazpines (BZs) are clinically useful anxiolytics, sedatives, and anticonvulsants. Their \n",
      "mechanism of action is positive allosteric modulation of alpha-aminobutyric acid type A (…\n",
      "\n",
      "10.7554/eLife.49953.001 The voltage-gated K + channel Kv2.1 serves a major structural \n",
      "role in the soma and proximal dendrites of mammalian brain neurons, tethering the plasma …\n",
      "\n",
      "… with NeuroMab, a joint facility of the University of California-Davis and the NIH (http://www.neuromab.org)… \n",
      "N144/32 detects 11 of the 12 proteins in the A subfamily, but none in the B or C …\n",
      "\n",
      "… the Ms mAb to pan-Pcdh-γA (clone N144/32, to amino acids 720–804 of the variable \n",
      "cytoplasmic domain of mouse Pcdh-γA3), were from NeuroMab (Davis, CA). In immunoblots, both …\n",
      "\n",
      "It has been proposed that the combinatorial expression of γ‐protocadherins (Pcdh‐γs) and \n",
      "other clustered protocadherins (Pcdhs) provides a code of molecular identity and individuality …\n",
      "\n",
      "Growth of a properly complex dendrite arbor is a key step in neuronal differentiation and a \n",
      "prerequisite for neural circuit formation. Diverse cell surface molecules, such as the clustered …\n",
      "\n",
      "… 1:500–1:1000) were generated by NeuroMab in collaboration with the Weiner laboratory [55] … \n",
      "2 and thus all 22 γ-Pcdh isoforms); N144/32 (detecting all γA subfamily isoforms); N148/30 (…\n",
      "\n",
      "… years, including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection \n",
      "of … The sections were also labelled with the mouse anti-Calbindin L109/39 mAb (IgG2b). …\n",
      "\n",
      "… years, including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection \n",
      "of … The sections were also labelled with the mouse anti-Calbindin L109/39 mAb (IgG2b). …\n",
      "\n",
      "Protein palmitoylation is a reversible lipid modification that plays critical roles in protein \n",
      "sorting and targeting to specific cellular compartments. The neuronal microtubule-regulatory …\n",
      "\n",
      "… The specificity of each of the primary antibodies has been verified using Western blots \n",
      "from brain tissue by the UC, Davis/NeuroMab Facility and also in our laboratory in BLA tissue …\n",
      "\n",
      "… The monoclonal antibody against KChiP2 (clone K60/73) was obtained from NeuroMab \n",
      "Facility, supported by NIH grant U24NS050606 and maintained by the Department of …\n",
      "\n",
      "… The NeuroMab KChIP1 antibody stained a double band with … The NeuroMab KChIP2 antibody \n",
      "stained a single band with a … The NeuroMab KChIP3 antibody stained a single band with …\n",
      "\n",
      "Widely expressed in the adult central nervous system, the cellular prion protein (PrP C ) is \n",
      "implicated in a variety of processes, including neuronal excitability. Dipeptidyl aminopeptidase…\n",
      "\n",
      "Aging increases the risk of cardiac pathologies including atrial fibrillation and can alter \n",
      "myocardial responsiveness to therapeutic agents. Here, seeking molecular correlates of …\n",
      "\n",
      "… Mouse anti-KChIP2 (NeuroMab, Cat. no. 75-004, clone K60/73, RRID:AB_2280942) was \n",
      "raised against a recombinant protein corresponding to amino acids 1–252 (full length) of rat …\n",
      "\n",
      "… K60/73 (IgG1), and anti-KChIP3 K66/38 (IgG2a). These mAbs are available from the UC \n",
      "Davis/ NIH NeuroMab Facility (neuromab.… staining with the mouse mAb K60/73 in red) is found …\n",
      "\n",
      "Patients with diabetes exhibit significantly altered renin-angiotensin system (RAS) control. \n",
      "Recently, it has been determined that hyperglycemic conditions induce an increase in …\n",
      "\n",
      "… K87A/10: does not recognize denatured Nav1.6 on immunoblots, but yields specific \n",
      "labeling of nodes of Ranvier and the AIS in brain sections, and labels cerebellar granule cells …\n",
      "\n",
      "The axon initial segment (AIS) contains the site of action potential initiation and plays a \n",
      "major role in neuronal excitability. AIS function relies on high concentrations of different ion …\n",
      "\n",
      "… (polyclonal ASC-009 and monoclonal K87A/10) in cerebella from mice aged from P10 to … \n",
      "K87A/10 was developed by and obtained from the UC Davis/NINDS/NIMH NeuroMab Facility, …\n",
      "\n",
      "… -G (AnkG; NeuroMab, Davis, CA; N106/36, 1:200) and the urn:x-wiley:08941491:media:glia22991:glia22991-math-0003 \n",
      "channel isoform 1.6 (Na v 1.6; NeuroMab; K87A/10, 1:100) …\n",
      "\n",
      "Key points • Inappropriate electrochemical signalling between neurons is a central problem \n",
      "in epilepsy. Only a fraction of all heritable epilepsies are solved, and many are due to ion …\n",
      "\n",
      "Over the recent years, it has been found that microglia pseudopodia contact synapses, \n",
      "detect sick ones and prune them, even in adult animals. Myelinated nerves also carry out …\n",
      "\n",
      "… Therefore, mouse Na v 1.6 in cultured cells was detected with anti-Na v 1.6 Na + channel, \n",
      "clone K87A/10 mouse monoclonal IgG (Neuromab Facility, lot number: 440-8HK-09) used at 1:…\n",
      "\n",
      "… Monoclonal antibodies K74/71, K69/3, and K87A/10 were developed by and obtained from \n",
      "the University of California Davis/NINDS/NIMH NeuroMab Facility, supported by NIH grant …\n",
      "\n",
      "… The mouse K v 1.2 (clone K14/16, 1: 500) and Na v 1.6 (clone K87A/10, 1:500) antibodies \n",
      "were purchased from the University of California, Davis NINDS/NIMH NeuroMab Facility. The …\n",
      "\n",
      "A mouse embryonic stem (ES) cell line containing an inducible transgene for the proneural \n",
      "gene Neurog1 has been used to generate glutamatergic neurons at a high efficiency. The …\n",
      "\n",
      "Neuron-glia interactions establish functional membrane domains along myelinated axons. \n",
      "These include nodes of Ranvier, paranodal axoglial junctions and juxtaparanodes. …\n",
      "\n",
      "The human NTERA2/D1 (NT2) cells generate postmitotic neurons (NT2N cells) upon retinoic \n",
      "acid (RA) treatment and are functionally integrated in the host tissue following grafting into …\n",
      "\n",
      "The exact site of initiation and shape of action potentials vary among different neuronal types. \n",
      "The reason for this variability is largely unknown, but the subunit composition, density and …\n",
      "\n",
      "… Monoclonal antibodies K74/71, K69/3, and K87A/10 were developed by as well as obtained \n",
      "from the University of California Davis/NINDS/NIMH NeuroMab Facility, supported by NIH …\n",
      "\n",
      "Background Increased electrical activity in peripheral sensory neurons including dorsal root \n",
      "ganglia (DRG) and trigeminal ganglia neurons is an important mechanism underlying pain. …\n",
      "\n",
      "… Monoclonal antibodies K74/71, K69/3, and K87A/10 were developed by as well as obtained \n",
      "from the University of California Davis/NINDS/NIMH NeuroMab Facility, supported by NIH …\n",
      "\n",
      "The axon initial segment (AIS) is the site of action potential (AP) initiation, thus a crucial \n",
      "regulator of neuronal activity. In excitatory pyramidal neurons, the high density of voltage-gated …\n",
      "\n",
      "In neurons, voltage-gated sodium channel β subunits regulate the expression levels, subcellular \n",
      "localization, and electrophysiological properties of sodium channel α subunits. However…\n",
      "\n",
      "… monoclonal (clone K74/71, NeuroMab) antibodies raised respectively against amino acids \n",
      "… ) and mouse monoclonal (clone K87A/10, NeuroMab) antibodies raised respectively against …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "… (K87A/10) mouse monoclonal antibodies were purchased from NeuroMab. Affinity-purified \n",
      "rabbit polyclonal antibodies against βIV spectrin were described previously (Yang et al., 2004)…\n",
      "\n",
      "Schwann cells (SCs), ensheathing glia of the peripheral nervous system, support axonal \n",
      "survival and function. Abnormalities in SC metabolism affect their ability to provide this support …\n",
      "\n",
      "The pathogenesis of peripheral neuropathies in adults is linked to maintenance mechanisms \n",
      "that are not well understood. Here, we elucidate a novel critical maintenance mechanism …\n",
      "\n",
      "Mutations in the gene encoding the chromatin remodeler CHD8 are strongly associated \n",
      "with autism spectrum disorder (ASD). CHD8 haploinsufficiency also results in autistic …\n",
      "\n",
      "Autosomal dominant lateral temporal epilepsy (ADLTE) is an inherited syndrome caused by \n",
      "mutations in the leucine-rich glioma inactivated 1 (LGI1) gene. It is known that glutamatergic …\n",
      "\n",
      "Action potential initiation and propagation in myelinated axons require ion channel clustering \n",
      "at axon initial segments (AIS) and nodes of Ranvier. Disruption of these domains after …\n",
      "\n",
      "… anti-Na V 1.6 (K87A,10), mouse anti-K V 1.1 (K20/78), mouse anti-K V 1.2 (K14/16), and \n",
      "mouse anti-PSD95 (K28/43) were obtained from University of California, Davis/NIH NeuroMab …\n",
      "\n",
      "The axon initial segment (AIS) is a highly specialized axonal compartment where the action \n",
      "potential is initiated. The heterogeneity of AISs has been suggested to occur between …\n",
      "\n",
      "Many therapies for lysosomal storage disorders rely on cross-correction of lysosomal enzymes. \n",
      "In globoid cell leukodystrophy (GLD), mutations in GALC cause psychosine accumulation…\n",
      "\n",
      "The axon initial segment ( AIS ) is a specialized domain essential for neuronal function, the \n",
      "formation of which begins with localization of an ankyrin‐G (AnkG) scaffold. However, the …\n",
      "\n",
      "10.7554/eLife.49253.001 Inhibiting high-voltage-activated calcium channels (HVACCs; Ca V \n",
      "1/Ca V 2) is therapeutic for myriad cardiovascular and neurological diseases. For particular …\n",
      "\n",
      "Opioids modulate the electrical activity of magnocellular neurons (MCN) and inhibit neuropeptide \n",
      "release at their terminals in the neurohypophysis. We have previously shown that µ‐…\n",
      "\n",
      "Ca 2+ influx through high-voltage-activated calcium channels (HVACCs) controls diverse \n",
      "cellular functions. A critical feature enabling a singular signal, Ca 2+ influx, to mediate …\n",
      "\n",
      "… N7/18 (Neuromab; UC Davis). The images were obtained on an Olympus IX70 microscope \n",
      "using a SPOT/RT3 digital camera (Diagnostic Instruments Inc.) and processed with Adobe® …\n",
      "\n",
      "Excitation‐contraction (EC) coupling in skeletal muscles operates through a physical \n",
      "interaction between the dihydropyridine receptor (DHPR), acting as a voltage sensor, and the …\n",
      "\n",
      "… Jerusalem, Israel); mouse monoclonal anti-Ca v β 1 and anti-Ca v β 4 (1 : 1000 for immunoblotting \n",
      "and immunohistoblotting, and 1 : 250 for immunohistochemistry; both from Neuromab, …\n",
      "\n",
      "Presynaptic Ca V 2 channels are essential for Ca 2+ -triggered exocytosis. In addition, there \n",
      "are two competing models for their roles in synapse structure. First, Ca 2+ channels or Ca 2…\n",
      "\n",
      "Synapse formation requires the organization of presynaptic active zones, the synaptic \n",
      "vesicle release sites, in precise apposition to postsynaptic neurotransmitter receptor clusters; …\n",
      "\n",
      "Formation of synapses between motor neurons and muscles is initiated by clustering of \n",
      "acetylcholine receptors (AChRs) in the center of muscle fibers prior to nerve arrival. This AChR …\n",
      "\n",
      "… 75-058; RRID:AB_2261666) recognized a single band of 105 kDa in Western blots of rat \n",
      "and mouse brain membranes (UC Davis/NIH NeuroMab Facility). SAP102 labeling was …\n",
      "\n",
      "… antibody (0.5 μg/ml) (K64/15, UC Davis/NIH NeuroMab Facility), anti-PSD95 mouse \n",
      "monoclonal antibody (0.5 μg/ml) (K28/43, UC Davis/NIH NeuroMab Facility), anti-GluA1 mouse …\n",
      "\n",
      "In vitro studies suggest that the intracellular C terminus of Neuroligin1 (NL1) could play a \n",
      "central role in the maturation of excitatory synapses. However, it is unknown how this activity …\n",
      "\n",
      "… Laboratories, San Jose, CA), anti-SAP97 clone K64/15 (mouse monoclonal antibody Cat# \n",
      "75-030, dilution 1:3000, NeuroMab, Davis, CA). To estimate potential variability in protein …\n",
      "\n",
      "It has been suggested that synapse-associated protein of 97-kDa molecular weight (SAP97) \n",
      "is a susceptibility factor for childhood and adult neuropsychiatric disorders. SAP97 is a …\n",
      "\n",
      "Synaptic signaling complexes are held together by scaffold proteins, each of which is selectively \n",
      "capable of interacting with a number of other proteins. In previous studies of rabbit retina…\n",
      "\n",
      "Aβ peptide that accumulates in Alzheimer’s disease brain, derives from proteolytic processing \n",
      "of the amyloid precursor protein (APP) that exists in three main isoforms derived by …\n",
      "\n",
      "… Anti–PAN MAGUK antibody (clone K28/86) and anti-SAP97 antibody (clone K64/15) were \n",
      "purchased from the University of California Davis/NIH NeuroMab Facility. Anti–human SAP97 …\n",
      "\n",
      "The number and function of synaptic AMPA receptors (AMPARs) tightly regulates excitatory \n",
      "synaptic transmission. Current evidence suggests that AMPARs are inserted into the …\n",
      "\n",
      "AMPA receptors and interacting proteins are importantly involved in mediating stress-dependent \n",
      "plasticity. Previously we reported that GluA1-containing AMPA receptors and their …\n",
      "\n",
      "Background Postsynaptic density (PSD)-95-like membrane-associated guanylate kinases (PSD-MAGUKs) \n",
      "are scaffold proteins in PSDs that cluster signaling molecules near NMDA …\n",
      "\n",
      "… K64/15) from NeuroMab (Davis, CA); neuronal nitric oxide synthase (nNOS) (clone 16), and \n",
      "β-catenin (clone 14) from BD Transduction Labs (San Jose, CA); and CRMP2 (clone C4G) …\n",
      "\n",
      "… Abs against Dlg1 (K64/15), pan-Kvβ potassium channel subunit (K40/17), Kv1.3 (L23/27), \n",
      "and Kv1.5 (K7/45) were purchased from Neuromab, and anti–β-actin Abs were purchased …\n",
      "\n",
      "Despite the central role PSD-95 plays in anchoring postsynaptic AMPARs, how PSD-95 \n",
      "itself is tethered to postsynaptic sites is not well understood. Here we show that the F-actin …\n",
      "\n",
      "Excitatory glutamatergic synapses facilitate important aspects of communication between \n",
      "the neurons that govern complex forms of behavior. Accordingly, small differences in the …\n",
      "\n",
      "The PI 3-kinase (PI 3-K) signaling pathway is essential for Schwann cell myelination. Here \n",
      "we have characterized PI 3-K effectors activated during myelination by probing myelinating …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "… were all based on mAbs from NeuroMab (UC Davis/NIH NeuroMab Facility, nd). This facility \n",
      "… 2013), our current efforts to systematically convert NeuroMab mAbs into scFvs that retain the …\n",
      "\n",
      "… Number APC-057 from Alomone labs and mouse anti-HCN3, clone N141/28 from NeuroMab), \n",
      "previous studies have suggested that HCN3 expression in these regions of the central …\n",
      "\n",
      "Properties, developmental regulation, and cAMP modulation of the hyperpolarization-activated \n",
      "current (I h ) were investigated by the whole cell patch-clamp technique in vestibular …\n",
      "\n",
      "… NeuroMab, a joint facility of the University of California-Davis and the NIH (http://www.neuromab.… \n",
      "One mAb was produced from the B2 VCD fusion (N148/30, anti-B2), which is specific for …\n",
      "\n",
      "… The mouse anti-γ-Pcdh constant domain (N159/5) and mouse anti-γ-Pcdh-B2 (N148/30) \n",
      "were generated by the UC Davis/NIH NeuroMab Facility using antigens produced in our …\n",
      "\n",
      "… :500–1:1000) were generated by NeuroMab in collaboration with the Weiner laboratory [55] … \n",
      "N144/32 (detecting all γA subfamily isoforms); N148/30 (specific for γB2); N174B/27 (specific …\n",
      "\n",
      "… Anti-GABA-A β3 receptor N87/25: yields a single band of expected size (55 kDa) on \n",
      "immunoblots of rat, mouse, and human brain samples, and does not cross-react with GABA-A β1 …\n",
      "\n",
      "… After fixation (3% paraformaldehyde; Sigma) and blockade (5% FBS; Gibco), the cells were \n",
      "stained with mouse-anti human β 3 (NeuroMab; clone N87/25; UC Davis/NIH NeuroMab …\n",
      "\n",
      "… A monoclonal antibody raised against intracellular residues 370–433 of the GABA A receptor \n",
      "β3 subunit (4 μg/ml, clone N87/25, UC Davis/NIH NeuroMab Facility) was used to detect β3 …\n",
      "\n",
      "… layer (ML), and anti-GABA-AR b3 mAb N87/25 (IgG1, green) in the granule cell layer (GL). … \n",
      "We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and …\n",
      "\n",
      "The evaluation of potential health risks associated with neuronal exposure to space radiation \n",
      "is critical for future long duration space travel. The purpose of this study was to evaluate …\n",
      "\n",
      "GABA A and glycine receptors are thought to compete for gephyrin-binding sites at mixed \n",
      "inhibitory synapses. Changes in the occupancy of one receptor type are therefore expected to …\n",
      "\n",
      "AB. Diagram of the structure of the human SHANK2 gene and protein.(A) The nonsense \n",
      "mutation R841X is found in an individual with ASD and ID.(B) SHANK2 protein contains 5 …\n",
      "\n",
      "… the N59/36 R-mAb project. (D) Agarose gel analysis of products of restriction enzyme digestion \n",
      "of N59/36 … We thank the current and former staff members of the UC Davis/NIH NeuroMab …\n",
      "\n",
      "… N59/36; NeuroMab) and anti-α-tubulin antibody (1:1000, mouse monoclonal DM1A clone, \n",
      "Upstate). Protein bands were visualized by using secondary goat peroxidase-conjugated anti-…\n",
      "\n",
      "Genetic code expansion exploiting unnatural amino acids (Uaas) is a powerful technique to \n",
      "create novel protein function in vivo. Here, we provide a protocol for the incorporation of two …\n",
      "\n",
      "… Primary antibodies used were as follows: PSD-95 (clone K28/43 UC Davis/NIH Neuromab \n",
      "Facility), vGlut1 (clone N59/36, UC Davis/NIH Neuromab Facility), Syp (sc-7568, Santa Cruz …\n",
      "\n",
      "Recycling of internalized receptors from endosomal compartments is essential for the \n",
      "receptors' cell-surface homeostasis. Sorting nexin 27 (SNX27) cooperates with the retromer …\n",
      "\n",
      "Accumulating genetic evidence suggests that schizophrenia (SZ) is associated with individually \n",
      "rare copy number variations (CNVs) of diverse genes, often specific to single cases. …\n",
      "\n",
      "… Primary antibodies used were as follows: PSD-95 (clone K28/43 UC Davis/NIH Neuromab \n",
      "Facility), vGlut1 (clone N59/36, UC Davis/NIH Neuromab Facility), Syp (sc-7568, Santa Cruz …\n",
      "\n",
      "Allostery represents a fundamental mechanism of biological regulation that involves long-range \n",
      "communication between distant protein sites. It also provides a powerful framework for …\n",
      "\n",
      "… The mouse monoclonal antibodies PSD95 (Clone K28/43) for immunoblotting, and NR2B \n",
      "(Clone N59-36) were developed by and obtained from the UC Davis/NIH Neuromab facility. …\n",
      "\n",
      "… In contrast, when probed with an antibody raised against the N terminus of SHANK3 (Neuromab \n",
      "367/62), we detected truncated SHANK3 bands in striatal PSD preparations from the …\n",
      "\n",
      "While the higher prevalence of Alzheimer’s disease (AD) in women is clear, studies suggest \n",
      "that biological sex may also influence AD pathogenesis. However, mechanisms behind …\n",
      "\n",
      "The calcium–calmodulin activated kinase CaMKII mediates many forms of learning and \n",
      "memory. Activity-regulated translocation of CaMKII to synapses is important for its functions in …\n",
      "\n",
      "… N59/36 or N59/20; Neuromab), anti-GABA-B1 antibody (1:500; rabbit monoclonal antibody \n",
      "raised against the GABA-B1 CTD; generous gift from Julie Kniazeff, Montpellier, France), anti-…\n",
      "\n",
      "… 10; NeuroMab, N59/36, mouse). Antibody specificity was determined by lack of staining after \n",
      "omission of primary or secondary antibodies. Alexa647 (1:200), Alexa488 (1:200), or Cy3 (1:…\n",
      "\n",
      "… (NeuroMab clone N59/36; dilution 1:1000) for 48 h at 4C. NR2B antibody was visualized \n",
      "with Cy3-donkey anti-mouse secondary antibody (1:400), and analyzed with an LSM 700 …\n",
      "\n",
      "Phosphorylation regulates surface and synaptic expression of NMDA receptors (NMDARs). \n",
      "Both the tyrosine kinase Fyn and the tyrosine phosphatase striatal-enriched protein tyrosine …\n",
      "\n",
      "… N59/36, NeuroMab). Protein bands were visualized using secondary goat peroxidase-linked \n",
      "anti-mouse antibodies (1:10,000, Jackson ImmunoResearch catalog number #115-035-003…\n",
      "\n",
      "Coordinated development of excitatory and inhibitory synapses is essential for higher brain \n",
      "function, and impairment in this development is associated with neuropsychiatric disorders. …\n",
      "\n",
      "… against NR2B (clone N59/36, at 1:100), SAP102 (clone N19/2 at 1:50), GKAP (clone N127/31 \n",
      "at 1:100), and Shank2 (clone N23B/6 at 1:200) were from NeuroMab; mouse monoclonal …\n",
      "\n",
      "L-methionine, the principal sulfur-containing amino acid in proteins, plays critical roles in \n",
      "cell physiology as an antioxidant and in the breakdown of fats and heavy metals. Previous …\n",
      "\n",
      "Heterotetrameric glutamate receptors are essential for the development, function, and \n",
      "plasticity of spine synapses but how they are organized to achieve this is not known. Here we …\n",
      "\n",
      "… The dilutions of antibodies for immunoblot are as follows: 1:100 NR2B (NeuroMab, clone \n",
      "N59/36, 73-101), 1:100 PSD-95 (NeuroMab, clone K28/43, 73-028), 1:1000 Arc (Santa Cruz …\n",
      "\n",
      "Organization of signaling complexes at excitatory synapses by membrane-associated guanylate \n",
      "kinase (MAGUK) proteins regulates synapse development, plasticity, senescence and …\n",
      "\n",
      "The metalloproteinase ADAM10 is of importance for Notch-dependent cortical brain \n",
      "development. The protease is tightly linked with α-secretase activity toward the amyloid precursor …\n",
      "\n",
      "Excitatory glutamatergic synapses facilitate important aspects of communication between \n",
      "the neurons that govern complex forms of behavior. Accordingly, small differences in the …\n",
      "\n",
      "Glutamate and its receptor N-methyl-D-aspartate receptor (NMDAR) have been associated \n",
      "with cancer, although their functions are not fully understood. Herein, we implicate glutamate-…\n",
      "\n",
      "Synaptic plasticity, the cellular basis of learning and memory, involves the dynamic \n",
      "trafficking of AMPA receptors (AMPARs) into and out of synapses. One of the remaining key …\n",
      "\n",
      "Localization of the N-methyl-D-aspartate type glutamate receptor (NMDAR) to dendritic \n",
      "spines is essential for excitatory synaptic transmission and plasticity. Rather than remaining …\n",
      "\n",
      "Protein ubiquitination has a significant influence on diverse aspects of neuronal development \n",
      "and function. Dorfin, also known as Rnf19a, is a RING finger E3 ubiquitin ligase implicated …\n",
      "\n",
      "Synaptic adhesion molecules regulate various aspects of synapse development, function \n",
      "and plasticity. These functions mainly involve trans-synaptic interactions and positive …\n",
      "\n",
      "… , mouse anti-GluA2 monoclonal antibody (1:250, Neuromab, L21/32), mouse anti-GluN2B \n",
      "monoclonal antibody (1:250, Neuromab, N59/36), mouse anti-GluN2A monoclonal antibody (1:…\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "… N207/27; UC Davis/NIH NeuroMab Facility) in the carbonate buffer (500 mM, pH 9.6) overnight \n",
      "at 4C. After washing with 0.1% TBST, samples were incubated with the blocking buffer (5…\n",
      "\n",
      "Myasthenia gravis (MG) is the most common disorder affecting the neuromuscular junction (NMJ). \n",
      "MG is frequently caused by autoantibodies against acetylcholine receptor (AChR) and …\n",
      "\n",
      "10.7554/eLife.04287.001 Lrp4, the muscle receptor for neuronal Agrin, is expressed in the \n",
      "hippocampus and areas involved in cognition. The function of Lrp4 in the brain, however, is …\n",
      "\n",
      "Background The neuromuscular junction (NMJ) is a peripheral synapse critical to muscle \n",
      "contraction. Like acetylcholine receptors (AChRs), many essential proteins of NMJ are …\n",
      "\n",
      "Simple Summary Schwann cells (SCs) are the primary glial cells in the peripheral nervous \n",
      "system and play an indispensable role in peripheral nerve regeneration. A recent study …\n",
      "\n",
      "The neuromuscular junction (NMJ) is a synapse between motoneurons and skeletal muscles \n",
      "to control motor behavior. Acetylcholine receptors (AChRs) are restricted at the synaptic …\n",
      "\n",
      "Neuromuscular junction (NMJ) formation requires precise interaction between motoneurons \n",
      "and muscle fibers. LRP4 is a receptor of agrin that is thought to act in cis to stimulate MuSK …\n",
      "\n",
      "… N207/27, UC Davis/NIH NeuroMab Facility). After washing, the membrane was incubated \n",
      "with TBST buffer containing HRP-conjugated goat anti-mouse and rabbit IgG from Pierce (1:…\n",
      "\n",
      "Hyperactive Notch signalling is frequently observed in breast cancer and correlates with \n",
      "poor prognosis. However, relatively few mutations in the core Notch signalling pathway have …\n",
      "\n",
      "… (NeuroMab, clone N286/74) and imaged as described above. Fold-changes were determined \n",
      "using the ratio of SCG10 immunofluorescence in expressing (mEos2 or mRFP-positive) to …\n",
      "\n",
      "… The monoclonal antibodies were generated by Neuromab … clone N39/B8, and CaVβ4 clone \n",
      "N10/7. A polyclonal antibody to … use of antibodies mainly from Neuromab to start to examine …\n",
      "\n",
      "Opioids modulate the electrical activity of magnocellular neurons (MCN) and inhibit neuropeptide \n",
      "release at their terminals in the neurohypophysis. We have previously shown that µ‐…\n",
      "\n",
      "… Jerusalem, Israel); mouse monoclonal anti-Ca v β 1 and anti-Ca v β 4 (1 : 1000 for immunoblotting \n",
      "and immunohistoblotting, and 1 : 250 for immunohistochemistry; both from Neuromab, …\n",
      "\n",
      "Synapse formation requires the organization of presynaptic active zones, the synaptic \n",
      "vesicle release sites, in precise apposition to postsynaptic neurotransmitter receptor clusters; …\n",
      "\n",
      "… Primary antibodies used were rabbit anti-Gem (Novus Biologicals, 1:100–250), mouse \n",
      "anti-βIII-Tubulin (Sigma, 1:500), mouse monoclonal anti-Ca V β4 (NeuroMab, clone N10/7; 1:…\n",
      "\n",
      "… Antibodies to the following astrocytes markers have been used: monoclonal anti-Aldh1L1, \n",
      "clone N103/39 (Cat #MABN495, NeuroMab UC, Davis, CA) is used as a protoplasmic …\n",
      "\n",
      "8 Abstract 9 Introduction: Several neurodegenerative diseases are classified as proteopathies \n",
      "as they are associated with the 10 aggregation of misfolded proteins. Synucleinopathies …\n",
      "\n",
      "BACE1 is the sole secretase for generating β-amyloid (Aβ) in vivo and is being actively pursued \n",
      "as a drug target for the treatment of Alzheimer's disease. Transmembrane BACE1 exerts …\n",
      "\n",
      "Remyelination within the central nervous system (CNS) most often is the result of oligodendrocyte \n",
      "progenitor cells differentiating into myelin-forming oligodendrocytes. In some cases, …\n",
      "\n",
      "Relating changes in gene expression to discrete developmental events remains an elusive \n",
      "challenge in neuroscience, in part because most neural territories are comprised of multiple …\n",
      "\n",
      "The human amygdala grows during childhood, and its abnormal development is linked to \n",
      "mood disorders. The primate amygdala contains a large population of immature neurons in the …\n",
      "\n",
      "Astrocytes express a variety of G protein–coupled receptors and might influence cognitive \n",
      "functions, such as learning and memory. However, the roles of astrocytic G s -coupled …\n",
      "\n",
      "Neocortical heterotopia consist of ectopic neuronal clusters that are frequently found in \n",
      "individuals with cognitive disability and epilepsy. However, their pathogenesis remains poorly …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Several lines of evidence suggest that the aggregation and deposition of amyloid‐β peptide \n",
      "(Aβ) initiate the pathology of Alzheimer's disease (AD). Recently, a genome‐wide …\n",
      "\n",
      "The normal BBB (blood–brain barrier) consists of a series of structures collectively known as \n",
      "neurovascular units, or NVU, that are composed of endothelial cells and astrocyte endfeet …\n",
      "\n",
      "Polycomb Repressive Complex 2 (PRC2) mediates transcriptional silencing by catalyzing \n",
      "histone H3 lysine 27 trimethylation (H3K27me3), but its role in the maturation of postmitotic …\n",
      "\n",
      "Postnatal and adult neurogenesis in the subventricular zone and subgranular zone of \n",
      "animals such as rodents and non‐human primates has been observed with many different …\n",
      "\n",
      "More than 1,200 mutations in neuronal voltage‐gated sodium channel (VGSC) genes have \n",
      "been identified in patients with several epilepsy syndromes. A common feature of genetic …\n",
      "\n",
      "Thalamocortical neurons (TCNs) play a critical role in the maintenance of thalamocortical \n",
      "oscillations, dysregulation of which can result in certain types of seizures. Precise control over …\n",
      "\n",
      "In the primate visual system, the ganglion cells of the magnocellular pathway underlie motion \n",
      "and flicker detection and are relatively transient, while the more sustained ganglion cells of …\n",
      "\n",
      "… ); b-actin (ab8226, Abcam; 1:10000), mouse antiCav3.1, clone N178A/9 (1mg/mL, ref. no. \n",
      "75-206, … (1:150, clone N178A/9, Cat. No. 75-206, Neuromab) and goat anti-mouse Alexa594-…\n",
      "\n",
      "Neuronal GABAergic responses switch from excitatory to inhibitory at an early postnatal \n",
      "period in rodents. The timing of this switch is controlled by intracellular Cl− concentrations, but …\n",
      "\n",
      "Adapting between scotopic and photopic illumination involves switching the routing of \n",
      "retinal signals between rod and cone-dominated circuits. In the daytime, cone signals pass …\n",
      "\n",
      "10.7554/eLife.49953.001 The voltage-gated K + channel Kv2.1 serves a major structural \n",
      "role in the soma and proximal dendrites of mammalian brain neurons, tethering the plasma …\n",
      "\n",
      "Neuronal phenotypes are controlled by terminal selector transcription factors in invertebrates, \n",
      "but few examples of such regulators have been provided in vertebrates. TCF7L2 has been …\n",
      "\n",
      "Neuronal phenotypes are controlled by terminal selector transcription factors in invertebrates, \n",
      "but only a few examples of such regulators have been provided in vertebrates. We …\n",
      "\n",
      "Hyperpolarization-activated cyclic-nucleotide-gated cation nonselective (HCN) channels \n",
      "are involved in the pathology of nervous system diseases. HCN channels and γ-aminobutyric …\n",
      "\n",
      "The hyperpolarization-activated cyclic-nucleotide-gated non-selective cation (HCN) channels \n",
      "play a vital role in the neurological basis underlying nervous system diseases. However, …\n",
      "\n",
      "Dysregulation of hyperpolarization-activated cyclic nucleotide-gated cation (HCN) channels \n",
      "alters neuronal excitability. However, the role of HCN channels in status epilepticus is not …\n",
      "\n",
      "… L11A/41, mouse monoclonal anti-neurofascin is described in detail in Schafer et al. (… \n",
      "L11A/41 can be obtained from the UC Davis/NINDS/NIMH NeuroMab facility (http://www.neuromab…\n",
      "\n",
      "Background: Modulation of M-type currents has been proposed as a new strategy for the \n",
      "treatment of neuropathic pain due to their role in regulating neuronal excitability. Using …\n",
      "\n",
      "The axon initial segment (AIS) is a highly specialized neuronal compartment that plays a key \n",
      "role in neuronal development and excitability. It concentrates multiple membrane proteins …\n",
      "\n",
      "Cerebrospinal fluid samples collected from children during initial presentation of central \n",
      "nervous system inflammation, who may or may not subsequently be diagnosed as having …\n",
      "\n",
      "The voltage-gated K + channels Kv7.2 and Kv7.3 are located at the axon initial segment (AIS) \n",
      "and exert strong control over action potential generation. Therefore, changes in their …\n",
      "\n",
      "Neuronal GABAergic responses switch from excitatory to inhibitory at an early postnatal \n",
      "period in rodents. The timing of this switch is controlled by intracellular Cl− concentrations, but …\n",
      "\n",
      "The exact site of initiation and shape of action potentials vary among different neuronal types. \n",
      "The reason for this variability is largely unknown, but the subunit composition, density and …\n",
      "\n",
      "… , mouse clone A12/18, and anti-Pan-neurofascin against an intracellular epitope, mouse \n",
      "clone L11A/41, were obtained from University of California, Davis, CA (NIH NeuroMab facility). …\n",
      "\n",
      "Axon formation, the initial step in establishing neuronal polarity, critically depends on local \n",
      "microtubule reorganization and is characterized by the formation of parallel microtubule …\n",
      "\n",
      "Neurons implement a variety of plasticity mechanisms to alter their function over timescales \n",
      "ranging from seconds to days. One powerful means of controlling excitability is to directly …\n",
      "\n",
      "Neurofascin (NF) is a cell-adhesion molecule that is found at the nodes of Ranvier. The 186 \n",
      "kDa isoform of neurofascin (NF186) is expressed on the axon in the exposed node, and the …\n",
      "\n",
      "… The pan-NF monoclonal antibody (clone L11A/41; RRID:AB_10672370) was made by \n",
      "immunizing mice with a glutathione S-transferase fusion protein containing the intracellular …\n",
      "\n",
      "Many therapies for lysosomal storage disorders rely on cross-correction of lysosomal enzymes. \n",
      "In globoid cell leukodystrophy (GLD), mutations in GALC cause psychosine accumulation…\n",
      "\n",
      "Polarized cargo transport is essential for neuronal function. However, the minimal basic \n",
      "components required for selective cargo sorting and distribution in neurons remain elusive. We …\n",
      "\n",
      "… ) and a positive control NeuroMab mAb against a different target (… -positive cells do not label \n",
      "with L106/4. D. An adjacent 400 … from adult mouse cortex with L106/4 mAb against gephyrin (…\n",
      "\n",
      "Axonal ion channels control spike initiation and propagation along the axon and determine \n",
      "action potential waveform. We show here that functional suppression of axonal Kv1 channels …\n",
      "\n",
      "Fast, saltatory conduction in myelinated nerves requires the clustering of voltage-gated \n",
      "sodium channels (Nav) at nodes of Ranvier in a nodal complex. The Neurofascin (Nfasc) gene …\n",
      "\n",
      "Abstract Changes in ion channel expression are implicated in the etiology of epilepsy. \n",
      "However, the molecular leading to long-term aberrant expression of ion channels are not well …\n",
      "\n",
      "… from NeuroMab (UC Davis/NIH NeuroMab facility): mouse monoclonal anti-Kv1.1 (K36/15) … \n",
      "with the Kv1.1 mouse monoclonal antibody (NeuroMab) as above. After several rinses, …\n",
      "\n",
      "… 1e mAb (K36/15; UC Davis/NIH NeuroMab Facility) before detergent permeabilization to \n",
      "detect the cell surface pool. The total cellular pools of the respective proteins were detected by …\n",
      "\n",
      "To gain insights into the phenotype of voltage‐gated potassium (Kv)1.1 and Kv4.2 knockout \n",
      "mice, we used immunohistochemistry to analyze the expression of component principal or α …\n",
      "\n",
      "Homeostatic plasticity of intrinsic excitability goes hand in hand with homeostatic plasticity \n",
      "of synaptic transmission. However, the mechanisms linking the two forms of homeostatic …\n",
      "\n",
      "Homeostatic plasticity of intrinsic excitability goes hand-in-hand with homeostatic plasticity \n",
      "of synaptic transmission. However, the mechanisms linking the two forms of homeostatic …\n",
      "\n",
      "… 1C (mouse monoclonal K36/15; Neuromab 73–105), and anti-Kv1.1D (mouse \n",
      "monoclonal K20/78; Neuromab 73–007); preimmune control IgG was from rabbit (Millipore/Upstate, …\n",
      "\n",
      "The exact site of initiation and shape of action potentials vary among different neuronal types. \n",
      "The reason for this variability is largely unknown, but the subunit composition, density and …\n",
      "\n",
      "Spike timing-dependent synaptic plasticity (STDP) serves as a key cellular correlate of \n",
      "associative learning, which is facilitated by elevated attentional and emotional states involving …\n",
      "\n",
      "Congenital peripheral nerve hyperexcitability (PNH) is usually associated with impaired \n",
      "function of voltage-gated K + channels (VGKCs) in neuromyotonia and demyelination in …\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "… K36/15) have also been described previously (21, 24). These monoclonal antibodies are \n",
      "available from NeuroMab (www.neuromab… ectodomain-directed K36/15 mouse monoclonal …\n",
      "\n",
      "Supplementary Figure 2 (continued) Ten electron micrographs taken from serial sections of \n",
      "neurobiotin-labeled (HRP end-product) boutons (B) of AAC B45a making type II synapses (…\n",
      "\n",
      "… (1:200, Neuromab #75-105, clone K36/15, external), and mouse anti-ankyrin G (1:200, \n",
      "Neuromab #75-146, clone N106/36). The following secondary antibodies were used for …\n",
      "\n",
      "The spiral ganglion conveys afferent auditory information predominantly through a single \n",
      "class of type I neurons that receive signals from inner hair cell sensory receptors. These …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "10.7554/eLife.21392.001 A high density of Na + channels at nodes of Ranvier is necessary \n",
      "for rapid and efficient action potential propagation in myelinated axons. Na+ channel …\n",
      "\n",
      "Fast-spiking cells (FS cells) are a prominent subtype of neocortical GABAergic interneurons \n",
      "with important functional roles. Multiple FS cell properties are coordinated for rapid response…\n",
      "\n",
      "Dominantly inherited missense mutations of the KCNA1 gene, which encodes the K V 1.1 \n",
      "potassium channel subunit, cause Episodic Ataxia type 1 (EA1). Although the cerebellar …\n",
      "\n",
      "Selection of the correct targets for myelination and regulation of myelin sheath growth are \n",
      "essential for central nervous system (CNS) formation and function. Through a genetic screen …\n",
      "\n",
      "The axon initial segment (AIS) is a highly specialized axonal compartment where the action \n",
      "potential is initiated. The heterogeneity of AISs has been suggested to occur between …\n",
      "\n",
      "Light as a trigger is ideal to perturb biological function due to its high spatial and temporal \n",
      "precision. This is evidenced by the ever-expanding arsenal of genetically-encoded optical …\n",
      "\n",
      "… Membranes were probed with mouse anti-Kirrel3 1:100 (Neuromab Clone N321C/49; \n",
      "catalog no. 75-333) that detects the long and short isoforms of Kirrel3, rat anti-HSC70/HSP73 1:…\n",
      "\n",
      "Projections from sensory neurons of olfactory systems coalesce into glomeruli in the brain. \n",
      "The Kirrel receptors are believed to homodimerize via their ectodomains and help separate …\n",
      "\n",
      "The formation of olfactory maps in the olfactory bulb (OB) is crucial for the control of innate \n",
      "and learned mouse behaviors. Olfactory sensory neurons (OSNs) expressing a specific …\n",
      "\n",
      "… Primary antibodies were as follows: mouse anti-MXD3 (NeuroMab clone N129/15) 1:1,000; \n",
      "mouse anti-histone3 (Millipore) 1:10,000. Secondary antibody was goat anti-mouse IRDye …\n",
      "\n",
      "… and AnkB (N105/13 (c) or N105/17 (d)). The arrows indicate … cell (Sc) cultures (AnkB, N105/17). \n",
      "The molecular weight is … cells and immunostained for AnkB (N105/17), neurofilament-M (…\n",
      "\n",
      "… Mouse monoclonal antibodies against ankyrinB (N105/17) and ankyrinG (N106/36) were \n",
      "purchased from the UC Davis/NIH NeuroMab Facility (Davis, CA). Mouse monoclonal …\n",
      "\n",
      "… S1 a, b); (ii) the guinea pig, rabbit and mouse monoclonal anti-AnkG (N106/65; IgG2b; \n",
      "Neuromab) antibodies were shown to strictly colocalize at the AISs (Fig. S1 d); (iii) western blots …\n",
      "\n",
      "The scaffolding protein ankyrin-G is required for Na + channel clustering at axon initial \n",
      "segments. It is also considered essential for Na + channel clustering at nodes of Ranvier to …\n",
      "\n",
      "TDP-43 is extensively studied in neurons in physiological and pathological contexts. \n",
      "However, emerging evidence indicates that glial cells are also reliant on TDP-43 function. We …\n",
      "\n",
      "Haploinsufficiency of the SHANK3 gene is causally linked to autism spectrum disorder (ASD), \n",
      "and ASD-associated genes are also enriched for chromatin remodelers. Here we found …\n",
      "\n",
      "α-Synuclein plays a key role in the pathogenesis of Parkinson’s disease and related disorders, \n",
      "but critical interacting partners and molecular mechanisms mediating neurotoxicity are …\n",
      "\n",
      "Action potential conduction along myelinated axons depends on high densities of voltage-gated \n",
      "Na + channels at the nodes of Ranvier. Flanking each node, paranodal junctions (…\n",
      "\n",
      "The major determinant of poor outcome in Guillain‐Barré syndrome (GBS) is axonal \n",
      "degeneration. Pathways leading to primary axonal injury in the motor axonal variant are well …\n",
      "\n",
      "The histological processing of musculoskeletal tissue might be challenging. The alteration \n",
      "of tissue composition eg by calcification of soft tissue in the elderly, after trauma or surgical …\n",
      "\n",
      "AnkyrinG (ankG) is highly enriched in neurons at axon initial segments (AISs) where it \n",
      "clusters Na + and K + channels and maintains neuronal polarity. How ankG becomes …\n",
      "\n",
      "… We provide examples from NeuroMab screens, and from the subsequent specialized … of \n",
      "NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs at the …\n",
      "\n",
      "Interleukin-6 (IL-6) is an important neuroimmune factor that is increased in the brain by alcohol \n",
      "exposure/withdrawal and is thought to play a role in the actions of alcohol on the brain. …\n",
      "\n",
      "… N415/24, 1–500, UC Davis/NIH NeuroMab Facility); a purified mouse monoclonal antibody \n",
      "to gamma-aminobutyric acid receptor subunit alpha-1 (GABA A R alpha-1) produced by …\n",
      "\n",
      "GABA A receptors are ligand-gated anion channels that are important regulators of neuronal \n",
      "inhibition. Mutations in several genes encoding receptor subunits have been identified in …\n",
      "\n",
      "The excitatory synapse between hippocampal CA3 and CA1 pyramidal neurons exhibits \n",
      "long-term potentiation (LTP), a positive feedback process implicated in learning and memory in …\n",
      "\n",
      "Opioids modulate the electrical activity of magnocellular neurons (MCN) and inhibit neuropeptide \n",
      "release at their terminals in the neurohypophysis. We have previously shown that µ‐…\n",
      "\n",
      "Presynaptic Ca V 2 channels are essential for Ca 2+ -triggered exocytosis. In addition, there \n",
      "are two competing models for their roles in synapse structure. First, Ca 2+ channels or Ca 2…\n",
      "\n",
      "… 75-058; RRID:AB_2261666) recognized a single band of 105 kDa in Western blots of rat \n",
      "and mouse brain membranes (UC Davis/NIH NeuroMab Facility). SAP102 labeling was …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "Glutamate plays a role in hair cell afferent transmission, but the receptors that mediate \n",
      "neurotransmission between outer hair cells (OHCs) and type II ganglion neurons are not well …\n",
      "\n",
      "The modulation of AMPA receptor (AMPAR) content at synapses is thought to be an underlying \n",
      "molecular mechanism of memory and learning. AMPAR content at synapses is highly …\n",
      "\n",
      "… Mouse monoclonal anti-PSD-93 (N18/30) and anti-ADAM22 (cytoplasmic, N46/30; extracellular, \n",
      "N57/2) were obtained from the University of California Davis/NIH NeuroMab Facility and …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "Synaptic signaling complexes are held together by scaffold proteins, each of which is selectively \n",
      "capable of interacting with a number of other proteins. In previous studies of rabbit retina…\n",
      "\n",
      "… 1e mAb (K36/15; UC Davis/NIH NeuroMab Facility) before detergent permeabilization to \n",
      "detect the cell surface pool. The total cellular pools of the respective proteins were detected by …\n",
      "\n",
      "… PSD93 was not detected in varicosity but identified in the whole gut extract using a \n",
      "monoclonal antibody (clone N18/30), and signal was not seen when the crude gut extract was …\n",
      "\n",
      "The mechanisms controlling synapse growth and maintenance are of critical importance for \n",
      "learning and memory. The MAGUK family of synaptic scaffolding proteins is abundantly …\n",
      "\n",
      "Membrane-associated guanylate kinases (MAGUKs), including SAP102, PSD-95, PSD-93, \n",
      "and SAP97, are scaffolding proteins for ionotropic glutamate receptors at excitatory synapses…\n",
      "\n",
      "How does chronic activity modulation lead to global remodeling of proteins at synapses and \n",
      "synaptic scaling? Here we report that guanylate kinase–associated protein (GKAP; also …\n",
      "\n",
      "… N18/30), and SAP97 (clone K64/15) from NeuroMab (Davis, CA); neuronal nitric oxide \n",
      "synthase (nNOS) (clone 16), and β-catenin (clone 14) from BD Transduction Labs (San Jose, CA)…\n",
      "\n",
      "… Mouse monoclonal anti-PSD-93 (N18/30) and polyclonal anti-PSD-93 antibodies were \n",
      "obtained from NeuroMab and from Alomone Labs, respectively. Additional rabbit polyclonal anti-…\n",
      "\n",
      "Despite the central role PSD-95 plays in anchoring postsynaptic AMPARs, how PSD-95 \n",
      "itself is tethered to postsynaptic sites is not well understood. Here we show that the F-actin …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "Neurite outgrowth is key to the formation of functional circuits during neuronal development. \n",
      "Neurotrophins, including nerve growth factor (NGF), increase neurite outgrowth in part by …\n",
      "\n",
      "Prior to maturation, mouse oocytes are arrested at the germinal vesicle (GV) stage during \n",
      "which they experience constitutive calcium (Ca 2+ ) influx and spontaneous Ca 2+ oscillations. …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "… Acknowledgements: Most of the antibodies used in this study are available from the UC \n",
      "Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). Correspondence should be addressed to …\n",
      "\n",
      "… incubated in NeuroMab monoclonal antibodies to Kv2.1 (K89/34; 10 μg/mL; NeuroMab #73-… \n",
      "/1 or N372B/60 (for Kv2.2) antibodies (both from NeuroMab) in 4 Thy1 mice (P26–P42) (see …\n",
      "\n",
      "The Kv2 family of voltage-gated potassium channel α subunits, comprising Kv2.1 and Kv2.2, \n",
      "mediate the bulk of the neuronal delayed rectifier K + current in many mammalian central …\n",
      "\n",
      "… available from the UC Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). 27 … dilution \n",
      "of Tissue Culture Supernatant, concentration unknown N372C/51, anti-Kv2.2 mouse IgG1 mAb …\n",
      "\n",
      "The endoplasmic reticulum (ER) and plasma membrane (PM) form junctions crucial to ion \n",
      "and lipid signaling and homeostasis. The Kv2.1 ion channel is localized at ER–PM junctions …\n",
      "\n",
      "… mAbs K14/39 (IgG1) and L76/36 (IgG2a) were generated against the same GST fusion \n",
      "protein (rat Kv1.2 amino acids 428–499) used to generate K14/16, and found to bind within the …\n",
      "\n",
      "… The mouse anti-Kv 1.2 antibodies were obtained from Neuromab and have both been \n",
      "knock-out validated. The main Caspr antibody used was rabbit anti-Caspr from Abcam. The …\n",
      "\n",
      "… (f) Immunolabeling by NiDAB histochemistry using NeuroMab N425/45 TCS in a sagittal \n",
      "section through the brainstem of a rat shows prominent signal for VAChT in the same cell nuclei …\n",
      "\n",
      "Schizophrenia is associated with cognitive and behavioral dysfunctions thought to reflect \n",
      "imbalances in neurotransmission systems. Recent screenings suggested that lack of (functional…\n",
      "\n",
      "The modulation of AMPA receptor (AMPAR) content at synapses is thought to be an underlying \n",
      "molecular mechanism of memory and learning. AMPAR content at synapses is highly …\n",
      "\n",
      "Associative learning processes have an important role in the initiation and persistence of \n",
      "heroin-seeking. Here we show in a rat self-administration model that reexposure to cues …\n",
      "\n",
      "… anti-PanShank (1:100, clone N23B/49; NeuroMab, Davis, CA, USA), monoclonal mouse \n",
      "anti-GluR2 (1:100, clone L21/32; NeuroMab), or monoclonal mouse anti-gephyrin (1:1000, cat. …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "… of endogenously expressed BK channels and NMDARs with in situ PLA, we examined \n",
      "the antigen specificities of a mouse monoclonal anti-BKα antibody (L6/60; NeuroMab) and a …\n",
      "\n",
      "… no: M0761) diluted to 1:4,000, anti-AMPA-R subunit 2 (Neuro Mab, Davis, CA, USA, cat. \n",
      "no: L21/32) diluted to 1:500, biotinylated antimouse (Jackson Immunoresearch Laboratories, …\n",
      "\n",
      "Establishment, specification, and validation of synaptic connections are thought to be mediated \n",
      "by interactions between pre- and postsynaptic cell-adhesion molecules. Arguably, the …\n",
      "\n",
      "… K66/38 (IgG2a). These mAbs are available from the UC Davis/ NIH NeuroMab Facility (neuromab.… \n",
      "KChIP3 expression, as defined here by staining with the mouse mAb K66/38 in red, is …\n",
      "\n",
      "Primary astrocyte cell cultures have become a valuable tool for studies of signaling pathways \n",
      "that regulate astrocyte physiology, reactivity, and function; however, differences in culture …\n",
      "\n",
      "Regulation of intracellular calcium is crucial both for proper neuronal function and survival. \n",
      "By coupling ATP hydrolysis with Ca 2+ extrusion from the cell, the plasma membrane calcium…\n",
      "\n",
      "We report a signaling pathway linking two fundamental functions of the ER, oxidative protein \n",
      "folding, and intracellular calcium regulation. Cells sense ER oxidative protein folding …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "After contusion spinal cord injury (SCI), astrocytes become reactive and form a glial scar. \n",
      "While this reduces spreading of the damage by containing the area of injury, it inhibits …\n",
      "\n",
      "… The monoclonal antibody was generated using the same protein fragment at the UC \n",
      "Davis/NIH NeuroMab Facility (Clone N202/7). The monoclonal was purified from cell culture and …\n",
      "\n",
      "… gels and analyzed by Western blot using monoclonal anti-FIG4 antibody (NeuroMab \n",
      "clone N202/7) and rabbit polyclonal anti-actin antibody (Sigma-Aldrich, St. Louis, MO). …\n",
      "\n",
      "… gels and analyzed by Western blot using monoclonal anti-FIG4 antibody (NeuroMab \n",
      "clone N202/7) and rabbit polyclonal anti-actin antibody (Sigma-Aldrich, St. Louis, MO). …\n",
      "\n",
      "Mutations in FIG4, coding for a phosphoinositol(3,5) bisphosphate 5′ phosphatase and \n",
      "involved in vesicular trafficking and fusion, have been shown causing a recessive form of …\n",
      "\n",
      "There is growing evidence that virus particles also contain host cell proteins, which provide \n",
      "viruses with certain properties required for entry and release. A proteomic analysis …\n",
      "\n",
      "DYRKs (dual specificity, tyrosine phosphorylation regulated kinases) and CLKs (cdc2-like \n",
      "kinases) are implicated in the onset and development of Alzheimer’s disease and Down …\n",
      "\n",
      "Inhibitory neurons control the firing of glutamatergic neurons and synchronize brain activity. \n",
      "However, little is known about mechanisms of excitatory synapse formation in inhibitory …\n",
      "\n",
      "Background Postsynaptic density (PSD)-95-like membrane-associated guanylate kinases (PSD-MAGUKs) \n",
      "are scaffold proteins in PSDs that cluster signaling molecules near NMDA …\n",
      "\n",
      "10.7554/eLife.24192.001 Glutamatergic synapses rely on AMPA receptors (AMPARs) for fast \n",
      "synaptic transmission and plasticity. AMPAR auxiliary proteins regulate receptor trafficking, …\n",
      "\n",
      "The acoustic environment an animal experiences early in life shapes the structure and function \n",
      "of its auditory system. This process of experience-dependent development is thought to …\n",
      "\n",
      "… The mouse monoclocal Kv1.1 antiserum (NeuroMab; catalog No. K20/78, RRID:AB_2312366) \n",
      "was prepared against a synthetic peptide representing amino acids 458–476 (…\n",
      "\n",
      "… (K20/78) antibody (UC Davis/NIH NeuroMab Facility). All incubations were performed at 4C \n",
      "with constant agitation. Antibody-bound protein complexes were captured by the addition of …\n",
      "\n",
      "… Cells transfected with p513-KCNA1 vectors were incubated overnight at 4C with primary \n",
      "antibody anti-Kv1.1, clone K20/78 (1:50; NeuroMab) and for 1 hour at room temperature with …\n",
      "\n",
      "… For detection of Kv1.1 subunits, we used mouse monoclonal anti-Kv1.1 antibodies (clone \n",
      "K20/78; 1:1000 dilution; UC Davis/NIH NeuroMab Facility) and rabbit polyclonal anti-Kv1.1 …\n",
      "\n",
      "Purpose: Mice lacking the Kv1.1 potassium channel α subunit encoded by the Kcna1 gene \n",
      "develop recurrent behavioral seizures early in life. We examined the neuropathological …\n",
      "\n",
      "… 1C (mouse monoclonal K36/15; Neuromab 73–105), and anti-Kv1.1D (mouse \n",
      "monoclonal K20/78; Neuromab 73–007); preimmune control IgG was from rabbit (Millipore/Upstate, …\n",
      "\n",
      "For vocal communicators like humans and songbirds, survival and reproduction depend on \n",
      "highly developed auditory processing systems that can detect and differentiate nuanced …\n",
      "\n",
      "… Anti-Kv1.2 antibody was obtained from NeuroMab. Kv1.2 has a calculated MW of 56.7 kDa \n",
      "but may run at 66–88 kDa depending on the degree of posttranslational modification. On …\n",
      "\n",
      "… anti-Kvβ2 (1:50, Clone 17/70, NeuroMab, Davis, CA; http://www.neuromab.org; Bekele-Arcuri \n",
      "et al. 1996) or anti-Kv1.1 (1:50, clone K20/78, NeuroMab). After being rinsed in TBST (TBS …\n",
      "\n",
      "… Total cellular Kv1.1 was detected by cytoplasmic directed monoclonal antibody K20/78 \n",
      "following detergent permeabilization. Bound primary antibodies were detected using Alexa-…\n",
      "\n",
      "Spiral ganglion neurons (SGNs) relay acoustic code from cochlear hair cells to the brainstem, \n",
      "and their stimulation enables electrical hearing via cochlear implants. Rapid adaptation, a …\n",
      "\n",
      "Action potential (AP) responses of dentate gyrus granule (DG) cells have to be tightly regulated \n",
      "to maintain hippocampal function. However, which ion channels control the response …\n",
      "\n",
      "Precise axon pathfinding is crucial for establishment of the initial neuronal network during \n",
      "development. Pioneer axons navigate without the help of preexisting axons and pave the way …\n",
      "\n",
      "Voltage-gated Kv1.1 channels encoded by the Kcna1 gene are traditionally regarded as being \n",
      "neural-specific with no known expression or intrinsic functional role in the heart. However…\n",
      "\n",
      "… 1e mAb (K36/15; UC Davis/NIH NeuroMab Facility) before detergent permeabilization to \n",
      "detect the cell surface pool. The total cellular pools of the respective proteins were detected by …\n",
      "\n",
      "… the supplier's (NeuroMab) datasheets online. The protein bands detected match with the \n",
      "expected sizes from endogenous and heterologous expression as reported by NeuroMab. We …\n",
      "\n",
      "… Additional commercial available mouse monoclonal antibodies from NeuroMab (Anti-Kv1.1: \n",
      "K20/78, Anti-Kv1.2: K14/16, Anti-Kv1.4: K13/31 and Anti-Kv1.6: …\n",
      "\n",
      "A specialized axonal ending, the basket cell “pinceau,” encapsulates the Purkinje cell axon \n",
      "initial segment (AIS), exerting final inhibitory control over the integrated outflow of the …\n",
      "\n",
      "TDP-43 is extensively studied in neurons in physiological and pathological contexts. \n",
      "However, emerging evidence indicates that glial cells are also reliant on TDP-43 function. We …\n",
      "\n",
      "… -K V 1.1 (K20/78), mouse anti-K V 1.2 (K14/16), and mouse anti-PSD95 (K28/43) were \n",
      "obtained from University of California, Davis/NIH NeuroMab Facility. Secondary antibodies used …\n",
      "\n",
      "Cortical dysplasia (CD) is a common cause for intractable epilepsy. Hyperactivation of the \n",
      "mechanistic target of rapamycin (mTOR) pathway has been implicated in CD; however, the …\n",
      "\n",
      "The nodes of Ranvier are essential regions for action potential conduction in myelinated fibers. \n",
      "They are enriched in multimolecular complexes composed of voltage-gated Nav and Kv7 …\n",
      "\n",
      "miR-17-92 is a microRNA cluster with six distinct members. Here, we show that the miR-17-92 \n",
      "cluster and its individual members modulate chronic neuropathic pain. All cluster …\n",
      "\n",
      "In myelinated fibers, the voltage-gated sodium channels Nav1 are concentrated at the nodal \n",
      "gap to ensure the saltatory propagation of action potentials. The voltage-gated potassium …\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "Spike timing-dependent synaptic plasticity (STDP) serves as a key cellular correlate of \n",
      "associative learning, which is facilitated by elevated attentional and emotional states involving …\n",
      "\n",
      "The Kv7 (KCNQ) channel is a family of voltage-gated potassium channels that is considered \n",
      "to be important in the regulation of cellular excitability and axonal conduction. Previous …\n",
      "\n",
      "… 1–500, UC Davis/NIH NeuroMab Facility); an antibody raised … anti-VGAT clone L118/80, UC \n",
      "Davis/NIH NeuroMab Facility); … L106/83, UC Davis/NIH NeuroMab Facility); a purified mouse …\n",
      "\n",
      "… 500, UC Davis/NIH NeuroMab Facility); a monoclonal antibody … VGAT clone L118/80, 1-500, \n",
      "UC Davis/NIH NeuroMab Facility)… L106/83, UC Davis/NIH NeuroMab Facility); a monoclonal …\n",
      "\n",
      "With an incidence of ~1 in 800 births, Down syndrome (DS) is the most common chromosomal \n",
      "condition linked to intellectual disability worldwide. While the genetic basis of DS has …\n",
      "\n",
      "… N221/17; NeuroMab) antibody in blocking solution overnight at 4C. After washing in \n",
      "blocking solution (three times for 10 min each at room temperature with vigorous agitation), the …\n",
      "\n",
      "… (N221/17, NeuroMab, Davis, CA). Membranes were later stripped and reblotted with an \n",
      "anti-actin antibody at a 1:2000 dilution (66,009, Proteintech, Rosemont, IL) as a loading control (…\n",
      "\n",
      "Patients with diabetes develop endothelial dysfunction shortly after diabetes onset that \n",
      "progresses to vascular disease underlying the majority of diabetes-associated comorbidities. …\n",
      "\n",
      "Neurite outgrowth is key to the formation of functional circuits during neuronal development. \n",
      "Neurotrophins, including nerve growth factor (NGF), increase neurite outgrowth in part by …\n",
      "\n",
      "… The mouse monoclonal (clone N221/17) from NeuroMab detected bands in T84 cells, \n",
      "human prostate PC3 cells (epithelial cell line generously provided by Dr. Vander Griend, …\n",
      "\n",
      "… /ml; clone N221/17; NeuroMab, Davis, CA) or mouse monoclonal antibody against PTH1R \n",
      "(1:200; Millipore, Billerica, MA). After washing cells were incubated with goat anti-mouse IgG …\n",
      "\n",
      "Little is known about mechanical regulation of morphological and functional polarity of central \n",
      "neurons. In this study, we report that mechanical stress specifically induces varicosities in …\n",
      "\n",
      "Epilepsy is the most common neurological disorder, with over 50 million people worldwide \n",
      "affected. Recent evidence suggests that the transient receptor potential cation channel …\n",
      "\n",
      "nan\n",
      "\n",
      "… to make these validated NeuroMabs available to the research community on a nonprofit \n",
      "basis. Although this single facility, whose goal is to generate NeuroMabs against 50 targets per …\n",
      "\n",
      "… -Kv4.2 antibody (5 μg per reaction; clone K57/1; NeuroMab) or anti-protein phosphatase 2A \n",
      "(PP2A) … (1:1000; clone K57/1; NeuroMab), and the signal intensities quantified using the NIH …\n",
      "\n",
      "Kv4 channels mediate most of the somatodendritic subthreshold operating A‐type current (I \n",
      "SA ) in neurons. This current plays essential roles in the regulation of spike timing, repetitive …\n",
      "\n",
      "… using the antibodies mainly from Neuromab for IP from extracts … Recently, the Neuromab \n",
      "facility started to provide a unique … use of antibodies mainly from Neuromab to start to examine …\n",
      "\n",
      "… The NeuroMab KChIP1 antibody stained a double band with … The NeuroMab KChIP2 antibody \n",
      "stained a single band with a … The NeuroMab KChIP3 antibody stained a single band with …\n",
      "\n",
      "… (K57/1) was purchased from Neuromab and has been described previously ( … of \n",
      "California, Davis, NINDS/NIMH NeuroMab Facility. Monoclonal antibody to synaptophysin was …\n",
      "\n",
      "Voltage-gated A-type K + channel Kv4.2 subunits are highly expressed in the dendrites of \n",
      "hippocampal CA1 neurons. However, little is known about the subcellular distribution and …\n",
      "\n",
      "Objective Increased faecal butyrate levels have been reported in irritable bowel syndrome. \n",
      "Rectal instillation of sodium butyrate (NaB) increases visceral sensitivity in rats by an …\n",
      "\n",
      "… K57/1 antibody is highly specific for Kv4.2 over its closely related family members Kv4.1 and \n",
      "Kv4.3 (Rhodes et al., 2004). K57/1 … we have tested, including the K57/1 mAb, have proven …\n",
      "\n",
      "… comparison with Kv4.2 potassium channel (K57/1, IgG1) in the cerebellum where both are \n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and …\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "… A mouse monoclonal anti-Kv4.2 antibody, NeuroMab clone K57/1 (mouse strain, Balb/C; \n",
      "myeloma cell, SP2/0), was obtained from the UC Davis/NIH NeuroMab Facility (Davis, CA). The …\n",
      "\n",
      "… To observe the internalization of Kv4.2 in NRVMs, surface-expressed Kv4.2 in NRVMs were \n",
      "labeled with anti-Kv4.2 antibody (K57/1; 1:200; NeuroMab) for 30 min at 4C, and washed …\n",
      "\n",
      "… An antibody targeting an extracellular Kv4.2 epitope (K57/1, Neuromab) effectively \n",
      "detected total Kv4.2 expression in permeabilized cells but inadequately stained Kv4.2 at the …\n",
      "\n",
      "… The specificity of each of the primary antibodies has been verified using Western blots \n",
      "from brain tissue by the UC, Davis/NeuroMab Facility and also in our laboratory in BLA tissue …\n",
      "\n",
      "… As representative examples, the catalog of the UC Davis/NIH NeuroMab facility (as of 12/17/11) \n",
      "contains 278 mAbs, and a representative part of the commercial catalog of EMD …\n",
      "\n",
      "… Immunodetections of K v 4.2 and K v 4.3 channels were performed using monoclonal \n",
      "antibodies from Neuro Mab (clones K57/1 and K75/41, respectively). These antibodies have been …\n",
      "\n",
      "Widely expressed in the adult central nervous system, the cellular prion protein (PrP C ) is \n",
      "implicated in a variety of processes, including neuronal excitability. Dipeptidyl aminopeptidase…\n",
      "\n",
      "To gain insights into the phenotype of voltage‐gated potassium (Kv)1.1 and Kv4.2 knockout \n",
      "mice, we used immunohistochemistry to analyze the expression of component principal or α …\n",
      "\n",
      "Patients with diabetes exhibit significantly altered renin-angiotensin system (RAS) control. \n",
      "Recently, it has been determined that hyperglycemic conditions induce an increase in …\n",
      "\n",
      "Potassium channels regulate numerous aspects of neuronal excitability, and several voltage-gated \n",
      "K + channel subunits have been identified in pyramidal neurons of rat neocortex. …\n",
      "\n",
      "… Rabbit anti-DPP6 antibodies and mouse monoclonal anti-Kv4.2; K57/1 and anti-Kv4.3 \n",
      "K75/41 Neuromab; www.neuromab.org were used for immunolocalization experiments in brain. …\n",
      "\n",
      "… Labeling was performed using mouse monoclonal anti-K v 4.2 (K57/1; University of \n",
      "California at Davis, National Institutes of Health NeuroMab facility, Davis, CA) dilution 1:500 and …\n",
      "\n",
      "… from NeuroMab (UC Davis/NIH, Davis, CA, USA): (1) monoclonal anti-Kv4.2 (K57/1; amino … \n",
      "mice have been described by the manufacturer (http://neuromab.ucdavis.edu/catalog.cfm, UC …\n",
      "\n",
      "Modulation of voltage-gated potassium (Kv) channel surface expression can profoundly \n",
      "affect neuronal excitability. Some but not all mammalian Shaker or Kv1 α subunits contain a …\n",
      "\n",
      "The Kv4 family of voltage-gated K + channels underlie the fast transient (A-type) outward K \n",
      "+ current. Although A-type currents are critical to determine somato-dendritic integration in …\n",
      "\n",
      "What percentage of the protein function is required to prevent disease symptoms is a \n",
      "fundamental question in genetic disorders. Decreased transsynaptic LGI1-ADAM22 protein …\n",
      "\n",
      "The transient, A-type K + current (I A ) controls the excitability of CA1 pyramidal neuron \n",
      "dendrites by regulating the back-propagation of action potentials and by shaping synaptic input. …\n",
      "\n",
      "The subthreshold voltage-gated transient K + current (I A ) carried by pore-forming Kv4.2 \n",
      "subunits regulates the propagation of synaptic input, dendritic excitability, and synaptic …\n",
      "\n",
      "Molecular studies examining the impact of mitochondrial morphology on the mammalian \n",
      "heart have previously focused on dynamin related protein-1 (Drp-1) and mitofusin-2 (Mfn-2), …\n",
      "\n",
      "Mitochondrial dysfunction has been implicated in the pathogenesis of type 2 diabetes. \n",
      "Identifying novel regulators of mitochondrial bioenergetics will broaden our understanding of …\n",
      "\n",
      "RNA splicing entails the coordinated interaction of more than 150 proteins in the spliceosome, \n",
      "one of the most complex of the cell's molecular machines. We previously discovered that …\n",
      "\n",
      "Kainate receptors are members of the glutamate receptor family that regulate synaptic function \n",
      "in the brain. They modulate synaptic transmission and the excitability of neurons; however…\n",
      "\n",
      "(論文の内容の要旨)[背景と目的] シナプス後肥厚部 (postsynaptic density, PSD) とはシナプス膜\n",
      "直下にある, 神経細胞に高度に分化した特殊な細胞骨格系で, 極めて精巧なタンパク質複合体である…\n",
      "\n",
      "It is essential to study the molecular architecture of post‐synaptic density ( PSD ) to understand \n",
      "the molecular mechanism underlying the dynamic nature of PSD , one of the bases of …\n",
      "\n",
      "Accumulating genetic evidence suggests that schizophrenia (SZ) is associated with individually \n",
      "rare copy number variations (CNVs) of diverse genes, often specific to single cases. …\n",
      "\n",
      "Depolarization of neurons in 3-week-old rat hippocampal cultures promotes a rapid increase \n",
      "in the density of surface NMDA receptors (NRs), accompanied by transient formation of …\n",
      "\n",
      "How does chronic activity modulation lead to global remodeling of proteins at synapses and \n",
      "synaptic scaling? Here we report that guanylate kinase–associated protein (GKAP; also …\n",
      "\n",
      "Synaptic plasticity, the cellular basis of learning and memory, involves the dynamic \n",
      "trafficking of AMPA receptors (AMPARs) into and out of synapses. One of the remaining key …\n",
      "\n",
      "LRRTMs are postsynaptic cell adhesion proteins that have region-restricted expression in the \n",
      "brain. To determine their role in the molecular organization of synapses in vivo, we studied …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "… by UC Davis/NIH NeuroMab Facility (Davis, CA, … N121A/1. Using a library of overlapping \n",
      "peptides covering the amino acid sequence of Panx2, we mapped the epitope of clone N121A/1 …\n",
      "\n",
      "Endomembrane specialization allows functional compartmentalization but imposes physical \n",
      "constraints to information flow within the cell. However, the evolution of an endomembrane …\n",
      "\n",
      "Presynaptic neurexins (Nrxs) and type IIa receptor-type protein tyrosine phosphatases (RPTPs) \n",
      "organize synapses through a network of postsynaptic ligands. We show that leucine-rich-…\n",
      "\n",
      "LRRTM4 is a transsynaptic adhesion protein regulating glutamatergic synapse assembly on \n",
      "dendrites of central neurons. In the mouse retina, we find that LRRTM4 is enriched at …\n",
      "\n",
      "The four members of the LRRTM family (LRRTM1-4) are postsynaptic adhesion molecules \n",
      "essential for excitatory synapse development. They have also been implicated in …\n",
      "\n",
      "… (N29/29), VGLUT3 (N34/34), as well as the mouse monoclonal antibody for VAChT (N6/38) \n",
      "were purchased from Antibodies Incorporated (UC Davis/NIH NeuroMab Facility, Davis, CA). …\n",
      "\n",
      "Glutamate and its receptor N-methyl-D-aspartate receptor (NMDAR) have been associated \n",
      "with cancer, although their functions are not fully understood. Herein, we implicate glutamate-…\n",
      "\n",
      "A hallmark of inflammatory diseases is the excessive recruitment and influx of monocytes to \n",
      "sites of tissue damage and their ensuing differentiation into macrophages. Numerous stimuli …\n",
      "\n",
      "… monoclonal antibodies that specifically target QKI-5 (N195A/16), QKI-6 (N182/17), or QKI-7 \n",
      "(N183/15.1; UC Davis/NIH NeuroMab Facility). Mouse monoclonal antibodies were used to …\n",
      "\n",
      "Alternative splicing is critical for animal ontogeny; however, its role in the earliest developmental \n",
      "decision, the specification of the three embryonic germ layers, is poorly understood. By …\n",
      "\n",
      "… (APC-021; d) and a mouse monoclonal (L6/23; e) antibody against the BK channel. f, The … \n",
      "(APC-021 recognizes an epitope between amino acid residues 1089 and 1196, and L6/23 …\n",
      "\n",
      "… (APC-021; d) and a mouse monoclonal (L6/23; e) antibody against the BK channel. f, The … \n",
      "(APC-021 recognizes an epitope between amino acid residues 1089 and 1196, and L6/23 …\n",
      "\n",
      "… N121A/1 and N121A/31) were generated by UC Davis/NIH NeuroMab Facility (Davis, CA, … \n",
      "Panx2 monoclonal antibodies (N121A/1 and N121A/31) by showing that both clones identified …\n",
      "\n",
      "Developmental and epileptic encephalopathies (DEE) are a group of severe epilepsies that \n",
      "usually present with intractable seizures, developmental delay, and often have elevated risk …\n",
      "\n",
      "… (1:500; KC; RRID: AB_2315767) and as loading controls mouse anti-mortalin (1:2, N52A/42 \n",
      "tissue culture supernatant; RRID: AB_10674108, from the UC Davis/NIH NeuroMab Facility) …\n",
      "\n",
      "Voltage-gated K + (Kv) channels play important roles in regulating neuronal excitability. Kv \n",
      "channels comprise four principal α subunits, and transmembrane and/or cytoplasmic …\n",
      "\n",
      "The plasma membrane (PM) comprises distinct subcellular domains with diverse functions \n",
      "that need to be dynamically coordinated with intracellular events, one of the most impactful …\n",
      "\n",
      "… (K7/45) and Mortalin/Grp75 (N52A/42) using monoclonal antibodies (clone number in \n",
      "parentheses) obtained from the UC Davis/NIH NeuroMab Facility. Alexa-conjugated fluorescent …\n",
      "\n",
      "… N52A/42; NeuroMab), and subsequently with goat anti-mouse or anti-rabbit IgG–HRP \n",
      "secondary antibodies (1:10,000; Antibodies Inc.). Immunoreactive proteins on membranes were …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… The plasmid backbone was generated by EcoRI/HindIII restriction digestion of the N52A/42 …\n",
      "\n",
      "… , including at the UC Davis/NIH NeuroMab Facility, we have generated a large collection of \n",
      "… The plasmid backbone was generated by EcoRI/HindIII restriction digestion of the N52A/42 …\n",
      "\n",
      "… 1b (N16B/8, NeuroMab, San Diego, CA), rabbit polyclonal anti-Kv3.1b (laboratory-generated), \n",
      "mouse monoclonal anti-mortalin (N52A/42, NeuroMab), mouse monoclonal anti-HA tag (…\n",
      "\n",
      "The disc-large (DLG)–membrane-associated guanylate kinase (MAGUK) family of proteins \n",
      "forms a central signaling hub of the glutamate receptor complex. Among this family, some …\n",
      "\n",
      "Dynamic modulation of ion channel expression, localization, and/or function drives plasticity \n",
      "in intrinsic neuronal excitability. Voltage-gated Kv2.1 potassium channels are constitutively …\n",
      "\n",
      "… The mouse mAbs N52A/42, K13/31, K89/34, L83/11, L86/33, and L86A/37 are available \n",
      "from the UC Davis/NIH NeuroMab Facility (http://neuromab.ucdavis.edu/), a not-for profit …\n",
      "\n",
      "… (N372B/60) and Grp75/Mortalin (N52A/42) using mAbs (clone number in parentheses) \n",
      "obtained from the UC-Davis/NIH NeuroMab Facility (http://neuromab.ucdavis.edu/), a not-for profit …\n",
      "\n",
      "… Mouse Mono K89/34 Amino acids 837–853 ratHMLPGGGAHGSTRDQSI NeuroMab 75-014 … \n",
      "Ankyrin-G Mouse Mono N106/65 full length human NeuroMab 75–147 Purified IgG 10 μg/…\n",
      "\n",
      "… encoding functional recombinant K89/34 mAb (IgG2a). Recombinant K89/34 was produced \n",
      "… Sections were labeled free floating and probed with K89/34 IgG1 antibody diluted in vehicle …\n",
      "\n",
      "… Briefly, Kv2.1 and Kv2.2 expression was characterized using the K89/34 (for Kv2.1) and \n",
      "N372B/1 or N372B/60 (for Kv2.2) antibodies (both from NeuroMab) in 4 Thy1 mice (P26–P42) (…\n",
      "\n",
      "… (K89/34). Sections were then incubated with fluorescein isothiocyanate-conjugated or cyanine \n",
      "3-conjugated secondary antibodies (Jackson ImmunoResearch, West Grove, PA). Images …\n",
      "\n",
      "… K89/34 (green) antibodies in cerebral cortex of CO 2 -treated (2 min) mouse… (K89/34 and \n",
      "K39/25), Kv1.2 (K14/16) and PSD-95 (K28/43) were obtained from the UC Davis/NIH NeuroMab …\n",
      "\n",
      "… Neurons were then incubated with the Kv2.1 primary antibody K89/34 (NeuroMab) and a \n",
      "GFP primary antibody (A10262, Invitrogen) overnight at 4 C. Cells were washed three times …\n",
      "\n",
      "Lysosomes communicate through cholesterol transfer at endoplasmic reticulum (ER) contact \n",
      "sites. At these sites, the Niemann Pick C1 cholesterol transporter (NPC1) facilitates the …\n",
      "\n",
      "… Similarly, an anti-Kv2.1 mouse mAb (IgG1 mAb K89/34 in green) only labels the Kv2.1 pool \n",
      "in RBM, and the sample from heterologous cells expressing recombinant Kv2.1. Note that …\n",
      "\n",
      "Altered neuronal excitability is emerging as an important feature in Alzheimer’s disease (AD). \n",
      "Kv2. 1 potassium channels are important modulators of neuronal excitability and synaptic …\n",
      "\n",
      "… We found the Kv8.2 antibody (Clone N448/88 from NeuroMab) generated a non-specific \n",
      "signal in immunohistochemistry (Supplementary Material, Fig. S2). Courtesy of James Trimmer, …\n",
      "\n",
      "… As a positive control, cells colabeled with two distinct Ca V 1.2 antibodies [rabbit FP1 (28) \n",
      "and monoclonal NeuroMab clone N263/31] showed surface membrane staining (fig. S1A) and …\n",
      "\n",
      "… These mAbs are available from the UC Davis/ NIH NeuroMab Facility (neuromab.ucdavis.… \n",
      "(determined by staining with the mouse mAb K89/34), and the Slo1 large conductance calcium…\n",
      "\n",
      "Voltage-gated sodium (Nav) channels initiate action potentials in brain neurons and are \n",
      "primary therapeutic targets for anti-epileptic drugs controlling neuronal hyperexcitability in …\n",
      "\n",
      "In arterial myocytes, the canonical function of voltage-gated Ca V 1.2 and KV 2.1 channels \n",
      "is to induce myocyte contraction and relaxation through their responses to membrane …\n",
      "\n",
      "The Kv2 channels encode delayed rectifier currents that regulate membrane potential in \n",
      "many tissues. They also have a non-conducting function to form stable junctions between the …\n",
      "\n",
      "… K89/34 and Kv1.2 K14/16 were obtained from UC Davis/NIH NeuroMab Facility (Western \n",
      "blotting). Kv2.1 (APC-012) was obtained from Alomone Labs (immunohistochemistry). …\n",
      "\n",
      "… available from the UC Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). 27 … RRID:AB_2571516 \n",
      "1:3 dilution of Tissue Culture Supernatant, concentration unknown K89/34, antiKv2.…\n",
      "\n",
      "The distinct organization of Kv2 voltage-gated potassium channels on and near the cell body \n",
      "of brain neurons enables their regulation of action potentials and specialized membrane …\n",
      "\n",
      "… ) using a monoclonal antibody specific for K V 2.1 subunits (NeuroMab; 75-159; clone \n",
      "K39/25, University of California, Davis). The secondary antibody was an Alexa Fluor 488-…\n",
      "\n",
      "… experiments, we used the K89/34 mouse IgG1 antibody (NeuroMab; mAb binds within aa … \n",
      ", we used the N372B/1 mouse IgG1 antibody (NeuroMab; mAb binds within aa 764 −907, cat#…\n",
      "\n",
      "Diverse classes of voltage-gated potassium channels (Kv) are integral to the variety of \n",
      "electrical activity patterns that distinguish different classes of neurons in the brain. A feature of …\n",
      "\n",
      "Potassium channels regulate numerous aspects of neuronal excitability, and several voltage-gated \n",
      "K + channel subunits have been identified in pyramidal neurons of rat neocortex. …\n",
      "\n",
      "The axon initial segment (AIS) is the site of action potential generation and a locus of activity-dependent \n",
      "homeostatic plasticity. A multimeric complex of sodium channels, linked via a …\n",
      "\n",
      "… Acknowledgements: Most of the antibodies used in this study are available from the UC \n",
      "Davis/NIH NeuroMab Facility (neuromab.ucdavis.edu). Correspondence should be addressed to …\n",
      "\n",
      "… Cultured neocortical neurons (14–16 DIV) were fixed and stained with monoclonal \n",
      "antibodies (NeuroMAB) against Kv2.1, Cav1.3, and Cav1.2, and Alexa-conjugated secondary …\n",
      "\n",
      "By studying healthy women who do not request analgesia during their first delivery, we \n",
      "investigate genetic effects on labor pain. Such women have normal sensory and psychometric …\n",
      "\n",
      "… (clone K89/34; NeuroMab); rabbit polyclonal against NRG1 (SC-348; Santa Cruz \n",
      "Biotechnology); mouse monoclonal against V5 tag (clone SV-Pk1; Bio-Rad); mouse monoclonal …\n",
      "\n",
      "The endoplasmic reticulum (ER) and plasma membrane (PM) form junctions crucial to ion \n",
      "and lipid signaling and homeostasis. The Kv2.1 ion channel is localized at ER–PM junctions …\n",
      "\n",
      "Myosin Va (MyoVa) is a plus-end filamentous-actin motor protein that is highly and broadly \n",
      "expressed in the vertebrate body, including in the nervous system. In excitatory neurons, …\n",
      "\n",
      "… For co-immunoprecipitation experiments, 50 μl of protein G-coated magnetic beads (Life \n",
      "Technologies) were incubated (4 C, overnight with nutation) with 10 μg of anti-Kvβ2 (NeuroMab…\n",
      "\n",
      "… The commercial antibody used was from NeuroMab, Davis, CA, USA, GABA A α1 antibody \n",
      "(N95/35, … Gabra6-a antibody and Gabrg2 antibody (Neuromab, Davis, CA, USA, N452/69, …\n",
      "\n",
      "Latrophilin-2 (Lphn2) and latrophilin-3 (Lphn3) are adhesion GPCRs that serve as postsynaptic \n",
      "recognition molecules in CA1 pyramidal neurons of the hippocampus, where they are …\n",
      "\n",
      "Although synaptogenesis within the retina is obviously essential for vision, mechanisms \n",
      "responsible for the initiation and maintenance of retinal synapses are poorly understood. In …\n",
      "\n",
      "… using the antibodies mainly from Neuromab for IP from extracts … Recently, the Neuromab \n",
      "facility started to provide a unique … use of antibodies mainly from Neuromab to start to examine …\n",
      "\n",
      "The cellular and molecular mechanisms underlying photoreceptor synaptogenesis are poorly \n",
      "understood. Furthermore, a detailed picture of the molecular composition of photoreceptor …\n",
      "\n",
      "Leukocyte common antigen-related receptor tyrosine phosphatases (LAR-RPTPs) are \n",
      "evolutionarily conserved presynaptic organizers. The synaptic role of vertebrate LAR-RPTPs in …\n",
      "\n",
      "BIN1, a member of the BAR adaptor protein family, is a significant late-onset Alzheimer \n",
      "disease risk factor. Here, we investigate BIN1 function in the brain using conditional knockout (…\n",
      "\n",
      "… ); mouse monoclonal anti-CASK (clone K56A/50; NeuroMab, RRID:AB_2068730), mouse \n",
      "monoclonal anti-PSD-95 (clone K28/43; NeuroMab, RRID:AB_2307331), and mouse …\n",
      "\n",
      "… monoclonal anti-PSD-95 (clone K28/43; NeuroMab); mouse monoclonal anti-β-actin (clone … \n",
      "mouse monoclonal anti-CASK (clone K56A/50; NeuroMab). The following antibodies have …\n",
      "\n",
      "… Davis/NIH NeuroMab Facility; K14/16), mouse anti-CASK (University of California Davis/NIH \n",
      "NeuroMab Facility; K56A/50), mouse anti-Ankyrin G (University of California Davis/NIH …\n",
      "\n",
      "Leukocyte common antigen-related receptor tyrosine phosphatases (LAR-RPTPs) are \n",
      "evolutionarily conserved presynaptic organizers. The synaptic role of vertebrate LAR-RPTPs in …\n",
      "\n",
      "The cellular and molecular mechanisms underlying photoreceptor synaptogenesis are \n",
      "poorly understood. In fact, there is no clear picture of the molecular composition of a typical …\n",
      "\n",
      "Myelinated neurons are organized into distinct molecular domains that are essential for \n",
      "electrical conduction. These domains include the axon initial segment, nodes of Ranvier, …\n",
      "\n",
      "… 2+ channel (NeuroMab clone L48A/9) directed against amino acids 859–875 in the N terminus \n",
      "of rat Ca v 1.3, and 10) monoclonal anti-Ca v 1.3 Ca 2+ channel (NeuroMab clone N38/8) …\n",
      "\n",
      "… L57/46,) or commercially available anti-Ca v 1.3 (Alo: Alomone Labs, ACC-005; NM: Neuromab \n",
      "Antibodies Inc. N38/8) antibodies. Blots were developed using Amersham ECL Plus and …\n",
      "\n",
      "… Specificity of the Ca V 1 subtype antibodies was determined by the supplier (NeuroMab) \n",
      "by immunohistochemistry in brain tissue and by western blot. They have been used (http://…\n",
      "\n",
      "… , 1∶4 dilution) or mouse anti-α1D (clone N38/8, #73-080, lot 437-4VA-10, 1∶4 dilution) \n",
      "obtained from the UC Davis/NIH NeuroMab Facility – mixed into TST containing 5% w/v nonfat …\n",
      "\n",
      "… Furthermore, immunostaining of Ca v 1.3α 1 using our and commercial (Alomone, Millipore, \n",
      "Neuromab) antibodies was conducted under two different fixation conditions, brief 2% …\n",
      "\n",
      "Intracellular chloride concentration ([Cl − ] i ) in pancreatic β-cells is kept above electrochemical \n",
      "equilibrium due to the predominant functional presence of Cl − loaders such as the Na + …\n",
      "\n",
      "Proteins of the major histocompatibility complex class I (MHCI) are known for their role in \n",
      "immunity and have recently been implicated in long-term plasticity of excitatory synaptic …\n",
      "\n",
      "Copper (Cu), an essential trace element present throughout the mammalian nervous system, \n",
      "is crucial for normal synaptic function. Neuronal handling of Cu is poorly understood. We …\n",
      "\n",
      "… ; NeuroMab) diluted 1:10; a mouse monoclonal supernatant NR2B glutamate receptor antibody \n",
      "(clone N59/20; NeuroMab) … receptor antibody (clone N308/48; NeuroMab) diluted 1:10; a …\n",
      "\n",
      "Brain radiotherapy is frequently used successfully to treat brain tumors. However, radiotherapy \n",
      "is often associated with declines in short-term and long-term memory, learning ability, …\n",
      "\n",
      "Postsynaptic density (PSD) fractions were isolated from rat forebrain and sonicated. Pellets \n",
      "from sonicated samples examined by electron microscopy revealed particles with an electron …\n",
      "\n",
      "… N59/20; Neuromab), anti-GABA-B1 antibody (1:500; rabbit monoclonal antibody raised \n",
      "against the GABA-B1 CTD; generous gift from Julie Kniazeff, Montpellier, France), anti-αTubuline …\n",
      "\n",
      "The proper function of neural circuits requires spatially and temporally balanced development \n",
      "of excitatory and inhibitory synapses. However, the molecular mechanisms coordinating …\n",
      "\n",
      "Autism spectrum disorder (ASD) is a heritable, common neurodevelopmental disorder with \n",
      "diverse genetic causes. Several studies have implicated protein synthesis as one among …\n",
      "\n",
      "The axon initial segment (AIS) is a specialized region in neurons that encompasses two \n",
      "essential functions, the generation of action potentials and the regulation of the axodendritic …\n",
      "\n",
      "Interleukin-6 (IL-6) is an important neuroimmune factor that is increased in the brain by alcohol \n",
      "exposure/withdrawal and is thought to play a role in the actions of alcohol on the brain. …\n",
      "\n",
      "… anti-Caspr, clone K65/35, NeuroMab, 1:2000 dilution; rabbit polyclonal antimyelin … clone \n",
      "N206A/8, NeuroMab, 1:2000 dilution; mouse monoclonal anti-PSD95, clone K28/43, NeuroMab, …\n",
      "\n",
      "… ), glial fibrillary acidic protein (GFAP, N206A/8), and Nav1.6 (K87/10A) antibodies were \n",
      "purchased from the UC Davis/NIH funded Neuromab monoclonal antibody resource. Mouse pan-…\n",
      "\n",
      "Parvalbumin (PV)-expressing basket interneurons in the prefrontal cortex (PFC) regulate \n",
      "pyramidal cell firing, synchrony, and network oscillations. Yet, it is unclear how their perisomatic …\n",
      "\n",
      "The region surrounding the central canal (CC) of the spinal cord is a highly plastic area, \n",
      "defined as a postnatal neurogenic niche. Within this region are ependymal cells that can …\n",
      "\n",
      "Chronic pain, broadly classified as inflammatory (caused by tissue damage) or neuropathic (caused \n",
      "by nerve damage), affects approximately 100 million Americans and only a minority …\n",
      "\n",
      "Among all voltage-gated potassium (Kv) channels, Kv2 channels are the most widely \n",
      "expressed in the mammalian brain. However, studying Kv2 in neurons has been challenging …\n",
      "\n",
      "… Anti-Kv1.2 antibody was obtained from NeuroMab. Kv1.2 has a calculated MW of 56.7 kDa \n",
      "but may run at 66–88 kDa depending on the degree of posttranslational modification. On …\n",
      "\n",
      "Numerous studies have documented the effects of chronic N-methyl-D-aspartate receptor (NMDAR) \n",
      "blockade on excitatory circuits, but the effects on inhibitory circuitry are not well …\n",
      "\n",
      "10.7554/eLife.12661.001 Neuropathic pain following peripheral nerve injury is associated \n",
      "with hyperexcitability in damaged myelinated sensory axons, which begins to normalise over …\n",
      "\n",
      "Neuropathic pain following peripheral nerve injury is associated with hyperexcitability in \n",
      "damaged myelinated sensory axons, which begins to normalise over time. We investigated the …\n",
      "\n",
      "Neuropathic pain (NP) embraces a broad range of conditions linked with a disease or \n",
      "lesion of the peripheral or central somatosensory system and its prevalence in the general …\n",
      "\n",
      "Neuropathic pain (NP) embraces a broad range of conditions linked with a disease or \n",
      "lesion of the peripheral or central somatosensory system and its prevalence in the general …\n",
      "\n",
      "… Additional commercial available mouse monoclonal antibodies from NeuroMab (Anti-Kv1.1: … \n",
      "K19/36) were used. Whereas monoclonal antibodies (mAntiKv1.x) mAnti-Kv1.1, mAnti-Kv1.6 …\n",
      "\n",
      "Autosomal dominant lateral temporal epilepsy (ADLTE) is an inherited syndrome caused by \n",
      "mutations in the leucine-rich glioma inactivated 1 (LGI1) gene. It is known that glutamatergic …\n",
      "\n",
      "… primary antibodies anti-GluN1 1:1000 (NeuroMab clone N308/48), anti-GluN2A 1:500 (NeuroMab \n",
      "clone N327/95) or anti-GluN2B 1:1000 (NeuroMab clone N59/36), and then washed …\n",
      "\n",
      "The efficient knock-in of large DNA fragments to label endogenous proteins remains especially \n",
      "challenging in non-dividing cells such as neurons. We developed Targeted Knock-In with …\n",
      "\n",
      "Excessive stimulation of NMDA receptors with glutamate or other potent agonists such as \n",
      "NMDA leads to excitotoxicity and neural injury. In this study, we aimed to provide insight into …\n",
      "\n",
      "Membrane-associated guanylate kinases (MAGUKs), including SAP102, PSD-95, PSD-93, \n",
      "and SAP97, are scaffolding proteins for ionotropic glutamate receptors at excitatory synapses…\n",
      "\n",
      "Proteins derived from the Kalrn gene, encoding 2 Rho guanine nucleotide exchange factor (GEF) \n",
      "domains, affect dendritic and axonal morphogenesis. The roles of endogenous Kalirin-…\n",
      "\n",
      "10.7554/eLife.04287.001 Lrp4, the muscle receptor for neuronal Agrin, is expressed in the \n",
      "hippocampus and areas involved in cognition. The function of Lrp4 in the brain, however, is …\n",
      "\n",
      "… The anti-GluN1 (clone N308/48) and anti-GluA1 (N355/1) monoclonal antibodies were \n",
      "obtained from the UC Davis/NIH NeuroMab Facility. …\n",
      "\n",
      "… F/F ) spinal cords (AnkG, N106/36). Two mice per genotype and more than … (N106/36 and \n",
      "N106/65, NeuroMab) (1:200), Caspr (K65/35, NeuroMab) (1:200), contactin (K73/20, NeuroMab…\n",
      "\n",
      "The node of Ranvier is a functionally important site on the myelinated axon where sodium \n",
      "channels are clustered and regeneration of action potentials occurs, allowing fast saltatory …\n",
      "\n",
      "… Mouse monoclonal anti-AnkG (N106/36; RRID: AB_10697718) and anti-GST (N100/13; \n",
      "RRID:AB_10671817) antibodies were purchased from UC Davis and the NIH NeuroMab facility. …\n",
      "\n",
      "… ankyrin-G (AnkG; NeuroMab, Davis, CA; N106/36, 1:200) and the urn:x-wiley:08941491:media:glia22991:glia22991-math-0003 \n",
      "channel isoform 1.6 (Na v 1.6; NeuroMab; K87A/10, 1:…\n",
      "\n",
      "The precise and remarkable subdivision of myelinated axons into molecularly and functionally \n",
      "distinct membrane domains depends on axoglial junctions that function as barriers. …\n",
      "\n",
      "Introduction Peripheral nerves are often exposed to mechanical stress leading to compression \n",
      "neuropathies. The pathophysiology underlying nerve dysfunction by chronic compression …\n",
      "\n",
      "Axonal ion channels control spike initiation and propagation along the axon and determine \n",
      "action potential waveform. We show here that functional suppression of axonal Kv1 channels …\n",
      "\n",
      "Background Tripartite motif (TRIM46) is a relatively novel protein that belongs to tripartite \n",
      "motif family. TRIM46 organizes parallel microtubule arrays on the axons, which are important …\n",
      "\n",
      "… The following other primary antibodies were used: mouse monoclonal anti-ankG for \n",
      "immunostaining (N106/36, UC Davis/NIH NeuroMab facility, Davis, CA, USA); rabbit anti-GAPDH (…\n",
      "\n",
      "The axon initial segment (AIS) is a highly specialized neuronal compartment that plays a key \n",
      "role in neuronal development and excitability. It concentrates multiple membrane proteins …\n",
      "\n",
      "TDP-43 is extensively studied in neurons in physiological and pathological contexts. \n",
      "However, emerging evidence indicates that glial cells are also reliant on TDP-43 function. We …\n",
      "\n",
      "… (1:100, Neuromab K36/15) and mouse IgG2a anti-ankyrinG (1:100, Neuromab N106/36). \n",
      "After extensive washing in incubation buffer, the secondary antibodies, Alexa 568 goat anti-…\n",
      "\n",
      "Axon formation, the initial step in establishing neuronal polarity, critically depends on local \n",
      "microtubule reorganization and is characterized by the formation of parallel microtubule …\n",
      "\n",
      "Axon initial segments (AISs) are specialized neuronal compartments crucial for action potential \n",
      "generation and neuronal polarity. However, a detailed understanding of the mechanisms …\n",
      "\n",
      "… N18/28, University of California Davis/NIH NeuroMab Facility), mouse anti-Ankyrin G (clone \n",
      "N106/36, University of California Davis/NIH NeuroMab Facility), guinea pig anti-VGlut2 (…\n",
      "\n",
      "Establishment of neuronal polarity depends on local microtubule (MT) reorganization. The \n",
      "endoplasmic reticulum (ER) consists of cisternae and tubules and, like MTs, forms an …\n",
      "\n",
      "Brain enriched voltage-gated sodium channel (VGSC) Na v 1.2 and Na v 1.6 are critical for \n",
      "electrical signaling in the CNS. Previous studies have extensively characterized cell-type-…\n",
      "\n",
      "Dysregulated neuronal excitability is a hallmark of amyotrophic lateral sclerosis (ALS). We \n",
      "sought to investigate how functional changes to the axon initial segment (AIS), the site of …\n",
      "\n",
      "Core planar cell polarity (PCP) genes, which are involved in various neurodevelopmental \n",
      "disorders such as neural tube closure, epilepsy, and autism spectrum disorder, have poorly …\n",
      "\n",
      "… mouse monoclonal anti-ankyrin G antibody (clone N106/36) and Caspr antibody (clone \n",
      "K65/35) were from University of California at Davis/NIH NeuroMab Facility (Davis, CA). Rabbit …\n",
      "\n",
      "… The primary antibodies used were: mouse IgG2b anti-K v 1.1 (1:100, Neuromab K36/15) \n",
      "and mouse IgG2a anti-ankyrinG (1:100, Neuromab N106/36). After extensive washing in …\n",
      "\n",
      "Alzheimer’s disease (AD) is a devastating neurodegenerative disorder characterized by \n",
      "pathological brain lesions and a decline in cognitive function. β-Amyloid peptides (Aβ), derived …\n",
      "\n",
      "… were used: mouse monoclonal ankyrinG (N106/36, UC Davis/NIH NeuroMab Facility Cat# \n",
      "75-146 RRID:AB_10673030), Kv1.2 (K14/16, UC Davis/NIH NeuroMab Facility Cat# 75-008 …\n",
      "\n",
      "The axon initial segment ( AIS ) is a specialized domain essential for neuronal function, the \n",
      "formation of which begins with localization of an ankyrin‐G (AnkG) scaffold. However, the …\n",
      "\n",
      "… Coverslips were then placed in primary antibody, made in 2% GS, to relevant concentrations: \n",
      "mouse monoclonal anti-ankyrin-G (clone N106/36, 1:500; catalog #75-146; NeuroMab); …\n",
      "\n",
      "Axon initial segments (AISs) generate action potentials and regulate the polarized distribution \n",
      "of proteins, lipids, and organelles in neurons. While the mechanisms of AIS Na + and K + …\n",
      "\n",
      "… Other mouse monoclonal antibodies used were sourced as follows: AnkG (N106/36, UC \n",
      "Davis/NIH NeuroMab facility, 1:250), AnkB (N105/17 and N105/13, UC Davis/NIH NeuroMab …\n",
      "\n",
      "P2X 7 receptors (P2X 7 Rs) are associated with numerous pathophysiological mechanisms, \n",
      "and this promotes them as therapeutic targets for certain neurodegenerative conditions. …\n",
      "\n",
      "… clone N106/20 (western blot) and clone N106/36 (immunocytochemistry) from Neuromab \n",
      "(Davis, CA, United States), PSD-95 (Neuromab), turboGFP mAb (to identify RNAi expression, …\n",
      "\n",
      "The axon initial segment (AIS) is the site of action potential (AP) initiation, thus a crucial \n",
      "regulator of neuronal activity. In excitatory pyramidal neurons, the high density of voltage-gated …\n",
      "\n",
      "Neuronal GABAergic responses switch from excitatory to inhibitory at an early postnatal \n",
      "period in rodents. The timing of this switch is controlled by intracellular Cl− concentrations, but …\n",
      "\n",
      "… Mouse anti-AnkG (clone N106/36 and clone N106/20) and anti-Pan-Neurofascin (cloneA12/18) \n",
      "were from Neuromab. Mouse anti-EB1 (clone 5/EB1), and mouse anti-Rab11 (clone 47/…\n",
      "\n",
      "TMEM106B, a gene encoding a lysosome membrane protein, is tightly associated with \n",
      "brain aging, hypomyelinating leukodystrophy, and multiple neurodegenerative diseases, …\n",
      "\n",
      "… (1:200, Neuromab #75-105, clone K36/15, external), and mouse anti-ankyrin G (1:200, \n",
      "Neuromab #75-146, clone N106/36). The following secondary antibodies were used for …\n",
      "\n",
      "Axon initial segments (AISs) initiate action potentials and regulate the trafficking of vesicles \n",
      "between somatodendritic and axonal compartments. However, the mechanisms controlling …\n",
      "\n",
      "… The following primary antibodies were used: mouse ankyrin-G (Neuromab; N106/36), \n",
      "rabbit ankyrin-G (SantaCruz; sc-28561), rabbit Homer1b/c (Antibodies-online; ABIN-1742339), …\n",
      "\n",
      "… Ankyrin G (AnkG; 1:3000; Clone N106/36 catalog #75–187) were all obtained from the \n",
      "University of California, Davis (UC Davis)/NIH NeuroMab Facility. A goat anti-Ankyrin G antibody (…\n",
      "\n",
      "… 1 h with these mAbs and mouse anti-Ank-G mAb (N106/36 or N106/65), or mouse anti-PSD-93 \n",
      "(N18/30) all from the UC Davis/NIH NeuroMab Facility; rat anti-EB1 mAb (KT-51; AbCam)…\n",
      "\n",
      "In vertebrates, fast saltatory conduction along myelinated axons relies on the node of \n",
      "Ranvier. How nodes assemble on CNS neurons is not yet fully understood. We previously …\n",
      "\n",
      "Axon initial segment (AIS) cell surface proteins mediate key biological processes in neurons \n",
      "including action potential initiation and axo-axonic synapse formation. However, few AIS …\n",
      "\n",
      "… Anti-AnkG (75-146, RRID:AB_10673030) mouse monoclonal antibody N106/36 was obtained \n",
      "from the UC Davis/NIH NeuroMab Facility (United States), and rabbit polyclonal anti-FLAG …\n",
      "\n",
      "The axon initial segment (AIS) is a unique neuronal compartment that plays a crucial role in \n",
      "the generation of action potential and neuronal polarity. The assembly of the AIS requires …\n",
      "\n",
      "Microglia, the resident immune cells of the central nervous system, are key players in healthy \n",
      "brain homeostasis and plasticity. In neurological diseases, such as Multiple Sclerosis, …\n",
      "\n",
      "The tetrodotoxin‐resistant ( TTX ‐R) voltage‐gated sodium channel Na v 1.8 is predominantly \n",
      "expressed in peripheral afferent neurons, but in case of neuronal injury an ectopic and …\n",
      "\n",
      "Integrins are adhesion and survival molecules involved in axon growth during CNS development, \n",
      "as well as axon regeneration after injury in the peripheral nervous system (PNS). Adult …\n",
      "\n",
      "Action potential conduction along myelinated axons depends on high densities of voltage-gated \n",
      "Na + channels at the nodes of Ranvier. Flanking each node, paranodal junctions (…\n",
      "\n",
      "… et al., 2004), mouse α-K v 1.2 (K14/16; NeuroMab), chicken α-MAP2 (EnCor Biotechnology), \n",
      "mouse α-ankyrin G (ankG) (N106/36; NeuroMab), mouse α-pan Na v α (K58/35; Sigma), …\n",
      "\n",
      "… Monoclonal antibodies for ankyrin-G (clone N106/36) and K ir 2.1 (clone N112B/14) were \n",
      "obtained from the University of California Davis/National Institutes of Health NeuroMab Facility. …\n",
      "\n",
      "… (clone K13/31), anti-ADAM22 (clone N46/30), and anti-ankyrinG (clone N106/36) mAbs \n",
      "were obtained from NeuroMab (UC Davis/NIH NeuroMab Facility). Mouse anti-TAG-1 (1C12) …\n",
      "\n",
      "Axons must withstand mechanical forces, including tension, torsion, and compression. \n",
      "Spectrins and actin form a periodic cytoskeleton proposed to protect axons against these forces. …\n",
      "\n",
      "… Anti-AnkG (N106/36), anti-FGF14 (N56/21), and anti-Nav1.6 (K87A/10) mouse monoclonal \n",
      "antibodies were purchased from NeuroMab. Affinity-purified rabbit polyclonal antibodies …\n",
      "\n",
      "10.7554/eLife.21392.001 A high density of Na + channels at nodes of Ranvier is necessary \n",
      "for rapid and efficient action potential propagation in myelinated axons. Na+ channel …\n",
      "\n",
      "KCNQ2 variants in children with neurodevelopmental impairment are difficult to assess due \n",
      "their heterogeneity and unclear pathogenic mechanisms. We describe a child with neonatal-…\n",
      "\n",
      "The role of myelination for axonal conduction is well-established in projection neurons but \n",
      "little is known about its significance in GABAergic interneurons. Myelination is discontinuous …\n",
      "\n",
      "Supplementary Figure 2 (continued) Ten electron micrographs taken from serial sections of \n",
      "neurobiotin-labeled (HRP end-product) boutons (B) of AAC B45a making type II synapses (…\n",
      "\n",
      "AnkyrinG (ankG) is highly enriched in neurons at axon initial segments (AISs) where it \n",
      "clusters Na + and K + channels and maintains neuronal polarity. How ankG becomes …\n",
      "\n",
      "The formation, function, and plasticity of synapses require dynamic changes in synaptic \n",
      "receptor composition. Here, we identify the sorting receptor SorCS1 as a key regulator of …\n",
      "\n",
      "… The primary antibodies used were the following: anti-AnkG (mouse IgG2a, Neuromab, clone \n",
      "N106/36, 1:200), anti-PLP (hybridoma of polyclonal rat IgGs, clone AA3, gift from K. Ikenaka, …\n",
      "\n",
      "Neurons implement a variety of plasticity mechanisms to alter their function over timescales \n",
      "ranging from seconds to days. One powerful means of controlling excitability is to directly …\n",
      "\n",
      "Fast-spiking (FS) neurons can fire action potentials (APs) up to 1,000 Hz and play key roles \n",
      "in vital functions such as sound location, motor coordination, and cognition. Here we report …\n",
      "\n",
      "… IgG 1 ; clone N168/6), Kv1.2 (subtype IgG 2b ; clone K14/16) and Caspr (subtype IgG 1 ; \n",
      "clone K65/35) were obtained from the UC Davis/NIH NeuroMab Facility. Mouse monoclonal …\n",
      "\n",
      "… using monoclonal antibodies from the UC Davis/NIH NeuroMab Facility (University of California, \n",
      "Davis, CA 95616, USA), clones N105/13 and N106/20, respectively. F-actin was labeled …\n",
      "\n",
      "Dendritic spines (DS) are tiny protrusions implicated in excitatory postsynaptic responses in \n",
      "the CNS. To achieve their function, DS concentrate a high density of ion channels and …\n",
      "\n",
      "Heterotetrameric glutamate receptors are essential for the development, function, and \n",
      "plasticity of spine synapses but how they are organized to achieve this is not known. Here we …\n",
      "\n",
      "Exo70 is a subunit of the greater exocyst complex, a collection of proteins that oversees \n",
      "cellular membrane addition and polarized exocytosis by acting as a tethering intermediate …\n",
      "\n",
      "Background Exo70 is a subunit of the greater exocyst complex, a collection of proteins that \n",
      "oversees cellular membrane addition and polarized exocytosis by acting as a tethering …\n",
      "\n",
      "During development, pyramidal neurons undergo dynamic regulation of AMPA receptor (AMPAR) \n",
      "subunit composition and density to help drive synaptic plasticity and maturation. …\n",
      "\n",
      "… In contrast, when probed with an antibody raised against the N terminus of SHANK3 (Neuromab \n",
      "367/62), we detected truncated SHANK3 bands in striatal PSD preparations from the …\n",
      "\n",
      "The Par3 polarity protein is critical for subcellular compartmentalization in different developmental \n",
      "processes. Variants of PARD3, which encodes PAR3, are associated with intelligence …\n",
      "\n",
      "Haploinsufficiency of the SHANK3 gene is causally linked to autism spectrum disorder (ASD), \n",
      "and ASD-associated genes are also enriched for chromatin remodelers. Here we found …\n",
      "\n",
      "Thalamocortical neurons (TCNs) play a critical role in the maintenance of thalamocortical \n",
      "oscillations, dysregulation of which can result in certain types of seizures. Precise control over …\n",
      "\n",
      "Properties, developmental regulation, and cAMP modulation of the hyperpolarization-activated \n",
      "current (I h ) were investigated by the whole cell patch-clamp technique in vestibular …\n",
      "\n",
      "The chemosensory glomus cells of the carotid body (CB) detect changes in O 2 tension. \n",
      "Carotid sinus nerve fibers, which originate from peripheral sensory neurons located within the …\n",
      "\n",
      "Mutations in leucine-rich repeat kinase 2 (LRRK2), which are associated with autosomal \n",
      "dominant Parkinson's disease, elicit progressive dendrite degeneration in neurons. We …\n",
      "\n",
      "AB. Diagram of the structure of the human SHANK2 gene and protein.(A) The nonsense \n",
      "mutation R841X is found in an individual with ASD and ID.(B) SHANK2 protein contains 5 …\n",
      "\n",
      "α-Synuclein plays a key role in the pathogenesis of Parkinson’s disease and related disorders, \n",
      "but critical interacting partners and molecular mechanisms mediating neurotoxicity are …\n",
      "\n",
      "The pathogenesis of schizophrenia is believed to involve combined dysfunctions of many \n",
      "proteins including microtubule-associated protein 6 (MAP6) and Kv3.1 voltage-gated K + (Kv) …\n",
      "\n",
      "The conventional kinesin motor transports many different cargos to specific locations in neurons. \n",
      "How cargos regulate motor function remains unclear. Here we focus on KIF5, the heavy …\n",
      "\n",
      "… the UC Davis/NIH NeuroMab facility: AnkG (106/36; RRID:AB_10673030), AnkG (106/65; \n",
      "RRID:AB_10675130), PanNav (N419/78; RRID:AB_2493099), b1 spectrin (N385/21; RRID:…\n",
      "\n",
      "Skeletal muscle contraction depends on activation of clustered acetylcholine receptors (AchRs) \n",
      "and muscle-specific Na + channels (Nav1.4). Some Nav1.4 channels are highly …\n",
      "\n",
      "Clustered ion channels at nodes of Ranvier are critical for fast action potential propagation \n",
      "in myelinated axons. Axon-glia interactions converge on ankyrin and spectrin cytoskeletal …\n",
      "\n",
      "… Mouse monoclonal antibody against GKAP (clone N1427/31, used at 1:100), pan Shank (clone \n",
      "N23B/49, which recognizes all three members of the Shank family: Shank 1, 2 and 3, …\n",
      "\n",
      "… 1:1000 for Western), and pan Shank (clone N23B/49, used at 1:200 for microscopy and 1:1000 \n",
      "for Western) were from NeuroMab, Davis, CA, USA. Rabbit polyclonal antibody against …\n",
      "\n",
      "… The monoclonal mouse antibodies against Shank (N23B/49) and PSD-95 (K28/43) were \n",
      "from NeuroMAB. ECL and ECL plus reagents were from Amersham. Other reagents were from …\n",
      "\n",
      "IRSp53 (BAIAP2) is an abundant protein at the postsynaptic density (PSD) that binds to major \n",
      "PSD scaffolds, PSD-95 and Shanks, as well as to F-actin. The distribution of IRSp53 at the …\n",
      "\n",
      "Early in development spontaneous activity modulates survival and connectivity of neurons \n",
      "and thus plays a crucial role in the formation of neural networks. The emergence of …\n",
      "\n",
      "The family of Shank scaffolding molecules (comprising Shank1, 2 and 3) are core components \n",
      "of the postsynaptic density ( PSD ) in neuronal synapses. Shanks link surface receptors to …\n",
      "\n",
      "Most neurological disorders display impaired synaptic connectivity. Hence, modulation of \n",
      "synapse formation may have therapeutic relevance. However, the high density and small size …\n",
      "\n",
      "In vitro studies suggest that the intracellular C terminus of Neuroligin1 (NL1) could play a \n",
      "central role in the maturation of excitatory synapses. However, it is unknown how this activity …\n",
      "\n",
      "… ), mouse monoclonal anti-Pan-Shank (clone N23B/49; 1:400, Neuromab, N23B/49), mouse \n",
      "monoclonal anti-NR2A (clone N327/95; 1:400, Neuromab, Q00959), mouse monoclonal anti-…\n",
      "\n",
      "… for Western), and pan-Shank (clone N23B/49, which recognizes all three members of the \n",
      "Shank family: Shank1, 2 and 3, used at 1:250 for EM) were from NeuroMab (Davis, CA). Mouse …\n",
      "\n",
      "… (clone N23B/49, which recognizes all three members of the Shank family: Shank 1, 2 and 3, \n",
      "used at 1:250), and Shank 2 (clone N23B/6, used at 1:200) were from NeuroMab (Davis, CA)…\n",
      "\n",
      "Excitatory glutamatergic synapses facilitate important aspects of communication between \n",
      "the neurons that govern complex forms of behavior. Accordingly, small differences in the …\n",
      "\n",
      "Synapses are specialized adhesion sites between neurons that are connected by protein \n",
      "complexes spanning the synaptic cleft. These trans‐synaptic interactions can organize …\n",
      "\n",
      "Despite the central role PSD-95 plays in anchoring postsynaptic AMPARs, how PSD-95 \n",
      "itself is tethered to postsynaptic sites is not well understood. Here we show that the F-actin …\n",
      "\n",
      "Mutations or altered protein levels of SHANK3 are implicated in neurodevelopmental disorders \n",
      "such as Phelan–McDermid syndrome, autism spectrum disorders, and schizophrenia ( …\n",
      "\n",
      "A core network of widely expressed proteins within the glutamatergic post-synapse mediates \n",
      "activity-dependent synaptic plasticity throughout the brain, but the specific proteomic …\n",
      "\n",
      "Scribble (Scrib) is a key regulator of apicobasal polarity, presynaptic architecture, and short-term \n",
      "synaptic plasticity in Drosophila. In mammals, its homolog Scrib1 has been implicated …\n",
      "\n",
      "LRRTMs are postsynaptic cell adhesion proteins that have region-restricted expression in the \n",
      "brain. To determine their role in the molecular organization of synapses in vivo, we studied …\n",
      "\n",
      "The spatial and temporal balance of spinal α‐motoneuron (αMN) intrinsic membrane \n",
      "conductances underlies the neural output of the final common pathway for motor commands. …\n",
      "\n",
      "Haploinsufficiency of the Shank3 gene, which encodes a scaffolding protein at glutamatergic \n",
      "synapses, is a highly prevalent and penetrant risk factor for autism. Using combined …\n",
      "\n",
      "Rho GTPases activated by GDP/GTP exchange factors (GEFs) play key roles in the \n",
      "developing and adult nervous system. Kalirin-7 (Kal7), the predominant adult splice form of the …\n",
      "\n",
      "… of NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs \n",
      "… for specific labeling (N355/1 in a1 and N399/19 in b1) and two other candidates from these …\n",
      "\n",
      "… The primary antibodies used for western blotting were mouse monoclonal anti-GFP (1∶1000, \n",
      "NeuroMab, N86/8) and rabbit polyclonal anti-FGF14 (1∶1000). Secondary antibodies …\n",
      "\n",
      "Aggregation of the multifunctional RNA‐binding protein TDP‐43 defines large subgroups of \n",
      "amyotrophic lateral sclerosis and frontotemporal dementia and correlates with …\n",
      "\n",
      "Chemotherapy-related cognitive impairment (CRCI) is a common adverse effect of treatment \n",
      "and is characterized by deficits involving multiple cognitive domains including memory. …\n",
      "\n",
      "Control of translation is a fundamental source of regulation in gene expression. The induction \n",
      "of protein synthesis by brain-derived neurotrophic factor (BDNF) critically contributes to …\n",
      "\n",
      "… We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and \n",
      "the Trimmer laboratory for their contributions and dedicated efforts, and Dr. Randall Stewart …\n",
      "\n",
      "Altered excitatory/inhibitory (E/I) balance is implicated in neuropsychiatric and neurodevelopmental \n",
      "disorders, but the underlying genetic etiology remains poorly understood. Copy …\n",
      "\n",
      "Phosphorylation regulates surface and synaptic expression of NMDA receptors (NMDARs). \n",
      "Both the tyrosine kinase Fyn and the tyrosine phosphatase striatal-enriched protein tyrosine …\n",
      "\n",
      "The C9orf72 repeat expansion causes amyotrophic lateral sclerosis and frontotemporal \n",
      "dementia, but the poor correlation between C9orf72‐specific pathology and TDP ‐43 pathology …\n",
      "\n",
      "The pathogenesis of peripheral neuropathies in adults is linked to maintenance mechanisms \n",
      "that are not well understood. Here, we elucidate a novel critical maintenance mechanism …\n",
      "\n",
      "Structural plasticity of dendritic spines and synapses is a fundamental mechanism governing \n",
      "neuronal circuits and may form an enduring basis for information storage in the brain. We …\n",
      "\n",
      "… The following antibodies were used: Anti-GFP (mouse N86/8, Neuromab, Davis, CA, USA \n",
      "and rabbit, Clontech, Mountainview, CA, USA), anti-β-actin (Sigma Aldrich, St. Louis, MO, USA)…\n",
      "\n",
      "The DISC1 protein is implicated in major mental illnesses including schizophrenia, depression, \n",
      "bipolar disorder, and autism. Aberrant mitochondrial dynamics are also associated with …\n",
      "\n",
      "MicroRNAs (miRNAs) are important regulators of localized mRNA translation in neuronal \n",
      "dendrites. The presence of RNA-induced silencing complex proteins in these compartments …\n",
      "\n",
      "Synaptic plasticity requires remodeling of the actin cytoskeleton. Although two actin isoforms, \n",
      "β- and γ-actin, are expressed in dendritic spines, the specific contribution of γ-actin in the …\n",
      "\n",
      "Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) are severe \n",
      "neurodegenerative diseases with overlapping clinical and pathological features. Recently, a …\n",
      "\n",
      "Abeta peptides derived from the amyloid precursor protein (APP) have been strongly implicated \n",
      "in the pathogenesis of Alzheimer's disease. However, the normal function of APP and …\n",
      "\n",
      "Schizophrenia is associated with cognitive and behavioral dysfunctions thought to reflect \n",
      "imbalances in neurotransmission systems. Recent screenings suggested that lack of (functional…\n",
      "\n",
      "… antibody to PSD-95 (clone K28/43, NeuroMab), rabbit antibody to GRIP1 (Millipore), mouse \n",
      "antibody to PICK1 (clone L20/8, NeuroMab) and rabbit antibody to phospho-Ser880 GluA2 (…\n",
      "\n",
      "… N295B/66; UC Davis/NIH NeuroMab Facility, Davis, CA) at a dilution of 1:400 incubated \n",
      "for 45 minutes at room temperature followed by a 30-minute incubation with Poly-AP anti-mouse …\n",
      "\n",
      "The primary cilium is a conserved, microtubule-based organelle that protrudes from the \n",
      "surface of most vertebrate cells as well as sensory cells of many organisms. It transduces …\n",
      "\n",
      "Cilia are near ubiquitous small, cellular appendages critical for cell-to-cell communication. \n",
      "As such, they are involved in diverse developmental and homeostatic processes, including …\n",
      "\n",
      "Polymer microparticles possess great potential as functional building blocks for advanced \n",
      "bottom-up engineering of complex tissues. Tailoring the three-dimensional architectural …\n",
      "\n",
      "… NeuroMab, a joint facility of the University of California-Davis and the NIH (http://www.neuromab.… \n",
      "Two mAbs were produced from the A3 VCD fusion: N144/17 (anti-A3), which is specific …\n",
      "\n",
      "Presynaptic Ca V 2 channels are essential for Ca 2+ -triggered exocytosis. In addition, there \n",
      "are two competing models for their roles in synapse structure. First, Ca 2+ channels or Ca 2…\n",
      "\n",
      "… 75-058; RRID:AB_2261666) recognized a single band of 105 kDa in Western blots of rat \n",
      "and mouse brain membranes (UC Davis/NIH NeuroMab Facility). SAP102 labeling was …\n",
      "\n",
      "Glutamate plays a role in hair cell afferent transmission, but the receptors that mediate \n",
      "neurotransmission between outer hair cells (OHCs) and type II ganglion neurons are not well …\n",
      "\n",
      "The modulation of AMPA receptor (AMPAR) content at synapses is thought to be an underlying \n",
      "molecular mechanism of memory and learning. AMPAR content at synapses is highly …\n",
      "\n",
      "Synaptic signaling complexes are held together by scaffold proteins, each of which is selectively \n",
      "capable of interacting with a number of other proteins. In previous studies of rabbit retina…\n",
      "\n",
      "… PSD93 was not detected in varicosity but identified in the whole gut extract using a \n",
      "monoclonal antibody (clone N18/30), and signal was not seen when the crude gut extract was …\n",
      "\n",
      "The mechanisms controlling synapse growth and maintenance are of critical importance for \n",
      "learning and memory. The MAGUK family of synaptic scaffolding proteins is abundantly …\n",
      "\n",
      "How does chronic activity modulation lead to global remodeling of proteins at synapses and \n",
      "synaptic scaling? Here we report that guanylate kinase–associated protein (GKAP; also …\n",
      "\n",
      "… N18/30), and SAP97 (clone K64/15) from NeuroMab (Davis, CA); neuronal nitric oxide \n",
      "synthase (nNOS) (clone 16), and β-catenin (clone 14) from BD Transduction Labs (San Jose, CA)…\n",
      "\n",
      "Neurite outgrowth is key to the formation of functional circuits during neuronal development. \n",
      "Neurotrophins, including nerve growth factor (NGF), increase neurite outgrowth in part by …\n",
      "\n",
      "… The mouse anti-Kv 1.2 antibodies were obtained from Neuromab and have both been \n",
      "knock-out validated. The main Caspr antibody used was rabbit anti-Caspr from Abcam. The …\n",
      "\n",
      "… of endogenously expressed BK channels and NMDARs with in situ PLA, we examined \n",
      "the antigen specificities of a mouse monoclonal anti-BKα antibody (L6/60; NeuroMab) and a …\n",
      "\n",
      "Alzheimer's disease (AD) is characterized by memory impairments and age-dependent \n",
      "synapse loss. Experimental and clinical studies have shown decreased expression of the …\n",
      "\n",
      "… The monoclonal antibodies PSD-95 (Clone K28/43) and GluA2 (Clone L21/32) were \n",
      "developed by and/or obtained from the UC Davis/NIH NeuroMab Facility and maintained by the …\n",
      "\n",
      "A subset of amyotrophic lateral sclerosis (ALS) and frontotemporal lobar degeneration (FTLD) \n",
      "patients present pathological redistribution and aggregation of the nuclear protein fused in …\n",
      "\n",
      "Glutamate AMPA receptors (AMPARs) lacking GluA2 subunit are calcium permeable (CP‐AMPARs), \n",
      "which are increased in the hypothalamic paraventricular nucleus (PVN) and …\n",
      "\n",
      "Alzheimer’s Disease (AD) is the primary cause of dementia among the elderly population. \n",
      "Elevated plasma levels of homocysteine (HCy), an amino acid derived from methionine …\n",
      "\n",
      "10.7554/eLife.24192.001 Glutamatergic synapses rely on AMPA receptors (AMPARs) for fast \n",
      "synaptic transmission and plasticity. AMPAR auxiliary proteins regulate receptor trafficking, …\n",
      "\n",
      "Alzheimer's disease (AD) is a neurodegenerative disorder characterized by a progressive \n",
      "deterioration of cognitive abilities, accumulation of the amyloid-β peptide (Aβ), increase of …\n",
      "\n",
      "… Other lanes include antiserum from one of the immunized mice (HB), a mouse serum negative \n",
      "control, and a positive control NeuroMab mAb against a different target (L86A/37, AMIGO-…\n",
      "\n",
      "… This probe was based on the well-characterized mouse mAb L109/57 (UC Davis/NIH NeuroMab \n",
      "… This is the same penetration depth as seen for the L109/57 mAb from which the scFv is …\n",
      "\n",
      "… (K7/45) were purchased from Neuromab, and anti–β-actin Abs were purchased from Bioeasy. \n",
      "Digital images were captured using the Amersham Image 600 system (GE Healthcare). …\n",
      "\n",
      "… (NeuroMab) recognized a single protein band at 70 kDa on western blot of lysates from \n",
      "Kv1.5-transfected COS cells. No band of equivalent size is observed from rat brain samples and …\n",
      "\n",
      "… Although both the Alomone anti-BKbeta2 Ab and the NeuroMab N53/32 β2 Ab were of use \n",
      "for IP and identification of β2 protein in our Western blot procedures, in our hands these Abs …\n",
      "\n",
      "Background: Inflammation-induced sensitization of primary afferents is associated with a \n",
      "decrease in K + current. However, the type of K + current and basis for the decrease varies as a …\n",
      "\n",
      "… The membranes were incubated with primary antibodies including anti-BK α (Neuromab, \n",
      "L6/60, 1:1000), anti-BK β2 (Neuromab, N53/32 1:1000), anti-BK β4 (Neuromab, L18A.3, 1:1000…\n",
      "\n",
      "Inactivation is an intrinsic property of several voltage-dependent ion channels, closing the \n",
      "conduction pathway during membrane depolarization and dynamically regulating neuronal …\n",
      "\n",
      "… polyclonal KCNE1 antibody (1:100; Alomone Labs, Jerusalem, Israel), and Kv1.4 was also \n",
      "labeled with a mouse monoclonal Kv1.4 antibody (1:250, clone K13/31, 1:300, NeuroMab). A …\n",
      "\n",
      "… -CD98 (Abcam, 2 µg mL −1 ), anti-KCC2 transporter NeuroMab clone N1/12 (1:1,000), \n",
      "anti-NKCC1 (NeuroMab clone N161/20, 1:1,000) at 4C overnight. Membranes were washed in …\n",
      "\n",
      "… (K13/31) mouse monoclonal antibodies used in this study are available from NeuroMab (http://www.neuromab.org), \n",
      "a not-for-profit supplier of monoclonal antibodies administered …\n",
      "\n",
      "… (K13/31) were obtained from NeuroMab. Rabbit pAbs to mitogen-activated protein kinase \n",
      "(MAPK) and to phospho-MAPK were from Promega. A mouse mAb to Src as well as to PP2 …\n",
      "\n",
      "… #: K14/16, K17/70, and K13/31, respectively; UC Davis/NIH Neuromab Facility, Davis, CA, \n",
      "USA), … We thank University of California, Davis-NIH NeuroMab facility (a nonprofit supplier) for …\n",
      "\n",
      "… The total cellular pool was detected by cytoplasmic-directed mAb K14/16 (Kv1.2) or K13/31 \n",
      "(Kv1.4) after detergent permeabilization. Alternatively, surface staining of intact cells with Kv1.…\n",
      "\n",
      "A specialized axonal ending, the basket cell “pinceau,” encapsulates the Purkinje cell axon \n",
      "initial segment (AIS), exerting final inhibitory control over the integrated outflow of the …\n",
      "\n",
      "… the supplier's (NeuroMab) datasheets online. The protein bands detected match with the \n",
      "expected sizes from endogenous and heterologous expression as reported by NeuroMab. We …\n",
      "\n",
      "miR-17-92 is a microRNA cluster with six distinct members. Here, we show that the miR-17-92 \n",
      "cluster and its individual members modulate chronic neuropathic pain. All cluster …\n",
      "\n",
      "10.7554/eLife.40316.001 Protein palmitoylation and depalmitoylation alter protein function. \n",
      "This post-translational modification is critical for synaptic transmission and plasticity. …\n",
      "\n",
      "Synapses are found in vast numbers in the brain and contain complex proteomes. We \n",
      "developed genetic labeling and imaging methods to examine synaptic proteins in individual …\n",
      "\n",
      "Although considerable evidence suggests that the chemical synapse is a lynchpin underlying \n",
      "affective disorders, how molecular insults differentially affect specific synaptic connections …\n",
      "\n",
      "Synapses in the brain exhibit cell-type-specific differences in basal synaptic transmission \n",
      "and plasticity. Here, we evaluated cell-type-specific specializations in the composition of …\n",
      "\n",
      "Gray matter degeneration contributes to progressive disability in multiple sclerosis (MS) and \n",
      "can occur out of proportion to measures of white matter disease. Although white matter …\n",
      "\n",
      "… /ml) (K28/43, UC Davis/NIH NeuroMab Facility), anti-GluA1 mouse monoclonal antibody (0.5 \n",
      "μg/ml, N355/1, UC Davis/NIH NeuroMab Facility), anti-β-actin mouse monoclonal antibody (…\n",
      "\n",
      "Tau-mediated toxicity is associated with cognitive decline and Alzheimer's disease (AD) \n",
      "progression. In particular, tau post-translational modifications (PTMs) are thought to generate …\n",
      "\n",
      "… FHF (N235/22, Catalogue 75-246, NeuroMab) and pan-sodium channel (K58/35, Catalogue \n",
      "S8809, Sigma Aldrich) followed by secondary peroxidase-conjugated antibodies (1:10,000, …\n",
      "\n",
      "Nav1.6 and Nav1.6-mediated resurgent currents have been implicated in several pain \n",
      "pathologies. However, our knowledge of how fast resurgent currents are modulated in neurons is …\n",
      "\n",
      "… -iFGF13 mouse monoclonal antibody (NeuroMab clone N91/27, 1:300; Antibodies Incorporated), \n",
      "and the anti-Pan-iFGF-A mouse monoclonal antibody (NeuroMab clone N235/22, 1:300…\n",
      "\n",
      "We report a signaling pathway linking two fundamental functions of the ER, oxidative protein \n",
      "folding, and intracellular calcium regulation. Cells sense ER oxidative protein folding …\n",
      "\n",
      "… 1e or K39/25 antibodies after fixation and before detergent permeabilization. Our previous \n",
      "studies (Manganas and Trimmer, 2000; Tiffany et al., 2000; Manganas et al., 2001a,b; Shibata …\n",
      "\n",
      "… Using a different TRPC 7 antibody (N64A/36; 100% homology to the rat sequence) we \n",
      "detected no staining of pre-glomerular vessels. A double staining of rat brain and renal sections …\n",
      "\n",
      "… The mouse anti-pan-neurofascin antibody (extNF; A12/18, Neuromab) was generated \n",
      "against the rat-specific extracellular domain common to NF155 and NF186 (amino acids 25-1110)…\n",
      "\n",
      "… clone N70/28 Fusion protein amino acids 778-910 at C-terminus of rat HCN1 NeuroMab, 75-… \n",
      "-1108 at C-terminus of rat HCN4 NeuroMab, 75-150 Mouse monoclonal AB_2248534 1:200 …\n",
      "\n",
      "… Sections were then immunostained as described 51 using mouse antibodies against HCN1 \n",
      "(clone N70/28, NeuroMab), PanNav 52 , or Caspr (clone K65/35, NeuroMab), and rabbit …\n",
      "\n",
      "… The HCN1 results were obtained with a mouse monoclonal antibody directed against amino \n",
      "acids 778–910 (clone N70/28; NeuroMab) and an affinity-purified rabbit polyclonal antibody …\n",
      "\n",
      "Parallel visual pathways originate at the first synapse in the retina, where cones make \n",
      "connections with cone bipolar cells that in turn contact ganglion cells. There are more ganglion …\n",
      "\n",
      "… to HCN1 obtained from NeuroMab, a polyclonal antibody to HCN1 obtained from Alomone \n",
      "Labs and a monoclonal antibody to Ca V 3.2 obtained from Neuromab. To examine the …\n",
      "\n",
      "De novo mutations in voltage-and ligand-gated channels have been associated with an \n",
      "increasing number of cases of developmental and epileptic encephalopathies, which often fail to …\n",
      "\n",
      "… Neuromab N231B/34, which was raised against a synthetic peptide corresponding to amino \n",
      "acids 841–960 did not detect recombinant LRRK2-(883–1300), suggesting that the epitope …\n",
      "\n",
      "… (a) Cells were stained with specific antibodies against FAK and LRRK2 (N231B/34). \n",
      "Magnified images of boxed areas in 4–6 and 13–15 were shown in 7–9 and 16–18, respectively. …\n",
      "\n",
      "… Interestingly, a different picture was observed using anti-LRRK2 N231B/34, a high affinity \n",
      "antibody against human LRRK2 [26]. This latter antibody showed a 3–4 fold increase in human …\n",
      "\n",
      "… The mouse K v 1.2 (clone K14/16, 1: 500) and Na v 1.6 (clone K87A/10, 1:500) antibodies \n",
      "were purchased from the University of California, Davis NINDS/NIMH NeuroMab Facility. The …\n",
      "\n",
      "… For co-localization experiments, we used mouse monoclonal anti-Kv1.2 antibodies (clone \n",
      "K14/16; 1:1000 dilution; UC Davis/NIH NeuroMab Facility) and rabbit polyclonal anti-KCNQ2 …\n",
      "\n",
      "Changes to the structure of nodes of Ranvier in the normal-appearing white matter (NAWM) \n",
      "of multiple sclerosis (MS) brains are associated with chronic inflammation. We show that the …\n",
      "\n",
      "Actin, spectrin, and associated molecules form a membrane-associated periodic skeleton (MPS) \n",
      "in neurons. The molecular composition and functions of the MPS remain incompletely …\n",
      "\n",
      "… Kv1.2 was detected using a mouse monoclonal Kv1.2 antibody (1:2,000 dilution, clone \n",
      "K14/16 75-008; NeuroMab) and HRP-conjugated goat anti-mouse antibody (1:20,000 dilution, …\n",
      "\n",
      "… K+ channel, clone K14/16 and anti-Caspr, clone K65/35 were obtained from the UC Davis/NINDS/NIMH \n",
      "NeuroMab Facility, supported by NIH grant U24NS050606 and maintained by …\n",
      "\n",
      "Abstract Changes to the structure of nodes of Ranvier in the normal-appearing white matter (NAWM) \n",
      "of MS brains are associated with chronic inflammation. We show that the paranodal …\n",
      "\n",
      "Damage to long axons in white matter tracts is a major pathology in closed head traumatic \n",
      "brain injury (TBI). Acute TBI treatments are needed that protect against axon damage and …\n",
      "\n",
      "… Other commercial antibodies used were as follows: goat anti-human contactin-1 polyclonal \n",
      "antibodies (R&D Systems), mouse monoclonal antibody K14/16 against the K v 1.2 α-subunit …\n",
      "\n",
      "Spiral ganglion neurons (SGNs) relay acoustic code from cochlear hair cells to the brainstem, \n",
      "and their stimulation enables electrical hearing via cochlear implants. Rapid adaptation, a …\n",
      "\n",
      "The leucine-rich glioma-inactivated (LGI) family consists of four highly conserved paralogous \n",
      "genes, LGI1-4, that are highly expressed in mammalian central and/or peripheral nervous …\n",
      "\n",
      "What percentage of the protein function is required to prevent disease symptoms is a \n",
      "fundamental question in genetic disorders. Decreased transsynaptic LGI1-ADAM22 protein …\n",
      "\n",
      "… V 1.1 (K20/78), mouse anti-K V 1.2 (K14/16), and mouse anti-PSD95 (K28/43) were obtained \n",
      "from University of California, Davis/NIH NeuroMab Facility. Secondary antibodies used for …\n",
      "\n",
      "The evolution of myelin was a major key event in vertebrates which aimed to enhance \n",
      "conduction velocity of electrical impulses in axons. Distinct myelination patterns along axons can …\n",
      "\n",
      "(論文の内容の要旨)[背景と目的] シナプス後肥厚部 (postsynaptic density, PSD) とはシナプス膜\n",
      "直下にある, 神経細胞に高度に分化した特殊な細胞骨格系で, 極めて精巧なタンパク質複合体である…\n",
      "\n",
      "It is essential to study the molecular architecture of post‐synaptic density ( PSD ) to understand \n",
      "the molecular mechanism underlying the dynamic nature of PSD , one of the bases of …\n",
      "\n",
      "… ; NeuroMab) or anti-Slack (1:3,000, clone N3/26; NeuroMab) antibody for 2.5 hours at \n",
      "room temperature. Membranes were incubated with HRP-labeled goat anti-mouse IgG1 (1:150,000…\n",
      "\n",
      "… monoclonal Slack antibody (clone N3/26, dilution 1 : 50; NeuroMab, Davis, CA, USA) and \n",
      "mouse monoclonal Slick antibody (clone N11/33, dilution 1 : 50; NeuroMab). We also used …\n",
      "\n",
      "… For each experiment, 40 µg of either anti-Slick (clone N11/33, Neuromab) or anti-Slack (clone \n",
      "N3/26, Neuromab) channel antibody was used and 2 mg of synaptic plasma membranes …\n",
      "\n",
      "… Middle, brain membrane proteins were separated and probed with N3/26 anti-Slo2.2 \n",
      "mAb (… N3/26, were developed by and obtained from the UC Davis/NIH NeuroMab Facility, …\n",
      "\n",
      "… N11/33, UC Davis/NIH NeuroMab Facility) and Slack (Anti-Slo2.2 sodium-activated potassium \n",
      "channel, clone N3/26, UC Davis/NIH NeuroMab Facility). For the immunocytochemical …\n",
      "\n",
      "… (1:5, clone K75/41; NeuroMab, UC Davis/NIH NeuroMab Facility) and mouse monoclonal \n",
      "anti-K V 7.4 (1:50, clone N43/6; NeuroMab) were used to detect various potassium channel α- …\n",
      "\n",
      "… Primary antibodies used were Neuromab monoclonal antibodies against KCNQ1 (N37/A10, \n",
      "0.6 μg/ml), KCNQ2 (N26A/23, 0.6 μg/ml), and KCNQ4 (N43/6, 0.6 μg/ml) and a non-…\n",
      "\n",
      "GABA A receptors mediate fast inhibitory transmission in the brain, and their number can be \n",
      "rapidly up- or downregulated to alter synaptic strength. Neuroligin-2 plays a critical role in …\n",
      "\n",
      "Neurodevelopmental disorders, such as intellectual disability (ID), epilepsy, and autism, \n",
      "involve altered synaptic transmission and plasticity. Functional characterization of their …\n",
      "\n",
      "Lateral diffusion on the neuronal plasma membrane of the AMPA-type glutamate receptor (AMPAR) \n",
      "serves an important role in synaptic plasticity. We investigated the role of the …\n",
      "\n",
      "… : N67/15, NeuroMab, Davis, CA) and diluted in DaVinci Green diluent (Biocare) applied for \n",
      "1 h. For double labeling, a cocktail of secondary antibodies (Alexa 555-conjugated goat anti…\n",
      "\n",
      "Intracellular Fibroblast Growth Factor 14 (iFGF14) and the other intracellular FGFs (iFGF11-13) \n",
      "regulate the properties and densities of voltage-gated neuronal and cardiac Na + (Nav) …\n",
      "\n",
      "… A monoclonal antibody anti-GABAB1 (Clone N93A/49; NeuroMab, Davis, CA, USA) and \n",
      "anti-GABAB2 (Clone N81/37; NeuroMab) were used. A guinea-pig polyclonal antibody anti-…\n",
      "\n",
      "Genes with multiple co-active promoters appear common in brain, yet little is known about \n",
      "functional requirements for these potentially redundant genomic regulatory elements. SCN1A, …\n",
      "\n",
      "… -reactivity validated by NeuroMab. Antibodies N257/25 and … The subclone N262/16 raised \n",
      "against the hapten 4 showed … Given these results, N257/25.1.1 and N262/16 represent …\n",
      "\n",
      "Key points This paper provides a comprehensive electrophysiological characterization of the \n",
      "external [K + ] dependence and inward rectifying properties of Kir channels in fast skeletal …\n",
      "\n",
      "Gamma aminobutyric acid (GABA) subtype A receptors (GABA A Rs) are integral membrane \n",
      "ion channels composed of five individual proteins or subunits. Up to 19 different GABA A R …\n",
      "\n",
      "Impaired inhibitory signaling underlies the pathophysiology of many neuropsychiatric and \n",
      "neurodevelopmental disorders including autism spectrum disorders and epilepsy. Neuronal …\n",
      "\n",
      "Retromer is an evolutionarily conserved endosomal trafficking complex that mediates the \n",
      "retrieval of cargo proteins from a degradative pathway for sorting back to the cell surface. To …\n",
      "\n",
      "Multiple trans-synaptic complexes organize synapse development, yet their roles in the \n",
      "mature brain and cooperation remain unclear. We analyzed the postsynaptic adhesion protein …\n",
      "\n",
      "… PSD-95 (K28/43) and SAP97 (N19/2) antibodies were obtained from Neuromab. ERK 1/2 \n",
      "(extracellular signal-regulated kinase 1/2) antibody was purchased from Cell Signaling …\n",
      "\n",
      "… Anti-PSD95/SAP90 antibody (K28/43) and SAP102 (N19/2) was purchased from \n",
      "University of California Davis/National Institutes of Health (NIH) NeuroMab facility. Anti-α-tubulin …\n",
      "\n",
      "Neurodevelopmental disorders such as autism spectrum disorders and schizophrenia \n",
      "differentially impact males and females and are highly heritable. The ways in which sex and …\n",
      "\n",
      "Classical cadherins, a set of ~20 related recognition and signaling molecules, have been \n",
      "implicated in many aspects of neural development, including the formation and remodeling of …\n",
      "\n",
      "Sensitive and specific antibodies are essential for detecting molecules in cells and tissues. \n",
      "However, currently used polyclonal and monoclonal antibodies are often less sensitive than …\n",
      "\n",
      "In tauopathic conditions, such as Alzheimer's disease (AD), highly soluble and natively \n",
      "unfolded tau polymerizes into an insoluble filament; however, the mechanistic details of this …\n",
      "\n",
      "Glycoprotein M6B and the closely related proteolipid protein regulate oligodendrocyte \n",
      "myelination in the central nervous system, but their role in the peripheral nervous system is less …\n",
      "\n",
      "Myelin is a highly specialized structure, organized into short periodical segments, which are \n",
      "extensions of the Schwann cell membrane in the peripheral nervous system (PNS). Myelin …\n",
      "\n",
      "The ability of dendritic spines to change size and shape rapidly is critical in modulating synaptic \n",
      "strength; these morphological changes are dependent upon rearrangements of the actin …\n",
      "\n",
      "… Anti–PAN MAGUK antibody (clone K28/86) and anti-SAP97 antibody (clone K64/15) were \n",
      "purchased from the University of California Davis/NIH NeuroMab Facility. Anti–human SAP97 …\n",
      "\n",
      "… 407140), and 3) monoclonal mouse anti-IP3 receptor clone L24/18, obtained from the \n",
      "UC Davis/NIH NeuroMab Facility, University of California, Davis (catalog No. 75-035). …\n",
      "\n",
      "The GABA-B receptor is densely expressed throughout the brain and has been implicated in \n",
      "many CNS functions and disorders, including addiction, epilepsy, spasticity, schizophrenia, …\n",
      "\n",
      "… , N6/38) from NeuroMab; mouse anti-SATB2 (clone, 4B19) from Abcam; rabbit anti-fluorescein \n",
      "(Life Technologies); and rabbit anti-protein kinase C alpha (P4334) from Sigma. Rabbit …\n",
      "\n",
      "… , N6/38) from NeuroMab (Davis, CA); mouse anti-protein kinase C- α (PKCα) (clone MC5) \n",
      "and rabbit anti-PKCα (P4334) from Sigma; rabbit anti-Opn4 from Thermo Fisher (PA1-780), …\n",
      "\n",
      "… with 0.1% Triton X-100 in Tris-buffered saline (TBS), blocked in 5% horse serum in TBS (vehicle), \n",
      "and incubated overnight in vehicle containing K55/7 (anti-KChIP1) or K75/30 (anti-Kv4.…\n",
      "\n",
      "The ENU-induced neurological mutant ataxia3 was mapped to distal mouse chromosome \n",
      "15. Sequencing of the positional candidate gene Scn8a encoding the sodium channel Na v …\n",
      "\n",
      "… (A) Representative images of the sensory epithelia showing cristae immunolabeled with the \n",
      "K65/35 anti-CASPR1 antibody (green) and the AB19013 anti-tenascin antibody (red). In the …\n",
      "\n",
      "… Trimmer, University of California, Davis, CA) and/or mouse monoclonal anti-Caspr (K65/35) \n",
      "antibody (1:500 dilution in vehicle; UC Davis/NIH NeuroMab Facility). Subsequently, …\n",
      "\n",
      "Action potential initiation and propagation in myelinated axons require ion channel clustering \n",
      "at axon initial segments (AIS) and nodes of Ranvier. Disruption of these domains after …\n",
      "\n",
      "… anti-CASPR1 (clone K65/35, RRID: AB_2083496, used at 1:400 dilution) and anti-PSD95 \n",
      "(clone K28/43, RRID: AB_2292909, 1:100) from the UC Davis/NIH NeuroMab Facility; mouse …\n",
      "\n",
      "… -associated protein (Caspr1) (clone K65/35, IgG1, cat. # 75–001) and anti-PSD95 (clone \n",
      "K28/43, IgG2a, cat# 75–028) from Neuromab. We used the following secondary antibodies …\n",
      "\n",
      "Astrocytes, once considered passive support cells, are increasingly appreciated as dynamic \n",
      "regulators of neuronal development and function, in part via secreted factors. The extent to …\n",
      "\n",
      "… QKI was immunoprecipitated with a QKI-5 specific (Bethyl, A300-183A) or panQKI (Neuromab, \n",
      "N147/6) antibodies and sequencing libraries prepared including a size matched input …\n",
      "\n",
      "… Mouse anti-GFP (University of California at Davis/NIH NeuroMab Facility, Davis, CA; \n",
      "abbreviated NeuroMab; Cat. no. 75-132, clone N86/38, RRID:AB_10672529) was generated …\n",
      "\n",
      "The strength of synaptic inhibition can be controlled by the stability and endocytosis of \n",
      "surface and synaptic GABA A receptors (GABA A Rs), but the surface receptor dynamics that …\n",
      "\n",
      "… The monoclonal anti-PSD95 (clone K28/43), GABA B R1 (clone NR3A/49) and GABA B R2 \n",
      "(clone N81/2) antibodies were obtained from the UC Davis/NIH NeuroMab Facility, supported …\n",
      "\n",
      "… Mouse monoclonal antibodies were developed by UC Davis/NIH NeuroMab Facility in mouse \n",
      "generating hybridomas. The anti-GABAAR α1 mouse monoclonal antibody (clone N95/35 …\n",
      "\n",
      "The axon initial segment (AIS) contains the site of action potential initiation and plays a \n",
      "major role in neuronal excitability. AIS function relies on high concentrations of different ion …\n",
      "\n",
      "Purpose Intracellular free calcium ions (Ca 2+) are an important element in retinal ganglion \n",
      "cell response. Two major EF-hand (E-helix-loop-F-helix-hand) calcium binding proteins in …\n",
      "\n",
      "Proteins that interact with voltage-gated sodium (Na v ) channels are important in channel \n",
      "sorting and modulation. In this study, we identified the transcriptional regulator, Sin3B, as a …\n",
      "\n",
      "Purpose: Recent studies indicate the presence of functional voltage-gated sodium channels \n",
      "(Na v channels) in the distal retina in several species. This study examined the distribution …\n",
      "\n",
      "Purpose: Recent studies indicate the presence of functional voltage-gated sodium channels \n",
      "(Nav channels) in the distal retina in several species. This study examined the distribution of …\n",
      "\n",
      "In neurons, voltage-gated sodium channel β subunits regulate the expression levels, subcellular \n",
      "localization, and electrophysiological properties of sodium channel α subunits. However…\n",
      "\n",
      "… ) and mouse monoclonal (clone K74/71, NeuroMab) antibodies raised respectively against \n",
      "… monoclonal (clone K87A/10, NeuroMab) antibodies raised respectively against amino acids …\n",
      "\n",
      "Alzheimer's disease (AD) is the most frequent neurodegenerative disease and a common \n",
      "dementia in elderly individuals. Previous studies found a strong correlation between axon …\n",
      "\n",
      "The axon initial segment (AIS) plays an important role in maintaining neuronal polarity and \n",
      "initiating action potentials (APs). The AIS adapts to its environment by changing its length and …\n",
      "\n",
      "Cerebellar granule (CG) cells generate high‐frequency action potentials that have been \n",
      "proposed to depend on the unique properties of their voltage‐gated ion channels. To address …\n",
      "\n",
      "Precise axon pathfinding is crucial for establishment of the initial neuronal network during \n",
      "development. Pioneer axons navigate without the help of preexisting axons and pave the way …\n",
      "\n",
      "… The anti-neurofascin antibody (extNF; A12/18, 75-172) and anti-Nav1.2 antibody (K69/3) \n",
      "were obtained from NeuroMab (USA). The anti-ankyrin G antibody (33-8800) was purchased …\n",
      "\n",
      "The role played by the transient receptor potential vanilloid 1 (TRPV1) channel on the thin \n",
      "fibre afferents evoking the exercise pressor reflex is controversial. To shed light on this …\n",
      "\n",
      "Dopamine‐ and tyrosine hydroxylase–immunopositive cells (TH cells) modulate visually \n",
      "driven signals as they flow through retinal photoreceptor, bipolar, and ganglion cells. Previous …\n",
      "\n",
      "Growth of a properly complex dendrite arbor is a key step in neuronal differentiation and a \n",
      "prerequisite for neural circuit formation. Diverse cell surface molecules, such as the clustered …\n",
      "\n",
      "… :500–1:1000) were generated by NeuroMab in collaboration with the Weiner laboratory [55] … \n",
      "); N148/30 (specific for γB2); N174B/27 (specific for γC3). A rabbit polyclonal antibody raised …\n",
      "\n",
      "SLITRK1 is an obsessive-compulsive disorder spectrum-disorders-associated gene that \n",
      "encodes a neuronal transmembrane protein. Here we show that SLITRK1 suppresses …\n",
      "\n",
      "… Anti-vesicular glutamate transporter 1 (VGLUT1; NeuroMAB, University of California, \n",
      "Davis, Davis, CA; clone N28/9), a mouse monoclonal antibody, raised against fusion protein …\n",
      "\n",
      "The proper formation and maintenance of functional synapses in the central nervous system \n",
      "(CNS) requires communication between neurons and astrocytes and the ability of …\n",
      "\n",
      "Brain-derived neurotrophic factor (BDNF) signals through its high affinity receptor Tropomyosin \n",
      "receptor kinase-B (TrkB) to regulate neuronal development, synapse formation and …\n",
      "\n",
      "The accumulation of insoluble amyloid-beta (Aβ) peptides is associated with neurodegenerative \n",
      "disorders, such as Alzheimer's disease (AD). As essential tremor (ET) could involve …\n",
      "\n",
      "PPM1H phosphatase reverses Parkinson's disease-associated, LRRK2-mediated Rab \n",
      "GTPase phosphorylation. We show here that PPM1H relies on an N-terminal amphipathic helix …\n",
      "\n",
      "Mutations in the leucine-rich repeat kinase-2 (LRRK2) gene cause late-onset Parkinson’s \n",
      "disease, but its physiological function has remained largely unknown. Here we report that …\n",
      "\n",
      "Mutations and variations in the leucine-rich repeat kinase 2 (LRRK2) gene are strongly \n",
      "associated with an increased risk to develop Parkinson’s disease (PD). Most pathogenic LRRK2 …\n",
      "\n",
      "… The panel shows a TEM image of uEVs and dual immunogold labeling with anti-LRRK2 \n",
      "antibodies (RabMab UDD3 and Neuromab N241, revealed by large and small gold particles, …\n",
      "\n",
      "Parkinson disease (PD) is the most common neurodegenerative movement disorder for which \n",
      "no treatments exist capable of slowing progression of the disease. One obstacle slowing …\n",
      "\n",
      "… For blots, primary antibodies used were: Mouse anti-total LRRK2 (Neuromab N241A/34), \n",
      "Rabbit antiLRRK2 pS935 (UDD2 10(12), MRC PPU Reagents and Services, or ab133450, …\n",
      "\n",
      "The Parkinson’s VPS35[D620N] mutation causes lysosome dysfunction enhancing LRRK2 \n",
      "kinase activity. We find the VPS35[D620N] mutation alters expression of ∼350 lysosomal …\n",
      "\n",
      "… The whole brain crude membrane fractions from WT (2 μg total protein) and KI (20 μg total \n",
      "protein) mice were analyzed with an anti-PSD-95 K28/43 antibody. F, Immunohistochemical …\n",
      "\n",
      "Compact myelin, the paranode, and the juxtaparanode are discrete domains that are formed \n",
      "on myelinated axons. In humans, neurological disorders associated with loss of myelin, …\n",
      "\n",
      "Ionotropic neurotransmitter receptors mediate fast synaptic transmission by functioning as \n",
      "ligand-gated ion channels. Fast inhibitory transmission in the brain is mediated mostly by …\n",
      "\n",
      "Autism spectrum disorder (ASD) is a neurological condition characterized by difficulties in \n",
      "social 22 interaction, communication, and behavior. The classical type II cadherins cadherin-8 (…\n",
      "\n",
      "AMPA receptors and interacting proteins are importantly involved in mediating stress-dependent \n",
      "plasticity. Previously we reported that GluA1-containing AMPA receptors and their …\n",
      "\n",
      "Calsyntenin-3 (Clstn3) is a postsynaptic adhesion molecule that induces presynaptic \n",
      "differentiation via presynaptic neurexins (Nrxns), but whether Nrxns directly bind to Clstn3 has …\n",
      "\n",
      "Gephyrin is a central scaffold protein that mediates development, function, and plasticity of \n",
      "mammalian inhibitory synapses by interacting with various inhibitory synaptic proteins. Here, …\n",
      "\n",
      "Depolarization drives neuronal plasticity. However, whether depolarization drives \n",
      "sensitization of peripheral nociceptive neurons remains elusive. By high-content screening (HCS) …\n",
      "\n",
      "… The NeuroMab KChIP1 antibody stained a double band with … The NeuroMab KChIP2 antibody \n",
      "stained a single band with a … The NeuroMab KChIP3 antibody stained a single band with …\n",
      "\n",
      "… The specificity of each of the primary antibodies has been verified using Western blots \n",
      "from brain tissue by the UC, Davis/NeuroMab Facility and also in our laboratory in BLA tissue …\n",
      "\n",
      "… Mouse anti-KChIP1 (University of California at Davis / NIH NeuroMab Facility, Davis, CA; \n",
      "abbreviated NeuroMab; Cat. no. 75-003, clone K55/7, RRID:AB_10673162) was generated …\n",
      "\n",
      "Potassium channels comprise the most diverse family of ion channels. In nerve cells, their \n",
      "critical roles in synaptic integration and output generation have been demonstrated. Here, we …\n",
      "\n",
      "… We used the following primary antibodies from NeuroMab (UC Davis/NIH, Davis, CA, \n",
      "USA): (1) monoclonal anti-Kv4.2 (K57/1; amino acid. 209–225 of human Kv4.2, Q9NZV8), (2) …\n",
      "\n",
      "… The Ms mAb to pan-Pcdh-γ (clone N159/5, to amino acids 808–931 of the constant \n",
      "cytoplasmic domain of mouse Pcdh-γA1 that is shared by all 22 Pcdh-γs) and the Ms mAb to pan-…\n",
      "\n",
      "It has been proposed that the combinatorial expression of γ‐protocadherins (Pcdh‐γs) and \n",
      "other clustered protocadherins (Pcdhs) provides a code of molecular identity and individuality …\n",
      "\n",
      "… The mouse anti-γ-Pcdh constant domain (N159/5) and mouse anti-γ-Pcdh-B2 (N148/30) \n",
      "were generated by the UC Davis/NIH NeuroMab Facility using antigens produced in our …\n",
      "\n",
      "Hyperpolarisation-activated (I h ) currents are considered important for dendritic integration, \n",
      "synaptic transmission, setting membrane potential and rhythmic action potential (AP) …\n",
      "\n",
      "Kv4 channels mediate most of the somatodendritic subthreshold operating A‐type current (I \n",
      "SA ) in neurons. This current plays essential roles in the regulation of spike timing, repetitive …\n",
      "\n",
      "… Immunodetections of K v 4.2 and K v 4.3 channels were performed using monoclonal \n",
      "antibodies from Neuro Mab (clones K57/1 and K75/41, respectively). These antibodies have been …\n",
      "\n",
      "… -S2 loop, Q9NZV8; NeuroMab, UC Davis/NIH, Davis, CA, USA); monoclonal anti-Kv4.3 (K75/41; \n",
      "aa. 415–636 of rat Kv4.3, cytoplasmic C-terminus, Q62897; NeuroMab, UC Davis/NIH, …\n",
      "\n",
      "… K75/41 Neuromab; www.neuromab.org were used for immunolocalization experiments in \n",
      "brain. Immunoperoxidase [DPP6 antibodies diluted (1:1000); Kv4.2 (1:2000)] and …\n",
      "\n",
      "Inhibitory neurons control the firing of glutamatergic neurons and synchronize brain activity. \n",
      "However, little is known about mechanisms of excitatory synapse formation in inhibitory …\n",
      "\n",
      "Establishment, specification, and validation of synaptic connections are thought to be mediated \n",
      "by interactions between pre- and postsynaptic cell-adhesion molecules. Arguably, the …\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# print the snippets out line by line for the google scholar search\n",
    "for index, row in df_final.iterrows():\n",
    "    print(row['snippet'])\n",
    "    print('')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fa17ffe9",
   "metadata": {},
   "source": [
    "# Final Output"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6b197e8f",
   "metadata": {},
   "source": [
    "## TODO merge in the old outputs we are missing as well ~600 and talk to Jim about best strategy for those"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "c36dfd8c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Chemical corrector treatment ameliorates incre...</td>\n",
       "      <td>https://www.nature.com/articles/nm.3759</td>\n",
       "      <td>N Yokoi, Y Fukata, D Kase, T Miyazaki, M Jaegl...</td>\n",
       "      <td>Epilepsy is one of the most common and intract...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=35472...</td>\n",
       "      <td>74.0</td>\n",
       "      <td>https://www.nature.com/articles/nm.3759.pdf</td>\n",
       "      <td>N144/14</td>\n",
       "      <td>6xHis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>669</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K36/15</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>739</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K13/31</td>\n",
       "      <td>Kv1.4 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>781</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>L23/27</td>\n",
       "      <td>Kv1.3 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>811</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K14/16</td>\n",
       "      <td>Kv1.2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>838</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>N46/30</td>\n",
       "      <td>ADAM22 (cytoplasmic)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>846</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>N56/21</td>\n",
       "      <td>FGF14/FHF4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1001</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K65/35</td>\n",
       "      <td>CASPR/Neurexin IV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1036</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K67/25</td>\n",
       "      <td>CASPR2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "10    Chemical corrector treatment ameliorates incre...   \n",
       "669    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "739    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "781    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "811    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "838    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "846   Proteomic analysis of native cerebellar iFGF14...   \n",
       "1001   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1036   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "\n",
       "                                             title_link  \\\n",
       "10              https://www.nature.com/articles/nm.3759   \n",
       "669   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "739   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "781   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "811   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "838   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "846   https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "1001  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1036  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "\n",
       "                                       publication_info  \\\n",
       "10    N Yokoi, Y Fukata, D Kase, T Miyazaki, M Jaegl...   \n",
       "669   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "739   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "781   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "811   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "838   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "846   MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "1001  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1036  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "\n",
       "                                                snippet  \\\n",
       "10    Epilepsy is one of the most common and intract...   \n",
       "669   Neurodevelopmental disorders, such as intellec...   \n",
       "739   Neurodevelopmental disorders, such as intellec...   \n",
       "781   Neurodevelopmental disorders, such as intellec...   \n",
       "811   Neurodevelopmental disorders, such as intellec...   \n",
       "838   Neurodevelopmental disorders, such as intellec...   \n",
       "846   Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "1001  Neurodevelopmental disorders, such as intellec...   \n",
       "1036  Neurodevelopmental disorders, such as intellec...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "10    https://scholar.google.com/scholar?cites=35472...            74.0   \n",
       "669   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "739   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "781   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "811   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "838   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "846   https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "1001  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1036  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "\n",
       "                                               pdf_file Antibody  \\\n",
       "10          https://www.nature.com/articles/nm.3759.pdf  N144/14   \n",
       "669   https://www.cell.com/cell-reports/pdf/S2211-12...   K36/15   \n",
       "739   https://www.cell.com/cell-reports/pdf/S2211-12...   K13/31   \n",
       "781   https://www.cell.com/cell-reports/pdf/S2211-12...   L23/27   \n",
       "811   https://www.cell.com/cell-reports/pdf/S2211-12...   K14/16   \n",
       "838   https://www.cell.com/cell-reports/pdf/S2211-12...   N46/30   \n",
       "846   https://www.tandfonline.com/doi/pdf/10.1080/19...   N56/21   \n",
       "1001  https://www.cell.com/cell-reports/pdf/S2211-12...   K65/35   \n",
       "1036  https://www.cell.com/cell-reports/pdf/S2211-12...   K67/25   \n",
       "\n",
       "                           Target  \n",
       "10                          6xHis  \n",
       "669   Kv1.1 K+ channel (external)  \n",
       "739              Kv1.4 K+ channel  \n",
       "781              Kv1.3 K+ channel  \n",
       "811              Kv1.2 K+ channel  \n",
       "838          ADAM22 (cytoplasmic)  \n",
       "846                    FGF14/FHF4  \n",
       "1001            CASPR/Neurexin IV  \n",
       "1036                       CASPR2  "
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# read this file googescholar_mabidsearch\n",
    "df_final_old= pd.read_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/googescholar_mabidsearch_5-20-24.csv\")\n",
    "df_final_old[df_final_old['publication_info'].str.contains('Miyazaki')]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "de60cadc",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>Author</th>\n",
       "      <th>Publication Date</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Date</th>\n",
       "      <th>Publication</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>920</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K36/15</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>969</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K13/31</td>\n",
       "      <td>Kv1.4 K+ channel</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1003</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>L23/27</td>\n",
       "      <td>Kv1.3 K+ channel</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1026</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K14/16</td>\n",
       "      <td>Kv1.2 K+ channel</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1045</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>N46/30</td>\n",
       "      <td>ADAM22 (cytoplasmic)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1051</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>N56/21</td>\n",
       "      <td>FGF14/FHF4</td>\n",
       "      <td>26889602.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2016-02-19</td>\n",
       "      <td>Bosch et al. Channels (Austin, Tex.)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1168</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K65/35</td>\n",
       "      <td>CASPR/Neurexin IV</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1188</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K67/25</td>\n",
       "      <td>CASPR2</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1894</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>K74/71</td>\n",
       "      <td>Nav1.1 Na+ channel</td>\n",
       "      <td>26889602.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2016-02-19</td>\n",
       "      <td>Bosch et al. Channels (Austin, Tex.)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1918</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>K69/3</td>\n",
       "      <td>Nav1.2 Na+ channel</td>\n",
       "      <td>26889602.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2016-02-19</td>\n",
       "      <td>Bosch et al. Channels (Austin, Tex.)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3313</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>N46/30, K36/15, K67/25, L23/27, K13/31, K65/35</td>\n",
       "      <td>ADAM22 (cytoplasmic)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3316</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>K74/71, K69/3, N56/21</td>\n",
       "      <td>FGF14/FHF4</td>\n",
       "      <td>26889602.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2016-02-19</td>\n",
       "      <td>Bosch et al. Channels (Austin, Tex.)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "920    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "969    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1003   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1026   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1045   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1051  Proteomic analysis of native cerebellar iFGF14...   \n",
       "1168   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1188   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1894  Proteomic analysis of native cerebellar iFGF14...   \n",
       "1918  Proteomic analysis of native cerebellar iFGF14...   \n",
       "3313   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "3316  Proteomic analysis of native cerebellar iFGF14...   \n",
       "\n",
       "                                             title_link  \\\n",
       "920   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "969   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1003  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1026  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1045  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1051  https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "1168  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1188  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1894  https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "1918  https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "3313  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "3316  https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "\n",
       "                                       publication_info  \\\n",
       "920   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "969   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1003  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1026  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1045  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1051  MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "1168  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1188  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1894  MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "1918  MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "3313  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "3316  MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "\n",
       "                                                snippet  \\\n",
       "920   Neurodevelopmental disorders, such as intellec...   \n",
       "969   Neurodevelopmental disorders, such as intellec...   \n",
       "1003  Neurodevelopmental disorders, such as intellec...   \n",
       "1026  Neurodevelopmental disorders, such as intellec...   \n",
       "1045  Neurodevelopmental disorders, such as intellec...   \n",
       "1051  Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "1168  Neurodevelopmental disorders, such as intellec...   \n",
       "1188  Neurodevelopmental disorders, such as intellec...   \n",
       "1894  Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "1918  Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "3313  Neurodevelopmental disorders, such as intellec...   \n",
       "3316  Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "920   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "969   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1003  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1026  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1045  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1051  https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "1168  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1188  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1894  https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "1918  https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "3313                                                NaN             NaN   \n",
       "3316  https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "\n",
       "                                               pdf_file  \\\n",
       "920   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "969   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1003  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1026  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1045  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1051  https://www.tandfonline.com/doi/pdf/10.1080/19...   \n",
       "1168  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1188  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1894  https://www.tandfonline.com/doi/pdf/10.1080/19...   \n",
       "1918  https://www.tandfonline.com/doi/pdf/10.1080/19...   \n",
       "3313  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "3316  https://www.tandfonline.com/doi/pdf/10.1080/19...   \n",
       "\n",
       "                                            Antibody  \\\n",
       "920                                           K36/15   \n",
       "969                                           K13/31   \n",
       "1003                                          L23/27   \n",
       "1026                                          K14/16   \n",
       "1045                                          N46/30   \n",
       "1051                                          N56/21   \n",
       "1168                                          K65/35   \n",
       "1188                                          K67/25   \n",
       "1894                                          K74/71   \n",
       "1918                                           K69/3   \n",
       "3313  N46/30, K36/15, K67/25, L23/27, K13/31, K65/35   \n",
       "3316                           K74/71, K69/3, N56/21   \n",
       "\n",
       "                           Target        PMID  Author  Publication Date  \\\n",
       "920   Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "969              Kv1.4 K+ channel  38194969.0     NaN               NaN   \n",
       "1003             Kv1.3 K+ channel  38194969.0     NaN               NaN   \n",
       "1026             Kv1.2 K+ channel  38194969.0     NaN               NaN   \n",
       "1045         ADAM22 (cytoplasmic)  38194969.0     NaN               NaN   \n",
       "1051                   FGF14/FHF4  26889602.0     NaN               NaN   \n",
       "1168            CASPR/Neurexin IV  38194969.0     NaN               NaN   \n",
       "1188                       CASPR2  38194969.0     NaN               NaN   \n",
       "1894           Nav1.1 Na+ channel  26889602.0     NaN               NaN   \n",
       "1918           Nav1.2 Na+ channel  26889602.0     NaN               NaN   \n",
       "3313         ADAM22 (cytoplasmic)  38194969.0     NaN               NaN   \n",
       "3316                   FGF14/FHF4  26889602.0     NaN               NaN   \n",
       "\n",
       "      Journal        Date                           Publication  \n",
       "920       NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "969       NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1003      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1026      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1045      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1051      NaN  2016-02-19  Bosch et al. Channels (Austin, Tex.)  \n",
       "1168      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1188      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1894      NaN  2016-02-19  Bosch et al. Channels (Austin, Tex.)  \n",
       "1918      NaN  2016-02-19  Bosch et al. Channels (Austin, Tex.)  \n",
       "3313      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "3316      NaN  2016-02-19  Bosch et al. Channels (Austin, Tex.)  "
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_final = pd.read_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/googlescholar_mabidsearch_pubmed_5-20-24.csv\")\n",
    "# filter for Miyazaki\n",
    "df_final[df_final['publication_info'].str.contains('Miyazaki')]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "5fac45d8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>Author</th>\n",
       "      <th>Publication Date</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Date</th>\n",
       "      <th>Publication</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>920</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K36/15</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>969</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K13/31</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1003</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>L23/27</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1026</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K14/16</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1045</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>N46/30</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1051</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>N56/21</td>\n",
       "      <td>FGF14/FHF4</td>\n",
       "      <td>26889602.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2016-02-19</td>\n",
       "      <td>Bosch et al. Channels (Austin, Tex.)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1168</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K65/35</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1188</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=40492...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>K67/25</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1894</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>K74/71</td>\n",
       "      <td>FGF14/FHF4</td>\n",
       "      <td>26889602.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2016-02-19</td>\n",
       "      <td>Bosch et al. Channels (Austin, Tex.)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1918</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>K69/3</td>\n",
       "      <td>FGF14/FHF4</td>\n",
       "      <td>26889602.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2016-02-19</td>\n",
       "      <td>Bosch et al. Channels (Austin, Tex.)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3313</th>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor...</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...</td>\n",
       "      <td>Neurodevelopmental disorders, such as intellec...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.cell.com/cell-reports/pdf/S2211-12...</td>\n",
       "      <td>N46/30, K36/15, K67/25, L23/27, K13/31, K65/35</td>\n",
       "      <td>Kv1.1 K+ channel (external)</td>\n",
       "      <td>38194969.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2024-01-10</td>\n",
       "      <td>Miyazaki et al. Cell reports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3316</th>\n",
       "      <td>Proteomic analysis of native cerebellar iFGF14...</td>\n",
       "      <td>https://www.tandfonline.com/doi/abs/10.1080/19...</td>\n",
       "      <td>MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...</td>\n",
       "      <td>Intracellular Fibroblast Growth Factor 14 (iFG...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=13497...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/19...</td>\n",
       "      <td>K74/71, K69/3, N56/21</td>\n",
       "      <td>FGF14/FHF4</td>\n",
       "      <td>26889602.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2016-02-19</td>\n",
       "      <td>Bosch et al. Channels (Austin, Tex.)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  title  \\\n",
       "920    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "969    Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1003   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1026   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1045   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1051  Proteomic analysis of native cerebellar iFGF14...   \n",
       "1168   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1188   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "1894  Proteomic analysis of native cerebellar iFGF14...   \n",
       "1918  Proteomic analysis of native cerebellar iFGF14...   \n",
       "3313   Oligodendrocyte-derived LGI3 and its receptor...   \n",
       "3316  Proteomic analysis of native cerebellar iFGF14...   \n",
       "\n",
       "                                             title_link  \\\n",
       "920   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "969   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1003  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1026  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1045  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1051  https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "1168  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1188  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1894  https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "1918  https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "3313  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "3316  https://www.tandfonline.com/doi/abs/10.1080/19...   \n",
       "\n",
       "                                       publication_info  \\\n",
       "920   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "969   Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1003  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1026  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1045  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1051  MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "1168  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1188  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "1894  MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "1918  MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "3313  Y Miyazaki, T Otsuka, Y Yamagata, T Endo, M Sa...   \n",
       "3316  MK Bosch, JM Nerbonne, RR Townsend, H Miyazaki...   \n",
       "\n",
       "                                                snippet  \\\n",
       "920   Neurodevelopmental disorders, such as intellec...   \n",
       "969   Neurodevelopmental disorders, such as intellec...   \n",
       "1003  Neurodevelopmental disorders, such as intellec...   \n",
       "1026  Neurodevelopmental disorders, such as intellec...   \n",
       "1045  Neurodevelopmental disorders, such as intellec...   \n",
       "1051  Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "1168  Neurodevelopmental disorders, such as intellec...   \n",
       "1188  Neurodevelopmental disorders, such as intellec...   \n",
       "1894  Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "1918  Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "3313  Neurodevelopmental disorders, such as intellec...   \n",
       "3316  Intracellular Fibroblast Growth Factor 14 (iFG...   \n",
       "\n",
       "                                          cited_by_link  cited_by_count  \\\n",
       "920   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "969   https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1003  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1026  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1045  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1051  https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "1168  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1188  https://scholar.google.com/scholar?cites=40492...             1.0   \n",
       "1894  https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "1918  https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "3313                                                NaN             NaN   \n",
       "3316  https://scholar.google.com/scholar?cites=13497...            14.0   \n",
       "\n",
       "                                               pdf_file  \\\n",
       "920   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "969   https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1003  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1026  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1045  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1051  https://www.tandfonline.com/doi/pdf/10.1080/19...   \n",
       "1168  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1188  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "1894  https://www.tandfonline.com/doi/pdf/10.1080/19...   \n",
       "1918  https://www.tandfonline.com/doi/pdf/10.1080/19...   \n",
       "3313  https://www.cell.com/cell-reports/pdf/S2211-12...   \n",
       "3316  https://www.tandfonline.com/doi/pdf/10.1080/19...   \n",
       "\n",
       "                                            Antibody  \\\n",
       "920                                           K36/15   \n",
       "969                                           K13/31   \n",
       "1003                                          L23/27   \n",
       "1026                                          K14/16   \n",
       "1045                                          N46/30   \n",
       "1051                                          N56/21   \n",
       "1168                                          K65/35   \n",
       "1188                                          K67/25   \n",
       "1894                                          K74/71   \n",
       "1918                                           K69/3   \n",
       "3313  N46/30, K36/15, K67/25, L23/27, K13/31, K65/35   \n",
       "3316                           K74/71, K69/3, N56/21   \n",
       "\n",
       "                           Target        PMID  Author  Publication Date  \\\n",
       "920   Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "969   Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "1003  Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "1026  Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "1045  Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "1051                   FGF14/FHF4  26889602.0     NaN               NaN   \n",
       "1168  Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "1188  Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "1894                   FGF14/FHF4  26889602.0     NaN               NaN   \n",
       "1918                   FGF14/FHF4  26889602.0     NaN               NaN   \n",
       "3313  Kv1.1 K+ channel (external)  38194969.0     NaN               NaN   \n",
       "3316                   FGF14/FHF4  26889602.0     NaN               NaN   \n",
       "\n",
       "      Journal        Date                           Publication  \n",
       "920       NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "969       NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1003      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1026      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1045      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1051      NaN  2016-02-19  Bosch et al. Channels (Austin, Tex.)  \n",
       "1168      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1188      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "1894      NaN  2016-02-19  Bosch et al. Channels (Austin, Tex.)  \n",
       "1918      NaN  2016-02-19  Bosch et al. Channels (Austin, Tex.)  \n",
       "3313      NaN  2024-01-10          Miyazaki et al. Cell reports  \n",
       "3316      NaN  2016-02-19  Bosch et al. Channels (Austin, Tex.)  "
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# for each row in the df_final, if df_final_old contains the title and title_link then replace target with the target from that dataframe\n",
    "for index, row in df_final.iterrows():\n",
    "    if len(df_final_old[(df_final_old['title'] == row['title']) & (df_final_old['title_link'] == row['title_link'])]) > 0:\n",
    "        df_final.at[index, 'Target'] = df_final_old[(df_final_old['title'] == row['title']) & (df_final_old['title_link'] == row['title_link'])]['Target'].values[0]\n",
    "df_final[df_final['publication_info'].str.contains('Miyazaki')]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "ef2d1c8a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "       TargetType                   Target AccessionNum     Clone  \\\n",
      "0     Epigenetics         5-formylcytidine          NaN   N358/68   \n",
      "1     Epigenetics  5-hydroxymethylcytidine          NaN   N297/59   \n",
      "2             Tag                    6xHis          NaN   N144/14   \n",
      "3       Receptors                   ADAM11       Q9R1V4   N441/35   \n",
      "4       Receptors     ADAM22 (cytoplasmic)       Q9R1V6    N46/30   \n",
      "..            ...                      ...          ...       ...   \n",
      "478  Rare Disease                      WRN       Q14191   N428/12   \n",
      "479         Other     Zebrafish PSD Marker          NaN   N286/74   \n",
      "480         Other                  Zinedin       Q9NRL3    K88/64   \n",
      "481  Transporters                     ZIP3       Q99K24    N476/9   \n",
      "482         Other                   ZNF423       Q2M1K9  N328B/37   \n",
      "\n",
      "    HumanGeneName DataSheetFileName IsoType ValidationT ValidationBrIB  \\\n",
      "0            ----       N358_68.pdf    IgG1         NaN            NaN   \n",
      "1            ----       N297_59.pdf   IgG2a         NaN            NaN   \n",
      "2            ----       N144_14.pdf    IgG1        Pass            NaN   \n",
      "3          ADAM11       N441_35.pdf   IgG2b        Pass           Pass   \n",
      "4          ADAM22        N46_30.pdf   IgG2b        Pass           Pass   \n",
      "..            ...               ...     ...         ...            ...   \n",
      "478           WRN       N428_12.pdf    IgG1        Pass           Pass   \n",
      "479          ----       N286_74.pdf    IgG1          ND             ND   \n",
      "480         STRN4        K88_64.pdf   IgG2a        Pass           Pass   \n",
      "481       SLC39A3        N476_9.pdf    IgG1        Pass           Pass   \n",
      "482        ZNF423      N328B_37.pdf    IgG1        Pass           Pass   \n",
      "\n",
      "    ValidationBrIHC ValidationKO   RRIDAB  \n",
      "0               NaN          NaN  2877251  \n",
      "1               NaN          NaN  2877253  \n",
      "2               NaN          NaN  2877254  \n",
      "3              Pass           ND  2877610  \n",
      "4              Pass         Pass  2877292  \n",
      "..              ...          ...      ...  \n",
      "478            Pass           ND  2877600  \n",
      "479              ND           ND  2877461  \n",
      "480            Pass           ND  2877335  \n",
      "481            Pass           ND  2716255  \n",
      "482            Pass         Pass  2877483  \n",
      "\n",
      "[483 rows x 12 columns]\n",
      "['N37A/10']\n",
      "['Kv7.1/KCNQ1 K+ channel']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N212/17']\n",
      "['TRIP8b (constant)']\n",
      "['N212/17']\n",
      "['TRIP8b (constant)']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N106/36', 'N105/17', 'N144/14']\n",
      "['6xHis', 'Ankyrin-B', 'Ankyrin-G (staining)']\n",
      "['N106/36', 'N144/14', 'N105/17']\n",
      "['6xHis', 'Ankyrin-B', 'Ankyrin-G (staining)']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N100/13', 'N144/14', 'N16B/8']\n",
      "['6xHis', 'GST', 'Kv3.1b K+ channel']\n",
      "['N100/13', 'N144/14', 'N16B/8']\n",
      "['6xHis', 'GST', 'Kv3.1b K+ channel']\n",
      "['N100/13', 'N144/14', 'N16B/8']\n",
      "['6xHis', 'GST', 'Kv3.1b K+ channel']\n",
      "['N100/13', 'N144/14', 'N16B/8']\n",
      "['6xHis', 'GST', 'Kv3.1b K+ channel']\n",
      "['N100/13', 'N144/14', 'N16B/8']\n",
      "['6xHis', 'GST', 'Kv3.1b K+ channel']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N385/21', 'L114/3', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N385/21', 'L114/3', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N385/21', 'L114/3', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N385/21', 'L114/3', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N385/21', 'L114/3', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N385/21', 'L114/3', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N385/21', 'L114/3', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N385/21', 'L114/3', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N385/21', 'N255/38', 'N419/78']\n",
      "['Beta1-spectrin', 'Nav1.4 Na+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N385/21', 'N255/38', 'N419/78']\n",
      "['Beta1-spectrin', 'Nav1.4 Na+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N385/21', 'N255/38', 'N419/78']\n",
      "['Beta1-spectrin', 'Nav1.4 Na+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N385/21', 'N255/38', 'N419/78']\n",
      "['Beta1-spectrin', 'Nav1.4 Na+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N385/21', 'N255/38', 'N419/78']\n",
      "['Beta1-spectrin', 'Nav1.4 Na+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N385/21', 'K14/16', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Kv1.2 K+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N385/21', 'K14/16', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Kv1.2 K+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N385/21', 'K14/16', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Kv1.2 K+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N385/21', 'K14/16', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Kv1.2 K+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N385/21', 'K14/16', 'N106/36', 'N419/78']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Kv1.2 K+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'L117/1']\n",
      "['IRSp53/BAIAP2', 'Pan-Shank']\n",
      "['N23B/49', 'L117/1']\n",
      "['IRSp53/BAIAP2', 'Pan-Shank']\n",
      "['N23B/49', 'L117/1']\n",
      "['IRSp53/BAIAP2', 'Pan-Shank']\n",
      "['N23B/49', 'L117/1']\n",
      "['IRSp53/BAIAP2', 'Pan-Shank']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49', 'N367/62', 'N22/21', 'N23B/6']\n",
      "['Pan-Shank', 'Shank1', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N22/21', 'N23B/6']\n",
      "['Pan-Shank', 'Shank1', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N22/21', 'N23B/6']\n",
      "['Pan-Shank', 'Shank1', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N22/21', 'N23B/6']\n",
      "['Pan-Shank', 'Shank1', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N22/21', 'N23B/6']\n",
      "['Pan-Shank', 'Shank1', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N22/21', 'N23B/6']\n",
      "['Pan-Shank', 'Shank1', 'Shank2', 'Shank3']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['62/3G1', 'L106/93', 'N97A/31']\n",
      "['Gephyrin', 'Neuroligin-1']\n",
      "['L20/8', 'N59/20', 'N19/2', 'N23B/49', 'K64/15', 'N97A/31']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['L20/8', 'N59/20', 'N19/2', 'N23B/49', 'K64/15', 'N97A/31']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['L20/8', 'N59/20', 'N19/2', 'N23B/49', 'K64/15', 'N97A/31']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['L20/8', 'N59/20', 'N19/2', 'N23B/49', 'K64/15', 'N97A/31']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['L20/8', 'N59/20', 'N19/2', 'N23B/49', 'K64/15', 'N97A/31']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['L20/8', 'N59/20', 'N19/2', 'N23B/49', 'K64/15', 'N97A/31']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['L20/8', 'N59/20', 'N19/2', 'N23B/49', 'K64/15', 'N97A/31']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['L20/8', 'N59/20', 'N19/2', 'N23B/49', 'K64/15', 'N97A/31']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['N327/95', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank']\n",
      "['N327/95', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank']\n",
      "['N327/95', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank']\n",
      "['N327/95', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank']\n",
      "['N327/95', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank']\n",
      "['N327/95', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank']\n",
      "['N23B/49', 'N367/62', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N367/62', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2', 'Shank3']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49', 'N23B/6']\n",
      "['Pan-Shank', 'Shank2']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L20/8', 'N97A/31', 'N59/36', 'N23B/49', 'N19/2', 'K64/15']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['L20/8', 'N19/2', 'N23B/49', 'K64/15', 'N97A/31', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Shank', 'PICK1', 'SAP102', 'SAP97']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49', 'K28/43', 'N19/2']\n",
      "['Pan-Shank', 'PSD-95', 'SAP102']\n",
      "['N23B/49', 'K28/43', 'N19/2']\n",
      "['Pan-Shank', 'PSD-95', 'SAP102']\n",
      "['N23B/49', 'K28/43', 'N19/2']\n",
      "['Pan-Shank', 'PSD-95', 'SAP102']\n",
      "['N23B/49', 'K28/43', 'N19/2']\n",
      "['Pan-Shank', 'PSD-95', 'SAP102']\n",
      "['N23B/49', 'K28/43', 'N19/2']\n",
      "['Pan-Shank', 'PSD-95', 'SAP102']\n",
      "['N23B/49', 'K28/43', 'N19/2']\n",
      "['Pan-Shank', 'PSD-95', 'SAP102']\n",
      "['N23B/49', 'K28/43', 'N19/2']\n",
      "['Pan-Shank', 'PSD-95', 'SAP102']\n",
      "['N23B/49', 'K28/43', 'N19/2']\n",
      "['Pan-Shank', 'PSD-95', 'SAP102']\n",
      "['N23B/49', 'K28/43', 'N23B/6']\n",
      "['Pan-Shank', 'PSD-95', 'Shank2']\n",
      "['N23B/49', 'K28/43', 'N23B/6']\n",
      "['Pan-Shank', 'PSD-95', 'Shank2']\n",
      "['N23B/49', 'K28/43', 'N23B/6']\n",
      "['Pan-Shank', 'PSD-95', 'Shank2']\n",
      "['N23B/49', 'K28/43', 'N23B/6']\n",
      "['Pan-Shank', 'PSD-95', 'Shank2']\n",
      "['N23B/49', 'K28/43', 'N23B/6']\n",
      "['Pan-Shank', 'PSD-95', 'Shank2']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N327/95', 'N367/62', 'N22/21', 'N23B/49']\n",
      "['GluN2A/NR2A glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank3']\n",
      "['N23B/49', 'N39B/8']\n",
      "['GIT1', 'Pan-Shank']\n",
      "['N23B/49', 'N39B/8']\n",
      "['GIT1', 'Pan-Shank']\n",
      "['N23B/49', 'N39B/8']\n",
      "['GIT1', 'Pan-Shank']\n",
      "['N23B/49', 'N39B/8']\n",
      "['GIT1', 'Pan-Shank']\n",
      "['N23B/49', 'N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'Pan-Shank', 'VGlut1']\n",
      "['N23B/49', 'N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'Pan-Shank', 'VGlut1']\n",
      "['N23B/49', 'N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'Pan-Shank', 'VGlut1']\n",
      "['N23B/49', 'N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'Pan-Shank', 'VGlut1']\n",
      "['N23B/49', 'N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'Pan-Shank', 'VGlut1']\n",
      "['N23B/49', 'N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'Pan-Shank', 'VGlut1']\n",
      "['N23B/49', 'N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'Pan-Shank', 'VGlut1']\n",
      "['N23B/49', 'N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'Pan-Shank', 'VGlut1']\n",
      "['N125/10']\n",
      "['Thorase/Atad1']\n",
      "['N479/107', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'VAPA + VAPB']\n",
      "['N479/107', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'VAPA + VAPB']\n",
      "['N479/107', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'VAPA + VAPB']\n",
      "['N69/46']\n",
      "['Shank3']\n",
      "['N69/46']\n",
      "['Shank3']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N367/62', 'N69/46']\n",
      "['Shank3', 'Shank3']\n",
      "['N69/46', 'N367/62']\n",
      "['Shank3', 'Shank3']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N415/24', 'N431/64', 'N244/5', 'N52A/42', 'N425/45', 'N71/37', 'N423/75', 'N419/40', 'N388A/27']\n",
      "['Ankyrin-R + Ankyrin-G', 'GABA(A)R, Alpha5', 'GABA(A)R, Pi', 'HCN2', 'KChIP4 K+ channel', 'Mortalin/GRP75', 'Pan-Nav1 Na+ channel', 'SynCAM4', 'VAChT']\n",
      "['N399/19']\n",
      "['GABA(A)R, Alpha2']\n",
      "['N399/19']\n",
      "['GABA(A)R, Alpha2']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8', 'N56/21', 'N106/36', 'N86/38']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'GFP', 'GFP']\n",
      "['N86/8', 'N56/21', 'N106/36', 'N86/38']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'GFP', 'GFP']\n",
      "['N86/8', 'N56/21', 'N106/36', 'N86/38']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'GFP', 'GFP']\n",
      "['N86/8', 'N56/21', 'N106/36', 'N86/38']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'GFP', 'GFP']\n",
      "['N86/8', 'N56/21', 'N106/36', 'N86/38']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'GFP', 'GFP']\n",
      "['N86/8', 'N56/21', 'N106/36', 'N86/38']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'GFP', 'GFP']\n",
      "['N86/8', 'N56/21', 'N106/36', 'N86/38']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'GFP', 'GFP']\n",
      "['N86/8', 'N56/21', 'N106/36', 'N86/38']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'GFP', 'GFP']\n",
      "['N86/8', 'N56/21', 'N106/36', 'N86/38']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'GFP', 'GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8', 'N206A/8']\n",
      "['GFAP', 'GFP']\n",
      "['N86/8', 'N206A/8']\n",
      "['GFAP', 'GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K20/78', 'N167/7', 'N86/38', 'N86/8']\n",
      "['Dicer', 'GFP', 'GFP', 'Kv1.1 K+ channel']\n",
      "['K20/78', 'N167/7', 'N86/38', 'N86/8']\n",
      "['Dicer', 'GFP', 'GFP', 'Kv1.1 K+ channel']\n",
      "['K20/78', 'N167/7', 'N86/38', 'N86/8']\n",
      "['Dicer', 'GFP', 'GFP', 'Kv1.1 K+ channel']\n",
      "['N86/8', 'N241A/34']\n",
      "['GFP', 'LRRK2/Dardarin, C-terminus']\n",
      "['N86/8', 'N52A/42', 'N86/38']\n",
      "['GFP', 'GFP', 'Mortalin/GRP75']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['K89/34', 'K28/86', 'K28/43', 'N28/9', 'N147/6', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Kv2.1 K+ channel', 'Pan-MAGUK', 'Pan-QKI', 'PSD-95', 'VGlut1']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8', '62/3G1', 'N110/29']\n",
      "['GFP', 'Neuroligin-3']\n",
      "['N86/8', '62/3G1', 'N110/29']\n",
      "['GFP', 'Neuroligin-3']\n",
      "['N86/8', '62/3G1', 'N110/29']\n",
      "['GFP', 'Neuroligin-3']\n",
      "['N86/8', '62/3G1', 'N110/29']\n",
      "['GFP', 'Neuroligin-3']\n",
      "['N86/8', '62/3G1', 'N110/29']\n",
      "['GFP', 'Neuroligin-3']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N59/36', 'N19/2']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102']\n",
      "['N59/36', 'N19/2']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102']\n",
      "['N59/36', 'N19/2']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102']\n",
      "['N59/36', 'N19/2']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102']\n",
      "['N59/36', 'N19/2']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102']\n",
      "['N59/36', 'N19/2']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N106/36', 'N86/8', 'K65/35', 'K14/16', 'K87A/10']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N106/36', 'N86/8', 'K65/35', 'K14/16', 'K87A/10']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N106/36', 'N86/8', 'K65/35', 'K14/16', 'K87A/10']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N106/36', 'N86/8', 'K65/35', 'K14/16', 'K87A/10']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N106/36', 'N86/8', 'K65/35', 'K14/16', 'K87A/10']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N106/36', 'N86/8', 'K65/35', 'K14/16', 'K87A/10']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N106/36', 'N86/8', 'K65/35', 'K14/16', 'K87A/10']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8', 'A12/18']\n",
      "['GFP', 'Pan-Neurofascin (extracellular)']\n",
      "['N86/8', 'A12/18']\n",
      "['GFP', 'Pan-Neurofascin (extracellular)']\n",
      "['N86/8', 'A12/18']\n",
      "['GFP', 'Pan-Neurofascin (extracellular)']\n",
      "['N86/8', 'A12/18']\n",
      "['GFP', 'Pan-Neurofascin (extracellular)']\n",
      "['N86/8', 'L20/8']\n",
      "['GFP', 'PICK1']\n",
      "['N86/8', 'L20/8']\n",
      "['GFP', 'PICK1']\n",
      "['N86/8', 'L20/8']\n",
      "['GFP', 'PICK1']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8', 'N86/44']\n",
      "['GFP', 'GFP']\n",
      "['N86/8', 'N22/21', 'N238/31']\n",
      "['GFP', 'SAPAP1 + SAPAP2', 'Shank1']\n",
      "['N86/8', 'N22/21', 'N238/31']\n",
      "['GFP', 'SAPAP1 + SAPAP2', 'Shank1']\n",
      "['N86/8', 'N22/21', 'N238/31']\n",
      "['GFP', 'SAPAP1 + SAPAP2', 'Shank1']\n",
      "['N86/8', 'N22/21', 'N238/31']\n",
      "['GFP', 'SAPAP1 + SAPAP2', 'Shank1']\n",
      "['N86/8', 'N22/21', 'N238/31']\n",
      "['GFP', 'SAPAP1 + SAPAP2', 'Shank1']\n",
      "['N86/8', 'N22/21', 'N238/31']\n",
      "['GFP', 'SAPAP1 + SAPAP2', 'Shank1']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N327/95', 'N59/36', 'L21/32', 'L20/8']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['N327/95', 'N59/36', 'L21/32', 'L20/8']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['N327/95', 'N59/36', 'L21/32', 'L20/8']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['N327/95', 'N59/36', 'L21/32', 'L20/8']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['N327/95', 'N59/36', 'L21/32', 'L20/8']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['N327/95', 'N59/36', 'L21/32', 'L20/8']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['N327/95', 'N59/36', 'L21/32', 'L20/8']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66', 'N241A/34']\n",
      "['Arl13b', 'LRRK2/Dardarin, C-terminus']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N144/32', 'N144/17', 'N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-A3', 'Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N144/32', 'N144/17', 'N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-A3', 'Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N144/32', 'N144/17', 'N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-A3', 'Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N144/32', 'N144/17', 'N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-A3', 'Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N144/32', 'N144/17', 'N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-A3', 'Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N144/32', 'N144/17', 'N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-A3', 'Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N126B/31']\n",
      "['Neuregulin-CRD (Cysteine-rich domain, Type III)']\n",
      "['N126B/31']\n",
      "['Neuregulin-CRD (Cysteine-rich domain, Type III)']\n",
      "['N126B/31', 'N235/22']\n",
      "['Neuregulin-CRD (Cysteine-rich domain, Type III)', 'Pan-FHF-A']\n",
      "['N126B/31', 'N235/22']\n",
      "['Neuregulin-CRD (Cysteine-rich domain, Type III)', 'Pan-FHF-A']\n",
      "['N126B/31', 'N235/22']\n",
      "['Neuregulin-CRD (Cysteine-rich domain, Type III)', 'Pan-FHF-A']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N419/78', 'N385/21', 'N410/17', 'N16B/8', 'L114/3']\n",
      "['Beta1-spectrin', 'Kv3.1b K+ channel', 'Kv3.2 K+ channel', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N419/78', 'N385/21', 'N410/17', 'N16B/8', 'L114/3']\n",
      "['Beta1-spectrin', 'Kv3.1b K+ channel', 'Kv3.2 K+ channel', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N419/78', 'N385/21', 'N410/17', 'N16B/8', 'L114/3']\n",
      "['Beta1-spectrin', 'Kv3.1b K+ channel', 'Kv3.2 K+ channel', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N419/78', 'N385/21', 'N410/17', 'N16B/8', 'L114/3']\n",
      "['Beta1-spectrin', 'Kv3.1b K+ channel', 'Kv3.2 K+ channel', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['N419/78', 'N385/21', 'N410/17', 'N16B/8', 'L114/3']\n",
      "['Beta1-spectrin', 'Kv3.1b K+ channel', 'Kv3.2 K+ channel', 'Pan-Nav1 Na+ channel', 'Parvalbumin']\n",
      "['K87A/10', 'K36/15']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K36/15']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K36/15']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K36/15']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K36/15']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K36/15']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K36/15']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.6 Na+ channel']\n",
      "['K13/31', 'L11A/41', 'K20/78', 'N57/2', 'K14/16', 'N46/30', 'N18/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Pan-Neurofascin', 'PSD-93/Chapsyn-110']\n",
      "['K13/31', 'L11A/41', 'K20/78', 'N57/2', 'K14/16', 'N46/30', 'N18/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Pan-Neurofascin', 'PSD-93/Chapsyn-110']\n",
      "['K13/31', 'L11A/41', 'K20/78', 'N57/2', 'K14/16', 'N46/30', 'N18/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Pan-Neurofascin', 'PSD-93/Chapsyn-110']\n",
      "['K13/31', 'L11A/41', 'K20/78', 'N57/2', 'K14/16', 'N46/30', 'N18/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Pan-Neurofascin', 'PSD-93/Chapsyn-110']\n",
      "['N57/2']\n",
      "['ADAM22 (extracellular)']\n",
      "['N104/37']\n",
      "['SNAT1']\n",
      "['N205B/22']\n",
      "['LRRTM4']\n",
      "['N205B/22']\n",
      "['LRRTM4']\n",
      "['N205B/22']\n",
      "['LRRTM4']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50', 'N205B/22']\n",
      "['CASK', 'LRRTM4']\n",
      "['N205B/22', 'K56A/50']\n",
      "['CASK', 'LRRTM4']\n",
      "['N205B/22']\n",
      "['LRRTM4']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35', 'N205B/22']\n",
      "['GABA(A)R, Alpha1', 'LRRTM4']\n",
      "['N205B/22', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'LRRTM4']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N34/34']\n",
      "['VGlut3']\n",
      "['N29/29', 'N6/38', 'N28/9', 'N34/34']\n",
      "['VAChT', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N6/38', 'N28/9', 'N34/34']\n",
      "['VAChT', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N6/38', 'N28/9', 'N34/34']\n",
      "['VAChT', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N6/38', 'N28/9', 'N34/34']\n",
      "['VAChT', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N6/38', 'N28/9', 'N34/34']\n",
      "['VAChT', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N6/38', 'N28/9', 'N34/34']\n",
      "['VAChT', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N59/36', 'N28/9', 'N34/34']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N59/36', 'N28/9', 'N34/34']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N59/36', 'N28/9', 'N34/34']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N59/36', 'N28/9', 'N34/34']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N59/36', 'N28/9', 'N34/34']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N29/29', 'N59/36', 'N28/9', 'N34/34']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1', 'VGlut2', 'VGlut3']\n",
      "['N359/28']\n",
      "['Foxi3']\n",
      "['K88/64']\n",
      "['Zinedin']\n",
      "['K88/64']\n",
      "['Zinedin']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N182/17', 'N165/38', 'N154/32', 'N147/6', 'N195A/16', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'JMJD2A', 'LAR/PTPRF', 'Pan-QKI', 'QKI-5', 'QKI-6', 'QKI-7']\n",
      "['N183/15', 'N182/17', 'N165/38', 'N154/32', 'N147/6', 'N195A/16', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'JMJD2A', 'LAR/PTPRF', 'Pan-QKI', 'QKI-5', 'QKI-6', 'QKI-7']\n",
      "['N183/15', 'N182/17', 'N165/38', 'N154/32', 'N147/6', 'N195A/16', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'JMJD2A', 'LAR/PTPRF', 'Pan-QKI', 'QKI-5', 'QKI-6', 'QKI-7']\n",
      "['N183/15', 'N182/17', 'N165/38', 'N154/32', 'N147/6', 'N195A/16', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'JMJD2A', 'LAR/PTPRF', 'Pan-QKI', 'QKI-5', 'QKI-6', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['N183/15', 'N106/65', 'N147/6']\n",
      "['Ankyrin-G (staining)', 'Pan-QKI', 'QKI-7']\n",
      "['L6/23', 'L6/60', 'L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Slo1/BKAlpha maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/23', 'L6/60', 'L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Slo1/BKAlpha maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/23', 'L6/60', 'L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Slo1/BKAlpha maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/23', 'L6/60', 'L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Slo1/BKAlpha maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/23', 'L6/60', 'L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Slo1/BKAlpha maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/23', 'L6/60', 'L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Slo1/BKAlpha maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N183/15', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'QKI-7']\n",
      "['N183/15', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'QKI-7']\n",
      "['N121A/1', 'N121A/31']\n",
      "['Pannexin-2', 'Pannexin-2']\n",
      "['N121A/1', 'N121A/31']\n",
      "['Pannexin-2', 'Pannexin-2']\n",
      "['N121A/1', 'N121A/31']\n",
      "['Pannexin-2', 'Pannexin-2']\n",
      "['N121A/1', 'N121A/31']\n",
      "['Pannexin-2', 'Pannexin-2']\n",
      "['N121A/31']\n",
      "['Pannexin-2']\n",
      "['N151/3']\n",
      "['GABA(A)R, Delta']\n",
      "['K39/25', 'N52A/42', 'K89/34', 'L114/3', 'L122/6']\n",
      "['Calretinin', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Mortalin/GRP75', 'Parvalbumin']\n",
      "['K39/25', 'N52A/42', 'K89/34', 'L114/3', 'L122/6']\n",
      "['Calretinin', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Mortalin/GRP75', 'Parvalbumin']\n",
      "['K39/25', 'N52A/42', 'K89/34', 'L114/3', 'L122/6']\n",
      "['Calretinin', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Mortalin/GRP75', 'Parvalbumin']\n",
      "['K39/25', 'N52A/42', 'K89/34', 'L114/3', 'L122/6']\n",
      "['Calretinin', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Mortalin/GRP75', 'Parvalbumin']\n",
      "['K39/25', 'N52A/42', 'K89/34', 'L114/3', 'L122/6']\n",
      "['Calretinin', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Mortalin/GRP75', 'Parvalbumin']\n",
      "['K39/25', 'N52A/42', 'K89/34', 'L114/3', 'L122/6']\n",
      "['Calretinin', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Mortalin/GRP75', 'Parvalbumin']\n",
      "['K39/25', 'N52A/42', 'K89/34', 'L114/3', 'L122/6']\n",
      "['Calretinin', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Mortalin/GRP75', 'Parvalbumin']\n",
      "['K39/25', 'N52A/42', 'K89/34', 'L114/3', 'L122/6']\n",
      "['Calretinin', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Mortalin/GRP75', 'Parvalbumin']\n",
      "['K39/25', 'N52A/42', 'K89/34', 'L114/3', 'L122/6']\n",
      "['Calretinin', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Mortalin/GRP75', 'Parvalbumin']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'N86/38']\n",
      "['GFP', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'N86/38']\n",
      "['GFP', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'N86/38']\n",
      "['GFP', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'N86/38']\n",
      "['GFP', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'N86/38']\n",
      "['GFP', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'L86/33', 'N372B/60']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'L86/33', 'N372B/60']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'L86/33', 'N372B/60']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'L86/33', 'N372B/60']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'L86/33', 'N372B/60']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'L86/33', 'N372B/60']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'L86/33', 'N372B/60']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34', 'L86/33', 'N372B/60']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['L83/11']\n",
      "['Kv2.1 K+ channel']\n",
      "['K13/31', 'N52A/42', 'L83/11', 'K89/34']\n",
      "['Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['L83/11', 'N52A/42', 'K13/31', 'K89/34']\n",
      "['Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['K13/31', 'N52A/42', 'K89/34', 'K14/16', 'K37/89', 'K7/45']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['K13/31', 'N52A/42', 'K89/34', 'K14/16', 'K37/89', 'K7/45']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['K13/31', 'N52A/42', 'K89/34', 'K14/16', 'K37/89', 'K7/45']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['K13/31', 'N52A/42', 'K89/34', 'K14/16', 'K37/89', 'K7/45']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['K13/31', 'N52A/42', 'K89/34', 'K14/16', 'K37/89', 'K7/45']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['K13/31', 'N52A/42', 'K89/34', 'K14/16', 'K37/89', 'K7/45']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['K13/31', 'N52A/42', 'K89/34', 'K14/16', 'K37/89', 'K7/45']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['K13/31', 'N52A/42', 'K89/34', 'K14/16', 'K37/89', 'K7/45']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['K13/31', 'N52A/42', 'K89/34', 'K14/16', 'K37/89', 'K7/45']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N221/17', 'N52A/42', 'N192/12']\n",
      "['Gs protein, alpha subunit', 'Mortalin/GRP75', 'TrpV1']\n",
      "['N221/17', 'N52A/42', 'N192/12']\n",
      "['Gs protein, alpha subunit', 'Mortalin/GRP75', 'TrpV1']\n",
      "['N221/17', 'N52A/42', 'N192/12']\n",
      "['Gs protein, alpha subunit', 'Mortalin/GRP75', 'TrpV1']\n",
      "['N221/17', 'N52A/42', 'N192/12']\n",
      "['Gs protein, alpha subunit', 'Mortalin/GRP75', 'TrpV1']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['L109/39']\n",
      "['Calbindin']\n",
      "['L86A/37']\n",
      "['AMIGO-1']\n",
      "['N468/37']\n",
      "['THIK-2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['L114/3']\n",
      "['Parvalbumin']\n",
      "['N398A/34']\n",
      "['GABA(A)R, Alpha4']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['L86A/37', 'N52A/42', 'N398A/34', 'L109/39', 'L114/3', 'N114/10', 'N93A/49', 'K28/43', 'N468/37', 'N106/20', 'N28/9', 'K14/16', 'K89/34']\n",
      "['AMIGO-1', 'Ankyrin-G (blotting)', 'Calbindin', 'GABA(A)R, Alpha4', 'GABA(B)R1', 'HCN4', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Mortalin/GRP75', 'Parvalbumin', 'PSD-95', 'THIK-2 K+ channel', 'VGlut1']\n",
      "['N106/20', 'K14/16', 'N52A/42', 'K28/43', 'N398A/34', 'N93A/49', 'L86A/37', 'L109/39', 'N114/10', 'N468/37', 'K89/34', 'L114/3', 'N28/9']\n",
      "['AMIGO-1', 'Ankyrin-G (blotting)', 'Calbindin', 'GABA(A)R, Alpha4', 'GABA(B)R1', 'HCN4', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Mortalin/GRP75', 'Parvalbumin', 'PSD-95', 'THIK-2 K+ channel', 'VGlut1']\n",
      "['N52A/42', 'N16B/8']\n",
      "['Kv3.1b K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'N16B/8']\n",
      "['Kv3.1b K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'N16B/8']\n",
      "['Kv3.1b K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['K56A/50', 'K20/78', 'N59/36', 'N52A/42', 'N19/2', 'N28/9', 'L21/32', 'N18/30']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'VGlut1']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L86A/37']\n",
      "['AMIGO-1']\n",
      "['L86/33']\n",
      "['AMIGO-1']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['L83/11']\n",
      "['Kv2.1 K+ channel']\n",
      "['L86A/37', 'N52A/42', 'L83/11', 'K13/31', 'L86/33', 'K89/34']\n",
      "['AMIGO-1', 'AMIGO-1', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['L83/11', 'N52A/42', 'L86/33', 'L86A/37', 'K89/34', 'K13/31']\n",
      "['AMIGO-1', 'AMIGO-1', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N372C/51', 'N52A/42', 'K89/34', 'N372B/60']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N372C/51', 'N52A/42', 'K89/34', 'N372B/60']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N372C/51', 'N52A/42', 'K89/34', 'N372B/60']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N372C/51', 'N52A/42', 'K89/34', 'N372B/60']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N372C/51', 'N52A/42', 'K89/34', 'N372B/60']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['N372C/51', 'N52A/42', 'K89/34', 'N372B/60']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Kv2.2 K+ channel', 'Mortalin/GRP75']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'K87A/10', 'N95/35', 'N96/55', 'K14/16', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K89/34', 'N86/38', 'L86A/37']\n",
      "['AMIGO-1', 'GFP', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N86/38', 'L86A/37']\n",
      "['AMIGO-1', 'GFP', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N86/38', 'L86A/37']\n",
      "['AMIGO-1', 'GFP', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N86/38', 'L86A/37']\n",
      "['AMIGO-1', 'GFP', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N86/38', 'L86A/37']\n",
      "['AMIGO-1', 'GFP', 'Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34', 'K14/16', 'N16B/8']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K89/34', 'K14/16', 'N16B/8']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K89/34', 'K14/16', 'N16B/8']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K89/34', 'K14/16', 'N16B/8']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34', 'K14/16', 'K39/25']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34', 'K14/16', 'K39/25']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34', 'K14/16', 'K39/25']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34', 'K14/16', 'K39/25']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34', 'K14/16', 'K39/25']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34', 'K14/16', 'K39/25']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L86A/37']\n",
      "['AMIGO-1']\n",
      "['L86A/37', 'K89/34']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'L86A/37']\n",
      "['AMIGO-1', 'Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31', 'L109/57']\n",
      "['Calbindin', 'Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31', 'L109/57']\n",
      "['Calbindin', 'Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31', 'L109/57']\n",
      "['Calbindin', 'Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31', 'L109/57']\n",
      "['Calbindin', 'Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31', 'L109/57']\n",
      "['Calbindin', 'Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31', 'L109/57']\n",
      "['Calbindin', 'Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31', 'L109/57']\n",
      "['Calbindin', 'Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31', 'L109/57']\n",
      "['Calbindin', 'Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8', 'L6/60', 'K89/34', 'K65/35', 'K14/16', 'N147/6', 'K57/1', 'L76/36', 'K55/7']\n",
      "['CASPR/Neurexin IV', 'GFAP', 'KChIP1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-QKI', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K89/34', 'K39/25']\n",
      "['Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34', 'K39/25']\n",
      "['Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34', 'K39/25']\n",
      "['Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34', 'K39/25']\n",
      "['Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34', 'N448/88']\n",
      "['Kv2.1 K+ channel', 'Kv8.2 K+ channel']\n",
      "['K89/34', 'N448/88']\n",
      "['Kv2.1 K+ channel', 'Kv8.2 K+ channel']\n",
      "['K89/34', 'N448/88']\n",
      "['Kv2.1 K+ channel', 'Kv8.2 K+ channel']\n",
      "['K89/34', 'N448/88']\n",
      "['Kv2.1 K+ channel', 'Kv8.2 K+ channel']\n",
      "['K89/34', 'N448/88']\n",
      "['Kv2.1 K+ channel', 'Kv8.2 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31']\n",
      "['Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31']\n",
      "['Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'N263/31']\n",
      "['Cav1.2 Ca2+ channel', 'Kv2.1 K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K75/41', 'L6/60', 'K89/34', 'K57/1', 'K60/73', 'K36/15', 'K66/38']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K37/89', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['K37/89', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['K37/89', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['K37/89', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['K37/89', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['K89/34', 'N479/22']\n",
      "['Kv2.1 K+ channel', 'VAPA']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34', 'N133/21']\n",
      "['Kv2.1 K+ channel', 'RGS14']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K7/45']\n",
      "['Kv1.5 K+ channel']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['K7/45', 'K39/25', 'K89/34']\n",
      "['Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K7/45', 'K89/34', 'K39/25']\n",
      "['Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'L23/27', 'K19/36', 'K89/34', 'K14/16', 'K57/1', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'K89/34', 'K57/1']\n",
      "['Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'K89/34', 'K57/1']\n",
      "['Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'K89/34', 'K57/1']\n",
      "['Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'K89/34', 'K57/1']\n",
      "['Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K75/41', 'K89/34', 'K57/1']\n",
      "['Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N393/2', 'K89/34', 'N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin', 'Kv2.1 K+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N393/2', 'K89/34', 'N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin', 'Kv2.1 K+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N393/2', 'K89/34', 'N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin', 'Kv2.1 K+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N393/2', 'K89/34', 'N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin', 'Kv2.1 K+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N393/2', 'K89/34', 'N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin', 'Kv2.1 K+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N393/2', 'K89/34', 'N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin', 'Kv2.1 K+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['N458/10']\n",
      "['Kv6.4 K+ channel']\n",
      "['N458/10']\n",
      "['Kv6.4 K+ channel']\n",
      "['N458/10']\n",
      "['Kv6.4 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['N59/20', 'N59/36', 'N19/2', 'N308/48', 'N367/62', 'N238/29', 'N18/30']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-93/Chapsyn-110', 'SAP102', 'SAPAP1', 'Shank3']\n",
      "['K13/31', 'K9/40', 'K17/70', 'L28/4', 'K7/45']\n",
      "['Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv4.2 K+ channel', 'Kvbeta1.1/KCNAB1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K9/40', 'K17/70', 'L28/4', 'K7/45']\n",
      "['Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv4.2 K+ channel', 'Kvbeta1.1/KCNAB1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K9/40', 'K17/70', 'L28/4', 'K7/45']\n",
      "['Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv4.2 K+ channel', 'Kvbeta1.1/KCNAB1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K9/40', 'K17/70', 'L28/4', 'K7/45']\n",
      "['Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv4.2 K+ channel', 'Kvbeta1.1/KCNAB1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K9/40', 'K17/70', 'L28/4', 'K7/45']\n",
      "['Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv4.2 K+ channel', 'Kvbeta1.1/KCNAB1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['L28/4']\n",
      "['Kv4.2 K+ channel']\n",
      "['N452/69', 'N229A/32', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha6', 'GABA(A)R, Gamma2L']\n",
      "['N452/69', 'N229A/32', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha6', 'GABA(A)R, Gamma2L']\n",
      "['N452/69', 'N229A/32', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha6', 'GABA(A)R, Gamma2L']\n",
      "['N452/69', 'N229A/32', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha6', 'GABA(A)R, Gamma2L']\n",
      "['N452/69', 'N229A/32', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha6', 'GABA(A)R, Gamma2L']\n",
      "['N452/69', 'N229A/32', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha6', 'GABA(A)R, Gamma2L']\n",
      "['N452/69', 'N229A/32', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha6', 'GABA(A)R, Gamma2L']\n",
      "['N452/69', 'N229A/32', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha6', 'GABA(A)R, Gamma2L']\n",
      "['K56A/50', 'N95/35', 'L120/12', 'L106/83', 'N59/20']\n",
      "['CASK', 'Collybistin', 'GABA(A)R, Alpha1', 'Gephyrin', 'GluN2B/NR2B glutamate receptor']\n",
      "['K56A/50', 'N95/35', 'L120/12', 'L106/83', 'N59/20']\n",
      "['CASK', 'Collybistin', 'GABA(A)R, Alpha1', 'Gephyrin', 'GluN2B/NR2B glutamate receptor']\n",
      "['K56A/50', 'N95/35', 'L120/12', 'L106/83', 'N59/20']\n",
      "['CASK', 'Collybistin', 'GABA(A)R, Alpha1', 'Gephyrin', 'GluN2B/NR2B glutamate receptor']\n",
      "['K56A/50', 'N95/35', 'L120/12', 'L106/83', 'N59/20']\n",
      "['CASK', 'Collybistin', 'GABA(A)R, Alpha1', 'Gephyrin', 'GluN2B/NR2B glutamate receptor']\n",
      "['K56A/50', 'N95/35', 'L120/12', 'L106/83', 'N59/20']\n",
      "['CASK', 'Collybistin', 'GABA(A)R, Alpha1', 'Gephyrin', 'GluN2B/NR2B glutamate receptor']\n",
      "['K56A/50', 'N95/35', 'L120/12', 'L106/83', 'N59/20']\n",
      "['CASK', 'Collybistin', 'GABA(A)R, Alpha1', 'Gephyrin', 'GluN2B/NR2B glutamate receptor']\n",
      "['N97A/31', 'K28/43', 'K56A/50']\n",
      "['CASK', 'Neuroligin-1', 'PSD-95']\n",
      "['N97A/31', 'K28/43', 'K56A/50']\n",
      "['CASK', 'Neuroligin-1', 'PSD-95']\n",
      "['N97A/31', 'K28/43', 'K56A/50']\n",
      "['CASK', 'Neuroligin-1', 'PSD-95']\n",
      "['K56A/50', 'N26A/23', 'N39B/8', 'K57/1', 'N10/7']\n",
      "['CASK', 'Cavbeta4 Ca2+ channel', 'GIT1', 'Kv4.2 K+ channel (external)', 'Kv7.2/KCNQ2 K+ channel']\n",
      "['K56A/50', 'N26A/23', 'N39B/8', 'K57/1', 'N10/7']\n",
      "['CASK', 'Cavbeta4 Ca2+ channel', 'GIT1', 'Kv4.2 K+ channel (external)', 'Kv7.2/KCNQ2 K+ channel']\n",
      "['K56A/50', 'N26A/23', 'N39B/8', 'K57/1', 'N10/7']\n",
      "['CASK', 'Cavbeta4 Ca2+ channel', 'GIT1', 'Kv4.2 K+ channel (external)', 'Kv7.2/KCNQ2 K+ channel']\n",
      "['K56A/50', 'N26A/23', 'N39B/8', 'K57/1', 'N10/7']\n",
      "['CASK', 'Cavbeta4 Ca2+ channel', 'GIT1', 'Kv4.2 K+ channel (external)', 'Kv7.2/KCNQ2 K+ channel']\n",
      "['K56A/50', 'N26A/23', 'N39B/8', 'K57/1', 'N10/7']\n",
      "['CASK', 'Cavbeta4 Ca2+ channel', 'GIT1', 'Kv4.2 K+ channel (external)', 'Kv7.2/KCNQ2 K+ channel']\n",
      "['K56A/50', 'N26A/23', 'N39B/8', 'K57/1', 'N10/7']\n",
      "['CASK', 'Cavbeta4 Ca2+ channel', 'GIT1', 'Kv4.2 K+ channel (external)', 'Kv7.2/KCNQ2 K+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N355/1', 'K56A/50']\n",
      "['CASK', 'GluA1/GluR1 glutamate receptor']\n",
      "['N355/1', 'K56A/50']\n",
      "['CASK', 'GluA1/GluR1 glutamate receptor']\n",
      "['N355/1', 'K56A/50']\n",
      "['CASK', 'GluA1/GluR1 glutamate receptor']\n",
      "['N355/1', 'K56A/50']\n",
      "['CASK', 'GluA1/GluR1 glutamate receptor']\n",
      "['N355/1', 'K56A/50']\n",
      "['CASK', 'GluA1/GluR1 glutamate receptor']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K28/43', 'K56A/50']\n",
      "['CASK', 'PSD-95']\n",
      "['K28/43', 'K56A/50']\n",
      "['CASK', 'PSD-95']\n",
      "['K28/43', 'K56A/50']\n",
      "['CASK', 'PSD-95']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K14/16', 'K56A/50']\n",
      "['CASK', 'Kv1.2 K+ channel']\n",
      "['K14/16', 'K56A/50']\n",
      "['CASK', 'Kv1.2 K+ channel']\n",
      "['K14/16', 'K56A/50']\n",
      "['CASK', 'Kv1.2 K+ channel']\n",
      "['K14/16', 'K56A/50']\n",
      "['CASK', 'Kv1.2 K+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N106/36', 'K56A/50', 'K14/16']\n",
      "['Ankyrin-G (staining)', 'CASK', 'Kv1.2 K+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['L57/46', 'N38/8']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N1/12', 'N1/66']\n",
      "['KCC2', 'KCC2']\n",
      "['N1/12', 'N1/66']\n",
      "['KCC2', 'KCC2']\n",
      "['N1/12', 'N1/66']\n",
      "['KCC2', 'KCC2']\n",
      "['N1/12', 'N1/66']\n",
      "['KCC2', 'KCC2']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L60/4', 'N28/9', 'N59/20']\n",
      "[\"Copper ATPase 1 (Menke's disease protein)\", 'GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['L60/4', 'N28/9', 'N59/20']\n",
      "[\"Copper ATPase 1 (Menke's disease protein)\", 'GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['L60/4', 'N28/9', 'N59/20']\n",
      "[\"Copper ATPase 1 (Menke's disease protein)\", 'GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['L60/4', 'N28/9', 'N59/20']\n",
      "[\"Copper ATPase 1 (Menke's disease protein)\", 'GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['L60/4', 'N28/9', 'N59/20']\n",
      "[\"Copper ATPase 1 (Menke's disease protein)\", 'GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/20', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/20', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/20', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/20', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/20', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N52A/42', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Mortalin/GRP75']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L117/1', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'IRSp53/BAIAP2']\n",
      "['L117/1', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'IRSp53/BAIAP2']\n",
      "['L117/1', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'IRSp53/BAIAP2']\n",
      "['L117/1', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'IRSp53/BAIAP2']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/36', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/36', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'N59/20']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'N59/20']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'N59/20']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'N59/20']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N327/95', 'N59/20']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'PSD-95']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N367/62', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Shank3']\n",
      "['N367/62', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Shank3']\n",
      "['N367/62', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Shank3']\n",
      "['N367/62', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Shank3']\n",
      "['N367/62', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Shank3']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N419/40']\n",
      "['Pan-Nav1 Na+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N415/24', 'N206A/8', 'L118/80', 'L106/83']\n",
      "['GABA(A)R, Alpha5', 'Gephyrin', 'GFAP', 'VGAT']\n",
      "['N415/24', 'N206A/8', 'L118/80', 'L106/83']\n",
      "['GABA(A)R, Alpha5', 'Gephyrin', 'GFAP', 'VGAT']\n",
      "['N415/24', 'N206A/8', 'L118/80', 'L106/83']\n",
      "['GABA(A)R, Alpha5', 'Gephyrin', 'GFAP', 'VGAT']\n",
      "['N415/24', 'N206A/8', 'L118/80', 'L106/83']\n",
      "['GABA(A)R, Alpha5', 'Gephyrin', 'GFAP', 'VGAT']\n",
      "['N415/24', 'N206A/8', 'L118/80', 'L106/83']\n",
      "['GABA(A)R, Alpha5', 'Gephyrin', 'GFAP', 'VGAT']\n",
      "['N415/24', 'N206A/8', 'L118/80', 'L106/83']\n",
      "['GABA(A)R, Alpha5', 'Gephyrin', 'GFAP', 'VGAT']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8', 'K56A/50', 'K65/35', 'N16B/8', 'L24/18']\n",
      "['CASK', 'CASPR/Neurexin IV', 'GFAP', 'IP3 receptor, type 1', 'Kv3.1b K+ channel']\n",
      "['N206A/8', 'K56A/50', 'K65/35', 'N16B/8', 'L24/18']\n",
      "['CASK', 'CASPR/Neurexin IV', 'GFAP', 'IP3 receptor, type 1', 'Kv3.1b K+ channel']\n",
      "['N206A/8', 'K56A/50', 'K65/35', 'N16B/8', 'L24/18']\n",
      "['CASK', 'CASPR/Neurexin IV', 'GFAP', 'IP3 receptor, type 1', 'Kv3.1b K+ channel']\n",
      "['N206A/8', 'K56A/50', 'K65/35', 'N16B/8', 'L24/18']\n",
      "['CASK', 'CASPR/Neurexin IV', 'GFAP', 'IP3 receptor, type 1', 'Kv3.1b K+ channel']\n",
      "['N206A/8', 'K56A/50', 'K65/35', 'N16B/8', 'L24/18']\n",
      "['CASK', 'CASPR/Neurexin IV', 'GFAP', 'IP3 receptor, type 1', 'Kv3.1b K+ channel']\n",
      "['N206A/8', 'K56A/50', 'K65/35', 'N16B/8', 'L24/18']\n",
      "['CASK', 'CASPR/Neurexin IV', 'GFAP', 'IP3 receptor, type 1', 'Kv3.1b K+ channel']\n",
      "['N206A/8', 'K56A/50', 'K65/35', 'N16B/8', 'L24/18']\n",
      "['CASK', 'CASPR/Neurexin IV', 'GFAP', 'IP3 receptor, type 1', 'Kv3.1b K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K87A/10', 'N206A/8', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'GFAP', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'N206A/8', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'GFAP', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'N206A/8', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'GFAP', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'N206A/8', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'GFAP', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'N206A/8', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'GFAP', 'Nav1.6 Na+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8', 'N28/9']\n",
      "['GFAP', 'VGlut1']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8', 'N147/6']\n",
      "['GFAP', 'Pan-QKI']\n",
      "['N206A/8', 'N147/6']\n",
      "['GFAP', 'Pan-QKI']\n",
      "['N206A/8', 'N147/6']\n",
      "['GFAP', 'Pan-QKI']\n",
      "['N206A/8', 'N147/6']\n",
      "['GFAP', 'Pan-QKI']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K20/78', 'L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['L23/27', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K19/36']\n",
      "['Kv1.6 K+ channel']\n",
      "['K19/36']\n",
      "['Kv1.6 K+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['K13/31', 'K19/36']\n",
      "['Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K13/31', 'K19/36']\n",
      "['Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K19/36', 'K13/31']\n",
      "['Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K19/36', 'K13/31']\n",
      "['Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K19/36']\n",
      "['Kv1.6 K+ channel']\n",
      "['K19/36']\n",
      "['Kv1.6 K+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['K13/31', 'K19/36']\n",
      "['Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K13/31', 'K19/36']\n",
      "['Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K19/36', 'K13/31']\n",
      "['Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K19/36', 'K13/31']\n",
      "['Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['K87A/10', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N327/95', 'N59/36', 'N308/48']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/36', 'N327/95', 'N308/48']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N308/48', 'N59/36']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N59/36']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N59/36']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N59/20', 'N59/36', 'N19/2', 'K28/86', 'N308/48', 'L21/32', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Pan-MAGUK', 'PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N59/20', 'N59/36', 'N19/2', 'K28/86', 'N308/48', 'L21/32', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Pan-MAGUK', 'PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N59/20', 'N59/36', 'N19/2', 'K28/86', 'N308/48', 'L21/32', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Pan-MAGUK', 'PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N59/20', 'N59/36', 'N19/2', 'K28/86', 'N308/48', 'L21/32', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Pan-MAGUK', 'PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N308/48', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N308/48', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N308/48', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N308/48', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N308/48', 'N207/27']\n",
      "['GluN1/NR1 glutamate receptor', 'LRP4 (extracellular)']\n",
      "['N308/48', 'N207/27']\n",
      "['GluN1/NR1 glutamate receptor', 'LRP4 (extracellular)']\n",
      "['N308/48', 'N207/27']\n",
      "['GluN1/NR1 glutamate receptor', 'LRP4 (extracellular)']\n",
      "['N308/48', 'N207/27']\n",
      "['GluN1/NR1 glutamate receptor', 'LRP4 (extracellular)']\n",
      "['N308/48', 'N207/27']\n",
      "['GluN1/NR1 glutamate receptor', 'LRP4 (extracellular)']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N308/48', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N308/48', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N308/48', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N308/48', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N312/10']\n",
      "['NSD1']\n",
      "['N312/10']\n",
      "['NSD1']\n",
      "['N312/10']\n",
      "['NSD1']\n",
      "['N312/10']\n",
      "['NSD1']\n",
      "['N312/10']\n",
      "['NSD1']\n",
      "['N312/10']\n",
      "['NSD1']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36', 'K73/20', 'N244/5', 'K65/35', 'N86/38', 'K14/16', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Contactin/F3', 'GFP', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel', 'SynCAM4']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N100/13']\n",
      "['GST']\n",
      "['N106/36', 'N100/13']\n",
      "['Ankyrin-G (staining)', 'GST']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N56/21', 'L11A/41']\n",
      "['FGF14/FHF4', 'Pan-Neurofascin']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K36/15', 'N106/36', 'K14/16']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['K36/15', 'N106/36', 'K14/16']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['K36/15', 'N106/36', 'K14/16']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['K36/15', 'N106/36', 'K14/16']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['L11A/41']\n",
      "['Pan-Neurofascin']\n",
      "['N106/36', 'L11A/41']\n",
      "['Ankyrin-G (staining)', 'Pan-Neurofascin']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K20/78', 'K36/15', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['K20/78', 'K36/15', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['K20/78', 'K36/15', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['K20/78', 'K36/15', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['K20/78', 'K36/15', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['K20/78', 'K36/15', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K20/78', 'N106/36', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-G (staining)', 'Kv1.1 K+ channel']\n",
      "['K20/78', 'N106/36', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-G (staining)', 'Kv1.1 K+ channel']\n",
      "['K20/78', 'N106/36', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-G (staining)', 'Kv1.1 K+ channel']\n",
      "['K20/78', 'N106/36', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-G (staining)', 'Kv1.1 K+ channel']\n",
      "['K20/78', 'N106/36', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-G (staining)', 'Kv1.1 K+ channel']\n",
      "['K20/78', 'N106/36', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-G (staining)', 'Kv1.1 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N18/28', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'PSD-93/Chapsyn-110']\n",
      "['N18/28', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'PSD-93/Chapsyn-110']\n",
      "['N18/28', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'PSD-93/Chapsyn-110']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N106/36', 'K28/43']\n",
      "['Ankyrin-G (staining)', 'PSD-95']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'N393/76']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'N393/76', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N56/21', 'N86/38']\n",
      "['FGF14/FHF4', 'GFP']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['N106/36', 'N100/13', 'K65/35', 'N86/38', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'GST']\n",
      "['N106/36', 'N100/13', 'K65/35', 'N86/38', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'GST']\n",
      "['N106/36', 'N100/13', 'K65/35', 'N86/38', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'GST']\n",
      "['N106/36', 'N100/13', 'K65/35', 'N86/38', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'GST']\n",
      "['N106/36', 'N100/13', 'K65/35', 'N86/38', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'GST']\n",
      "['N106/36', 'N100/13', 'K65/35', 'N86/38', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'GST']\n",
      "['N106/36', 'N100/13', 'K65/35', 'N86/38', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'GST']\n",
      "['N106/36', 'N100/13', 'K65/35', 'N86/38', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'GST']\n",
      "['N106/36', 'N100/13', 'K65/35', 'N86/38', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'GFP', 'GST']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N106/36', 'K87A/10']\n",
      "['Ankyrin-G (staining)', 'Nav1.6 Na+ channel']\n",
      "['K28/43', 'N106/36', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'PSD-95']\n",
      "['K28/43', 'N106/36', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'PSD-95']\n",
      "['K28/43', 'N106/36', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'PSD-95']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K74/71', 'N56/21', 'N70/28', 'N155/9', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'FGF14/FHF4', 'HCN1', 'Nav1.1 Na+ channel', 'Uncx']\n",
      "['K74/71', 'N56/21', 'N70/28', 'N155/9', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'FGF14/FHF4', 'HCN1', 'Nav1.1 Na+ channel', 'Uncx']\n",
      "['K74/71', 'N56/21', 'N70/28', 'N155/9', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'FGF14/FHF4', 'HCN1', 'Nav1.1 Na+ channel', 'Uncx']\n",
      "['K74/71', 'N56/21', 'N70/28', 'N155/9', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'FGF14/FHF4', 'HCN1', 'Nav1.1 Na+ channel', 'Uncx']\n",
      "['K74/71', 'N56/21', 'N70/28', 'N155/9', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'FGF14/FHF4', 'HCN1', 'Nav1.1 Na+ channel', 'Uncx']\n",
      "['K74/71', 'N56/21', 'N70/28', 'N155/9', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'FGF14/FHF4', 'HCN1', 'Nav1.1 Na+ channel', 'Uncx']\n",
      "['K74/71', 'N56/21', 'N70/28', 'N155/9', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'FGF14/FHF4', 'HCN1', 'Nav1.1 Na+ channel', 'Uncx']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18', 'K65/35', 'N106/20', 'N393/76']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin', 'CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K13/31', 'N106/36', 'K20/78', 'K17/70', 'K14/16', 'K36/15']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'N106/36', 'K20/78', 'K17/70', 'K14/16', 'K36/15']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'N106/36', 'K20/78', 'K17/70', 'K14/16', 'K36/15']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'N106/36', 'K20/78', 'K17/70', 'K14/16', 'K36/15']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'N106/36', 'K20/78', 'K17/70', 'K14/16', 'K36/15']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'N106/36', 'K20/78', 'K17/70', 'K14/16', 'K36/15']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'N106/36', 'K20/78', 'K17/70', 'K14/16', 'K36/15']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'N106/36', 'K20/78', 'K17/70', 'K14/16', 'K36/15']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['L88/86']\n",
      "['TrpM4']\n",
      "['N106/36', 'L88/86']\n",
      "['Ankyrin-G (staining)', 'TrpM4']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K20/78', 'K36/15', 'K14/16', 'N18/30']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['K20/78', 'K36/15', 'K14/16', 'N18/30']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['K20/78', 'K36/15', 'K14/16', 'N18/30']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['K20/78', 'K36/15', 'K14/16', 'N18/30']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['K20/78', 'K36/15', 'K14/16', 'N18/30']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['K20/78', 'K36/15', 'K14/16', 'N18/30']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['K20/78', 'K36/15', 'K14/16', 'N18/30']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['K20/78', 'K36/15', 'K14/16', 'N18/30']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['K20/78', 'K36/15', 'K14/16', 'N18/30']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N395/68', 'K14/16', 'N168/6']\n",
      "['Kv1.2 K+ channel', 'Navbeta2 Na+ channel', 'Navbeta4 Na+ channel']\n",
      "['N395/68', 'K14/16', 'N168/6']\n",
      "['Kv1.2 K+ channel', 'Navbeta2 Na+ channel', 'Navbeta4 Na+ channel']\n",
      "['N395/68', 'K14/16', 'N168/6']\n",
      "['Kv1.2 K+ channel', 'Navbeta2 Na+ channel', 'Navbeta4 Na+ channel']\n",
      "['N395/68', 'K14/16', 'N168/6']\n",
      "['Kv1.2 K+ channel', 'Navbeta2 Na+ channel', 'Navbeta4 Na+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N319A/14']\n",
      "['SUR2A']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K87A/10', 'N56/21', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'N56/21', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'N56/21', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'N56/21', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'FGF14/FHF4', 'Nav1.6 Na+ channel']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N105/17', 'K14/16']\n",
      "['Ankyrin-B', 'Kv1.2 K+ channel']\n",
      "['N105/17', 'K14/16']\n",
      "['Ankyrin-B', 'Kv1.2 K+ channel']\n",
      "['N105/17', 'K14/16']\n",
      "['Ankyrin-B', 'Kv1.2 K+ channel']\n",
      "['N105/17', 'K14/16']\n",
      "['Ankyrin-B', 'Kv1.2 K+ channel']\n",
      "['N105/17', 'K14/16']\n",
      "['Ankyrin-B', 'Kv1.2 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K20/78', 'K13/31', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['N106/36', 'N114/10', 'K36/15']\n",
      "['Ankyrin-G (staining)', 'HCN4', 'Kv1.1 K+ channel (external)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['L11A/41', 'N105/17', 'A12/18', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)', 'Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['L11A/41', 'N105/17', 'A12/18', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)', 'Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['L11A/41', 'N105/17', 'A12/18', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)', 'Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['L11A/41', 'N105/17', 'A12/18', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)', 'Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['N106/36', 'K69/3']\n",
      "['Ankyrin-G (staining)', 'Nav1.2 Na+ channel']\n",
      "['K36/15', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel (external)']\n",
      "['K36/15', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel (external)']\n",
      "['K36/15', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Kv1.1 K+ channel (external)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N87/25', 'N452/69']\n",
      "['GABA(A)R, Beta3', 'GABA(A)R, Gamma2L']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N170A/1']\n",
      "['Neurexin-1-Beta']\n",
      "['N106/36', 'N170A/1']\n",
      "['Ankyrin-G (staining)', 'Neurexin-1-Beta']\n",
      "['N395/68', 'N168/6']\n",
      "['Navbeta2 Na+ channel', 'Navbeta4 Na+ channel']\n",
      "['N395/68', 'K65/35', 'K14/16', 'N168/6']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel', 'Navbeta2 Na+ channel', 'Navbeta4 Na+ channel']\n",
      "['N395/68', 'K65/35', 'K14/16', 'N168/6']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel', 'Navbeta2 Na+ channel', 'Navbeta4 Na+ channel']\n",
      "['N395/68', 'K65/35', 'K14/16', 'N168/6']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel', 'Navbeta2 Na+ channel', 'Navbeta4 Na+ channel']\n",
      "['N395/68', 'K65/35', 'K14/16', 'N168/6']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel', 'Navbeta2 Na+ channel', 'Navbeta4 Na+ channel']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['N105/13', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)']\n",
      "['N105/13', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)']\n",
      "['N105/13', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)']\n",
      "['N105/13', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)']\n",
      "['N105/13', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['N327/95', 'K28/74']\n",
      "['GluN2A/NR2A glutamate receptor', 'PSD-95']\n",
      "['N327/95', 'N59/36', 'K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-95']\n",
      "['N327/95', 'N59/36', 'K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-95']\n",
      "['N327/95', 'N59/36', 'K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-95']\n",
      "['N327/95', 'N59/36', 'K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'PSD-95']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N75/3']\n",
      "['mGluR1/5 (Group I) glutamate receptor']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N367/62', 'N28/9', 'N23B/6']\n",
      "['Shank2', 'Shank3', 'VGlut1']\n",
      "['N367/62', 'N28/9', 'N23B/6']\n",
      "['Shank2', 'Shank3', 'VGlut1']\n",
      "['N367/62', 'N28/9', 'N23B/6']\n",
      "['Shank2', 'Shank3', 'VGlut1']\n",
      "['N367/62', 'N28/9', 'N23B/6']\n",
      "['Shank2', 'Shank3', 'VGlut1']\n",
      "['N367/62', 'N28/9', 'N23B/6']\n",
      "['Shank2', 'Shank3', 'VGlut1']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N367/62', 'K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'PSD-95', 'Shank3']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N105/17', 'N367/62']\n",
      "['Ankyrin-B', 'Shank3']\n",
      "['N71/37', 'N114/10']\n",
      "['HCN2', 'HCN4']\n",
      "['N71/37', 'N114/10']\n",
      "['HCN2', 'HCN4']\n",
      "['N71/37', 'N114/10']\n",
      "['HCN2', 'HCN4']\n",
      "['N71/37', 'N114/10']\n",
      "['HCN2', 'HCN4']\n",
      "['N71/37']\n",
      "['HCN2']\n",
      "['N71/37']\n",
      "['HCN2']\n",
      "['N71/37']\n",
      "['HCN2']\n",
      "['N71/37']\n",
      "['HCN2']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['K13/31', 'L6/60', 'N26A/23', 'N71/37']\n",
      "['HCN2', 'Kv1.4 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'L6/60', 'N26A/23', 'N71/37']\n",
      "['HCN2', 'Kv1.4 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'L6/60', 'N26A/23', 'N71/37']\n",
      "['HCN2', 'Kv1.4 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'L6/60', 'N26A/23', 'N71/37']\n",
      "['HCN2', 'Kv1.4 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'L6/60', 'N26A/23', 'N71/37']\n",
      "['HCN2', 'Kv1.4 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N59/36', 'N28/9']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['N59/36', 'N28/9']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['N59/36', 'N28/9']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N95/35', 'N87/25']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta3']\n",
      "['K28/74', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'PSD-95']\n",
      "['K28/74', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'PSD-95']\n",
      "['K28/74', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'PSD-95']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N86/38', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GFP']\n",
      "['N86/38', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GFP']\n",
      "['N86/38', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GFP']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N95/35', 'K28/43']\n",
      "['GABA(A)R, Alpha1', 'PSD-95']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N398A/34', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha4']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N398A/34', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha4']\n",
      "['N398A/34', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha4']\n",
      "['N398A/34', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha4']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N186/29']\n",
      "['Dopamine D2 receptor']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66', 'N186/29']\n",
      "['Arl13b', 'Dopamine D2 receptor']\n",
      "['L106/93']\n",
      "['Gephyrin']\n",
      "['L106/4']\n",
      "['Gephyrin']\n",
      "['L86A/37']\n",
      "['AMIGO-1']\n",
      "['L113/27']\n",
      "['Homer1L']\n",
      "['L113/130']\n",
      "['Homer1L + Homer1S']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['L113/13']\n",
      "['Homer1L']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L86A/37', 'L113/13', 'L106/93', 'L113/27', 'L113/130', 'K28/43', 'K28/86', 'L106/4', 'K28/74']\n",
      "['AMIGO-1', 'Gephyrin', 'Gephyrin', 'Homer1L', 'Homer1L', 'Homer1L + Homer1S', 'Pan-MAGUK', 'PSD-95', 'PSD-95']\n",
      "['N414/25']\n",
      "['NCKX4 Na+/Ca2+/K+ exchanger']\n",
      "['N97A/31', 'L21/32', 'N110/29']\n",
      "['GluA2/GluR2 glutamate receptor', 'Neuroligin-1', 'Neuroligin-3']\n",
      "['N97A/31', 'L21/32', 'N110/29']\n",
      "['GluA2/GluR2 glutamate receptor', 'Neuroligin-1', 'Neuroligin-3']\n",
      "['N97A/31', 'L21/32', 'N110/29']\n",
      "['GluA2/GluR2 glutamate receptor', 'Neuroligin-1', 'Neuroligin-3']\n",
      "['N97A/31', 'L21/32', 'N110/29']\n",
      "['GluA2/GluR2 glutamate receptor', 'Neuroligin-1', 'Neuroligin-3']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6', 'N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46', 'N263/31', 'N134/12']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.2 Ca2+ channel', 'Nav1.8 Na+ channel']\n",
      "['L57/46', 'N263/31', 'N134/12']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.2 Ca2+ channel', 'Nav1.8 Na+ channel']\n",
      "['L57/46', 'N263/31', 'N134/12']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.2 Ca2+ channel', 'Nav1.8 Na+ channel']\n",
      "['L57/46', 'N263/31', 'N134/12']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.2 Ca2+ channel', 'Nav1.8 Na+ channel']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N55/10']\n",
      "['Cav3.2 Ca2+ channel']\n",
      "['N55/10']\n",
      "['Cav3.2 Ca2+ channel']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N55/10']\n",
      "['Cav3.2 Ca2+ channel']\n",
      "['N55/10']\n",
      "['Cav3.2 Ca2+ channel']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['N393/76', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35', 'N393/76']\n",
      "['Beta4-spectrin', 'CASPR/Neurexin IV']\n",
      "['N225A/10', 'N235/22']\n",
      "['FGF13/FHF2, B isoform', 'Pan-FHF-A']\n",
      "['N225A/10', 'N235/22']\n",
      "['FGF13/FHF2, B isoform', 'Pan-FHF-A']\n",
      "['N225A/10', 'N235/22']\n",
      "['FGF13/FHF2, B isoform', 'Pan-FHF-A']\n",
      "['N229A/32', 'N95/35', 'N28/9', 'L118/80', 'L106/83', 'N415/24', 'N355/1']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha5', 'GABA(A)R, Alpha6', 'Gephyrin', 'GluA1/GluR1 glutamate receptor', 'VGAT', 'VGlut1']\n",
      "['N229A/32', 'N95/35', 'L106/83', 'N415/24', 'N355/1']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha5', 'GABA(A)R, Alpha6', 'Gephyrin', 'GluA1/GluR1 glutamate receptor']\n",
      "['N95/35', 'L118/80', 'L106/83', 'N415/24', 'N355/1']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha5', 'Gephyrin', 'GluA1/GluR1 glutamate receptor', 'VGAT']\n",
      "['N95/35', 'L118/80', 'L106/83', 'N415/24', 'N355/1']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha5', 'Gephyrin', 'GluA1/GluR1 glutamate receptor', 'VGAT']\n",
      "['N95/35', 'L118/80', 'L106/83', 'N415/24', 'N355/1']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha5', 'Gephyrin', 'GluA1/GluR1 glutamate receptor', 'VGAT']\n",
      "['N95/35', 'L118/80', 'L106/83', 'N415/24', 'N355/1']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha5', 'Gephyrin', 'GluA1/GluR1 glutamate receptor', 'VGAT']\n",
      "['N7/18', 'N263/31']\n",
      "['Cav1.2 Ca2+ channel', 'Cavbeta1 Ca2+ channel']\n",
      "['N7/18', 'N263/31']\n",
      "['Cav1.2 Ca2+ channel', 'Cavbeta1 Ca2+ channel']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['K28/43', 'N263/31']\n",
      "['Cav1.2 Ca2+ channel', 'PSD-95']\n",
      "['K28/43', 'N263/31']\n",
      "['Cav1.2 Ca2+ channel', 'PSD-95']\n",
      "['K28/43', 'N263/31']\n",
      "['Cav1.2 Ca2+ channel', 'PSD-95']\n",
      "['N144/32']\n",
      "['Pan-Gamma-protocadherin-A']\n",
      "['N148/30']\n",
      "['Gamma-protocadherin-B2']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N174B/27']\n",
      "['Gamma-protocadherin-C3']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N419/78']\n",
      "['Pan-Nav1 Na+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['N8B/1', 'N10/7', 'N7/18']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta2 Ca2+ channel', 'Cavbeta4 Ca2+ channel']\n",
      "['N8B/1', 'N10/7', 'N7/18']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta2 Ca2+ channel', 'Cavbeta4 Ca2+ channel']\n",
      "['N8B/1', 'N10/7', 'N7/18']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta2 Ca2+ channel', 'Cavbeta4 Ca2+ channel']\n",
      "['N7/18', 'N10/7']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta4 Ca2+ channel']\n",
      "['N7/18', 'N10/7']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta4 Ca2+ channel']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['N8B/1', 'N10/7', 'K28/43', 'N7/18']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta2 Ca2+ channel', 'Cavbeta4 Ca2+ channel', 'PSD-95']\n",
      "['N8B/1', 'N10/7', 'K28/43', 'N7/18']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta2 Ca2+ channel', 'Cavbeta4 Ca2+ channel', 'PSD-95']\n",
      "['N8B/1', 'N10/7', 'K28/43', 'N7/18']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta2 Ca2+ channel', 'Cavbeta4 Ca2+ channel', 'PSD-95']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['N7/18', 'N10/7']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta4 Ca2+ channel']\n",
      "['N7/18', 'N10/7']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta4 Ca2+ channel']\n",
      "['K74/71', 'N86/8', 'N144/14', 'L114/81', 'K14/16', 'N16B/8']\n",
      "['6xHis', 'GFP', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel', 'Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['K74/71', 'N86/8', 'N144/14', 'L114/81', 'K14/16', 'N16B/8']\n",
      "['6xHis', 'GFP', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel', 'Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['K74/71', 'N86/8', 'N144/14', 'L114/81', 'K14/16', 'N16B/8']\n",
      "['6xHis', 'GFP', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel', 'Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['K74/71', 'N86/8', 'N144/14', 'L114/81', 'K14/16', 'N16B/8']\n",
      "['6xHis', 'GFP', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel', 'Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['K74/71', 'N86/8', 'N144/14', 'L114/81', 'K14/16', 'N16B/8']\n",
      "['6xHis', 'GFP', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel', 'Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['K74/71', 'N86/8', 'N144/14', 'L114/81', 'K14/16', 'N16B/8']\n",
      "['6xHis', 'GFP', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel', 'Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['K74/71', 'N86/8', 'N144/14', 'L114/81', 'K14/16', 'N16B/8']\n",
      "['6xHis', 'GFP', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel', 'Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['K74/71', 'N86/8', 'N144/14', 'L114/81', 'K14/16', 'N16B/8']\n",
      "['6xHis', 'GFP', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel', 'Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['L114/81']\n",
      "['Parvalbumin']\n",
      "['L109/57']\n",
      "['Calbindin']\n",
      "['N206B/9']\n",
      "['GFAP R416WT']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L115/13']\n",
      "['NPY/Neuropeptide Y']\n",
      "['L109/57']\n",
      "['Calbindin']\n",
      "['N206B/9']\n",
      "['GFAP R416WT']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L115/13']\n",
      "['NPY/Neuropeptide Y']\n",
      "['L114/81']\n",
      "['Parvalbumin']\n",
      "['N86/38', 'L114/81', 'K28/43', 'N206B/9', 'N28/9', 'K14/16', 'L109/57', 'L115/13']\n",
      "['Calbindin', 'GFAP R416WT', 'GFP', 'Kv1.2 K+ channel', 'NPY/Neuropeptide Y', 'Parvalbumin', 'PSD-95', 'VGlut1']\n",
      "['N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N148/30', 'N159/5']\n",
      "['Gamma-protocadherin-B2', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N148/30']\n",
      "['Gamma-protocadherin-B2']\n",
      "['N174B/27']\n",
      "['Gamma-protocadherin-C3']\n",
      "['N174B/27']\n",
      "['Gamma-protocadherin-C3']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['N144/32']\n",
      "['Pan-Gamma-protocadherin-A']\n",
      "['N148/30']\n",
      "['Gamma-protocadherin-B2']\n",
      "['N144/32', 'N148/30', 'N159/5', 'N174B/27']\n",
      "['Gamma-protocadherin-B2', 'Gamma-protocadherin-C3', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N212/17']\n",
      "['TRIP8b (constant)']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N479/107']\n",
      "['VAPA + VAPB']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N86/8', 'K28/43', 'N38/8']\n",
      "['Cav1.3 Ca2+ channel', 'GFP', 'PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N183/15']\n",
      "['QKI-7']\n",
      "['N415/24', '62/3G1']\n",
      "['GABA(A)R, Alpha5']\n",
      "['N415/24', '62/3G1']\n",
      "['GABA(A)R, Alpha5']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N372B/60']\n",
      "['Kv2.2 K+ channel']\n",
      "['N255/38']\n",
      "['Nav1.4 Na+ channel']\n",
      "['N255/38']\n",
      "['Nav1.4 Na+ channel']\n",
      "['N255/38']\n",
      "['Nav1.4 Na+ channel']\n",
      "['N255/38']\n",
      "['Nav1.4 Na+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N372B/60']\n",
      "['Kv2.2 K+ channel']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['K14/16', 'N134/12', 'N372B/60', 'N68/6', 'K89/34', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Nav1.7 Na+ channel', 'Nav1.8 Na+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N372B/1']\n",
      "['Kv2.2 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N106/36', 'K14/16', 'N16B/8']\n",
      "['Ankyrin-G (staining)', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N178A/9']\n",
      "['Cav3.1 Ca2+ channel']\n",
      "['N178A/9']\n",
      "['Cav3.1 Ca2+ channel']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['N178A/9']\n",
      "['Cav3.1 Ca2+ channel']\n",
      "['N178A/9']\n",
      "['Cav3.1 Ca2+ channel']\n",
      "['N178A/9']\n",
      "['Cav3.1 Ca2+ channel']\n",
      "['N212A/34']\n",
      "['TRIP8b (exon 1b)']\n",
      "['N212/3']\n",
      "['TRIP8b (exon 4)']\n",
      "['N212/3']\n",
      "['TRIP8b (exon 4)']\n",
      "['N212A/34']\n",
      "['TRIP8b (exon 1b)']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N18/28']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K67/25', 'K36/15', 'K13/31', 'L23/27']\n",
      "['CASPR2', 'Kv1.1 K+ channel (external)', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K67/25', 'K36/15', 'K13/31', 'L23/27']\n",
      "['CASPR2', 'Kv1.1 K+ channel (external)', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K67/25', 'K36/15', 'K13/31', 'L23/27']\n",
      "['CASPR2', 'Kv1.1 K+ channel (external)', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K67/25', 'K36/15', 'K13/31', 'L23/27']\n",
      "['CASPR2', 'Kv1.1 K+ channel (external)', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K67/25', 'K36/15', 'K13/31', 'L23/27']\n",
      "['CASPR2', 'Kv1.1 K+ channel (external)', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K67/25', 'K36/15', 'K13/31', 'L23/27']\n",
      "['CASPR2', 'Kv1.1 K+ channel (external)', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K67/25', 'K36/15', 'K13/31', 'L23/27']\n",
      "['CASPR2', 'Kv1.1 K+ channel (external)', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K67/25', 'K36/15', 'K13/31', 'L23/27']\n",
      "['CASPR2', 'Kv1.1 K+ channel (external)', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N415/24', 'N419/40', 'N206A/8', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha5', 'GFAP', 'Pan-Nav1 Na+ channel']\n",
      "['N415/24', 'N419/40', 'N206A/8', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha5', 'GFAP', 'Pan-Nav1 Na+ channel']\n",
      "['N415/24']\n",
      "['GABA(A)R, Alpha5']\n",
      "['N415/24']\n",
      "['GABA(A)R, Alpha5']\n",
      "['L124/59']\n",
      "['Bassoon']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['L109/57']\n",
      "['Calbindin']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K73/20']\n",
      "['Contactin/F3']\n",
      "['L6/60', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L18A/3', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'N53/32']\n",
      "['BKbeta2 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['K28/43', 'K28/74']\n",
      "['PSD-95', 'PSD-95']\n",
      "['K28/43', 'K28/74']\n",
      "['PSD-95', 'PSD-95']\n",
      "['K28/43', 'K28/74']\n",
      "['PSD-95', 'PSD-95']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'PSD-95']\n",
      "['K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'PSD-95']\n",
      "['K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'PSD-95']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N235/22', 'N91/27']\n",
      "['FGF13/FHF2', 'Pan-FHF-A']\n",
      "['N235/22', 'N91/27']\n",
      "['FGF13/FHF2', 'Pan-FHF-A']\n",
      "['N235/22', 'N91/27']\n",
      "['FGF13/FHF2', 'Pan-FHF-A']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['N388A/10']\n",
      "['Ankyrin-R']\n",
      "['N388A/10']\n",
      "['Ankyrin-R']\n",
      "['N388A/10']\n",
      "['Ankyrin-R']\n",
      "['N65/37']\n",
      "['Ataxin-1, WT']\n",
      "['N65/37']\n",
      "['Ataxin-1, WT']\n",
      "['N206B/9']\n",
      "['GFAP R416WT']\n",
      "['N206B/9']\n",
      "['GFAP R416WT']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N106/43']\n",
      "['Ankyrin-G (staining)']\n",
      "['N105/13']\n",
      "['Ankyrin-B']\n",
      "['N105/13']\n",
      "['Ankyrin-B']\n",
      "['N105/13']\n",
      "['Ankyrin-B']\n",
      "['N105/13']\n",
      "['Ankyrin-B']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['L36/12']\n",
      "['CNGA1 + CNGA3']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['N105/13']\n",
      "['Ankyrin-B']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['K13/31', 'K20/78', 'K17/70', 'N70/28', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'HCN1', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K20/78', 'K17/70', 'N70/28', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'HCN1', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K20/78', 'K17/70', 'N70/28', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'HCN1', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K20/78', 'K17/70', 'N70/28', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'HCN1', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K20/78', 'K17/70', 'N70/28', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'HCN1', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K20/78', 'K17/70', 'N70/28', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'HCN1', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K20/78', 'K17/70', 'N70/28', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'HCN1', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K20/78', 'K17/70', 'N70/28', 'K14/16', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'HCN1', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['N57/2', 'K69/3', 'A12/18', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Nav1.2 Na+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N57/2', 'K69/3', 'A12/18', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Nav1.2 Na+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N57/2', 'K69/3', 'A12/18', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Nav1.2 Na+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N57/2', 'K69/3', 'A12/18', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Nav1.2 Na+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N57/2', 'K69/3', 'A12/18', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Nav1.2 Na+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N57/2', 'K69/3', 'A12/18', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'ADAM22 (extracellular)', 'Nav1.2 Na+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N46/30', 'K14/16', 'K57/1']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.2 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['N46/30', 'K14/16', 'K57/1']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.2 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['N46/30', 'K14/16', 'K57/1']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.2 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['N46/30', 'K14/16', 'K57/1']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.2 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L60/4']\n",
      "[\"Copper ATPase 1 (Menke's disease protein)\"]\n",
      "['N56/21', 'N235/22']\n",
      "['FGF14/FHF4', 'Pan-FHF-A']\n",
      "['N56/21', 'N235/22']\n",
      "['FGF14/FHF4', 'Pan-FHF-A']\n",
      "['N56/21', 'N235/22']\n",
      "['FGF14/FHF4', 'Pan-FHF-A']\n",
      "['N56/21', 'N235/22']\n",
      "['FGF14/FHF4', 'Pan-FHF-A']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N56/21', 'N235/22', 'K74/71', 'K69/3']\n",
      "['FGF14/FHF4', 'Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Pan-FHF-A']\n",
      "['N56/21', 'N235/22', 'K74/71', 'K69/3']\n",
      "['FGF14/FHF4', 'Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Pan-FHF-A']\n",
      "['N56/21', 'N235/22', 'K74/71', 'K69/3']\n",
      "['FGF14/FHF4', 'Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Pan-FHF-A']\n",
      "['N56/21', 'N235/22', 'K74/71', 'K69/3']\n",
      "['FGF14/FHF4', 'Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Pan-FHF-A']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N167/7']\n",
      "['Dicer']\n",
      "['L48A/9']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['L48A/9']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'VGlut1']\n",
      "['N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'VGlut1']\n",
      "['N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'VGlut1']\n",
      "['N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'VGlut1']\n",
      "['N28/9', 'N110/29']\n",
      "['Neuroligin-3', 'VGlut1']\n",
      "['N170A/1']\n",
      "['Neurexin-1-Beta']\n",
      "['N170A/1']\n",
      "['Neurexin-1-Beta']\n",
      "['N170A/1']\n",
      "['Neurexin-1-Beta']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N133/35']\n",
      "['RGS14']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['N244/5', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'SynCAM4']\n",
      "['N244/5', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'SynCAM4']\n",
      "['N244/5', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'SynCAM4']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N244/5', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'SynCAM4']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/65']\n",
      "['Ankyrin-G (staining)']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N332B/15']\n",
      "['BAF53b']\n",
      "['N332B/15']\n",
      "['BAF53b']\n",
      "['N332B/15']\n",
      "['BAF53b']\n",
      "['L43/40']\n",
      "['DLX1']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N70/28', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'HCN1']\n",
      "['N70/28', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'HCN1']\n",
      "['N70/28', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'HCN1']\n",
      "['L11A/41', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Pan-Neurofascin']\n",
      "['L11A/41', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Pan-Neurofascin']\n",
      "['K87A/10', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K36/15', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Kv1.1 K+ channel (external)']\n",
      "['K36/15', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Kv1.1 K+ channel (external)']\n",
      "['K20/78', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Kv1.1 K+ channel']\n",
      "['K20/78', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Kv1.1 K+ channel']\n",
      "['K20/78', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Kv1.1 K+ channel']\n",
      "['K73/20', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Contactin/F3']\n",
      "['K73/20', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Contactin/F3']\n",
      "['K73/20', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Contactin/F3']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K65/35', 'K28/43']\n",
      "['CASPR/Neurexin IV', 'PSD-95']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K87A/10', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35', 'N59/20']\n",
      "['CASPR/Neurexin IV', 'GluN2B/NR2B glutamate receptor']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L127/12']\n",
      "['GAD65 + GAD67']\n",
      "['L127/12']\n",
      "['GAD65 + GAD67']\n",
      "['L127/12']\n",
      "['GAD65 + GAD67']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['K67/25']\n",
      "['CASPR2']\n",
      "['K67/25']\n",
      "['CASPR2']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['K28/86', 'N19/2']\n",
      "['Pan-MAGUK', 'SAP102']\n",
      "['K28/86', 'N19/2']\n",
      "['Pan-MAGUK', 'SAP102']\n",
      "['K28/86', 'N19/2']\n",
      "['Pan-MAGUK', 'SAP102']\n",
      "['K28/86', 'N19/2']\n",
      "['Pan-MAGUK', 'SAP102']\n",
      "['K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP3 K+ channel', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP3 K+ channel', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP3 K+ channel', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP3 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N81/2']\n",
      "['GABA(B)R2']\n",
      "['N81/2']\n",
      "['GABA(B)R2']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N81/2']\n",
      "['GABA(B)R2']\n",
      "['N81/2']\n",
      "['GABA(B)R2']\n",
      "['N81/2']\n",
      "['GABA(B)R2']\n",
      "['N81/2']\n",
      "['GABA(B)R2']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N174B/27']\n",
      "['Gamma-protocadherin-C3']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['N144/32']\n",
      "['Pan-Gamma-protocadherin-A']\n",
      "['N144/32', 'N159/5', 'N174B/27']\n",
      "['Gamma-protocadherin-C3', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49', 'N11/33']\n",
      "['GABA(B)R1', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N93A/49', 'N11/33']\n",
      "['GABA(B)R1', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N93A/49', 'N11/33']\n",
      "['GABA(B)R1', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'PSD-95']\n",
      "['K28/43', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L18A/3']\n",
      "['BKbeta4 K+ channel']\n",
      "['L18A/3']\n",
      "['BKbeta4 K+ channel']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N258/5']\n",
      "['Clcn3']\n",
      "['N258/5']\n",
      "['Clcn3']\n",
      "['N258/5']\n",
      "['Clcn3']\n",
      "['N258/5']\n",
      "['Clcn3']\n",
      "['N327A/38']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['K78/29']\n",
      "['SK2 K+ channel']\n",
      "['N116/14']\n",
      "['Botch']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N59/36', 'N355/1', 'N95/35', 'N97A/31']\n",
      "['GABA(A)R, Alpha1', 'GluA1/GluR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Neuroligin-1']\n",
      "['N59/36', 'N355/1', 'N95/35', 'N97A/31']\n",
      "['GABA(A)R, Alpha1', 'GluA1/GluR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Neuroligin-1']\n",
      "['N59/36', 'N355/1', 'N95/35', 'N97A/31']\n",
      "['GABA(A)R, Alpha1', 'GluA1/GluR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Neuroligin-1']\n",
      "['N59/36', 'N355/1', 'N95/35', 'N97A/31']\n",
      "['GABA(A)R, Alpha1', 'GluA1/GluR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Neuroligin-1']\n",
      "['N59/36', 'N355/1', 'N95/35', 'N97A/31']\n",
      "['GABA(A)R, Alpha1', 'GluA1/GluR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Neuroligin-1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['N59/36', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/36', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/36', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N245/36']\n",
      "['TARPGamma2+4+8']\n",
      "['N245/36', 'K28/43', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-95', 'TARPGamma2+4+8']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L86A/37']\n",
      "['AMIGO-1']\n",
      "['L113/27']\n",
      "['Homer1L']\n",
      "['L113/130']\n",
      "['Homer1L + Homer1S']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['L113/13']\n",
      "['Homer1L']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['L106/93']\n",
      "['Gephyrin']\n",
      "['L106/4']\n",
      "['Gephyrin']\n",
      "['N201/35']\n",
      "['Iduna/RNF146']\n",
      "['N201/35']\n",
      "['Iduna/RNF146']\n",
      "['N201/35']\n",
      "['Iduna/RNF146']\n",
      "['N201/35']\n",
      "['Iduna/RNF146']\n",
      "['N201/35']\n",
      "['Iduna/RNF146']\n",
      "['N276A/15']\n",
      "['Synaptotagmin-9']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N325B/65']\n",
      "['THAP1']\n",
      "['N212/3', 'N212A/34']\n",
      "['TRIP8b (exon 1b)', 'TRIP8b (exon 4)']\n",
      "['N212/3', 'N212A/34']\n",
      "['TRIP8b (exon 1b)', 'TRIP8b (exon 4)']\n",
      "['N212/3', 'N212A/34']\n",
      "['TRIP8b (exon 1b)', 'TRIP8b (exon 4)']\n",
      "['N212/3', 'N212A/34']\n",
      "['TRIP8b (exon 1b)', 'TRIP8b (exon 4)']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['N72/16']\n",
      "['Kv3.4 K+ channel']\n",
      "['N72/16']\n",
      "['Kv3.4 K+ channel']\n",
      "['N72/16']\n",
      "['Kv3.4 K+ channel']\n",
      "['N1/12', 'N161/20']\n",
      "['KCC2', 'NKCC1']\n",
      "['N1/12', 'N161/20']\n",
      "['KCC2', 'NKCC1']\n",
      "['N1/12', 'N161/20']\n",
      "['KCC2', 'NKCC1']\n",
      "['K20/78', 'K17/70', 'K13/31']\n",
      "['Kv1.1 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K20/78', 'K17/70', 'K13/31']\n",
      "['Kv1.1 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K20/78', 'K17/70', 'K13/31']\n",
      "['Kv1.1 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K20/78', 'K17/70', 'K13/31']\n",
      "['Kv1.1 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K17/70']\n",
      "['Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K17/70']\n",
      "['Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K17/70']\n",
      "['Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K17/70', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K17/70', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K17/70', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31', 'K17/70', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K13/31', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K13/31', 'N72/16']\n",
      "['Kv1.4 K+ channel', 'Kv3.4 K+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K13/31', 'K14/16', 'K20/78']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K20/78', 'K36/15', 'K13/31', 'K57/1']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['K20/78', 'K36/15', 'K13/31', 'K57/1']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['K20/78', 'K36/15', 'K13/31', 'K57/1']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['K20/78', 'K36/15', 'K13/31', 'K57/1']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['K20/78', 'K36/15', 'K13/31', 'K57/1']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['N149/25']\n",
      "['Olig1']\n",
      "['K91/36']\n",
      "['WAVE1/SCAR']\n",
      "['K91/36']\n",
      "['WAVE1/SCAR']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'N28/9', 'N19/2', 'K28/74']\n",
      "['PSD-95', 'PSD-95', 'SAP102', 'VGlut1']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['L6/60', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N39B/8']\n",
      "['GIT1']\n",
      "['N83/48']\n",
      "['GIT2']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N75/33']\n",
      "['mGluR1/5 (Group I) glutamate receptor']\n",
      "['N75/33']\n",
      "['mGluR1/5 (Group I) glutamate receptor']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N212/7']\n",
      "['TRIP8b (constant)']\n",
      "['N70/28', 'N114/10']\n",
      "['HCN1', 'HCN4']\n",
      "['N70/28', 'N114/10']\n",
      "['HCN1', 'HCN4']\n",
      "['N70/28', 'N114/10']\n",
      "['HCN1', 'HCN4']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28', 'N114/10']\n",
      "['HCN1', 'HCN4']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28', 'N114/10']\n",
      "['HCN1', 'HCN4']\n",
      "['N70/28', 'N114/10']\n",
      "['HCN1', 'HCN4']\n",
      "['N70/28', 'N114/10']\n",
      "['HCN1', 'HCN4']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N70/28', 'K28/43']\n",
      "['HCN1', 'PSD-95']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N138/6', 'N152B/23', 'N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)', 'LRRK2/Dardarin, N-terminus', 'VDAC1']\n",
      "['N138/6', 'N152B/23', 'N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)', 'LRRK2/Dardarin, N-terminus', 'VDAC1']\n",
      "['N138/6', 'N152B/23', 'N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)', 'LRRK2/Dardarin, N-terminus', 'VDAC1']\n",
      "['N138/6', 'N152B/23', 'N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)', 'LRRK2/Dardarin, N-terminus', 'VDAC1']\n",
      "['N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)']\n",
      "['N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)']\n",
      "['N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)']\n",
      "['N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)']\n",
      "['N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)']\n",
      "['N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N231B/34']\n",
      "['LRRK2/Dardarin, N3 (non-mouse-reactive)']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N29/29', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'VGlut2']\n",
      "['N29/29', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'VGlut2']\n",
      "['N29/29', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'VGlut2']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N166A/26']\n",
      "['OCRL/INPP5b']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K74/71', 'L11A/41']\n",
      "['Nav1.1 Na+ channel', 'Pan-Neurofascin']\n",
      "['K74/71', 'L11A/41']\n",
      "['Nav1.1 Na+ channel', 'Pan-Neurofascin']\n",
      "['K74/71', 'L11A/41']\n",
      "['Nav1.1 Na+ channel', 'Pan-Neurofascin']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K87A/10', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K20/78', 'K14/16']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel']\n",
      "['K20/78', 'K14/16']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel']\n",
      "['K20/78', 'K14/16']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K36/15', 'K14/16']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['K36/15', 'K14/16']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K17/70', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K87A/10', 'K74/71', 'K36/15', 'L11A/41']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.1 Na+ channel', 'Nav1.6 Na+ channel', 'Pan-Neurofascin']\n",
      "['K87A/10', 'K74/71', 'K36/15', 'L11A/41']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.1 Na+ channel', 'Nav1.6 Na+ channel', 'Pan-Neurofascin']\n",
      "['K87A/10', 'K74/71', 'K36/15', 'L11A/41']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.1 Na+ channel', 'Nav1.6 Na+ channel', 'Pan-Neurofascin']\n",
      "['K87A/10', 'K74/71', 'K36/15', 'L11A/41']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.1 Na+ channel', 'Nav1.6 Na+ channel', 'Pan-Neurofascin']\n",
      "['K87A/10', 'K74/71', 'K36/15', 'L11A/41']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.1 Na+ channel', 'Nav1.6 Na+ channel', 'Pan-Neurofascin']\n",
      "['K20/78', 'K14/16', 'N16B/8']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K20/78', 'K14/16', 'N16B/8']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K20/78', 'K14/16', 'N16B/8']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K20/78', 'K14/16', 'N16B/8']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K20/78', 'K14/16', 'N16B/8']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'PSD-95']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K36/15', 'K14/16']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['K36/15', 'K14/16']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K20/78', 'K14/16', 'K87A/10']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K20/78', 'K14/16', 'K87A/10']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K20/78', 'K14/16', 'K87A/10']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K20/78', 'K14/16', 'K87A/10']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K20/78', 'K14/16', 'K87A/10']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N275/14']\n",
      "['Synaptotagmin-7']\n",
      "['N275/14']\n",
      "['Synaptotagmin-7']\n",
      "['N275/14']\n",
      "['Synaptotagmin-7']\n",
      "['N275/14']\n",
      "['Synaptotagmin-7']\n",
      "['N275/14']\n",
      "['Synaptotagmin-7']\n",
      "['N275/14']\n",
      "['Synaptotagmin-7']\n",
      "['N230/21']\n",
      "['MMACHC']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N19/2', 'N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel', 'SAP102']\n",
      "['N19/2', 'N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel', 'SAP102']\n",
      "['N19/2', 'N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel', 'SAP102']\n",
      "['N19/2', 'N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel', 'SAP102']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N377/20']\n",
      "['MAP3K12']\n",
      "['N377/20']\n",
      "['MAP3K12']\n",
      "['N345/51']\n",
      "['REEP1']\n",
      "['N43/6', 'K75/41']\n",
      "['Kv4.3 K+ channel', 'Kv7.4/KCNQ4 K+ channel']\n",
      "['N43/6', 'K75/41']\n",
      "['Kv4.3 K+ channel', 'Kv7.4/KCNQ4 K+ channel']\n",
      "['N43/6', 'K75/41']\n",
      "['Kv4.3 K+ channel', 'Kv7.4/KCNQ4 K+ channel']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N43/6', 'N37A/10', 'N26A/23']\n",
      "['Kv7.1/KCNQ1 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Kv7.4/KCNQ4 K+ channel']\n",
      "['N43/6', 'N37A/10', 'N26A/23']\n",
      "['Kv7.1/KCNQ1 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Kv7.4/KCNQ4 K+ channel']\n",
      "['N43/6', 'N37A/10', 'N26A/23']\n",
      "['Kv7.1/KCNQ1 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Kv7.4/KCNQ4 K+ channel']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N100/13']\n",
      "['GST']\n",
      "['N100/13']\n",
      "['GST']\n",
      "['K96/7']\n",
      "['Pancortin']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K96/7', 'K28/43']\n",
      "['Pancortin', 'PSD-95']\n",
      "['K96/7']\n",
      "['Pancortin']\n",
      "['K96/7']\n",
      "['Pancortin']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['L45/30']\n",
      "['SynCAM1']\n",
      "['L114/3', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['L114/3', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['L114/3', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Parvalbumin']\n",
      "['N262/16']\n",
      "['Phosphoserine-2-OMe']\n",
      "['N257/25']\n",
      "['Phosphoserine-2-OEt']\n",
      "['N257/25', 'N262/16']\n",
      "['Phosphoserine-2-OEt', 'Phosphoserine-2-OMe']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['N112B/14', 'N124B/38']\n",
      "['Kir2.1 K+ channel', 'Kir2.2 K+ channel']\n",
      "['N112B/14', 'N124B/38']\n",
      "['Kir2.1 K+ channel', 'Kir2.2 K+ channel']\n",
      "['N112B/14', 'N124B/38']\n",
      "['Kir2.1 K+ channel', 'Kir2.2 K+ channel']\n",
      "['N97A/31', 'N110/29']\n",
      "['Neuroligin-1', 'Neuroligin-3']\n",
      "['N97A/31', 'N110/29']\n",
      "['Neuroligin-1', 'Neuroligin-3']\n",
      "['N97A/31', 'N110/29']\n",
      "['Neuroligin-1', 'Neuroligin-3']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N19/2', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N19/2', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N19/2', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N59/36', 'N19/2', 'N23B/6']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102', 'Shank2']\n",
      "['N59/36', 'N19/2', 'N23B/6']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102', 'Shank2']\n",
      "['N59/36', 'N19/2', 'N23B/6']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102', 'Shank2']\n",
      "['N59/36', 'N19/2', 'N23B/6']\n",
      "['GluN2B/NR2B glutamate receptor', 'SAP102', 'Shank2']\n",
      "['N19/2', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N19/2', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N19/2', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N18/30', 'N19/2', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N170A/1', 'N97A/31']\n",
      "['Neurexin-1-Beta', 'Neuroligin-1']\n",
      "['N170A/1', 'N97A/31']\n",
      "['Neurexin-1-Beta', 'Neuroligin-1']\n",
      "['N170A/1', 'N97A/31']\n",
      "['Neurexin-1-Beta', 'Neuroligin-1']\n",
      "['N170A/1']\n",
      "['Neurexin-1-Beta']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N170A/1', 'N97A/31']\n",
      "['Neurexin-1-Beta', 'Neuroligin-1']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-MAGUK']\n",
      "['K28/86', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-MAGUK']\n",
      "['K28/86', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K17/70']\n",
      "['Kvbeta2/KCNAB2 K+ channel']\n",
      "['K17/70']\n",
      "['Kvbeta2/KCNAB2 K+ channel']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N4/15']\n",
      "['PINK1']\n",
      "['N4/15']\n",
      "['PINK1']\n",
      "['N182/17']\n",
      "['QKI-6']\n",
      "['L6/60', 'L24/18']\n",
      "['IP3 receptor, type 1', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['N11/33']\n",
      "['KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N11/33']\n",
      "['KCNT2/Slo2.1/Slick K+ channel']\n",
      "['K28/86', 'N6/38', 'N28/9', 'N114/10']\n",
      "['HCN4', 'Pan-MAGUK', 'VAChT', 'VGlut1']\n",
      "['K28/86', 'N6/38', 'N28/9', 'N114/10']\n",
      "['HCN4', 'Pan-MAGUK', 'VAChT', 'VGlut1']\n",
      "['K28/86', 'N6/38', 'N28/9', 'N114/10']\n",
      "['HCN4', 'Pan-MAGUK', 'VAChT', 'VGlut1']\n",
      "['N6/38']\n",
      "['VAChT']\n",
      "['K28/86', 'N6/38', 'N28/9', 'N114/10']\n",
      "['HCN4', 'Pan-MAGUK', 'VAChT', 'VGlut1']\n",
      "['K28/86', 'N6/38', 'N28/9', 'N114/10']\n",
      "['HCN4', 'Pan-MAGUK', 'VAChT', 'VGlut1']\n",
      "['K28/86', 'N6/38', 'N28/9', 'N114/10']\n",
      "['HCN4', 'Pan-MAGUK', 'VAChT', 'VGlut1']\n",
      "['K28/86', 'N6/38', 'N28/9', 'N114/10']\n",
      "['HCN4', 'Pan-MAGUK', 'VAChT', 'VGlut1']\n",
      "['N6/38']\n",
      "['VAChT']\n",
      "['K75/41', 'K55/7']\n",
      "['KChIP1 K+ channel', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K55/7']\n",
      "['KChIP1 K+ channel', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K55/7']\n",
      "['KChIP1 K+ channel', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K55/7']\n",
      "['KChIP1 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N192/12']\n",
      "['Gs protein, alpha subunit']\n",
      "['N192/12']\n",
      "['Gs protein, alpha subunit']\n",
      "['N192/12']\n",
      "['Gs protein, alpha subunit']\n",
      "['N192/12']\n",
      "['Gs protein, alpha subunit']\n",
      "['N192/12']\n",
      "['Gs protein, alpha subunit']\n",
      "['N279B/27']\n",
      "['GluK5/Grik5/KA2 kainate receptor']\n",
      "['N279B/27']\n",
      "['GluK5/Grik5/KA2 kainate receptor']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['L51/82']\n",
      "['MMP9']\n",
      "['L51/82']\n",
      "['MMP9']\n",
      "['L51/82']\n",
      "['MMP9']\n",
      "['L51/82']\n",
      "['MMP9']\n",
      "['L51/82']\n",
      "['MMP9']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel']\n",
      "['K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel']\n",
      "['K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K69/3', 'K74/71', 'K87A/10']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['K87A/10', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['K87A/10', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['N56/21', 'N94/17', 'K69/3']\n",
      "['FGF12/FHF1', 'FGF14/FHF4', 'Nav1.2 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K20/78', 'K75/41', 'K69/3']\n",
      "['Kv1.1 K+ channel', 'Kv4.3 K+ channel', 'Nav1.2 Na+ channel']\n",
      "['K20/78', 'K75/41', 'K69/3']\n",
      "['Kv1.1 K+ channel', 'Kv4.3 K+ channel', 'Nav1.2 Na+ channel']\n",
      "['K20/78', 'K75/41', 'K69/3']\n",
      "['Kv1.1 K+ channel', 'Kv4.3 K+ channel', 'Nav1.2 Na+ channel']\n",
      "['K20/78', 'K75/41', 'K69/3']\n",
      "['Kv1.1 K+ channel', 'Kv4.3 K+ channel', 'Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18', 'K69/3']\n",
      "['Nav1.2 Na+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L102/45', 'L21/32', 'L42/17', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-93/Chapsyn-110', 'SynDIG1/Tmem90b', 'SynDIG4/Prrt1']\n",
      "['L102/45', 'L21/32', 'L42/17', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-93/Chapsyn-110', 'SynDIG1/Tmem90b', 'SynDIG4/Prrt1']\n",
      "['L102/45', 'L21/32', 'L42/17', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-93/Chapsyn-110', 'SynDIG1/Tmem90b', 'SynDIG4/Prrt1']\n",
      "['L102/45', 'L21/32', 'L42/17', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-93/Chapsyn-110', 'SynDIG1/Tmem90b', 'SynDIG4/Prrt1']\n",
      "['L102/45', 'L21/32', 'L42/17', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-93/Chapsyn-110', 'SynDIG1/Tmem90b', 'SynDIG4/Prrt1']\n",
      "['L102/45', 'L21/32', 'L42/17', 'N18/30']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-93/Chapsyn-110', 'SynDIG1/Tmem90b', 'SynDIG4/Prrt1']\n",
      "['L42/17']\n",
      "['SynDIG1/Tmem90b']\n",
      "['N171/17']\n",
      "['INPP4b']\n",
      "['N171/17']\n",
      "['INPP4b']\n",
      "['N18/28', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'PSD-93/Chapsyn-110']\n",
      "['N18/28', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'PSD-93/Chapsyn-110']\n",
      "['N18/28', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'PSD-93/Chapsyn-110']\n",
      "['N18/28', 'K28/43']\n",
      "['PSD-93/Chapsyn-110', 'PSD-95']\n",
      "['N18/28', 'K28/43']\n",
      "['PSD-93/Chapsyn-110', 'PSD-95']\n",
      "['N18/28', 'K28/43']\n",
      "['PSD-93/Chapsyn-110', 'PSD-95']\n",
      "['L80/21']\n",
      "['Kv2.1 K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9', 'A12/18']\n",
      "['Pan-Neurofascin (extracellular)', 'VGlut1']\n",
      "['N28/9', 'A12/18']\n",
      "['Pan-Neurofascin (extracellular)', 'VGlut1']\n",
      "['N28/9', 'A12/18']\n",
      "['Pan-Neurofascin (extracellular)', 'VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N138/6', 'N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N-terminus']\n",
      "['N138/6', 'N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N-terminus']\n",
      "['N138/6', 'N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N138/6', 'N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N-terminus']\n",
      "['N138/6', 'N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N-terminus']\n",
      "['N138/6', 'N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'K56A/50']\n",
      "['CASK', 'PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'PSD-95']\n",
      "['K28/43', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'PSD-95']\n",
      "['K28/43', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'PSD-95']\n",
      "['K28/43', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N23B/49', 'N127/31', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-SAPAP', 'Pan-Shank']\n",
      "['N23B/49', 'N127/31', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-SAPAP', 'Pan-Shank']\n",
      "['N23B/49', 'N127/31', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-SAPAP', 'Pan-Shank']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N245/36', 'K64/15']\n",
      "['SAP97', 'TARPGamma2+4+8']\n",
      "['N245/36', 'K64/15']\n",
      "['SAP97', 'TARPGamma2+4+8']\n",
      "['N245/36', 'K64/15']\n",
      "['SAP97', 'TARPGamma2+4+8']\n",
      "['N245/36', 'K64/15']\n",
      "['SAP97', 'TARPGamma2+4+8']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31', 'K28/43']\n",
      "['Neuroligin-1', 'PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N127/31', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-SAPAP']\n",
      "['N127/31', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-SAPAP']\n",
      "['N127/31', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-SAPAP']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K28/43', 'K64/15']\n",
      "['PSD-95', 'SAP97']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K56A/50', 'L21/32', 'K64/15']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'SAP97']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N127/31', 'N18/30']\n",
      "['Pan-SAPAP', 'PSD-93/Chapsyn-110']\n",
      "['N127/31', 'N18/30']\n",
      "['Pan-SAPAP', 'PSD-93/Chapsyn-110']\n",
      "['N127/31', 'N18/30']\n",
      "['Pan-SAPAP', 'PSD-93/Chapsyn-110']\n",
      "['N127/31', 'N18/30']\n",
      "['Pan-SAPAP', 'PSD-93/Chapsyn-110']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L88/86']\n",
      "['TrpM4']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N134/12', 'N68/6']\n",
      "['Nav1.7 Na+ channel', 'Nav1.8 Na+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N25/35']\n",
      "['Kir2.3 K+ channel']\n",
      "['N25/35']\n",
      "['Kir2.3 K+ channel']\n",
      "['N77/15']\n",
      "['TrpC4']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['L8/15']\n",
      "['Snapin/SNAPAP']\n",
      "['K57/1', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K57/1', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K57/1', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K57/1', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K57/1', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K55/7']\n",
      "['KChIP1 K+ channel']\n",
      "['K60/73', 'K75/41', 'K55/7', 'K57/1']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K60/73', 'K75/41', 'K55/7', 'K57/1']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K60/73', 'K75/41', 'K55/7', 'K57/1']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K60/73', 'K75/41', 'K55/7', 'K57/1']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K60/73', 'K75/41', 'K55/7', 'K57/1']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K55/7']\n",
      "['KChIP1 K+ channel']\n",
      "['N423/75', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'KChIP4 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N423/75', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'KChIP4 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N423/75', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'KChIP4 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N423/75', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'KChIP4 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N423/75', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'KChIP4 K+ channel', 'Kv4.3 K+ channel']\n",
      "['K55/7']\n",
      "['KChIP1 K+ channel']\n",
      "['K75/41', 'K55/7']\n",
      "['KChIP1 K+ channel', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K55/7']\n",
      "['KChIP1 K+ channel', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K55/7']\n",
      "['KChIP1 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N423/75', 'K75/41', 'K66/38', 'K57/1']\n",
      "['KChIP3 K+ channel', 'KChIP4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N423/75', 'K75/41', 'K66/38', 'K57/1']\n",
      "['KChIP3 K+ channel', 'KChIP4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N423/75', 'K75/41', 'K66/38', 'K57/1']\n",
      "['KChIP3 K+ channel', 'KChIP4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N423/75', 'K75/41', 'K66/38', 'K57/1']\n",
      "['KChIP3 K+ channel', 'KChIP4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['N70/28', 'N144/32', 'N159/5']\n",
      "['HCN1', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N70/28', 'N144/32', 'N159/5']\n",
      "['HCN1', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N70/28', 'N144/32', 'N159/5']\n",
      "['HCN1', 'Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['N144/32']\n",
      "['Pan-Gamma-protocadherin-A']\n",
      "['N144/32', 'N159/5']\n",
      "['Pan-Gamma-protocadherin-A', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K74/71', 'K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Nav1.1 Na+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1', 'K75/41']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['N50/36']\n",
      "['NGL-2/LRRC4']\n",
      "['N283/7']\n",
      "['Lgi1']\n",
      "['N245/1', 'N245/36']\n",
      "['TARPGamma2/Stargazin', 'TARPGamma2+4+8']\n",
      "['N245/1', 'N245/36']\n",
      "['TARPGamma2/Stargazin', 'TARPGamma2+4+8']\n",
      "['N245/1', 'N245/36']\n",
      "['TARPGamma2/Stargazin', 'TARPGamma2+4+8']\n",
      "['N85/18']\n",
      "['Malin']\n",
      "['N26A/23']\n",
      "['Kv7.2/KCNQ2 K+ channel']\n",
      "['N26A/23']\n",
      "['Kv7.2/KCNQ2 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N26A/23']\n",
      "['Kv7.2/KCNQ2 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N26A/23']\n",
      "['Kv7.2/KCNQ2 K+ channel']\n",
      "['N26A/23']\n",
      "['Kv7.2/KCNQ2 K+ channel']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N170A/26']\n",
      "['Neurexin-1-Beta']\n",
      "['N432/21']\n",
      "['VSP']\n",
      "['N414/25']\n",
      "['NCKX4 Na+/Ca2+/K+ exchanger']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N97A/31', 'K56A/50']\n",
      "['CASK', 'Neuroligin-1']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6', 'N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N135/37']\n",
      "['JIP-2/IB-2']\n",
      "['N135/37']\n",
      "['JIP-2/IB-2']\n",
      "['N289/16']\n",
      "['SUR1']\n",
      "['N289/16']\n",
      "['SUR1']\n",
      "['N289/16']\n",
      "['SUR1']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['L11A/41', 'A12/18']\n",
      "['Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['L11A/41', 'A12/18']\n",
      "['Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['L11A/41', 'A12/18']\n",
      "['Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['L11A/41', 'A12/18']\n",
      "['Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N154/32']\n",
      "['JMJD2A']\n",
      "['N154/32']\n",
      "['JMJD2A']\n",
      "['N154/32']\n",
      "['JMJD2A']\n",
      "['N74/25']\n",
      "['TrpM7']\n",
      "['N74/25']\n",
      "['TrpM7']\n",
      "['N74/25']\n",
      "['TrpM7']\n",
      "['N74/25']\n",
      "['TrpM7']\n",
      "['N74/25']\n",
      "['TrpM7']\n",
      "['N74/25']\n",
      "['TrpM7']\n",
      "['N23B/6']\n",
      "['Shank2']\n",
      "['N23B/6']\n",
      "['Shank2']\n",
      "['N2/16']\n",
      "['KCNU1/Slo3 maxi-K+ channel']\n",
      "['N2/16']\n",
      "['KCNU1/Slo3 maxi-K+ channel']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['L5/1']\n",
      "['SCG10/Stathmin-2']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K60/73', 'K57/1']\n",
      "['KChIP2b K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['K60/73', 'K57/1']\n",
      "['KChIP2b K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L21/32', 'N19/2', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GluA2/GluR2 glutamate receptor', 'SAP102']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36', 'K28/43']\n",
      "['GluN2B/NR2B glutamate receptor', 'PSD-95']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N286/74']\n",
      "['Zebrafish PSD Marker']\n",
      "['L11A/41']\n",
      "['Pan-Neurofascin']\n",
      "['L11A/41']\n",
      "['Pan-Neurofascin']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K20/78', 'K36/15']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['K20/78', 'K36/15']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['N321C/49']\n",
      "['Kirrel3, short and long']\n",
      "['N321C/49']\n",
      "['Kirrel3, short and long']\n",
      "['N321C/49']\n",
      "['Kirrel3, short and long']\n",
      "['N129/15']\n",
      "['Mad3 (human)']\n",
      "['K13/31', 'K17/70', 'N18/30']\n",
      "['Kv1.4 K+ channel', 'Kvbeta2/KCNAB2 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['N221/17', 'N15/4']\n",
      "['TrpV1', 'TrpV3']\n",
      "['N221/17', 'N15/4']\n",
      "['TrpV1', 'TrpV3']\n",
      "['N221/17', 'N15/4']\n",
      "['TrpV1', 'TrpV3']\n",
      "['N15/4']\n",
      "['TrpV3']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K20/78', 'K17/70']\n",
      "['Kv1.1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['N336B/83']\n",
      "['BAF53a']\n",
      "['N336B/83']\n",
      "['BAF53a']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K55/82', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Pan-KChIP K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N111/24']\n",
      "['Mitofusin-1']\n",
      "['N111/24']\n",
      "['Mitofusin-1']\n",
      "['N219/5']\n",
      "['RBM17/SPF45']\n",
      "['N127/31']\n",
      "['Pan-SAPAP']\n",
      "['N121A/1']\n",
      "['Pannexin-2']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N68/6', 'N372B/60', 'N134/12', 'K14/16', 'K65/35', 'K89/34']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Nav1.7 Na+ channel', 'Nav1.8 Na+ channel']\n",
      "['N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Pan-Neurofascin (extracellular)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['K66/38']\n",
      "['KChIP3 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K73/20']\n",
      "['Contactin/F3']\n",
      "['K87A/10', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['L6/48', 'L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K7/45']\n",
      "['Kv1.4 K+ channel', 'Kv1.5 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K55/82', 'K57/1', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Pan-KChIP K+ channel']\n",
      "['K20/78', 'K17/70']\n",
      "['Kv1.1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K91/36']\n",
      "['WAVE1/SCAR']\n",
      "['K51/1', 'L11A/41']\n",
      "['Pan-Neurofascin', 'SH3GL1/Endophilin A2']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['K60/73', 'K75/41']\n",
      "['KChIP2b K+ channel', 'Kv4.3 K+ channel']\n",
      "['K66/38']\n",
      "['KChIP3 K+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K75/41', 'K55/82']\n",
      "['Kv4.3 K+ channel', 'Pan-KChIP K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K87A/10', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K7/45']\n",
      "['Kv1.5 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['N6/38']\n",
      "['VAChT']\n",
      "['K13/31', 'K17/70', 'K57/1', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['K87A/10', 'N56/21']\n",
      "['FGF14/FHF4', 'Nav1.6 Na+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L11A/41', 'K69/3']\n",
      "['Nav1.2 Na+ channel', 'Pan-Neurofascin']\n",
      "['K55/82']\n",
      "['Pan-KChIP K+ channel']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K17/70', 'N68/6']\n",
      "['Kvbeta2/KCNAB2 K+ channel', 'Nav1.7 Na+ channel']\n",
      "['N1/12', 'L11A/41', 'A12/18']\n",
      "['KCC2', 'Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['K87A/10', 'K74/71', 'K65/35', 'K69/3']\n",
      "['CASPR/Neurexin IV', 'Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['K20/78', 'K36/15', 'K7/45']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.5 K+ channel']\n",
      "['N15/4']\n",
      "['TrpV3']\n",
      "['L20/8', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['L20/8', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['K51/1', 'L11A/41', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Pan-Neurofascin', 'SH3GL1/Endophilin A2']\n",
      "['N59/36', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['K60/73', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel']\n",
      "['N39B/8']\n",
      "['GIT1']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['N71/37']\n",
      "['HCN2']\n",
      "['N2/16']\n",
      "['KCNU1/Slo3 maxi-K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K60/73', 'K75/41']\n",
      "['KChIP2b K+ channel', 'Kv4.3 K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K65/35', 'K14/16']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['K74/71', 'K7/45']\n",
      "['Kv1.5 K+ channel', 'Nav1.1 Na+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N37A/10']\n",
      "['Kv7.1/KCNQ1 K+ channel']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N2/16']\n",
      "['KCNU1/Slo3 maxi-K+ channel']\n",
      "['N71/37']\n",
      "['HCN2']\n",
      "['L6/60', 'N2/16']\n",
      "['KCNU1/Slo3 maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K91/36']\n",
      "['WAVE1/SCAR']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel']\n",
      "['N46/30']\n",
      "['ADAM22 (cytoplasmic)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K67/25']\n",
      "['CASPR2']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['K89/34', 'K55/82']\n",
      "['Kv2.1 K+ channel', 'Pan-KChIP K+ channel']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['K87A/10', 'K65/35', 'K69/3']\n",
      "['CASPR/Neurexin IV', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['N8B/1', 'N7/18']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta2 Ca2+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['N8B/1']\n",
      "['Cavbeta2 Ca2+ channel']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['N37A/10']\n",
      "['Kv7.1/KCNQ1 K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K66/38']\n",
      "['KChIP3 K+ channel']\n",
      "['N29/29', 'N28/9']\n",
      "['VGlut1', 'VGlut2']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['N7/18', 'N10/7']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta4 Ca2+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K55/7', 'K57/1']\n",
      "['KChIP1 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['N149/25']\n",
      "['Olig1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['K20/78', 'K14/16']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['N2/16']\n",
      "['KCNU1/Slo3 maxi-K+ channel']\n",
      "['K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel']\n",
      "['K60/73', 'K55/7', 'K66/38', 'K55/82']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Pan-KChIP K+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['K89/34', 'N6/38']\n",
      "['Kv2.1 K+ channel', 'VAChT']\n",
      "['K17/70']\n",
      "['Kvbeta2/KCNAB2 K+ channel']\n",
      "['K37/89', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N210/5']\n",
      "['PhyH/PAHX']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['N23B/49', 'N28/9', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-Shank', 'VGlut1']\n",
      "['N71/37', 'N141/28']\n",
      "['HCN2', 'HCN3']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N29/29', 'N28/9']\n",
      "['VGlut1', 'VGlut2']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K87A/10', 'K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K13/31', 'K60/73', 'L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'KChIP2b K+ channel', 'Kv1.4 K+ channel']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N46/30']\n",
      "['ADAM22 (cytoplasmic)']\n",
      "['L11A/41', 'K69/3']\n",
      "['Nav1.2 Na+ channel', 'Pan-Neurofascin']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['N124B/38']\n",
      "['Kir2.2 K+ channel']\n",
      "['K7/45']\n",
      "['Kv1.5 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N18/30', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP97']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['N72/16', 'N16B/8']\n",
      "['Kv3.1b K+ channel', 'Kv3.4 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N59/36', 'L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N235/22']\n",
      "['Pan-FHF-A']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K28/86', 'N23B/49']\n",
      "['Pan-MAGUK', 'Pan-Shank']\n",
      "['K60/73', 'K57/1']\n",
      "['KChIP2b K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['N6/38']\n",
      "['VAChT']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['K87A/10', 'K65/35', 'K14/16']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['N86/8', 'K37/89', 'K89/34', 'N86/38']\n",
      "['GFP', 'GFP', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['K65/35', 'K14/16']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K60/73', 'K75/41', 'K66/38', 'K57/1']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N195A/16', 'N147/6']\n",
      "['Pan-QKI', 'QKI-5']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N72/16']\n",
      "['Kv3.4 K+ channel']\n",
      "['K60/73', 'K89/34']\n",
      "['KChIP2b K+ channel', 'Kv2.1 K+ channel']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N23B/49', 'N100/13']\n",
      "['GST', 'Pan-Shank']\n",
      "['K65/35', 'K14/16']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/23', 'L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N59/36', 'N28/9', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['K89/34', 'K39/25']\n",
      "['Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['K75/41', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N134/12', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Nav1.8 Na+ channel']\n",
      "['K55/7']\n",
      "['KChIP1 K+ channel']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['K60/73', 'K75/41', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'Kv4.3 K+ channel']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['L5/1']\n",
      "['SCG10/Stathmin-2']\n",
      "['K20/78', 'K36/15']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['K13/31', 'L23/27', 'K19/36', 'K20/78', 'K14/16', 'K7/45']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['K65/35', 'K14/16']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N153/5']\n",
      "['Mitofusin-2']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K75/41', 'K19/36', 'K57/1']\n",
      "['Kv1.6 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['K13/31', 'K14/16', 'K19/36']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel']\n",
      "['L11A/41', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Pan-Neurofascin']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['L5/1']\n",
      "['SCG10/Stathmin-2']\n",
      "['K65/35', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'CASPR/Neurexin IV']\n",
      "['K37/89', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N2/16']\n",
      "['KCNU1/Slo3 maxi-K+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['K14/16', 'N72/16', 'K57/1']\n",
      "['Kv1.2 K+ channel', 'Kv3.4 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['L20/8', 'N59/36', 'N125/10']\n",
      "['GluN2B/NR2B glutamate receptor', 'PICK1', 'Thorase/Atad1']\n",
      "['N39B/8']\n",
      "['GIT1']\n",
      "['K13/31', 'K9/40', 'N124B/38', 'K47/42', 'K17/70', 'N112B/14', 'K57/1', 'K60/73', 'K75/41']\n",
      "['KChIP2b K+ channel', 'Kir2.1 K+ channel', 'Kir2.2 K+ channel', 'Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Kvbeta1.1/KCNAB1 K+ channel', 'Kvbeta1.2/KCNAB1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N244/5', 'N105/17']\n",
      "['Ankyrin-B', 'SynCAM4']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N23B/6']\n",
      "['Shank2']\n",
      "['N70/28', 'N212A/34', 'N212/3', 'N212/17']\n",
      "['HCN1', 'TRIP8b (constant)', 'TRIP8b (exon 1b)', 'TRIP8b (exon 4)']\n",
      "['K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N129/15']\n",
      "['Mad3 (human)']\n",
      "['K67/25', 'N86/8', 'K17/70', 'K73/20', 'K14/16', 'N86/38']\n",
      "['CASPR2', 'Contactin/F3', 'GFP', 'GFP', 'Kv1.2 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['K20/78', 'K14/16']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel']\n",
      "['K20/78', 'K14/16']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K60/73', 'K75/41']\n",
      "['KChIP2b K+ channel', 'Kv4.3 K+ channel']\n",
      "['N85/18']\n",
      "['Malin']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L57/46', 'K14/16']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Kv1.2 K+ channel']\n",
      "['K74/71', 'K20/78', 'L6/60', 'K14/16', 'N16B/8', 'K75/41', 'K69/3', 'K57/1']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv3.1b K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['K74/71', 'K67/25', 'K20/78', 'K73/20', 'K65/35', 'K87A/10', 'N105/13', 'N105/17']\n",
      "['Ankyrin-B', 'Ankyrin-B', 'CASPR/Neurexin IV', 'CASPR2', 'Contactin/F3', 'Kv1.1 K+ channel', 'Nav1.1 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['K73/20']\n",
      "['Contactin/F3']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N23B/49', 'N59/36', 'N22/21', 'N23B/6']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-Shank', 'Shank1', 'Shank2']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N7/18', 'L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cavbeta1 Ca2+ channel']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['K37/89']\n",
      "['Kv2.2 K+ channel']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N196/16']\n",
      "['PARIS/ZNF746']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N39B/8']\n",
      "['GIT1']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['K14/16', 'N16B/8']\n",
      "['Kv1.2 K+ channel', 'Kv3.1b K+ channel']\n",
      "['N70/28', 'N71/37']\n",
      "['HCN1', 'HCN2']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K89/34', 'K14/16', 'K64/15']\n",
      "['Kv1.2 K+ channel', 'Kv2.1 K+ channel', 'SAP97']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N38/8']\n",
      "['Cav1.3 Ca2+ channel']\n",
      "['K13/31', 'N72/16', 'K20/78', 'K55/82', 'K57/1', 'K60/73', 'K75/41', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv1.1 K+ channel', 'Kv1.4 K+ channel', 'Kv3.4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Pan-KChIP K+ channel']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['K20/78', 'K17/70', 'K65/35', 'K14/16', 'K87A/10', 'K36/15', 'N18/30']\n",
      "['CASPR/Neurexin IV', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kvbeta2/KCNAB2 K+ channel', 'Nav1.6 Na+ channel', 'PSD-93/Chapsyn-110']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N23B/6']\n",
      "['Shank2']\n",
      "['L42/17']\n",
      "['SynDIG1/Tmem90b']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N1/12', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'KCC2']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K74/71', 'L11A/41', 'K20/78', 'N56/21', 'N26A/23', 'K14/16', 'K69/3', 'N105/17']\n",
      "['Ankyrin-B', 'FGF14/FHF4', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Pan-Neurofascin']\n",
      "['N155/9', 'N38/8']\n",
      "['Cav1.3 Ca2+ channel', 'Uncx']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['N72/16']\n",
      "['Kv3.4 K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N19/2', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N126B/31']\n",
      "['Neuregulin-CRD (Cysteine-rich domain, Type III)']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K20/78', 'K14/16']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel']\n",
      "['N149/25']\n",
      "['Olig1']\n",
      "['K66/38', 'K57/1']\n",
      "['KChIP3 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N23B/49', 'N86/38']\n",
      "['GFP', 'Pan-Shank']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N170A/1', 'N97A/31']\n",
      "['Neurexin-1-Beta', 'Neuroligin-1']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N212/17']\n",
      "['TRIP8b (constant)']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['L20/8', 'N19/2', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'PICK1', 'SAP102']\n",
      "['N112B/14', 'N25/35']\n",
      "['Kir2.1 K+ channel', 'Kir2.3 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K47/42', 'K17/70', 'K9/40']\n",
      "['Kvbeta1.1/KCNAB1 K+ channel', 'Kvbeta1.2/KCNAB1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L6/60', 'L18A/3']\n",
      "['BKbeta4 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N116/14']\n",
      "['Botch']\n",
      "['L11A/41']\n",
      "['Pan-Neurofascin']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K67/25', 'K36/15', 'K14/16', 'K19/36']\n",
      "['CASPR2', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.6 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['N111/24']\n",
      "['Mitofusin-1']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K60/73', 'K57/1']\n",
      "['KChIP2b K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['N87/25', 'N96/55', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3']\n",
      "['N159/5']\n",
      "['Pan-Gamma-protocadherin-Constant']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['K60/73', 'K55/7', 'K66/38', 'K55/82']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'KChIP3 K+ channel', 'Pan-KChIP K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L60/4']\n",
      "[\"Copper ATPase 1 (Menke's disease protein)\"]\n",
      "['N165/43']\n",
      "['Pan-PTPR']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N28/9', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'VGlut1']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K75/41', 'K66/38', 'K57/1']\n",
      "['KChIP3 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N230/21']\n",
      "['MMACHC']\n",
      "['N59/36', 'N28/9']\n",
      "['GluN2B/NR2B glutamate receptor', 'VGlut1']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N70/28', 'N71/37', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN4']\n",
      "['N144/17', 'N159/5']\n",
      "['Gamma-protocadherin-A3', 'Pan-Gamma-protocadherin-Constant']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K20/78', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Kv1.1 K+ channel']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N192/12', 'N144/14']\n",
      "['6xHis', 'Gs protein, alpha subunit']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N22/21']\n",
      "['Shank1']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K20/78', 'K75/41', 'K57/1']\n",
      "['Kv1.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K20/78', 'L23/27']\n",
      "['Kv1.1 K+ channel', 'Kv1.3 K+ channel']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N26A/23', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv7.2/KCNQ2 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['L11A/41']\n",
      "['Pan-Neurofascin']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N50/36']\n",
      "['NGL-2/LRRC4']\n",
      "['K37/89', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['N23B/49', 'K56A/50']\n",
      "['CASK', 'Pan-Shank']\n",
      "['N100/13']\n",
      "['GST']\n",
      "['L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['K73/20']\n",
      "['Contactin/F3']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N230/21']\n",
      "['MMACHC']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L57/46', 'N155/9']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Uncx']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K65/35', 'K14/16']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['N206A/8', 'N149/25']\n",
      "['GFAP', 'Olig1']\n",
      "['K13/31', 'K9/40', 'L23/27', 'K19/36', 'K20/78', 'K47/42', 'K17/70', 'K14/16', 'K7/45']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kvbeta1.1/KCNAB1 K+ channel', 'Kvbeta1.2/KCNAB1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N201/35']\n",
      "['Iduna/RNF146']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N212/7', 'N141/28']\n",
      "['HCN3', 'TRIP8b (constant)']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N170A/1']\n",
      "['Neurexin-1-Beta']\n",
      "['N100/13']\n",
      "['GST']\n",
      "['K36/15', 'K14/16']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N6/38']\n",
      "['VAChT']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N37A/10']\n",
      "['Kv7.1/KCNQ1 K+ channel']\n",
      "['N70/28', 'N71/37', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN4']\n",
      "['K13/31', 'K20/78', 'N70/28', 'K14/16', 'K57/1', 'N71/37']\n",
      "['HCN1', 'HCN2', 'Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['L57/46', 'N263/31']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.2 Ca2+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N151/3']\n",
      "['GABA(A)R, Delta']\n",
      "['N26A/23']\n",
      "['Kv7.2/KCNQ2 K+ channel']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP3 K+ channel']\n",
      "['N134/12', 'N68/6']\n",
      "['Nav1.7 Na+ channel', 'Nav1.8 Na+ channel']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N111/24']\n",
      "['Mitofusin-1']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['N151/3']\n",
      "['GABA(A)R, Delta']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['K36/15', 'N16B/8']\n",
      "['Kv1.1 K+ channel (external)', 'Kv3.1b K+ channel']\n",
      "['N196/16']\n",
      "['PARIS/ZNF746']\n",
      "['N87/25', 'N96/55', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GABA(A)R, Beta3']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N100/13']\n",
      "['GST']\n",
      "['K20/78', 'K36/15']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N238/31']\n",
      "['SAPAP1 + SAPAP2']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K20/78', 'N134/12', 'K14/16']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel', 'Nav1.8 Na+ channel']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N71/37', 'N141/28', 'N114/10']\n",
      "['HCN2', 'HCN3', 'HCN4']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['K20/78', 'K36/15', 'K57/1']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv4.2 K+ channel (external)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K65/35', 'K67/25']\n",
      "['CASPR/Neurexin IV', 'CASPR2']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N15/4']\n",
      "['TrpV3']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['N289/16', 'N67/15']\n",
      "['SUR1', 'TrpC5']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K87A/10', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Nav1.6 Na+ channel']\n",
      "['N50/36']\n",
      "['NGL-2/LRRC4']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N149/25']\n",
      "['Olig1']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['K87A/10', 'N56/21', 'K74/71']\n",
      "['FGF14/FHF4', 'Nav1.1 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['K36/15', 'K14/16']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['N59/20', 'N52A/42', 'N59/36', 'N19/2', 'K64/15', 'N18/30']\n",
      "['GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110', 'SAP102', 'SAP97']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['N195A/16', 'N154/32', 'N182/17']\n",
      "['JMJD2A', 'QKI-5', 'QKI-6']\n",
      "['N28/9', 'N165/38']\n",
      "['LAR/PTPRF', 'VGlut1']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N161/20']\n",
      "['NKCC1']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['N308/48', 'N59/36']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N84/37']\n",
      "['Laforin']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N245/1']\n",
      "['TARPGamma2/Stargazin']\n",
      "['N52A/42', 'N59/36', 'N18/30', 'L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Mortalin/GRP75', 'PSD-93/Chapsyn-110']\n",
      "['L20/8', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'PICK1']\n",
      "['N308/48', 'N367/62']\n",
      "['GluN1/NR1 glutamate receptor', 'Shank3']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['N327/95', 'N355/1', 'N308/48', 'L21/32', 'N59/20']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['L5/1', 'K55/82']\n",
      "['Pan-KChIP K+ channel', 'SCG10/Stathmin-2']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N127/31', 'N22/21', 'N23B/6', 'K91/36', 'N87/25']\n",
      "['GABA(A)R, Beta3', 'Pan-SAPAP', 'Shank1', 'Shank2', 'WAVE1/SCAR']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N23B/6']\n",
      "['Shank2']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['N110/29']\n",
      "['Neuroligin-3']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N308/48', 'N206A/8', 'N29/29', 'N59/20']\n",
      "['GFAP', 'GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'VGlut2']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N327/95', 'N59/36', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N26A/23']\n",
      "['Kv7.2/KCNQ2 K+ channel']\n",
      "['K89/34', 'K57/1', 'K60/73', 'K75/41', 'K7/45']\n",
      "['KChIP2b K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N70/28', 'N212/3']\n",
      "['HCN1', 'TRIP8b (exon 4)']\n",
      "['N106/36', 'N86/8', 'N144/14', 'N86/38', 'K14/16']\n",
      "['6xHis', 'Ankyrin-G (staining)', 'GFP', 'GFP', 'Kv1.2 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K91/36']\n",
      "['WAVE1/SCAR']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K47/42', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kvbeta1.2/KCNAB1 K+ channel']\n",
      "['N97A/31', 'N59/20', 'K56A/50']\n",
      "['CASK', 'GluN2B/NR2B glutamate receptor', 'Neuroligin-1']\n",
      "['N154/32', 'N183/15', 'N182/17', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'JMJD2A', 'QKI-6', 'QKI-7']\n",
      "['N212/17']\n",
      "['TRIP8b (constant)']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N207/27', 'K55/82']\n",
      "['LRP4 (extracellular)', 'Pan-KChIP K+ channel']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['L11A/41']\n",
      "['Pan-Neurofascin']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N87/25', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta3']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['K89/34', 'K57/1', 'K39/25']\n",
      "['Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)', 'Kv4.2 K+ channel (external)']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['K37/89', 'K89/34']\n",
      "['Kv2.1 K+ channel', 'Kv2.2 K+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K36/15', 'K89/34', 'K39/25']\n",
      "['Kv1.1 K+ channel (external)', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K39/25']\n",
      "['Kv2.1 K+ channel (external)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N8B/1', 'N263/31', 'K69/3']\n",
      "['Cav1.2 Ca2+ channel', 'Cavbeta2 Ca2+ channel', 'Nav1.2 Na+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['N18/30', 'K64/15']\n",
      "['PSD-93/Chapsyn-110', 'SAP97']\n",
      "['N86/8', 'N86/38']\n",
      "['GFP', 'GFP']\n",
      "['L11A/41', 'A12/18']\n",
      "['Pan-Neurofascin', 'Pan-Neurofascin (extracellular)']\n",
      "['N23B/49', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'Pan-Shank']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N1/66']\n",
      "['KCC2']\n",
      "['K36/15', 'K13/31', 'K75/41', 'N72/16', 'L23/27', 'K19/36', 'K20/78', 'K89/34', 'N16B/8', 'K14/16', 'K57/1', 'K37/89', 'K7/45']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.5 K+ channel', 'Kv1.6 K+ channel', 'Kv2.1 K+ channel', 'Kv2.2 K+ channel', 'Kv3.1b K+ channel', 'Kv3.4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K66/38']\n",
      "['KChIP3 K+ channel']\n",
      "['N116/14']\n",
      "['Botch']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K56A/50', 'N238/29', 'N110/29', 'N52A/42', 'N19/2', 'N105/17', 'N23B/49', 'L28/4', 'K64/15', 'K57/1', 'N238/31', 'N97A/31']\n",
      "['Ankyrin-B', 'CASK', 'Kv4.2 K+ channel', 'Kv4.2 K+ channel (external)', 'Mortalin/GRP75', 'Neuroligin-1', 'Neuroligin-3', 'Pan-Shank', 'SAP102', 'SAP97', 'SAPAP1', 'SAPAP1 + SAPAP2']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N125/10']\n",
      "['Thorase/Atad1']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['L21/32', 'N245/36']\n",
      "['GluA2/GluR2 glutamate receptor', 'TARPGamma2+4+8']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N96/55', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1']\n",
      "['N241A/34', 'N231B/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N3 (non-mouse-reactive)']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N374/48']\n",
      "['TASK1 K+ channel']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N39B/8']\n",
      "['GIT1']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N111/24']\n",
      "['Mitofusin-1']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['K55/82']\n",
      "['Pan-KChIP K+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['L5/1', 'K55/82']\n",
      "['Pan-KChIP K+ channel', 'SCG10/Stathmin-2']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['K60/73', 'K75/41']\n",
      "['KChIP2b K+ channel', 'Kv4.3 K+ channel']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N129/15']\n",
      "['Mad3 (human)']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N327/95', 'N59/20']\n",
      "['GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N196/16']\n",
      "['PARIS/ZNF746']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['K65/35', 'K14/16']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N277/7']\n",
      "['Synaptotagmin-12']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['K28/43', 'K56A/50']\n",
      "['CASK', 'PSD-95']\n",
      "['K89/34', 'K57/1']\n",
      "['Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N129/15']\n",
      "['Mad3 (human)']\n",
      "['N70/28', 'N212/17']\n",
      "['HCN1', 'TRIP8b (constant)']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['K74/71', 'K69/3']\n",
      "['Nav1.1 Na+ channel', 'Nav1.2 Na+ channel']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N202/7']\n",
      "['Fig4/Sac3']\n",
      "['L21/32', 'N245/36']\n",
      "['GluA2/GluR2 glutamate receptor', 'TARPGamma2+4+8']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N125/10', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'Thorase/Atad1']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K91/36']\n",
      "['WAVE1/SCAR']\n",
      "['K74/71', 'K65/35', 'K69/3']\n",
      "['CASPR/Neurexin IV', 'Nav1.1 Na+ channel', 'Nav1.2 Na+ channel']\n",
      "['N308/48', 'N105/13']\n",
      "['Ankyrin-B', 'GluN1/NR1 glutamate receptor']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N355/1', 'K56A/50', 'N159/5', 'N59/20', 'N19/2', 'N23B/49', 'N308/48', 'L21/32', 'N97A/31']\n",
      "['CASK', 'GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Neuroligin-1', 'Pan-Gamma-protocadherin-Constant', 'Pan-Shank', 'SAP102']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K65/35', 'K14/16']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel']\n",
      "['K87A/10', 'K74/71', 'N106/36', 'K69/3']\n",
      "['Ankyrin-G (staining)', 'Nav1.1 Na+ channel', 'Nav1.2 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N123/48', 'N307/12', 'N117/9', 'N299/39', 'N309A/21', 'N123/19', 'N158/28', 'N167/7', 'N312/10', 'N310/21']\n",
      "['Dicer', 'DNMT3L', 'Histone H2A.Z/V', 'Histone H2A.Z-acetyl-Lys4/7/11', 'Histone H3.3', 'Histone H3-acetyl-Lys56', 'Histone H3-pThr11', 'Histone H3-pThr11', 'Histone H4-dimethyl-Arg3', 'NSD1']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['K75/41', 'K7/45']\n",
      "['Kv1.5 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N308/48', 'N59/36']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K14/16', 'N144/14']\n",
      "['6xHis', 'Kv1.2 K+ channel']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N111/24']\n",
      "['Mitofusin-1']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N19/2', 'N18/30']\n",
      "['PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['K13/31', 'N59/36', 'K66/38', 'K57/1']\n",
      "['GluN2B/NR2B glutamate receptor', 'KChIP3 K+ channel', 'Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)']\n",
      "['L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N23B/49', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-Shank']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['K55/7']\n",
      "['KChIP1 K+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N127/31']\n",
      "['Pan-SAPAP']\n",
      "['L20/8', 'N245/1', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'PICK1', 'TARPGamma2/Stargazin']\n",
      "['N23B/6']\n",
      "['Shank2']\n",
      "['K74/71', 'N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Nav1.1 Na+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N127/31', 'N23B/6']\n",
      "['Pan-SAPAP', 'Shank2']\n",
      "['N308/48', 'N59/36']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K14/16', 'K89/34', 'K7/45', 'K39/25']\n",
      "['Kv1.2 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv2.1 K+ channel (external)']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['N2/16']\n",
      "['KCNU1/Slo3 maxi-K+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['K20/78', 'N59/20', 'K65/35', 'K36/15', 'L21/32', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'CASPR/Neurexin IV', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N289/16']\n",
      "['SUR1']\n",
      "['K65/35', 'A12/18']\n",
      "['CASPR/Neurexin IV', 'Pan-Neurofascin (extracellular)']\n",
      "['N26A/23']\n",
      "['Kv7.2/KCNQ2 K+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N195A/16']\n",
      "['QKI-5']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N59/36', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N308/48', 'N59/36', 'N327/95']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N155/9']\n",
      "['Uncx']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N374/48']\n",
      "['TASK1 K+ channel']\n",
      "['N56/21', 'K75/41']\n",
      "['FGF14/FHF4', 'Kv4.3 K+ channel']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N258/5']\n",
      "['Clcn3']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['L124/59', 'L21/32', 'K67/25', 'N355/1']\n",
      "['Bassoon', 'CASPR2', 'GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K87A/10', 'L6/60', 'K65/35', 'L6/48']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel', 'Slo1/BKAlpha maxi-K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Pan-Neurofascin (extracellular)']\n",
      "['N43/6', 'N37A/10', 'N26A/23']\n",
      "['Kv7.1/KCNQ1 K+ channel', 'Kv7.2/KCNQ2 K+ channel', 'Kv7.4/KCNQ4 K+ channel']\n",
      "['N209C/35']\n",
      "['LRRTM2']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K28/86', 'L20/8', 'N18/28', 'N19/2']\n",
      "['Pan-MAGUK', 'PICK1', 'PSD-93/Chapsyn-110', 'SAP102']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K60/73', 'K75/41']\n",
      "['KChIP2b K+ channel', 'Kv4.3 K+ channel']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N28/9', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'VGlut1']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['N110/29']\n",
      "['Neuroligin-3']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['ANTIBODY/1F1', 'N67/15']\n",
      "['TrpC5']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L60/4']\n",
      "[\"Copper ATPase 1 (Menke's disease protein)\"]\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K60/73', 'K89/34', 'N112B/14', 'K7/45']\n",
      "['KChIP2b K+ channel', 'Kir2.1 K+ channel', 'Kv1.5 K+ channel', 'Kv2.1 K+ channel']\n",
      "['L60/4']\n",
      "[\"Copper ATPase 1 (Menke's disease protein)\"]\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N22/21', 'N19/2', 'N23B/6']\n",
      "['SAP102', 'Shank1', 'Shank2']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N34/34']\n",
      "['VGlut3']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N308/48', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N308/48', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['62/3G1']\n",
      "[]\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['K13/31', 'N52A/42', 'K89/34']\n",
      "['Kv1.4 K+ channel', 'Kv2.1 K+ channel', 'Mortalin/GRP75']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N141/28']\n",
      "['HCN3']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['K69/3']\n",
      "['Nav1.2 Na+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N29/29', 'N28/9']\n",
      "['VGlut1', 'VGlut2']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N155/9']\n",
      "['Uncx']\n",
      "['N93A/49', 'N160/21']\n",
      "['GABA(B)R1', 'SynDIG3/Tmem91']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N85/18']\n",
      "['Malin']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N336B/83', 'N59/20']\n",
      "['BAF53a', 'GluN2B/NR2B glutamate receptor']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N86/38']\n",
      "['GFP']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['K13/31', 'N124B/38', 'N112B/14', 'K60/73', 'K75/41']\n",
      "['KChIP2b K+ channel', 'Kir2.1 K+ channel', 'Kir2.2 K+ channel', 'Kv1.4 K+ channel', 'Kv4.3 K+ channel']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N308/48', 'N23B/49']\n",
      "['GluN1/NR1 glutamate receptor', 'Pan-Shank']\n",
      "['N23B/49', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-Shank']\n",
      "['K64/15']\n",
      "['SAP97']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N429/19']\n",
      "['ANO6/TMEM16F']\n",
      "['N327/95', 'N59/36', 'N308/48', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N43/6', 'N37A/10', 'N152B/23']\n",
      "['Kv7.1/KCNQ1 K+ channel', 'Kv7.4/KCNQ4 K+ channel', 'VDAC1']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K75/41', 'N263/31']\n",
      "['Cav1.2 Ca2+ channel', 'Kv4.3 K+ channel']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N106/36', 'K14/16']\n",
      "['Ankyrin-G (staining)', 'Kv1.2 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K74/71', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.1 Na+ channel']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['K87A/10', 'N56/21', 'K65/35', 'K74/71']\n",
      "['CASPR/Neurexin IV', 'FGF14/FHF4', 'Nav1.1 Na+ channel', 'Nav1.6 Na+ channel']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N112B/14']\n",
      "['Kir2.1 K+ channel']\n",
      "['N105/13', 'N106/36']\n",
      "['Ankyrin-B', 'Ankyrin-G (staining)']\n",
      "['N95/35', 'N97A/31']\n",
      "['GABA(A)R, Alpha1', 'Neuroligin-1']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K13/31', 'K75/41', 'K57/1']\n",
      "['Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N70/28', 'N71/37']\n",
      "['HCN1', 'HCN2']\n",
      "['N398A/34', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha4']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N308/48', 'N327/95', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N308/48', 'N59/36', 'K57/1']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv4.2 K+ channel (external)']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['N127/31', 'N59/36']\n",
      "['GluN2B/NR2B glutamate receptor', 'Pan-SAPAP']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N34/34']\n",
      "['VGlut3']\n",
      "['N183/15', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'QKI-7']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K67/25']\n",
      "['CASPR2']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['K60/73', 'K75/41', 'K57/1']\n",
      "['KChIP2b K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N59/20', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['K9/40']\n",
      "['Kvbeta1.1/KCNAB1 K+ channel']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['N23B/49', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'Pan-Shank']\n",
      "['N308/48', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['N106/36', 'A12/18', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Pan-Neurofascin (extracellular)']\n",
      "['N263/31', 'K56A/50']\n",
      "['CASK', 'Cav1.2 Ca2+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N138/6']\n",
      "['LRRK2/Dardarin, N-terminus']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N183/15', 'N182/17', 'N154/32', 'N147/6', 'N195A/16', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'JMJD2A', 'Pan-QKI', 'QKI-5', 'QKI-6', 'QKI-7']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N28/9', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'VGlut1']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K67/25']\n",
      "['CASPR2']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N414/25']\n",
      "['NCKX4 Na+/Ca2+/K+ exchanger']\n",
      "['N245/1']\n",
      "['TARPGamma2/Stargazin']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['K96/7']\n",
      "['Pancortin']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['K75/41', 'K55/82', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel', 'Pan-KChIP K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N422/18']\n",
      "['GluN2C/NR2C glutamate receptor']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['K13/31', 'L23/27', 'K19/36', 'K20/78', 'L6/60']\n",
      "['Kv1.1 K+ channel', 'Kv1.3 K+ channel', 'Kv1.4 K+ channel', 'Kv1.6 K+ channel', 'Slo1/BKAlpha maxi-K+ channel']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['ANTIBODY/1F1']\n",
      "[]\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['L8/15']\n",
      "['Snapin/SNAPAP']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K7/45']\n",
      "['Kv1.5 K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N23B/49', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'Pan-Shank']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K7/45']\n",
      "['Kv1.5 K+ channel']\n",
      "['N52A/42', 'N19/2']\n",
      "['Mortalin/GRP75', 'SAP102']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N86/8', 'N206A/8']\n",
      "['GFAP', 'GFP']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K36/15', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.6 K+ channel']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N23B/6']\n",
      "['Shank2']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N87/25', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta3']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N87/25', 'K28/43', 'L21/32']\n",
      "['GABA(A)R, Beta3', 'GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['K60/73', 'K75/41']\n",
      "['KChIP2b K+ channel', 'Kv4.3 K+ channel']\n",
      "['K13/31', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv1.4 K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N138/6', 'N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus', 'LRRK2/Dardarin, N-terminus']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['K60/73', 'K75/41', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP2b K+ channel', 'Kv4.3 K+ channel']\n",
      "['K7/45']\n",
      "['Kv1.5 K+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N97A/31', 'N110/29']\n",
      "['Neuroligin-1', 'Neuroligin-3']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['L21/32', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GluA2/GluR2 glutamate receptor']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N18/30', 'N25/35', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'Kir2.3 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K91/36']\n",
      "['WAVE1/SCAR']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['L5/1']\n",
      "['SCG10/Stathmin-2']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N186/29']\n",
      "['Dopamine D2 receptor']\n",
      "['N106/36', 'A12/18', 'K56A/50']\n",
      "['Ankyrin-G (staining)', 'CASK', 'Pan-Neurofascin (extracellular)']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['N37A/10']\n",
      "['Kv7.1/KCNQ1 K+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K14/16', 'K36/15', 'N106/36', 'N114/10']\n",
      "['Ankyrin-G (staining)', 'HCN4', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N86/8', 'K91/36']\n",
      "['GFP', 'WAVE1/SCAR']\n",
      "['K20/78', 'N16B/8']\n",
      "['Kv1.1 K+ channel', 'Kv3.1b K+ channel']\n",
      "['K7/45', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kv1.5 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N8B/1']\n",
      "['Cavbeta2 Ca2+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['N393/76', 'A12/18']\n",
      "['Beta4-spectrin', 'Pan-Neurofascin (extracellular)']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N86/8', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'GFP']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['K28/43', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'PSD-95']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N86/8', 'L20/8']\n",
      "['GFP', 'PICK1']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N28/9', 'N19/2']\n",
      "['SAP102', 'VGlut1']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N295B/66', 'K74/71']\n",
      "['Arl13b', 'Nav1.1 Na+ channel']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N11/33', 'N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel', 'KCNT2/Slo2.1/Slick K+ channel']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N133/21', 'K28/74']\n",
      "['PSD-95', 'RGS14']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N11/33']\n",
      "['KCNT2/Slo2.1/Slick K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N414/25']\n",
      "['NCKX4 Na+/Ca2+/K+ exchanger']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['N86/8', 'N241A/34']\n",
      "['GFP', 'LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['N116/14']\n",
      "['Botch']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['L43/40']\n",
      "['DLX1']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['K65/35', 'N106/36', 'K14/16']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'Kv1.2 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K89/34', 'N448/88']\n",
      "['Kv2.1 K+ channel', 'Kv8.2 K+ channel']\n",
      "['N308/48', 'N59/36', 'N327/95']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N72/16']\n",
      "['Kv3.4 K+ channel']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N244/5', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'SynCAM4']\n",
      "['N367/62', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'Shank3']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N50/36']\n",
      "['NGL-2/LRRC4']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N28/9', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'VGlut1']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N15/4']\n",
      "['TrpV3']\n",
      "['N421A/85']\n",
      "['ANO5/TMEM16E']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N46/30']\n",
      "['ADAM22 (cytoplasmic)']\n",
      "['N106/36', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N96/55', '62/3G1']\n",
      "['GABA(A)R, Beta1']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K55/7']\n",
      "['KChIP1 K+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K28/43', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N86/8', 'N206A/8']\n",
      "['GFAP', 'GFP']\n",
      "['K13/31', 'K75/41', 'K57/1']\n",
      "['Kv1.4 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K75/41', 'K57/1']\n",
      "['Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K74/71', 'N68/6', 'N106/20', 'N105/13', 'N393/76']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)', 'Beta4-spectrin', 'Nav1.1 Na+ channel', 'Nav1.7 Na+ channel']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['N86/8', 'N52A/42']\n",
      "['GFP', 'Mortalin/GRP75']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N18/28', 'N19/2', 'K28/74']\n",
      "['PSD-93/Chapsyn-110', 'PSD-95', 'SAP102']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['L43/40']\n",
      "['DLX1']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N97A/31']\n",
      "['Neuroligin-1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N28/9', 'N19/2', 'K28/74']\n",
      "['PSD-95', 'SAP102', 'VGlut1']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K87A/10', 'K74/71', 'N68/6']\n",
      "['Nav1.1 Na+ channel', 'Nav1.6 Na+ channel', 'Nav1.7 Na+ channel']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['K91/36']\n",
      "['WAVE1/SCAR']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['L102/45']\n",
      "['SynDIG4/Prrt1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N59/36', 'L21/32', 'N19/2', 'K56A/50']\n",
      "['CASK', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'SAP102']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['K55/7']\n",
      "['KChIP1 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K14/16', 'A12/18', 'N46/30']\n",
      "['ADAM22 (cytoplasmic)', 'Kv1.2 K+ channel', 'Pan-Neurofascin (extracellular)']\n",
      "['N28/9', 'N355/1', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GluA1/GluR1 glutamate receptor', 'VGlut1']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N106/20']\n",
      "['Ankyrin-G (blotting)']\n",
      "['L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['N7/18', 'N10/7']\n",
      "['Cavbeta1 Ca2+ channel', 'Cavbeta4 Ca2+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['62/3G1', 'N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['N106/36', 'N46/30', 'N106/20']\n",
      "['ADAM22 (cytoplasmic)', 'Ankyrin-G (blotting)', 'Ankyrin-G (staining)']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N15/4']\n",
      "['TrpV3']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N59/36', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N59/36', 'L21/32', 'N19/2']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'SAP102']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K55/82', 'K14/16', 'K57/1', 'K60/73', 'L28/4']\n",
      "['KChIP2b K+ channel', 'Kv1.2 K+ channel', 'Kv4.2 K+ channel', 'Kv4.2 K+ channel (external)', 'Pan-KChIP K+ channel']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N18/30']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K60/73']\n",
      "['KChIP2b K+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N422/18']\n",
      "['GluN2C/NR2C glutamate receptor']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N28/9', 'N34/34']\n",
      "['VGlut1', 'VGlut3']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N308/48', 'N59/36', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N419/40', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N86/8', 'N355/1']\n",
      "['GFP', 'GluA1/GluR1 glutamate receptor']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K36/15', 'K75/41', 'K14/16', 'K19/36']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.6 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N244/5', 'K67/25', 'K56A/50']\n",
      "['CASK', 'CASPR2', 'SynCAM4']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N2/16']\n",
      "['KCNU1/Slo3 maxi-K+ channel']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K55/7']\n",
      "['KChIP1 K+ channel']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N134/12', 'N68/6']\n",
      "['Nav1.7 Na+ channel', 'Nav1.8 Na+ channel']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N56/21', 'N393/76']\n",
      "['Beta4-spectrin', 'FGF14/FHF4']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['L106/83']\n",
      "['Gephyrin']\n",
      "['N308/48', 'N28/9']\n",
      "['GluN1/NR1 glutamate receptor', 'VGlut1']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N141/28']\n",
      "['HCN3']\n",
      "['N393/76']\n",
      "['Beta4-spectrin']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N28/9', 'L118/80']\n",
      "['VGAT', 'VGlut1']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N87/25', 'L124/59', 'K28/43', 'L21/32']\n",
      "['Bassoon', 'GABA(A)R, Beta3', 'GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N1/66']\n",
      "['KCC2']\n",
      "['N59/36', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['K74/71', 'N106/36', 'N56/21', 'K65/35', 'K87A/10', 'N419/40']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV', 'FGF14/FHF4', 'Nav1.1 Na+ channel', 'Nav1.6 Na+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N106/36', 'N105/17', 'N106/20']\n",
      "['Ankyrin-B', 'Ankyrin-G (blotting)', 'Ankyrin-G (staining)']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N23B/49', 'N59/20', 'K56A/50']\n",
      "['CASK', 'GluN2B/NR2B glutamate receptor', 'Pan-Shank']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N206A/8', 'N411/51']\n",
      "['Arx', 'GFAP']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['K65/35', 'N393/76']\n",
      "['Beta4-spectrin', 'CASPR/Neurexin IV']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['N10/7']\n",
      "['Cavbeta4 Ca2+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K28/43', 'N23B/6']\n",
      "['PSD-95', 'Shank2']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K20/78', 'K36/15']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K20/78', 'K36/15']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['K75/41', 'N423/75', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP4 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K73/20']\n",
      "['Contactin/F3']\n",
      "['L114/3', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'Parvalbumin']\n",
      "['K28/43', 'K56A/50']\n",
      "['CASK', 'PSD-95']\n",
      "['N106/36', 'N393/76']\n",
      "['Ankyrin-G (staining)', 'Beta4-spectrin']\n",
      "['N106/36', 'A12/18']\n",
      "['Ankyrin-G (staining)', 'Pan-Neurofascin (extracellular)']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['K91/36', 'K66/38', 'K55/7']\n",
      "['KChIP1 K+ channel', 'KChIP3 K+ channel', 'WAVE1/SCAR']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N87/25', 'N415/24', 'N452/69', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha5', 'GABA(A)R, Beta3', 'GABA(A)R, Gamma2L']\n",
      "['N308/48', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N103/39', 'N103/31']\n",
      "['Aldh1L1 (blotting)', 'Aldh1L1 (staining)']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['L109/57', 'N59/36', 'K89/34', 'N106/20', 'K75/41', 'N355/1']\n",
      "['Ankyrin-G (blotting)', 'Calbindin', 'GluA1/GluR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor', 'Kv2.1 K+ channel', 'Kv4.3 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N183/15', 'N106/65']\n",
      "['Ankyrin-G (staining)', 'QKI-7']\n",
      "['N106/36', 'N393/76', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)', 'Beta4-spectrin']\n",
      "['N28/9', 'N34/34']\n",
      "['VGlut1', 'VGlut3']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K28/43', 'N95/35', 'N97A/31']\n",
      "['GABA(A)R, Alpha1', 'Neuroligin-1', 'PSD-95']\n",
      "['K60/73', 'K75/41', 'K66/38', 'K57/1']\n",
      "['KChIP2b K+ channel', 'KChIP3 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N18/28']\n",
      "['PSD-93/Chapsyn-110']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N106/36', 'N106/20']\n",
      "['Ankyrin-G (blotting)', 'Ankyrin-G (staining)']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N15/4']\n",
      "['TrpV3']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['L6/23']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N72/16', 'N16B/8']\n",
      "['Kv3.1b K+ channel', 'Kv3.4 K+ channel']\n",
      "['K20/78', 'K14/16']\n",
      "['Kv1.1 K+ channel', 'Kv1.2 K+ channel']\n",
      "['N327/95', 'N59/36', 'N308/48', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['K20/78', 'K36/15', 'K9/40']\n",
      "['Kv1.1 K+ channel', 'Kv1.1 K+ channel (external)', 'Kvbeta1.1/KCNAB1 K+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K13/31', 'N106/36', 'N59/36', 'K14/16', 'K36/15', 'N18/30']\n",
      "['Ankyrin-G (staining)', 'GluN2B/NR2B glutamate receptor', 'Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel', 'Kv1.4 K+ channel', 'PSD-93/Chapsyn-110']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['K91/36', '62/3G1', 'N110/29']\n",
      "['Neuroligin-3', 'WAVE1/SCAR']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N414/25']\n",
      "['NCKX4 Na+/Ca2+/K+ exchanger']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N308/48', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N419/40', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Pan-Nav1 Na+ channel']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N43/6']\n",
      "['Kv7.4/KCNQ4 K+ channel']\n",
      "['N72/16']\n",
      "['Kv3.4 K+ channel']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N327/95', 'N95/35', 'N308/48', 'L21/32', 'N59/20']\n",
      "['GABA(A)R, Alpha1', 'GluA2/GluR2 glutamate receptor', 'GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N422/18']\n",
      "['GluN2C/NR2C glutamate receptor']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N70/28', 'N52A/42']\n",
      "['HCN1', 'Mortalin/GRP75']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['K91/36', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'WAVE1/SCAR']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L106/83']\n",
      "['Gephyrin']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['K37/89', 'K14/16']\n",
      "['Kv1.2 K+ channel', 'Kv2.2 K+ channel']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['N133/35']\n",
      "['RGS14']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N271/44']\n",
      "['ASIC1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N144/14', 'N59/20']\n",
      "['6xHis', 'GluN2B/NR2B glutamate receptor']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['K28/43', 'N28/9', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'PSD-95', 'VGlut1']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N103/31']\n",
      "['Aldh1L1 (staining)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N59/36']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['K28/43', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['N327/95', 'N59/36', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['L102/45', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'SynDIG4/Prrt1']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N398A/34', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Alpha4']\n",
      "['L106/83', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'Gephyrin']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['K65/35', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'CASPR/Neurexin IV']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K20/78']\n",
      "['Kv1.1 K+ channel']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L57/46', 'N263/31']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels', 'Cav1.2 Ca2+ channel']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N385/21', 'N106/36', 'L109/57']\n",
      "['Ankyrin-G (staining)', 'Beta1-spectrin', 'Calbindin']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K87A/10', 'K65/35']\n",
      "['CASPR/Neurexin IV', 'Nav1.6 Na+ channel']\n",
      "['N327/95']\n",
      "['GluN2A/NR2A glutamate receptor']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['N206A/8', 'N106/36']\n",
      "['Ankyrin-G (staining)', 'GFAP']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K67/25']\n",
      "['CASPR2']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['N87/25', 'N452/69', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta3', 'GABA(A)R, Gamma2L']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N308/48', 'N59/36']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N134/12']\n",
      "['Nav1.8 Na+ channel']\n",
      "['N147/6']\n",
      "['Pan-QKI']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N94/17', 'N152B/23']\n",
      "['FGF12/FHF1', 'VDAC1']\n",
      "['N43/6', 'N37A/10']\n",
      "['Kv7.1/KCNQ1 K+ channel', 'Kv7.4/KCNQ4 K+ channel']\n",
      "['N377/20']\n",
      "['MAP3K12']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K87A/10']\n",
      "['Nav1.6 Na+ channel']\n",
      "['N10/7', 'K28/43']\n",
      "['Cavbeta4 Ca2+ channel', 'PSD-95']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['K74/71']\n",
      "['Nav1.1 Na+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['N103/39']\n",
      "['Aldh1L1 (blotting)']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['62/3G1', 'N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['K72/8']\n",
      "['Co-Rest/RCOR1']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N419/40', 'K74/71']\n",
      "['Nav1.1 Na+ channel', 'Pan-Nav1 Na+ channel']\n",
      "['N86/8', 'N23B/49']\n",
      "['GFP', 'Pan-Shank']\n",
      "['K17/70', 'K9/40', 'L23/27']\n",
      "['Kv1.3 K+ channel', 'Kvbeta1.1/KCNAB1 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['L23/27']\n",
      "['Kv1.3 K+ channel']\n",
      "['N70/28']\n",
      "['HCN1']\n",
      "['K17/70', 'K7/45']\n",
      "['Kv1.5 K+ channel', 'Kvbeta2/KCNAB2 K+ channel']\n",
      "['N244/5']\n",
      "['SynCAM4']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['N3/26']\n",
      "['KCNT1/Slo2.2/Slack K+ channel']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['N105/17']\n",
      "['Ankyrin-B']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L20/8']\n",
      "['PICK1']\n",
      "['K65/35', 'K14/16']\n",
      "['CASPR/Neurexin IV', 'Kv1.2 K+ channel']\n",
      "['N415/24']\n",
      "['GABA(A)R, Alpha5']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['N95/35', 'N28/9', 'L118/80', 'L106/83', 'N308/48']\n",
      "['GABA(A)R, Alpha1', 'Gephyrin', 'GluN1/NR1 glutamate receptor', 'VGAT', 'VGlut1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N114/10']\n",
      "['HCN4']\n",
      "['N59/36', 'K28/43']\n",
      "['GluN2B/NR2B glutamate receptor', 'PSD-95']\n",
      "['N67/15']\n",
      "['TrpC5']\n",
      "['N308/48', 'N327/95', 'N59/20']\n",
      "['GluN1/NR1 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N422/18']\n",
      "['GluN2C/NR2C glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['L28/4']\n",
      "['Kv4.2 K+ channel']\n",
      "['N59/20']\n",
      "['GluN2B/NR2B glutamate receptor']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N59/36', 'L21/32']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L6/23']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K36/15', 'K14/16']\n",
      "['Kv1.1 K+ channel (external)', 'Kv1.2 K+ channel']\n",
      "['N25/35']\n",
      "['Kir2.3 K+ channel']\n",
      "['N486/32']\n",
      "['c-Fos']\n",
      "['K60/73', 'K75/41']\n",
      "['KChIP2b K+ channel', 'Kv4.3 K+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['L57/46']\n",
      "['Cav1.2 + Cav1.3 Ca2+ channels']\n",
      "['N7/18']\n",
      "['Cavbeta1 Ca2+ channel']\n",
      "['K75/41']\n",
      "['Kv4.3 K+ channel']\n",
      "['K36/15']\n",
      "['Kv1.1 K+ channel (external)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K91/36']\n",
      "['WAVE1/SCAR']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K56A/50']\n",
      "['CASK']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['L119/106']\n",
      "['NPY2R/Neuropeptide Y receptor type 2']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N56/21']\n",
      "['FGF14/FHF4']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L124/59']\n",
      "['Bassoon']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N206A/8', 'N147/6']\n",
      "['GFAP', 'Pan-QKI']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N70/28', 'N71/37', 'N141/28', 'N114/10']\n",
      "['HCN1', 'HCN2', 'HCN3', 'HCN4']\n",
      "['N93A/49']\n",
      "['GABA(B)R1']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43', 'N59/20']\n",
      "['GluN2B/NR2B glutamate receptor', 'PSD-95']\n",
      "['L21/32', 'N59/20']\n",
      "['GluA2/GluR2 glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['K57/1']\n",
      "['Kv4.2 K+ channel (external)']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['N244/5', 'K28/43']\n",
      "['PSD-95', 'SynCAM4']\n",
      "['K28/43', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'PSD-95']\n",
      "['N28/9', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'VGlut1']\n",
      "['N221/17']\n",
      "['TrpV1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N355/1']\n",
      "['GluA1/GluR1 glutamate receptor']\n",
      "['L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['N385/21']\n",
      "['Beta1-spectrin']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N393/76', 'N95/35']\n",
      "['Beta4-spectrin', 'GABA(A)R, Alpha1']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['L124/59']\n",
      "['Bassoon']\n",
      "['N308/48']\n",
      "['GluN1/NR1 glutamate receptor']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K65/35', 'N144/14']\n",
      "['6xHis', 'CASPR/Neurexin IV']\n",
      "['N144/14']\n",
      "['6xHis']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N327/95', 'N59/20', 'L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'GluN2A/NR2A glutamate receptor', 'GluN2B/NR2B glutamate receptor']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N414/25']\n",
      "['NCKX4 Na+/Ca2+/K+ exchanger']\n",
      "['K75/41', 'K89/34', 'K7/45', 'K57/1']\n",
      "['Kv1.5 K+ channel', 'Kv2.1 K+ channel', 'Kv4.2 K+ channel (external)', 'Kv4.3 K+ channel']\n",
      "['N23B/49']\n",
      "['Pan-Shank']\n",
      "['N76/3']\n",
      "['Ataxin-1, 11NQ']\n",
      "['A12/18']\n",
      "['Pan-Neurofascin (extracellular)']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N87/25']\n",
      "['GABA(A)R, Beta3']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N95/35', 'N96/55', 'K28/43', 'N28/9', 'N97A/31', 'L21/32', 'N355/1']\n",
      "['GABA(A)R, Alpha1', 'GABA(A)R, Beta1', 'GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor', 'Neuroligin-1', 'PSD-95', 'VGlut1']\n",
      "['K9/40']\n",
      "['Kvbeta1.1/KCNAB1 K+ channel']\n",
      "['L21/32', 'N355/1']\n",
      "['GluA1/GluR1 glutamate receptor', 'GluA2/GluR2 glutamate receptor']\n",
      "['N72/16']\n",
      "['Kv3.4 K+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['K87A/10', 'K36/15']\n",
      "['Kv1.1 K+ channel (external)', 'Nav1.6 Na+ channel']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N97A/31', 'N110/29']\n",
      "['Neuroligin-1', 'Neuroligin-3']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K89/34', 'K28/43']\n",
      "['Kv2.1 K+ channel', 'PSD-95']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['K7/45']\n",
      "['Kv1.5 K+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N207/27']\n",
      "['LRP4 (extracellular)']\n",
      "['N183/15']\n",
      "['QKI-7']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['N241A/34']\n",
      "['LRRK2/Dardarin, C-terminus']\n",
      "['N28/9']\n",
      "['VGlut1']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N105/13', 'N106/36']\n",
      "['Ankyrin-B', 'Ankyrin-G (staining)']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['L109/57']\n",
      "['Calbindin']\n",
      "['N95/35']\n",
      "['GABA(A)R, Alpha1']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N59/36', 'K28/43']\n",
      "['GluN2B/NR2B glutamate receptor', 'PSD-95']\n",
      "['K14/16']\n",
      "['Kv1.2 K+ channel']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N206A/8']\n",
      "['GFAP']\n",
      "['L28/4']\n",
      "['Kv4.2 K+ channel']\n",
      "['N1/12']\n",
      "['KCC2']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N23B/6']\n",
      "['Shank2']\n",
      "['N68/6']\n",
      "['Nav1.7 Na+ channel']\n",
      "['N86/8']\n",
      "['GFP']\n",
      "['N263/31']\n",
      "['Cav1.2 Ca2+ channel']\n",
      "['N18/28', 'K28/43']\n",
      "['PSD-93/Chapsyn-110', 'PSD-95']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N52A/42']\n",
      "['Mortalin/GRP75']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['N116/14']\n",
      "['Botch']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['L24/18']\n",
      "['IP3 receptor, type 1']\n",
      "['K28/43', 'N95/35']\n",
      "['GABA(A)R, Alpha1', 'PSD-95']\n",
      "['N295B/66']\n",
      "['Arl13b']\n",
      "['N106/36']\n",
      "['Ankyrin-G (staining)']\n",
      "['N152B/23']\n",
      "['VDAC1']\n",
      "['L21/32']\n",
      "['GluA2/GluR2 glutamate receptor']\n",
      "['N393/2']\n",
      "['Beta4-spectrin']\n",
      "['K65/35']\n",
      "['CASPR/Neurexin IV']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N46/30']\n",
      "['ADAM22 (cytoplasmic)']\n",
      "['K28/43']\n",
      "['PSD-95']\n",
      "['N19/2']\n",
      "['SAP102']\n",
      "['L6/60']\n",
      "['Slo1/BKAlpha maxi-K+ channel']\n",
      "['N367/62']\n",
      "['Shank3']\n",
      "['N133/21']\n",
      "['RGS14']\n",
      "['K28/74']\n",
      "['PSD-95']\n",
      "['K89/34']\n",
      "['Kv2.1 K+ channel']\n",
      "['N16B/8']\n",
      "['Kv3.1b K+ channel']\n",
      "['N29/29']\n",
      "['VGlut2']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K28/86']\n",
      "['Pan-MAGUK']\n",
      "['K13/31']\n",
      "['Kv1.4 K+ channel']\n",
      "['N97A/31', 'K56A/50']\n",
      "['CASK', 'Neuroligin-1']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-42-def262d400c5>:46: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df_cure['PMID'] = df_cure['PMID'].astype(str)\n",
      "<ipython-input-42-def262d400c5>:48: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df_cure['MAB_ID'] = df_cure['MAB_ID'].str.split(',')\n",
      "<ipython-input-42-def262d400c5>:60: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df_cure[\"citation\"] = \"\"\n",
      "<ipython-input-42-def262d400c5>:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df_cure[\"title\"] = \"\"\n",
      "<ipython-input-42-def262d400c5>:62: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df_cure[\"year\"] = \"\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Error fetching data from PubMed for 28320946.\n",
      "'AuthorList'\n",
      "For 28320946, no record was found.\n",
      "Error fetching data from PubMed for 28320946.\n",
      "'AuthorList'\n",
      "For 28320946, no record was found.\n",
      "Error fetching data from PubMed for 28320946.\n",
      "'AuthorList'\n",
      "For 28320946, no record was found.\n",
      "Error fetching data from PubMed for 26186538.\n",
      "'LastName'\n",
      "For 26186538, no record was found.\n",
      "Error fetching data from PubMed for 26186538.\n",
      "'LastName'\n",
      "For 26186538, no record was found.\n",
      "Error fetching data from PubMed for 31816021.\n",
      "'AuthorList'\n",
      "For 31816021, no record was found.\n",
      "Error fetching data from PubMed for 31816021.\n",
      "'AuthorList'\n",
      "For 31816021, no record was found.\n",
      "Error fetching data from PubMed for 31816021.\n",
      "'AuthorList'\n",
      "For 31816021, no record was found.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>MAB_ID</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>citation</th>\n",
       "      <th>title</th>\n",
       "      <th>year</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[N37A/10]</td>\n",
       "      <td>[Kv7.1/KCNQ1 K+ channel]</td>\n",
       "      <td>22024150</td>\n",
       "      <td>Krzystanek K et al. 2012 Heart Rhythm 9 440-8.</td>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>2012</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>[N70/28]</td>\n",
       "      <td>[HCN1]</td>\n",
       "      <td>29427087</td>\n",
       "      <td>Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...</td>\n",
       "      <td>Neuroinflammation Alters Integrative Propertie...</td>\n",
       "      <td>2018</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>[N212/17]</td>\n",
       "      <td>[TRIP8b (constant)]</td>\n",
       "      <td>28887304</td>\n",
       "      <td>Lyman KA et al. 2017 J Biol Chem 292 17718-17730.</td>\n",
       "      <td>Allostery between two binding sites in the ion...</td>\n",
       "      <td>2017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>[N212/17]</td>\n",
       "      <td>[TRIP8b (constant)]</td>\n",
       "      <td>28127990</td>\n",
       "      <td>Lyman KA et al. 2017 Expert Opin Ther Targets ...</td>\n",
       "      <td>Animal models suggest the TRIP8b-HCN interacti...</td>\n",
       "      <td>2017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>[N144/14]</td>\n",
       "      <td>[6xHis]</td>\n",
       "      <td>22184198</td>\n",
       "      <td>Emond MR et al. 2011 J Cell Biol 195 1115-21.</td>\n",
       "      <td>A complex of Protocadherin-19 and N-cadherin m...</td>\n",
       "      <td>2011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5557</th>\n",
       "      <td>[N29/29]</td>\n",
       "      <td>[VGlut2]</td>\n",
       "      <td>38219919</td>\n",
       "      <td>Go J et al. 2024 Life Sci 339 122413.</td>\n",
       "      <td>Agathobaculum butyriciproducens improves agein...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5558</th>\n",
       "      <td>[K28/86]</td>\n",
       "      <td>[Pan-MAGUK]</td>\n",
       "      <td>38294129</td>\n",
       "      <td>Lee DW et al. 2024 Muscle Nerve 69 498-503.</td>\n",
       "      <td>Protective effect of alpha-lipoic acid and epa...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5559</th>\n",
       "      <td>[K28/86]</td>\n",
       "      <td>[Pan-MAGUK]</td>\n",
       "      <td>38360747</td>\n",
       "      <td>Delhaye M et al. 2024 J Neurosci 44.</td>\n",
       "      <td>Adaptation of Magnified Analysis of the Proteo...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5560</th>\n",
       "      <td>[K13/31]</td>\n",
       "      <td>[Kv1.4 K+ channel]</td>\n",
       "      <td>38367666</td>\n",
       "      <td>Kabakov AY et al. 2024 J Biol Chem 300 105759.</td>\n",
       "      <td>E3 ubiquitin ligase rififylin has yin and yang...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5561</th>\n",
       "      <td>[N97A/31,  K56A/50]</td>\n",
       "      <td>[CASK, Neuroligin-1]</td>\n",
       "      <td>38472649</td>\n",
       "      <td>Marcó de la Cruz B et al. 2024 Nat Neurosci 27...</td>\n",
       "      <td>Liprin-α proteins are master regulators of hum...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5562 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                   MAB_ID                    Target      PMID  \\\n",
       "0               [N37A/10]  [Kv7.1/KCNQ1 K+ channel]  22024150   \n",
       "1                [N70/28]                    [HCN1]  29427087   \n",
       "2               [N212/17]       [TRIP8b (constant)]  28887304   \n",
       "3               [N212/17]       [TRIP8b (constant)]  28127990   \n",
       "4               [N144/14]                   [6xHis]  22184198   \n",
       "...                   ...                       ...       ...   \n",
       "5557             [N29/29]                  [VGlut2]  38219919   \n",
       "5558             [K28/86]               [Pan-MAGUK]  38294129   \n",
       "5559             [K28/86]               [Pan-MAGUK]  38360747   \n",
       "5560             [K13/31]        [Kv1.4 K+ channel]  38367666   \n",
       "5561  [N97A/31,  K56A/50]      [CASK, Neuroligin-1]  38472649   \n",
       "\n",
       "                                               citation  \\\n",
       "0        Krzystanek K et al. 2012 Heart Rhythm 9 440-8.   \n",
       "1     Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...   \n",
       "2     Lyman KA et al. 2017 J Biol Chem 292 17718-17730.   \n",
       "3     Lyman KA et al. 2017 Expert Opin Ther Targets ...   \n",
       "4         Emond MR et al. 2011 J Cell Biol 195 1115-21.   \n",
       "...                                                 ...   \n",
       "5557              Go J et al. 2024 Life Sci 339 122413.   \n",
       "5558        Lee DW et al. 2024 Muscle Nerve 69 498-503.   \n",
       "5559               Delhaye M et al. 2024 J Neurosci 44.   \n",
       "5560     Kabakov AY et al. 2024 J Biol Chem 300 105759.   \n",
       "5561  Marcó de la Cruz B et al. 2024 Nat Neurosci 27...   \n",
       "\n",
       "                                                  title  year  \n",
       "0     Deubiquitylating enzyme USP2 counteracts Nedd4...  2012  \n",
       "1     Neuroinflammation Alters Integrative Propertie...  2018  \n",
       "2     Allostery between two binding sites in the ion...  2017  \n",
       "3     Animal models suggest the TRIP8b-HCN interacti...  2017  \n",
       "4     A complex of Protocadherin-19 and N-cadherin m...  2011  \n",
       "...                                                 ...   ...  \n",
       "5557  Agathobaculum butyriciproducens improves agein...  2024  \n",
       "5558  Protective effect of alpha-lipoic acid and epa...  2024  \n",
       "5559  Adaptation of Magnified Analysis of the Proteo...  2024  \n",
       "5560  E3 ubiquitin ligase rififylin has yin and yang...  2024  \n",
       "5561  Liprin-α proteins are master regulators of hum...  2024  \n",
       "\n",
       "[5562 rows x 6 columns]"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "# import BytesIO to read the data from the url\n",
    "from io import BytesIO\n",
    "import requests\n",
    "import os\n",
    "\n",
    "# merge df_final and dfantiinc on the pmid columns do an outer join\n",
    "# df_final = pd.read_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/googlescholar_mabidsearch_pubmed.csv\")\n",
    "dfantiinc = pd.read_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/antibodiesinc_final-5-20-24.csv\")\n",
    "# set them both PMID and PMID_ID to int\n",
    "df_final['PMID'] = df_final['PMID'].astype(int)\n",
    "dfantiinc['PMID_ID'] = dfantiinc['PMID_ID'].astype(int)\n",
    "df_final = df_final.merge(dfantiinc, left_on='PMID', right_on='PMID_ID', how='outer')\n",
    "# create a final mab_id column that takes all the unique values from the MABID column and the Antibody column\n",
    "df_final['MAB_ID'] = df_final['MABID'].fillna(df_final['Antibody'])\n",
    "df_final['PMID'] = df_final['PMID'].fillna(df_final['PMID_ID'])\n",
    "df_final['PMID'] = df_final['PMID'].astype(int).astype(str)\n",
    "\n",
    "# read the cfm file like before to get the target information\n",
    "data = requests.get(\"https://neuromab.ucdavis.edu/catalog-download.cfm\").content\n",
    "df = pd.read_csv(BytesIO(data))\n",
    "print(df)\n",
    "# for the MAB_ID column create a new column called target that is the target from the df where aall the clones match\n",
    "for index, row in df_final.iterrows():\n",
    "    # look for -f with a number after it with regex\n",
    "    # get the list of clones in the df_final[\"MAB_ID\"]\n",
    "    clones = [i.strip() for i in row[\"MAB_ID\"].split(',')]\n",
    "    print(clones)\n",
    "    print(df[df['Clone'].isin(clones)]['Target'].tolist())\n",
    "    df_final.at[index, 'Target'] = df[df['Clone'].isin(clones)]['Target'].tolist()\n",
    "\n",
    "# now add the pubmed dictionary columns to the df_final\n",
    "df_cure = df_final[['MAB_ID', 'Target', 'PMID']]\n",
    "# do another groupby on the PMID and join the Target column with a comma and the PMID column with a comma\n",
    "# then ensure that the Target values are unique and the MAID_ID values are unique\n",
    "# turn the list columns into strings with a comma by joining\n",
    "\n",
    "# for index, row in df_cure.iterrows():\n",
    "#     if type(row['Target']) == list:\n",
    "#         df_cure.at[index, 'Target'] = ', '.join(set(row['Target']))\n",
    "#     if type(row['MAB_ID']) == list:\n",
    "#         df_cure.at[index, 'MAB_ID'] = ', '.join(set(row['MAB_ID']))\n",
    "\n",
    "# groupby the pmid and join the target and mab_id columns with a comma but make them a set before joining\n",
    "# make PMID a string\n",
    "df_cure['PMID'] = df_cure['PMID'].astype(str)\n",
    "# turn MAB_D and Target into lists\n",
    "df_cure['MAB_ID'] = df_cure['MAB_ID'].str.split(',')\n",
    "# df_cure['Target'] = df_cure['Target'].str.split(',')\n",
    "# df_cure = df_cure.groupby('PMID').agg({'MAB_ID': lambda x: ', '.join(set(str(x))), \"Target\": lambda x: ', '.join(set(str(x)))}).reset_index()\n",
    "\n",
    "# now turn these back into a list and make sure they are unique\n",
    "# for index, row in df_cure.iterrows():\n",
    "#     if type(row['Target']) == str:\n",
    "#         df_cure.at[index, 'Target'] = set(row['Target'].split(', '))\n",
    "#     if type(row['MAB_ID']) == str:\n",
    "#         df_cure.at[index, 'MAB_ID'] = set(row['MAB_ID'].split(', '))\n",
    "\n",
    "# add the citation column to the df_cure using the column output as a dictionary (citation title and year)\n",
    "df_cure[\"citation\"] = \"\"\n",
    "df_cure[\"title\"] = \"\"\n",
    "df_cure[\"year\"] = \"\"\n",
    "for index, row in df_cure.iterrows():\n",
    "    # look for -f with a number after it with regex\n",
    "    # get the list of clones in the df_final[\"MAB_ID\"]\n",
    "    pmid = row[\"PMID\"]\n",
    "    record = fetch_details(pmid)\n",
    "    # check if the record is a dictionary with the relevant keys\n",
    "    if type(record) == dict and 'citation' in record and 'title' in record and 'year' in record:\n",
    "        df_cure.at[index, 'citation'] = record['citation']\n",
    "        df_cure.at[index, 'title'] = record['title']\n",
    "        df_cure.at[index, 'year'] = record['year']\n",
    "    else:\n",
    "        print(f\"For {pmid}, no record was found.\")\n",
    "        \n",
    "df_cure\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "f16def79",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>MAB_ID</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>citation</th>\n",
       "      <th>title</th>\n",
       "      <th>year</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1845</th>\n",
       "      <td>[K67/25,  K36/15,  K13/31,  L23/27]</td>\n",
       "      <td>[CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...</td>\n",
       "      <td>38194969</td>\n",
       "      <td>Miyazaki Y et al. 2024 Cell Rep 43 113634.</td>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor ...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1846</th>\n",
       "      <td>[K67/25,  K36/15,  K13/31,  L23/27]</td>\n",
       "      <td>[CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...</td>\n",
       "      <td>38194969</td>\n",
       "      <td>Miyazaki Y et al. 2024 Cell Rep 43 113634.</td>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor ...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1847</th>\n",
       "      <td>[K67/25,  K36/15,  K13/31,  L23/27]</td>\n",
       "      <td>[CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...</td>\n",
       "      <td>38194969</td>\n",
       "      <td>Miyazaki Y et al. 2024 Cell Rep 43 113634.</td>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor ...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1848</th>\n",
       "      <td>[K67/25,  K36/15,  K13/31,  L23/27]</td>\n",
       "      <td>[CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...</td>\n",
       "      <td>38194969</td>\n",
       "      <td>Miyazaki Y et al. 2024 Cell Rep 43 113634.</td>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor ...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1849</th>\n",
       "      <td>[K67/25,  K36/15,  K13/31,  L23/27]</td>\n",
       "      <td>[CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...</td>\n",
       "      <td>38194969</td>\n",
       "      <td>Miyazaki Y et al. 2024 Cell Rep 43 113634.</td>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor ...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1850</th>\n",
       "      <td>[K67/25,  K36/15,  K13/31,  L23/27]</td>\n",
       "      <td>[CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...</td>\n",
       "      <td>38194969</td>\n",
       "      <td>Miyazaki Y et al. 2024 Cell Rep 43 113634.</td>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor ...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1851</th>\n",
       "      <td>[K67/25,  K36/15,  K13/31,  L23/27]</td>\n",
       "      <td>[CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...</td>\n",
       "      <td>38194969</td>\n",
       "      <td>Miyazaki Y et al. 2024 Cell Rep 43 113634.</td>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor ...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1852</th>\n",
       "      <td>[K67/25,  K36/15,  K13/31,  L23/27]</td>\n",
       "      <td>[CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...</td>\n",
       "      <td>38194969</td>\n",
       "      <td>Miyazaki Y et al. 2024 Cell Rep 43 113634.</td>\n",
       "      <td>Oligodendrocyte-derived LGI3 and its receptor ...</td>\n",
       "      <td>2024</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                   MAB_ID  \\\n",
       "1845  [K67/25,  K36/15,  K13/31,  L23/27]   \n",
       "1846  [K67/25,  K36/15,  K13/31,  L23/27]   \n",
       "1847  [K67/25,  K36/15,  K13/31,  L23/27]   \n",
       "1848  [K67/25,  K36/15,  K13/31,  L23/27]   \n",
       "1849  [K67/25,  K36/15,  K13/31,  L23/27]   \n",
       "1850  [K67/25,  K36/15,  K13/31,  L23/27]   \n",
       "1851  [K67/25,  K36/15,  K13/31,  L23/27]   \n",
       "1852  [K67/25,  K36/15,  K13/31,  L23/27]   \n",
       "\n",
       "                                                 Target      PMID  \\\n",
       "1845  [CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...  38194969   \n",
       "1846  [CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...  38194969   \n",
       "1847  [CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...  38194969   \n",
       "1848  [CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...  38194969   \n",
       "1849  [CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...  38194969   \n",
       "1850  [CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...  38194969   \n",
       "1851  [CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...  38194969   \n",
       "1852  [CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+...  38194969   \n",
       "\n",
       "                                        citation  \\\n",
       "1845  Miyazaki Y et al. 2024 Cell Rep 43 113634.   \n",
       "1846  Miyazaki Y et al. 2024 Cell Rep 43 113634.   \n",
       "1847  Miyazaki Y et al. 2024 Cell Rep 43 113634.   \n",
       "1848  Miyazaki Y et al. 2024 Cell Rep 43 113634.   \n",
       "1849  Miyazaki Y et al. 2024 Cell Rep 43 113634.   \n",
       "1850  Miyazaki Y et al. 2024 Cell Rep 43 113634.   \n",
       "1851  Miyazaki Y et al. 2024 Cell Rep 43 113634.   \n",
       "1852  Miyazaki Y et al. 2024 Cell Rep 43 113634.   \n",
       "\n",
       "                                                  title  year  \n",
       "1845  Oligodendrocyte-derived LGI3 and its receptor ...  2024  \n",
       "1846  Oligodendrocyte-derived LGI3 and its receptor ...  2024  \n",
       "1847  Oligodendrocyte-derived LGI3 and its receptor ...  2024  \n",
       "1848  Oligodendrocyte-derived LGI3 and its receptor ...  2024  \n",
       "1849  Oligodendrocyte-derived LGI3 and its receptor ...  2024  \n",
       "1850  Oligodendrocyte-derived LGI3 and its receptor ...  2024  \n",
       "1851  Oligodendrocyte-derived LGI3 and its receptor ...  2024  \n",
       "1852  Oligodendrocyte-derived LGI3 and its receptor ...  2024  "
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# filter df_cure for the same PMID\n",
    "df_cure[df_cure['PMID'] == \"38194969\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "4df5bbaa",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-44-b8b1931b25da>:9: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df_cure[\"source\"] = \"\"\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:25: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': row['Clone'], 'Target': row['Target']}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Error fetching data from PubMed for 24077800.\n",
      "list index out of range\n",
      "For 24077800, no record was found.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
      "<ipython-input-44-b8b1931b25da>:23: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n"
     ]
    }
   ],
   "source": [
    "# lets add the legacy info so iterate through the df and see if the pubmed is not in the df_cure, if not append it\n",
    "df = pd.read_excel(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/publications 041922.xlsx\")\n",
    "# filter out anything that saays review in the clone\n",
    "# remove all Pubmed that are empty\n",
    "df = df[df['Pubmed'].notna()]\n",
    "# set the Clone to a string column and remove any that are na\n",
    "df['Clone'] = df['Clone'].astype(str)\n",
    "df = df[df['Clone'] != 'nan']\n",
    "df_cure[\"source\"] = \"\"\n",
    "\n",
    "for index, row in df.iterrows():\n",
    "    # look for -f with a number after it with regex\n",
    "    # get the list of clones in the df_final[\"MAB_ID\"]\n",
    "    pmid = str(row[\"Pubmed\"]).split('.')[0]\n",
    "    if pmid not in df_cure['PMID'].tolist():\n",
    "        record = fetch_details(pmid)\n",
    "        # check if the record is a dictionary with the relevant keys\n",
    "        if type(record) == dict and 'citation' in record and 'title' in record and 'year' in record:\n",
    "            \n",
    "            # use the clone column for MAB_ID and the target column for Target just make them a set after splitting on the comma\n",
    "            # check if both are not na first\n",
    "            if type(row['Clone']) == str and type(row['Target']) == str:\n",
    "                df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': set(row['Clone'].split(', ')), 'Target': set(row['Target'].split(', '))}, ignore_index=True)\n",
    "            else:\n",
    "                df_cure = df_cure.append({'PMID': pmid, 'citation': record['citation'], 'title': record['title'], 'year': record['year'], 'source':'legacy', 'MAB_ID': row['Clone'], 'Target': row['Target']}, ignore_index=True)\n",
    "\n",
    "        else:\n",
    "            print(f\"For {pmid}, no record was found.\")\n",
    "\n",
    "\n",
    "# now also iterate through the df_cure is not in the df, if so update the source column to google scholar unique\n",
    "for index, row in df_cure.iterrows():\n",
    "    # look for -f with a number after it with regex\n",
    "    # get the list of clones in the df_final[\"MAB_ID\"]\n",
    "    pmid = str(row[\"PMID\"])\n",
    "    if pmid not in df['Pubmed'].astype(str).tolist():\n",
    "        df_cure.at[index, 'source'] = 'google scholar unique'\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "588cfdbb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>MAB_ID</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>citation</th>\n",
       "      <th>title</th>\n",
       "      <th>year</th>\n",
       "      <th>source</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[N37A/10]</td>\n",
       "      <td>[Kv7.1/KCNQ1 K+ channel]</td>\n",
       "      <td>22024150</td>\n",
       "      <td>Krzystanek K et al. 2012 Heart Rhythm 9 440-8.</td>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>2012</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>[N70/28]</td>\n",
       "      <td>[HCN1]</td>\n",
       "      <td>29427087</td>\n",
       "      <td>Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...</td>\n",
       "      <td>Neuroinflammation Alters Integrative Propertie...</td>\n",
       "      <td>2018</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>[N212/17]</td>\n",
       "      <td>[TRIP8b (constant)]</td>\n",
       "      <td>28887304</td>\n",
       "      <td>Lyman KA et al. 2017 J Biol Chem 292 17718-17730.</td>\n",
       "      <td>Allostery between two binding sites in the ion...</td>\n",
       "      <td>2017</td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>[N212/17]</td>\n",
       "      <td>[TRIP8b (constant)]</td>\n",
       "      <td>28127990</td>\n",
       "      <td>Lyman KA et al. 2017 Expert Opin Ther Targets ...</td>\n",
       "      <td>Animal models suggest the TRIP8b-HCN interacti...</td>\n",
       "      <td>2017</td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>[N144/14]</td>\n",
       "      <td>[6xHis]</td>\n",
       "      <td>22184198</td>\n",
       "      <td>Emond MR et al. 2011 J Cell Biol 195 1115-21.</td>\n",
       "      <td>A complex of Protocadherin-19 and N-cadherin m...</td>\n",
       "      <td>2011</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6254</th>\n",
       "      <td>{N106/65}</td>\n",
       "      <td>{Ankyrin-G}</td>\n",
       "      <td>33536249</td>\n",
       "      <td>Ogawa Y et al. 2021 J Cell Sci 134.</td>\n",
       "      <td>Endogenously expressed Ranbp2 is not at the ax...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6255</th>\n",
       "      <td>{L6/60}</td>\n",
       "      <td>{Slo1/BKAlpha}</td>\n",
       "      <td>33556372</td>\n",
       "      <td>Bukiya AN et al. 2021 J Biol Chem 296 100381.</td>\n",
       "      <td>Cholesterol activates BK channels by increasin...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6256</th>\n",
       "      <td>{N212B}</td>\n",
       "      <td>{TRIP8b}</td>\n",
       "      <td>33547341</td>\n",
       "      <td>Rátkai A et al. 2021 Sci Rep 11 3236.</td>\n",
       "      <td>Homeostatic plasticity and burst activity are ...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6257</th>\n",
       "      <td>{K28/43}</td>\n",
       "      <td>{PSD-95}</td>\n",
       "      <td>33663561</td>\n",
       "      <td>Perkins EM et al. 2021 Mol Neurodegener 16 13.</td>\n",
       "      <td>Altered network properties in C9ORF72 repeat e...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6258</th>\n",
       "      <td>{K28/43}</td>\n",
       "      <td>{PSD-95}</td>\n",
       "      <td>33615226</td>\n",
       "      <td>Gavrilovici C et al. 2021 Cereb Cortex Commun ...</td>\n",
       "      <td>Behavioral Deficits in Mice with Postnatal Dis...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>6259 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         MAB_ID                    Target      PMID  \\\n",
       "0     [N37A/10]  [Kv7.1/KCNQ1 K+ channel]  22024150   \n",
       "1      [N70/28]                    [HCN1]  29427087   \n",
       "2     [N212/17]       [TRIP8b (constant)]  28887304   \n",
       "3     [N212/17]       [TRIP8b (constant)]  28127990   \n",
       "4     [N144/14]                   [6xHis]  22184198   \n",
       "...         ...                       ...       ...   \n",
       "6254  {N106/65}               {Ankyrin-G}  33536249   \n",
       "6255    {L6/60}            {Slo1/BKAlpha}  33556372   \n",
       "6256    {N212B}                  {TRIP8b}  33547341   \n",
       "6257   {K28/43}                  {PSD-95}  33663561   \n",
       "6258   {K28/43}                  {PSD-95}  33615226   \n",
       "\n",
       "                                               citation  \\\n",
       "0        Krzystanek K et al. 2012 Heart Rhythm 9 440-8.   \n",
       "1     Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...   \n",
       "2     Lyman KA et al. 2017 J Biol Chem 292 17718-17730.   \n",
       "3     Lyman KA et al. 2017 Expert Opin Ther Targets ...   \n",
       "4         Emond MR et al. 2011 J Cell Biol 195 1115-21.   \n",
       "...                                                 ...   \n",
       "6254                Ogawa Y et al. 2021 J Cell Sci 134.   \n",
       "6255      Bukiya AN et al. 2021 J Biol Chem 296 100381.   \n",
       "6256              Rátkai A et al. 2021 Sci Rep 11 3236.   \n",
       "6257     Perkins EM et al. 2021 Mol Neurodegener 16 13.   \n",
       "6258  Gavrilovici C et al. 2021 Cereb Cortex Commun ...   \n",
       "\n",
       "                                                  title  year  \\\n",
       "0     Deubiquitylating enzyme USP2 counteracts Nedd4...  2012   \n",
       "1     Neuroinflammation Alters Integrative Propertie...  2018   \n",
       "2     Allostery between two binding sites in the ion...  2017   \n",
       "3     Animal models suggest the TRIP8b-HCN interacti...  2017   \n",
       "4     A complex of Protocadherin-19 and N-cadherin m...  2011   \n",
       "...                                                 ...   ...   \n",
       "6254  Endogenously expressed Ranbp2 is not at the ax...  2021   \n",
       "6255  Cholesterol activates BK channels by increasin...  2021   \n",
       "6256  Homeostatic plasticity and burst activity are ...  2021   \n",
       "6257  Altered network properties in C9ORF72 repeat e...  2021   \n",
       "6258  Behavioral Deficits in Mice with Postnatal Dis...  2021   \n",
       "\n",
       "                     source  \n",
       "0                            \n",
       "1                            \n",
       "2     google scholar unique  \n",
       "3     google scholar unique  \n",
       "4                            \n",
       "...                     ...  \n",
       "6254                 legacy  \n",
       "6255                 legacy  \n",
       "6256                 legacy  \n",
       "6257                 legacy  \n",
       "6258                 legacy  \n",
       "\n",
       "[6259 rows x 7 columns]"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_cure"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "fb4b3090",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "config": {
        "plotlyServerURL": "https://plot.ly"
       },
       "data": [
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "source=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "",
         "offsetgroup": "",
         "orientation": "v",
         "showlegend": false,
         "type": "histogram",
         "x": [
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "google scholar unique",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy"
         ],
         "xaxis": "x",
         "yaxis": "y"
        }
       ],
       "layout": {
        "barmode": "relative",
        "legend": {
         "tracegroupgap": 0
        },
        "margin": {
         "t": 60
        },
        "template": {
         "data": {
          "bar": [
           {
            "error_x": {
             "color": "#2a3f5f"
            },
            "error_y": {
             "color": "#2a3f5f"
            },
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "bar"
           }
          ],
          "barpolar": [
           {
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "barpolar"
           }
          ],
          "carpet": [
           {
            "aaxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "baxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "type": "carpet"
           }
          ],
          "choropleth": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "choropleth"
           }
          ],
          "contour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "contour"
           }
          ],
          "contourcarpet": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "contourcarpet"
           }
          ],
          "heatmap": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmap"
           }
          ],
          "heatmapgl": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmapgl"
           }
          ],
          "histogram": [
           {
            "marker": {
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "histogram"
           }
          ],
          "histogram2d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2d"
           }
          ],
          "histogram2dcontour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2dcontour"
           }
          ],
          "mesh3d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "mesh3d"
           }
          ],
          "parcoords": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "parcoords"
           }
          ],
          "pie": [
           {
            "automargin": true,
            "type": "pie"
           }
          ],
          "scatter": [
           {
            "fillpattern": {
             "fillmode": "overlay",
             "size": 10,
             "solidity": 0.2
            },
            "type": "scatter"
           }
          ],
          "scatter3d": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatter3d"
           }
          ],
          "scattercarpet": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattercarpet"
           }
          ],
          "scattergeo": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergeo"
           }
          ],
          "scattergl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergl"
           }
          ],
          "scattermapbox": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattermapbox"
           }
          ],
          "scatterpolar": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolar"
           }
          ],
          "scatterpolargl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolargl"
           }
          ],
          "scatterternary": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterternary"
           }
          ],
          "surface": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "surface"
           }
          ],
          "table": [
           {
            "cells": {
             "fill": {
              "color": "#EBF0F8"
             },
             "line": {
              "color": "white"
             }
            },
            "header": {
             "fill": {
              "color": "#C8D4E3"
             },
             "line": {
              "color": "white"
             }
            },
            "type": "table"
           }
          ]
         },
         "layout": {
          "annotationdefaults": {
           "arrowcolor": "#2a3f5f",
           "arrowhead": 0,
           "arrowwidth": 1
          },
          "autotypenumbers": "strict",
          "coloraxis": {
           "colorbar": {
            "outlinewidth": 0,
            "ticks": ""
           }
          },
          "colorscale": {
           "diverging": [
            [
             0,
             "#8e0152"
            ],
            [
             0.1,
             "#c51b7d"
            ],
            [
             0.2,
             "#de77ae"
            ],
            [
             0.3,
             "#f1b6da"
            ],
            [
             0.4,
             "#fde0ef"
            ],
            [
             0.5,
             "#f7f7f7"
            ],
            [
             0.6,
             "#e6f5d0"
            ],
            [
             0.7,
             "#b8e186"
            ],
            [
             0.8,
             "#7fbc41"
            ],
            [
             0.9,
             "#4d9221"
            ],
            [
             1,
             "#276419"
            ]
           ],
           "sequential": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ],
           "sequentialminus": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ]
          },
          "colorway": [
           "#636efa",
           "#EF553B",
           "#00cc96",
           "#ab63fa",
           "#FFA15A",
           "#19d3f3",
           "#FF6692",
           "#B6E880",
           "#FF97FF",
           "#FECB52"
          ],
          "font": {
           "color": "#2a3f5f"
          },
          "geo": {
           "bgcolor": "white",
           "lakecolor": "white",
           "landcolor": "#E5ECF6",
           "showlakes": true,
           "showland": true,
           "subunitcolor": "white"
          },
          "hoverlabel": {
           "align": "left"
          },
          "hovermode": "closest",
          "mapbox": {
           "style": "light"
          },
          "paper_bgcolor": "white",
          "plot_bgcolor": "#E5ECF6",
          "polar": {
           "angularaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "radialaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "scene": {
           "xaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "yaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "zaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           }
          },
          "shapedefaults": {
           "line": {
            "color": "#2a3f5f"
           }
          },
          "ternary": {
           "aaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "baxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "caxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "title": {
           "x": 0.05
          },
          "xaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          },
          "yaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          }
         }
        },
        "xaxis": {
         "anchor": "y",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "source"
         }
        },
        "yaxis": {
         "anchor": "x",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "count"
         }
        }
       }
      }
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# summarize the source column with a figure \n",
    "import plotly.express as px\n",
    "fig = px.histogram(df_cure, x=\"source\")\n",
    "fig.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "id": "475a3226",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "config": {
        "plotlyServerURL": "https://plot.ly"
       },
       "data": [
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "source=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "",
         "offsetgroup": "",
         "orientation": "v",
         "showlegend": false,
         "type": "histogram",
         "x": [
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "",
          "",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "",
          "google scholar unique",
          "",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "google scholar unique",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "google scholar unique",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy",
          "legacy"
         ],
         "xaxis": "x",
         "yaxis": "y"
        }
       ],
       "layout": {
        "barmode": "relative",
        "legend": {
         "tracegroupgap": 0
        },
        "margin": {
         "t": 60
        },
        "template": {
         "data": {
          "bar": [
           {
            "error_x": {
             "color": "#2a3f5f"
            },
            "error_y": {
             "color": "#2a3f5f"
            },
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "bar"
           }
          ],
          "barpolar": [
           {
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "barpolar"
           }
          ],
          "carpet": [
           {
            "aaxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "baxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "type": "carpet"
           }
          ],
          "choropleth": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "choropleth"
           }
          ],
          "contour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "contour"
           }
          ],
          "contourcarpet": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "contourcarpet"
           }
          ],
          "heatmap": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmap"
           }
          ],
          "heatmapgl": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmapgl"
           }
          ],
          "histogram": [
           {
            "marker": {
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "histogram"
           }
          ],
          "histogram2d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2d"
           }
          ],
          "histogram2dcontour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2dcontour"
           }
          ],
          "mesh3d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "mesh3d"
           }
          ],
          "parcoords": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "parcoords"
           }
          ],
          "pie": [
           {
            "automargin": true,
            "type": "pie"
           }
          ],
          "scatter": [
           {
            "fillpattern": {
             "fillmode": "overlay",
             "size": 10,
             "solidity": 0.2
            },
            "type": "scatter"
           }
          ],
          "scatter3d": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatter3d"
           }
          ],
          "scattercarpet": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattercarpet"
           }
          ],
          "scattergeo": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergeo"
           }
          ],
          "scattergl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergl"
           }
          ],
          "scattermapbox": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattermapbox"
           }
          ],
          "scatterpolar": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolar"
           }
          ],
          "scatterpolargl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolargl"
           }
          ],
          "scatterternary": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterternary"
           }
          ],
          "surface": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "surface"
           }
          ],
          "table": [
           {
            "cells": {
             "fill": {
              "color": "#EBF0F8"
             },
             "line": {
              "color": "white"
             }
            },
            "header": {
             "fill": {
              "color": "#C8D4E3"
             },
             "line": {
              "color": "white"
             }
            },
            "type": "table"
           }
          ]
         },
         "layout": {
          "annotationdefaults": {
           "arrowcolor": "#2a3f5f",
           "arrowhead": 0,
           "arrowwidth": 1
          },
          "autotypenumbers": "strict",
          "coloraxis": {
           "colorbar": {
            "outlinewidth": 0,
            "ticks": ""
           }
          },
          "colorscale": {
           "diverging": [
            [
             0,
             "#8e0152"
            ],
            [
             0.1,
             "#c51b7d"
            ],
            [
             0.2,
             "#de77ae"
            ],
            [
             0.3,
             "#f1b6da"
            ],
            [
             0.4,
             "#fde0ef"
            ],
            [
             0.5,
             "#f7f7f7"
            ],
            [
             0.6,
             "#e6f5d0"
            ],
            [
             0.7,
             "#b8e186"
            ],
            [
             0.8,
             "#7fbc41"
            ],
            [
             0.9,
             "#4d9221"
            ],
            [
             1,
             "#276419"
            ]
           ],
           "sequential": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ],
           "sequentialminus": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ]
          },
          "colorway": [
           "#636efa",
           "#EF553B",
           "#00cc96",
           "#ab63fa",
           "#FFA15A",
           "#19d3f3",
           "#FF6692",
           "#B6E880",
           "#FF97FF",
           "#FECB52"
          ],
          "font": {
           "color": "#2a3f5f"
          },
          "geo": {
           "bgcolor": "white",
           "lakecolor": "white",
           "landcolor": "#E5ECF6",
           "showlakes": true,
           "showland": true,
           "subunitcolor": "white"
          },
          "hoverlabel": {
           "align": "left"
          },
          "hovermode": "closest",
          "mapbox": {
           "style": "light"
          },
          "paper_bgcolor": "white",
          "plot_bgcolor": "#E5ECF6",
          "polar": {
           "angularaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "radialaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "scene": {
           "xaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "yaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "zaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           }
          },
          "shapedefaults": {
           "line": {
            "color": "#2a3f5f"
           }
          },
          "ternary": {
           "aaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "baxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "caxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "title": {
           "x": 0.05
          },
          "xaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          },
          "yaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          }
         }
        },
        "xaxis": {
         "anchor": "y",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "source"
         }
        },
        "yaxis": {
         "anchor": "x",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "count"
         }
        }
       }
      }
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# summarize the source column with a figure \n",
    "import plotly.express as px\n",
    "fig = px.histogram(df_cure, x=\"source\")\n",
    "fig.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "308a0318",
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "__new__() missing 2 required positional arguments: 'tag' and 'attributes'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "Input \u001b[0;32mIn [47]\u001b[0m, in \u001b[0;36m<cell line: 3>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[38;5;66;03m# plot based on year\u001b[39;00m\n\u001b[1;32m      2\u001b[0m \u001b[38;5;28;01mimport\u001b[39;00m \u001b[38;5;21;01mplotly\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mexpress\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m \u001b[38;5;21;01mpx\u001b[39;00m\n\u001b[0;32m----> 3\u001b[0m fig \u001b[38;5;241m=\u001b[39m \u001b[43mpx\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mhistogram\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdf_cure\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mx\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43myear\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[1;32m      4\u001b[0m fig\u001b[38;5;241m.\u001b[39mshow()\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/plotly/express/_chart_types.py:480\u001b[0m, in \u001b[0;36mhistogram\u001b[0;34m(data_frame, x, y, color, pattern_shape, facet_row, facet_col, facet_col_wrap, facet_row_spacing, facet_col_spacing, hover_name, hover_data, animation_frame, animation_group, category_orders, labels, color_discrete_sequence, color_discrete_map, pattern_shape_sequence, pattern_shape_map, marginal, opacity, orientation, barmode, barnorm, histnorm, log_x, log_y, range_x, range_y, histfunc, cumulative, nbins, text_auto, title, template, width, height)\u001b[0m\n\u001b[1;32m    434\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mhistogram\u001b[39m(\n\u001b[1;32m    435\u001b[0m     data_frame\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m,\n\u001b[1;32m    436\u001b[0m     x\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    472\u001b[0m     height\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m,\n\u001b[1;32m    473\u001b[0m ) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m go\u001b[38;5;241m.\u001b[39mFigure:\n\u001b[1;32m    474\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m    475\u001b[0m \u001b[38;5;124;03m    In a histogram, rows of `data_frame` are grouped together into a\u001b[39;00m\n\u001b[1;32m    476\u001b[0m \u001b[38;5;124;03m    rectangular mark to visualize the 1D distribution of an aggregate\u001b[39;00m\n\u001b[1;32m    477\u001b[0m \u001b[38;5;124;03m    function `histfunc` (e.g. the count or sum) of the value `y` (or `x` if\u001b[39;00m\n\u001b[1;32m    478\u001b[0m \u001b[38;5;124;03m    `orientation` is `'h'`).\u001b[39;00m\n\u001b[1;32m    479\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m--> 480\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mmake_figure\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    481\u001b[0m \u001b[43m        \u001b[49m\u001b[43margs\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mlocals\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    482\u001b[0m \u001b[43m        \u001b[49m\u001b[43mconstructor\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mgo\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mHistogram\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    483\u001b[0m \u001b[43m        \u001b[49m\u001b[43mtrace_patch\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mdict\u001b[39;49m\u001b[43m(\u001b[49m\n\u001b[1;32m    484\u001b[0m \u001b[43m            \u001b[49m\u001b[43mhistnorm\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mhistnorm\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    485\u001b[0m \u001b[43m            \u001b[49m\u001b[43mhistfunc\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mhistfunc\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    486\u001b[0m \u001b[43m            \u001b[49m\u001b[43mcumulative\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mdict\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43menabled\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mcumulative\u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    487\u001b[0m \u001b[43m        \u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    488\u001b[0m \u001b[43m        \u001b[49m\u001b[43mlayout_patch\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mdict\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43mbarmode\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbarmode\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mbarnorm\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbarnorm\u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    489\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/plotly/express/_core.py:2246\u001b[0m, in \u001b[0;36mmake_figure\u001b[0;34m(args, constructor, trace_patch, layout_patch)\u001b[0m\n\u001b[1;32m   2237\u001b[0m         _set_trace_grid_reference(\n\u001b[1;32m   2238\u001b[0m             trace,\n\u001b[1;32m   2239\u001b[0m             fig\u001b[38;5;241m.\u001b[39mlayout,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m   2242\u001b[0m             trace\u001b[38;5;241m.\u001b[39m_subplot_col,\n\u001b[1;32m   2243\u001b[0m         )\n\u001b[1;32m   2245\u001b[0m \u001b[38;5;66;03m# Add traces, layout and frames to figure\u001b[39;00m\n\u001b[0;32m-> 2246\u001b[0m \u001b[43mfig\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43madd_traces\u001b[49m\u001b[43m(\u001b[49m\u001b[43mframe_list\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m0\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mdata\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mif\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[38;5;28;43mlen\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43mframe_list\u001b[49m\u001b[43m)\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m>\u001b[39;49m\u001b[43m \u001b[49m\u001b[38;5;241;43m0\u001b[39;49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01melse\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43m[\u001b[49m\u001b[43m]\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   2247\u001b[0m fig\u001b[38;5;241m.\u001b[39mupdate_layout(layout_patch)\n\u001b[1;32m   2248\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtemplate\u001b[39m\u001b[38;5;124m\"\u001b[39m \u001b[38;5;129;01min\u001b[39;00m args \u001b[38;5;129;01mand\u001b[39;00m args[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtemplate\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/plotly/graph_objs/_figure.py:961\u001b[0m, in \u001b[0;36mFigure.add_traces\u001b[0;34m(self, data, rows, cols, secondary_ys, exclude_empty_subplots)\u001b[0m\n\u001b[1;32m    885\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21madd_traces\u001b[39m(\n\u001b[1;32m    886\u001b[0m     \u001b[38;5;28mself\u001b[39m,\n\u001b[1;32m    887\u001b[0m     data,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    891\u001b[0m     exclude_empty_subplots\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mFalse\u001b[39;00m,\n\u001b[1;32m    892\u001b[0m ) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFigure\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n\u001b[1;32m    893\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m    894\u001b[0m \n\u001b[1;32m    895\u001b[0m \u001b[38;5;124;03m    Add traces to the figure\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    959\u001b[0m \n\u001b[1;32m    960\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m--> 961\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43msuper\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43mFigure\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m)\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43madd_traces\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    962\u001b[0m \u001b[43m        \u001b[49m\u001b[43mdata\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mrows\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mcols\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43msecondary_ys\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mexclude_empty_subplots\u001b[49m\n\u001b[1;32m    963\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/plotly/basedatatypes.py:2181\u001b[0m, in \u001b[0;36mBaseFigure.add_traces\u001b[0;34m(self, data, rows, cols, secondary_ys, exclude_empty_subplots)\u001b[0m\n\u001b[1;32m   2113\u001b[0m \u001b[38;5;250m\u001b[39m\u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m   2114\u001b[0m \u001b[38;5;124;03mAdd traces to the figure\u001b[39;00m\n\u001b[1;32m   2115\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m   2177\u001b[0m \u001b[38;5;124;03mFigure(...)\u001b[39;00m\n\u001b[1;32m   2178\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m   2180\u001b[0m \u001b[38;5;66;03m# Validate traces\u001b[39;00m\n\u001b[0;32m-> 2181\u001b[0m data \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_data_validator\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mvalidate_coerce\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdata\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   2183\u001b[0m \u001b[38;5;66;03m# Set trace indexes\u001b[39;00m\n\u001b[1;32m   2184\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m ind, new_trace \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28menumerate\u001b[39m(data):\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/_plotly_utils/basevalidators.py:2645\u001b[0m, in \u001b[0;36mBaseDataValidator.validate_coerce\u001b[0;34m(self, v, skip_invalid, _validate)\u001b[0m\n\u001b[1;32m   2641\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m v_el \u001b[38;5;129;01min\u001b[39;00m v:\n\u001b[1;32m   2643\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(v_el, BaseTraceType):\n\u001b[1;32m   2644\u001b[0m         \u001b[38;5;66;03m# Clone input traces\u001b[39;00m\n\u001b[0;32m-> 2645\u001b[0m         v_el \u001b[38;5;241m=\u001b[39m \u001b[43mv_el\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mto_plotly_json\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   2647\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(v_el, \u001b[38;5;28mdict\u001b[39m):\n\u001b[1;32m   2648\u001b[0m         v_copy \u001b[38;5;241m=\u001b[39m deepcopy(v_el)\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/site-packages/plotly/basedatatypes.py:5590\u001b[0m, in \u001b[0;36mBasePlotlyType.to_plotly_json\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   5582\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mto_plotly_json\u001b[39m(\u001b[38;5;28mself\u001b[39m):\n\u001b[1;32m   5583\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m   5584\u001b[0m \u001b[38;5;124;03m    Return plotly JSON representation of object as a Python dict\u001b[39;00m\n\u001b[1;32m   5585\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m   5588\u001b[0m \u001b[38;5;124;03m    dict\u001b[39;00m\n\u001b[1;32m   5589\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m-> 5590\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_props\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mif\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_props\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;129;43;01mis\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[38;5;129;43;01mnot\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mNone\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[38;5;28;43;01melse\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43m{\u001b[49m\u001b[43m}\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    144\u001b[0m copier \u001b[38;5;241m=\u001b[39m _deepcopy_dispatch\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;28mcls\u001b[39m)\n\u001b[1;32m    145\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m copier \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[0;32m--> 146\u001b[0m     y \u001b[38;5;241m=\u001b[39m \u001b[43mcopier\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    147\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    148\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28missubclass\u001b[39m(\u001b[38;5;28mcls\u001b[39m, \u001b[38;5;28mtype\u001b[39m):\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/copy.py:229\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    227\u001b[0m memo[\u001b[38;5;28mid\u001b[39m(x)] \u001b[38;5;241m=\u001b[39m y\n\u001b[1;32m    228\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m key, value \u001b[38;5;129;01min\u001b[39;00m x\u001b[38;5;241m.\u001b[39mitems():\n\u001b[0;32m--> 229\u001b[0m     y[deepcopy(key, memo)] \u001b[38;5;241m=\u001b[39m \u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43mvalue\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    230\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m y\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/copy.py:153\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    151\u001b[0m copier \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mgetattr\u001b[39m(x, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m__deepcopy__\u001b[39m\u001b[38;5;124m\"\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m)\n\u001b[1;32m    152\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m copier \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[0;32m--> 153\u001b[0m     y \u001b[38;5;241m=\u001b[39m \u001b[43mcopier\u001b[49m\u001b[43m(\u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    154\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    155\u001b[0m     reductor \u001b[38;5;241m=\u001b[39m dispatch_table\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;28mcls\u001b[39m)\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    170\u001b[0m                 y \u001b[38;5;241m=\u001b[39m x\n\u001b[1;32m    171\u001b[0m             \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m--> 172\u001b[0m                 y \u001b[38;5;241m=\u001b[39m \u001b[43m_reconstruct\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mrv\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    174\u001b[0m \u001b[38;5;66;03m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[1;32m    175\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m y \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m x:\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/copy.py:263\u001b[0m, in \u001b[0;36m_reconstruct\u001b[0;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[1;32m    261\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m deep \u001b[38;5;129;01mand\u001b[39;00m args:\n\u001b[1;32m    262\u001b[0m     args \u001b[38;5;241m=\u001b[39m (deepcopy(arg, memo) \u001b[38;5;28;01mfor\u001b[39;00m arg \u001b[38;5;129;01min\u001b[39;00m args)\n\u001b[0;32m--> 263\u001b[0m y \u001b[38;5;241m=\u001b[39m \u001b[43mfunc\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    264\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m deep:\n\u001b[1;32m    265\u001b[0m     memo[\u001b[38;5;28mid\u001b[39m(x)] \u001b[38;5;241m=\u001b[39m y\n",
      "File \u001b[0;32m/opt/anaconda3/envs/trimmer_lab/lib/python3.8/copyreg.py:91\u001b[0m, in \u001b[0;36m__newobj__\u001b[0;34m(cls, *args)\u001b[0m\n\u001b[1;32m     90\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m__newobj__\u001b[39m(\u001b[38;5;28mcls\u001b[39m, \u001b[38;5;241m*\u001b[39margs):\n\u001b[0;32m---> 91\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mcls\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[38;5;21;43m__new__\u001b[39;49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mcls\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[0;31mTypeError\u001b[0m: __new__() missing 2 required positional arguments: 'tag' and 'attributes'"
     ]
    }
   ],
   "source": [
    "# plot based on year\n",
    "import plotly.express as px\n",
    "fig = px.histogram(df_cure, x=\"year\")\n",
    "fig.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "id": "6f562747",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "config": {
        "plotlyServerURL": "https://plot.ly"
       },
       "data": [
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "",
         "offsetgroup": "",
         "orientation": "v",
         "showlegend": false,
         "type": "histogram",
         "x": [
          "2012",
          "2018",
          "2017",
          "2017",
          "2011",
          "2012",
          "2018",
          "2023",
          "2023",
          "2023",
          "2023",
          "2017",
          "2013",
          "2013",
          "2013",
          "2013",
          "2014",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2021",
          "2021",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2015",
          "2015",
          "2015",
          "2015",
          "2010",
          "2010",
          "2010",
          "2010",
          "2012",
          "2012",
          "2012",
          "2017",
          "2017",
          "2017",
          "2010",
          "2010",
          "2010",
          "2010",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2020",
          "2020",
          "2020",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2011",
          "2011",
          "2011",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2020",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2010",
          "2010",
          "2010",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2021",
          "2021",
          "2021",
          "2010",
          "2015",
          "2015",
          "2015",
          "2008",
          "2008",
          "2008",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2020",
          "2015",
          "2019",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2012",
          "2022",
          "2022",
          "2022",
          "2019",
          "2022",
          "2017",
          "2023",
          "2021",
          "2016",
          "2018",
          "2012",
          "2012",
          "2012",
          "2021",
          "2012",
          "2017",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2022",
          "2012",
          "2009",
          "2010",
          "2020",
          "2019",
          "2019",
          "2019",
          "2019",
          "2022",
          "2016",
          "2015",
          "2022",
          "2015",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2022",
          "2012",
          "2012",
          "2021",
          "2020",
          "2020",
          "2020",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2011",
          "2011",
          "2011",
          "2016",
          "2019",
          "2016",
          "2014",
          "2014",
          "2014",
          "2018",
          "2017",
          "2013",
          "2016",
          "2016",
          "2016",
          "2017",
          "2017",
          "2017",
          "2016",
          "2019",
          "2014",
          "2020",
          "2018",
          "2017",
          "2018",
          "2020",
          "2017",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2012",
          "2013",
          "2021",
          "2019",
          "2017",
          "2014",
          "2020",
          "2021",
          "2021",
          "2021",
          "2020",
          "2015",
          "2023",
          "2023",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2010",
          "2010",
          "2010",
          "2021",
          "2016",
          "2019",
          "2014",
          "2013",
          "2021",
          "2021",
          "2021",
          "2023",
          "2022",
          "2020",
          "2020",
          "2016",
          "2021",
          "2021",
          "2019",
          "2020",
          "2017",
          "2017",
          "2017",
          "2015",
          "2017",
          "2020",
          "2016",
          "2019",
          "2023",
          "2019",
          "2018",
          "2016",
          "2023",
          "2018",
          "2017",
          "2023",
          "2017",
          "2021",
          "2019",
          "2020",
          "2012",
          "2020",
          "2021",
          "2016",
          "2015",
          "2018",
          "2012",
          "2019",
          "2023",
          "2023",
          "2020",
          "2023",
          "2020",
          "2019",
          "2022",
          "2017",
          "2020",
          "2018",
          "2018",
          "2018",
          "2019",
          "2020",
          "2018",
          "2020",
          "2016",
          "2022",
          "2017",
          "2017",
          "2019",
          "2023",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2014",
          "2014",
          "2014",
          "2019",
          "2021",
          "2021",
          "2021",
          "2021",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2010",
          "2010",
          "2010",
          "2021",
          "2010",
          "2013",
          "2022",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2016",
          "2016",
          "2016",
          "2016",
          "2011",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2023",
          "2008",
          "2019",
          "2019",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2013",
          "2013",
          "2013",
          "2013",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2023",
          "2014",
          "2014",
          "2014",
          "2022",
          "2011",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2012",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2012",
          "2012",
          "2012",
          "2012",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2015",
          "2015",
          "2016",
          "2017",
          "2015",
          "2022",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2018",
          "2018",
          "2018",
          "2010",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2022",
          "2016",
          "2011",
          "2011",
          "2011",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2017",
          "2013",
          "2010",
          "2010",
          "2010",
          "2020",
          "2012",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2022",
          "2022",
          "2022",
          "2006",
          "2014",
          "2021",
          "2023",
          "2023",
          "2023",
          "2023",
          "2009",
          "2017",
          "2011",
          "2021",
          "2013",
          "2012",
          "2022",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2016",
          "2016",
          "2016",
          "2013",
          "2022",
          "2022",
          "2022",
          "2022",
          "2020",
          "2017",
          "2017",
          "2017",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2018",
          "2018",
          "2014",
          "2014",
          "2014",
          "2013",
          "2023",
          "2023",
          "2021",
          "2021",
          "2021",
          "2021",
          "2020",
          "2019",
          "2017",
          "2017",
          "2015",
          "2016",
          "2016",
          "2016",
          "2015",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2016",
          "2020",
          "2021",
          "2013",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2017",
          "2017",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2015",
          "2019",
          "2019",
          "2019",
          "2015",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2011",
          "2013",
          "2013",
          "2013",
          "2013",
          "2020",
          "2020",
          "2020",
          "2019",
          "2017",
          "2017",
          "2017",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2021",
          "2023",
          "2023",
          "2023",
          "2023",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2010",
          "2010",
          "2010",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2014",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2018",
          "2020",
          "2020",
          "2020",
          "2020",
          "2018",
          "2018",
          "2018",
          "2019",
          "2019",
          "2019",
          "2023",
          "2013",
          "2013",
          "2013",
          "2013",
          "2023",
          "2022",
          "2020",
          "2013",
          "2013",
          "2013",
          "2013",
          "2017",
          "2015",
          "2015",
          "2015",
          "2016",
          "2014",
          "2014",
          "2014",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2009",
          "2011",
          "2023",
          "2013",
          "2013",
          "2013",
          "2012",
          "2012",
          "2012",
          "2017",
          "2017",
          "2017",
          "2018",
          "2010",
          "2010",
          "2010",
          "2013",
          "2013",
          "2013",
          "2013",
          "2011",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2016",
          "2016",
          "2016",
          "2016",
          "2023",
          "2009",
          "2021",
          "2011",
          "2013",
          "2019",
          "2019",
          "2019",
          "2015",
          "2012",
          "2014",
          "2014",
          "2014",
          "2011",
          "2018",
          "2018",
          "2018",
          "2018",
          "2013",
          "2013",
          "2011",
          "2016",
          "2016",
          "2016",
          "2018",
          "2015",
          "2015",
          "2015",
          "2015",
          "2021",
          "2021",
          "2021",
          "2021",
          "2019",
          "2016",
          "2017",
          "2019",
          "2019",
          "2020",
          "2014",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2022",
          "2016",
          "2023",
          "2019",
          "2017",
          "2018",
          "2023",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2018",
          "2013",
          "2013",
          "2013",
          "2013",
          "2015",
          "2016",
          "2020",
          "2020",
          "2013",
          "2020",
          "2020",
          "2020",
          "2015",
          "2015",
          "2015",
          "2023",
          "2016",
          "2023",
          "2021",
          "2021",
          "2021",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2012",
          "2012",
          "2012",
          "2012",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2021",
          "2018",
          "2018",
          "2018",
          "2018",
          "2016",
          "2021",
          "2021",
          "2021",
          "2021",
          "2016",
          "2016",
          "2016",
          "2012",
          "2021",
          "2015",
          "2013",
          "2013",
          "2013",
          "2013",
          "2015",
          "2015",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2016",
          "2016",
          "2016",
          "2022",
          "2017",
          "2021",
          "2021",
          "2020",
          "2022",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2017",
          "2017",
          "2017",
          "2017",
          "2018",
          "2018",
          "2018",
          "2021",
          "2010",
          "2017",
          "2016",
          "2016",
          "2016",
          "2013",
          "2020",
          "2020",
          "2019",
          "2017",
          "2013",
          "2013",
          "2013",
          "2020",
          "2017",
          "2017",
          "2017",
          "2017",
          "2021",
          "2012",
          "2020",
          "2017",
          "2017",
          "2017",
          "2018",
          "2013",
          "2016",
          "2016",
          "2016",
          "2017",
          "2017",
          "2017",
          "2016",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2019",
          "2021",
          "2020",
          "2011",
          "2019",
          "2014",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2017",
          "2017",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2024",
          "2024",
          "2017",
          "2017",
          "2017",
          "2014",
          "2022",
          "2022",
          "2022",
          "2022",
          "2023",
          "2019",
          "2019",
          "2019",
          "2022",
          "2022",
          "2015",
          "2015",
          "2015",
          "2013",
          "2012",
          "2012",
          "2012",
          "2022",
          "2022",
          "2022",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2015",
          "2021",
          "2021",
          "2021",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2018",
          "2018",
          "2018",
          "2019",
          "2019",
          "2021",
          "2021",
          "2021",
          "2023",
          "2019",
          "2017",
          "2017",
          "2017",
          "2017",
          "2023",
          "2023",
          "2023",
          "2019",
          "2014",
          "2014",
          "2014",
          "2022",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2014",
          "2014",
          "2014",
          "2021",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2013",
          "2018",
          "2018",
          "2018",
          "2021",
          "2021",
          "2021",
          "2016",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2019",
          "2019",
          "2015",
          "2015",
          "2015",
          "2014",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2022",
          "2022",
          "2012",
          "2013",
          "2013",
          "2013",
          "2013",
          "2018",
          "2022",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2023",
          "2023",
          "2012",
          "2019",
          "2019",
          "2017",
          "2023",
          "2021",
          "2022",
          "2022",
          "2016",
          "2015",
          "2015",
          "2015",
          "2015",
          "2020",
          "2020",
          "2020",
          "2020",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2014",
          "2022",
          "2022",
          "2021",
          "2014",
          "2013",
          "2013",
          "2013",
          "2013",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2022",
          "2020",
          "2012",
          "2012",
          "2012",
          "2012",
          "2023",
          "2018",
          "2018",
          "2018",
          "2018",
          "2017",
          "2017",
          "2017",
          "2014",
          "2019",
          "2015",
          "2015",
          "2015",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2015",
          "2015",
          "2023",
          "2014",
          "2014",
          "2014",
          "2021",
          "2013",
          "2016",
          "2012",
          "2015",
          "2022",
          "2017",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2023",
          "2023",
          "2015",
          "2020",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2023",
          "2019",
          "2016",
          "2016",
          "2016",
          "2023",
          "2023",
          "2023",
          "2023",
          "2018",
          "2018",
          "2018",
          "2021",
          "2021",
          "2021",
          "2021",
          "2014",
          "2017",
          "2011",
          "2022",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2014",
          "2014",
          "2014",
          "",
          "",
          "",
          "2019",
          "2014",
          "2023",
          "2023",
          "2023",
          "2023",
          "2016",
          "2010",
          "2010",
          "2010",
          "2016",
          "2009",
          "2012",
          "2012",
          "2012",
          "2015",
          "2015",
          "2015",
          "2023",
          "2022",
          "2022",
          "2022",
          "2011",
          "2010",
          "2014",
          "2014",
          "2020",
          "2020",
          "2020",
          "2018",
          "2018",
          "2018",
          "2018",
          "2014",
          "2014",
          "2013",
          "2013",
          "2013",
          "2013",
          "2010",
          "2021",
          "2021",
          "2017",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2017",
          "2019",
          "2019",
          "2020",
          "2017",
          "2023",
          "2021",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2014",
          "2015",
          "2015",
          "2014",
          "2014",
          "2023",
          "2023",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2013",
          "2013",
          "2013",
          "2014",
          "2014",
          "2014",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2018",
          "2018",
          "2018",
          "2018",
          "2016",
          "2016",
          "2016",
          "2011",
          "2011",
          "2011",
          "2015",
          "2015",
          "2015",
          "2007",
          "2007",
          "2007",
          "2007",
          "2010",
          "2010",
          "2010",
          "2012",
          "2012",
          "2012",
          "2012",
          "2007",
          "2007",
          "2007",
          "2012",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2017",
          "2017",
          "2017",
          "2007",
          "2007",
          "2007",
          "2007",
          "2007",
          "2018",
          "2018",
          "2016",
          "2014",
          "2015",
          "2019",
          "2019",
          "2019",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2014",
          "2014",
          "2014",
          "2014",
          "2023",
          "2023",
          "2023",
          "2018",
          "2016",
          "2016",
          "2016",
          "2018",
          "2018",
          "2018",
          "2018",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2023",
          "2023",
          "2023",
          "2020",
          "2016",
          "2016",
          "2016",
          "2017",
          "2017",
          "2017",
          "2023",
          "2023",
          "2023",
          "2022",
          "2022",
          "2019",
          "2019",
          "2019",
          "2015",
          "2019",
          "2008",
          "2008",
          "2008",
          "2014",
          "2019",
          "2015",
          "2022",
          "2022",
          "2017",
          "2017",
          "2016",
          "2014",
          "2014",
          "",
          "2017",
          "2017",
          "2013",
          "2010",
          "2010",
          "2010",
          "2019",
          "2022",
          "2016",
          "2016",
          "2016",
          "2016",
          "2022",
          "2017",
          "2017",
          "2017",
          "2017",
          "2016",
          "2016",
          "2016",
          "2013",
          "2009",
          "2016",
          "2016",
          "2016",
          "2021",
          "2017",
          "2016",
          "2014",
          "2011",
          "2011",
          "2011",
          "2013",
          "2017",
          "2017",
          "2017",
          "2011",
          "2011",
          "2011",
          "2022",
          "2022",
          "2022",
          "2018",
          "2023",
          "2012",
          "2020",
          "2010",
          "2010",
          "2013",
          "2013",
          "2013",
          "2013",
          "2015",
          "2015",
          "2015",
          "2014",
          "2014",
          "2014",
          "2017",
          "2016",
          "2014",
          "2015",
          "2015",
          "2021",
          "2019",
          "2008",
          "2010",
          "2016",
          "2011",
          "2011",
          "2011",
          "2019",
          "2009",
          "2018",
          "2022",
          "2020",
          "2013",
          "2012",
          "2014",
          "2007",
          "2010",
          "2020",
          "2019",
          "2020",
          "2018",
          "2017",
          "2023",
          "2017",
          "2016",
          "2019",
          "2018",
          "2015",
          "2021",
          "2012",
          "2018",
          "2007",
          "2007",
          "2007",
          "2020",
          "2018",
          "2012",
          "2012",
          "2012",
          "2012",
          "2017",
          "2012",
          "2012",
          "2012",
          "2015",
          "2015",
          "2011",
          "2011",
          "2011",
          "2017",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2015",
          "2016",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2015",
          "2010",
          "2022",
          "2022",
          "2022",
          "2019",
          "2016",
          "2020",
          "2020",
          "2020",
          "2009",
          "2009",
          "2009",
          "2022",
          "2022",
          "2022",
          "2009",
          "2009",
          "2009",
          "2012",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2022",
          "2022",
          "2022",
          "2019",
          "2014",
          "2014",
          "2013",
          "2017",
          "2017",
          "2021",
          "2021",
          "2021",
          "2021",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2021",
          "2022",
          "2016",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2017",
          "2018",
          "2021",
          "2019",
          "2020",
          "2021",
          "2014",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2015",
          "2015",
          "2015",
          "2015",
          "2022",
          "2012",
          "2015",
          "2015",
          "2015",
          "2013",
          "2013",
          "2013",
          "2013",
          "2022",
          "2023",
          "2022",
          "2018",
          "2022",
          "2021",
          "2016",
          "2023",
          "2021",
          "2018",
          "2020",
          "2014",
          "2013",
          "2013",
          "2013",
          "2018",
          "2013",
          "2013",
          "2013",
          "2019",
          "2013",
          "2019",
          "2019",
          "2019",
          "2019",
          "2022",
          "2018",
          "2018",
          "2018",
          "2012",
          "2011",
          "2013",
          "2022",
          "2022",
          "2012",
          "2022",
          "2012",
          "2023",
          "2018",
          "2016",
          "2016",
          "2016",
          "2016",
          "2022",
          "2020",
          "2018",
          "2022",
          "2012",
          "2012",
          "2012",
          "2014",
          "2012",
          "2021",
          "2021",
          "2021",
          "2014",
          "2014",
          "2014",
          "2017",
          "2015",
          "2015",
          "2015",
          "2021",
          "2021",
          "2015",
          "2012",
          "2018",
          "2018",
          "2018",
          "2022",
          "2022",
          "2022",
          "2019",
          "2019",
          "2019",
          "2017",
          "2017",
          "2017",
          "2017",
          "2023",
          "2023",
          "2023",
          "2012",
          "2011",
          "2011",
          "2011",
          "2011",
          "2015",
          "2015",
          "2015",
          "2015",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2016",
          "2013",
          "2013",
          "2013",
          "2015",
          "2015",
          "2015",
          "2015",
          "2009",
          "2009",
          "2009",
          "2007",
          "2007",
          "2007",
          "2007",
          "2007",
          "2016",
          "2016",
          "2016",
          "2013",
          "2018",
          "2022",
          "2022",
          "2022",
          "2015",
          "2020",
          "2023",
          "2018",
          "2018",
          "2018",
          "2023",
          "2018",
          "2018",
          "2018",
          "2014",
          "2019",
          "2023",
          "2022",
          "2021",
          "2023",
          "2014",
          "2014",
          "2014",
          "2011",
          "2023",
          "2023",
          "2023",
          "2018",
          "2018",
          "2018",
          "2021",
          "2021",
          "2021",
          "2021",
          "2015",
          "2020",
          "2022",
          "2021",
          "2017",
          "2014",
          "2017",
          "2018",
          "2011",
          "2011",
          "2011",
          "2016",
          "2012",
          "2012",
          "2012",
          "2012",
          "2020",
          "2020",
          "2021",
          "2018",
          "2022",
          "2021",
          "2013",
          "2018",
          "2019",
          "2017",
          "2019",
          "2014",
          "2013",
          "2013",
          "2014",
          "2023",
          "2008",
          "2012",
          "2014",
          "2019",
          "2010",
          "2009",
          "2009",
          "2009",
          "2008",
          "2012",
          "1996",
          "2023",
          "2020",
          "2013",
          "2016",
          "2023",
          "2022",
          "2022",
          "2015",
          "2017",
          "2017",
          "2017",
          "2015",
          "2015",
          "2015",
          "2015",
          "2018",
          "2018",
          "2018",
          "2018",
          "2013",
          "2019",
          "2008",
          "2008",
          "2008",
          "2008",
          "2019",
          "2015",
          "2011",
          "2019",
          "2015",
          "2016",
          "2010",
          "2021",
          "2019",
          "2021",
          "2011",
          "2017",
          "2017",
          "2009",
          "2009",
          "2009",
          "2016",
          "2016",
          "2016",
          "2010",
          "2010",
          "2010",
          "2021",
          "2012",
          "2012",
          "2009",
          "2013",
          "2010",
          "2021",
          "2017",
          "2017",
          "2017",
          "2011",
          "2011",
          "2020",
          "2020",
          "2013",
          "2016",
          "2016",
          "2016",
          "2021",
          "2021",
          "2021",
          "2015",
          "2020",
          "2022",
          "2018",
          "2014",
          "2019",
          "2022",
          "2022",
          "2020",
          "2017",
          "2022",
          "2020",
          "2022",
          "2022",
          "2023",
          "2023",
          "2023",
          "2021",
          "2021",
          "2021",
          "2016",
          "2016",
          "2016",
          "2020",
          "2021",
          "2014",
          "2020",
          "2013",
          "2013",
          "2020",
          "2021",
          "2018",
          "2019",
          "2011",
          "2016",
          "2010",
          "2023",
          "2015",
          "2010",
          "2013",
          "2015",
          "2015",
          "2015",
          "2015",
          "2012",
          "2015",
          "2019",
          "2020",
          "2015",
          "2015",
          "2021",
          "2018",
          "2021",
          "2015",
          "2012",
          "2022",
          "2022",
          "2008",
          "2018",
          "2015",
          "2018",
          "2018",
          "2015",
          "2015",
          "2014",
          "2008",
          "2020",
          "2016",
          "2012",
          "2012",
          "2012",
          "2018",
          "2013",
          "2015",
          "2011",
          "2011",
          "2011",
          "2011",
          "2010",
          "2017",
          "2022",
          "2022",
          "2022",
          "2021",
          "2011",
          "2014",
          "2012",
          "2014",
          "2017",
          "2021",
          "2010",
          "2010",
          "2010",
          "2010",
          "2017",
          "2008",
          "2008",
          "2008",
          "2008",
          "2013",
          "2013",
          "2013",
          "2007",
          "2007",
          "2007",
          "2007",
          "2022",
          "2022",
          "2022",
          "2022",
          "2023",
          "2007",
          "2007",
          "2007",
          "2022",
          "2008",
          "2008",
          "2008",
          "2010",
          "2009",
          "2015",
          "2021",
          "2021",
          "2021",
          "2022",
          "2022",
          "2022",
          "2010",
          "2010",
          "2010",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2019",
          "2017",
          "2022",
          "2022",
          "2022",
          "2015",
          "2010",
          "2014",
          "2017",
          "2021",
          "2010",
          "2012",
          "2010",
          "2009",
          "2012",
          "2023",
          "2012",
          "2014",
          "2014",
          "2022",
          "2015",
          "2009",
          "2014",
          "2012",
          "2019",
          "2012",
          "2016",
          "2013",
          "2021",
          "2013",
          "2013",
          "2008",
          "2010",
          "2015",
          "2018",
          "2014",
          "2014",
          "2010",
          "2017",
          "2009",
          "2023",
          "2020",
          "2016",
          "2022",
          "2019",
          "2009",
          "2021",
          "2017",
          "2022",
          "2016",
          "2018",
          "2018",
          "2019",
          "2011",
          "2022",
          "2022",
          "2017",
          "2017",
          "2023",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2010",
          "2017",
          "2012",
          "2021",
          "2017",
          "2019",
          "2022",
          "2022",
          "2014",
          "2014",
          "2014",
          "2017",
          "2017",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2020",
          "2023",
          "2012",
          "2021",
          "2017",
          "2016",
          "2016",
          "2012",
          "2021",
          "2018",
          "2018",
          "2015",
          "2012",
          "2022",
          "2019",
          "2010",
          "2014",
          "2018",
          "2020",
          "2009",
          "2011",
          "2019",
          "2023",
          "2022",
          "2022",
          "2022",
          "2019",
          "2013",
          "2013",
          "2013",
          "2016",
          "2017",
          "2016",
          "2014",
          "2019",
          "2023",
          "2017",
          "2013",
          "2018",
          "2018",
          "2018",
          "2018",
          "2020",
          "2020",
          "2020",
          "2022",
          "2017",
          "2015",
          "2015",
          "2015",
          "2017",
          "2015",
          "2009",
          "2020",
          "2014",
          "2022",
          "2016",
          "2017",
          "2020",
          "2019",
          "2019",
          "2020",
          "2014",
          "2021",
          "2014",
          "2019",
          "2015",
          "2022",
          "2022",
          "2012",
          "2012",
          "2012",
          "2022",
          "2022",
          "2021",
          "2015",
          "2017",
          "2019",
          "2020",
          "2020",
          "2017",
          "2018",
          "2021",
          "2020",
          "2019",
          "2019",
          "2019",
          "2018",
          "2023",
          "2023",
          "2023",
          "2022",
          "2019",
          "2018",
          "2023",
          "2020",
          "2021",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2018",
          "2017",
          "2020",
          "2015",
          "2021",
          "2023",
          "2019",
          "2018",
          "2022",
          "2021",
          "2014",
          "2014",
          "2011",
          "2012",
          "2022",
          "2013",
          "2018",
          "2016",
          "2023",
          "2017",
          "2008",
          "2016",
          "2015",
          "2016",
          "2010",
          "2013",
          "2015",
          "2014",
          "2020",
          "2021",
          "2021",
          "2020",
          "2019",
          "2017",
          "2015",
          "2012",
          "2018",
          "2012",
          "2011",
          "2011",
          "2009",
          "2016",
          "2014",
          "2010",
          "2015",
          "2020",
          "2020",
          "2022",
          "2020",
          "2018",
          "2021",
          "2010",
          "2008",
          "2018",
          "2015",
          "2015",
          "2010",
          "2010",
          "2022",
          "2013",
          "2013",
          "2016",
          "2012",
          "2016",
          "2021",
          "2021",
          "2021",
          "2021",
          "2011",
          "2015",
          "2015",
          "2015",
          "2015",
          "2013",
          "2013",
          "2012",
          "2012",
          "2012",
          "2012",
          "2021",
          "2010",
          "2019",
          "2019",
          "2010",
          "2019",
          "2023",
          "2009",
          "2010",
          "2014",
          "2012",
          "2021",
          "2013",
          "2014",
          "2014",
          "2014",
          "2014",
          "2013",
          "2022",
          "2023",
          "2023",
          "2017",
          "2023",
          "2010",
          "2010",
          "2010",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2021",
          "2021",
          "2021",
          "2011",
          "2020",
          "2023",
          "2017",
          "2017",
          "2020",
          "2015",
          "2010",
          "2012",
          "2012",
          "2012",
          "2023",
          "2011",
          "2023",
          "2019",
          "2022",
          "2013",
          "2021",
          "2019",
          "2019",
          "2008",
          "2018",
          "2015",
          "2015",
          "2014",
          "2019",
          "2023",
          "2021",
          "2023",
          "2018",
          "2019",
          "2019",
          "2019",
          "2016",
          "2022",
          "2015",
          "2009",
          "2015",
          "2017",
          "2018",
          "2013",
          "2014",
          "2020",
          "2012",
          "2012",
          "2012",
          "2012",
          "2016",
          "2019",
          "2019",
          "2015",
          "2015",
          "2019",
          "2022",
          "2012",
          "2017",
          "2017",
          "2007",
          "2015",
          "2023",
          "2020",
          "2012",
          "2019",
          "2023",
          "2023",
          "2011",
          "2017",
          "2021",
          "2011",
          "2018",
          "2013",
          "2017",
          "2012",
          "2022",
          "2013",
          "2012",
          "2015",
          "2015",
          "2014",
          "2017",
          "2019",
          "2024",
          "2015",
          "2012",
          "2019",
          "2019",
          "2019",
          "2011",
          "2016",
          "2015",
          "2018",
          "2020",
          "2020",
          "2020",
          "2021",
          "2015",
          "2016",
          "2022",
          "2023",
          "2014",
          "2019",
          "2023",
          "2012",
          "2016",
          "2016",
          "2016",
          "2021",
          "2017",
          "2017",
          "2011",
          "2013",
          "2021",
          "2018",
          "2015",
          "2017",
          "2022",
          "2020",
          "2023",
          "2022",
          "2013",
          "2021",
          "2021",
          "2021",
          "2021",
          "2014",
          "2014",
          "2012",
          "2014",
          "2013",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2021",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2009",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2013",
          "2006",
          "2006",
          "2006",
          "2020",
          "2020",
          "2020",
          "2020",
          "2015",
          "2010",
          "2010",
          "2010",
          "2017",
          "2020",
          "2020",
          "2020",
          "2011",
          "2011",
          "2011",
          "2018",
          "2011",
          "2012",
          "2015",
          "2014",
          "2012",
          "2013",
          "2013",
          "2013",
          "2013",
          "2011",
          "2010",
          "2008",
          "2008",
          "2008",
          "2006",
          "2023",
          "2019",
          "2019",
          "2015",
          "2015",
          "2015",
          "2007",
          "2019",
          "2019",
          "2019",
          "2014",
          "2008",
          "2008",
          "2008",
          "2007",
          "2022",
          "2008",
          "2020",
          "2013",
          "2013",
          "2013",
          "2017",
          "2016",
          "2015",
          "2016",
          "2015",
          "2019",
          "2019",
          "2019",
          "2018",
          "2013",
          "2015",
          "2018",
          "2010",
          "",
          "",
          "",
          "2014",
          "2021",
          "2022",
          "2019",
          "2018",
          "2017",
          "2008",
          "2020",
          "2015",
          "2018",
          "2018",
          "2014",
          "2021",
          "2021",
          "2013",
          "2022",
          "2017",
          "2022",
          "2022",
          "2010",
          "2012",
          "2012",
          "2013",
          "2021",
          "2016",
          "2012",
          "2014",
          "2017",
          "2021",
          "2020",
          "2013",
          "2013",
          "2023",
          "2010",
          "2018",
          "2012",
          "2021",
          "2010",
          "2010",
          "2012",
          "2016",
          "2012",
          "2008",
          "2011",
          "2016",
          "2017",
          "1997",
          "2014",
          "2016",
          "2015",
          "2014",
          "2021",
          "2013",
          "2014",
          "2013",
          "2016",
          "2014",
          "2019",
          "2020",
          "2016",
          "2015",
          "2017",
          "2017",
          "2021",
          "2016",
          "2016",
          "2015",
          "2016",
          "2017",
          "2016",
          "2023",
          "2012",
          "2012",
          "2019",
          "2020",
          "2022",
          "2015",
          "2017",
          "2015",
          "2016",
          "2016",
          "2011",
          "2012",
          "2014",
          "2017",
          "2020",
          "2021",
          "2019",
          "2018",
          "2014",
          "2018",
          "2020",
          "2023",
          "2023",
          "2023",
          "2013",
          "2011",
          "2019",
          "2019",
          "2018",
          "2011",
          "2011",
          "2009",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2019",
          "2010",
          "2016",
          "2015",
          "2016",
          "2020",
          "2020",
          "2020",
          "2010",
          "2010",
          "2010",
          "2022",
          "2014",
          "2018",
          "2012",
          "2012",
          "2020",
          "2020",
          "2020",
          "2011",
          "2011",
          "2018",
          "2021",
          "2016",
          "2017",
          "2020",
          "2014",
          "2012",
          "2012",
          "2014",
          "2019",
          "2016",
          "2017",
          "2016",
          "2021",
          "2020",
          "2012",
          "2015",
          "2018",
          "2019",
          "2013",
          "2020",
          "2016",
          "2021",
          "2013",
          "2021",
          "2020",
          "2012",
          "2014",
          "2016",
          "2015",
          "2014",
          "2014",
          "2013",
          "2015",
          "2016",
          "2022",
          "2015",
          "2021",
          "2020",
          "2009",
          "2013",
          "2021",
          "2016",
          "2020",
          "2020",
          "2020",
          "2023",
          "2017",
          "2014",
          "2015",
          "2012",
          "2012",
          "2008",
          "2023",
          "2019",
          "2021",
          "2021",
          "2019",
          "2015",
          "2023",
          "2019",
          "2018",
          "2021",
          "2008",
          "2015",
          "2015",
          "2015",
          "2020",
          "2008",
          "2022",
          "2011",
          "2010",
          "2014",
          "2011",
          "2014",
          "2014",
          "2017",
          "2016",
          "2012",
          "2023",
          "2017",
          "2020",
          "2020",
          "2008",
          "2013",
          "2015",
          "2017",
          "2014",
          "2022",
          "2022",
          "2020",
          "2013",
          "2013",
          "2015",
          "2017",
          "2014",
          "2006",
          "2016",
          "2007",
          "2007",
          "2013",
          "2013",
          "2013",
          "2022",
          "2019",
          "2011",
          "2008",
          "2022",
          "2012",
          "2012",
          "2018",
          "2017",
          "2019",
          "2023",
          "2023",
          "2023",
          "2024",
          "2023",
          "2023",
          "2023",
          "2023",
          "2024",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2017",
          "2024",
          "2023",
          "2023",
          "2023",
          "2024",
          "2023",
          "2024",
          "2023",
          "2023",
          "2024",
          "2023",
          "2023",
          "2006",
          "2006",
          "2006",
          "2006",
          "2006",
          "2007",
          "2006",
          "2006",
          "2006",
          "2006",
          "2007",
          "2007",
          "2007",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2009",
          "2008",
          "2008",
          "2009",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2009",
          "2009",
          "2008",
          "2008",
          "2009",
          "2008",
          "2008",
          "2008",
          "2009",
          "2008",
          "2009",
          "2009",
          "2009",
          "2008",
          "2008",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2010",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2010",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2010",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2010",
          "2009",
          "2009",
          "2010",
          "2010",
          "2010",
          "2009",
          "2009",
          "2009",
          "2010",
          "2010",
          "2011",
          "2009",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2009",
          "2009",
          "2009",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2011",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2011",
          "2011",
          "2010",
          "2010",
          "2012",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2011",
          "2010",
          "2011",
          "2010",
          "2010",
          "2010",
          "2011",
          "2011",
          "2010",
          "2010",
          "2011",
          "2010",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2012",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2012",
          "2012",
          "2011",
          "2011",
          "2012",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2012",
          "2011",
          "2012",
          "2012",
          "2012",
          "2011",
          "2011",
          "2011",
          "2011",
          "2012",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2012",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2012",
          "2011",
          "2011",
          "2012",
          "2011",
          "2012",
          "2011",
          "2011",
          "2011",
          "2012",
          "2011",
          "2012",
          "2012",
          "2011",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2011",
          "2012",
          "2011",
          "2012",
          "2011",
          "2011",
          "2012",
          "2011",
          "2012",
          "2012",
          "2012",
          "2011",
          "2012",
          "2012",
          "2012",
          "2011",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2011",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2012",
          "2011",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2013",
          "2012",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2012",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2012",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2014",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2014",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2014",
          "2013",
          "2013",
          "2014",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2014",
          "2013",
          "2013",
          "2013",
          "2013",
          "2014",
          "2013",
          "2013",
          "2013",
          "2014",
          "2013",
          "2013",
          "2014",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2014",
          "2015",
          "2013",
          "2014",
          "2014",
          "2013",
          "2014",
          "2014",
          "2013",
          "2013",
          "2014",
          "2013",
          "2013",
          "2014",
          "2014",
          "2014",
          "2014",
          "2013",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2013",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2014",
          "2014",
          "2014",
          "2015",
          "2015",
          "2014",
          "2014",
          "2014",
          "2015",
          "2015",
          "2014",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2014",
          "2014",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2015",
          "2014",
          "2014",
          "2015",
          "2014",
          "2015",
          "2014",
          "2015",
          "2015",
          "2015",
          "2014",
          "2015",
          "2015",
          "2014",
          "2015",
          "2015",
          "2014",
          "2015",
          "2014",
          "2014",
          "2015",
          "2015",
          "2015",
          "2014",
          "2015",
          "2015",
          "2015",
          "2015",
          "2014",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2014",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2014",
          "2015",
          "2015",
          "2016",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2016",
          "2015",
          "2015",
          "2015",
          "2016",
          "2015",
          "2014",
          "2015",
          "2016",
          "2015",
          "2015",
          "2015",
          "2016",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2016",
          "2015",
          "2016",
          "2015",
          "2015",
          "2016",
          "2016",
          "2017",
          "2016",
          "2015",
          "2016",
          "2016",
          "2016",
          "2015",
          "2016",
          "2015",
          "2016",
          "2015",
          "2016",
          "2015",
          "2015",
          "2016",
          "2015",
          "2016",
          "2015",
          "2016",
          "2016",
          "2016",
          "2016",
          "2015",
          "2016",
          "2016",
          "2016",
          "2016",
          "2015",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2017",
          "2016",
          "2017",
          "2016",
          "2017",
          "2016",
          "2017",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2017",
          "2017",
          "2016",
          "2017",
          "2017",
          "2017",
          "2016",
          "2016",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2016",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2018",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2018",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2018",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2018",
          "2017",
          "2017",
          "2018",
          "2017",
          "2017",
          "2018",
          "2018",
          "2019",
          "2017",
          "2017",
          "2018",
          "2018",
          "2018",
          "2017",
          "2018",
          "2017",
          "2017",
          "2018",
          "2018",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2017",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2016",
          "2018",
          "2018",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2018",
          "2018",
          "2019",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2019",
          "2018",
          "2018",
          "2018",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2019",
          "2019",
          "2019",
          "2018",
          "2019",
          "2019",
          "2020",
          "2018",
          "2019",
          "2018",
          "2018",
          "2019",
          "2019",
          "2018",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2020",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2018",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2020",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2020",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2020",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2020",
          "2020",
          "2019",
          "2019",
          "2019",
          "2019",
          "2020",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2020",
          "2020",
          "2019",
          "2020",
          "2020",
          "2019",
          "2020",
          "2019",
          "2019",
          "2020",
          "2020",
          "2019",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2019",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2020",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2020",
          "2020",
          "2021",
          "2020",
          "2021",
          "2020",
          "2020",
          "2022",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2021",
          "2020",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2021",
          "2021",
          "2020",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2020",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2020",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2022",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2022",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2022",
          "2022",
          "2022",
          "2021",
          "2021",
          "2022",
          "2022",
          "2022",
          "2022",
          "2021",
          "2022",
          "2022",
          "2021",
          "2021",
          "2021",
          "2022",
          "2021",
          "2022",
          "2020",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2023",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2023",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2023",
          "2022",
          "2022",
          "2023",
          "2022",
          "2022",
          "2023",
          "2022",
          "2022",
          "2022",
          "2023",
          "2022",
          "2023",
          "2022",
          "2023",
          "2023",
          "2023",
          "2022",
          "2022",
          "2022",
          "2023",
          "2022",
          "2023",
          "2022",
          "2023",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2022",
          "2023",
          "2022",
          "2022",
          "2023",
          "2023",
          "2023",
          "2023",
          "2022",
          "2023",
          "2022",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2024",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2023",
          "2024",
          "2023",
          "2023",
          "2023",
          "2023",
          "2024",
          "2023",
          "2006",
          "2006",
          "2006",
          "2007",
          "2007",
          "2007",
          "2007",
          "2007",
          "2007",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2008",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2009",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2010",
          "2011",
          "2011",
          "2011",
          "2012",
          "2011",
          "2011",
          "2012",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2012",
          "2011",
          "2012",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "2011",
          "1975",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2012",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2013",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2014",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2015",
          "2016",
          "2015",
          "2015",
          "2015",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2016",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2016",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2018",
          "2018",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2017",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2018",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2019",
          "2020",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2020",
          "2021",
          "2020",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021",
          "2021"
         ],
         "xaxis": "x",
         "yaxis": "y"
        }
       ],
       "layout": {
        "barmode": "relative",
        "legend": {
         "tracegroupgap": 0
        },
        "margin": {
         "t": 60
        },
        "template": {
         "data": {
          "bar": [
           {
            "error_x": {
             "color": "#2a3f5f"
            },
            "error_y": {
             "color": "#2a3f5f"
            },
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "bar"
           }
          ],
          "barpolar": [
           {
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "barpolar"
           }
          ],
          "carpet": [
           {
            "aaxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "baxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "type": "carpet"
           }
          ],
          "choropleth": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "choropleth"
           }
          ],
          "contour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "contour"
           }
          ],
          "contourcarpet": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "contourcarpet"
           }
          ],
          "heatmap": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmap"
           }
          ],
          "heatmapgl": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmapgl"
           }
          ],
          "histogram": [
           {
            "marker": {
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "histogram"
           }
          ],
          "histogram2d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2d"
           }
          ],
          "histogram2dcontour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2dcontour"
           }
          ],
          "mesh3d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "mesh3d"
           }
          ],
          "parcoords": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "parcoords"
           }
          ],
          "pie": [
           {
            "automargin": true,
            "type": "pie"
           }
          ],
          "scatter": [
           {
            "fillpattern": {
             "fillmode": "overlay",
             "size": 10,
             "solidity": 0.2
            },
            "type": "scatter"
           }
          ],
          "scatter3d": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatter3d"
           }
          ],
          "scattercarpet": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattercarpet"
           }
          ],
          "scattergeo": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergeo"
           }
          ],
          "scattergl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergl"
           }
          ],
          "scattermapbox": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattermapbox"
           }
          ],
          "scatterpolar": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolar"
           }
          ],
          "scatterpolargl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolargl"
           }
          ],
          "scatterternary": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterternary"
           }
          ],
          "surface": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "surface"
           }
          ],
          "table": [
           {
            "cells": {
             "fill": {
              "color": "#EBF0F8"
             },
             "line": {
              "color": "white"
             }
            },
            "header": {
             "fill": {
              "color": "#C8D4E3"
             },
             "line": {
              "color": "white"
             }
            },
            "type": "table"
           }
          ]
         },
         "layout": {
          "annotationdefaults": {
           "arrowcolor": "#2a3f5f",
           "arrowhead": 0,
           "arrowwidth": 1
          },
          "autotypenumbers": "strict",
          "coloraxis": {
           "colorbar": {
            "outlinewidth": 0,
            "ticks": ""
           }
          },
          "colorscale": {
           "diverging": [
            [
             0,
             "#8e0152"
            ],
            [
             0.1,
             "#c51b7d"
            ],
            [
             0.2,
             "#de77ae"
            ],
            [
             0.3,
             "#f1b6da"
            ],
            [
             0.4,
             "#fde0ef"
            ],
            [
             0.5,
             "#f7f7f7"
            ],
            [
             0.6,
             "#e6f5d0"
            ],
            [
             0.7,
             "#b8e186"
            ],
            [
             0.8,
             "#7fbc41"
            ],
            [
             0.9,
             "#4d9221"
            ],
            [
             1,
             "#276419"
            ]
           ],
           "sequential": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ],
           "sequentialminus": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ]
          },
          "colorway": [
           "#636efa",
           "#EF553B",
           "#00cc96",
           "#ab63fa",
           "#FFA15A",
           "#19d3f3",
           "#FF6692",
           "#B6E880",
           "#FF97FF",
           "#FECB52"
          ],
          "font": {
           "color": "#2a3f5f"
          },
          "geo": {
           "bgcolor": "white",
           "lakecolor": "white",
           "landcolor": "#E5ECF6",
           "showlakes": true,
           "showland": true,
           "subunitcolor": "white"
          },
          "hoverlabel": {
           "align": "left"
          },
          "hovermode": "closest",
          "mapbox": {
           "style": "light"
          },
          "paper_bgcolor": "white",
          "plot_bgcolor": "#E5ECF6",
          "polar": {
           "angularaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "radialaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "scene": {
           "xaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "yaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "zaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           }
          },
          "shapedefaults": {
           "line": {
            "color": "#2a3f5f"
           }
          },
          "ternary": {
           "aaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "baxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "caxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "title": {
           "x": 0.05
          },
          "xaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          },
          "yaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          }
         }
        },
        "xaxis": {
         "anchor": "y",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "year"
         }
        },
        "yaxis": {
         "anchor": "x",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "count"
         }
        }
       }
      }
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# plot based on year\n",
    "import plotly.express as px\n",
    "fig = px.histogram(df_cure, x=\"year\")\n",
    "fig.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "cfc9aca5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>MAB_ID</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>citation</th>\n",
       "      <th>title</th>\n",
       "      <th>year</th>\n",
       "      <th>source</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1560</th>\n",
       "      <td>[N95/35]</td>\n",
       "      <td>[GABA(A)R, Alpha1]</td>\n",
       "      <td>28320946</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1561</th>\n",
       "      <td>[N87/25]</td>\n",
       "      <td>[GABA(A)R, Beta3]</td>\n",
       "      <td>28320946</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1562</th>\n",
       "      <td>[N95/35,  N87/25]</td>\n",
       "      <td>[GABA(A)R, Alpha1, GABA(A)R, Beta3]</td>\n",
       "      <td>28320946</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1909</th>\n",
       "      <td>[K72/8]</td>\n",
       "      <td>[Co-Rest/RCOR1]</td>\n",
       "      <td>26186538</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1910</th>\n",
       "      <td>[K72/8]</td>\n",
       "      <td>[Co-Rest/RCOR1]</td>\n",
       "      <td>26186538</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2246</th>\n",
       "      <td>[L21/32]</td>\n",
       "      <td>[GluA2/GluR2 glutamate receptor]</td>\n",
       "      <td>31816021</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2247</th>\n",
       "      <td>[N59/36]</td>\n",
       "      <td>[GluN2B/NR2B glutamate receptor]</td>\n",
       "      <td>31816021</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2248</th>\n",
       "      <td>[N59/36,  L21/32]</td>\n",
       "      <td>[GluA2/GluR2 glutamate receptor, GluN2B/NR2B g...</td>\n",
       "      <td>31816021</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                 MAB_ID                                             Target  \\\n",
       "1560           [N95/35]                                 [GABA(A)R, Alpha1]   \n",
       "1561           [N87/25]                                  [GABA(A)R, Beta3]   \n",
       "1562  [N95/35,  N87/25]                [GABA(A)R, Alpha1, GABA(A)R, Beta3]   \n",
       "1909            [K72/8]                                    [Co-Rest/RCOR1]   \n",
       "1910            [K72/8]                                    [Co-Rest/RCOR1]   \n",
       "2246           [L21/32]                   [GluA2/GluR2 glutamate receptor]   \n",
       "2247           [N59/36]                   [GluN2B/NR2B glutamate receptor]   \n",
       "2248  [N59/36,  L21/32]  [GluA2/GluR2 glutamate receptor, GluN2B/NR2B g...   \n",
       "\n",
       "          PMID citation title year                 source  \n",
       "1560  28320946                      google scholar unique  \n",
       "1561  28320946                      google scholar unique  \n",
       "1562  28320946                      google scholar unique  \n",
       "1909  26186538                      google scholar unique  \n",
       "1910  26186538                      google scholar unique  \n",
       "2246  31816021                      google scholar unique  \n",
       "2247  31816021                      google scholar unique  \n",
       "2248  31816021                      google scholar unique  "
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# filter year that is ''\n",
    "df_cure[df_cure['year'] == '']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "de7295af",
   "metadata": {},
   "outputs": [],
   "source": [
    "# output df_cure\n",
    "# convert to int\n",
    "# fill empty year with 0\n",
    "df_cure['year'] = df_cure['year'].replace('', 0)\n",
    "df_cure['year'] = df_cure['year'].astype(int)\n",
    "df_cure = df_cure[df_cure['year'] >= 2000]\n",
    "\n",
    "df_cure.to_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/df_cure-5-20-24.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "07360ddf",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>MAB_ID</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>citation</th>\n",
       "      <th>title</th>\n",
       "      <th>year</th>\n",
       "      <th>source</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>['N37A/10']</td>\n",
       "      <td>['Kv7.1/KCNQ1 K+ channel']</td>\n",
       "      <td>22024150</td>\n",
       "      <td>Krzystanek K et al. 2012 Heart Rhythm 9 440-8.</td>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>2012</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>['N70/28']</td>\n",
       "      <td>['HCN1']</td>\n",
       "      <td>29427087</td>\n",
       "      <td>Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...</td>\n",
       "      <td>Neuroinflammation Alters Integrative Propertie...</td>\n",
       "      <td>2018</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>['N212/17']</td>\n",
       "      <td>['TRIP8b (constant)']</td>\n",
       "      <td>28887304</td>\n",
       "      <td>Lyman KA et al. 2017 J Biol Chem 292 17718-17730.</td>\n",
       "      <td>Allostery between two binding sites in the ion...</td>\n",
       "      <td>2017</td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>['N212/17']</td>\n",
       "      <td>['TRIP8b (constant)']</td>\n",
       "      <td>28127990</td>\n",
       "      <td>Lyman KA et al. 2017 Expert Opin Ther Targets ...</td>\n",
       "      <td>Animal models suggest the TRIP8b-HCN interacti...</td>\n",
       "      <td>2017</td>\n",
       "      <td>google scholar unique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>['N144/14']</td>\n",
       "      <td>['6xHis']</td>\n",
       "      <td>22184198</td>\n",
       "      <td>Emond MR et al. 2011 J Cell Biol 195 1115-21.</td>\n",
       "      <td>A complex of Protocadherin-19 and N-cadherin m...</td>\n",
       "      <td>2011</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6243</th>\n",
       "      <td>{'N106/65'}</td>\n",
       "      <td>{'Ankyrin-G'}</td>\n",
       "      <td>33536249</td>\n",
       "      <td>Ogawa Y et al. 2021 J Cell Sci 134.</td>\n",
       "      <td>Endogenously expressed Ranbp2 is not at the ax...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6244</th>\n",
       "      <td>{'L6/60'}</td>\n",
       "      <td>{'Slo1/BKAlpha'}</td>\n",
       "      <td>33556372</td>\n",
       "      <td>Bukiya AN et al. 2021 J Biol Chem 296 100381.</td>\n",
       "      <td>Cholesterol activates BK channels by increasin...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6245</th>\n",
       "      <td>{'N212B'}</td>\n",
       "      <td>{'TRIP8b'}</td>\n",
       "      <td>33547341</td>\n",
       "      <td>Rátkai A et al. 2021 Sci Rep 11 3236.</td>\n",
       "      <td>Homeostatic plasticity and burst activity are ...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6246</th>\n",
       "      <td>{'K28/43'}</td>\n",
       "      <td>{'PSD-95'}</td>\n",
       "      <td>33663561</td>\n",
       "      <td>Perkins EM et al. 2021 Mol Neurodegener 16 13.</td>\n",
       "      <td>Altered network properties in C9ORF72 repeat e...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6247</th>\n",
       "      <td>{'K28/43'}</td>\n",
       "      <td>{'PSD-95'}</td>\n",
       "      <td>33615226</td>\n",
       "      <td>Gavrilovici C et al. 2021 Cereb Cortex Commun ...</td>\n",
       "      <td>Behavioral Deficits in Mice with Postnatal Dis...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>6248 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           MAB_ID                      Target      PMID  \\\n",
       "0     ['N37A/10']  ['Kv7.1/KCNQ1 K+ channel']  22024150   \n",
       "1      ['N70/28']                    ['HCN1']  29427087   \n",
       "2     ['N212/17']       ['TRIP8b (constant)']  28887304   \n",
       "3     ['N212/17']       ['TRIP8b (constant)']  28127990   \n",
       "4     ['N144/14']                   ['6xHis']  22184198   \n",
       "...           ...                         ...       ...   \n",
       "6243  {'N106/65'}               {'Ankyrin-G'}  33536249   \n",
       "6244    {'L6/60'}            {'Slo1/BKAlpha'}  33556372   \n",
       "6245    {'N212B'}                  {'TRIP8b'}  33547341   \n",
       "6246   {'K28/43'}                  {'PSD-95'}  33663561   \n",
       "6247   {'K28/43'}                  {'PSD-95'}  33615226   \n",
       "\n",
       "                                               citation  \\\n",
       "0        Krzystanek K et al. 2012 Heart Rhythm 9 440-8.   \n",
       "1     Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...   \n",
       "2     Lyman KA et al. 2017 J Biol Chem 292 17718-17730.   \n",
       "3     Lyman KA et al. 2017 Expert Opin Ther Targets ...   \n",
       "4         Emond MR et al. 2011 J Cell Biol 195 1115-21.   \n",
       "...                                                 ...   \n",
       "6243                Ogawa Y et al. 2021 J Cell Sci 134.   \n",
       "6244      Bukiya AN et al. 2021 J Biol Chem 296 100381.   \n",
       "6245              Rátkai A et al. 2021 Sci Rep 11 3236.   \n",
       "6246     Perkins EM et al. 2021 Mol Neurodegener 16 13.   \n",
       "6247  Gavrilovici C et al. 2021 Cereb Cortex Commun ...   \n",
       "\n",
       "                                                  title  year  \\\n",
       "0     Deubiquitylating enzyme USP2 counteracts Nedd4...  2012   \n",
       "1     Neuroinflammation Alters Integrative Propertie...  2018   \n",
       "2     Allostery between two binding sites in the ion...  2017   \n",
       "3     Animal models suggest the TRIP8b-HCN interacti...  2017   \n",
       "4     A complex of Protocadherin-19 and N-cadherin m...  2011   \n",
       "...                                                 ...   ...   \n",
       "6243  Endogenously expressed Ranbp2 is not at the ax...  2021   \n",
       "6244  Cholesterol activates BK channels by increasin...  2021   \n",
       "6245  Homeostatic plasticity and burst activity are ...  2021   \n",
       "6246  Altered network properties in C9ORF72 repeat e...  2021   \n",
       "6247  Behavioral Deficits in Mice with Postnatal Dis...  2021   \n",
       "\n",
       "                     source  \n",
       "0                       NaN  \n",
       "1                       NaN  \n",
       "2     google scholar unique  \n",
       "3     google scholar unique  \n",
       "4                       NaN  \n",
       "...                     ...  \n",
       "6243                 legacy  \n",
       "6244                 legacy  \n",
       "6245                 legacy  \n",
       "6246                 legacy  \n",
       "6247                 legacy  \n",
       "\n",
       "[6248 rows x 7 columns]"
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# expand out the list columns into multiple rows (do MAB_ID)\n",
    "df_cure = pd.read_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/df_cure-5-20-24.csv\")\n",
    "# turn the string columns into lists after getting rid of the {} for MAB_ID \n",
    "# some are split on or as well fix that by replacing or with a comma space\n",
    "# df_cure['MAB_ID'] = df_cure['MAB_ID'].str.replace('{', '').str.replace('}', '').str.replace(\"'\", '').str.replace(' or ', ', ').str.split(', ')\n",
    "# also reaplce the [ in the \n",
    "# replace the html stuff in the title that <>\n",
    "df_cure['title'] = df_cure['title'].str.replace('<i>', '').str.replace('</i>', '').str.replace('<b>', '').str.replace('</b>', '')\n",
    "df_cure['title'] = df_cure['title'].str.replace('<sub>', '').str.replace('</sub>', '').str.replace('<sup>', '').str.replace('</sup>', '')\n",
    "df_cure['title'] = df_cure['title'].str.replace('<u>', '').str.replace('</u>', '')\n",
    "# output a df_cure_cleaaned\n",
    "# turn year into an int\n",
    "#df_cure['year'] = df_cure['year'].astype(int)\n",
    "df_cure.to_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/df_cure_cleaned-5-20-24.csv\", index=False)\n",
    "df_cure = df_cure.explode('MAB_ID')\n",
    "df_cure"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "b83a58ab",
   "metadata": {},
   "outputs": [],
   "source": [
    "# create a new column that is the citation_final\n",
    "# make it blank for now\n",
    "df_cure['citation_final'] = ''\n",
    "# iterate through each row get the pubmed id and grab the corresponding json and get a citation in the following format Anderson, C.N.G., et al. 2006. J Physiol. 575:367-72.\n",
    "for index, row in df_cure.iterrows():\n",
    "    # look for -f with a number after it with regex\n",
    "    # get the list of clones in the df_final[\"MAB_ID\"]\n",
    "    pmid = row[\"PMID\"]\n",
    "    record = fetch_details(pmid)\n",
    "    # set the citation_final column to the citation value\n",
    "    if type(record) == dict and 'citation' in record and 'title' in record and 'year' in record:\n",
    "        df_cure.at[index, 'citation_final'] = record['citation']\n",
    "    else:\n",
    "        print(f\"For {pmid}, no record was found.\")\n",
    "    # # get the first author et al. \n",
    "    # # check if the record is a dictionary with the relevant keys\n",
    "    # if type(record) == dict and 'citation' in record and 'title' in record and 'year' in record:\n",
    "    #     df_cure.at[index, 'citation_final'] = record['citation']\n",
    "    # else:\n",
    "    #     print(f\"For {pmid}, no record was found.\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "ac7c0e3a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>MAB_ID</th>\n",
       "      <th>Target</th>\n",
       "      <th>PMID</th>\n",
       "      <th>citation</th>\n",
       "      <th>title</th>\n",
       "      <th>year</th>\n",
       "      <th>source</th>\n",
       "      <th>citation_final</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>['N37A/10']</td>\n",
       "      <td>['Kv7.1/KCNQ1 K+ channel']</td>\n",
       "      <td>22024150</td>\n",
       "      <td>Krzystanek K et al. 2012 Heart Rhythm 9 440-8.</td>\n",
       "      <td>Deubiquitylating enzyme USP2 counteracts Nedd4...</td>\n",
       "      <td>2012</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Krzystanek K et al. 2012 Heart Rhythm 9 440-8.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>['N70/28']</td>\n",
       "      <td>['HCN1']</td>\n",
       "      <td>29427087</td>\n",
       "      <td>Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...</td>\n",
       "      <td>Neuroinflammation Alters Integrative Propertie...</td>\n",
       "      <td>2018</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>['N212/17']</td>\n",
       "      <td>['TRIP8b (constant)']</td>\n",
       "      <td>28887304</td>\n",
       "      <td>Lyman KA et al. 2017 J Biol Chem 292 17718-17730.</td>\n",
       "      <td>Allostery between two binding sites in the ion...</td>\n",
       "      <td>2017</td>\n",
       "      <td>google scholar unique</td>\n",
       "      <td>Lyman KA et al. 2017 J Biol Chem 292 17718-17730.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>['N212/17']</td>\n",
       "      <td>['TRIP8b (constant)']</td>\n",
       "      <td>28127990</td>\n",
       "      <td>Lyman KA et al. 2017 Expert Opin Ther Targets ...</td>\n",
       "      <td>Animal models suggest the TRIP8b-HCN interacti...</td>\n",
       "      <td>2017</td>\n",
       "      <td>google scholar unique</td>\n",
       "      <td>Lyman KA et al. 2017 Expert Opin Ther Targets ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>['N144/14']</td>\n",
       "      <td>['6xHis']</td>\n",
       "      <td>22184198</td>\n",
       "      <td>Emond MR et al. 2011 J Cell Biol 195 1115-21.</td>\n",
       "      <td>A complex of Protocadherin-19 and N-cadherin m...</td>\n",
       "      <td>2011</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Emond MR et al. 2011 J Cell Biol 195 1115-21.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6243</th>\n",
       "      <td>{'N106/65'}</td>\n",
       "      <td>{'Ankyrin-G'}</td>\n",
       "      <td>33536249</td>\n",
       "      <td>Ogawa Y et al. 2021 J Cell Sci 134.</td>\n",
       "      <td>Endogenously expressed Ranbp2 is not at the ax...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "      <td>Ogawa Y et al. 2021 J Cell Sci 134.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6244</th>\n",
       "      <td>{'L6/60'}</td>\n",
       "      <td>{'Slo1/BKAlpha'}</td>\n",
       "      <td>33556372</td>\n",
       "      <td>Bukiya AN et al. 2021 J Biol Chem 296 100381.</td>\n",
       "      <td>Cholesterol activates BK channels by increasin...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "      <td>Bukiya AN et al. 2021 J Biol Chem 296 100381.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6245</th>\n",
       "      <td>{'N212B'}</td>\n",
       "      <td>{'TRIP8b'}</td>\n",
       "      <td>33547341</td>\n",
       "      <td>Rátkai A et al. 2021 Sci Rep 11 3236.</td>\n",
       "      <td>Homeostatic plasticity and burst activity are ...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "      <td>Rátkai A et al. 2021 Sci Rep 11 3236.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6246</th>\n",
       "      <td>{'K28/43'}</td>\n",
       "      <td>{'PSD-95'}</td>\n",
       "      <td>33663561</td>\n",
       "      <td>Perkins EM et al. 2021 Mol Neurodegener 16 13.</td>\n",
       "      <td>Altered network properties in C9ORF72 repeat e...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "      <td>Perkins EM et al. 2021 Mol Neurodegener 16 13.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6247</th>\n",
       "      <td>{'K28/43'}</td>\n",
       "      <td>{'PSD-95'}</td>\n",
       "      <td>33615226</td>\n",
       "      <td>Gavrilovici C et al. 2021 Cereb Cortex Commun ...</td>\n",
       "      <td>Behavioral Deficits in Mice with Postnatal Dis...</td>\n",
       "      <td>2021</td>\n",
       "      <td>legacy</td>\n",
       "      <td>Gavrilovici C et al. 2021 Cereb Cortex Commun ...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>6248 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           MAB_ID                      Target      PMID  \\\n",
       "0     ['N37A/10']  ['Kv7.1/KCNQ1 K+ channel']  22024150   \n",
       "1      ['N70/28']                    ['HCN1']  29427087   \n",
       "2     ['N212/17']       ['TRIP8b (constant)']  28887304   \n",
       "3     ['N212/17']       ['TRIP8b (constant)']  28127990   \n",
       "4     ['N144/14']                   ['6xHis']  22184198   \n",
       "...           ...                         ...       ...   \n",
       "6243  {'N106/65'}               {'Ankyrin-G'}  33536249   \n",
       "6244    {'L6/60'}            {'Slo1/BKAlpha'}  33556372   \n",
       "6245    {'N212B'}                  {'TRIP8b'}  33547341   \n",
       "6246   {'K28/43'}                  {'PSD-95'}  33663561   \n",
       "6247   {'K28/43'}                  {'PSD-95'}  33615226   \n",
       "\n",
       "                                               citation  \\\n",
       "0        Krzystanek K et al. 2012 Heart Rhythm 9 440-8.   \n",
       "1     Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...   \n",
       "2     Lyman KA et al. 2017 J Biol Chem 292 17718-17730.   \n",
       "3     Lyman KA et al. 2017 Expert Opin Ther Targets ...   \n",
       "4         Emond MR et al. 2011 J Cell Biol 195 1115-21.   \n",
       "...                                                 ...   \n",
       "6243                Ogawa Y et al. 2021 J Cell Sci 134.   \n",
       "6244      Bukiya AN et al. 2021 J Biol Chem 296 100381.   \n",
       "6245              Rátkai A et al. 2021 Sci Rep 11 3236.   \n",
       "6246     Perkins EM et al. 2021 Mol Neurodegener 16 13.   \n",
       "6247  Gavrilovici C et al. 2021 Cereb Cortex Commun ...   \n",
       "\n",
       "                                                  title  year  \\\n",
       "0     Deubiquitylating enzyme USP2 counteracts Nedd4...  2012   \n",
       "1     Neuroinflammation Alters Integrative Propertie...  2018   \n",
       "2     Allostery between two binding sites in the ion...  2017   \n",
       "3     Animal models suggest the TRIP8b-HCN interacti...  2017   \n",
       "4     A complex of Protocadherin-19 and N-cadherin m...  2011   \n",
       "...                                                 ...   ...   \n",
       "6243  Endogenously expressed Ranbp2 is not at the ax...  2021   \n",
       "6244  Cholesterol activates BK channels by increasin...  2021   \n",
       "6245  Homeostatic plasticity and burst activity are ...  2021   \n",
       "6246  Altered network properties in C9ORF72 repeat e...  2021   \n",
       "6247  Behavioral Deficits in Mice with Postnatal Dis...  2021   \n",
       "\n",
       "                     source                                     citation_final  \n",
       "0                       NaN     Krzystanek K et al. 2012 Heart Rhythm 9 440-8.  \n",
       "1                       NaN  Frigerio F et al. 2018 Mol Neurobiol 55 7500-7...  \n",
       "2     google scholar unique  Lyman KA et al. 2017 J Biol Chem 292 17718-17730.  \n",
       "3     google scholar unique  Lyman KA et al. 2017 Expert Opin Ther Targets ...  \n",
       "4                       NaN      Emond MR et al. 2011 J Cell Biol 195 1115-21.  \n",
       "...                     ...                                                ...  \n",
       "6243                 legacy                Ogawa Y et al. 2021 J Cell Sci 134.  \n",
       "6244                 legacy      Bukiya AN et al. 2021 J Biol Chem 296 100381.  \n",
       "6245                 legacy              Rátkai A et al. 2021 Sci Rep 11 3236.  \n",
       "6246                 legacy     Perkins EM et al. 2021 Mol Neurodegener 16 13.  \n",
       "6247                 legacy  Gavrilovici C et al. 2021 Cereb Cortex Commun ...  \n",
       "\n",
       "[6248 rows x 8 columns]"
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_cure"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "5fd3459f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# filter ADAM22 contained in Target\n",
    "df_cure[df_cure['PMID']==38194969]\n",
    "df_curec = df_cure"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "72284c0e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "MAB_ID            object\n",
       "Target            object\n",
       "PMID               int64\n",
       "citation          object\n",
       "title             object\n",
       "year               int64\n",
       "source            object\n",
       "citation_final    object\n",
       "dtype: object"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_curec.dtypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 187,
   "id": "173bfa74",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-187-efce0a3337cd>:2: FutureWarning:\n",
      "\n",
      "The default value of regex will change from True to False in a future version. In addition, single character regular expressions will *not* be treated as literal strings when regex=True.\n",
      "\n",
      "<ipython-input-187-efce0a3337cd>:3: FutureWarning:\n",
      "\n",
      "The default value of regex will change from True to False in a future version. In addition, single character regular expressions will *not* be treated as literal strings when regex=True.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# conver both MAB_ID and Target to strings and remove the {} and [] and split on the comma to make them a list\n",
    "# df_curec['MAB_ID'] = df_cure['MAB_ID'].str.replace('{', '').str.replace('}', '').str.replace(\"'\", '').str.replace(\"]\", '').str.replace(\"[\", '').str.replace(' or ', ', ').str.split(',')\n",
    "# df_curec['Target'] = df_cure['Target'].str.replace('{', '').str.replace('}', '').str.replace(\"'\", '').str.replace(\"]\", '').str.replace(\"[\", '').str.replace(' or ', ', ').str.split(',')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 195,
   "id": "a8e599b5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "MAB_ID            object\n",
       "Target            object\n",
       "PMID               int64\n",
       "citation          object\n",
       "title             object\n",
       "year               int64\n",
       "source            object\n",
       "citation_final    object\n",
       "dtype: object"
      ]
     },
     "execution_count": 195,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_curec[df_curec['PMID']==38194969]\n",
    "# get the column data types\n",
    "df_curec.dtypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "6638acc6",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-55-8a05630c10f7>:4: FutureWarning:\n",
      "\n",
      "The default value of regex will change from True to False in a future version. In addition, single character regular expressions will *not* be treated as literal strings when regex=True.\n",
      "\n",
      "<ipython-input-55-8a05630c10f7>:5: FutureWarning:\n",
      "\n",
      "The default value of regex will change from True to False in a future version. In addition, single character regular expressions will *not* be treated as literal strings when regex=True.\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PMID</th>\n",
       "      <th>MAB_ID</th>\n",
       "      <th>Target</th>\n",
       "      <th>citation_final</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>16511870</td>\n",
       "      <td>\"K28/43\"</td>\n",
       "      <td>PSD-95</td>\n",
       "      <td>Mo Z et al. 2006 J Neurosci Res 83 1323-30.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>16525047</td>\n",
       "      <td>\"K75/41,  K55/7\"</td>\n",
       "      <td>KChIP1 K+ channel, Kv4.3 K+ channel</td>\n",
       "      <td>Kollo M et al. 2006 J Neurosci 26 2684-91.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>16566008</td>\n",
       "      <td>\"L6/60\"</td>\n",
       "      <td>Slo1/BKAlpha maxi-K+ channel</td>\n",
       "      <td>Misonou H et al. 2006 J Comp Neurol 496 289-302.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>16687515</td>\n",
       "      <td>\"K73/20\"</td>\n",
       "      <td>Contactin/F3</td>\n",
       "      <td>Ogawa Y et al. 2006 J Neurosci 26 5230-9.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>16793899</td>\n",
       "      <td>\"review article\"</td>\n",
       "      <td>review article</td>\n",
       "      <td>Anderson CN et al. 2006 J Physiol 575 367-72.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4586</th>\n",
       "      <td>38736483</td>\n",
       "      <td>\"N59/36\", \"K28/43\"</td>\n",
       "      <td>GluN2B/NR2B glutamate receptor, PSD-95</td>\n",
       "      <td>Matsumoto Y et al. 2024 Front Mol Neurosci 17 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4587</th>\n",
       "      <td>38746081</td>\n",
       "      <td>\"N106/36\", \"N56/21\", \"K87A/10\"</td>\n",
       "      <td>Ankyrin-G (staining), Nav1.6 Na+ channel, FGF1...</td>\n",
       "      <td>Ransdell JL et al. 2024 bioRxiv.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4588</th>\n",
       "      <td>38767050</td>\n",
       "      <td>\"N295B/66\"</td>\n",
       "      <td>Arl13b</td>\n",
       "      <td>Kumari A et al. 2024 Cytoskeleton (Hoboken).</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4589</th>\n",
       "      <td>38767486</td>\n",
       "      <td>\"K28/74\"</td>\n",
       "      <td>PSD-95</td>\n",
       "      <td>Sun SY et al. 2025 Neural Regen Res 20 209-223.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4590</th>\n",
       "      <td>38767832</td>\n",
       "      <td>\"K74/71\", \"N106/36\"</td>\n",
       "      <td>Nav1.1 Na+ channel, Ankyrin-G (staining)</td>\n",
       "      <td>Cui Y et al. 2024 Neurosci Bull.</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4591 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          PMID                          MAB_ID  \\\n",
       "0     16511870                        \"K28/43\"   \n",
       "1     16525047                \"K75/41,  K55/7\"   \n",
       "2     16566008                         \"L6/60\"   \n",
       "3     16687515                        \"K73/20\"   \n",
       "4     16793899                \"review article\"   \n",
       "...        ...                             ...   \n",
       "4586  38736483              \"N59/36\", \"K28/43\"   \n",
       "4587  38746081  \"N106/36\", \"N56/21\", \"K87A/10\"   \n",
       "4588  38767050                      \"N295B/66\"   \n",
       "4589  38767486                        \"K28/74\"   \n",
       "4590  38767832             \"K74/71\", \"N106/36\"   \n",
       "\n",
       "                                                 Target  \\\n",
       "0                                                PSD-95   \n",
       "1                   KChIP1 K+ channel, Kv4.3 K+ channel   \n",
       "2                          Slo1/BKAlpha maxi-K+ channel   \n",
       "3                                          Contactin/F3   \n",
       "4                                        review article   \n",
       "...                                                 ...   \n",
       "4586             GluN2B/NR2B glutamate receptor, PSD-95   \n",
       "4587  Ankyrin-G (staining), Nav1.6 Na+ channel, FGF1...   \n",
       "4588                                             Arl13b   \n",
       "4589                                             PSD-95   \n",
       "4590           Nav1.1 Na+ channel, Ankyrin-G (staining)   \n",
       "\n",
       "                                         citation_final  \n",
       "0           Mo Z et al. 2006 J Neurosci Res 83 1323-30.  \n",
       "1            Kollo M et al. 2006 J Neurosci 26 2684-91.  \n",
       "2      Misonou H et al. 2006 J Comp Neurol 496 289-302.  \n",
       "3             Ogawa Y et al. 2006 J Neurosci 26 5230-9.  \n",
       "4         Anderson CN et al. 2006 J Physiol 575 367-72.  \n",
       "...                                                 ...  \n",
       "4586  Matsumoto Y et al. 2024 Front Mol Neurosci 17 ...  \n",
       "4587                   Ransdell JL et al. 2024 bioRxiv.  \n",
       "4588       Kumari A et al. 2024 Cytoskeleton (Hoboken).  \n",
       "4589    Sun SY et al. 2025 Neural Regen Res 20 209-223.  \n",
       "4590                   Cui Y et al. 2024 Neurosci Bull.  \n",
       "\n",
       "[4591 rows x 4 columns]"
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "# make the MAB_ID and Target columns sets \n",
    "\n",
    "df_curegb = df_curec.groupby('PMID').agg({'MAB_ID': lambda x: set(x), 'Target': lambda x: set(x), 'citation_final': 'first'}).reset_index()\n",
    "df_curegb['MAB_ID'] = df_curegb['MAB_ID'].astype(str).str.replace('{', '').str.replace('}', '').str.replace(\"'\", '').str.replace(\"]\", '').str.replace(\"[\", '')\n",
    "df_curegb['Target'] = df_curegb['Target'].astype(str).str.replace('{', '').str.replace('}', '').str.replace(\"'\", '').str.replace(\"]\", '').str.replace(\"[\", '')\n",
    "# remove \"\" from Target \n",
    "df_curegb['Target'] = df_curegb['Target'].str.replace('\"', '')\n",
    "df_curegb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "b3de1c08",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PMID</th>\n",
       "      <th>MAB_ID</th>\n",
       "      <th>Target</th>\n",
       "      <th>citation_final</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>4530</th>\n",
       "      <td>38194969</td>\n",
       "      <td>\"K67/25,  K36/15,  K13/31,  L23/27\"</td>\n",
       "      <td>CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+ ...</td>\n",
       "      <td>Miyazaki Y et al. 2024 Cell Rep 43 113634.</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          PMID                               MAB_ID  \\\n",
       "4530  38194969  \"K67/25,  K36/15,  K13/31,  L23/27\"   \n",
       "\n",
       "                                                 Target  \\\n",
       "4530  CASPR2, Kv1.1 K+ channel (external), Kv1.3 K+ ...   \n",
       "\n",
       "                                  citation_final  \n",
       "4530  Miyazaki Y et al. 2024 Cell Rep 43 113634.  "
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# filter for the PMID again\n",
    "df_curegb[df_curegb['PMID'] == 38194969]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "a5a8ce38",
   "metadata": {},
   "outputs": [],
   "source": [
    "# output this to a csv file\n",
    "df_curegb.to_csv(\"/Users/keithmitchell/Desktop/Repositories/NeuroMabSeq/trimmer/picklequeries/df_cure_cleaned_grouped-5-20-24.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "80e13fcb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "config": {
        "plotlyServerURL": "https://plot.ly"
       },
       "data": [
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N37A/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N37A/10']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N37A/10']",
         "offsetgroup": "['N37A/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2009,
          2010,
          2013,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28']",
         "offsetgroup": "['N70/28']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2011,
          2011,
          2011,
          2023,
          2013,
          2009,
          2017,
          2014,
          2013,
          2017,
          2022,
          2022,
          2022,
          2018,
          2012,
          2020,
          2010,
          2010,
          2010,
          2013,
          2014,
          2015,
          2016,
          2016,
          2016,
          2017,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2020,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N212/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N212/17']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N212/17']",
         "offsetgroup": "['N212/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2024,
          2012,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N144/14']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N144/14']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N144/14']",
         "offsetgroup": "['N144/14']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2012,
          2018,
          2023,
          2017,
          2014,
          2012,
          2009,
          2014,
          2016,
          2018,
          2019,
          2020,
          2021,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36']",
         "offsetgroup": "['N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2020,
          2013,
          2021,
          2021,
          2021,
          2017,
          2018,
          2021,
          2017,
          2017,
          2013,
          2020,
          2020,
          2019,
          2017,
          2020,
          2020,
          2021,
          2012,
          2020,
          2018,
          2018,
          2013,
          2017,
          2016,
          2019,
          2021,
          2021,
          2020,
          2020,
          2011,
          2019,
          2014,
          2014,
          2021,
          2021,
          2021,
          2021,
          2017,
          2017,
          2024,
          2024,
          2017,
          2014,
          2022,
          2023,
          2022,
          2022,
          2015,
          2015,
          2015,
          2013,
          2017,
          2017,
          2017,
          2019,
          2019,
          2021,
          2021,
          2021,
          2023,
          2017,
          2017,
          2019,
          2022,
          2021,
          2013,
          2013,
          2018,
          2021,
          2016,
          2019,
          2019,
          2014,
          2020,
          2021,
          2022,
          2022,
          2012,
          2018,
          2022,
          2023,
          2023,
          2019,
          2019,
          2017,
          2023,
          2021,
          2022,
          2022,
          2016,
          2014,
          2022,
          2022,
          2021,
          2014,
          2013,
          2022,
          2020,
          2023,
          2023,
          2018,
          2018,
          2014,
          2015,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2023,
          2023,
          2023,
          2024,
          2024,
          2024,
          2023,
          2021,
          2020,
          2018,
          2023,
          2023,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2017,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2020,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2021,
          2021,
          2021,
          2021,
          2022,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N105/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N105/17']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N105/17']",
         "offsetgroup": "['N105/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2013,
          2013,
          2013,
          2018,
          2018,
          2014,
          2014,
          2014,
          2014,
          2023,
          2019,
          2017,
          2019,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N105/17', ' N144/14']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N105/17', ' N144/14']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N105/17', ' N144/14']",
         "offsetgroup": "['N106/36', ' N105/17', ' N144/14']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N144/14', ' N105/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N144/14', ' N105/17']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N144/14', ' N105/17']",
         "offsetgroup": "['N106/36', ' N144/14', ' N105/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N100/13', ' N144/14', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N100/13', ' N144/14', ' N16B/8']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N100/13', ' N144/14', ' N16B/8']",
         "offsetgroup": "['N100/13', ' N144/14', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N385/21', ' L114/3', ' N106/36', ' N419/78']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N385/21', ' L114/3', ' N106/36', ' N419/78']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N385/21', ' L114/3', ' N106/36', ' N419/78']",
         "offsetgroup": "['N385/21', ' L114/3', ' N106/36', ' N419/78']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N385/21', ' N255/38', ' N419/78']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N385/21', ' N255/38', ' N419/78']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N385/21', ' N255/38', ' N419/78']",
         "offsetgroup": "['N385/21', ' N255/38', ' N419/78']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N385/21', ' K14/16', ' N106/36', ' N419/78']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N385/21', ' K14/16', ' N106/36', ' N419/78']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N385/21', ' K14/16', ' N106/36', ' N419/78']",
         "offsetgroup": "['N385/21', ' K14/16', ' N106/36', ' N419/78']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N23B/6']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N23B/6']",
         "offsetgroup": "['N23B/49', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015,
          2010,
          2010,
          2010,
          2010,
          2010,
          2012,
          2012,
          2012,
          2012,
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' L117/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' L117/1']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' L117/1']",
         "offsetgroup": "['N23B/49', ' L117/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49']",
         "offsetgroup": "['N23B/49']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2010,
          2010,
          2022,
          2011,
          2011,
          2011,
          2011,
          2024,
          2010,
          2010,
          2010,
          2011,
          2011,
          2011,
          2012,
          2014,
          2014,
          2015,
          2016,
          2016,
          2018,
          2019,
          2022,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N367/62', ' N22/21', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N367/62', ' N22/21', ' N23B/6']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N367/62', ' N22/21', ' N23B/6']",
         "offsetgroup": "['N23B/49', ' N367/62', ' N22/21', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['62/3G1', ' L106/93', ' N97A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['62/3G1', ' L106/93', ' N97A/31']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['62/3G1', ' L106/93', ' N97A/31']",
         "offsetgroup": "['62/3G1', ' L106/93', ' N97A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L20/8', ' N59/20', ' N19/2', ' N23B/49', ' K64/15', ' N97A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L20/8', ' N59/20', ' N19/2', ' N23B/49', ' K64/15', ' N97A/31']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L20/8', ' N59/20', ' N19/2', ' N23B/49', ' K64/15', ' N97A/31']",
         "offsetgroup": "['L20/8', ' N59/20', ' N19/2', ' N23B/49', ' K64/15', ' N97A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N23B/49']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N23B/49']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N23B/49']",
         "offsetgroup": "['N327/95', ' N23B/49']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N367/62', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N367/62', ' N23B/6']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N367/62', ' N23B/6']",
         "offsetgroup": "['N23B/49', ' N367/62', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L20/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L20/8']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L20/8']",
         "offsetgroup": "['L20/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2010,
          2010,
          2010,
          2021,
          2016,
          2008,
          2009,
          2009,
          2010,
          2011,
          2011,
          2011,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2016,
          2017,
          2016,
          2017,
          2018,
          2018,
          2019,
          2020,
          2020,
          2020,
          2021,
          2021,
          2021,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N19/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N19/2']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N19/2']",
         "offsetgroup": "['N19/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2024,
          2019,
          2019,
          2019,
          2022,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2012,
          2016,
          2013,
          2013,
          2013,
          2016,
          2016,
          2016,
          2009,
          2009,
          2010,
          2010,
          2011,
          2011,
          2011,
          2011,
          2011,
          2014,
          2014,
          2016,
          2017,
          2019,
          2018,
          2019,
          2021,
          2021,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N97A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N97A/31']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N97A/31']",
         "offsetgroup": "['N97A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2016,
          2016,
          2016,
          2016,
          2018,
          2021,
          2020,
          2020,
          2020,
          2016,
          2016,
          2016,
          2022,
          2022,
          2015,
          2016,
          2016,
          2017,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K64/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K64/15']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K64/15']",
         "offsetgroup": "['K64/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2012,
          2012,
          2012,
          2019,
          2021,
          2016,
          2017,
          2020,
          2014,
          2010,
          2010,
          2013,
          2014,
          2015,
          2015,
          2015,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36']",
         "offsetgroup": "['N59/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2018,
          2023,
          2024,
          2021,
          2014,
          2014,
          2019,
          2019,
          2015,
          2015,
          2015,
          2015,
          2016,
          2017,
          2016,
          2021,
          2012,
          2015,
          2018,
          2019,
          2013,
          2016,
          2011,
          2011,
          2012,
          2012,
          2012,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2016,
          2016,
          2017,
          2018,
          2018,
          2018,
          2020,
          2020,
          2021,
          2020,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L20/8', ' N97A/31', ' N59/36', ' N23B/49', ' N19/2', ' K64/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L20/8', ' N97A/31', ' N59/36', ' N23B/49', ' N19/2', ' K64/15']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L20/8', ' N97A/31', ' N59/36', ' N23B/49', ' N19/2', ' K64/15']",
         "offsetgroup": "['L20/8', ' N97A/31', ' N59/36', ' N23B/49', ' N19/2', ' K64/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L20/8', ' N19/2', ' N23B/49', ' K64/15', ' N97A/31', ' N59/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L20/8', ' N19/2', ' N23B/49', ' K64/15', ' N97A/31', ' N59/36']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L20/8', ' N19/2', ' N23B/49', ' K64/15', ' N97A/31', ' N59/36']",
         "offsetgroup": "['L20/8', ' N19/2', ' N23B/49', ' K64/15', ' N97A/31', ' N59/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' K28/43', ' N19/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' K28/43', ' N19/2']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' K28/43', ' N19/2']",
         "offsetgroup": "['N23B/49', ' K28/43', ' N19/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' K28/43', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' K28/43', ' N23B/6']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' K28/43', ' N23B/6']",
         "offsetgroup": "['N23B/49', ' K28/43', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N367/62', ' N22/21', ' N23B/49']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N367/62', ' N22/21', ' N23B/49']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N367/62', ' N22/21', ' N23B/49']",
         "offsetgroup": "['N327/95', ' N367/62', ' N22/21', ' N23B/49']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N39B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N39B/8']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N39B/8']",
         "offsetgroup": "['N23B/49', ' N39B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N28/9', ' N110/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N28/9', ' N110/29']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N28/9', ' N110/29']",
         "offsetgroup": "['N23B/49', ' N28/9', ' N110/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N125/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N125/10']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N125/10']",
         "offsetgroup": "['N125/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N479/107', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N479/107', ' N106/36']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N479/107', ' N106/36']",
         "offsetgroup": "['N479/107', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N69/46']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N69/46']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N69/46']",
         "offsetgroup": "['N69/46']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N367/62']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N367/62']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N367/62']",
         "offsetgroup": "['N367/62']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2020,
          2018,
          2023,
          2019,
          2016,
          2016,
          2016,
          2023,
          2018,
          2024,
          2017,
          2019,
          2020,
          2022,
          2023,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N367/62', ' N69/46']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N367/62', ' N69/46']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N367/62', ' N69/46']",
         "offsetgroup": "['N367/62', ' N69/46']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N69/46', ' N367/62']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N69/46', ' N367/62']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N69/46', ' N367/62']",
         "offsetgroup": "['N69/46', ' N367/62']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/20']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/20']",
         "offsetgroup": "['N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2008,
          2008,
          2008,
          2018,
          2010,
          2010,
          2010,
          2011,
          2023,
          2023,
          2009,
          2011,
          2013,
          2015,
          2012,
          2011,
          2013,
          2013,
          2011,
          2016,
          2018,
          2010,
          2010,
          2011,
          2012,
          2012,
          2013,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2014,
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2015,
          2014,
          2015,
          2015,
          2015,
          2015,
          2018,
          2018,
          2018,
          2020,
          2020,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N415/24', ' N431/64', ' N244/5', ' N52A/42', ' N425/45', ' N71/37', ' N423/75', ' N419/40', ' N388A/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N415/24', ' N431/64', ' N244/5', ' N52A/42', ' N425/45', ' N71/37', ' N423/75', ' N419/40', ' N388A/27']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N415/24', ' N431/64', ' N244/5', ' N52A/42', ' N425/45', ' N71/37', ' N423/75', ' N419/40', ' N388A/27']",
         "offsetgroup": "['N415/24', ' N431/64', ' N244/5', ' N52A/42', ' N425/45', ' N71/37', ' N423/75', ' N419/40', ' N388A/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N399/19']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N399/19']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N399/19']",
         "offsetgroup": "['N399/19']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8']",
         "offsetgroup": "['N86/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2014,
          2022,
          2019,
          2017,
          2023,
          2021,
          2016,
          2018,
          2018,
          2017,
          2022,
          2020,
          2022,
          2016,
          2015,
          2022,
          2015,
          2022,
          2021,
          2020,
          2020,
          2020,
          2016,
          2019,
          2016,
          2014,
          2014,
          2014,
          2018,
          2017,
          2016,
          2020,
          2018,
          2017,
          2018,
          2020,
          2021,
          2019,
          2017,
          2014,
          2020,
          2021,
          2020,
          2015,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2024,
          2023,
          2023,
          2024,
          2023,
          2014,
          2014,
          2015,
          2015,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2022,
          2022,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N56/21', ' N106/36', ' N86/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N56/21', ' N106/36', ' N86/38']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N56/21', ' N106/36', ' N86/38']",
         "offsetgroup": "['N86/8', ' N56/21', ' N106/36', ' N86/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N86/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N86/38']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N86/38']",
         "offsetgroup": "['N86/8', ' N86/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2022,
          2012,
          2009,
          2010,
          2012,
          2012,
          2011,
          2011,
          2011,
          2013,
          2014,
          2012,
          2013,
          2009,
          2009,
          2010,
          2010,
          2010,
          2010,
          2011,
          2011,
          2011,
          2011,
          2012,
          2011,
          2013,
          2012,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2013,
          2014,
          2013,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/38']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/38']",
         "offsetgroup": "['N86/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2023,
          2023,
          2015,
          2013,
          2013,
          2019,
          2012,
          2012,
          2021,
          2015,
          2015,
          2020,
          2021,
          2015,
          2018,
          2022,
          2022,
          2010,
          2015,
          2012,
          2018,
          2015,
          2015,
          2015,
          2008,
          2018,
          2016,
          2014,
          2020,
          2013,
          2018,
          2008,
          2011,
          2011,
          2011,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2014,
          2013,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2014,
          2015,
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N206A/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N206A/8']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N206A/8']",
         "offsetgroup": "['N86/8', ' N206A/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2022,
          2017,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' N167/7', ' N86/38', ' N86/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' N167/7', ' N86/38', ' N86/8']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' N167/7', ' N86/38', ' N86/8']",
         "offsetgroup": "['K20/78', ' N167/7', ' N86/38', ' N86/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N241A/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N241A/34']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N241A/34']",
         "offsetgroup": "['N86/8', ' N241A/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N52A/42', ' N86/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N52A/42', ' N86/38']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N52A/42', ' N86/38']",
         "offsetgroup": "['N86/8', ' N52A/42', ' N86/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K28/86', ' K28/43', ' N28/9', ' N147/6', ' N87/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K28/86', ' K28/43', ' N28/9', ' N147/6', ' N87/25']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K28/86', ' K28/43', ' N28/9', ' N147/6', ' N87/25']",
         "offsetgroup": "['K89/34', ' K28/86', ' K28/43', ' N28/9', ' N147/6', ' N87/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' 62/3G1', ' N110/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' 62/3G1', ' N110/29']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' 62/3G1', ' N110/29']",
         "offsetgroup": "['N86/8', ' 62/3G1', ' N110/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' N19/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' N19/2']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' N19/2']",
         "offsetgroup": "['N59/36', ' N19/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N86/8', ' K65/35', ' K14/16', ' K87A/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N86/8', ' K65/35', ' K14/16', ' K87A/10']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N86/8', ' K65/35', ' K14/16', ' K87A/10']",
         "offsetgroup": "['N106/36', ' N86/8', ' K65/35', ' K14/16', ' K87A/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' A12/18']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' A12/18']",
         "offsetgroup": "['N86/8', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' L20/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' L20/8']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' L20/8']",
         "offsetgroup": "['N86/8', ' L20/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N103/39']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N103/39']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N103/39']",
         "offsetgroup": "['N103/39']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2014,
          2014,
          2014,
          2013,
          2015,
          2022,
          2022,
          2016,
          2015,
          2021,
          2020,
          2009,
          2009,
          2021,
          2013,
          2013,
          2010,
          2010,
          2011,
          2012,
          2012,
          2013,
          2013,
          2013,
          2014,
          2014,
          2014,
          2016,
          2016,
          2017,
          2017,
          2019,
          2018,
          2018,
          2018,
          2019,
          2020,
          2019,
          2021,
          2021,
          2021,
          2022,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N86/44']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N86/44']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N86/44']",
         "offsetgroup": "['N86/8', ' N86/44']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N22/21', ' N238/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N22/21', ' N238/31']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N22/21', ' N238/31']",
         "offsetgroup": "['N86/8', ' N22/21', ' N238/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N38/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N38/8']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N38/8']",
         "offsetgroup": "['N38/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2017,
          2015,
          2015,
          2015,
          2015,
          2015,
          2016,
          2016,
          2013,
          2013,
          2009,
          2009,
          2011,
          2023,
          2012,
          2012,
          2012,
          2023,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43']",
         "offsetgroup": "['K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2023,
          2016,
          2017,
          2023,
          2017,
          2023,
          2023,
          2023,
          2023,
          2024,
          2024,
          2024,
          2016,
          2021,
          2021,
          2021,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2024,
          2024,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2024,
          2023,
          2023,
          2024,
          2012,
          2018,
          2013,
          2022,
          2017,
          2022,
          2018,
          2021,
          2014,
          2014,
          2011,
          2012,
          2022,
          2013,
          2018,
          2016,
          2017,
          2016,
          2015,
          2016,
          2010,
          2013,
          2015,
          2014,
          2020,
          2021,
          2021,
          2020,
          2019,
          2017,
          2015,
          2012,
          2018,
          2012,
          2011,
          2011,
          2009,
          2016,
          2014,
          2010,
          2015,
          2020,
          2020,
          2022,
          2020,
          2018,
          2021,
          2010,
          2008,
          2018,
          2015,
          2015,
          2010,
          2010,
          2022,
          2013,
          2013,
          2016,
          2012,
          2016,
          2011,
          2015,
          2013,
          2013,
          2021,
          2010,
          2019,
          2019,
          2010,
          2009,
          2010,
          2014,
          2012,
          2021,
          2013,
          2014,
          2014,
          2013,
          2017,
          2017,
          2017,
          2021,
          2011,
          2020,
          2017,
          2017,
          2020,
          2015,
          2010,
          2011,
          2022,
          2013,
          2021,
          2008,
          2018,
          2014,
          2019,
          2021,
          2018,
          2019,
          2016,
          2022,
          2015,
          2009,
          2017,
          2018,
          2013,
          2014,
          2020,
          2016,
          2019,
          2019,
          2015,
          2015,
          2019,
          2022,
          2012,
          2017,
          2017,
          2007,
          2015,
          2020,
          2012,
          2019,
          2011,
          2017,
          2021,
          2011,
          2018,
          2013,
          2017,
          2012,
          2013,
          2012,
          2014,
          2017,
          2019,
          2024,
          2015,
          2011,
          2016,
          2015,
          2018,
          2021,
          2015,
          2016,
          2022,
          2014,
          2019,
          2012,
          2021,
          2017,
          2017,
          2011,
          2011,
          2017,
          2016,
          2017,
          2017,
          2017,
          2017,
          2018,
          2019,
          2019,
          2020,
          2020,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2022,
          2021,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2023,
          2022,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N59/36', ' L21/32', ' L20/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N59/36', ' L21/32', ' L20/8']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N59/36', ' L21/32', ' L20/8']",
         "offsetgroup": "['N327/95', ' N59/36', ' L21/32', ' L20/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N295B/66']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N295B/66']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N295B/66']",
         "offsetgroup": "['N295B/66']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2014,
          2014,
          2013,
          2013,
          2021,
          2021,
          2021,
          2021,
          2023,
          2022,
          2022,
          2020,
          2020,
          2016,
          2016,
          2021,
          2019,
          2020,
          2017,
          2017,
          2017,
          2015,
          2017,
          2017,
          2020,
          2016,
          2019,
          2019,
          2023,
          2023,
          2019,
          2019,
          2018,
          2018,
          2016,
          2023,
          2018,
          2017,
          2023,
          2023,
          2017,
          2021,
          2019,
          2020,
          2020,
          2012,
          2020,
          2021,
          2016,
          2015,
          2015,
          2018,
          2018,
          2012,
          2019,
          2019,
          2023,
          2023,
          2020,
          2023,
          2023,
          2020,
          2020,
          2019,
          2022,
          2022,
          2017,
          2020,
          2018,
          2018,
          2018,
          2019,
          2020,
          2018,
          2020,
          2016,
          2022,
          2017,
          2017,
          2019,
          2023,
          2024,
          2024,
          2017,
          2024,
          2024,
          2024,
          2023,
          2023,
          2018,
          2017,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2020,
          2021,
          2020,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N295B/66', ' N241A/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N295B/66', ' N241A/34']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N295B/66', ' N241A/34']",
         "offsetgroup": "['N295B/66', ' N241A/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N144/32', ' N144/17', ' N148/30', ' N159/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N144/32', ' N144/17', ' N148/30', ' N159/5']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N144/32', ' N144/17', ' N148/30', ' N159/5']",
         "offsetgroup": "['N144/32', ' N144/17', ' N148/30', ' N159/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N126B/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N126B/31']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N126B/31']",
         "offsetgroup": "['N126B/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2014,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N126B/31', ' N235/22']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N126B/31', ' N235/22']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N126B/31', ' N235/22']",
         "offsetgroup": "['N126B/31', ' N235/22']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34']",
         "offsetgroup": "['K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2016,
          2023,
          2009,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2017,
          2013,
          2020,
          2012,
          2022,
          2006,
          2014,
          2021,
          2009,
          2017,
          2011,
          2021,
          2013,
          2012,
          2022,
          2012,
          2013,
          2020,
          2018,
          2018,
          2013,
          2023,
          2023,
          2019,
          2017,
          2017,
          2015,
          2015,
          2016,
          2020,
          2021,
          2013,
          2016,
          2017,
          2017,
          2017,
          2015,
          2019,
          2019,
          2019,
          2015,
          2011,
          2019,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2021,
          2024,
          2024,
          2024,
          2023,
          2010,
          2011,
          2011,
          2011,
          2012,
          2012,
          2012,
          2012,
          2012,
          2013,
          2014,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2016,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2021,
          2021,
          2022,
          2022,
          2022,
          2022,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N419/78', ' N385/21', ' N410/17', ' N16B/8', ' L114/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N419/78', ' N385/21', ' N410/17', ' N16B/8', ' L114/3']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N419/78', ' N385/21', ' N410/17', ' N16B/8', ' L114/3']",
         "offsetgroup": "['N419/78', ' N385/21', ' N410/17', ' N16B/8', ' L114/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K36/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K36/15']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K36/15']",
         "offsetgroup": "['K87A/10', ' K36/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' L11A/41', ' K20/78', ' N57/2', ' K14/16', ' N46/30', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' L11A/41', ' K20/78', ' N57/2', ' K14/16', ' N46/30', ' N18/30']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' L11A/41', ' K20/78', ' N57/2', ' K14/16', ' N46/30', ' N18/30']",
         "offsetgroup": "['K13/31', ' L11A/41', ' K20/78', ' N57/2', ' K14/16', ' N46/30', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N57/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N57/2']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N57/2']",
         "offsetgroup": "['N57/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N104/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N104/37']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N104/37']",
         "offsetgroup": "['N104/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N205B/22']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N205B/22']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N205B/22']",
         "offsetgroup": "['N205B/22']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2022,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K56A/50']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K56A/50']",
         "offsetgroup": "['K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2014,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2018,
          2018,
          2018,
          2018,
          2023,
          2023,
          2022,
          2020,
          2013,
          2009,
          2010,
          2012,
          2013,
          2013,
          2013,
          2015,
          2015,
          2016,
          2016,
          2016,
          2017,
          2018,
          2018,
          2018,
          2019,
          2020,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K56A/50', ' N205B/22']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K56A/50', ' N205B/22']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K56A/50', ' N205B/22']",
         "offsetgroup": "['K56A/50', ' N205B/22']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N205B/22', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N205B/22', ' K56A/50']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N205B/22', ' K56A/50']",
         "offsetgroup": "['N205B/22', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N95/35']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N95/35']",
         "offsetgroup": "['N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2019,
          2010,
          2010,
          2022,
          2022,
          2022,
          2022,
          2022,
          2017,
          2014,
          2014,
          2022,
          2022,
          2022,
          2021,
          2022,
          2015,
          2018,
          2016,
          2019,
          2019,
          2018,
          2013,
          2015,
          2015,
          2018,
          2018,
          2015,
          2011,
          2011,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2015,
          2015,
          2015,
          2016,
          2015,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2018,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2021,
          2021,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N95/35', ' N205B/22']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N95/35', ' N205B/22']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N95/35', ' N205B/22']",
         "offsetgroup": "['N95/35', ' N205B/22']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N205B/22', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N205B/22', ' N95/35']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N205B/22', ' N95/35']",
         "offsetgroup": "['N205B/22', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N34/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N34/34']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N34/34']",
         "offsetgroup": "['N34/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2015,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N29/29', ' N6/38', ' N28/9', ' N34/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N29/29', ' N6/38', ' N28/9', ' N34/34']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N29/29', ' N6/38', ' N28/9', ' N34/34']",
         "offsetgroup": "['N29/29', ' N6/38', ' N28/9', ' N34/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N29/29', ' N59/36', ' N28/9', ' N34/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N29/29', ' N59/36', ' N28/9', ' N34/34']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N29/29', ' N59/36', ' N28/9', ' N34/34']",
         "offsetgroup": "['N29/29', ' N59/36', ' N28/9', ' N34/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N359/28']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N359/28']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N359/28']",
         "offsetgroup": "['N359/28']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K88/64']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K88/64']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K88/64']",
         "offsetgroup": "['K88/64']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N183/15', ' N106/65', ' N147/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N183/15', ' N106/65', ' N147/6']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N183/15', ' N106/65', ' N147/6']",
         "offsetgroup": "['N183/15', ' N106/65', ' N147/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N183/15', ' N182/17', ' N165/38', ' N154/32', ' N147/6', ' N195A/16', ' N106/65']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N183/15', ' N182/17', ' N165/38', ' N154/32', ' N147/6', ' N195A/16', ' N106/65']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N183/15', ' N182/17', ' N165/38', ' N154/32', ' N147/6', ' N195A/16', ' N106/65']",
         "offsetgroup": "['N183/15', ' N182/17', ' N165/38', ' N154/32', ' N147/6', ' N195A/16', ' N106/65']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/23', ' L6/60', ' L57/46']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/23', ' L6/60', ' L57/46']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/23', ' L6/60', ' L57/46']",
         "offsetgroup": "['L6/23', ' L6/60', ' L57/46']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N183/15', ' N106/65']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N183/15', ' N106/65']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N183/15', ' N106/65']",
         "offsetgroup": "['N183/15', ' N106/65']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2016,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N121A/1', ' N121A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N121A/1', ' N121A/31']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N121A/1', ' N121A/31']",
         "offsetgroup": "['N121A/1', ' N121A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N121A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N121A/31']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N121A/31']",
         "offsetgroup": "['N121A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N151/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N151/3']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N151/3']",
         "offsetgroup": "['N151/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K39/25', ' N52A/42', ' K89/34', ' L114/3', ' L122/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K39/25', ' N52A/42', ' K89/34', ' L114/3', ' L122/6']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K39/25', ' N52A/42', ' K89/34', ' L114/3', ' L122/6']",
         "offsetgroup": "['K39/25', ' N52A/42', ' K89/34', ' L114/3', ' L122/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N52A/42']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N52A/42']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N52A/42']",
         "offsetgroup": "['N52A/42']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2015,
          2015,
          2012,
          2016,
          2015,
          2015,
          2016,
          2017,
          2015,
          2022,
          2023,
          2010,
          2022,
          2016,
          2009,
          2024,
          2017,
          2010,
          2012,
          2013,
          2013,
          2014,
          2016,
          2016,
          2016,
          2017,
          2017,
          2018,
          2019,
          2019,
          2019,
          2020,
          2020,
          2021,
          2022,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N52A/42', ' K89/34', ' N86/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N52A/42', ' K89/34', ' N86/38']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N52A/42', ' K89/34', ' N86/38']",
         "offsetgroup": "['N52A/42', ' K89/34', ' N86/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N52A/42', ' K89/34', ' L86/33', ' N372B/60']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N52A/42', ' K89/34', ' L86/33', ' N372B/60']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N52A/42', ' K89/34', ' L86/33', ' N372B/60']",
         "offsetgroup": "['N52A/42', ' K89/34', ' L86/33', ' N372B/60']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31']",
         "offsetgroup": "['K13/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2009,
          2016,
          2016,
          2016,
          2016,
          2007,
          2016,
          2018,
          2008,
          2009,
          2010,
          2012,
          2015,
          2016,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L83/11']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L83/11']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L83/11']",
         "offsetgroup": "['L83/11']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' N52A/42', ' L83/11', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' N52A/42', ' L83/11', ' K89/34']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' N52A/42', ' L83/11', ' K89/34']",
         "offsetgroup": "['K13/31', ' N52A/42', ' L83/11', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L83/11', ' N52A/42', ' K13/31', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L83/11', ' N52A/42', ' K13/31', ' K89/34']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L83/11', ' N52A/42', ' K13/31', ' K89/34']",
         "offsetgroup": "['L83/11', ' N52A/42', ' K13/31', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' N52A/42', ' K89/34', ' K14/16', ' K37/89', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' N52A/42', ' K89/34', ' K14/16', ' K37/89', ' K7/45']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' N52A/42', ' K89/34', ' K14/16', ' K37/89', ' K7/45']",
         "offsetgroup": "['K13/31', ' N52A/42', ' K89/34', ' K14/16', ' K37/89', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N221/17', ' N52A/42', ' N192/12']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N221/17', ' N52A/42', ' N192/12']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N221/17', ' N52A/42', ' N192/12']",
         "offsetgroup": "['N221/17', ' N52A/42', ' N192/12']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L114/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L114/3']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L114/3']",
         "offsetgroup": "['L114/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/20']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/20']",
         "offsetgroup": "['N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2015,
          2015,
          2023,
          2011,
          2012,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L109/39']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L109/39']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L109/39']",
         "offsetgroup": "['L109/39']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L86A/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L86A/37']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L86A/37']",
         "offsetgroup": "['L86A/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2009,
          2022,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N468/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N468/37']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N468/37']",
         "offsetgroup": "['N468/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K14/16']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K14/16']",
         "offsetgroup": "['K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2013,
          2017,
          2017,
          2017,
          2018,
          2018,
          2018,
          2017,
          2017,
          2017,
          2017,
          2017,
          2023,
          2023,
          2024,
          2023,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2023,
          2017,
          2017,
          2017,
          2022,
          2022,
          2022,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2007,
          2016,
          2012,
          2018,
          2020,
          2018,
          2012,
          2017,
          2015,
          2015,
          2011,
          2017,
          2015,
          2016,
          2015,
          2010,
          2022,
          2019,
          2016,
          2020,
          2020,
          2020,
          2022,
          2019,
          2017,
          2017,
          2021,
          2022,
          2016,
          2023,
          2010,
          2010,
          2010,
          2010,
          2010,
          2012,
          2012,
          2012,
          2012,
          2013,
          2012,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2019,
          2019,
          2019,
          2020,
          2023,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N398A/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N398A/34']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N398A/34']",
         "offsetgroup": "['N398A/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N93A/49']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N93A/49']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N93A/49']",
         "offsetgroup": "['N93A/49']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2012,
          2012,
          2012,
          2014,
          2011,
          2011,
          2011,
          2011,
          2020,
          2017,
          2023,
          2021,
          2012,
          2012,
          2016,
          2016,
          2011,
          2012,
          2014,
          2015,
          2015,
          2016,
          2019,
          2020,
          2021,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9']",
         "offsetgroup": "['N28/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2023,
          2022,
          2012,
          2021,
          2018,
          2018,
          2015,
          2012,
          2022,
          2019,
          2010,
          2014,
          2018,
          2020,
          2009,
          2011,
          2019,
          2019,
          2013,
          2013,
          2013,
          2016,
          2017,
          2016,
          2014,
          2019,
          2017,
          2013,
          2018,
          2018,
          2018,
          2018,
          2020,
          2020,
          2020,
          2022,
          2017,
          2015,
          2015,
          2015,
          2017,
          2015,
          2009,
          2022,
          2008,
          2009,
          2010,
          2010,
          2010,
          2011,
          2011,
          2011,
          2012,
          2012,
          2013,
          2012,
          2013,
          2013,
          2014,
          2014,
          2014,
          2015,
          2015,
          2016,
          2015,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2018,
          2019,
          2018,
          2020,
          2019,
          2019,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2021,
          2021,
          2022,
          2022,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N114/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N114/10']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N114/10']",
         "offsetgroup": "['N114/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2013,
          2012,
          2015,
          2014,
          2012,
          2013,
          2013,
          2013,
          2010,
          2012,
          2013,
          2013,
          2014,
          2014,
          2016,
          2016,
          2017,
          2017,
          2018,
          2018,
          2019,
          2019,
          2020,
          2020,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L86A/37', ' N52A/42', ' N398A/34', ' L109/39', ' L114/3', ' N114/10', ' N93A/49', ' K28/43', ' N468/37', ' N106/20', ' N28/9', ' K14/16', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L86A/37', ' N52A/42', ' N398A/34', ' L109/39', ' L114/3', ' N114/10', ' N93A/49', ' K28/43', ' N468/37', ' N106/20', ' N28/9', ' K14/16', ' K89/34']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L86A/37', ' N52A/42', ' N398A/34', ' L109/39', ' L114/3', ' N114/10', ' N93A/49', ' K28/43', ' N468/37', ' N106/20', ' N28/9', ' K14/16', ' K89/34']",
         "offsetgroup": "['L86A/37', ' N52A/42', ' N398A/34', ' L109/39', ' L114/3', ' N114/10', ' N93A/49', ' K28/43', ' N468/37', ' N106/20', ' N28/9', ' K14/16', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/20', ' K14/16', ' N52A/42', ' K28/43', ' N398A/34', ' N93A/49', ' L86A/37', ' L109/39', ' N114/10', ' N468/37', ' K89/34', ' L114/3', ' N28/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/20', ' K14/16', ' N52A/42', ' K28/43', ' N398A/34', ' N93A/49', ' L86A/37', ' L109/39', ' N114/10', ' N468/37', ' K89/34', ' L114/3', ' N28/9']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/20', ' K14/16', ' N52A/42', ' K28/43', ' N398A/34', ' N93A/49', ' L86A/37', ' L109/39', ' N114/10', ' N468/37', ' K89/34', ' L114/3', ' N28/9']",
         "offsetgroup": "['N106/20', ' K14/16', ' N52A/42', ' K28/43', ' N398A/34', ' N93A/49', ' L86A/37', ' L109/39', ' N114/10', ' N468/37', ' K89/34', ' L114/3', ' N28/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N52A/42', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N52A/42', ' N16B/8']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N52A/42', ' N16B/8']",
         "offsetgroup": "['N52A/42', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K56A/50', ' K20/78', ' N59/36', ' N52A/42', ' N19/2', ' N28/9', ' L21/32', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K56A/50', ' K20/78', ' N59/36', ' N52A/42', ' N19/2', ' N28/9', ' L21/32', ' N18/30']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K56A/50', ' K20/78', ' N59/36', ' N52A/42', ' N19/2', ' N28/9', ' L21/32', ' N18/30']",
         "offsetgroup": "['K56A/50', ' K20/78', ' N59/36', ' N52A/42', ' N19/2', ' N28/9', ' L21/32', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N52A/42', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N52A/42', ' K89/34']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N52A/42', ' K89/34']",
         "offsetgroup": "['N52A/42', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L86/33']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L86/33']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L86/33']",
         "offsetgroup": "['L86/33']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L86A/37', ' N52A/42', ' L83/11', ' K13/31', ' L86/33', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L86A/37', ' N52A/42', ' L83/11', ' K13/31', ' L86/33', ' K89/34']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L86A/37', ' N52A/42', ' L83/11', ' K13/31', ' L86/33', ' K89/34']",
         "offsetgroup": "['L86A/37', ' N52A/42', ' L83/11', ' K13/31', ' L86/33', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L83/11', ' N52A/42', ' L86/33', ' L86A/37', ' K89/34', ' K13/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L83/11', ' N52A/42', ' L86/33', ' L86A/37', ' K89/34', ' K13/31']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L83/11', ' N52A/42', ' L86/33', ' L86A/37', ' K89/34', ' K13/31']",
         "offsetgroup": "['L83/11', ' N52A/42', ' L86/33', ' L86A/37', ' K89/34', ' K13/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N372C/51', ' N52A/42', ' K89/34', ' N372B/60']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N372C/51', ' N52A/42', ' K89/34', ' N372B/60']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N372C/51', ' N52A/42', ' K89/34', ' N372B/60']",
         "offsetgroup": "['N372C/51', ' N52A/42', ' K89/34', ' N372B/60']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K87A/10', ' N95/35', ' N96/55', ' K14/16', ' N87/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K87A/10', ' N95/35', ' N96/55', ' K14/16', ' N87/25']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K87A/10', ' N95/35', ' N96/55', ' K14/16', ' N87/25']",
         "offsetgroup": "['K89/34', ' K87A/10', ' N95/35', ' N96/55', ' K14/16', ' N87/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' N86/38', ' L86A/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' N86/38', ' L86A/37']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' N86/38', ' L86A/37']",
         "offsetgroup": "['K89/34', ' N86/38', ' L86A/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K14/16', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K14/16', ' N16B/8']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K14/16', ' N16B/8']",
         "offsetgroup": "['K89/34', ' K14/16', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K14/16', ' K39/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K14/16', ' K39/25']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K14/16', ' K39/25']",
         "offsetgroup": "['K89/34', ' K14/16', ' K39/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011,
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L86A/37', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L86A/37', ' K89/34']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L86A/37', ' K89/34']",
         "offsetgroup": "['L86A/37', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' L86A/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' L86A/37']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' L86A/37']",
         "offsetgroup": "['K89/34', ' L86A/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' N263/31', ' L109/57']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' N263/31', ' L109/57']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' N263/31', ' L109/57']",
         "offsetgroup": "['K89/34', ' N263/31', ' L109/57']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N206A/8', ' L6/60', ' K89/34', ' K65/35', ' K14/16', ' N147/6', ' K57/1', ' L76/36', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N206A/8', ' L6/60', ' K89/34', ' K65/35', ' K14/16', ' N147/6', ' K57/1', ' L76/36', ' K55/7']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N206A/8', ' L6/60', ' K89/34', ' K65/35', ' K14/16', ' N147/6', ' K57/1', ' L76/36', ' K55/7']",
         "offsetgroup": "['N206A/8', ' L6/60', ' K89/34', ' K65/35', ' K14/16', ' N147/6', ' K57/1', ' L76/36', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K39/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K39/25']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K39/25']",
         "offsetgroup": "['K89/34', ' K39/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' N448/88']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' N448/88']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' N448/88']",
         "offsetgroup": "['K89/34', ' N448/88']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2022,
          2022,
          2022,
          2022,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' N263/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' N263/31']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' N263/31']",
         "offsetgroup": "['K89/34', ' N263/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K75/41', ' L6/60', ' K89/34', ' K57/1', ' K60/73', ' K36/15', ' K66/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K75/41', ' L6/60', ' K89/34', ' K57/1', ' K60/73', ' K36/15', ' K66/38']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K75/41', ' L6/60', ' K89/34', ' K57/1', ' K60/73', ' K36/15', ' K66/38']",
         "offsetgroup": "['K13/31', ' K75/41', ' L6/60', ' K89/34', ' K57/1', ' K60/73', ' K36/15', ' K66/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K69/3']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K69/3']",
         "offsetgroup": "['K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2018,
          2022,
          2010,
          2009,
          2013,
          2019,
          2017,
          2022,
          2015,
          2010,
          2014,
          2017,
          2021,
          2010,
          2010,
          2010,
          2010,
          2010,
          2013,
          2015,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K37/89', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K37/89', ' K89/34']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K37/89', ' K89/34']",
         "offsetgroup": "['K37/89', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021,
          2021,
          2010,
          2012,
          2012,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' N479/22']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' N479/22']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' N479/22']",
         "offsetgroup": "['K89/34', ' N479/22']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K14/16']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K14/16']",
         "offsetgroup": "['K89/34', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' N133/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' N133/21']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' N133/21']",
         "offsetgroup": "['K89/34', ' N133/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K7/45']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K7/45']",
         "offsetgroup": "['K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2008,
          2010,
          2017,
          2017,
          2017,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K39/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K39/25']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K39/25']",
         "offsetgroup": "['K39/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2008,
          2008,
          2008,
          2014,
          2019,
          2015,
          2022,
          2009,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K7/45', ' K39/25', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K7/45', ' K39/25', ' K89/34']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K7/45', ' K39/25', ' K89/34']",
         "offsetgroup": "['K7/45', ' K39/25', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K7/45', ' K89/34', ' K39/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K7/45', ' K89/34', ' K39/25']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K7/45', ' K89/34', ' K39/25']",
         "offsetgroup": "['K7/45', ' K89/34', ' K39/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K75/41', ' L23/27', ' K19/36', ' K89/34', ' K14/16', ' K57/1', ' K36/15', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K75/41', ' L23/27', ' K19/36', ' K89/34', ' K14/16', ' K57/1', ' K36/15', ' K7/45']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K75/41', ' L23/27', ' K19/36', ' K89/34', ' K14/16', ' K57/1', ' K36/15', ' K7/45']",
         "offsetgroup": "['K13/31', ' K75/41', ' L23/27', ' K19/36', ' K89/34', ' K14/16', ' K57/1', ' K36/15', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K75/41', ' K89/34', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K75/41', ' K89/34', ' K57/1']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K75/41', ' K89/34', ' K57/1']",
         "offsetgroup": "['K13/31', ' K75/41', ' K89/34', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N393/2', ' K89/34', ' N106/36', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N393/2', ' K89/34', ' N106/36', ' A12/18']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N393/2', ' K89/34', ' N106/36', ' A12/18']",
         "offsetgroup": "['N393/2', ' K89/34', ' N106/36', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L57/46', ' N38/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L57/46', ' N38/8']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L57/46', ' N38/8']",
         "offsetgroup": "['L57/46', ' N38/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2014,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N458/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N458/10']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N458/10']",
         "offsetgroup": "['N458/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/20', ' N59/36', ' N19/2', ' N308/48', ' N367/62', ' N238/29', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/20', ' N59/36', ' N19/2', ' N308/48', ' N367/62', ' N238/29', ' N18/30']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/20', ' N59/36', ' N19/2', ' N308/48', ' N367/62', ' N238/29', ' N18/30']",
         "offsetgroup": "['N59/20', ' N59/36', ' N19/2', ' N308/48', ' N367/62', ' N238/29', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K9/40', ' K17/70', ' L28/4', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K9/40', ' K17/70', ' L28/4', ' K7/45']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K9/40', ' K17/70', ' L28/4', ' K7/45']",
         "offsetgroup": "['K13/31', ' K9/40', ' K17/70', ' L28/4', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L28/4']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L28/4']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L28/4']",
         "offsetgroup": "['L28/4']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N452/69', ' N229A/32', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N452/69', ' N229A/32', ' N95/35']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N452/69', ' N229A/32', ' N95/35']",
         "offsetgroup": "['N452/69', ' N229A/32', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K56A/50', ' N95/35', ' L120/12', ' L106/83', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K56A/50', ' N95/35', ' L120/12', ' L106/83', ' N59/20']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K56A/50', ' N95/35', ' L120/12', ' L106/83', ' N59/20']",
         "offsetgroup": "['K56A/50', ' N95/35', ' L120/12', ' L106/83', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N97A/31', ' K28/43', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N97A/31', ' K28/43', ' K56A/50']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N97A/31', ' K28/43', ' K56A/50']",
         "offsetgroup": "['N97A/31', ' K28/43', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K56A/50', ' N26A/23', ' N39B/8', ' K57/1', ' N10/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K56A/50', ' N26A/23', ' N39B/8', ' K57/1', ' N10/7']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K56A/50', ' N26A/23', ' N39B/8', ' K57/1', ' N10/7']",
         "offsetgroup": "['K56A/50', ' N26A/23', ' N39B/8', ' K57/1', ' N10/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2009,
          2009,
          2009,
          2009,
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N355/1', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N355/1', ' K56A/50']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N355/1', ' K56A/50']",
         "offsetgroup": "['N355/1', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' K56A/50']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' K56A/50']",
         "offsetgroup": "['K28/43', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019,
          2008,
          2014,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K14/16', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K14/16', ' K56A/50']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K14/16', ' K56A/50']",
         "offsetgroup": "['K14/16', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' K56A/50', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' K56A/50', ' K14/16']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' K56A/50', ' K14/16']",
         "offsetgroup": "['N106/36', ' K56A/50', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N1/12', ' N1/66']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N1/12', ' N1/66']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N1/12', ' N1/66']",
         "offsetgroup": "['N1/12', ' N1/66']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L60/4', ' N28/9', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L60/4', ' N28/9', ' N59/20']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L60/4', ' N28/9', ' N59/20']",
         "offsetgroup": "['L60/4', ' N28/9', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N327/95', ' L21/32', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N327/95', ' L21/32', ' N59/20']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N327/95', ' L21/32', ' N59/20']",
         "offsetgroup": "['N308/48', ' N327/95', ' L21/32', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/20', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/20', ' N95/35']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/20', ' N95/35']",
         "offsetgroup": "['N59/20', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N52A/42', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N52A/42', ' N59/20']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N52A/42', ' N59/20']",
         "offsetgroup": "['N52A/42', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L117/1', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L117/1', ' N59/20']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L117/1', ' N59/20']",
         "offsetgroup": "['L117/1', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' N59/20']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' N59/20']",
         "offsetgroup": "['N59/36', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N327/95', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N327/95', ' N59/20']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N327/95', ' N59/20']",
         "offsetgroup": "['N308/48', ' N327/95', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N59/20']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N59/20']",
         "offsetgroup": "['K28/43', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N367/62', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N367/62', ' N59/20']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N367/62', ' N59/20']",
         "offsetgroup": "['N367/62', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N56/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N56/21']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N56/21']",
         "offsetgroup": "['N56/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2024,
          2022,
          2020,
          2018,
          2022,
          2007,
          2009,
          2013,
          2014,
          2016,
          2016,
          2016,
          2017,
          2017,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N419/40']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N419/40']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N419/40']",
         "offsetgroup": "['N419/40']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N206A/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N206A/8']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N206A/8']",
         "offsetgroup": "['N206A/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2017,
          2019,
          2019,
          2019,
          2020,
          2014,
          2022,
          2016,
          2023,
          2019,
          2017,
          2018,
          2023,
          2018,
          2015,
          2016,
          2020,
          2013,
          2020,
          2020,
          2020,
          2020,
          2023,
          2016,
          2023,
          2024,
          2024,
          2023,
          2023,
          2011,
          2012,
          2012,
          2012,
          2013,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016,
          2016,
          2017,
          2017,
          2018,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2021,
          2020,
          2021,
          2020,
          2020,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2022,
          2022,
          2022,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N415/24', ' N206A/8', ' L118/80', ' L106/83']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N415/24', ' N206A/8', ' L118/80', ' L106/83']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N415/24', ' N206A/8', ' L118/80', ' L106/83']",
         "offsetgroup": "['N415/24', ' N206A/8', ' L118/80', ' L106/83']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N206A/8', ' K56A/50', ' K65/35', ' N16B/8', ' L24/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N206A/8', ' K56A/50', ' K65/35', ' N16B/8', ' L24/18']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N206A/8', ' K56A/50', ' K65/35', ' N16B/8', ' L24/18']",
         "offsetgroup": "['N206A/8', ' K56A/50', ' K65/35', ' N16B/8', ' L24/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' N206A/8', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' N206A/8', ' N106/20']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' N206A/8', ' N106/20']",
         "offsetgroup": "['K87A/10', ' N206A/8', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N206A/8', ' N28/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N206A/8', ' N28/9']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N206A/8', ' N28/9']",
         "offsetgroup": "['N206A/8', ' N28/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N206A/8', ' N147/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N206A/8', ' N147/6']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N206A/8', ' N147/6']",
         "offsetgroup": "['N206A/8', ' N147/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' L23/27', ' K14/16', ' K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' L23/27', ' K14/16', ' K19/36']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' L23/27', ' K14/16', ' K19/36']",
         "offsetgroup": "['K20/78', ' L23/27', ' K14/16', ' K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L23/27', ' K14/16', ' K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L23/27', ' K14/16', ' K19/36']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L23/27', ' K14/16', ' K19/36']",
         "offsetgroup": "['L23/27', ' K14/16', ' K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K19/36']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K19/36']",
         "offsetgroup": "['K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K19/36']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K19/36']",
         "offsetgroup": "['K13/31', ' K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K19/36', ' K13/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K19/36', ' K13/31']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K19/36', ' K13/31']",
         "offsetgroup": "['K19/36', ' K13/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K13/31', ' K14/16', ' K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K13/31', ' K14/16', ' K19/36']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K13/31', ' K14/16', ' K19/36']",
         "offsetgroup": "['K20/78', ' K13/31', ' K14/16', ' K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K13/31', ' K14/16', ' K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K13/31', ' K14/16', ' K19/36']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K13/31', ' K14/16', ' K19/36']",
         "offsetgroup": "['K87A/10', ' K13/31', ' K14/16', ' K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48']",
         "offsetgroup": "['N308/48']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2018,
          2016,
          2021,
          2021,
          2021,
          2021,
          2012,
          2021,
          2015,
          2014,
          2013,
          2013,
          2014,
          2015,
          2016,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2021,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95']",
         "offsetgroup": "['N327/95']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2017,
          2022,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2023,
          2023,
          2013,
          2016,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N59/36', ' N308/48']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N59/36', ' N308/48']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N59/36', ' N308/48']",
         "offsetgroup": "['N327/95', ' N59/36', ' N308/48']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' N327/95', ' N308/48']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' N327/95', ' N308/48']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' N327/95', ' N308/48']",
         "offsetgroup": "['N59/36', ' N327/95', ' N308/48']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N59/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N59/36']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N59/36']",
         "offsetgroup": "['N308/48', ' N59/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2013,
          2014,
          2015,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/20', ' N59/36', ' N19/2', ' K28/86', ' N308/48', ' L21/32', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/20', ' N59/36', ' N19/2', ' K28/86', ' N308/48', ' L21/32', ' N18/30']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/20', ' N59/36', ' N19/2', ' K28/86', ' N308/48', ' L21/32', ' N18/30']",
         "offsetgroup": "['N59/20', ' N59/36', ' N19/2', ' K28/86', ' N308/48', ' L21/32', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' L21/32']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' L21/32']",
         "offsetgroup": "['N308/48', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N207/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N207/27']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N207/27']",
         "offsetgroup": "['N308/48', ' N207/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N355/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N355/1']",
         "offsetgroup": "['N308/48', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N312/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N312/10']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N312/10']",
         "offsetgroup": "['N312/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2017,
          2021,
          2021,
          2020,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' K73/20', ' N244/5', ' K65/35', ' N86/38', ' K14/16', ' K87A/10', ' N105/13', ' N105/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' K73/20', ' N244/5', ' K65/35', ' N86/38', ' K14/16', ' K87A/10', ' N105/13', ' N105/17']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' K73/20', ' N244/5', ' K65/35', ' N86/38', ' K14/16', ' K87A/10', ' N105/13', ' N105/17']",
         "offsetgroup": "['N106/36', ' K73/20', ' N244/5', ' K65/35', ' N86/38', ' K14/16', ' K87A/10', ' N105/13', ' N105/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' N106/36']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' N106/36']",
         "offsetgroup": "['K65/35', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2022,
          2022,
          2022,
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N100/13']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N100/13']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N100/13']",
         "offsetgroup": "['N100/13']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2019,
          2022,
          2012,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N100/13']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N100/13']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N100/13']",
         "offsetgroup": "['N106/36', ' N100/13']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N56/21', ' L11A/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N56/21', ' L11A/41']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N56/21', ' L11A/41']",
         "offsetgroup": "['N56/21', ' L11A/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10']",
         "offsetgroup": "['K87A/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2024,
          2024,
          2022,
          2010,
          2010,
          2010,
          2013,
          2019,
          2010,
          2016,
          2015,
          2016,
          2020,
          2020,
          2006,
          2009,
          2011,
          2011,
          2012,
          2016,
          2017,
          2017,
          2017,
          2019,
          2019,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15', ' N106/36', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15', ' N106/36', ' K14/16']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15', ' N106/36', ' K14/16']",
         "offsetgroup": "['K36/15', ' N106/36', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35']",
         "offsetgroup": "['K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2012,
          2012,
          2012,
          2016,
          2024,
          2024,
          2024,
          2016,
          2010,
          2024,
          2021,
          2021,
          2021,
          2011,
          2019,
          2017,
          2017,
          2010,
          2021,
          2016,
          2009,
          2009,
          2009,
          2009,
          2020,
          2021,
          2013,
          2020,
          2020,
          2021,
          2021,
          2017,
          2022,
          2021,
          2024,
          2019,
          2014,
          2020,
          2022,
          2020,
          2022,
          2022,
          2022,
          2018,
          2013,
          2022,
          2020,
          2008,
          2008,
          2008,
          2008,
          2009,
          2010,
          2010,
          2011,
          2012,
          2012,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2018,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2020,
          2018,
          2019,
          2019,
          2019,
          2020,
          2020,
          2022,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L11A/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L11A/41']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L11A/41']",
         "offsetgroup": "['L11A/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2023,
          2017,
          2012,
          2012,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' L11A/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' L11A/41']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' L11A/41']",
         "offsetgroup": "['N106/36', ' L11A/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K36/15', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K36/15', ' N106/36']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K36/15', ' N106/36']",
         "offsetgroup": "['K20/78', ' K36/15', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' N106/36', ' N105/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' N106/36', ' N105/17']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' N106/36', ' N105/17']",
         "offsetgroup": "['K20/78', ' N106/36', ' N105/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['A12/18']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['A12/18']",
         "offsetgroup": "['A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015,
          2022,
          2019,
          2019,
          2019,
          2019,
          2015,
          2015,
          2015,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2015,
          2015,
          2015,
          2015,
          2015,
          2023,
          2016,
          2016,
          2016,
          2016,
          2022,
          2022,
          2010,
          2010,
          2010,
          2016,
          2014,
          2019,
          2020,
          2016,
          2015,
          2017,
          2017,
          2021,
          2015,
          2016,
          2017,
          2016,
          2019,
          2020,
          2022,
          2015,
          2017,
          2015,
          2014,
          2014,
          2016,
          2016,
          2018,
          2019,
          2019,
          2020,
          2020,
          2020,
          2021,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N18/28', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N18/28', ' N106/36']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N18/28', ' N106/36']",
         "offsetgroup": "['N18/28', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' K28/43']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' K28/43']",
         "offsetgroup": "['N106/36', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N393/76']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N393/76']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N393/76']",
         "offsetgroup": "['N393/76']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2021,
          2021,
          2021,
          2022,
          2022,
          2022,
          2023,
          2023,
          2018,
          2016,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' A12/18', ' N393/76']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' A12/18', ' N393/76']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' A12/18', ' N393/76']",
         "offsetgroup": "['N106/36', ' A12/18', ' N393/76']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N393/76', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N393/76', ' A12/18']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N393/76', ' A12/18']",
         "offsetgroup": "['N106/36', ' N393/76', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N56/21', ' N86/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N56/21', ' N86/38']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N56/21', ' N86/38']",
         "offsetgroup": "['N56/21', ' N86/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N100/13', ' K65/35', ' N86/38', ' N105/13', ' N105/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N100/13', ' K65/35', ' N86/38', ' N105/13', ' N105/17']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N100/13', ' K65/35', ' N86/38', ' N105/13', ' N105/17']",
         "offsetgroup": "['N106/36', ' N100/13', ' K65/35', ' N86/38', ' N105/13', ' N105/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' K87A/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' K87A/10']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' K87A/10']",
         "offsetgroup": "['N106/36', ' K87A/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N106/36', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N106/36', ' N106/20']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N106/36', ' N106/20']",
         "offsetgroup": "['K28/43', ' N106/36', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N112B/14']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N112B/14']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N112B/14']",
         "offsetgroup": "['N112B/14']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2021,
          2017,
          2011,
          2011,
          2012,
          2013,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' N56/21', ' N70/28', ' N155/9', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' N56/21', ' N70/28', ' N155/9', ' N106/20']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' N56/21', ' N70/28', ' N155/9', ' N106/20']",
         "offsetgroup": "['K74/71', ' N56/21', ' N70/28', ' N155/9', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' A12/18', ' K65/35', ' N106/20', ' N393/76']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' A12/18', ' K65/35', ' N106/20', ' N393/76']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' A12/18', ' K65/35', ' N106/20', ' N393/76']",
         "offsetgroup": "['N106/36', ' A12/18', ' K65/35', ' N106/20', ' N393/76']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N134/12']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N134/12']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N134/12']",
         "offsetgroup": "['N134/12']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2024,
          2023,
          2023,
          2018,
          2017,
          2022,
          2020,
          2022,
          2013,
          2010,
          2011,
          2011,
          2012,
          2013,
          2013,
          2014,
          2015,
          2015,
          2019,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' N106/36', ' K20/78', ' K17/70', ' K14/16', ' K36/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' N106/36', ' K20/78', ' K17/70', ' K14/16', ' K36/15']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' N106/36', ' K20/78', ' K17/70', ' K14/16', ' K36/15']",
         "offsetgroup": "['K13/31', ' N106/36', ' K20/78', ' K17/70', ' K14/16', ' K36/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L88/86']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L88/86']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L88/86']",
         "offsetgroup": "['L88/86']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' L88/86']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' L88/86']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' L88/86']",
         "offsetgroup": "['N106/36', ' L88/86']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/65']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/65']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/65']",
         "offsetgroup": "['N106/65']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2023,
          2023,
          2024,
          2014,
          2011,
          2021,
          2020,
          2015,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N106/65']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N106/65']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N106/65']",
         "offsetgroup": "['N106/36', ' N106/65']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K36/15', ' K14/16', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K36/15', ' K14/16', ' N18/30']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K36/15', ' K14/16', ' N18/30']",
         "offsetgroup": "['K20/78', ' K36/15', ' K14/16', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' A12/18']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' A12/18']",
         "offsetgroup": "['N106/36', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2023,
          2015,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N395/68', ' K14/16', ' N168/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N395/68', ' K14/16', ' N168/6']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N395/68', ' K14/16', ' N168/6']",
         "offsetgroup": "['N395/68', ' K14/16', ' N168/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N319A/14']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N319A/14']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N319A/14']",
         "offsetgroup": "['N319A/14']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' N56/21', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' N56/21', ' N106/36']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' N56/21', ' N106/36']",
         "offsetgroup": "['K87A/10', ' N56/21', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N105/17', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N105/17', ' K14/16']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N105/17', ' K14/16']",
         "offsetgroup": "['N105/17', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K13/31', ' K14/16', ' N46/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K13/31', ' K14/16', ' N46/30']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K13/31', ' K14/16', ' N46/30']",
         "offsetgroup": "['K20/78', ' K13/31', ' K14/16', ' N46/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15']",
         "offsetgroup": "['K36/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2024,
          2024,
          2023,
          2023,
          2022,
          2014,
          2015,
          2008,
          2019,
          2010,
          2011,
          2012,
          2014,
          2014,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N114/10', ' K36/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N114/10', ' K36/15']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N114/10', ' K36/15']",
         "offsetgroup": "['N106/36', ' N114/10', ' K36/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L11A/41', ' N105/17', ' A12/18', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L11A/41', ' N105/17', ' A12/18', ' N106/20']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L11A/41', ' N105/17', ' A12/18', ' N106/20']",
         "offsetgroup": "['L11A/41', ' N105/17', ' A12/18', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' K69/3']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' K69/3']",
         "offsetgroup": "['N106/36', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15', ' N106/36']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15', ' N106/36']",
         "offsetgroup": "['K36/15', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N87/25', ' N452/69']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N87/25', ' N452/69']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N87/25', ' N452/69']",
         "offsetgroup": "['N87/25', ' N452/69']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N170A/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N170A/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N170A/1']",
         "offsetgroup": "['N170A/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2023,
          2023,
          2023,
          2018,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N170A/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N170A/1']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N170A/1']",
         "offsetgroup": "['N106/36', ' N170A/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N395/68', ' N168/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N395/68', ' N168/6']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N395/68', ' N168/6']",
         "offsetgroup": "['N395/68', ' N168/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N395/68', ' K65/35', ' K14/16', ' N168/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N395/68', ' K65/35', ' K14/16', ' N168/6']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N395/68', ' K65/35', ' K14/16', ' N168/6']",
         "offsetgroup": "['N395/68', ' K65/35', ' K14/16', ' N168/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N105/13', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N105/13', ' N106/20']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N105/13', ' N106/20']",
         "offsetgroup": "['N105/13', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N68/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N68/6']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N68/6']",
         "offsetgroup": "['N68/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2013,
          2016,
          2012,
          2015,
          2022,
          2024,
          2010,
          2011,
          2012,
          2012,
          2012,
          2012,
          2013,
          2013,
          2013,
          2016,
          2018,
          2019,
          2019,
          2019,
          2021,
          2022,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/74']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/74']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/74']",
         "offsetgroup": "['K28/74']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2023,
          2025,
          2023,
          2015,
          2022,
          2022,
          2014,
          2017,
          2018,
          2018,
          2019,
          2019,
          2020,
          2020,
          2022,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' K28/74']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' K28/74']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' K28/74']",
         "offsetgroup": "['N327/95', ' K28/74']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N59/36', ' K28/43', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N59/36', ' K28/43', ' N355/1']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N59/36', ' K28/43', ' N355/1']",
         "offsetgroup": "['N327/95', ' N59/36', ' K28/43', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2022,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N75/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N75/3']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N75/3']",
         "offsetgroup": "['N75/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N367/62', ' N28/9', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N367/62', ' N28/9', ' N23B/6']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N367/62', ' N28/9', ' N23B/6']",
         "offsetgroup": "['N367/62', ' N28/9', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N355/1']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N355/1']",
         "offsetgroup": "['N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2024,
          2024,
          2024,
          2023,
          2023,
          2018,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2020,
          2018,
          2018,
          2019,
          2019,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N367/62', ' K28/43', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N367/62', ' K28/43', ' N355/1']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N367/62', ' K28/43', ' N355/1']",
         "offsetgroup": "['N367/62', ' K28/43', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N105/17', ' N367/62']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N105/17', ' N367/62']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N105/17', ' N367/62']",
         "offsetgroup": "['N105/17', ' N367/62']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N71/37', ' N114/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N71/37', ' N114/10']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N71/37', ' N114/10']",
         "offsetgroup": "['N71/37', ' N114/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N71/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N71/37']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N71/37']",
         "offsetgroup": "['N71/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2017,
          2011,
          2022,
          2009,
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' N71/37', ' N141/28', ' N114/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' N71/37', ' N141/28', ' N114/10']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' N71/37', ' N141/28', ' N114/10']",
         "offsetgroup": "['N70/28', ' N71/37', ' N141/28', ' N114/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012,
          2012,
          2021,
          2010,
          2011,
          2012,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' L6/60', ' N26A/23', ' N71/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' L6/60', ' N26A/23', ' N71/37']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' L6/60', ' N26A/23', ' N71/37']",
         "offsetgroup": "['K13/31', ' L6/60', ' N26A/23', ' N71/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2008,
          2008,
          2008,
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' N28/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' N28/9']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' N28/9']",
         "offsetgroup": "['N59/36', ' N28/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/74', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/74', ' N95/35']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/74', ' N95/35']",
         "offsetgroup": "['K28/74', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/38', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/38', ' N95/35']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/38', ' N95/35']",
         "offsetgroup": "['N86/38', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N95/35', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N95/35', ' K28/43']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N95/35', ' K28/43']",
         "offsetgroup": "['N95/35', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N398A/34', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N398A/34', ' N95/35']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N398A/34', ' N95/35']",
         "offsetgroup": "['N398A/34', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2018,
          2018,
          2018,
          2016,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N186/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N186/29']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N186/29']",
         "offsetgroup": "['N186/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N295B/66', ' N186/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N295B/66', ' N186/29']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N295B/66', ' N186/29']",
         "offsetgroup": "['N295B/66', ' N186/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L106/93']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L106/93']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L106/93']",
         "offsetgroup": "['L106/93']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L106/4']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L106/4']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L106/4']",
         "offsetgroup": "['L106/4']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L113/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L113/27']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L113/27']",
         "offsetgroup": "['L113/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L113/130']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L113/130']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L113/130']",
         "offsetgroup": "['L113/130']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L113/13']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L113/13']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L113/13']",
         "offsetgroup": "['L113/13']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/86']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/86']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/86']",
         "offsetgroup": "['K28/86']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2024,
          2023,
          2021,
          2017,
          2014,
          2017,
          2018,
          2016,
          2012,
          2020,
          2020,
          2021,
          2018,
          2022,
          2021,
          2013,
          2018,
          2019,
          2008,
          2010,
          2011,
          2014,
          2015,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016,
          2017,
          2018,
          2018,
          2018,
          2019,
          2022,
          2023,
          2024,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L86A/37', ' L113/13', ' L106/93', ' L113/27', ' L113/130', ' K28/43', ' K28/86', ' L106/4', ' K28/74']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L86A/37', ' L113/13', ' L106/93', ' L113/27', ' L113/130', ' K28/43', ' K28/86', ' L106/4', ' K28/74']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L86A/37', ' L113/13', ' L106/93', ' L113/27', ' L113/130', ' K28/43', ' K28/86', ' L106/4', ' K28/74']",
         "offsetgroup": "['L86A/37', ' L113/13', ' L106/93', ' L113/27', ' L113/130', ' K28/43', ' K28/86', ' L106/4', ' K28/74']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N414/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N414/25']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N414/25']",
         "offsetgroup": "['N414/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2017,
          2017,
          2017,
          2021,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N97A/31', ' L21/32', ' N110/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N97A/31', ' L21/32', ' N110/29']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N97A/31', ' L21/32', ' N110/29']",
         "offsetgroup": "['N97A/31', ' L21/32', ' N110/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N138/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N138/6']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N138/6']",
         "offsetgroup": "['N138/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2024,
          2023,
          2012,
          2012,
          2013,
          2016,
          2012,
          2014,
          2017,
          2020,
          2013,
          2013,
          2009,
          2014,
          2014,
          2014,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N241A/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N241A/34']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N241A/34']",
         "offsetgroup": "['N241A/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2017,
          2020,
          2014,
          2022,
          2016,
          2017,
          2020,
          2019,
          2019,
          2020,
          2014,
          2021,
          2014,
          2019,
          2015,
          2022,
          2022,
          2022,
          2022,
          2021,
          2015,
          2017,
          2019,
          2020,
          2020,
          2017,
          2018,
          2021,
          2020,
          2018,
          2022,
          2019,
          2018,
          2023,
          2020,
          2021,
          2021,
          2020,
          2020,
          2020,
          2020,
          2021,
          2018,
          2017,
          2020,
          2015,
          2021,
          2019,
          2018,
          2022,
          2021,
          2024,
          2023,
          2013,
          2014,
          2014,
          2014,
          2015,
          2015,
          2015,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2018,
          2017,
          2018,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2020,
          2020,
          2020,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N138/6', ' N241A/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N138/6', ' N241A/34']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N138/6', ' N241A/34']",
         "offsetgroup": "['N138/6', ' N241A/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2012,
          2012,
          2012,
          2019,
          2019,
          2019,
          2021,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L57/46']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L57/46']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L57/46']",
         "offsetgroup": "['L57/46']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2021,
          2010,
          2010,
          2012,
          2016,
          2016,
          2012,
          2008,
          2016,
          2011,
          2011,
          2017,
          2014,
          2009,
          2009,
          2010,
          2010,
          2011,
          2011,
          2012,
          2012,
          2013,
          2014,
          2014,
          2014,
          2015,
          2015,
          2016,
          2016,
          2018,
          2019,
          2018,
          2019,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L57/46', ' N263/31', ' N134/12']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L57/46', ' N263/31', ' N134/12']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L57/46', ' N263/31', ' N134/12']",
         "offsetgroup": "['L57/46', ' N263/31', ' N134/12']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N55/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N55/10']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N55/10']",
         "offsetgroup": "['N55/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2016,
          2014,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N393/76', ' N106/65']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N393/76', ' N106/65']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N393/76', ' N106/65']",
         "offsetgroup": "['N393/76', ' N106/65']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' N393/76']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' N393/76']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' N393/76']",
         "offsetgroup": "['K65/35', ' N393/76']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N225A/10', ' N235/22']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N225A/10', ' N235/22']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N225A/10', ' N235/22']",
         "offsetgroup": "['N225A/10', ' N235/22']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N229A/32', ' N95/35', ' N28/9', ' L118/80', ' L106/83', ' N415/24', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N229A/32', ' N95/35', ' N28/9', ' L118/80', ' L106/83', ' N415/24', ' N355/1']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N229A/32', ' N95/35', ' N28/9', ' L118/80', ' L106/83', ' N415/24', ' N355/1']",
         "offsetgroup": "['N229A/32', ' N95/35', ' N28/9', ' L118/80', ' L106/83', ' N415/24', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N229A/32', ' N95/35', ' L106/83', ' N415/24', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N229A/32', ' N95/35', ' L106/83', ' N415/24', ' N355/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N229A/32', ' N95/35', ' L106/83', ' N415/24', ' N355/1']",
         "offsetgroup": "['N229A/32', ' N95/35', ' L106/83', ' N415/24', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N95/35', ' L118/80', ' L106/83', ' N415/24', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N95/35', ' L118/80', ' L106/83', ' N415/24', ' N355/1']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N95/35', ' L118/80', ' L106/83', ' N415/24', ' N355/1']",
         "offsetgroup": "['N95/35', ' L118/80', ' L106/83', ' N415/24', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N7/18', ' N263/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N7/18', ' N263/31']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N7/18', ' N263/31']",
         "offsetgroup": "['N7/18', ' N263/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N263/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N263/31']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N263/31']",
         "offsetgroup": "['N263/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2011,
          2013,
          2014,
          2014,
          2014,
          2016,
          2019,
          2020,
          2020,
          2021,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N263/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N263/31']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N263/31']",
         "offsetgroup": "['K28/43', ' N263/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N144/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N144/32']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N144/32']",
         "offsetgroup": "['N144/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2019,
          2016,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N148/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N148/30']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N148/30']",
         "offsetgroup": "['N148/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N174B/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N174B/27']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N174B/27']",
         "offsetgroup": "['N174B/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2019,
          2019,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N159/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N159/5']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N159/5']",
         "offsetgroup": "['N159/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2019,
          2016,
          2015,
          2017,
          2020,
          2018,
          2011,
          2009,
          2010,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N419/78']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N419/78']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N419/78']",
         "offsetgroup": "['N419/78']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N7/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N7/18']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N7/18']",
         "offsetgroup": "['N7/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2022,
          2018,
          2018,
          2014,
          2008,
          2009,
          2014,
          2017,
          2018,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N8B/1', ' N10/7', ' N7/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N8B/1', ' N10/7', ' N7/18']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N8B/1', ' N10/7', ' N7/18']",
         "offsetgroup": "['N8B/1', ' N10/7', ' N7/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N7/18', ' N10/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N7/18', ' N10/7']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N7/18', ' N10/7']",
         "offsetgroup": "['N7/18', ' N10/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2011,
          2011,
          2010,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N8B/1', ' N10/7', ' K28/43', ' N7/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N8B/1', ' N10/7', ' K28/43', ' N7/18']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N8B/1', ' N10/7', ' K28/43', ' N7/18']",
         "offsetgroup": "['N8B/1', ' N10/7', ' K28/43', ' N7/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' N86/8', ' N144/14', ' L114/81', ' K14/16', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' N86/8', ' N144/14', ' L114/81', ' K14/16', ' N16B/8']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' N86/8', ' N144/14', ' L114/81', ' K14/16', ' N16B/8']",
         "offsetgroup": "['K74/71', ' N86/8', ' N144/14', ' L114/81', ' K14/16', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L114/81']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L114/81']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L114/81']",
         "offsetgroup": "['L114/81']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L109/57']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L109/57']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L109/57']",
         "offsetgroup": "['L109/57']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2024,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N206B/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N206B/9']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N206B/9']",
         "offsetgroup": "['N206B/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L115/13']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L115/13']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L115/13']",
         "offsetgroup": "['L115/13']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/38', ' L114/81', ' K28/43', ' N206B/9', ' N28/9', ' K14/16', ' L109/57', ' L115/13']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/38', ' L114/81', ' K28/43', ' N206B/9', ' N28/9', ' K14/16', ' L109/57', ' L115/13']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/38', ' L114/81', ' K28/43', ' N206B/9', ' N28/9', ' K14/16', ' L109/57', ' L115/13']",
         "offsetgroup": "['N86/38', ' L114/81', ' K28/43', ' N206B/9', ' N28/9', ' K14/16', ' L109/57', ' L115/13']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N148/30', ' N159/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N148/30', ' N159/5']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N148/30', ' N159/5']",
         "offsetgroup": "['N148/30', ' N159/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N144/32', ' N148/30', ' N159/5', ' N174B/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N144/32', ' N148/30', ' N159/5', ' N174B/27']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N144/32', ' N148/30', ' N159/5', ' N174B/27']",
         "offsetgroup": "['N144/32', ' N148/30', ' N159/5', ' N174B/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N87/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N87/25']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N87/25']",
         "offsetgroup": "['N87/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2019,
          2012,
          2012,
          2014,
          2017,
          2009,
          2010,
          2010,
          2010,
          2011,
          2012,
          2012,
          2012,
          2012,
          2013,
          2013,
          2014,
          2015,
          2015,
          2017,
          2016,
          2016,
          2019,
          2020,
          2021,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N479/107']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N479/107']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N479/107']",
         "offsetgroup": "['N479/107']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' K28/43', ' N38/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' K28/43', ' N38/8']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' K28/43', ' N38/8']",
         "offsetgroup": "['N86/8', ' K28/43', ' N38/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N183/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N183/15']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N183/15']",
         "offsetgroup": "['N183/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N415/24', ' 62/3G1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N415/24', ' 62/3G1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N415/24', ' 62/3G1']",
         "offsetgroup": "['N415/24', ' 62/3G1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N372B/60']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N372B/60']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N372B/60']",
         "offsetgroup": "['N372B/60']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N255/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N255/38']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N255/38']",
         "offsetgroup": "['N255/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2024,
          2024,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K14/16', ' N134/12', ' N372B/60', ' N68/6', ' K89/34', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K14/16', ' N134/12', ' N372B/60', ' N68/6', ' K89/34', ' K65/35']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K14/16', ' N134/12', ' N372B/60', ' N68/6', ' K89/34', ' K65/35']",
         "offsetgroup": "['K14/16', ' N134/12', ' N372B/60', ' N68/6', ' K89/34', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N372B/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N372B/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N372B/1']",
         "offsetgroup": "['N372B/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71']",
         "offsetgroup": "['K74/71']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2023,
          2021,
          2022,
          2023,
          2022,
          2008,
          2011,
          2011,
          2013,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' K14/16', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' K14/16', ' N16B/8']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' K14/16', ' N16B/8']",
         "offsetgroup": "['N106/36', ' K14/16', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N207/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N207/27']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N207/27']",
         "offsetgroup": "['N207/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2021,
          2021,
          2021,
          2013,
          2021,
          2020,
          2012,
          2014,
          2016,
          2017,
          2019,
          2020,
          2020,
          2021,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N10/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N10/7']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N10/7']",
         "offsetgroup": "['N10/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2009,
          2011,
          2012,
          2013,
          2014,
          2014,
          2014,
          2017,
          2019,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N178A/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N178A/9']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N178A/9']",
         "offsetgroup": "['N178A/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2021,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N212A/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N212A/34']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N212A/34']",
         "offsetgroup": "['N212A/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N212/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N212/3']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N212/3']",
         "offsetgroup": "['N212/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N18/28']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N18/28']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N18/28']",
         "offsetgroup": "['N18/28']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78']",
         "offsetgroup": "['K20/78']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2016,
          2023,
          2017,
          2020,
          2020,
          2013,
          2015,
          2017,
          2014,
          2009,
          2009,
          2010,
          2010,
          2010,
          2012,
          2014,
          2020,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K67/25', ' K36/15', ' K13/31', ' L23/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K67/25', ' K36/15', ' K13/31', ' L23/27']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K67/25', ' K36/15', ' K13/31', ' L23/27']",
         "offsetgroup": "['K67/25', ' K36/15', ' K13/31', ' L23/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N415/24', ' N419/40', ' N206A/8', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N415/24', ' N419/40', ' N206A/8', ' N95/35']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N415/24', ' N419/40', ' N206A/8', ' N95/35']",
         "offsetgroup": "['N415/24', ' N419/40', ' N206A/8', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N415/24']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N415/24']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N415/24']",
         "offsetgroup": "['N415/24']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L124/59']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L124/59']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L124/59']",
         "offsetgroup": "['L124/59']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L21/32']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L21/32']",
         "offsetgroup": "['L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2024,
          2024,
          2024,
          2019,
          2014,
          2016,
          2010,
          2010,
          2010,
          2016,
          2009,
          2012,
          2011,
          2010,
          2014,
          2014,
          2020,
          2020,
          2020,
          2018,
          2014,
          2014,
          2013,
          2013,
          2010,
          2021,
          2017,
          2019,
          2008,
          2022,
          2009,
          2010,
          2011,
          2011,
          2012,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2015,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2021,
          2021,
          2021,
          2022,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K73/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K73/20']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K73/20']",
         "offsetgroup": "['K73/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2006,
          2011,
          2013,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/60', ' N53/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/60', ' N53/32']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/60', ' N53/32']",
         "offsetgroup": "['L6/60', ' N53/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/60', ' L18A/3', ' N53/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/60', ' L18A/3', ' N53/32']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/60', ' L18A/3', ' N53/32']",
         "offsetgroup": "['L6/60', ' L18A/3', ' N53/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012,
          2012,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' K28/74']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' K28/74']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' K28/74']",
         "offsetgroup": "['K28/43', ' K28/74']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N355/1']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N355/1']",
         "offsetgroup": "['K28/43', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N235/22', ' N91/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N235/22', ' N91/27']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N235/22', ' N91/27']",
         "offsetgroup": "['N235/22', ' N91/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K72/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K72/8']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K72/8']",
         "offsetgroup": "['K72/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2016,
          2014,
          2014,
          2015,
          2018,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N388A/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N388A/10']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N388A/10']",
         "offsetgroup": "['N388A/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N65/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N65/37']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N65/37']",
         "offsetgroup": "['N65/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/43']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/43']",
         "offsetgroup": "['N106/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N105/13']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N105/13']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N105/13']",
         "offsetgroup": "['N105/13']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L36/12']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L36/12']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L36/12']",
         "offsetgroup": "['L36/12']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N1/12']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N1/12']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N1/12']",
         "offsetgroup": "['N1/12']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2024,
          2009,
          2018,
          2022,
          2020,
          2013,
          2012,
          2014,
          2007,
          2010,
          2020,
          2019,
          2020,
          2018,
          2017,
          2017,
          2016,
          2012,
          2012,
          2014,
          2014,
          2015,
          2017,
          2017,
          2017,
          2018,
          2019,
          2019,
          2019,
          2020,
          2021,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N76/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N76/3']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N76/3']",
         "offsetgroup": "['N76/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2018,
          2018,
          2024,
          2010,
          2013,
          2013,
          2014,
          2017,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N3/26']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N3/26']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N3/26']",
         "offsetgroup": "['N3/26']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2022,
          2012,
          2013,
          2022,
          2023,
          2022,
          2010,
          2015,
          2016,
          2017,
          2017,
          2019,
          2021,
          2021,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N421A/85']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N421A/85']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N421A/85']",
         "offsetgroup": "['N421A/85']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2022,
          2022,
          2021,
          2016,
          2023,
          2023,
          2021,
          2021,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K20/78', ' K17/70', ' N70/28', ' K14/16', ' N46/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K20/78', ' K17/70', ' N70/28', ' K14/16', ' N46/30']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K20/78', ' K17/70', ' N70/28', ' K14/16', ' N46/30']",
         "offsetgroup": "['K13/31', ' K20/78', ' K17/70', ' N70/28', ' K14/16', ' N46/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N57/2', ' K69/3', ' A12/18', ' N46/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N57/2', ' K69/3', ' A12/18', ' N46/30']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N57/2', ' K69/3', ' A12/18', ' N46/30']",
         "offsetgroup": "['N57/2', ' K69/3', ' A12/18', ' N46/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N46/30', ' K14/16', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N46/30', ' K14/16', ' K57/1']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N46/30', ' K14/16', ' K57/1']",
         "offsetgroup": "['N46/30', ' K14/16', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N67/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N67/15']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N67/15']",
         "offsetgroup": "['N67/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2010,
          2010,
          2010,
          2013,
          2022,
          2022,
          2012,
          2022,
          2012,
          2023,
          2014,
          2023,
          2008,
          2009,
          2010,
          2010,
          2011,
          2011,
          2011,
          2012,
          2012,
          2014,
          2018,
          2018,
          2019,
          2019,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L60/4']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L60/4']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L60/4']",
         "offsetgroup": "['L60/4']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2012,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N56/21', ' N235/22']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N56/21', ' N235/22']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N56/21', ' N235/22']",
         "offsetgroup": "['N56/21', ' N235/22']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N56/21', ' N235/22', ' K74/71', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N56/21', ' N235/22', ' K74/71', ' K69/3']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N56/21', ' N235/22', ' K74/71', ' K69/3']",
         "offsetgroup": "['N56/21', ' N235/22', ' K74/71', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N167/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N167/7']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N167/7']",
         "offsetgroup": "['N167/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L48A/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L48A/9']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L48A/9']",
         "offsetgroup": "['L48A/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' N110/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' N110/29']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' N110/29']",
         "offsetgroup": "['N28/9', ' N110/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2023,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N133/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N133/21']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N133/21']",
         "offsetgroup": "['N133/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2023,
          2023,
          2023,
          2019,
          2014,
          2019,
          2022,
          2021,
          2010,
          2011,
          2013,
          2013,
          2014,
          2014,
          2015,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2017,
          2018,
          2019,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2020,
          2021,
          2021,
          2022,
          2022,
          2022,
          2023,
          2022,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N133/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N133/35']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N133/35']",
         "offsetgroup": "['N133/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N244/5', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N244/5', ' N106/36']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N244/5', ' N106/36']",
         "offsetgroup": "['N244/5', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N244/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N244/5']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N244/5']",
         "offsetgroup": "['N244/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2013,
          2013,
          2014,
          2023,
          2013,
          2015,
          2016,
          2017,
          2019,
          2018,
          2019,
          2019,
          2021,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N244/5', ' N106/65']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N244/5', ' N106/65']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N244/5', ' N106/65']",
         "offsetgroup": "['N244/5', ' N106/65']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N332B/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N332B/15']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N332B/15']",
         "offsetgroup": "['N332B/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2023,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L43/40']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L43/40']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L43/40']",
         "offsetgroup": "['L43/40']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' K65/35']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' K65/35']",
         "offsetgroup": "['N70/28', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L11A/41', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L11A/41', ' K65/35']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L11A/41', ' K65/35']",
         "offsetgroup": "['L11A/41', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K65/35']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K65/35']",
         "offsetgroup": "['K87A/10', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2009,
          2009,
          2016,
          2016,
          2016,
          2006,
          2008,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15', ' K65/35']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15', ' K65/35']",
         "offsetgroup": "['K36/15', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K65/35']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K65/35']",
         "offsetgroup": "['K20/78', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K73/20', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K73/20', ' K65/35']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K73/20', ' K65/35']",
         "offsetgroup": "['K73/20', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2009,
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' K28/43']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' K28/43']",
         "offsetgroup": "['K65/35', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' N59/20']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' N59/20']",
         "offsetgroup": "['K65/35', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L127/12']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L127/12']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L127/12']",
         "offsetgroup": "['L127/12']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N147/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N147/6']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N147/6']",
         "offsetgroup": "['N147/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2013,
          2013,
          2020,
          2021,
          2018,
          2019,
          2011,
          2016,
          2016,
          2021,
          2020,
          2021,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K67/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K67/25']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K67/25']",
         "offsetgroup": "['K67/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2023,
          2009,
          2016,
          2016,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/86', ' N19/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/86', ' N19/2']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/86', ' N19/2']",
         "offsetgroup": "['K28/86', ' N19/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K66/38', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K66/38', ' K55/7']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K66/38', ' K55/7']",
         "offsetgroup": "['K75/41', ' K66/38', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N81/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N81/2']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N81/2']",
         "offsetgroup": "['N81/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2010,
          2017,
          2017,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/60']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/60']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/60']",
         "offsetgroup": "['L6/60']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2023,
          2023,
          2024,
          2009,
          2009,
          2009,
          2012,
          2010,
          2009,
          2012,
          2012,
          2014,
          2014,
          2022,
          2015,
          2009,
          2014,
          2012,
          2019,
          2012,
          2016,
          2013,
          2021,
          2013,
          2013,
          2008,
          2010,
          2015,
          2018,
          2014,
          2014,
          2010,
          2017,
          2009,
          2020,
          2016,
          2022,
          2019,
          2009,
          2021,
          2017,
          2022,
          2016,
          2018,
          2018,
          2019,
          2011,
          2022,
          2022,
          2017,
          2017,
          2023,
          2006,
          2006,
          2006,
          2008,
          2008,
          2008,
          2009,
          2009,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2011,
          2011,
          2011,
          2012,
          2012,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2016,
          2017,
          2016,
          2016,
          2016,
          2017,
          2016,
          2017,
          2017,
          2017,
          2018,
          2017,
          2017,
          2016,
          2018,
          2019,
          2019,
          2019,
          2020,
          2021,
          2021,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N271/44']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N271/44']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N271/44']",
         "offsetgroup": "['N271/44']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2012,
          2021,
          2023,
          2017,
          2019,
          2014,
          2016,
          2016,
          2017,
          2020,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N144/32', ' N159/5', ' N174B/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N144/32', ' N159/5', ' N174B/27']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N144/32', ' N159/5', ' N174B/27']",
         "offsetgroup": "['N144/32', ' N159/5', ' N174B/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N93A/49', ' N11/33']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N93A/49', ' N11/33']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N93A/49', ' N11/33']",
         "offsetgroup": "['N93A/49', ' N11/33']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' L21/32']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' L21/32']",
         "offsetgroup": "['K28/43', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023,
          2012,
          2013,
          2022,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L18A/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L18A/3']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L18A/3']",
         "offsetgroup": "['L18A/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N258/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N258/5']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N258/5']",
         "offsetgroup": "['N258/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2011,
          2011,
          2010,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327A/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327A/38']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327A/38']",
         "offsetgroup": "['N327A/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K78/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K78/29']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K78/29']",
         "offsetgroup": "['K78/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N116/14']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N116/14']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N116/14']",
         "offsetgroup": "['N116/14']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2012,
          2014,
          2017,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' N355/1', ' N95/35', ' N97A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' N355/1', ' N95/35', ' N97A/31']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' N355/1', ' N95/35', ' N97A/31']",
         "offsetgroup": "['N59/36', ' N355/1', ' N95/35', ' N97A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' L21/32']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' L21/32']",
         "offsetgroup": "['N59/36', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2009,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L21/32', ' N355/1']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L21/32', ' N355/1']",
         "offsetgroup": "['L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2022,
          2022,
          2017,
          2019,
          2020,
          2021,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N245/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N245/36']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N245/36']",
         "offsetgroup": "['N245/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N245/36', ' K28/43', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N245/36', ' K28/43', ' L21/32']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N245/36', ' K28/43', ' L21/32']",
         "offsetgroup": "['N245/36', ' K28/43', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N201/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N201/35']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N201/35']",
         "offsetgroup": "['N201/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2019,
          2019,
          2020,
          2017,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N276A/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N276A/15']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N276A/15']",
         "offsetgroup": "['N276A/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N152B/23']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N152B/23']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N152B/23']",
         "offsetgroup": "['N152B/23']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2013,
          2018,
          2022,
          2022,
          2022,
          2015,
          2020,
          2014,
          2015,
          2016,
          2016,
          2016,
          2017,
          2018,
          2019,
          2022,
          2022,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N325B/65']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N325B/65']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N325B/65']",
         "offsetgroup": "['N325B/65']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N212/3', ' N212A/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N212/3', ' N212A/34']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N212/3', ' N212A/34']",
         "offsetgroup": "['N212/3', ' N212A/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K7/45', ' L23/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K7/45', ' L23/27']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K7/45', ' L23/27']",
         "offsetgroup": "['K7/45', ' L23/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019,
          2019,
          2019,
          2013,
          2013,
          2013,
          2010,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N72/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N72/16']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N72/16']",
         "offsetgroup": "['N72/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011,
          2011,
          2012,
          2018,
          2021,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N1/12', ' N161/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N1/12', ' N161/20']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N1/12', ' N161/20']",
         "offsetgroup": "['N1/12', ' N161/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K17/70', ' K13/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K17/70', ' K13/31']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K17/70', ' K13/31']",
         "offsetgroup": "['K20/78', ' K17/70', ' K13/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007,
          2007,
          2007,
          2007
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K17/70']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K17/70']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K17/70']",
         "offsetgroup": "['K13/31', ' K17/70']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K17/70', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K17/70', ' K14/16']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K17/70', ' K14/16']",
         "offsetgroup": "['K13/31', ' K17/70', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K14/16']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K14/16']",
         "offsetgroup": "['K13/31', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007,
          2014,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' N72/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' N72/16']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' N72/16']",
         "offsetgroup": "['K13/31', ' N72/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K14/16', ' K20/78']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K14/16', ' K20/78']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K14/16', ' K20/78']",
         "offsetgroup": "['K13/31', ' K14/16', ' K20/78']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41']",
         "offsetgroup": "['K75/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017,
          2023,
          2019,
          2019,
          2015,
          2007,
          2014,
          2007,
          2022,
          2009,
          2010,
          2011,
          2011,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2013,
          2015,
          2015,
          2019,
          2019,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K36/15', ' K13/31', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K36/15', ' K13/31', ' K57/1']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K36/15', ' K13/31', ' K57/1']",
         "offsetgroup": "['K20/78', ' K36/15', ' K13/31', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007,
          2007,
          2007,
          2007,
          2007
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N149/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N149/25']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N149/25']",
         "offsetgroup": "['N149/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2010,
          2012,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K91/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K91/36']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K91/36']",
         "offsetgroup": "['K91/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2014,
          2007,
          2009,
          2014,
          2014,
          2017,
          2018,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N28/9', ' N19/2', ' K28/74']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N28/9', ' N19/2', ' K28/74']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N28/9', ' N19/2', ' K28/74']",
         "offsetgroup": "['K28/43', ' N28/9', ' N19/2', ' K28/74']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/60', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/60', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/60', ' L21/32', ' N355/1']",
         "offsetgroup": "['L6/60', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N39B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N39B/8']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N39B/8']",
         "offsetgroup": "['N39B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2009,
          2011,
          2011,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N83/48']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N83/48']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N83/48']",
         "offsetgroup": "['N83/48']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N103/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N103/31']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N103/31']",
         "offsetgroup": "['N103/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019,
          2011,
          2011,
          2010,
          2014,
          2014,
          2011,
          2011,
          2011,
          2013,
          2012,
          2013,
          2013,
          2013,
          2015,
          2018,
          2019,
          2020,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N75/33']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N75/33']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N75/33']",
         "offsetgroup": "['N75/33']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N212/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N212/7']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N212/7']",
         "offsetgroup": "['N212/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' N114/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' N114/10']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' N114/10']",
         "offsetgroup": "['N70/28', ' N114/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' K28/43']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' K28/43']",
         "offsetgroup": "['N70/28', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N138/6', ' N152B/23', ' N241A/34', ' N231B/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N138/6', ' N152B/23', ' N241A/34', ' N231B/34']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N138/6', ' N152B/23', ' N241A/34', ' N231B/34']",
         "offsetgroup": "['N138/6', ' N152B/23', ' N241A/34', ' N231B/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N241A/34', ' N231B/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N241A/34', ' N231B/34']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N241A/34', ' N231B/34']",
         "offsetgroup": "['N241A/34', ' N231B/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N231B/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N231B/34']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N231B/34']",
         "offsetgroup": "['N231B/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L23/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L23/27']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L23/27']",
         "offsetgroup": "['L23/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2015,
          2015,
          2021,
          2019,
          2008,
          2010,
          2016,
          2010,
          2011,
          2011,
          2012,
          2013,
          2014,
          2014,
          2014,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2018,
          2018,
          2019,
          2021,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N29/29', ' L23/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N29/29', ' L23/27']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N29/29', ' L23/27']",
         "offsetgroup": "['N29/29', ' L23/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N29/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N29/29']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N29/29']",
         "offsetgroup": "['N29/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2009,
          2012,
          2012,
          2013,
          2015,
          2015,
          2017,
          2020,
          2022,
          2022,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N166A/26']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N166A/26']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N166A/26']",
         "offsetgroup": "['N166A/26']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' L11A/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' L11A/41']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' L11A/41']",
         "offsetgroup": "['K74/71', ' L11A/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007,
          2007,
          2007
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K14/16']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K14/16']",
         "offsetgroup": "['K87A/10', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2020,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K14/16']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K14/16']",
         "offsetgroup": "['K20/78', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2010,
          2011,
          2011,
          2012,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15', ' K14/16']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15', ' K14/16']",
         "offsetgroup": "['K36/15', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2014,
          2014,
          2013,
          2013,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['A12/18', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['A12/18', ' K14/16']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['A12/18', ' K14/16']",
         "offsetgroup": "['A12/18', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K17/70', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K17/70', ' K14/16']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K17/70', ' K14/16']",
         "offsetgroup": "['K17/70', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K74/71', ' K36/15', ' L11A/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K74/71', ' K36/15', ' L11A/41']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K74/71', ' K36/15', ' L11A/41']",
         "offsetgroup": "['K87A/10', ' K74/71', ' K36/15', ' L11A/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2008,
          2008,
          2008,
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K14/16', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K14/16', ' N16B/8']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K14/16', ' N16B/8']",
         "offsetgroup": "['K20/78', ' K14/16', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' K14/16']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' K14/16']",
         "offsetgroup": "['K28/43', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K14/16', ' K87A/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K14/16', ' K87A/10']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K14/16', ' K87A/10']",
         "offsetgroup": "['K20/78', ' K14/16', ' K87A/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N275/14']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N275/14']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N275/14']",
         "offsetgroup": "['N275/14']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2018,
          2021,
          2019,
          2020,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N230/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N230/21']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N230/21']",
         "offsetgroup": "['N230/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2012,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N11/33', ' N3/26']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N11/33', ' N3/26']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N11/33', ' N3/26']",
         "offsetgroup": "['N11/33', ' N3/26']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2015,
          2015,
          2015,
          2013,
          2013,
          2013,
          2009,
          2011,
          2016,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N19/2', ' N11/33', ' N3/26']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N19/2', ' N11/33', ' N3/26']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N19/2', ' N11/33', ' N3/26']",
         "offsetgroup": "['N19/2', ' N11/33', ' N3/26']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N377/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N377/20']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N377/20']",
         "offsetgroup": "['N377/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2020,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N345/51']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N345/51']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N345/51']",
         "offsetgroup": "['N345/51']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N43/6', ' K75/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N43/6', ' K75/41']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N43/6', ' K75/41']",
         "offsetgroup": "['N43/6', ' K75/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N43/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N43/6']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N43/6']",
         "offsetgroup": "['N43/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2019,
          2013,
          2010,
          2011,
          2012,
          2013,
          2013,
          2014,
          2015,
          2015,
          2016,
          2016,
          2017,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N43/6', ' N37A/10', ' N26A/23']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N43/6', ' N37A/10', ' N26A/23']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N43/6', ' N37A/10', ' N26A/23']",
         "offsetgroup": "['N43/6', ' N37A/10', ' N26A/23']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K96/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K96/7']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K96/7']",
         "offsetgroup": "['K96/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2012,
          2011,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K96/7', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K96/7', ' K28/43']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K96/7', ' K28/43']",
         "offsetgroup": "['K96/7', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L45/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L45/30']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L45/30']",
         "offsetgroup": "['L45/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L114/3', ' K74/71']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L114/3', ' K74/71']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L114/3', ' K74/71']",
         "offsetgroup": "['L114/3', ' K74/71']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N262/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N262/16']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N262/16']",
         "offsetgroup": "['N262/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N257/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N257/25']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N257/25']",
         "offsetgroup": "['N257/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N257/25', ' N262/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N257/25', ' N262/16']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N257/25', ' N262/16']",
         "offsetgroup": "['N257/25', ' N262/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N112B/14', ' N124B/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N112B/14', ' N124B/38']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N112B/14', ' N124B/38']",
         "offsetgroup": "['N112B/14', ' N124B/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N97A/31', ' N110/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N97A/31', ' N110/29']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N97A/31', ' N110/29']",
         "offsetgroup": "['N97A/31', ' N110/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2019,
          2017,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N19/2', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N19/2', ' N18/30']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N19/2', ' N18/30']",
         "offsetgroup": "['N19/2', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015,
          2009,
          2009,
          2009,
          2012,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N18/30', ' N19/2', ' K64/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N18/30', ' N19/2', ' K64/15']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N18/30', ' N19/2', ' K64/15']",
         "offsetgroup": "['N18/30', ' N19/2', ' K64/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2017,
          2017,
          2007,
          2007,
          2007,
          2007,
          2007
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' N19/2', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' N19/2', ' N23B/6']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' N19/2', ' N23B/6']",
         "offsetgroup": "['N59/36', ' N19/2', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N170A/1', ' N97A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N170A/1', ' N97A/31']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N170A/1', ' N97A/31']",
         "offsetgroup": "['N170A/1', ' N97A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2018,
          2018,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/86', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/86', ' N59/20']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/86', ' N59/20']",
         "offsetgroup": "['K28/86', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K17/70']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K17/70']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K17/70']",
         "offsetgroup": "['K17/70']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2013,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N4/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N4/15']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N4/15']",
         "offsetgroup": "['N4/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N182/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N182/17']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N182/17']",
         "offsetgroup": "['N182/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/60', ' L24/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/60', ' L24/18']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/60', ' L24/18']",
         "offsetgroup": "['L6/60', ' L24/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L24/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L24/18']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L24/18']",
         "offsetgroup": "['L24/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2012,
          2023,
          2020,
          2013,
          2016,
          2023,
          2022,
          2009,
          2008,
          2008,
          2008,
          2009,
          2010,
          2011,
          2013,
          2013,
          2015,
          2014,
          2016,
          2017,
          2017,
          2018,
          2018,
          2020,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N11/33']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N11/33']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N11/33']",
         "offsetgroup": "['N11/33']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2015,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/86', ' N6/38', ' N28/9', ' N114/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/86', ' N6/38', ' N28/9', ' N114/10']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/86', ' N6/38', ' N28/9', ' N114/10']",
         "offsetgroup": "['K28/86', ' N6/38', ' N28/9', ' N114/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015,
          2018,
          2018,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N6/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N6/38']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N6/38']",
         "offsetgroup": "['N6/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2013,
          2008,
          2010,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K55/7']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K55/7']",
         "offsetgroup": "['K75/41', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2008,
          2008,
          2008,
          2006,
          2006,
          2006
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N192/12']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N192/12']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N192/12']",
         "offsetgroup": "['N192/12']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2015,
          2011,
          2019,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N279B/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N279B/27']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N279B/27']",
         "offsetgroup": "['N279B/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N16B/8']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N16B/8']",
         "offsetgroup": "['N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2020,
          2008,
          2009,
          2010,
          2009,
          2010,
          2010,
          2010,
          2010,
          2011,
          2011,
          2012,
          2013,
          2013,
          2013,
          2013,
          2014,
          2015,
          2016,
          2017,
          2019,
          2018,
          2019,
          2020,
          2020,
          2021,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L51/82']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L51/82']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L51/82']",
         "offsetgroup": "['L51/82']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2014,
          2012,
          2014,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K74/71', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K74/71', ' K69/3']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K74/71', ' K69/3']",
         "offsetgroup": "['K87A/10', ' K74/71', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2010,
          2008,
          2008,
          2008,
          2008,
          2007,
          2007,
          2007,
          2007,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' K69/3']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' K69/3']",
         "offsetgroup": "['K74/71', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013,
          2008,
          2009,
          2010,
          2011,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K69/3', ' K74/71', ' K87A/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K69/3', ' K74/71', ' K87A/10']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K69/3', ' K74/71', ' K87A/10']",
         "offsetgroup": "['K69/3', ' K74/71', ' K87A/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K74/71']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K74/71']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K74/71']",
         "offsetgroup": "['K87A/10', ' K74/71']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007,
          2007,
          2007,
          2008,
          2008,
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N56/21', ' N94/17', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N56/21', ' N94/17', ' K69/3']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N56/21', ' N94/17', ' K69/3']",
         "offsetgroup": "['N56/21', ' N94/17', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K75/41', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K75/41', ' K69/3']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K75/41', ' K69/3']",
         "offsetgroup": "['K20/78', ' K75/41', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['A12/18', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['A12/18', ' K69/3']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['A12/18', ' K69/3']",
         "offsetgroup": "['A12/18', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L102/45', ' L21/32', ' L42/17', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L102/45', ' L21/32', ' L42/17', ' N18/30']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L102/45', ' L21/32', ' L42/17', ' N18/30']",
         "offsetgroup": "['L102/45', ' L21/32', ' L42/17', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L42/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L42/17']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L42/17']",
         "offsetgroup": "['L42/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N171/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N171/17']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N171/17']",
         "offsetgroup": "['N171/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N18/28', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N18/28', ' N18/30']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N18/28', ' N18/30']",
         "offsetgroup": "['N18/28', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N18/28', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N18/28', ' K28/43']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N18/28', ' K28/43']",
         "offsetgroup": "['N18/28', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L80/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L80/21']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L80/21']",
         "offsetgroup": "['L80/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' A12/18']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' A12/18']",
         "offsetgroup": "['N28/9', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N18/30']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N18/30']",
         "offsetgroup": "['K28/43', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N127/31', ' N59/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N127/31', ' N59/36']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N127/31', ' N59/36']",
         "offsetgroup": "['N23B/49', ' N127/31', ' N59/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N245/36', ' K64/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N245/36', ' K64/15']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N245/36', ' K64/15']",
         "offsetgroup": "['N245/36', ' K64/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N97A/31', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N97A/31', ' K28/43']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N97A/31', ' K28/43']",
         "offsetgroup": "['N97A/31', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N127/31', ' N59/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N127/31', ' N59/36']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N127/31', ' N59/36']",
         "offsetgroup": "['N127/31', ' N59/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' L21/32', ' N355/1']",
         "offsetgroup": "['K28/43', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' K64/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' K64/15']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' K64/15']",
         "offsetgroup": "['K28/43', ' K64/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K56A/50', ' L21/32', ' K64/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K56A/50', ' L21/32', ' K64/15']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K56A/50', ' L21/32', ' K64/15']",
         "offsetgroup": "['K56A/50', ' L21/32', ' K64/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N127/31', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N127/31', ' N18/30']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N127/31', ' N18/30']",
         "offsetgroup": "['N127/31', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N134/12', ' N68/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N134/12', ' N68/6']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N134/12', ' N68/6']",
         "offsetgroup": "['N134/12', ' N68/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2013,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N25/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N25/35']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N25/35']",
         "offsetgroup": "['N25/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N77/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N77/15']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N77/15']",
         "offsetgroup": "['N77/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L8/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L8/15']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L8/15']",
         "offsetgroup": "['L8/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "offsetgroup": "['K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014,
          2014,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K55/7']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K55/7']",
         "offsetgroup": "['K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2009,
          2013,
          2010,
          2015,
          2019,
          2019,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K75/41', ' K55/7', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K75/41', ' K55/7', ' K57/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K75/41', ' K55/7', ' K57/1']",
         "offsetgroup": "['K60/73', ' K75/41', ' K55/7', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N423/75', ' K60/73', ' K75/41', ' K66/38', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N423/75', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N423/75', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "offsetgroup": "['N423/75', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N423/75', ' K75/41', ' K66/38', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N423/75', ' K75/41', ' K66/38', ' K57/1']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N423/75', ' K75/41', ' K66/38', ' K57/1']",
         "offsetgroup": "['N423/75', ' K75/41', ' K66/38', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' N144/32', ' N159/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' N144/32', ' N159/5']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' N144/32', ' N159/5']",
         "offsetgroup": "['N70/28', ' N144/32', ' N159/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N144/32', ' N159/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N144/32', ' N159/5']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N144/32', ' N159/5']",
         "offsetgroup": "['N144/32', ' N159/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K57/1']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K57/1']",
         "offsetgroup": "['K75/41', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2008,
          2008,
          2019,
          2019,
          2019,
          2008,
          2008,
          2008,
          2009,
          2010,
          2010,
          2011,
          2015,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' K75/41', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' K75/41', ' K57/1']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' K75/41', ' K57/1']",
         "offsetgroup": "['K74/71', ' K75/41', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2006
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K57/1']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K57/1']",
         "offsetgroup": "['K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2013,
          2006,
          2016,
          2007,
          2007,
          2013,
          2013,
          2013,
          2022,
          2011,
          2008,
          2022,
          2008,
          2008,
          2008,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2010,
          2011,
          2010,
          2011,
          2011,
          2012,
          2012,
          2013,
          2014,
          2016,
          2017,
          2018,
          2018,
          2019,
          2019,
          2020,
          2020,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K57/1', ' K75/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K57/1', ' K75/41']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K57/1', ' K75/41']",
         "offsetgroup": "['K57/1', ' K75/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N50/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N50/36']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N50/36']",
         "offsetgroup": "['N50/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2012,
          2013,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N283/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N283/7']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N283/7']",
         "offsetgroup": "['N283/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N245/1', ' N245/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N245/1', ' N245/36']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N245/1', ' N245/36']",
         "offsetgroup": "['N245/1', ' N245/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N85/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N85/18']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N85/18']",
         "offsetgroup": "['N85/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2011,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N26A/23']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N26A/23']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N26A/23']",
         "offsetgroup": "['N26A/23']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2008,
          2015,
          2018,
          2014,
          2013,
          2014,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N170A/26']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N170A/26']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N170A/26']",
         "offsetgroup": "['N170A/26']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N432/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N432/21']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N432/21']",
         "offsetgroup": "['N432/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N97A/31', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N97A/31', ' K56A/50']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N97A/31', ' K56A/50']",
         "offsetgroup": "['N97A/31', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N135/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N135/37']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N135/37']",
         "offsetgroup": "['N135/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N289/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N289/16']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N289/16']",
         "offsetgroup": "['N289/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2014,
          2013,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L11A/41', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L11A/41', ' A12/18']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L11A/41', ' A12/18']",
         "offsetgroup": "['L11A/41', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2012,
          2012,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N154/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N154/32']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N154/32']",
         "offsetgroup": "['N154/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N74/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N74/25']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N74/25']",
         "offsetgroup": "['N74/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2014,
          2017,
          2020,
          2021,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/6']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/6']",
         "offsetgroup": "['N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2018,
          2011,
          2012,
          2014,
          2015,
          2017,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N2/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N2/16']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N2/16']",
         "offsetgroup": "['N2/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2011,
          2009,
          2009,
          2010,
          2011,
          2015,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L5/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L5/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L5/1']",
         "offsetgroup": "['L5/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2011,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73']",
         "offsetgroup": "['K60/73']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2013,
          2013,
          2008,
          2008,
          2010,
          2010,
          2010,
          2010,
          2012,
          2012,
          2012,
          2013,
          2013,
          2015,
          2014,
          2015,
          2016,
          2016,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K57/1']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K57/1']",
         "offsetgroup": "['K60/73', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2013,
          2010,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L21/32', ' N19/2', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L21/32', ' N19/2', ' N95/35']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L21/32', ' N19/2', ' N95/35']",
         "offsetgroup": "['L21/32', ' N19/2', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' K28/43']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' K28/43']",
         "offsetgroup": "['N59/36', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N286/74']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N286/74']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N286/74']",
         "offsetgroup": "['N286/74']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K36/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K36/15']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K36/15']",
         "offsetgroup": "['K20/78', ' K36/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2011,
          2013,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N321C/49']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N321C/49']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N321C/49']",
         "offsetgroup": "['N321C/49']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2021,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N129/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N129/15']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N129/15']",
         "offsetgroup": "['N129/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2011,
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K17/70', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K17/70', ' N18/30']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K17/70', ' N18/30']",
         "offsetgroup": "['K13/31', ' K17/70', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N221/17', ' N15/4']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N221/17', ' N15/4']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N221/17', ' N15/4']",
         "offsetgroup": "['N221/17', ' N15/4']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N15/4']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N15/4']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N15/4']",
         "offsetgroup": "['N15/4']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2008,
          2014,
          2018,
          2018,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N202/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N202/7']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N202/7']",
         "offsetgroup": "['N202/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2014,
          2014,
          2017,
          2016,
          2012,
          2012,
          2011,
          2012,
          2013,
          2015,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K17/70']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K17/70']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K17/70']",
         "offsetgroup": "['K20/78', ' K17/70']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2006
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N336B/83']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N336B/83']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N336B/83']",
         "offsetgroup": "['N336B/83']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N221/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N221/17']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N221/17']",
         "offsetgroup": "['N221/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022,
          2020,
          2013,
          2013,
          2015,
          2017,
          2014,
          2014,
          2014,
          2017,
          2021,
          2021,
          2022,
          2022,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K55/82', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K55/82', ' K57/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K55/82', ' K57/1']",
         "offsetgroup": "['K55/82', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N111/24']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N111/24']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N111/24']",
         "offsetgroup": "['N111/24']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2012,
          2012,
          2013,
          2014,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N219/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N219/5']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N219/5']",
         "offsetgroup": "['N219/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N127/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N127/31']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N127/31']",
         "offsetgroup": "['N127/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N121A/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N121A/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N121A/1']",
         "offsetgroup": "['N121A/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N68/6', ' N372B/60', ' N134/12', ' K14/16', ' K65/35', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N68/6', ' N372B/60', ' N134/12', ' K14/16', ' K65/35', ' K89/34']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N68/6', ' N372B/60', ' N134/12', ' K14/16', ' K65/35', ' K89/34']",
         "offsetgroup": "['N68/6', ' N372B/60', ' N134/12', ' K14/16', ' K65/35', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K66/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K66/38']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K66/38']",
         "offsetgroup": "['K66/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2024,
          2008,
          2009,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/48', ' L6/60']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/48', ' L6/60']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/48', ' L6/60']",
         "offsetgroup": "['L6/48', ' L6/60']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2006
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K7/45']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K7/45']",
         "offsetgroup": "['K13/31', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K55/82', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K55/82', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K55/82', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "offsetgroup": "['K55/82', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2006
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K51/1', ' L11A/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K51/1', ' L11A/41']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K51/1', ' L11A/41']",
         "offsetgroup": "['K51/1', ' L11A/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K75/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K75/41']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K75/41']",
         "offsetgroup": "['K60/73', ' K75/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2009,
          2011,
          2014,
          2015,
          2017,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K55/82']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K55/82']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K55/82']",
         "offsetgroup": "['K75/41', ' K55/82']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K17/70', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K17/70', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K17/70', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "offsetgroup": "['K13/31', ' K17/70', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' N56/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' N56/21']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' N56/21']",
         "offsetgroup": "['K87A/10', ' N56/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L11A/41', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L11A/41', ' K69/3']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L11A/41', ' K69/3']",
         "offsetgroup": "['L11A/41', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K55/82']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K55/82']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K55/82']",
         "offsetgroup": "['K55/82']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K17/70', ' N68/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K17/70', ' N68/6']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K17/70', ' N68/6']",
         "offsetgroup": "['K17/70', ' N68/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N1/12', ' L11A/41', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N1/12', ' L11A/41', ' A12/18']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N1/12', ' L11A/41', ' A12/18']",
         "offsetgroup": "['N1/12', ' L11A/41', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K74/71', ' K65/35', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K74/71', ' K65/35', ' K69/3']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K74/71', ' K65/35', ' K69/3']",
         "offsetgroup": "['K87A/10', ' K74/71', ' K65/35', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K36/15', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K36/15', ' K7/45']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K36/15', ' K7/45']",
         "offsetgroup": "['K20/78', ' K36/15', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L20/8', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L20/8', ' N59/20']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L20/8', ' N59/20']",
         "offsetgroup": "['L20/8', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2009,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K51/1', ' L11A/41', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K51/1', ' L11A/41', ' K65/35']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K51/1', ' L11A/41', ' K65/35']",
         "offsetgroup": "['K51/1', ' L11A/41', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K66/38', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K66/38', ' K55/7']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K66/38', ' K55/7']",
         "offsetgroup": "['K60/73', ' K66/38', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' K14/16']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' K14/16']",
         "offsetgroup": "['K65/35', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2010,
          2011,
          2012,
          2015,
          2014,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' K7/45']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' K7/45']",
         "offsetgroup": "['K74/71', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/60', ' N2/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/60', ' N2/16']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/60', ' N2/16']",
         "offsetgroup": "['L6/60', ' N2/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N46/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N46/30']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N46/30']",
         "offsetgroup": "['N46/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2010,
          2018,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K55/82']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K55/82']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K55/82']",
         "offsetgroup": "['K89/34', ' K55/82']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K65/35', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K65/35', ' K69/3']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K65/35', ' K69/3']",
         "offsetgroup": "['K87A/10', ' K65/35', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N8B/1', ' N7/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N8B/1', ' N7/18']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N8B/1', ' N7/18']",
         "offsetgroup": "['N8B/1', ' N7/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N8B/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N8B/1']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N8B/1']",
         "offsetgroup": "['N8B/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N29/29', ' N28/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N29/29', ' N28/9']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N29/29', ' N28/9']",
         "offsetgroup": "['N29/29', ' N28/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2010,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N18/30']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N18/30']",
         "offsetgroup": "['N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2010,
          2010,
          2013,
          2013,
          2014,
          2016,
          2016,
          2018,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K55/7', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K55/7', ' K57/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K55/7', ' K57/1']",
         "offsetgroup": "['K55/7', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K55/7', ' K66/38', ' K55/82']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K55/7', ' K66/38', ' K55/82']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K55/7', ' K66/38', ' K55/82']",
         "offsetgroup": "['K60/73', ' K55/7', ' K66/38', ' K55/82']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' N6/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' N6/38']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' N6/38']",
         "offsetgroup": "['K89/34', ' N6/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N210/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N210/5']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N210/5']",
         "offsetgroup": "['N210/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N28/9', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N28/9', ' N59/20']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N28/9', ' N59/20']",
         "offsetgroup": "['N23B/49', ' N28/9', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N71/37', ' N141/28']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N71/37', ' N141/28']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N71/37', ' N141/28']",
         "offsetgroup": "['N71/37', ' N141/28']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K60/73', ' L57/46']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K60/73', ' L57/46']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K60/73', ' L57/46']",
         "offsetgroup": "['K13/31', ' K60/73', ' L57/46']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N124B/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N124B/38']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N124B/38']",
         "offsetgroup": "['N124B/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N18/30', ' K64/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N18/30', ' K64/15']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N18/30', ' K64/15']",
         "offsetgroup": "['N18/30', ' K64/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N72/16', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N72/16', ' N16B/8']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N72/16', ' N16B/8']",
         "offsetgroup": "['N72/16', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' L21/32', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' L21/32', ' N59/20']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' L21/32', ' N59/20']",
         "offsetgroup": "['N59/36', ' L21/32', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N235/22']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N235/22']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N235/22']",
         "offsetgroup": "['N235/22']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/86', ' N23B/49']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/86', ' N23B/49']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/86', ' N23B/49']",
         "offsetgroup": "['K28/86', ' N23B/49']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K65/35', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K65/35', ' K14/16']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K65/35', ' K14/16']",
         "offsetgroup": "['K87A/10', ' K65/35', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' K37/89', ' K89/34', ' N86/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' K37/89', ' K89/34', ' N86/38']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' K37/89', ' K89/34', ' N86/38']",
         "offsetgroup": "['N86/8', ' K37/89', ' K89/34', ' N86/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K75/41', ' K66/38', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K75/41', ' K66/38', ' K57/1']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K75/41', ' K66/38', ' K57/1']",
         "offsetgroup": "['K60/73', ' K75/41', ' K66/38', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N195A/16', ' N147/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N195A/16', ' N147/6']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N195A/16', ' N147/6']",
         "offsetgroup": "['N195A/16', ' N147/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K89/34']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K89/34']",
         "offsetgroup": "['K60/73', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N100/13']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N100/13']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N100/13']",
         "offsetgroup": "['N23B/49', ' N100/13']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/23', ' L6/60']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/23', ' L6/60']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/23', ' L6/60']",
         "offsetgroup": "['L6/23', ' L6/60']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' N28/9', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' N28/9', ' N95/35']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' N28/9', ' N95/35']",
         "offsetgroup": "['N59/36', ' N28/9', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K89/34']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K89/34']",
         "offsetgroup": "['K75/41', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N134/12', ' K74/71']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N134/12', ' K74/71']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N134/12', ' K74/71']",
         "offsetgroup": "['N134/12', ' K74/71']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K75/41', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K75/41', ' K55/7']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K75/41', ' K55/7']",
         "offsetgroup": "['K60/73', ' K75/41', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' L23/27', ' K19/36', ' K20/78', ' K14/16', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' L23/27', ' K19/36', ' K20/78', ' K14/16', ' K7/45']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' L23/27', ' K19/36', ' K20/78', ' K14/16', ' K7/45']",
         "offsetgroup": "['K13/31', ' L23/27', ' K19/36', ' K20/78', ' K14/16', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N153/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N153/5']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N153/5']",
         "offsetgroup": "['N153/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K19/36', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K19/36', ' K57/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K19/36', ' K57/1']",
         "offsetgroup": "['K75/41', ' K19/36', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K14/16', ' K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K14/16', ' K19/36']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K14/16', ' K19/36']",
         "offsetgroup": "['K13/31', ' K14/16', ' K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' N105/13', ' N105/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' N105/13', ' N105/17']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' N105/13', ' N105/17']",
         "offsetgroup": "['K65/35', ' N105/13', ' N105/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K14/16', ' N72/16', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K14/16', ' N72/16', ' K57/1']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K14/16', ' N72/16', ' K57/1']",
         "offsetgroup": "['K14/16', ' N72/16', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L20/8', ' N59/36', ' N125/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L20/8', ' N59/36', ' N125/10']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L20/8', ' N59/36', ' N125/10']",
         "offsetgroup": "['L20/8', ' N59/36', ' N125/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K9/40', ' N124B/38', ' K47/42', ' K17/70', ' N112B/14', ' K57/1', ' K60/73', ' K75/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K9/40', ' N124B/38', ' K47/42', ' K17/70', ' N112B/14', ' K57/1', ' K60/73', ' K75/41']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K9/40', ' N124B/38', ' K47/42', ' K17/70', ' N112B/14', ' K57/1', ' K60/73', ' K75/41']",
         "offsetgroup": "['K13/31', ' K9/40', ' N124B/38', ' K47/42', ' K17/70', ' N112B/14', ' K57/1', ' K60/73', ' K75/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N244/5', ' N105/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N244/5', ' N105/17']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N244/5', ' N105/17']",
         "offsetgroup": "['N244/5', ' N105/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' N212A/34', ' N212/3', ' N212/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' N212A/34', ' N212/3', ' N212/17']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' N212A/34', ' N212/3', ' N212/17']",
         "offsetgroup": "['N70/28', ' N212A/34', ' N212/3', ' N212/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K67/25', ' N86/8', ' K17/70', ' K73/20', ' K14/16', ' N86/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K67/25', ' N86/8', ' K17/70', ' K73/20', ' K14/16', ' N86/38']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K67/25', ' N86/8', ' K17/70', ' K73/20', ' K14/16', ' N86/38']",
         "offsetgroup": "['K67/25', ' N86/8', ' K17/70', ' K73/20', ' K14/16', ' N86/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L57/46', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L57/46', ' K14/16']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L57/46', ' K14/16']",
         "offsetgroup": "['L57/46', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' K20/78', ' L6/60', ' K14/16', ' N16B/8', ' K75/41', ' K69/3', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' K20/78', ' L6/60', ' K14/16', ' N16B/8', ' K75/41', ' K69/3', ' K57/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' K20/78', ' L6/60', ' K14/16', ' N16B/8', ' K75/41', ' K69/3', ' K57/1']",
         "offsetgroup": "['K74/71', ' K20/78', ' L6/60', ' K14/16', ' N16B/8', ' K75/41', ' K69/3', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' K67/25', ' K20/78', ' K73/20', ' K65/35', ' K87A/10', ' N105/13', ' N105/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' K67/25', ' K20/78', ' K73/20', ' K65/35', ' K87A/10', ' N105/13', ' N105/17']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' K67/25', ' K20/78', ' K73/20', ' K65/35', ' K87A/10', ' N105/13', ' N105/17']",
         "offsetgroup": "['K74/71', ' K67/25', ' K20/78', ' K73/20', ' K65/35', ' K87A/10', ' N105/13', ' N105/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N59/36', ' N22/21', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N59/36', ' N22/21', ' N23B/6']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N59/36', ' N22/21', ' N23B/6']",
         "offsetgroup": "['N23B/49', ' N59/36', ' N22/21', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N7/18', ' L57/46']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N7/18', ' L57/46']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N7/18', ' L57/46']",
         "offsetgroup": "['N7/18', ' L57/46']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K37/89']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K37/89']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K37/89']",
         "offsetgroup": "['K37/89']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N196/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N196/16']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N196/16']",
         "offsetgroup": "['N196/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2013,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K14/16', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K14/16', ' N16B/8']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K14/16', ' N16B/8']",
         "offsetgroup": "['K14/16', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' N71/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' N71/37']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' N71/37']",
         "offsetgroup": "['N70/28', ' N71/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K14/16', ' K64/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K14/16', ' K64/15']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K14/16', ' K64/15']",
         "offsetgroup": "['K89/34', ' K14/16', ' K64/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' N72/16', ' K20/78', ' K55/82', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' N72/16', ' K20/78', ' K55/82', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' N72/16', ' K20/78', ' K55/82', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "offsetgroup": "['K13/31', ' N72/16', ' K20/78', ' K55/82', ' K57/1', ' K60/73', ' K75/41', ' K66/38', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K17/70', ' K65/35', ' K14/16', ' K87A/10', ' K36/15', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K17/70', ' K65/35', ' K14/16', ' K87A/10', ' K36/15', ' N18/30']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K17/70', ' K65/35', ' K14/16', ' K87A/10', ' K36/15', ' N18/30']",
         "offsetgroup": "['K20/78', ' K17/70', ' K65/35', ' K14/16', ' K87A/10', ' K36/15', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L21/32', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L21/32', ' N59/20']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L21/32', ' N59/20']",
         "offsetgroup": "['L21/32', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2012,
          2015,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N1/12', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N1/12', ' N95/35']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N1/12', ' N95/35']",
         "offsetgroup": "['N1/12', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' L11A/41', ' K20/78', ' N56/21', ' N26A/23', ' K14/16', ' K69/3', ' N105/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' L11A/41', ' K20/78', ' N56/21', ' N26A/23', ' K14/16', ' K69/3', ' N105/17']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' L11A/41', ' K20/78', ' N56/21', ' N26A/23', ' K14/16', ' K69/3', ' N105/17']",
         "offsetgroup": "['K74/71', ' L11A/41', ' K20/78', ' N56/21', ' N26A/23', ' K14/16', ' K69/3', ' N105/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N155/9', ' N38/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N155/9', ' N38/8']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N155/9', ' N38/8']",
         "offsetgroup": "['N155/9', ' N38/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K66/38', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K66/38', ' K57/1']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K66/38', ' K57/1']",
         "offsetgroup": "['K66/38', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N86/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N86/38']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N86/38']",
         "offsetgroup": "['N23B/49', ' N86/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L20/8', ' N19/2', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L20/8', ' N19/2', ' N59/20']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L20/8', ' N19/2', ' N59/20']",
         "offsetgroup": "['L20/8', ' N19/2', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N112B/14', ' N25/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N112B/14', ' N25/35']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N112B/14', ' N25/35']",
         "offsetgroup": "['N112B/14', ' N25/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K47/42', ' K17/70', ' K9/40']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K47/42', ' K17/70', ' K9/40']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K47/42', ' K17/70', ' K9/40']",
         "offsetgroup": "['K47/42', ' K17/70', ' K9/40']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/60', ' L18A/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/60', ' L18A/3']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/60', ' L18A/3']",
         "offsetgroup": "['L6/60', ' L18A/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K67/25', ' K36/15', ' K14/16', ' K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K67/25', ' K36/15', ' K14/16', ' K19/36']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K67/25', ' K36/15', ' K14/16', ' K19/36']",
         "offsetgroup": "['K67/25', ' K36/15', ' K14/16', ' K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N87/25', ' N96/55', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N87/25', ' N96/55', ' N95/35']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N87/25', ' N96/55', ' N95/35']",
         "offsetgroup": "['N87/25', ' N96/55', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N165/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N165/43']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N165/43']",
         "offsetgroup": "['N165/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' N18/30']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' N18/30']",
         "offsetgroup": "['N28/9', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K66/38', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K66/38', ' K57/1']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K66/38', ' K57/1']",
         "offsetgroup": "['K75/41', ' K66/38', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' N71/37', ' N114/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' N71/37', ' N114/10']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' N71/37', ' N114/10']",
         "offsetgroup": "['N70/28', ' N71/37', ' N114/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N144/17', ' N159/5']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N144/17', ' N159/5']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N144/17', ' N159/5']",
         "offsetgroup": "['N144/17', ' N159/5']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N192/12', ' N144/14']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N192/12', ' N144/14']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N192/12', ' N144/14']",
         "offsetgroup": "['N192/12', ' N144/14']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N22/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N22/21']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N22/21']",
         "offsetgroup": "['N22/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K75/41', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K75/41', ' K57/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K75/41', ' K57/1']",
         "offsetgroup": "['K20/78', ' K75/41', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' L23/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' L23/27']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' L23/27']",
         "offsetgroup": "['K20/78', ' L23/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N26A/23', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N26A/23', ' K14/16']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N26A/23', ' K14/16']",
         "offsetgroup": "['N26A/23', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' K56A/50']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' K56A/50']",
         "offsetgroup": "['N23B/49', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L57/46', ' N155/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L57/46', ' N155/9']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L57/46', ' N155/9']",
         "offsetgroup": "['L57/46', ' N155/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N206A/8', ' N149/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N206A/8', ' N149/25']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N206A/8', ' N149/25']",
         "offsetgroup": "['N206A/8', ' N149/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K9/40', ' L23/27', ' K19/36', ' K20/78', ' K47/42', ' K17/70', ' K14/16', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K9/40', ' L23/27', ' K19/36', ' K20/78', ' K47/42', ' K17/70', ' K14/16', ' K7/45']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K9/40', ' L23/27', ' K19/36', ' K20/78', ' K47/42', ' K17/70', ' K14/16', ' K7/45']",
         "offsetgroup": "['K13/31', ' K9/40', ' L23/27', ' K19/36', ' K20/78', ' K47/42', ' K17/70', ' K14/16', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N212/7', ' N141/28']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N212/7', ' N141/28']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N212/7', ' N141/28']",
         "offsetgroup": "['N212/7', ' N141/28']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K20/78', ' N70/28', ' K14/16', ' K57/1', ' N71/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K20/78', ' N70/28', ' K14/16', ' K57/1', ' N71/37']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K20/78', ' N70/28', ' K14/16', ' K57/1', ' N71/37']",
         "offsetgroup": "['K13/31', ' K20/78', ' N70/28', ' K14/16', ' K57/1', ' N71/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L57/46', ' N263/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L57/46', ' N263/31']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L57/46', ' N263/31']",
         "offsetgroup": "['L57/46', ' N263/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K66/38', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K66/38', ' K55/7']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K66/38', ' K55/7']",
         "offsetgroup": "['K66/38', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15', ' N16B/8']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15', ' N16B/8']",
         "offsetgroup": "['K36/15', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N238/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N238/31']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N238/31']",
         "offsetgroup": "['N238/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' N134/12', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' N134/12', ' K14/16']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' N134/12', ' K14/16']",
         "offsetgroup": "['K20/78', ' N134/12', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N71/37', ' N141/28', ' N114/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N71/37', ' N141/28', ' N114/10']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N71/37', ' N141/28', ' N114/10']",
         "offsetgroup": "['N71/37', ' N141/28', ' N114/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K36/15', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K36/15', ' K57/1']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K36/15', ' K57/1']",
         "offsetgroup": "['K20/78', ' K36/15', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' K67/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' K67/25']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' K67/25']",
         "offsetgroup": "['K65/35', ' K67/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N289/16', ' N67/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N289/16', ' N67/15']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N289/16', ' N67/15']",
         "offsetgroup": "['N289/16', ' N67/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' N56/21', ' K74/71']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' N56/21', ' K74/71']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' N56/21', ' K74/71']",
         "offsetgroup": "['K87A/10', ' N56/21', ' K74/71']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/20', ' N52A/42', ' N59/36', ' N19/2', ' K64/15', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/20', ' N52A/42', ' N59/36', ' N19/2', ' K64/15', ' N18/30']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/20', ' N52A/42', ' N59/36', ' N19/2', ' K64/15', ' N18/30']",
         "offsetgroup": "['N59/20', ' N52A/42', ' N59/36', ' N19/2', ' K64/15', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N195A/16', ' N154/32', ' N182/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N195A/16', ' N154/32', ' N182/17']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N195A/16', ' N154/32', ' N182/17']",
         "offsetgroup": "['N195A/16', ' N154/32', ' N182/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' N165/38']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' N165/38']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' N165/38']",
         "offsetgroup": "['N28/9', ' N165/38']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N161/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N161/20']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N161/20']",
         "offsetgroup": "['N161/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N84/37']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N84/37']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N84/37']",
         "offsetgroup": "['N84/37']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N245/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N245/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N245/1']",
         "offsetgroup": "['N245/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N52A/42', ' N59/36', ' N18/30', ' L21/32', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N52A/42', ' N59/36', ' N18/30', ' L21/32', ' N59/20']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N52A/42', ' N59/36', ' N18/30', ' L21/32', ' N59/20']",
         "offsetgroup": "['N52A/42', ' N59/36', ' N18/30', ' L21/32', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N367/62']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N367/62']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N367/62']",
         "offsetgroup": "['N308/48', ' N367/62']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N355/1', ' N308/48', ' L21/32', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N355/1', ' N308/48', ' L21/32', ' N59/20']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N355/1', ' N308/48', ' L21/32', ' N59/20']",
         "offsetgroup": "['N327/95', ' N355/1', ' N308/48', ' L21/32', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L5/1', ' K55/82']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L5/1', ' K55/82']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L5/1', ' K55/82']",
         "offsetgroup": "['L5/1', ' K55/82']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N127/31', ' N22/21', ' N23B/6', ' K91/36', ' N87/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N127/31', ' N22/21', ' N23B/6', ' K91/36', ' N87/25']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N127/31', ' N22/21', ' N23B/6', ' K91/36', ' N87/25']",
         "offsetgroup": "['N127/31', ' N22/21', ' N23B/6', ' K91/36', ' N87/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N110/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N110/29']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N110/29']",
         "offsetgroup": "['N110/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N206A/8', ' N29/29', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N206A/8', ' N29/29', ' N59/20']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N206A/8', ' N29/29', ' N59/20']",
         "offsetgroup": "['N308/48', ' N206A/8', ' N29/29', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N59/36', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N59/36', ' N355/1']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N59/36', ' N355/1']",
         "offsetgroup": "['N327/95', ' N59/36', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K57/1', ' K60/73', ' K75/41', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K57/1', ' K60/73', ' K75/41', ' K7/45']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K57/1', ' K60/73', ' K75/41', ' K7/45']",
         "offsetgroup": "['K89/34', ' K57/1', ' K60/73', ' K75/41', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' N212/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' N212/3']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' N212/3']",
         "offsetgroup": "['N70/28', ' N212/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N86/8', ' N144/14', ' N86/38', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N86/8', ' N144/14', ' N86/38', ' K14/16']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N86/8', ' N144/14', ' N86/38', ' K14/16']",
         "offsetgroup": "['N106/36', ' N86/8', ' N144/14', ' N86/38', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K47/42', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K47/42', ' K89/34']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K47/42', ' K89/34']",
         "offsetgroup": "['K47/42', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N97A/31', ' N59/20', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N97A/31', ' N59/20', ' K56A/50']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N97A/31', ' N59/20', ' K56A/50']",
         "offsetgroup": "['N97A/31', ' N59/20', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N154/32', ' N183/15', ' N182/17', ' N106/65']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N154/32', ' N183/15', ' N182/17', ' N106/65']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N154/32', ' N183/15', ' N182/17', ' N106/65']",
         "offsetgroup": "['N154/32', ' N183/15', ' N182/17', ' N106/65']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N207/27', ' K55/82']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N207/27', ' K55/82']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N207/27', ' K55/82']",
         "offsetgroup": "['N207/27', ' K55/82']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N87/25', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N87/25', ' N95/35']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N87/25', ' N95/35']",
         "offsetgroup": "['N87/25', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K57/1', ' K39/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K57/1', ' K39/25']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K57/1', ' K39/25']",
         "offsetgroup": "['K89/34', ' K57/1', ' K39/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15', ' K89/34', ' K39/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15', ' K89/34', ' K39/25']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15', ' K89/34', ' K39/25']",
         "offsetgroup": "['K36/15', ' K89/34', ' K39/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N8B/1', ' N263/31', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N8B/1', ' N263/31', ' K69/3']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N8B/1', ' N263/31', ' K69/3']",
         "offsetgroup": "['N8B/1', ' N263/31', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' L21/32']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' L21/32']",
         "offsetgroup": "['N23B/49', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N1/66']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N1/66']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N1/66']",
         "offsetgroup": "['N1/66']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15', ' K13/31', ' K75/41', ' N72/16', ' L23/27', ' K19/36', ' K20/78', ' K89/34', ' N16B/8', ' K14/16', ' K57/1', ' K37/89', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15', ' K13/31', ' K75/41', ' N72/16', ' L23/27', ' K19/36', ' K20/78', ' K89/34', ' N16B/8', ' K14/16', ' K57/1', ' K37/89', ' K7/45']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15', ' K13/31', ' K75/41', ' N72/16', ' L23/27', ' K19/36', ' K20/78', ' K89/34', ' N16B/8', ' K14/16', ' K57/1', ' K37/89', ' K7/45']",
         "offsetgroup": "['K36/15', ' K13/31', ' K75/41', ' N72/16', ' L23/27', ' K19/36', ' K20/78', ' K89/34', ' N16B/8', ' K14/16', ' K57/1', ' K37/89', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K56A/50', ' N238/29', ' N110/29', ' N52A/42', ' N19/2', ' N105/17', ' N23B/49', ' L28/4', ' K64/15', ' K57/1', ' N238/31', ' N97A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K56A/50', ' N238/29', ' N110/29', ' N52A/42', ' N19/2', ' N105/17', ' N23B/49', ' L28/4', ' K64/15', ' K57/1', ' N238/31', ' N97A/31']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K56A/50', ' N238/29', ' N110/29', ' N52A/42', ' N19/2', ' N105/17', ' N23B/49', ' L28/4', ' K64/15', ' K57/1', ' N238/31', ' N97A/31']",
         "offsetgroup": "['K56A/50', ' N238/29', ' N110/29', ' N52A/42', ' N19/2', ' N105/17', ' N23B/49', ' L28/4', ' K64/15', ' K57/1', ' N238/31', ' N97A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L21/32', ' N245/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L21/32', ' N245/36']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L21/32', ' N245/36']",
         "offsetgroup": "['L21/32', ' N245/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N96/55', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N96/55', ' N95/35']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N96/55', ' N95/35']",
         "offsetgroup": "['N96/55', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N374/48']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N374/48']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N374/48']",
         "offsetgroup": "['N374/48']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N59/20']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N59/20']",
         "offsetgroup": "['N327/95', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N277/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N277/7']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N277/7']",
         "offsetgroup": "['N277/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K57/1']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K57/1']",
         "offsetgroup": "['K89/34', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' N212/17']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' N212/17']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' N212/17']",
         "offsetgroup": "['N70/28', ' N212/17']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N125/10', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N125/10', ' L21/32']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N125/10', ' L21/32']",
         "offsetgroup": "['N125/10', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' K65/35', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' K65/35', ' K69/3']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' K65/35', ' K69/3']",
         "offsetgroup": "['K74/71', ' K65/35', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N105/13']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N105/13']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N105/13']",
         "offsetgroup": "['N308/48', ' N105/13']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N355/1', ' K56A/50', ' N159/5', ' N59/20', ' N19/2', ' N23B/49', ' N308/48', ' L21/32', ' N97A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N355/1', ' K56A/50', ' N159/5', ' N59/20', ' N19/2', ' N23B/49', ' N308/48', ' L21/32', ' N97A/31']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N355/1', ' K56A/50', ' N159/5', ' N59/20', ' N19/2', ' N23B/49', ' N308/48', ' L21/32', ' N97A/31']",
         "offsetgroup": "['N355/1', ' K56A/50', ' N159/5', ' N59/20', ' N19/2', ' N23B/49', ' N308/48', ' L21/32', ' N97A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K74/71', ' N106/36', ' K69/3']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K74/71', ' N106/36', ' K69/3']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K74/71', ' N106/36', ' K69/3']",
         "offsetgroup": "['K87A/10', ' K74/71', ' N106/36', ' K69/3']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N123/48', ' N307/12', ' N117/9', ' N299/39', ' N309A/21', ' N123/19', ' N158/28', ' N167/7', ' N312/10', ' N310/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N123/48', ' N307/12', ' N117/9', ' N299/39', ' N309A/21', ' N123/19', ' N158/28', ' N167/7', ' N312/10', ' N310/21']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N123/48', ' N307/12', ' N117/9', ' N299/39', ' N309A/21', ' N123/19', ' N158/28', ' N167/7', ' N312/10', ' N310/21']",
         "offsetgroup": "['N123/48', ' N307/12', ' N117/9', ' N299/39', ' N309A/21', ' N123/19', ' N158/28', ' N167/7', ' N312/10', ' N310/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K7/45']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K7/45']",
         "offsetgroup": "['K75/41', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K14/16', ' N144/14']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K14/16', ' N144/14']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K14/16', ' N144/14']",
         "offsetgroup": "['K14/16', ' N144/14']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' N59/36', ' K66/38', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' N59/36', ' K66/38', ' K57/1']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' N59/36', ' K66/38', ' K57/1']",
         "offsetgroup": "['K13/31', ' N59/36', ' K66/38', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N59/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N59/36']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N59/36']",
         "offsetgroup": "['N23B/49', ' N59/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L20/8', ' N245/1', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L20/8', ' N245/1', ' L21/32']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L20/8', ' N245/1', ' L21/32']",
         "offsetgroup": "['L20/8', ' N245/1', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' N106/36', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' N106/36', ' A12/18']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' N106/36', ' A12/18']",
         "offsetgroup": "['K74/71', ' N106/36', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N127/31', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N127/31', ' N23B/6']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N127/31', ' N23B/6']",
         "offsetgroup": "['N127/31', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K14/16', ' K89/34', ' K7/45', ' K39/25']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K14/16', ' K89/34', ' K7/45', ' K39/25']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K14/16', ' K89/34', ' K7/45', ' K39/25']",
         "offsetgroup": "['K14/16', ' K89/34', ' K7/45', ' K39/25']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' N59/20', ' K65/35', ' K36/15', ' L21/32', ' N46/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' N59/20', ' K65/35', ' K36/15', ' L21/32', ' N46/30']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' N59/20', ' K65/35', ' K36/15', ' L21/32', ' N46/30']",
         "offsetgroup": "['K20/78', ' N59/20', ' K65/35', ' K36/15', ' L21/32', ' N46/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' A12/18']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' A12/18']",
         "offsetgroup": "['K65/35', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N195A/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N195A/16']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N195A/16']",
         "offsetgroup": "['N195A/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' L21/32', ' N355/1']",
         "offsetgroup": "['N59/36', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N59/36', ' N327/95']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N59/36', ' N327/95']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N59/36', ' N327/95']",
         "offsetgroup": "['N308/48', ' N59/36', ' N327/95']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N155/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N155/9']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N155/9']",
         "offsetgroup": "['N155/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N56/21', ' K75/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N56/21', ' K75/41']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N56/21', ' K75/41']",
         "offsetgroup": "['N56/21', ' K75/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L124/59', ' L21/32', ' K67/25', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L124/59', ' L21/32', ' K67/25', ' N355/1']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L124/59', ' L21/32', ' K67/25', ' N355/1']",
         "offsetgroup": "['L124/59', ' L21/32', ' K67/25', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' L6/60', ' K65/35', ' L6/48']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' L6/60', ' K65/35', ' L6/48']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' L6/60', ' K65/35', ' L6/48']",
         "offsetgroup": "['K87A/10', ' L6/60', ' K65/35', ' L6/48']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N209C/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N209C/35']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N209C/35']",
         "offsetgroup": "['N209C/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/86', ' L20/8', ' N18/28', ' N19/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/86', ' L20/8', ' N18/28', ' N19/2']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/86', ' L20/8', ' N18/28', ' N19/2']",
         "offsetgroup": "['K28/86', ' L20/8', ' N18/28', ' N19/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' N355/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' N355/1']",
         "offsetgroup": "['N28/9', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['ANTIBODY/1F1', ' N67/15']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['ANTIBODY/1F1', ' N67/15']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['ANTIBODY/1F1', ' N67/15']",
         "offsetgroup": "['ANTIBODY/1F1', ' N67/15']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K89/34', ' N112B/14', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K89/34', ' N112B/14', ' K7/45']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K89/34', ' N112B/14', ' K7/45']",
         "offsetgroup": "['K60/73', ' K89/34', ' N112B/14', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N22/21', ' N19/2', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N22/21', ' N19/2', ' N23B/6']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N22/21', ' N19/2', ' N23B/6']",
         "offsetgroup": "['N22/21', ' N19/2', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' L21/32', ' N355/1']",
         "offsetgroup": "['N308/48', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['62/3G1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['62/3G1']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['62/3G1']",
         "offsetgroup": "['62/3G1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' N52A/42', ' K89/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' N52A/42', ' K89/34']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' N52A/42', ' K89/34']",
         "offsetgroup": "['K13/31', ' N52A/42', ' K89/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N141/28']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N141/28']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N141/28']",
         "offsetgroup": "['N141/28']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N93A/49', ' N160/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N93A/49', ' N160/21']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N93A/49', ' N160/21']",
         "offsetgroup": "['N93A/49', ' N160/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N336B/83', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N336B/83', ' N59/20']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N336B/83', ' N59/20']",
         "offsetgroup": "['N336B/83', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' N124B/38', ' N112B/14', ' K60/73', ' K75/41']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' N124B/38', ' N112B/14', ' K60/73', ' K75/41']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' N124B/38', ' N112B/14', ' K60/73', ' K75/41']",
         "offsetgroup": "['K13/31', ' N124B/38', ' N112B/14', ' K60/73', ' K75/41']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N23B/49']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N23B/49']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N23B/49']",
         "offsetgroup": "['N308/48', ' N23B/49']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N429/19']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N429/19']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N429/19']",
         "offsetgroup": "['N429/19']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N59/36', ' N308/48', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N59/36', ' N308/48', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N59/36', ' N308/48', ' L21/32', ' N355/1']",
         "offsetgroup": "['N327/95', ' N59/36', ' N308/48', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N43/6', ' N37A/10', ' N152B/23']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N43/6', ' N37A/10', ' N152B/23']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N43/6', ' N37A/10', ' N152B/23']",
         "offsetgroup": "['N43/6', ' N37A/10', ' N152B/23']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' N263/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' N263/31']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' N263/31']",
         "offsetgroup": "['K75/41', ' N263/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' K14/16']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' K14/16']",
         "offsetgroup": "['N106/36', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' K65/35']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' K65/35']",
         "offsetgroup": "['K74/71', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' N56/21', ' K65/35', ' K74/71']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' N56/21', ' K65/35', ' K74/71']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' N56/21', ' K65/35', ' K74/71']",
         "offsetgroup": "['K87A/10', ' N56/21', ' K65/35', ' K74/71']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N105/13', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N105/13', ' N106/36']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N105/13', ' N106/36']",
         "offsetgroup": "['N105/13', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N95/35', ' N97A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N95/35', ' N97A/31']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N95/35', ' N97A/31']",
         "offsetgroup": "['N95/35', ' N97A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' K75/41', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' K75/41', ' K57/1']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' K75/41', ' K57/1']",
         "offsetgroup": "['K13/31', ' K75/41', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N327/95', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N327/95', ' L21/32']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N327/95', ' L21/32']",
         "offsetgroup": "['N308/48', ' N327/95', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N59/36', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N59/36', ' K57/1']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N59/36', ' K57/1']",
         "offsetgroup": "['N308/48', ' N59/36', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K60/73', ' K75/41', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K60/73', ' K75/41', ' K57/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K60/73', ' K75/41', ' K57/1']",
         "offsetgroup": "['K60/73', ' K75/41', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/20', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/20', ' N355/1']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/20', ' N355/1']",
         "offsetgroup": "['N59/20', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K9/40']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K9/40']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K9/40']",
         "offsetgroup": "['K9/40']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' A12/18', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' A12/18', ' N106/20']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' A12/18', ' N106/20']",
         "offsetgroup": "['N106/36', ' A12/18', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N263/31', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N263/31', ' K56A/50']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N263/31', ' K56A/50']",
         "offsetgroup": "['N263/31', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N183/15', ' N182/17', ' N154/32', ' N147/6', ' N195A/16', ' N106/65']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N183/15', ' N182/17', ' N154/32', ' N147/6', ' N195A/16', ' N106/65']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N183/15', ' N182/17', ' N154/32', ' N147/6', ' N195A/16', ' N106/65']",
         "offsetgroup": "['N183/15', ' N182/17', ' N154/32', ' N147/6', ' N195A/16', ' N106/65']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K55/82', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K55/82', ' K57/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K55/82', ' K57/1']",
         "offsetgroup": "['K75/41', ' K55/82', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N422/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N422/18']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N422/18']",
         "offsetgroup": "['N422/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2019,
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' L23/27', ' K19/36', ' K20/78', ' L6/60']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' L23/27', ' K19/36', ' K20/78', ' L6/60']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' L23/27', ' K19/36', ' K20/78', ' L6/60']",
         "offsetgroup": "['K13/31', ' L23/27', ' K19/36', ' K20/78', ' L6/60']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['ANTIBODY/1F1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['ANTIBODY/1F1']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['ANTIBODY/1F1']",
         "offsetgroup": "['ANTIBODY/1F1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N355/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N355/1']",
         "offsetgroup": "['N23B/49', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N52A/42', ' N19/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N52A/42', ' N19/2']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N52A/42', ' N19/2']",
         "offsetgroup": "['N52A/42', ' N19/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15', ' K14/16', ' K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15', ' K14/16', ' K19/36']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15', ' K14/16', ' K19/36']",
         "offsetgroup": "['K36/15', ' K14/16', ' K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N87/25', ' K28/43', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N87/25', ' K28/43', ' L21/32']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N87/25', ' K28/43', ' L21/32']",
         "offsetgroup": "['N87/25', ' K28/43', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L21/32', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L21/32', ' N95/35']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L21/32', ' N95/35']",
         "offsetgroup": "['L21/32', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N18/30', ' N25/35', ' N46/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N18/30', ' N25/35', ' N46/30']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N18/30', ' N25/35', ' N46/30']",
         "offsetgroup": "['N18/30', ' N25/35', ' N46/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' A12/18', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' A12/18', ' K56A/50']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' A12/18', ' K56A/50']",
         "offsetgroup": "['N106/36', ' A12/18', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K14/16', ' K36/15', ' N106/36', ' N114/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K14/16', ' K36/15', ' N106/36', ' N114/10']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K14/16', ' K36/15', ' N106/36', ' N114/10']",
         "offsetgroup": "['K14/16', ' K36/15', ' N106/36', ' N114/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' K91/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' K91/36']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' K91/36']",
         "offsetgroup": "['N86/8', ' K91/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' N16B/8']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' N16B/8']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' N16B/8']",
         "offsetgroup": "['K20/78', ' N16B/8']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N393/76', ' A12/18']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N393/76', ' A12/18']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N393/76', ' A12/18']",
         "offsetgroup": "['N393/76', ' A12/18']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N106/36']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N106/36']",
         "offsetgroup": "['N86/8', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' N19/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' N19/2']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' N19/2']",
         "offsetgroup": "['N28/9', ' N19/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N295B/66', ' K74/71']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N295B/66', ' K74/71']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N295B/66', ' K74/71']",
         "offsetgroup": "['N295B/66', ' K74/71']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N133/21', ' K28/74']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N133/21', ' K28/74']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N133/21', ' K28/74']",
         "offsetgroup": "['N133/21', ' K28/74']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' N106/36', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' N106/36', ' K14/16']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' N106/36', ' K14/16']",
         "offsetgroup": "['K65/35', ' N106/36', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N244/5', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N244/5', ' K57/1']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N244/5', ' K57/1']",
         "offsetgroup": "['N244/5', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N367/62', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N367/62', ' L21/32']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N367/62', ' L21/32']",
         "offsetgroup": "['N367/62', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' L21/32', ' N355/1']",
         "offsetgroup": "['N28/9', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N106/20']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N106/20']",
         "offsetgroup": "['N106/36', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N96/55', ' 62/3G1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N96/55', ' 62/3G1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N96/55', ' 62/3G1']",
         "offsetgroup": "['N96/55', ' 62/3G1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' N68/6', ' N106/20', ' N105/13', ' N393/76']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' N68/6', ' N106/20', ' N105/13', ' N393/76']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' N68/6', ' N106/20', ' N105/13', ' N393/76']",
         "offsetgroup": "['K74/71', ' N68/6', ' N106/20', ' N105/13', ' N393/76']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N52A/42']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N52A/42']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N52A/42']",
         "offsetgroup": "['N86/8', ' N52A/42']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N18/28', ' N19/2', ' K28/74']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N18/28', ' N19/2', ' K28/74']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N18/28', ' N19/2', ' K28/74']",
         "offsetgroup": "['N18/28', ' N19/2', ' K28/74']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' N19/2', ' K28/74']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' N19/2', ' K28/74']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' N19/2', ' K28/74']",
         "offsetgroup": "['N28/9', ' N19/2', ' K28/74']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K87A/10', ' K74/71', ' N68/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K87A/10', ' K74/71', ' N68/6']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K87A/10', ' K74/71', ' N68/6']",
         "offsetgroup": "['K87A/10', ' K74/71', ' N68/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L102/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L102/45']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L102/45']",
         "offsetgroup": "['L102/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' L21/32', ' N19/2', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' L21/32', ' N19/2', ' K56A/50']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' L21/32', ' N19/2', ' K56A/50']",
         "offsetgroup": "['N59/36', ' L21/32', ' N19/2', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K14/16', ' A12/18', ' N46/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K14/16', ' A12/18', ' N46/30']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K14/16', ' A12/18', ' N46/30']",
         "offsetgroup": "['K14/16', ' A12/18', ' N46/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' N355/1', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' N355/1', ' N95/35']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' N355/1', ' N95/35']",
         "offsetgroup": "['N28/9', ' N355/1', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['62/3G1', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['62/3G1', ' N95/35']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['62/3G1', ' N95/35']",
         "offsetgroup": "['62/3G1', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N46/30', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N46/30', ' N106/20']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N46/30', ' N106/20']",
         "offsetgroup": "['N106/36', ' N46/30', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' L21/32', ' N19/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' L21/32', ' N19/2']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' L21/32', ' N19/2']",
         "offsetgroup": "['N59/36', ' L21/32', ' N19/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K55/82', ' K14/16', ' K57/1', ' K60/73', ' L28/4']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K55/82', ' K14/16', ' K57/1', ' K60/73', ' L28/4']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K55/82', ' K14/16', ' K57/1', ' K60/73', ' L28/4']",
         "offsetgroup": "['K55/82', ' K14/16', ' K57/1', ' K60/73', ' L28/4']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' N34/34']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' N34/34']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' N34/34']",
         "offsetgroup": "['N28/9', ' N34/34']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N59/36', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N59/36', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N59/36', ' L21/32', ' N355/1']",
         "offsetgroup": "['N308/48', ' N59/36', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N419/40', ' K74/71']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N419/40', ' K74/71']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N419/40', ' K74/71']",
         "offsetgroup": "['N419/40', ' K74/71']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N355/1']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N355/1']",
         "offsetgroup": "['N86/8', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K36/15', ' K75/41', ' K14/16', ' K19/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K36/15', ' K75/41', ' K14/16', ' K19/36']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K36/15', ' K75/41', ' K14/16', ' K19/36']",
         "offsetgroup": "['K36/15', ' K75/41', ' K14/16', ' K19/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N244/5', ' K67/25', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N244/5', ' K67/25', ' K56A/50']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N244/5', ' K67/25', ' K56A/50']",
         "offsetgroup": "['N244/5', ' K67/25', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N56/21', ' N393/76']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N56/21', ' N393/76']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N56/21', ' N393/76']",
         "offsetgroup": "['N56/21', ' N393/76']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L106/83']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L106/83']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L106/83']",
         "offsetgroup": "['L106/83']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N308/48', ' N28/9']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N308/48', ' N28/9']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N308/48', ' N28/9']",
         "offsetgroup": "['N308/48', ' N28/9']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' L118/80']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' L118/80']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' L118/80']",
         "offsetgroup": "['N28/9', ' L118/80']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N87/25', ' L124/59', ' K28/43', ' L21/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N87/25', ' L124/59', ' K28/43', ' L21/32']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N87/25', ' L124/59', ' K28/43', ' L21/32']",
         "offsetgroup": "['N87/25', ' L124/59', ' K28/43', ' L21/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N59/36', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N59/36', ' N355/1']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N59/36', ' N355/1']",
         "offsetgroup": "['N59/36', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K74/71', ' N106/36', ' N56/21', ' K65/35', ' K87A/10', ' N419/40']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K74/71', ' N106/36', ' N56/21', ' K65/35', ' K87A/10', ' N419/40']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K74/71', ' N106/36', ' N56/21', ' K65/35', ' K87A/10', ' N419/40']",
         "offsetgroup": "['K74/71', ' N106/36', ' N56/21', ' K65/35', ' K87A/10', ' N419/40']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N105/17', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N105/17', ' N106/20']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N105/17', ' N106/20']",
         "offsetgroup": "['N106/36', ' N105/17', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N23B/49', ' N59/20', ' K56A/50']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N23B/49', ' N59/20', ' K56A/50']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N23B/49', ' N59/20', ' K56A/50']",
         "offsetgroup": "['N23B/49', ' N59/20', ' K56A/50']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N206A/8', ' N411/51']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N206A/8', ' N411/51']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N206A/8', ' N411/51']",
         "offsetgroup": "['N206A/8', ' N411/51']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N23B/6']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N23B/6']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N23B/6']",
         "offsetgroup": "['K28/43', ' N23B/6']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' N423/75', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' N423/75', ' K55/7']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' N423/75', ' K55/7']",
         "offsetgroup": "['K75/41', ' N423/75', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L114/3', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L114/3', ' N106/36']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L114/3', ' N106/36']",
         "offsetgroup": "['L114/3', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N393/76']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N393/76']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N393/76']",
         "offsetgroup": "['N106/36', ' N393/76']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K91/36', ' K66/38', ' K55/7']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K91/36', ' K66/38', ' K55/7']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K91/36', ' K66/38', ' K55/7']",
         "offsetgroup": "['K91/36', ' K66/38', ' K55/7']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N87/25', ' N415/24', ' N452/69', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N87/25', ' N415/24', ' N452/69', ' N95/35']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N87/25', ' N415/24', ' N452/69', ' N95/35']",
         "offsetgroup": "['N87/25', ' N415/24', ' N452/69', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N103/39', ' N103/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N103/39', ' N103/31']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N103/39', ' N103/31']",
         "offsetgroup": "['N103/39', ' N103/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L109/57', ' N59/36', ' K89/34', ' N106/20', ' K75/41', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L109/57', ' N59/36', ' K89/34', ' N106/20', ' K75/41', ' N355/1']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L109/57', ' N59/36', ' K89/34', ' N106/20', ' K75/41', ' N355/1']",
         "offsetgroup": "['L109/57', ' N59/36', ' K89/34', ' N106/20', ' K75/41', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N106/36', ' N393/76', ' N106/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N106/36', ' N393/76', ' N106/20']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N106/36', ' N393/76', ' N106/20']",
         "offsetgroup": "['N106/36', ' N393/76', ' N106/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N95/35', ' N97A/31']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N95/35', ' N97A/31']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N95/35', ' N97A/31']",
         "offsetgroup": "['K28/43', ' N95/35', ' N97A/31']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L6/23']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L6/23']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L6/23']",
         "offsetgroup": "['L6/23']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021,
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K20/78', ' K36/15', ' K9/40']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K20/78', ' K36/15', ' K9/40']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K20/78', ' K36/15', ' K9/40']",
         "offsetgroup": "['K20/78', ' K36/15', ' K9/40']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K13/31', ' N106/36', ' N59/36', ' K14/16', ' K36/15', ' N18/30']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K13/31', ' N106/36', ' N59/36', ' K14/16', ' K36/15', ' N18/30']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K13/31', ' N106/36', ' N59/36', ' K14/16', ' K36/15', ' N18/30']",
         "offsetgroup": "['K13/31', ' N106/36', ' N59/36', ' K14/16', ' K36/15', ' N18/30']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K91/36', ' 62/3G1', ' N110/29']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K91/36', ' 62/3G1', ' N110/29']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K91/36', ' 62/3G1', ' N110/29']",
         "offsetgroup": "['K91/36', ' 62/3G1', ' N110/29']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N419/40', ' K65/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N419/40', ' K65/35']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N419/40', ' K65/35']",
         "offsetgroup": "['N419/40', ' K65/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N95/35', ' N308/48', ' L21/32', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N95/35', ' N308/48', ' L21/32', ' N59/20']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N95/35', ' N308/48', ' L21/32', ' N59/20']",
         "offsetgroup": "['N327/95', ' N95/35', ' N308/48', ' L21/32', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N70/28', ' N52A/42']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N70/28', ' N52A/42']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N70/28', ' N52A/42']",
         "offsetgroup": "['N70/28', ' N52A/42']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K91/36', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K91/36', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K91/36', ' L21/32', ' N355/1']",
         "offsetgroup": "['K91/36', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K37/89', ' K14/16']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K37/89', ' K14/16']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K37/89', ' K14/16']",
         "offsetgroup": "['K37/89', ' K14/16']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N144/14', ' N59/20']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N144/14', ' N59/20']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N144/14', ' N59/20']",
         "offsetgroup": "['N144/14', ' N59/20']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N28/9', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N28/9', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N28/9', ' L21/32', ' N355/1']",
         "offsetgroup": "['K28/43', ' N28/9', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N59/36', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N59/36', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N59/36', ' L21/32', ' N355/1']",
         "offsetgroup": "['N327/95', ' N59/36', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L102/45', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L102/45', ' N355/1']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L102/45', ' N355/1']",
         "offsetgroup": "['L102/45', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L106/83', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L106/83', ' N95/35']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L106/83', ' N95/35']",
         "offsetgroup": "['L106/83', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N385/21', ' N106/36', ' L109/57']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N385/21', ' N106/36', ' L109/57']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N385/21', ' N106/36', ' L109/57']",
         "offsetgroup": "['N385/21', ' N106/36', ' L109/57']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N206A/8', ' N106/36']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N206A/8', ' N106/36']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N206A/8', ' N106/36']",
         "offsetgroup": "['N206A/8', ' N106/36']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N87/25', ' N452/69', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N87/25', ' N452/69', ' N95/35']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N87/25', ' N452/69', ' N95/35']",
         "offsetgroup": "['N87/25', ' N452/69', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N94/17', ' N152B/23']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N94/17', ' N152B/23']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N94/17', ' N152B/23']",
         "offsetgroup": "['N94/17', ' N152B/23']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N43/6', ' N37A/10']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N43/6', ' N37A/10']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N43/6', ' N37A/10']",
         "offsetgroup": "['N43/6', ' N37A/10']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N10/7', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N10/7', ' K28/43']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N10/7', ' K28/43']",
         "offsetgroup": "['N10/7', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N86/8', ' N23B/49']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N86/8', ' N23B/49']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N86/8', ' N23B/49']",
         "offsetgroup": "['N86/8', ' N23B/49']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K17/70', ' K9/40', ' L23/27']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K17/70', ' K9/40', ' L23/27']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K17/70', ' K9/40', ' L23/27']",
         "offsetgroup": "['K17/70', ' K9/40', ' L23/27']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K17/70', ' K7/45']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K17/70', ' K7/45']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K17/70', ' K7/45']",
         "offsetgroup": "['K17/70', ' K7/45']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N95/35', ' N28/9', ' L118/80', ' L106/83', ' N308/48']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N95/35', ' N28/9', ' L118/80', ' L106/83', ' N308/48']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N95/35', ' N28/9', ' L118/80', ' L106/83', ' N308/48']",
         "offsetgroup": "['N95/35', ' N28/9', ' L118/80', ' L106/83', ' N308/48']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N486/32']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N486/32']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N486/32']",
         "offsetgroup": "['N486/32']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['L119/106']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['L119/106']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['L119/106']",
         "offsetgroup": "['L119/106']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N244/5', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N244/5', ' K28/43']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N244/5', ' K28/43']",
         "offsetgroup": "['N244/5', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N28/9', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N28/9', ' N95/35']",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N28/9', ' N95/35']",
         "offsetgroup": "['N28/9', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2022
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N385/21']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N385/21']",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N385/21']",
         "offsetgroup": "['N385/21']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N393/76', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N393/76', ' N95/35']",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N393/76', ' N95/35']",
         "offsetgroup": "['N393/76', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K65/35', ' N144/14']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K65/35', ' N144/14']",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K65/35', ' N144/14']",
         "offsetgroup": "['K65/35', ' N144/14']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N327/95', ' N59/20', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N327/95', ' N59/20', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N327/95', ' N59/20', ' L21/32', ' N355/1']",
         "offsetgroup": "['N327/95', ' N59/20', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K75/41', ' K89/34', ' K7/45', ' K57/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K75/41', ' K89/34', ' K7/45', ' K57/1']",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K75/41', ' K89/34', ' K7/45', ' K57/1']",
         "offsetgroup": "['K75/41', ' K89/34', ' K7/45', ' K57/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N95/35', ' N96/55', ' K28/43', ' N28/9', ' N97A/31', ' L21/32', ' N355/1']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N95/35', ' N96/55', ' K28/43', ' N28/9', ' N97A/31', ' L21/32', ' N355/1']",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N95/35', ' N96/55', ' K28/43', ' N28/9', ' N97A/31', ' L21/32', ' N355/1']",
         "offsetgroup": "['N95/35', ' N96/55', ' K28/43', ' N28/9', ' N97A/31', ' L21/32', ' N355/1']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K89/34', ' K28/43']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K89/34', ' K28/43']",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K89/34', ' K28/43']",
         "offsetgroup": "['K89/34', ' K28/43']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['K28/43', ' N95/35']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['K28/43', ' N95/35']",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['K28/43', ' N95/35']",
         "offsetgroup": "['K28/43', ' N95/35']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=['N393/2']<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "['N393/2']",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "['N393/2']",
         "offsetgroup": "['N393/2']",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2023
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'review article'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'review article'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'review article'}",
         "offsetgroup": "{'review article'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2006,
          2007,
          2008,
          2008,
          2008,
          2008,
          2009,
          2012,
          2012,
          2012,
          2012,
          2012,
          2013,
          2014,
          2014,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43'}",
         "offsetgroup": "{'K28/43'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2006,
          2006,
          2007,
          2008,
          2008,
          2008,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2011,
          2012,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2012,
          2011,
          2012,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2012,
          2012,
          2012,
          2012,
          2012,
          2013,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2018,
          2019,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K14/16'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K14/16'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K14/16'}",
         "offsetgroup": "{'K14/16'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007,
          2007,
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K89/34'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K89/34'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K89/34'}",
         "offsetgroup": "{'K89/34'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007,
          2008,
          2008,
          2009,
          2009
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L21/32'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L21/32'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L21/32'}",
         "offsetgroup": "{'L21/32'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2007,
          2010,
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K37/89'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K37/89'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K37/89'}",
         "offsetgroup": "{'K37/89'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N103/39', 'N103/31'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N103/39', 'N103/31'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N103/39', 'N103/31'}",
         "offsetgroup": "{'N103/39', 'N103/31'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L57/46'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L57/46'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L57/46'}",
         "offsetgroup": "{'L57/46'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N16B/8'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N16B/8'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N16B/8'}",
         "offsetgroup": "{'N16B/8'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N76/8'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N76/8'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N76/8'}",
         "offsetgroup": "{'N76/8'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2008,
          2009,
          2011,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N23B/49'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N23B/49'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N23B/49'}",
         "offsetgroup": "{'N23B/49'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N75/3'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N75/3'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N75/3'}",
         "offsetgroup": "{'N75/3'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N81/37'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N81/37'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N81/37'}",
         "offsetgroup": "{'N81/37'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2009,
          2013,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N86.36'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N86.36'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N86.36'}",
         "offsetgroup": "{'N86.36'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N154/32'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N154/32'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N154/32'}",
         "offsetgroup": "{'N154/32'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2016,
          2016,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N83/48'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N83/48'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N83/48'}",
         "offsetgroup": "{'N83/48'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2012,
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43', 'L21/32'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43', 'L21/32'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43', 'L21/32'}",
         "offsetgroup": "{'K28/43', 'L21/32'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N55/10'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N55/10'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N55/10'}",
         "offsetgroup": "{'N55/10'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2011,
          2011,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L51/82'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L51/82'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L51/82'}",
         "offsetgroup": "{'L51/82'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2012,
          2013,
          2013,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N74/25'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N74/25'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N74/25'}",
         "offsetgroup": "{'N74/25'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2011,
          2012,
          2014,
          2014,
          2014,
          2014,
          2016,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N69/46'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N69/46'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N69/46'}",
         "offsetgroup": "{'N69/46'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2010,
          2012,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N106/36'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N106/36'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N106/36'}",
         "offsetgroup": "{'N106/36'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2011,
          2012,
          2012,
          2013,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L36/12'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L36/12'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L36/12'}",
         "offsetgroup": "{'L36/12'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2014,
          2016,
          2016,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N127/31', 'N22/21', 'N23B/6', 'K28/43', 'N238/31', 'N110/29'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N127/31', 'N22/21', 'N23B/6', 'K28/43', 'N238/31', 'N110/29'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N127/31', 'N22/21', 'N23B/6', 'K28/43', 'N238/31', 'N110/29'}",
         "offsetgroup": "{'N127/31', 'N22/21', 'N23B/6', 'K28/43', 'N238/31', 'N110/29'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N106/36', 'N106/65'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N106/36', 'N106/65'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N106/36', 'N106/65'}",
         "offsetgroup": "{'N106/36', 'N106/65'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N29/29'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N29/29'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N29/29'}",
         "offsetgroup": "{'N29/29'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2017,
          2018,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N103/31'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N103/31'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N103/31'}",
         "offsetgroup": "{'N103/31'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N103/39'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N103/39'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N103/39'}",
         "offsetgroup": "{'N103/39'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N81/2'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N81/2'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N81/2'}",
         "offsetgroup": "{'N81/2'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2012,
          2013,
          2016,
          2018,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N106'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N106'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N106'}",
         "offsetgroup": "{'N106'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011,
          2013,
          2013,
          2014,
          2016,
          2017,
          2017,
          2018,
          2018,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N59', 'L21/32'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N59', 'L21/32'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N59', 'L21/32'}",
         "offsetgroup": "{'N59', 'L21/32'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2011
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K88/64'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K88/64'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K88/64'}",
         "offsetgroup": "{'K88/64'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N86'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N86'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N86'}",
         "offsetgroup": "{'N86'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2014,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016,
          2017,
          2016,
          2016,
          2017,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N295B/66'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N295B/66'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N295B/66'}",
         "offsetgroup": "{'N295B/66'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012,
          2013,
          2013,
          2014,
          2014,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N81/2', 'K57/1'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N81/2', 'K57/1'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N81/2', 'K57/1'}",
         "offsetgroup": "{'N81/2', 'K57/1'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N89/34'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N89/34'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N89/34'}",
         "offsetgroup": "{'N89/34'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2012
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K8/43'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K8/43'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K8/43'}",
         "offsetgroup": "{'K8/43'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N165/38'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N165/38'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N165/38'}",
         "offsetgroup": "{'N165/38'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L24/18'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L24/18'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L24/18'}",
         "offsetgroup": "{'L24/18'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N29/9'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N29/9'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N29/9'}",
         "offsetgroup": "{'N29/9'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID=method reference<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "method reference",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "method reference",
         "offsetgroup": "method reference",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N106/65'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N106/65'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N106/65'}",
         "offsetgroup": "{'N106/65'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2017,
          2019,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28', 'N86'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28', 'N86'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28', 'N86'}",
         "offsetgroup": "{'K28', 'N86'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43', 'N356'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43', 'N356'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43', 'N356'}",
         "offsetgroup": "{'K28/43', 'N356'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N8B/1'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N8B/1'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N8B/1'}",
         "offsetgroup": "{'N8B/1'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/86'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/86'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/86'}",
         "offsetgroup": "{'K28/86'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N78/8', 'N78/3'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N78/8', 'N78/3'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N78/8', 'N78/3'}",
         "offsetgroup": "{'N78/8', 'N78/3'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N295B/66', 'N103/31'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N295B/66', 'N103/31'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N295B/66', 'N103/31'}",
         "offsetgroup": "{'N295B/66', 'N103/31'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N308/48', 'K28/43'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N308/48', 'K28/43'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N308/48', 'K28/43'}",
         "offsetgroup": "{'N308/48', 'K28/43'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L48A/9'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L48A/9'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L48A/9'}",
         "offsetgroup": "{'L48A/9'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N39B/18', 'N83/48'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N39B/18', 'N83/48'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N39B/18', 'N83/48'}",
         "offsetgroup": "{'N39B/18', 'N83/48'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N68/6'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N68/6'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N68/6'}",
         "offsetgroup": "{'N68/6'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2013
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'EA7'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'EA7'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'EA7'}",
         "offsetgroup": "{'EA7'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N147/6'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N147/6'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N147/6'}",
         "offsetgroup": "{'N147/6'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N5/20', 'K28/43'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N5/20', 'K28/43'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N5/20', 'K28/43'}",
         "offsetgroup": "{'N5/20', 'K28/43'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N106', 'K28'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N106', 'K28'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N106', 'K28'}",
         "offsetgroup": "{'N106', 'K28'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N86/38'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N86/38'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N86/38'}",
         "offsetgroup": "{'N86/38'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2016,
          2016,
          2016,
          2017,
          2018,
          2018,
          2019,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N52A/42'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N52A/42'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N52A/42'}",
         "offsetgroup": "{'N52A/42'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L48A/9 or N38/8'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L48A/9 or N38/8'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L48A/9 or N38/8'}",
         "offsetgroup": "{'L48A/9 or N38/8'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N59', 'N308/48'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N59', 'N308/48'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N59', 'N308/48'}",
         "offsetgroup": "{'N59', 'N308/48'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N138/6'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N138/6'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N138/6'}",
         "offsetgroup": "{'N138/6'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K69/3'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K69/3'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K69/3'}",
         "offsetgroup": "{'K69/3'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2016,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N77/15'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N77/15'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N77/15'}",
         "offsetgroup": "{'N77/15'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014,
          2015,
          2016,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K20/78'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K20/78'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K20/78'}",
         "offsetgroup": "{'K20/78'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N40B/18'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N40B/18'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N40B/18'}",
         "offsetgroup": "{'N40B/18'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N75/33'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N75/33'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N75/33'}",
         "offsetgroup": "{'N75/33'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N327'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N327'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N327'}",
         "offsetgroup": "{'N327'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2014
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N385/21'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N385/21'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N385/21'}",
         "offsetgroup": "{'N385/21'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N327/95', 'K28/43'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N327/95', 'K28/43'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N327/95', 'K28/43'}",
         "offsetgroup": "{'N327/95', 'K28/43'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N351/24'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N351/24'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N351/24'}",
         "offsetgroup": "{'N351/24'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N271/44'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N271/44'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N271/44'}",
         "offsetgroup": "{'N271/44'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43 or K28/74'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43 or K28/74'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43 or K28/74'}",
         "offsetgroup": "{'K28/43 or K28/74'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2017,
          2017,
          2018,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K39/25 or K89/34 or L80/21 or L83/11'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K39/25 or K89/34 or L80/21 or L83/11'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K39/25 or K89/34 or L80/21 or L83/11'}",
         "offsetgroup": "{'K39/25 or K89/34 or L80/21 or L83/11'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2015,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2019,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N206'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N206'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N206'}",
         "offsetgroup": "{'N206'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43', 'N106/65'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43', 'N106/65'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43', 'N106/65'}",
         "offsetgroup": "{'K28/43', 'N106/65'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28'}",
         "offsetgroup": "{'K28'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N384/63'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N384/63'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N384/63'}",
         "offsetgroup": "{'N384/63'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N65/37'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N65/37'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N65/37'}",
         "offsetgroup": "{'N65/37'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2015
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28', 'N28/9'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28', 'N28/9'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28', 'N28/9'}",
         "offsetgroup": "{'K28', 'N28/9'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N170A'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N170A'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N170A'}",
         "offsetgroup": "{'N170A'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N221/17'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N221/17'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N221/17'}",
         "offsetgroup": "{'N221/17'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28', 'L21/32'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28', 'L21/32'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28', 'L21/32'}",
         "offsetgroup": "{'K28', 'L21/32'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N28/9'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N28/9'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N28/9'}",
         "offsetgroup": "{'N28/9'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2017,
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28', 'N86', 'N245/44'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28', 'N86', 'N245/44'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28', 'N86', 'N245/44'}",
         "offsetgroup": "{'K28', 'N86', 'N245/44'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N59/36', 'K28/43'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N59/36', 'K28/43'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N59/36', 'K28/43'}",
         "offsetgroup": "{'N59/36', 'K28/43'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N59/36', 'K28/43', 'L21/32'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N59/36', 'K28/43', 'L21/32'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N59/36', 'K28/43', 'L21/32'}",
         "offsetgroup": "{'N59/36', 'K28/43', 'L21/32'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N95/35'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N95/35'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N95/35'}",
         "offsetgroup": "{'N95/35'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N219/5'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N219/5'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N219/5'}",
         "offsetgroup": "{'N219/5'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N230/21'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N230/21'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N230/21'}",
         "offsetgroup": "{'N230/21'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N327/95', 'N59/20'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N327/95', 'N59/20'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N327/95', 'N59/20'}",
         "offsetgroup": "{'N327/95', 'N59/20'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N120A/9'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N120A/9'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N120A/9'}",
         "offsetgroup": "{'N120A/9'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N367/62'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N367/62'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N367/62'}",
         "offsetgroup": "{'N367/62'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N166A/26'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N166A/26'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N166A/26'}",
         "offsetgroup": "{'N166A/26'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N22/21', 'N23B/6', 'N18', 'N69/46 or N367/62', 'N81', 'N28/9'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N22/21', 'N23B/6', 'N18', 'N69/46 or N367/62', 'N81', 'N28/9'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N22/21', 'N23B/6', 'N18', 'N69/46 or N367/62', 'N81', 'N28/9'}",
         "offsetgroup": "{'N22/21', 'N23B/6', 'N18', 'N69/46 or N367/62', 'N81', 'N28/9'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N178A/9', 'N55/10'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N178A/9', 'N55/10'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N178A/9', 'N55/10'}",
         "offsetgroup": "{'N178A/9', 'N55/10'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N87/25', 'N308/48', 'K28/43', 'L21/32'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N87/25', 'N308/48', 'K28/43', 'L21/32'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N87/25', 'N308/48', 'K28/43', 'L21/32'}",
         "offsetgroup": "{'N87/25', 'N308/48', 'K28/43', 'L21/32'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L48A/9', 'N263/31'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L48A/9', 'N263/31'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L48A/9', 'N263/31'}",
         "offsetgroup": "{'L48A/9', 'N263/31'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28', 'L42/17'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28', 'L42/17'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28', 'L42/17'}",
         "offsetgroup": "{'K28', 'L42/17'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N23B/49', 'K28/43'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N23B/49', 'K28/43'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N23B/49', 'K28/43'}",
         "offsetgroup": "{'N23B/49', 'K28/43'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N23B/49', 'N127/31', 'K28/43'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N23B/49', 'N127/31', 'K28/43'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N23B/49', 'N127/31', 'K28/43'}",
         "offsetgroup": "{'N23B/49', 'N127/31', 'K28/43'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L43/40'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L43/40'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L43/40'}",
         "offsetgroup": "{'L43/40'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L36/12', 'N28/9'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L36/12', 'N28/9'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L36/12', 'N28/9'}",
         "offsetgroup": "{'L36/12', 'N28/9'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K64/15'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K64/15'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K64/15'}",
         "offsetgroup": "{'K64/15'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2017,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N87/25', 'N86'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N87/25', 'N86'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N87/25', 'N86'}",
         "offsetgroup": "{'N87/25', 'N86'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N332B/15'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N332B/15'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N332B/15'}",
         "offsetgroup": "{'N332B/15'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N81', 'N93A/49', 'N95/35'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N81', 'N93A/49', 'N95/35'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N81', 'N93A/49', 'N95/35'}",
         "offsetgroup": "{'N81', 'N93A/49', 'N95/35'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N26A/23', 'N106/36'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N26A/23', 'N106/36'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N26A/23', 'N106/36'}",
         "offsetgroup": "{'N26A/23', 'N106/36'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N393'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N393'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N393'}",
         "offsetgroup": "{'N393'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N343/26', 'L48A/9 or N38/8', 'L57/46 or N263/31'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N343/26', 'L48A/9 or N38/8', 'L57/46 or N263/31'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N343/26', 'L48A/9 or N38/8', 'L57/46 or N263/31'}",
         "offsetgroup": "{'N343/26', 'L48A/9 or N38/8', 'L57/46 or N263/31'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N1'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N1'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N1'}",
         "offsetgroup": "{'N1'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2016
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N18'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N18'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N18'}",
         "offsetgroup": "{'N18'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N59/36', 'L21/32'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N59/36', 'L21/32'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N59/36', 'L21/32'}",
         "offsetgroup": "{'N59/36', 'L21/32'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N18', 'N19/2'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N18', 'N19/2'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N18', 'N19/2'}",
         "offsetgroup": "{'N18', 'N19/2'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N4/15'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N4/15'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N4/15'}",
         "offsetgroup": "{'N4/15'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N262/16', 'N259/48', 'N257/25'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N262/16', 'N259/48', 'N257/25'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N262/16', 'N259/48', 'N257/25'}",
         "offsetgroup": "{'N262/16', 'N259/48', 'N257/25'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N125/10'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N125/10'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N125/10'}",
         "offsetgroup": "{'N125/10'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N129/15 or N129A/6'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N129/15 or N129A/6'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N129/15 or N129A/6'}",
         "offsetgroup": "{'N129/15 or N129A/6'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N100/13'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N100/13'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N100/13'}",
         "offsetgroup": "{'N100/13'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43 or K28/74', 'N28/9'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43 or K28/74', 'N28/9'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43 or K28/74', 'N28/9'}",
         "offsetgroup": "{'K28/43 or K28/74', 'N28/9'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43', 'N244/5'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43', 'N244/5'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43', 'N244/5'}",
         "offsetgroup": "{'K28/43', 'N244/5'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N202/7'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N202/7'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N202/7'}",
         "offsetgroup": "{'N202/7'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2018,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N19/2'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N19/2'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N19/2'}",
         "offsetgroup": "{'N19/2'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43', 'N97A/31'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43', 'N97A/31'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43', 'N97A/31'}",
         "offsetgroup": "{'K28/43', 'N97A/31'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N269/73'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N269/73'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N269/73'}",
         "offsetgroup": "{'N269/73'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N201/35'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N201/35'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N201/35'}",
         "offsetgroup": "{'N201/35'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N277/7', 'K28/43 or K28/74', 'N22/21'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N277/7', 'K28/43 or K28/74', 'N22/21'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N277/7', 'K28/43 or K28/74', 'N22/21'}",
         "offsetgroup": "{'N277/7', 'K28/43 or K28/74', 'N22/21'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N67/15'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N67/15'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N67/15'}",
         "offsetgroup": "{'N67/15'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2018,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N86', 'K69/3'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N86', 'K69/3'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N86', 'K69/3'}",
         "offsetgroup": "{'N86', 'K69/3'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N196/16'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N196/16'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N196/16'}",
         "offsetgroup": "{'N196/16'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2018,
          2019,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N69/46 or N367/62'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N69/46 or N367/62'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N69/46 or N367/62'}",
         "offsetgroup": "{'N69/46 or N367/62'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N18/28', 'K28/43', 'N110/29'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N18/28', 'K28/43', 'N110/29'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N18/28', 'K28/43', 'N110/29'}",
         "offsetgroup": "{'N18/28', 'K28/43', 'N110/29'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N336B/83', 'N332B/15'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N336B/83', 'N332B/15'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N336B/83', 'N332B/15'}",
         "offsetgroup": "{'N336B/83', 'N332B/15'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L60/4'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L60/4'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L60/4'}",
         "offsetgroup": "{'L60/4'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43 or K28/74', 'N238/31'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43 or K28/74', 'N238/31'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43 or K28/74', 'N238/31'}",
         "offsetgroup": "{'K28/43 or K28/74', 'N238/31'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N289/16'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N289/16'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N289/16'}",
         "offsetgroup": "{'N289/16'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N10/7'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N10/7'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N10/7'}",
         "offsetgroup": "{'N10/7'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N393/2'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N393/2'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N393/2'}",
         "offsetgroup": "{'N393/2'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N59/36', 'N59/20'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N59/36', 'N59/20'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N59/36', 'N59/20'}",
         "offsetgroup": "{'N59/36', 'N59/20'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L11A/41'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L11A/41'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L11A/41'}",
         "offsetgroup": "{'L11A/41'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2018,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N103'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N103'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N103'}",
         "offsetgroup": "{'N103'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43 or K28/74', 'K64/15'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43 or K28/74', 'K64/15'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43 or K28/74', 'K64/15'}",
         "offsetgroup": "{'K28/43 or K28/74', 'K64/15'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N327/95', 'N367/62', 'N75/3'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N327/95', 'N367/62', 'N75/3'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N327/95', 'N367/62', 'N75/3'}",
         "offsetgroup": "{'N327/95', 'N367/62', 'N75/3'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N112B/14'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N112B/14'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N112B/14'}",
         "offsetgroup": "{'N112B/14'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018,
          2019,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N29/29', 'N28/9', 'N59/20 or N59/36', 'N327/95 or N327A/38'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N29/29', 'N28/9', 'N59/20 or N59/36', 'N327/95 or N327A/38'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N29/29', 'N28/9', 'N59/20 or N59/36', 'N327/95 or N327A/38'}",
         "offsetgroup": "{'N29/29', 'N28/9', 'N59/20 or N59/36', 'N327/95 or N327A/38'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N212/3', 'N212A/34'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N212/3', 'N212A/34'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N212/3', 'N212A/34'}",
         "offsetgroup": "{'N212/3', 'N212A/34'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K17/70'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K17/70'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K17/70'}",
         "offsetgroup": "{'K17/70'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L118/66', 'L124/59', 'L117/1', 'K28/43', 'L120/12', 'L106/4', 'N28/9', 'L106/83', 'N178A/9', 'L113/130', 'L106/93'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L118/66', 'L124/59', 'L117/1', 'K28/43', 'L120/12', 'L106/4', 'N28/9', 'L106/83', 'N178A/9', 'L113/130', 'L106/93'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L118/66', 'L124/59', 'L117/1', 'K28/43', 'L120/12', 'L106/4', 'N28/9', 'L106/83', 'N178A/9', 'L113/130', 'L106/93'}",
         "offsetgroup": "{'L118/66', 'L124/59', 'L117/1', 'K28/43', 'L120/12', 'L106/4', 'N28/9', 'L106/83', 'N178A/9', 'L113/130', 'L106/93'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N421A/85'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N421A/85'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N421A/85'}",
         "offsetgroup": "{'N421A/85'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N212/17'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N212/17'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N212/17'}",
         "offsetgroup": "{'N212/17'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N70/28', 'N26A/23', 'N141/28', 'N114/10'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N70/28', 'N26A/23', 'N141/28', 'N114/10'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N70/28', 'N26A/23', 'N141/28', 'N114/10'}",
         "offsetgroup": "{'N70/28', 'N26A/23', 'N141/28', 'N114/10'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N212 or N212A'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N212 or N212A'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N212 or N212A'}",
         "offsetgroup": "{'N212 or N212A'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2018
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L6/60'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L6/60'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L6/60'}",
         "offsetgroup": "{'L6/60'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43 or K28/74', 'N59/20 or N59/36'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43 or K28/74', 'N59/20 or N59/36'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43 or K28/74', 'N59/20 or N59/36'}",
         "offsetgroup": "{'K28/43 or K28/74', 'N59/20 or N59/36'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N18/30', 'K64/15'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N18/30', 'K64/15'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N18/30', 'K64/15'}",
         "offsetgroup": "{'N18/30', 'K64/15'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L36/12', 'N295B/66'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L36/12', 'N295B/66'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L36/12', 'N295B/66'}",
         "offsetgroup": "{'L36/12', 'N295B/66'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N111/24'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N111/24'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N111/24'}",
         "offsetgroup": "{'N111/24'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K13/31 or L71/5', 'N68/6', 'K14/16 or L76/36'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K13/31 or L71/5', 'N68/6', 'K14/16 or L76/36'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K13/31 or L71/5', 'N68/6', 'K14/16 or L76/36'}",
         "offsetgroup": "{'K13/31 or L71/5', 'N68/6', 'K14/16 or L76/36'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N70/28', 'N26A/23'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N70/28', 'N26A/23'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N70/28', 'N26A/23'}",
         "offsetgroup": "{'N70/28', 'N26A/23'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N170A/1 or N170A/26'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N170A/1 or N170A/26'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N170A/1 or N170A/26'}",
         "offsetgroup": "{'N170A/1 or N170A/26'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N75/3 or N75/33', 'N18/28 or N18/30'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N75/3 or N75/33', 'N18/28 or N18/30'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N75/3 or N75/33', 'N18/28 or N18/30'}",
         "offsetgroup": "{'N75/3 or N75/33', 'N18/28 or N18/30'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N37A/10'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N37A/10'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N37A/10'}",
         "offsetgroup": "{'N37A/10'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N170A/1'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N170A/1'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N170A/1'}",
         "offsetgroup": "{'N170A/1'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N238/31'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N238/31'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N238/31'}",
         "offsetgroup": "{'N238/31'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N221/12 or N221/17'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N221/12 or N221/17'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N221/12 or N221/17'}",
         "offsetgroup": "{'N221/12 or N221/17'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N212/3', 'N212/17'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N212/3', 'N212/17'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N212/3', 'N212/17'}",
         "offsetgroup": "{'N212/3', 'N212/17'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2019
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N144/32'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N144/32'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N144/32'}",
         "offsetgroup": "{'N144/32'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N192/12'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N192/12'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N192/12'}",
         "offsetgroup": "{'N192/12'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N77/15', 'N67/15'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N77/15', 'N67/15'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N77/15', 'N67/15'}",
         "offsetgroup": "{'N77/15', 'N67/15'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N419/40', 'N106/65'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N419/40', 'N106/65'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N419/40', 'N106/65'}",
         "offsetgroup": "{'N419/40', 'N106/65'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N270/47', 'K28/43'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N270/47', 'K28/43'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N270/47', 'K28/43'}",
         "offsetgroup": "{'N270/47', 'K28/43'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N86/8 or N86/20 or N86/38 or N86/44'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N86/8 or N86/20 or N86/38 or N86/44'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N86/8 or N86/20 or N86/38 or N86/44'}",
         "offsetgroup": "{'N86/8 or N86/20 or N86/38 or N86/44'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020,
          2020,
          2020,
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N321C/49'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N321C/49'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N321C/49'}",
         "offsetgroup": "{'N321C/49'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'L86/33 or L86/36 or L86A/37'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'L86/33 or L86/36 or L86A/37'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'L86/33 or L86/36 or L86A/37'}",
         "offsetgroup": "{'L86/33 or L86/36 or L86A/37'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'62-3G1'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'62-3G1'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'62-3G1'}",
         "offsetgroup": "{'62-3G1'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N159/5'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N159/5'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N159/5'}",
         "offsetgroup": "{'N159/5'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N43/6'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N43/6'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N43/6'}",
         "offsetgroup": "{'N43/6'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N26A/23'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N26A/23'}",
         "marker": {
          "color": "#ab63fa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N26A/23'}",
         "offsetgroup": "{'N26A/23'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N294A/10'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N294A/10'}",
         "marker": {
          "color": "#FFA15A",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N294A/10'}",
         "offsetgroup": "{'N294A/10'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N241A/34'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N241A/34'}",
         "marker": {
          "color": "#19d3f3",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N241A/34'}",
         "offsetgroup": "{'N241A/34'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'1F1'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'1F1'}",
         "marker": {
          "color": "#FF6692",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'1F1'}",
         "offsetgroup": "{'1F1'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K28/43', 'N28/9'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K28/43', 'N28/9'}",
         "marker": {
          "color": "#B6E880",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K28/43', 'N28/9'}",
         "offsetgroup": "{'K28/43', 'N28/9'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N149/25'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N149/25'}",
         "marker": {
          "color": "#FF97FF",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N149/25'}",
         "offsetgroup": "{'N149/25'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K13/31 or L71/5'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K13/31 or L71/5'}",
         "marker": {
          "color": "#FECB52",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K13/31 or L71/5'}",
         "offsetgroup": "{'K13/31 or L71/5'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2020
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'K57/1'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'K57/1'}",
         "marker": {
          "color": "#636efa",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'K57/1'}",
         "offsetgroup": "{'K57/1'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N319A/14', 'N289/16'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N319A/14', 'N289/16'}",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N319A/14', 'N289/16'}",
         "offsetgroup": "{'N319A/14', 'N289/16'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "MAB_ID={'N212B'}<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "{'N212B'}",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "{'N212B'}",
         "offsetgroup": "{'N212B'}",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        }
       ],
       "layout": {
        "barmode": "relative",
        "legend": {
         "title": {
          "text": "MAB_ID"
         },
         "tracegroupgap": 0
        },
        "margin": {
         "t": 60
        },
        "template": {
         "data": {
          "bar": [
           {
            "error_x": {
             "color": "#2a3f5f"
            },
            "error_y": {
             "color": "#2a3f5f"
            },
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "bar"
           }
          ],
          "barpolar": [
           {
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "barpolar"
           }
          ],
          "carpet": [
           {
            "aaxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "baxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "type": "carpet"
           }
          ],
          "choropleth": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "choropleth"
           }
          ],
          "contour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "contour"
           }
          ],
          "contourcarpet": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "contourcarpet"
           }
          ],
          "heatmap": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmap"
           }
          ],
          "heatmapgl": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmapgl"
           }
          ],
          "histogram": [
           {
            "marker": {
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "histogram"
           }
          ],
          "histogram2d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2d"
           }
          ],
          "histogram2dcontour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2dcontour"
           }
          ],
          "mesh3d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "mesh3d"
           }
          ],
          "parcoords": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "parcoords"
           }
          ],
          "pie": [
           {
            "automargin": true,
            "type": "pie"
           }
          ],
          "scatter": [
           {
            "fillpattern": {
             "fillmode": "overlay",
             "size": 10,
             "solidity": 0.2
            },
            "type": "scatter"
           }
          ],
          "scatter3d": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatter3d"
           }
          ],
          "scattercarpet": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattercarpet"
           }
          ],
          "scattergeo": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergeo"
           }
          ],
          "scattergl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergl"
           }
          ],
          "scattermapbox": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattermapbox"
           }
          ],
          "scatterpolar": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolar"
           }
          ],
          "scatterpolargl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolargl"
           }
          ],
          "scatterternary": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterternary"
           }
          ],
          "surface": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "surface"
           }
          ],
          "table": [
           {
            "cells": {
             "fill": {
              "color": "#EBF0F8"
             },
             "line": {
              "color": "white"
             }
            },
            "header": {
             "fill": {
              "color": "#C8D4E3"
             },
             "line": {
              "color": "white"
             }
            },
            "type": "table"
           }
          ]
         },
         "layout": {
          "annotationdefaults": {
           "arrowcolor": "#2a3f5f",
           "arrowhead": 0,
           "arrowwidth": 1
          },
          "autotypenumbers": "strict",
          "coloraxis": {
           "colorbar": {
            "outlinewidth": 0,
            "ticks": ""
           }
          },
          "colorscale": {
           "diverging": [
            [
             0,
             "#8e0152"
            ],
            [
             0.1,
             "#c51b7d"
            ],
            [
             0.2,
             "#de77ae"
            ],
            [
             0.3,
             "#f1b6da"
            ],
            [
             0.4,
             "#fde0ef"
            ],
            [
             0.5,
             "#f7f7f7"
            ],
            [
             0.6,
             "#e6f5d0"
            ],
            [
             0.7,
             "#b8e186"
            ],
            [
             0.8,
             "#7fbc41"
            ],
            [
             0.9,
             "#4d9221"
            ],
            [
             1,
             "#276419"
            ]
           ],
           "sequential": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ],
           "sequentialminus": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ]
          },
          "colorway": [
           "#636efa",
           "#EF553B",
           "#00cc96",
           "#ab63fa",
           "#FFA15A",
           "#19d3f3",
           "#FF6692",
           "#B6E880",
           "#FF97FF",
           "#FECB52"
          ],
          "font": {
           "color": "#2a3f5f"
          },
          "geo": {
           "bgcolor": "white",
           "lakecolor": "white",
           "landcolor": "#E5ECF6",
           "showlakes": true,
           "showland": true,
           "subunitcolor": "white"
          },
          "hoverlabel": {
           "align": "left"
          },
          "hovermode": "closest",
          "mapbox": {
           "style": "light"
          },
          "paper_bgcolor": "white",
          "plot_bgcolor": "#E5ECF6",
          "polar": {
           "angularaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "radialaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "scene": {
           "xaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "yaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "zaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           }
          },
          "shapedefaults": {
           "line": {
            "color": "#2a3f5f"
           }
          },
          "ternary": {
           "aaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "baxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "caxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "title": {
           "x": 0.05
          },
          "xaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          },
          "yaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          }
         }
        },
        "xaxis": {
         "anchor": "y",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "year"
         }
        },
        "yaxis": {
         "anchor": "x",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "count"
         }
        }
       }
      }
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# remove anything before the year 2000\n",
    "df_cure = df_cure[df_cure['year'] >= 2000]\n",
    "# make the same plot\n",
    "fig = px.histogram(df_cure, x=\"year\", color=\"MAB_ID\")\n",
    "fig.show()\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "a52cc37a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "config": {
        "plotlyServerURL": "https://plot.ly"
       },
       "data": [
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "source=google scholar unique<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "google scholar unique",
         "marker": {
          "color": "#EF553B",
          "pattern": {
           "shape": ""
          }
         },
         "name": "google scholar unique",
         "offsetgroup": "google scholar unique",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2017,
          2017,
          2012,
          2023,
          2023,
          2023,
          2023,
          2023,
          2021,
          2021,
          2021,
          2021,
          2021,
          2015,
          2015,
          2015,
          2015,
          2015,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2020,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2015,
          2015,
          2015,
          2015,
          2015,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2016,
          2021,
          2017,
          2022,
          2022,
          2022,
          2022,
          2021,
          2018,
          2017,
          2016,
          2018,
          2021,
          2014,
          2020,
          2021,
          2021,
          2021,
          2020,
          2015,
          2023,
          2023,
          2019,
          2021,
          2021,
          2021,
          2023,
          2022,
          2022,
          2020,
          2020,
          2021,
          2021,
          2020,
          2020,
          2023,
          2023,
          2016,
          2023,
          2018,
          2023,
          2023,
          2021,
          2020,
          2020,
          2012,
          2020,
          2021,
          2015,
          2015,
          2012,
          2019,
          2019,
          2023,
          2023,
          2020,
          2023,
          2023,
          2022,
          2022,
          2017,
          2020,
          2018,
          2018,
          2020,
          2018,
          2020,
          2016,
          2022,
          2019,
          2023,
          2012,
          2021,
          2021,
          2021,
          2021,
          2021,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2021,
          2022,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2016,
          2016,
          2016,
          2016,
          2023,
          2019,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2023,
          2022,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2016,
          2017,
          2022,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2010,
          2022,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2020,
          2022,
          2022,
          2022,
          2022,
          2021,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2021,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2020,
          2017,
          2017,
          2017,
          2023,
          2023,
          2021,
          2021,
          2021,
          2021,
          2021,
          2016,
          2021,
          2016,
          2016,
          2016,
          2016,
          2016,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2019,
          2019,
          2013,
          2013,
          2013,
          2013,
          2020,
          2020,
          2020,
          2017,
          2017,
          2017,
          2017,
          2021,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2019,
          2019,
          2019,
          2019,
          2019,
          2021,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2018,
          2018,
          2018,
          2023,
          2023,
          2022,
          2013,
          2013,
          2013,
          2013,
          2017,
          2011,
          2023,
          2017,
          2017,
          2017,
          2017,
          2018,
          2023,
          2023,
          2011,
          2016,
          2016,
          2016,
          2018,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2019,
          2019,
          2020,
          2022,
          2023,
          2019,
          2017,
          2023,
          2018,
          2020,
          2023,
          2023,
          2021,
          2021,
          2021,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2021,
          2018,
          2018,
          2018,
          2018,
          2018,
          2021,
          2021,
          2021,
          2021,
          2015,
          2022,
          2021,
          2022,
          2020,
          2020,
          2021,
          2020,
          2017,
          2017,
          2017,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2017,
          2024,
          2024,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2017,
          2017,
          2017,
          2023,
          2023,
          2023,
          2019,
          2022,
          2021,
          2018,
          2018,
          2018,
          2021,
          2021,
          2021,
          2016,
          2019,
          2019,
          2020,
          2020,
          2020,
          2021,
          2022,
          2022,
          2022,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2023,
          2023,
          2017,
          2023,
          2021,
          2022,
          2022,
          2022,
          2022,
          2021,
          2014,
          2022,
          2020,
          2023,
          2023,
          2018,
          2018,
          2018,
          2018,
          2014,
          2015,
          2015,
          2015,
          2017,
          2023,
          2021,
          2022,
          2017,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2023,
          2023,
          2015,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2023,
          2023,
          2023,
          2023,
          2023,
          2018,
          2018,
          2018,
          2018,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2014,
          2014,
          2022,
          2022,
          2022,
          2019,
          2021,
          2022,
          2018,
          2018,
          2018,
          2018,
          2024,
          2024,
          2024,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2023,
          2021,
          2010,
          2016,
          2016,
          2021,
          2021,
          2021,
          2021,
          2016,
          2014,
          2016,
          2014,
          2021,
          2023,
          2021,
          2021,
          2021,
          2022,
          2022,
          2022,
          2023,
          2023,
          2023,
          2023,
          2023,
          2018,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2020,
          2023,
          2023,
          2023,
          2023,
          2023,
          2019,
          2019,
          2019,
          2019,
          2019,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2022,
          2022,
          2020,
          2020,
          2020,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2024,
          2024,
          2024,
          2023,
          2023,
          2023,
          2024,
          2024,
          2023,
          2023,
          2024,
          2023,
          2023,
          2023,
          2024,
          2017,
          2024,
          2024,
          2024,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2024,
          2024,
          2023,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2023,
          2023,
          2024,
          2024,
          2024,
          2023,
          2023,
          2023,
          2024,
          2024,
          2024,
          2024,
          2023,
          2023,
          2023,
          2024,
          2023,
          2023,
          2024,
          2024,
          2024,
          2022,
          2022,
          2021,
          2020,
          2021,
          2020,
          2020,
          2020,
          2023,
          2023,
          2023,
          2023,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2023,
          2023,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2023,
          2019,
          2019,
          2019,
          2024,
          2023,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2025,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2017,
          2017,
          2017,
          2022,
          2022,
          2022,
          2017,
          2017,
          2017,
          2017,
          2023,
          2023,
          2023,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024,
          2023,
          2024,
          2022,
          2022,
          2021,
          2023,
          2023,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2024,
          2024,
          2018,
          2022,
          2020,
          2022,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2024,
          2023,
          2023,
          2023,
          2021,
          2021,
          2021,
          2022,
          2022,
          2024,
          2023,
          2019,
          2024,
          2021,
          2021,
          2016,
          2016,
          2016,
          2020,
          2022,
          2022,
          2022,
          2020,
          2020,
          2021,
          2021,
          2022,
          2024,
          2022,
          2020,
          2022,
          2022,
          2022,
          2022,
          2021,
          2021,
          2021,
          2023,
          2023,
          2023,
          2023,
          2024,
          2023,
          2013,
          2013,
          2019,
          2021,
          2020,
          2021,
          2015,
          2022,
          2022,
          2016,
          2021,
          2023,
          2023,
          2023,
          2023,
          2024,
          2017,
          2021,
          2023,
          2016,
          2016,
          2016,
          2016,
          2020,
          2023,
          2021,
          2016,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2023,
          2023,
          2024,
          2024,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2024,
          2023,
          2023,
          2024,
          2024,
          2023,
          2023,
          2010,
          2023,
          2023,
          2017,
          2017,
          2017,
          2017,
          2017,
          2012,
          2012,
          2012,
          2015,
          2015,
          2015,
          2022,
          2022,
          2022,
          2018,
          2018,
          2018,
          2018,
          2021,
          2021,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2017,
          2019,
          2023,
          2021,
          2011,
          2011,
          2011,
          2016,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2018,
          2019,
          2019,
          2019,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2017,
          2017,
          2020,
          2022,
          2022,
          2019,
          2022,
          2022,
          2021,
          2013,
          2017,
          2017,
          2017,
          2022,
          2022,
          2022,
          2020,
          2010,
          2017,
          2015,
          2015,
          2021,
          2016,
          2019,
          2022,
          2020,
          2019,
          2017,
          2016,
          2021,
          2018,
          2020,
          2017,
          2016,
          2015,
          2022,
          2022,
          2022,
          2019,
          2022,
          2022,
          2022,
          2022,
          2016,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2020,
          2021,
          2022,
          2022,
          2023,
          2022,
          2020,
          2018,
          2019,
          2022,
          2018,
          2018,
          2018,
          2022,
          2022,
          2022,
          2023,
          2021,
          2021,
          2021,
          2022,
          2022,
          2022,
          2015,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2022,
          2021,
          2021,
          2021,
          2020,
          2020,
          2021,
          2022,
          2021,
          2023,
          2019,
          2023,
          2023,
          2022,
          2022,
          2015,
          2015,
          2018,
          2018,
          2018,
          2018,
          2019,
          2015,
          2020,
          2021,
          2018,
          2008,
          2021,
          2017,
          2022,
          2022,
          2022,
          2022,
          2023,
          2022,
          2017,
          2022,
          2022,
          2022,
          2017,
          2021,
          2012,
          2022,
          2019,
          2021,
          2020,
          2022,
          2019,
          2021,
          2017,
          2022,
          2016,
          2018,
          2018,
          2019,
          2022,
          2022,
          2023,
          2022,
          2022,
          2021,
          2018,
          2018,
          2022,
          2019,
          2019,
          2022,
          2022,
          2022,
          2017,
          2019,
          2018,
          2018,
          2018,
          2022,
          2017,
          2009,
          2020,
          2022,
          2019,
          2020,
          2021,
          2014,
          2019,
          2022,
          2022,
          2022,
          2022,
          2021,
          2017,
          2020,
          2020,
          2021,
          2020,
          2019,
          2019,
          2019,
          2018,
          2022,
          2019,
          2023,
          2021,
          2021,
          2021,
          2017,
          2021,
          2019,
          2018,
          2022,
          2021,
          2022,
          2017,
          2020,
          2021,
          2020,
          2018,
          2016,
          2015,
          2020,
          2022,
          2021,
          2018,
          2022,
          2021,
          2021,
          2021,
          2021,
          2021,
          2019,
          2019,
          2021,
          2014,
          2014,
          2014,
          2014,
          2022,
          2017,
          2017,
          2021,
          2021,
          2021,
          2020,
          2017,
          2019,
          2022,
          2021,
          2021,
          2022,
          2018,
          2019,
          2015,
          2019,
          2022,
          2017,
          2020,
          2013,
          2012,
          2022,
          2015,
          2015,
          2017,
          2024,
          2016,
          2021,
          2015,
          2022,
          2019,
          2021,
          2011,
          2021,
          2018,
          2017,
          2022,
          2022,
          2021,
          2020,
          2020,
          2020,
          2020,
          2017,
          2014,
          2023,
          2015,
          2015,
          2015,
          2014,
          2022,
          2021,
          2021,
          2022,
          2022,
          2022,
          2021,
          2021,
          2020,
          2018,
          2016,
          2021,
          2019,
          2020,
          2022,
          2015,
          2017,
          2015,
          2021,
          2019,
          2013,
          2011,
          2013,
          2013,
          2019,
          2020,
          2020,
          2021,
          2020,
          2016,
          2021,
          2018,
          2021,
          2021,
          2023,
          2021,
          2021,
          2019,
          2020,
          2022,
          2014,
          2014,
          2016,
          2023,
          2017,
          2020,
          2015,
          2022,
          2022,
          2022,
          2022,
          2019,
          2022,
          2012,
          2023,
          2023,
          2023,
          2023,
          2023,
          2017,
          2024,
          2023,
          2023,
          2024,
          2023,
          2009,
          2010,
          2011,
          2011,
          2011,
          2011,
          2011,
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2018,
          2019,
          2018,
          2020,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2020,
          2019,
          2020,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2020,
          2019,
          2019,
          2020,
          2020,
          2020,
          2020,
          2019,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2021,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2020,
          2020,
          2020,
          2020,
          2021,
          2020,
          2021,
          2020,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2020,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2022,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2022,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2022,
          2022,
          2022,
          2021,
          2021,
          2022,
          2022,
          2022,
          2022,
          2021,
          2022,
          2022,
          2021,
          2021,
          2021,
          2022,
          2021,
          2022,
          2020,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2022,
          2022,
          2023,
          2022,
          2022,
          2023,
          2022,
          2022,
          2022,
          2023,
          2022,
          2023,
          2022,
          2023,
          2023,
          2023,
          2022,
          2022,
          2022,
          2023,
          2022,
          2023,
          2022,
          2023,
          2022,
          2022,
          2022,
          2022,
          2022,
          2022,
          2023,
          2022,
          2022,
          2023,
          2023,
          2023,
          2023,
          2022,
          2023,
          2022,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2023,
          2024,
          2024,
          2023,
          2023,
          2024,
          2024,
          2024,
          2024,
          2024,
          2024
         ],
         "xaxis": "x",
         "yaxis": "y"
        },
        {
         "alignmentgroup": "True",
         "bingroup": "x",
         "hovertemplate": "source=legacy<br>year=%{x}<br>count=%{y}<extra></extra>",
         "legendgroup": "legacy",
         "marker": {
          "color": "#00cc96",
          "pattern": {
           "shape": ""
          }
         },
         "name": "legacy",
         "offsetgroup": "legacy",
         "orientation": "v",
         "showlegend": true,
         "type": "histogram",
         "x": [
          2006,
          2006,
          2006,
          2007,
          2007,
          2007,
          2007,
          2007,
          2007,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2008,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2009,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2010,
          2011,
          2011,
          2011,
          2012,
          2011,
          2011,
          2012,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2012,
          2011,
          2012,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2011,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2013,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2012,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2013,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2014,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2015,
          2016,
          2015,
          2015,
          2015,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2016,
          2016,
          2016,
          2016,
          2017,
          2016,
          2016,
          2017,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2016,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2017,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2018,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2019,
          2020,
          2021,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2020,
          2021,
          2020,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021,
          2021
         ],
         "xaxis": "x",
         "yaxis": "y"
        }
       ],
       "layout": {
        "barmode": "relative",
        "legend": {
         "title": {
          "text": "source"
         },
         "tracegroupgap": 0
        },
        "margin": {
         "t": 60
        },
        "template": {
         "data": {
          "bar": [
           {
            "error_x": {
             "color": "#2a3f5f"
            },
            "error_y": {
             "color": "#2a3f5f"
            },
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "bar"
           }
          ],
          "barpolar": [
           {
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             },
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "barpolar"
           }
          ],
          "carpet": [
           {
            "aaxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "baxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "type": "carpet"
           }
          ],
          "choropleth": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "choropleth"
           }
          ],
          "contour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "contour"
           }
          ],
          "contourcarpet": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "contourcarpet"
           }
          ],
          "heatmap": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmap"
           }
          ],
          "heatmapgl": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmapgl"
           }
          ],
          "histogram": [
           {
            "marker": {
             "pattern": {
              "fillmode": "overlay",
              "size": 10,
              "solidity": 0.2
             }
            },
            "type": "histogram"
           }
          ],
          "histogram2d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2d"
           }
          ],
          "histogram2dcontour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2dcontour"
           }
          ],
          "mesh3d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "mesh3d"
           }
          ],
          "parcoords": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "parcoords"
           }
          ],
          "pie": [
           {
            "automargin": true,
            "type": "pie"
           }
          ],
          "scatter": [
           {
            "fillpattern": {
             "fillmode": "overlay",
             "size": 10,
             "solidity": 0.2
            },
            "type": "scatter"
           }
          ],
          "scatter3d": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatter3d"
           }
          ],
          "scattercarpet": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattercarpet"
           }
          ],
          "scattergeo": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergeo"
           }
          ],
          "scattergl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergl"
           }
          ],
          "scattermapbox": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattermapbox"
           }
          ],
          "scatterpolar": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolar"
           }
          ],
          "scatterpolargl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolargl"
           }
          ],
          "scatterternary": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterternary"
           }
          ],
          "surface": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "surface"
           }
          ],
          "table": [
           {
            "cells": {
             "fill": {
              "color": "#EBF0F8"
             },
             "line": {
              "color": "white"
             }
            },
            "header": {
             "fill": {
              "color": "#C8D4E3"
             },
             "line": {
              "color": "white"
             }
            },
            "type": "table"
           }
          ]
         },
         "layout": {
          "annotationdefaults": {
           "arrowcolor": "#2a3f5f",
           "arrowhead": 0,
           "arrowwidth": 1
          },
          "autotypenumbers": "strict",
          "coloraxis": {
           "colorbar": {
            "outlinewidth": 0,
            "ticks": ""
           }
          },
          "colorscale": {
           "diverging": [
            [
             0,
             "#8e0152"
            ],
            [
             0.1,
             "#c51b7d"
            ],
            [
             0.2,
             "#de77ae"
            ],
            [
             0.3,
             "#f1b6da"
            ],
            [
             0.4,
             "#fde0ef"
            ],
            [
             0.5,
             "#f7f7f7"
            ],
            [
             0.6,
             "#e6f5d0"
            ],
            [
             0.7,
             "#b8e186"
            ],
            [
             0.8,
             "#7fbc41"
            ],
            [
             0.9,
             "#4d9221"
            ],
            [
             1,
             "#276419"
            ]
           ],
           "sequential": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ],
           "sequentialminus": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ]
          },
          "colorway": [
           "#636efa",
           "#EF553B",
           "#00cc96",
           "#ab63fa",
           "#FFA15A",
           "#19d3f3",
           "#FF6692",
           "#B6E880",
           "#FF97FF",
           "#FECB52"
          ],
          "font": {
           "color": "#2a3f5f"
          },
          "geo": {
           "bgcolor": "white",
           "lakecolor": "white",
           "landcolor": "#E5ECF6",
           "showlakes": true,
           "showland": true,
           "subunitcolor": "white"
          },
          "hoverlabel": {
           "align": "left"
          },
          "hovermode": "closest",
          "mapbox": {
           "style": "light"
          },
          "paper_bgcolor": "white",
          "plot_bgcolor": "#E5ECF6",
          "polar": {
           "angularaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "radialaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "scene": {
           "xaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "yaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "zaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           }
          },
          "shapedefaults": {
           "line": {
            "color": "#2a3f5f"
           }
          },
          "ternary": {
           "aaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "baxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "caxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "title": {
           "x": 0.05
          },
          "xaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          },
          "yaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          }
         }
        },
        "title": {
         "text": "NeuroMab General has not been done yet"
        },
        "xaxis": {
         "anchor": "y",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "year"
         }
        },
        "yaxis": {
         "anchor": "x",
         "domain": [
          0,
          1
         ],
         "title": {
          "text": "count"
         }
        }
       }
      }
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# remove anything before the year 2000\n",
    "df_cure = df_cure[df_cure['year'] >= 2000]\n",
    "# make the same plot\n",
    "fig = px.histogram(df_cure, x=\"year\", color=\"source\")\n",
    "# add a title that shows the neuromab generaal has not been done yet\n",
    "fig.update_layout(title_text='NeuroMab General has not been done yet')\n",
    "fig.show()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
