
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.1c31f4d4c46e.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.3c558ec9f8de.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.5c2920216256.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.6539a0a78536cfdc1fa6.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.e622fd8870cb011febf1.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Developing High-Quality Mouse Monoclonal Antibodies for Neuroscience Research &#8211; Approaches, Perspectives and Opportunities - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C2F776F7782C5000038C086E230A5.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="1.4.0.post1+1040be0" />

<meta name="ncbi_domain" content="nihpa" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC4884554/" />


    


        <script>
            
            var useOfficialGovtHeader = true;
        </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4884554/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="New biotechnology" /><meta name="citation_title" content="Developing High-Quality Mouse Monoclonal Antibodies for Neuroscience Research &#x02013; Approaches, Perspectives and Opportunities" /><meta name="citation_author" content="Belvin Gong" /><meta name="citation_author_institution" content="Department of Neurobiology, Physiology and Behavior, University of California, Davis, Davis, California 95616" /><meta name="citation_author" content="Karl D. Murray" /><meta name="citation_author_institution" content="Department of Neurobiology, Physiology and Behavior, University of California, Davis, Davis, California 95616" /><meta name="citation_author_institution" content="Center for Neuroscience, University of California, Davis, Davis, California 95616" /><meta name="citation_author" content="James S. Trimmer" /><meta name="citation_author_institution" content="Department of Neurobiology, Physiology and Behavior, University of California, Davis, Davis, California 95616" /><meta name="citation_author_institution" content="Department of Physiology and Membrane Biology, University of California Davis School of Medicine, Davis, California 95616" /><meta name="citation_publication_date" content="2016/09/09" /><meta name="citation_issue" content="5 Pt A" /><meta name="citation_volume" content="33" /><meta name="citation_firstpage" content="551" /><meta name="citation_doi" content="10.1016/j.nbt.2015.11.007" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4884554/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC4884554/" /><meta name="citation_pmid" content="26644354" /><meta name="DC.Title" content="Developing High-Quality Mouse Monoclonal Antibodies for Neuroscience Research &#x02013; Approaches, Perspectives and Opportunities" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="NIH Public Access" /><meta name="DC.Contributor" content="Belvin Gong" /><meta name="DC.Contributor" content="Karl D. Murray" /><meta name="DC.Contributor" content="James S. Trimmer" /><meta name="DC.Date" content="2016 Sep 25" /><meta name="DC.Identifier" content="10.1016/j.nbt.2015.11.007" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Developing High-Quality Mouse Monoclonal Antibodies for Neuroscience Research &#x02013; Approaches, Perspectives and Opportunities" /><meta property="og:type" content="article" /><meta property="og:description" content="High-quality antibodies (Abs) are critical to neuroscience research, as they remain the primary affinity proteomics reagent used to label and capture endogenously expressed protein targets in the nervous system. As in other fields, neuroscientists are ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4884554/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" />


</head>
<body >

   
    
    


    
        <section class="pmc-alerts">

</section>
    

    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC4884554
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pubreader-link other_item"><a href="/pmc/articles/PMC4884554/?report=reader" class="sidefm-pmclink">PubReader</a></li>
<li class="pdf-link other_item"><a href="/pmc/articles/PMC4884554/pdf/nihms741564.pdf" class="int-view">PDF (1.6M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/4884554/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/4884554/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<div class="collections-button-container" data-article-id="4884554" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4884554%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="7yC9bn4KciKdRXlftnRTsFK0b3cVPKUHpF1K1wESxw4jkZaIGc9Rz0FLewjqH41j">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4884554%2F&amp;text=Developing%20High-Quality%20Mouse%20Monoclonal%20Antibodies%20for%20Neuroscience%20Research%20%02013%20Approaches%2C%20Perspectives%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4884554%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4884554/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC4884554/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/26644354/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC4884554/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/26644354/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC4884554/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/4884554/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC4884554
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">
        Learn more about our disclaimer.
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area target="_blank" rel="noopener noreferrer" shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div><span id="pmcmata">N Biotechnol.</span> Author manuscript; available in PMC 2017 Sep 25.</div><div></div><div class="final-form"><em>Published in final edited form as:</em></div><div class="final-citation"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26644354" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">N Biotechnol. 2016 Sep 25; 33(5 Pt A): 551–564. </a></div><span class="fm-vol-iss-date">Published online 2015 Nov 28. </span> <span class="doi"><span>doi: </span><a href="//doi.org/10.1016%2Fj.nbt.2015.11.007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.nbt.2015.11.007</a></span></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4884554</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS741564</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/26644354">26644354</a></div></div></div><h1 class="content-title">Developing High-Quality Mouse Monoclonal Antibodies for Neuroscience Research &#x02013; Approaches, Perspectives and Opportunities</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gong%20B%5BAuthor%5D" class="affpopup" co-rid="_co_idm139909184860688" co-class="co-affbox">Belvin Gong</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Murray%20KD%5BAuthor%5D" class="affpopup" co-rid="_co_idm139909183908912" co-class="co-affbox">Karl D. Murray</a>,<sup>1,</sup><sup>2</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Trimmer%20JS%5BAuthor%5D" class="affpopup" co-rid="_co_idm139909182925616" co-class="co-affbox">James S. Trimmer</a><sup>1,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139909184860688"><h3 class="no_margin">Belvin Gong</h3><p><sup>1</sup>Department of Neurobiology, Physiology and Behavior, University of California, Davis, Davis, California 95616</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gong%20B%5BAuthor%5D">Belvin Gong</a></div></div><div id="_co_idm139909183908912"><h3 class="no_margin">Karl D. Murray</h3><p><sup>1</sup>Department of Neurobiology, Physiology and Behavior, University of California, Davis, Davis, California 95616</p><p><sup>2</sup>Center for Neuroscience, University of California, Davis, Davis, California 95616</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Murray%20KD%5BAuthor%5D">Karl D. Murray</a></div></div><div id="_co_idm139909182925616"><h3 class="no_margin">James S. Trimmer</h3><p><sup>1</sup>Department of Neurobiology, Physiology and Behavior, University of California, Davis, Davis, California 95616</p><p><sup>3</sup>Department of Physiology and Membrane Biology, University of California Davis School of Medicine, Davis, California 95616</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Trimmer%20JS%5BAuthor%5D">James S. Trimmer</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139909183949552_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139909183949552_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139909183949552_ai" style="display:none"><div class="fm-affl" id="A1"><sup>1</sup>Department of Neurobiology, Physiology and Behavior, University of California, Davis, Davis, California 95616</div><div class="fm-affl" id="A2"><sup>2</sup>Center for Neuroscience, University of California, Davis, Davis, California 95616</div><div class="fm-affl" id="A3"><sup>3</sup>Department of Physiology and Membrane Biology, University of California Davis School of Medicine, Davis, California 95616</div><div id="FN1">Corresponding Author: Dr. James S. Trimmer, Department of Neurobiology, Physiology and Behavior, University of California Davis, One Shields Ave, Davis, CA, 95616-8519. <a href="mailto:dev@null" data-email="ude.sivadcu@remmirtj" class="oemail">ude.sivadcu@remmirtj</a></div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm139909183949552_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN2">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26644354" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">N Biotechnol</a></div></div></div><div class="sec"></div><div id="abstract-1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-1title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">High-quality antibodies (Abs) are critical to neuroscience research, as they remain the primary affinity proteomics reagent used to label and capture endogenously expressed protein targets in the nervous system. As in other fields, neuroscientists are frequently confronted with inaccurate and irreproducible Ab-based results and/or reporting. The UC Davis/NIH NeuroMab Facility was created with the mission of addressing the unmet need for high-quality Abs in neuroscience research by applying a unique approach to generate and validate mouse monoclonal antibodies (mAbs) optimized for use against mammalian brain (<em>i.e.</em>, NeuroMabs). Here we describe our methodology of multi-step mAb screening focused on identifying mAbs exhibiting efficacy and specificity in labeling mammalian brain samples. We provide examples from NeuroMab screens, and from the subsequent specialized validation of those selected as NeuroMabs. We highlight the particular challenges and considerations of determining specificity for brain immunolabeling. We also describe why our emphasis on extensive validation of large numbers of candidates by immunoblotting and immunohistochemistry against brain samples is essential for identifying those that exhibit efficacy and specificity in those applications to become NeuroMabs. We describe the special attention given to candidates with less common non-IgG1 IgG subclasses that can facilitate simultaneous multiplex labeling with subclass-specific secondary antibodies. We detail our recent use of recombinant cloning of NeuroMabs as a method to archive all NeuroMabs, to unambiguously define NeuroMabs at the DNA sequence level, and to re-engineer IgG1 NeuroMabs to less common IgG subclasses to facilitate their use in multiplex labeling. Finally, we provide suggestions to facilitate Ab development and use, as to design, execution and interpretation of Ab-based neuroscience experiments. Reproducibility in neuroscience research will improve with enhanced Ab validation, unambiguous identification of Abs used in published experiments, and end user proficiency in Ab-based assays.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Brain, immunoblot, immunofluorescence, immunohistochemistry, neuroscience, validation</span></div></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">Introduction</h2><p id="P3" class="p p-first">Antibodies (Abs) are valuable and essential reagents for many proteomic level applications that are key to the effective pursuit of molecular and cellular neuroscience research. While Abs are only one class of the diverse binder types used in neuroscience research, they remain the primary tool for the labeling and capture of molecular targets in cells and tissues from the nervous system. They provide neuroscientists with familiar, stable and high-affinity reagents for which there are a wide array of readily available reagents for their subsequent detection and/or capture. Antibodies can be developed and validated for a broad range of labeling and capture applications, and compared to many other classes of binders, can generally be used under a broad range of assay conditions (although, as discussed below, sample preparation and assay conditions can fundamentally impact the nature of Ab-antigen interaction). High-quality and well-characterized polyclonal or monoclonal Abs (pAbs or mAbs, respectively) provide enormous benefits to neuroscience research due to their wide availability, familiarity and ease of use, such that the same Ab can be used globally across many independent laboratories. Antibodies have been crucial to the expansion of knowledge regarding the expression, localization, structure, function, and molecular interactions of a wide variety of proteins expressed in the nervous system, leading to significant advances in the field. However, in spite of the importance of Abs in neuroscience research, and the large number of commercial and public suppliers of Abs, neuroscientists are often faced with challenges and frustrations when applying Abs in their experiments [<a href="#R1" rid="R1" class=" bibr popnode">1</a>&#x02013;<a href="#R3" rid="R3" class=" bibr popnode">3</a>]. Lack of reliable results using Abs, and their reproducibility, whether due to the poor quality of the Abs themselves, or their application in experiments under conditions for which the Ab was not validated for use, or insufficient reporting of the details of the Abs themselves, has caused considerable frustration among researchers in many fields of research [<a href="#R4" rid="R4" class=" bibr popnode">4</a>]. In the neuroscience arena this has resulted in documentation of numerous incorrect and irreproducible results using Abs, for examples see [<a href="#R5" rid="R5" class=" bibr popnode">5</a>&#x02013;<a href="#R11" rid="R11" class=" bibr popnode">11</a>]. This is coupled with frustration with the high cost of many commercial Abs (<span class="underline">francesscientist.wordpress.com/2010/10/12/scamming-the-antibody/</span>). The estimates of the resulting cost of &#x0201c;bad&#x0201d; Abs to global research budgets in the hundreds of millions of US dollars annually [<a href="#R12" rid="R12" class=" bibr popnode">12</a>].</p><p id="P4" class="p p-last">The UC Davis/NIH NeuroMab Facility was created in 2005, in large part as a response to inquiries from neuroscientists interested in having the same systematic approach to mAb generation and characterization we were using to generate high quality anti-ion channel mAbs in our research laboratory, for examples see [<a href="#R13" rid="R13" class=" bibr popnode">13</a>&#x02013;<a href="#R16" rid="R16" class=" bibr popnode">16</a>], applied more broadly to targets of general importance to the neuroscience research community. This approach is similar to that used in many other laboratories for stringent in-house validation of Abs, and similar to that described in a prominent publication on Ab quality and validation [<a href="#R17" rid="R17" class=" bibr popnode">17</a>], but with a specific focus on validation in brain samples. NeuroMab aims to provide mAbs (<em>i.e.</em>, NeuroMabs), validated for the critical applications of immunoblotting (IB) and immunohistochemistry (IHC) against mammalian brain samples, and to distribute them on a non-profit low cost basis. Over the last ten years, the NeuroMab Facility has established a strong track record towards this mission by both following a proven neuroscience-based approach to screening that is briefly described here, and by its subsequent involvement in controlling quality of the final mAb product distributed to end users. Here we describe the approaches used at the NeuroMab Facility to generate and validate mAbs for neuroscience research applications, provide examples of validation data from such efforts, and use this as a platform to discuss some aspects of the general state of Abs for neuroscience research.</p></div><div id="S2" class="tsec sec"><h2 class="head no_bottom_margin" id="S2title">NeuroMab generation, characterization and validation</h2><p id="P5" class="p p-first-last">NeuroMab projects begin with input from the neuroscience community regarding potential targets that are important for neuroscience research and for which no reliable, widely available, and/or renewable binder currently exists. A Scientific Advisory Board and NIH program official prioritize target suggestions based on information regarding the scientific justification and biological importance of the target, and the availability and suitability of existing Abs or other binders. The suggesting lab is also asked to provide a list of available reagents for the project, including plasmids for expression of full-length recombinant target protein in mammalian cells, and samples from knockout (KO) mice if available. Suggesting labs are also asked to provide any other information relevant to the target that may facilitate the screening and validation (<em>e.g.</em>, the spatial and temporal expression pattern of the target in brain, any unusual aspects of the target protein as far as instability during sample preparation, behavior on SDS gels, etc.). Through this process, the NeuroMab Facility has taken on projects for a broad array of targets, ranging from novel proteins for which little information and few reagents were available, to well-characterized proteins for which the quality of, or accessibility to, existing Abs was not reliable, or for which there is a dwindling supply of a non-renewable reagent (<em>e.g.</em>, a pAb). The array of targets we have pursued include cytoskeletal proteins, enzymes, receptors, ion channels, transcription factors and other proteins of diverse and/or unknown function, as well as specific epigenetic marks on DNA and proteins, specific phosphorylation sites, splice junctions of alternatively spliced proteins, newly generated ends of proteolytic cleavage products, disease-associated mutations, adducts arising from bioterrorism/chemical warfare agents, and many other targets for which no reliable or renewable reagent existed.</p></div><div id="S3" class="tsec sec"><h2 class="head no_bottom_margin" id="S3title">Immunogen design and production</h2><p id="P6" class="p p-first">A critical step in Ab production is the design and production of an appropriate immunogen. While many sophisticated approaches can be used in immunogen design, guided by intensive efforts in the area of vaccine development, for example see [<a href="#R18" rid="R18" class=" bibr popnode">18</a>], our proximate goal is to generate research reagents for labeling and capturing targets from brain tissue, so our immunogen design is guided by only a few simple considerations. The design of our immunogens is similar to that used by the Human Protein Atlas project except we do not limit the length of our immunogens to 150 amino acids [<a href="#R19" rid="R19" class=" bibr popnode">19</a>]. One key feature is that the immunogen be hydrophilic and, in a best-case scenario, have extensive representation of charged residues to increase the probability that these regions will be surface-accessible in aqueous solutions. The immunogen should also contain extensive regions not present in other proteins, including other highly related members of the same family. It should also provide a reagent that is likely to yield mAbs that recognize the specific target across species (<em>i.e.</em>, mAbs that recognize mammalian orthologs, but not paralogs within a given species). For complex polypeptide targets such as ion channel subunits and other integral membrane proteins, we have commonly used fragments of intracellular or extracellular loops, or N- or C-terminal domains. For smaller targets, we have used full-length protein immunogens to successfully generate target-specific NeuroMabs. We prefer that our immunogens are at least 5 kDa in size, and do not have any known requirements for establishing or maintaining proper folding. In cases where the only attractive options for immunogens were a few separate short stretches of less than 50 amino acids each, we generated fusions of these fragments. We have a focus on human targets, as many of our neuroscience end users study human samples and the mechanisms underlying human neurological disease. However, much of this research also involves animal models, especially rodents, such that specific immunoreactivity against both human and rodent orthologs is desirable. Selecting human sequences that share substantial identity with rodent sequences allows us to use rat and mouse brain tissue in screens, which offers advantages as discussed below.</p><p id="P7">Our most common immunogens are recombinant protein fragments, which typically generate a more robust and varied immune response, and yield a high number of mAbs that work well in different applications. Smaller synthetic peptides are used almost exclusively for highly focused projects such as disease-associated mutations, posttranslational modifications (PTMs), splice junctions, etc. Because of their size and complexity, designing recombinant protein immunogens that produce maximal target specificity and minimal cross-reactivity can be challenging. Expression, solubility, and affinity purification of recombinant mammalian proteins in <em>E coli</em> can be difficult compared with the ease of outsourcing chemical synthesis of a peptide immunogen. However, in our experience, the time, effort and cost put towards generating or obtaining recombinant protein immunogens has been worthwhile. From a retrospective analysis of 414 NeuroMab projects, we find a much higher project success rate for generating at least one validated NeuroMab when using recombinant protein immunogens (211 out of 290 projects or a 73% success rate) than when using synthetic peptide immunogens (47 out of 124 projects or a 38% success rate). One step we have incorporated into our workflow is to validate each affinity-purified immunogen by tandem mass spectrometry (MS) analysis, which has substantially reduced the number of failed projects due to an incorrect immunogen.</p><p id="P8" class="p p-last">Mice are immunized using a short (30 day) but immunogen-intensive immunization protocol [<a href="#R13" rid="R13" class=" bibr popnode">13</a>] that is a modified version of protocols developed to generate a strong IgG response in a much shorter time period than conventional immunization protocols [<a href="#R20" rid="R20" class=" bibr popnode">20</a>; <a href="#R21" rid="R21" class=" bibr popnode">21</a>]. For each project, two na&#x000ef;ve female BALB/c mice (6&#x02013;8 weeks old) are immunized intraperitoneally using immunogen mixed with Sigma Adjuvant System adjuvant weekly for four weeks. Antiserum from the mice is analyzed for immunoreactivity against the immunogen by enzyme-linked immunosorbent assay (ELISA), by immunocytochemistry (ICC) against the target protein heterologously overexpressed in mammalian COS-1 cells, and by immunoblot against the target protein endogenously expressed in brain tissue. For three days preceding splenocyte isolation and fusion, mice receive intravenous tail vein injections of immunogen in physiological saline. Splenocytes from both mice are then harvested and pooled to maximize the population of mAb specificities, and used in a fusion with Sp2/0-Ag14 (ATCC CRL-1581) B-lymphocyte hybridomas. A small fraction (&#x0003c;1% or 5 &#x000d7; 10<sup>4</sup> cells) of pooled splenocytes is frozen as archival material and for potential future recombinant Ab generation efforts. The remainder of the splenocytes (typically 4 &#x000d7; 10<sup>8</sup>) is used for hybridoma generation <em>via</em> electrofusion with a Nepa Gene ECFG21 apparatus, which replaced polyethylene glycol (PEG)-based chemical fusion after an in-house comparison revealed higher levels of long term survival of hybridoma colonies (<a href="/pmc/articles/PMC4884554/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), as well as increased stability of mAb production. These results are typical of what other investigators have found, reviewed in [<a href="#R22" rid="R22" class=" bibr popnode">22</a>]. Following electrofusion, nascent hybridomas are plated (32 &#x000d7; 96-well plates for a total of 2,944 wells) and incubated for one week, after which the mAb-conditioned media (<em>i.e.</em>, tissue culture supernatants or TCS) is harvested for use in primary screens.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Fusion efficiency with electrofusion (EF) and polyethylene glycol (PEG)</p></div><div class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th valign="bottom" align="left" rowspan="1" colspan="1"></th><th colspan="3" valign="bottom" align="left" rowspan="1">% of wells with hybridomas @ 1 &#x000d7; 10<sup>7</sup> splenocytes/plate</th><th colspan="3" valign="bottom" align="left" rowspan="1">% of wells with hybridomas @ 1 &#x000d7; 10<sup>6</sup> splenocytes/plate</th><th colspan="3" valign="bottom" align="left" rowspan="1">% of wells with hybridomas @ 1 &#x000d7; 10<sup>5</sup> splenocytes/plate</th></tr><tr><th valign="bottom" align="left" rowspan="1" colspan="1">Project</th><th valign="bottom" align="right" rowspan="1" colspan="1">PEG</th><th valign="bottom" align="right" rowspan="1" colspan="1">EF</th><th valign="bottom" align="right" rowspan="1" colspan="1">Fold &#x00394;</th><th valign="bottom" align="right" rowspan="1" colspan="1">PEG</th><th valign="bottom" align="right" rowspan="1" colspan="1">EF</th><th valign="bottom" align="right" rowspan="1" colspan="1">Fold &#x00394;</th><th valign="bottom" align="right" rowspan="1" colspan="1">PEG</th><th valign="bottom" align="right" rowspan="1" colspan="1">EF</th><th valign="bottom" align="right" rowspan="1" colspan="1">Fold &#x00394;</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">N372A, B</td><td align="right" valign="top" rowspan="1" colspan="1">75%</td><td align="right" valign="top" rowspan="1" colspan="1">100%</td><td align="right" valign="top" rowspan="1" colspan="1">&#x0003e;1</td><td align="right" valign="top" rowspan="1" colspan="1">16%</td><td align="right" valign="top" rowspan="1" colspan="1">94%</td><td align="right" valign="top" rowspan="1" colspan="1">5.9</td><td align="right" valign="top" rowspan="1" colspan="1">0%</td><td align="right" valign="top" rowspan="1" colspan="1">18%</td><td align="right" valign="top" rowspan="1" colspan="1">NA</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">N378</td><td align="right" valign="top" rowspan="1" colspan="1">54%</td><td align="right" valign="top" rowspan="1" colspan="1">100%</td><td align="right" valign="top" rowspan="1" colspan="1">&#x0003e;1</td><td align="right" valign="top" rowspan="1" colspan="1">15%</td><td align="right" valign="top" rowspan="1" colspan="1">83%</td><td align="right" valign="top" rowspan="1" colspan="1">5.5</td><td align="right" valign="top" rowspan="1" colspan="1">2%</td><td align="right" valign="top" rowspan="1" colspan="1">40%</td><td align="right" valign="top" rowspan="1" colspan="1">20.0</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">L93</td><td align="right" valign="top" rowspan="1" colspan="1">32%</td><td align="right" valign="top" rowspan="1" colspan="1">86%</td><td align="right" valign="top" rowspan="1" colspan="1">2.7</td><td align="right" valign="top" rowspan="1" colspan="1">2%</td><td align="right" valign="top" rowspan="1" colspan="1">46%</td><td align="right" valign="top" rowspan="1" colspan="1">23.0</td><td align="right" valign="top" rowspan="1" colspan="1">ND</td><td align="right" valign="top" rowspan="1" colspan="1">ND</td><td align="right" valign="top" rowspan="1" colspan="1">ND</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">N379</td><td align="right" valign="top" rowspan="1" colspan="1">51%</td><td align="right" valign="top" rowspan="1" colspan="1">66%</td><td align="right" valign="top" rowspan="1" colspan="1">1.3</td><td align="right" valign="top" rowspan="1" colspan="1">8%</td><td align="right" valign="top" rowspan="1" colspan="1">24%</td><td align="right" valign="top" rowspan="1" colspan="1">3.0</td><td align="right" valign="top" rowspan="1" colspan="1">1%</td><td align="right" valign="top" rowspan="1" colspan="1">3%</td><td align="right" valign="top" rowspan="1" colspan="1">3.0</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">N380</td><td align="right" valign="top" rowspan="1" colspan="1">96%</td><td align="right" valign="top" rowspan="1" colspan="1">100%</td><td align="right" valign="top" rowspan="1" colspan="1">&#x0003e;1</td><td align="right" valign="top" rowspan="1" colspan="1">14%</td><td align="right" valign="top" rowspan="1" colspan="1">100%</td><td align="right" valign="top" rowspan="1" colspan="1">7.1</td><td align="right" valign="top" rowspan="1" colspan="1">2%</td><td align="right" valign="top" rowspan="1" colspan="1">66%</td><td align="right" valign="top" rowspan="1" colspan="1">33.0</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">N376A<sup>*</sup></td><td align="right" valign="top" rowspan="1" colspan="1">92%</td><td align="right" valign="top" rowspan="1" colspan="1">100%</td><td align="right" valign="top" rowspan="1" colspan="1">&#x0003e;1</td><td align="right" valign="top" rowspan="1" colspan="1">32%</td><td align="right" valign="top" rowspan="1" colspan="1">74%</td><td align="right" valign="top" rowspan="1" colspan="1">2.3</td><td align="right" valign="top" rowspan="1" colspan="1">2%</td><td align="right" valign="top" rowspan="1" colspan="1">9%</td><td align="right" valign="top" rowspan="1" colspan="1">4.5</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139909177030592"><a target="object" rel="noopener" href="/pmc/articles/PMC4884554/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p class="p p-first-last">NA = not applicable; ND = not done</p></div><div id="TFN2"><sup>*</sup>Using larger capacity chamber: 100%, 100% and 50% of wells with hybridomas @ 1 &#x000d7; 10<sup>7</sup>, 1 &#x000d7; 10<sup>6</sup> and 1 &#x000d7; 10<sup>5</sup> splenocytes/plate, resulting in &#x0003e;1, &#x0003e;3 and 25.0 fold increases, respectively.</div></div></div></div><div id="S4" class="tsec sec"><h2 class="head no_bottom_margin" id="S4title">Primary ELISA screens</h2><p id="P9" class="p p-first">The primary screen entails parallel ELISAs in two different 96-well plate formats, one employing plates coated with recombinant protein or synthetic peptide immunogen, and the other with plates containing transfected COS-1 cells. Both types of ELISAs are performed using a balanced cocktail of IgG1, IgG2a and IgG2b subclass-specific anti-mouse secondary Abs to avoid the bias inherent in &#x02018;generic&#x02019; anti-mouse IgG heavy-and-light-chain (H+L) specific secondary Abs that we identified in a previous study [<a href="#R23" rid="R23" class=" bibr popnode">23</a>]. For ELISAs against immunogens, recombinant protein fragments are expressed in and purified from <em>E. coli</em> bacteria, typically fused to a molecular tag to allow for affinity purification, and in some cases (<em>e.g.</em>, glutathione S-transferase (GST) and maltose binding protein (MBP) tags) [<a href="#R24" rid="R24" class=" bibr popnode">24</a>] to enhance expression and solubility. However, if used directly as an immunogen this can result in tag-specific mAbs that can interfere with effective identification of target-specific mAbs. Many of our recent projects have utilized immunogens generated by a modified version [<a href="#R25" rid="R25" class=" bibr popnode">25</a>] of the chitin-intein system [<a href="#R26" rid="R26" class=" bibr popnode">26</a>], which eliminates the tag during the purification process, resulting in the use of predominantly untagged immunogens and a low incidence of tag-specific mAbs. For projects employing GST-tagged proteins as immunogens, we preincubate the TCS from each of the 2,944 wells with soluble GST, which binds to and competes away anti-GST mAbs. This has yielded an effective suppression of the interfering signal from anti-GST mAbs and a subsequent increase in our ability to identify target-specific mAbs. Shorter tags (<em>e.g.</em>, 6x His) have proven less problematic. A set of tag-specific mAbs effective for detecting tagged proteins by IB, ICC and IHC, generated as a byproduct of NeuroMab projects against tagged immunogens, have been added to the NeuroMab catalog for those end users in need of such a reagent. For ELISAs against synthetic peptide immunogens, we coat plates with bovine serum albumin (BSA)-conjugated peptides, while keyhole limpet hemocyanin (KLH)-conjugated peptides are used for immunizations. This means that only the peptide itself is in common between immunogen and ELISA target, avoiding selection of KLH-specific mAbs. The chemical properties of BSA generally predominate within the peptide-BSA conjugate, allowing for ease and predictability of use, including reliable binding to microtiter plates that may not be typical of all peptides, given their varying amino acid compositions.</p><p id="P10">In parallel, ELISA screens against transiently transfected COS-1 cells expressing full-length target protein are performed to identify mAbs that recognize the full-length target protein in a cellular context, in which it may exhibit folding conformations, binding partners, and PTMs not present in the bacterially expressed immunogen. Cells are transiently transfected, plated onto 96-well plates, and cultured for two days. Cells are fixed under standard ICC conditions, using a freshly prepared solution of 4% formaldehyde, with 0.1% Triton X-100 for permeabilization. These conditions are also similar to those used for IHC, thereby enhancing the predictive value of this ELISA screen for that critical downstream application. Another distinction between immunogen- and cell-based ELISAs is their mode of detection. While both use horseradish peroxidase (HRP)-conjugated secondary Abs, the former uses colorimetric detection, and the latter a more sensitive chemiluminescence detection [<a href="#R27" rid="R27" class=" bibr popnode">27</a>]. ELISA screens are done in a semiautomated manner using a customized Agilent BioCel 1800 platform with plate- and liquid-handling skills.</p><p id="P11" class="p p-last">Examples of primary ELISA screen data for four NeuroMab projects are shown in <a href="/pmc/articles/PMC4884554/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, where the signal from the immunogen protein/peptide ELISA (x axis) is plotted against the signal from the cell-based ELISA (y axis) on a scatter plot. For each of the four projects shown, the squares represent the signal from each well of the 2,944 candidate wells, as well as signals from the positive (green squares) and negative (blue squares) serum and media controls, which are presented as an average of their 32 replicates (one from each fusion plate). The color of the candidate mAb symbols depicts those not selected for further consideration (unfilled squares), and of the 96 that were selected, those that ultimately became a NeuroMab (red squares), and those that did not (grey squares). In some cases, such as the N423, N424 and N429 projects shown in the panels in <a href="/pmc/articles/PMC4884554/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1A&#x02013;C</span></a>, it is rather simple to select 96 candidates for further consideration from among those with high signals in both assays (<em>i.e.</em>, those trending towards the upper right quadrant). However, for other projects, such as the N420 project shown in <a href="/pmc/articles/PMC4884554/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1D</span></a>, there is a wide distribution of data points, from which we may also select from wells with &#x0201c;low cell, high protein&#x0201d; signals (upper left quadrant) and from those with &#x0201c;high cell, low protein&#x0201d; signals (lower right quadrant). One important theme that has emerged from these screens, evident from the location of the plotted red squares, is the absolute magnitude of the signal strength on primary ELISAs has not been predictive of which candidates ultimately become NeuroMabs. This reinforces our need to take as many candidates as is manageable, from a cell culture and screening standpoint, forward into the downstream screens for NeuroMab-critical applications. As soon as ELISA results are available, the selected 96 candidate hybridoma cultures are expanded into 24-well culture plates.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4884554/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139909190361488"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4884554_nihms741564f1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms741564f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4884554/bin/nihms741564f1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139909190361488"><a target="object" rel="noopener" href="/pmc/articles/PMC4884554/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4884554/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Representative ELISA primary screen data</strong><p>Scatter plots show the relative distribution of ELISA data from NeuroMab projects N423 (A), N424 (B), N429 (C) and N420 (D), where protein and cell ELISA data are plotted on the x- and y-axis, respectively. Each project began with 2,944 hybridoma-containing wells and each graph shows 2,948 data points (squares) that together comprise positive (green) and negative (blue) control wells (each the average of 32 replicates), unselected wells (unfilled squares) and selected wells (grey squares), with red squares denoting the wells with candidates that ultimately became NeuroMabs.</p></div></div></div></div><div id="S5" class="tsec sec"><h2 class="head no_bottom_margin" id="S5title">Secondary screens</h2><p id="P12" class="p p-first">Three days after expansion, TCS is collected from each well and tested in parallel secondary screens that include the NeuroMab-critical applications of IB and IHC against mammalian brain samples. However, prior to use in these labor-intensive assays, a confirmation fluorescence-linked ELISA (FLISA) <em>versus</em> the immunogen is performed with subclass-specific secondary Abs conjugated to fluorophores (<em>e.g.</em>, Alexa 555-conjugated anti-IgG1, Alexa 488-conjugated anti-IgG2a, and Alexa 647-conjugated anti-IgG2b). This assay serves to confirm that TCS contain sufficient immunoreactivity to justify subsequent testing, and to determine the IgG subclass of the target-positive mAbs in these likely polyclonal TCS samples. Determining IgG subclass early in the screening process can enhance focus on candidates containing the less common IgG2a and IgG2b mAbs, which have increased value for multiplex labeling as most available mAbs are of the IgG1 subclass (<em>e.g.</em>, the NeuroMab and Millipore mAb collections in 2012 were 70% IgG1, 20% IgG2a and 10% IgG2b) [<a href="#R23" rid="R23" class=" bibr popnode">23</a>]. Positives from FLISA (typically most if not all of the 96 candidates) are then subjected to three parallel downstream assays. These comprise the two NeuroMab-critical applications of IB <em>versus</em> crude subcellular fractions from adult rat brain, and IHC on sagittal adult rat brain sections, as well as ICC on transfected cells, performed in a critical time window of a few days, due to cell culture and cryopreservation considerations detailed below. For IB analyses of brain samples, the benchmarks relate to the established or predicted molecular characteristics of the target protein (relative electrophoretic mobility and microheterogeneity of the target protein population due to alternative splicing of mRNA, PTMs, etc.) on sodium dodecyl sulfate (SDS) gels.</p><p id="P13">For IHC analyses on brain sections, the established or predicted anatomical features of endogenous protein expression at the regional, cellular and/or subcellular level need to be established <em>a priori</em> in order to gauge the specificity of immunolabeling for each candidate. Some NeuroMab projects target well-characterized proteins for which some validated immunolabeling data exists, and where the intent is to recapitulate this in a NeuroMab. In other cases the target is a novel protein that has never been characterized, in which case we need to develop diagnostic predictions on molecular characteristics (<em>e.g.</em>, from UniProt and other databases) and expression patterns in brain (<em>e.g.</em>, from Allen Brain Atlas (ABA) and other databases of mRNA expression from <em>in situ</em> hybridization).</p><p id="P14">Immunoblot screening is performed on crude subcellular fractions of adult rat brain prepared from homogenates by differential centrifugation, and typically constituting a nuclear/cytoskeletal fraction, or a crude membrane fraction, the so-called P1 and P2 fractions, respectively, of Gray and Whittaker [<a href="#R28" rid="R28" class=" bibr popnode">28</a>]. The choice of brain fraction is determined by the predicted subcellular location of the target protein, or empirically in pilot studies using available Abs or immune serum collected from the NeuroMab project mice. Samples (3 mg total protein) are size-fractionated on large format (14 cm wide &#x000d7; 15 cm long) SDS gels, with the resulting nitrocellulose blots cut into 30 &#x000d7; 0.4 cm wide vertical strips, each containing 100 &#x003bc;g of brain protein. The 30 strips from each gel allow for analysis of TCS from 26 candidates and a total of four positive and negative controls. <a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> shows a subset of candidate TCS from three different NeuroMab projects, together with interspersed positive and negative controls (plus and minus symbols, respectively) for IB analyses of a crude membrane or P2 fraction (<a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A and 2C</span></a>), used for membrane protein projects (N419: voltage-gated sodium channel principal subunits; N423: an A-type potassium channel auxiliary subunit), and a low-speed pellet or P1 fraction (<a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>) used for nuclear/cytoskeleton projects (N431: a GABA(A) receptor subunit). In these examples, the expected relative electrophoretic mobility of the target proteins (all are somewhat microheterogeneous due to PTMs) are 250 kDa for the N419 candidates (<a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>), 50 kDa for the N431 candidates (<a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>) and 30 kDa for the N423 candidates (<a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>). The immunoblot analyses of N419 and N431 yielded a wide variety of banding patterns, with a minority of candidates selected as being as target-specific, driving the eventual selection of these NeuroMabs (labeled with &#x0201c;N&#x0201d;s above the lanes): N419/40 (<a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A</span></a>) and N431/64 (<a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2B</span></a>) as NeuroMabs. In contrast, the N423 project yielded numerous candidates with varying IgG concentrations that exhibited different band intensities but also monospecific immunoreactivity to a band with the expected molecular characteristics, one of which, N423/75 (&#x0201c;N&#x0201d;-labeled lane in <a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2C</span></a>) was selected as a NeuroMab based on results of other assays.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4884554/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139909172310928"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4884554_nihms741564f2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms741564f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4884554/bin/nihms741564f2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139909172310928"><a target="object" rel="noopener" href="/pmc/articles/PMC4884554/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4884554/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Representative immunoblot secondary screen data</strong><p>Images show immunoblot data from three different NeuroMab projects. (A) Anti-pan Nav1 channel N419 NeuroMab project. (B) Anti-GABA(A) receptor &#x003c0; subunit N431 NeuroMab project. (C) Anti-KChIP4 N423 NeuroMab project. Numbers to the left of each panel show the mobility of molecular weight standards in kDa. Adult rat brain protein samples analyzed were a P2 crude membrane fraction or RBM (A and C) or a P1 low-speed pellet or RBLSP (B). Expected relative electrophoretic mobilities for target bands were 250 kDa (A), 50 kDa (B), and 30 kDa (C). The subset of strips shown depicts multiple candidates. (A) N419/28 to N419/40. (B) N431/55 to N431/64. (C) N423/28 to N423/37. Each set includes respective positive and negative controls (plus and minus symbols, respectively). The &#x0201c;N&#x0201d; denotes candidates that ultimately became a NeuroMab. (A) N419/40. (B) N431/64. (C) N423/75.</p></div></div></div><p id="P15">The IHC screens are typically performed on sagittal sections prepared from a formaldehyde-perfused and sucrose-cryoprotected adult female rat brain, which yields sufficient 40 &#x003bc;m thick sections to test all 96 candidates, as well as all appropriate controls. <a href="/pmc/articles/PMC4884554/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> shows examples of immunolabeling from typical NeuroMab IHC screens, in which immunolabeling is detected using a protocol employing avidin-biotin complex amplification, and a nickel-enhanced immunoperoxidase/diaminobenzidine (NiDAB) reaction [<a href="#R29" rid="R29" class=" bibr popnode">29</a>]. Each section is digitally imaged at high resolution using a high-speed automated slide scanner (Aperio ScanScope CS, Leica). Based on available <em>in situ</em> hybridization analyses, and previously published IHC data, immunolabeling for the GluA1/GluR1 glutamate receptor (<a href="/pmc/articles/PMC4884554/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3A</span></a>) was expected to be high in the hippocampus (bracketed region in <a href="/pmc/articles/PMC4884554/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>) and cerebellum relative to other brain structures (<em>e.g.</em> thalamus), and within these regions the immunolabeling was expected to be localized to fields of neuropil that are rich in neuronal synapses, as opposed to neuronal cell bodies or other non-neuronal cell types. Among the examples shown, immunolabeling in panel A1 fits these criteria, such that N355/1, which ultimately became a NeuroMab, was judged as target-specific.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4884554/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139909189311328"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4884554_nihms741564f3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms741564f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4884554/bin/nihms741564f3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139909189311328"><a target="object" rel="noopener" href="/pmc/articles/PMC4884554/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4884554/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Representative images from IHC secondary screens</strong><p>Photomicrographs show DAB/NAS immunolabeling of sagittal sections of adult rat brain from NeuroMab projects N355 (A1, A2 and A3) and N399 (B1, B2 and B3). There were 96 candidates screened for each project and each column shown here depicts images collected from one candidate judged to meet the criteria set out for specific labeling (N355/1 in A1 and N399/19 in B1) and two other candidates from these projects that failed to meet these criteria (N355/2 in A2, N355/3 in A3, N399/17 in B2 and N399/18 in B3). The bracketed region in A1 and B1 is the hippocampus, and the arrow in A2 and B3 identifies neocortex layer 5. CA3, hippocampal <em>cornu ammonis</em> area 3, CB, cerebellum, CPU, caudoputamen, CTX, neocortex DT, dorsal thalamus, HC, hippocampus.</p></div></div></div><p id="P16">Immunolabeling for the GABA(A) receptor &#x003b1;2 subunit (<a href="/pmc/articles/PMC4884554/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3B</span></a>) was expected to be higher in neocortex, caudoputamen and hippocampus relative to other brain regions, and was also expected to give a synaptic neuropil-enriched pattern of labeling. Among these examples, N399/19 (panel B1) was judged target-specific, and ultimately became a NeuroMab. Other panels show candidates with labeling that did not satisfy the criteria for correct positive labeling (<a href="/pmc/articles/PMC4884554/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3A2, A3, B2, B3</span></a>). Note that panels A2 and B3 show examples of candidate TCS from unrelated projects that exhibit similar off-target labeling across layer 5 of cortex, and in the principal cells of hippocampus, and especially in CA3. Candidates with this pattern can typically be eliminated, as this is a commonly observed pattern of non-specific immunolabeling in adult rat brain [<a href="#R2" rid="R2" class=" bibr popnode">2</a>].</p><p id="P17">While not a NeuroMab-critical application, each of the 96 candidate TCS is subjected to multiplex immunofluorescence (IF) ICC (IF-ICC) <em>versus</em> heterologous cells expressing full-length target protein. This is done as additional confirmation of target specificity. Transient transfection yields target-positive (transfected) cells in the same field of view as target-negative (untransfected) cells, allowing for a simple, straightforward analysis of concordance (or lack thereof) of candidate TCS when independent tags are utilized in the transfection. <a href="/pmc/articles/PMC4884554/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a> shows examples from such a screen, in this case from the N244 anti-SynCAM4 project, in which Myc-tagged SynCAM4 was expressed in COS-1 cells. Concordance of Myc (red) and mAb candidate (green) immunolabeling signals can be seen for N244/5 and N244/20, and to a lesser extent, N244/19, yielding a yellow signal in the merged image, while N244/3 exhibits discord in the signals, in this case due to a negative candidate, although in some cases it can be due to the candidate TCS labeling all cells.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC4884554/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139909172837328"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4884554_nihms741564f4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms741564f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4884554/bin/nihms741564f4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139909172837328"><a target="object" rel="noopener" href="/pmc/articles/PMC4884554/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC4884554/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Representative images from IF-ICC secondary screens</strong><p>Photomicrographs show IF immunolabeling of COS-1 cells transiently transfected with a Myc-tagged rat SynCAM4 mammalian expression construct using a rabbit anti-Myc pAb (red), candidate mouse mAbs (green) and Hoechst nuclear stain (blue). Each row depicts a set of images collected from a different candidates, one presented as typical of a negative candidate (N244/3), and three represented as positive candidates (N244/5, N244/19 and N244/20), of which N244/5 ultimately became a NeuroMab.</p></div></div></div><p id="P18" class="p p-last">In the 3&#x02013;4 day period during which these secondary screens are being completed, the hybridoma cells have been further expanded into 6 cm dishes, and are sufficiently dense to cryopreserve as two seed vials in liquid nitrogen. The goal is to have the results of the secondary screens available by this time, so that a small subset (<em>e.g.</em>, 5) of the top candidates can be subcloned by limiting dilution. This is critical as these cultures have substantial polyclonality at this stage, and it is advantageous to separate the desired hybridomas from the others prior to subjecting the polyclonal cultures to a freeze/thaw cycle, which empirical evidence suggests imposes a negative selection on the most valuable hybridomas. Harvesting for cryopreservation yields another 12 mL of TCS that can be used for any tertiary screens.</p></div><div id="S6" class="tsec sec"><h2 class="head no_bottom_margin" id="S6title">Tertiary screens</h2><p id="P19" class="p p-first-last">A variety of tertiary screens, the nature of which in part depends on the availability of samples, are performed on selected TCS, and/or on TCS from subcloned candidates, after their confirmation by ELISA. IB-positive mAbs are assayed against blots containing human, mouse, and rat brain (a positive control) samples. Due to the large size of the human brain it is not practical to make whole brain samples, and the region from which the sample is derived needs to be selected based on information regarding the regional pattern of target expression to avoid a false negative signal from <em>bona fide</em> human-positive mAbs. Human samples can have variable postmortem intervals prior to preservation during which the target proteins can be degraded, again resulting in a false positive. However, the translational value of assaying for anti-human brain efficacy is sufficiently high to pursue these assays with these caveats in mind. <a href="/pmc/articles/PMC4884554/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a> shows examples from such IB assays, employing the anti-Mortalin/GRP75 NeuroMab N52A/42 as a loading control, showing two NeuroMabs (anti-PSD-95 NeuroMab K28/43 and anti-KCC2 NeuroMab N1/12) that exhibit comparable immunoreactivity across rodent and human brain samples, and two (anti-Kv3.1b NeuroMab N16B/8 and anti-HCN2 NeuroMab N71/37) that exhibit preferential immunoreactivity against rodent samples. We also perform IB analyses against samples from KO or transgenic mice. Such validation is crucial in order to determine Ab specificity in native tissue such as brain [<a href="#R2" rid="R2" class=" bibr popnode">2</a>]. Examples of such analyses are shown in <a href="/pmc/articles/PMC4884554/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5E&#x02013;H</span></a>, again using N52A/42 as a loading control. The same comparison of KO <em>versus</em> WT brain samples can be used to define IHC labeling specificity (<a href="/pmc/articles/PMC4884554/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6C,D</span></a>), as detailed below. When justified by the project, we perform additional rounds of IF-ICC against transfected cells assaying for cross-reactivity against related members of the target protein family. While we typically aim to generate monospecific mAbs, recombinant immunogens often contain both target-unique regions, and regions conserved across family members, and can yield &#x0201c;pan-specific&#x0201d; mAbs, or mAbs that recognizes all members of a protein family. Such &#x0201c;pan-specific&#x0201d; mAbs can be useful for labeling [<a href="#R15" rid="R15" class=" bibr popnode">15</a>] or capturing [<a href="#R30" rid="R30" class=" bibr popnode">30</a>] all members of a protein family.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC4884554/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139909190358640"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4884554_nihms741564f5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms741564f5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4884554/bin/nihms741564f5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139909190358640"><a target="object" rel="noopener" href="/pmc/articles/PMC4884554/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC4884554/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Tertiary screening by IB against samples from human brain, and from KO mouse brain</strong><p>Panels (A&#x02013;D) depict results from NeuroMab TCS screening against mouse, rat and human brain P2 crude membrane fraction samples (MBM: adult mouse brain; RBM: adult rat brain; HBM (adult human brain, from Cx: neocortex; Hi: hippocampus). (A) Anti-PSD-95 NeuroMab K28/43. (B) Anti-KCC2 NeuroMab N1/12. (C) Anti-Kv3.1b NeuroMab N16B/8. (D) Anti-HCN2 NeuroMab N71/37. Panels (E&#x02013;H) depict results from NeuroMab TCS screening against brain samples from WT mice, and from the KO mouse corresponding to the NeuroMab target. Samples in panels (E, F) and (H) are P2 crude membrane fractions from adult rat brain (RBM), or adult mouse brains from WT and KO mice. Panel (G) has P1 low-speed pellet fractions from adult rat brain (RBLSP), or from adult WT and KO mouse brains. (E) Anti-Nav&#x003b2;2 subunit NeuroMab N395/68. (F) Anti-GABA(A) receptor &#x003b1;2 subunit NeuroMab N399/19. (G) Anti-GABA(A) receptor &#x003b1;5 subunit NeuroMab N415/24. (H) Anti-KChIP4 Kv channel subunit NeuroMab N423/75. In all cases duplicate immunoblots were probed the anti-Mortalin/GRP75 NeuroMab N52A/42 as a control to show comparable loading of all samples, with the exception of (H) in which the anti-pan-KChIP NeuroMab K55/82 was used to show the selective elimination of the KChIP4 band within the brain KChIP population in the KChIP4 KO sample. Numbers to the left of each panel show the mobility of molecular weight standards in kDa.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC4884554/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139909172422224"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4884554_nihms741564f6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms741564f6.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4884554/bin/nihms741564f6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139909172422224"><a target="object" rel="noopener" href="/pmc/articles/PMC4884554/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC4884554/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Tertiary screening by IHC</strong><p>Panels (A, B) show that discordance between regional localization of transcript and protein in mammalian brain can confound simple Ab evaluation. (A) Pseudocolor image of a sagittal section from mouse brain processed for <em>in situ</em> hybridization to mRNA for <em>Kchip4</em> obtained from the ABA (image 70546039_100) showing prominent transcript expression in the principal cell layers of the hippocampus, where cell bodies reside. (B) A sagittal section from rat brain immunolabeled for KChIP4 using NeuroMab N423/75 TCS by nickel enhanced diaminobenzidine (NiDAB) histochemistry showing a lack of labeling in the principal cellular layers of the hippocampus, but prominent labeling in the molecular layers, particularly stratum radiatum (sr) which is rich in dendrites. (C, D) Antibody specificity confirmed by KO validation. (C) IF labeling for KChIP4 using NeuroMab N423/75 TCS (magenta) in a sagittal section from a WT mouse brain shows the same prominent immunolabeling pattern in the dendrite rich molecular layers of hippocampus observed in rat (B). (D) Immunolabeling for KChIP4 is absent in the KChIP4 KO brain. In contrast, immunolabeling with the anti-ankyrin-R/G NeuroMab N388A/27 TCS (green) does not differ between WT and KO. Dye staining with Hoechst 33342 (blue) labels cellular nuclei. (E, F) Cellular specificity of transcript expression can guide Ab evaluation. (E) Pseudocolor image of <em>in situ</em> hybridization to <em>VAChT</em> mRNA in a sagittal mouse section from the ABA (image 79762433_86) showing specific transcript expression in the cranial trigeminal motor nucleus (V) and facial nucleus (VII) of the brainstem. (F) Immunolabeling by NiDAB histochemistry using NeuroMab N425/45 TCS in a sagittal section through the brainstem of a rat shows prominent signal for VAChT in the same cell nuclei (V, VII) as mRNA. (G,H) Subcellular target localization can guide NeuroMab evaluation. (G) Pseudocolor image of <em>in situ</em> hybridization of <em>Ankyrin-G</em> mRNA from the ABA (image 68196979_98) showing intense cellular expression throughout all cortical laminae (II&#x02013;VI). (H) IF labeling for Ankyrin-G (magenta) with NeuroMab N106/36 TCS and staining against cellular nuclei with the dye Hoechst 33342 (blue). In contrast to mRNA, which is localized to cell bodies and resembles the nuclear dye stain, protein expression is restricted to axon initial segments. CA1&#x02013;3, hippocampal <em>cornu ammonis</em> areas 1&#x02013;3, CB, cerebellum, DG, hippocampal dentate gyrus, Pn, pons, sg, stratum granulosum (granule cell body layer), sp, stratum pyramidale (pyramidal cell body layer), sr, stratum radiatum (pyramidal cell molecular layer).</p></div></div></div></div><div id="S7" class="tsec sec"><h2 class="head no_bottom_margin" id="S7title">Value of a multistep, real time approach to screening</h2><p id="P20" class="p p-first">Taking a large set of candidate mAbs through the multistep screening process described above, including the NeuroMab-critical screens of IB and IHC on mammalian brain samples, typically yields a set of mAbs (<em>i.e.</em>, NeuroMabs) extensively validated for these specific applications. Our experience suggests that taking such large numbers (96) of ELISA-positive candidates through these screens is important to identifying at least one mAb that is effective and specific for these applications. Conversely, we posit that performing mAb projects in which a more limited number of ELISA-positive candidates (<em>e.g.</em>, 5&#x02013;10) enter the downstream validation pipeline, such as typically offered by commercial custom mAb vendors, is not likely to yield such mAbs. While we spend a great deal of time and effort on screening this large number of candidates for these NeuroMab-critical applications, there are levels of characterization that are routine at other facilities, and that we do not perform. One prominent example is determination of absolute binding affinity against a recombinant protein or synthetic peptide. We have focused instead on identifying mAbs that recognize endogenous target in native tissue, in which mAb binding may be impacted by PTMs, protein-protein interactions, tissue characteristics or other aspects of the biological target protein, and that may alter binding of certain mAbs and not others in a manner unrelated to binding affinities against &#x0201c;naked&#x0201d; recombinant or synthetic preparations.</p><p id="P21">The candidate mAbs that pass all of the tests mentioned above are added to the NeuroMab catalog for distribution to end users in the research community. However, we retain frozen hybridoma seeds and TCS samples for all ELISA-positive candidates. We select NeuroMabs based on the two NeuroMab-critical applications and we cannot predict how the selected NeuroMabs will perform in other applications or even in the same applications performed under sample preparation and/or binding conditions distinct from those under which they were validated. Should a given NeuroMab not work for an application that is not part of our screening paradigm (<em>e.g.</em>, immunoelectron microscopy), it is possible to return to the original candidate sample set, rescreen for these applications, and provide them to end users with those specialized research needs.</p><p id="P22" class="p p-last">By definition Ab binding is dependent on the conditions under which the sample is prepared, and the binding is performed [<a href="#R3" rid="R3" class=" bibr popnode">3</a>; <a href="#R31" rid="R31" class=" bibr popnode">31</a>&#x02013;<a href="#R33" rid="R33" class=" bibr popnode">33</a>]. For IB analyses, this includes extraction and denaturation of the sample prior to SDS-PAGE [<a href="#R34" rid="R34" class=" bibr popnode">34</a>], and for ICC and IHC, the nature of fixation (which fixative, concentration, pH, temperature and/or time of fixation). Fundamentally results can be obtained with changes in fixation (<em>e.g.</em>, changing from 4% formaldehyde, pH 7.4 fixation to 2% formaldehyde, pH 6.0 fixation) [<a href="#R33" rid="R33" class=" bibr popnode">33</a>]. Making more extensive changes, for example adding glutaraldehyde as is typical for sample preparation for immunoelectron microscopy, presents even additional challenges. In these cases, Ab validation needs to be redone under these specific conditions. There are also numerous protocols for retrieving antigenicity from fixed tissue samples in which fixation has altered access to or the nature of the epitope [<a href="#R35" rid="R35" class=" bibr popnode">35</a>&#x02013;<a href="#R38" rid="R38" class=" bibr popnode">38</a>], and preparing samples in this manner requires revalidation. Tissues from KO animals are invaluable in these assays [<a href="#R2" rid="R2" class=" bibr popnode">2</a>; <a href="#R33" rid="R33" class=" bibr popnode">33</a>]. As to the binding reaction itself, the pH, buffer composition, salt concentration, blocking agent, detergents, temperature and time are among factors that can profoundly influence the interaction of Ab and target. Given that the details of sample preparation and binding can profoundly influence efficacy and specificity, it is likely that discrepancies in these procedures, due to a variety of factors including lack of transparency in reporting, and/or inattention to detail by experimentalists, are the basis for at least some of the issues underlying the irreproducibility of Ab-based results [<a href="#R4" rid="R4" class=" bibr popnode">4</a>]. Protocols used for validation at NeuroMab are posted on our website, so that end users can replicate these exact conditions in their own laboratories, or are at least aware when they are using conditions distinct from those used for mAb validation.</p></div><div id="S8" class="tsec sec"><h2 class="head no_bottom_margin" id="S8title">Challenges and considerations in determining Ab specificity in brain IHC</h2><p id="P23" class="p p-first-last">The mammalian brain is the most heterogeneous tissue in the body, and presents special challenges for generation and validation of specific Abs for use therein. Target and off-target proteins could potentially be localized in a variety of cell types including those related to vasculature (endothelial cells, smooth muscle), multiple types of glia, or any of the myriad classes of neurons present in specific brain regions, networks and laminae [<a href="#R39" rid="R39" class=" bibr popnode">39</a>&#x02013;<a href="#R42" rid="R42" class=" bibr popnode">42</a>]. Neuroscientists, therefore, must be exceptionally cognizant of the potential for off-target binding to molecular entities. However, the defined architecture of the brain and its cells at the regional, network, cellular and subcellular level can provide a wealth of information that can be used in Ab validation. Open access databases of mRNA expression patterns in mammalian brain, such as the ABA, can provide valuable information on regional and cellular patterns. Regional differences in protein expression can be extremely useful in determining specificity of Ab labeling, as these can be quite dramatic. In the case of the NeuroMab target RGS14, <em>a priori</em> knowledge that within the brain, <em>RGS14</em> was primarily expressed in hippocampal region CA2, led to the relatively straightforward identification of the N133/21 NeuroMab, that has subsequently found use in defining CA2 circuitry [<a href="#R43" rid="R43" class=" bibr popnode">43</a>]. In some cases, while gene expression can be extremely informative, one must also take into consideration the subcellular localization of the encoded protein. Having knowledge of brain architecture and the network of interconnectivity between regions and cells within regions is crucial to evaluating the specificity of target binding when proteins are trafficked to distinct compartments which may lie at some distance from the site of the mRNA in the cell body, see discussion in [<a href="#R44" rid="R44" class=" bibr popnode">44</a>]. As one example of a target for which there exists partial overlap of mRNA and protein, the <em>VAChT</em> mRNA is specifically localized to brainstem cranial nerve motor neurons and immunolabeling of VAChT protein with NeuroMab N425/45 shows the same expression pattern (<a href="/pmc/articles/PMC4884554/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6E, F</span></a>). However, there is also additional dense neuropil labeling throughout the caudoputamen, the target of dopaminergic neurons in the substantia nigra. Striatal labeling of this kind was therefore one of the diagnostic criteria for identifying a successful candidate. In some cases there is a strong discordance between transcript and protein expression for proteins localized to axonal and dendritic neuronal processes, which may lie far from the cell soma where mRNA is typically localized (<a href="/pmc/articles/PMC4884554/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6</span></a>). The potassium channel interacting protein KChIP4 is a good example of such discordance (<a href="/pmc/articles/PMC4884554/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6A,B</span></a>). While the mRNA is localized to the principal cell layers of the hippocampus, protein labeling with NeuroMab N423/75 is found overlying the molecular layers that are primarily comprised of the apical dendrites of these cells, with little labeling in the principal cell layers themselves. The scaffolding protein Ankyrin-G provides another example, as its localization to axon initial segments and nodes of Ranvier yields a specific immunolabeling pattern quite distinct from its mRNA that is present in cell somata (<a href="/pmc/articles/PMC4884554/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6G, H</span></a>). While such considerations can be confounding, they can also be used to advantage by a trained neuroanatomist who is able to correlate the sites of immunolabeling to the <em>in situ</em> hybridization signal in the cell bodies of origin. While the cellular heterogeneity of the brain, and the complexity of the cells therein, present challenges to Ab validation, they also offer distinct opportunities for determining Ab specificity.</p></div><div id="S9" class="tsec sec"><h2 class="head no_bottom_margin" id="S9title">Distinct mouse IgG subclasses as a benefit to multiplex labeling</h2><p id="P24" class="p p-first-last">Mice have four subclasses of IgG (in order of typical abundance): IgG1, IgG2a, IgG2b, and IgG3. We recently found that &#x02018;generic&#x02019; anti-mouse IgG H+L secondary Abs have a bias towards recognizing IgG1 mAbs that confounds equivalent detection of bound primary mouse Abs of different IgG subclasses [<a href="#R23" rid="R23" class=" bibr popnode">23</a>]. As such, we now perform all early stage assays using balanced cocktails of anti-IgG1, anti-IgG2a and anti-IgG2b subclass-specific anti-mouse secondary Abs to avoid detection bias when the IgG subclasses of candidate mAbs are not yet known; we do not include anti-IgG3 secondary Abs as the characteristics of these mAbs can complicate purification and stability in storage [<a href="#R45" rid="R45" class=" bibr popnode">45</a>]. The availability of mouse IgG mAbs that are by definition a single IgG subclass, along with specific and effective subclass-specific secondary Abs conjugated to different fluorophores, provides to mouse mAbs an important asset not present in rabbit mAbs, that allows for combining multiple Abs in simultaneous multiplex IF labeling [<a href="#R23" rid="R23" class=" bibr popnode">23</a>]. As the vast majority ( 70%) of commercially available mouse mAbs are of the IgG1 subclass, identifying and isolating IgG2a and IgG2b mouse mAbs has become a high priority, as this allows our end users to have additional experimental versatility. From a given project we may select more than one validated NeuroMab specifically to include representatives of different IgG subclasses.</p></div><div id="S10" class="tsec sec"><h2 class="head no_bottom_margin" id="S10title">Recombinant NeuroMabs as a route to archiving and engineering</h2><p id="P25" class="p p-first-last">As detailed above, we use natural as opposed to recombinant approaches to generate NeuroMabs. However, we are now generating recombinant NeuroMabs by <em>post facto</em> cloning from hybridomas, using a protocol developed by Wright and colleagues [<a href="#R46" rid="R46" class=" bibr popnode">46</a>; <a href="#R47" rid="R47" class=" bibr popnode">47</a>]. This archiving of NeuroMabs is independent of cryopreserved hybridomas and is a route to enhancing their unambiguous identification by sequence-based methods such as MS-based sequencing of the native or recombinant mAb preparation [<a href="#R12" rid="R12" class=" bibr popnode">12</a>]. We express and validate each recombinant NeuroMab in COS-1 cells in a side-by-side comparison with the native mAb to ensure that the native characteristics are faithfully represented in the recombinant form. Cloning of NeuroMabs also allows for their engineering into alternate forms that may facilitate their use in specific applications, our initial focus being re-engineering the constant region to alter the IgG subclass, especially for the extremely common IgG1s, to generate collections of otherwise identical mAbs of different IgG subclasses that enhances the flexibility of their use in simultaneous multiplex labeling experiments [<a href="#R23" rid="R23" class=" bibr popnode">23</a>].</p></div><div id="S11" class="tsec sec"><h2 class="head no_bottom_margin" id="S11title">NeuroMab and the outlook for the future of Ab quality</h2><p id="P26" class="p p-first">In general, neuroscientists have been at the forefront in advocating for enhanced reliability and reproducibility of Ab-based research, promoting both stringent validation of Abs [<a href="#R2" rid="R2" class=" bibr popnode">2</a>; <a href="#R33" rid="R33" class=" bibr popnode">33</a>; <a href="#R48" rid="R48" class=" bibr popnode">48</a>], and transparent, unambiguous description of those used in published research [<a href="#R49" rid="R49" class=" bibr popnode">49</a>]. Neuroscience journals with high publication standards such as the <em>Journal of Comparative Neurology</em> (<em>JCN</em>) have implemented strict measures for manuscripts reporting Ab-based research [<a href="#R1" rid="R1" class=" bibr popnode">1</a>; <a href="#R50" rid="R50" class=" bibr popnode">50</a>]. This is in part due to numerous instances of variable Ab characterization and data quality, some of which was revealed by the availability of KO mice, and in which the reported Ab-based localization was found to be unchanged in the KO samples [<a href="#R1" rid="R1" class=" bibr popnode">1</a>; <a href="#R50" rid="R50" class=" bibr popnode">50</a>]. These mandatory standards include a requirement for detailed descriptions of Ab validation, regardless of whether the Ab originated from a commercial vendor, came from another academic laboratory, or was generated <em>de novo</em> by the authors for the study. Once an Ab meets these standards, it is added to a database of <em>JCN</em>-Abs (onlinelibrary.wiley.com/journal/10.1002/(ISSN)1096&#x02013;9861/homepage/jcn_antibody_database.htm).</p><p id="P27" class="p p-last">The neuroscience community has also been instrumental in promoting the use of a standardized reporting system for research reagents, including and especially Abs, through the Neuroscience Information Framework (<a href="http://www.neuinfo.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.neuinfo.org</a>), which serves as a prominent portal to the Antibody Registry (<a href="http://www.antibodyregistry.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.antibodyregistry.org</a>), a database of over 2.4 million Abs, each of which has a unique identifier (<em>e.g.</em>, &#x0201c;AB_1234567&#x0201d;), to ensure the unambiguous description of any particular Ab. This unique identifier is then built into the larger Research Resource Identification Initiative (<a href="http://www.scicrunch.org/resources" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.scicrunch.org/resources</a>) or RRID (<a href="/pmc/articles/PMC4884554/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) [<a href="#R51" rid="R51" class=" bibr popnode">51</a>] such that each Antibody Registry unique identifier becomes a corresponding RRID identifier (<em>e.g.</em>, &#x0201c;AB_1234567&#x0201d; becomes &#x0201c;RRID:AB_1234567&#x0201d;). Certain neuroscience journals, including <em>JCN</em>, have embraced this effort, requiring an Antibody Registry/RRID number for all Abs used, while other journals have been making slower progress in this regard [<a href="#R51" rid="R51" class=" bibr popnode">51</a>]. The full implementation of the RRID:AB system should go a long way towards clearly defining which Ab was used for which experiment, as it is fairly common that references to Abs are incomplete, insufficiently clear, or in some cases completely absent, a problem exacerbated by journals with strict space limitations that have decided to dispense with experimental details [<a href="#R52" rid="R52" class=" bibr popnode">52</a>]. As both a service to our end users, and as a reporting tool, the NeuroMab Facility strives to track publications employing the use of NeuroMabs (neuromab.ucdavis.edu/publications.cfm). However, these efforts are often confounded by the lack of unambiguous detail of exactly which NeuroMab was used (<em>e.g.</em>, the NeuroMab catalog contains four anti-Ankyrin-G mAbs, which vary in their efficacy in IB versus and IHC, and in IgG subclass). As such, stating that an experiment employed an anti-Ankyrin G mAb from NeuroMab would not allow for reproducibility of the experiment. This is even more common for large Ab distributors, who may distribute several Abs against a single target. We have long encouraged our end users to provide the specific NeuroMab clone number (<em>e.g.</em>, N106/36, N106/65, etc.) as an unambiguous transparent identifier in all publications, a practice that should be even clearer now that a unique &#x0201c;RRID:AB_&#x0201d; identifier has been assigned to each NeuroMab (and extended to the level of TCS <em>versus</em> pure IgG preparations). This systematic registry of research resources is a major step towards increasing the reproducibility of experiments involving Abs [<a href="#R53" rid="R53" class=" bibr popnode">53</a>] and is consistent with calls for greater transparency in research [<a href="#R49" rid="R49" class=" bibr popnode">49</a>], and research reproducibility [<a href="#R54" rid="R54" class=" bibr popnode">54</a>], an emerging focus of the research community.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>NeuroMabs highlighted in current manuscript and their associated Research Resource Identifiers (RRIDs)</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th valign="top" align="left" rowspan="1" colspan="1">Target</th><th valign="top" align="left" rowspan="1" colspan="1">Accession #</th><th valign="top" align="left" rowspan="1" colspan="1">Clone</th><th valign="top" align="left" rowspan="1" colspan="1">IgG Subclass</th><th valign="top" align="left" rowspan="1" colspan="1">RRID TCS</th><th valign="top" align="left" rowspan="1" colspan="1">RRID Pure</th></tr></thead><tbody><tr><td align="left" valign="top" rowspan="1" colspan="1">Ankyrin-G (staining)</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_540032529" class="tag_hotlink" href="/protein/E9PE32" ref="/protein/E9PE32"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"E9PE32","term_id":"257051061"}}</span>E9PE32</a></td><td align="left" valign="top" rowspan="1" colspan="1">N106/36</td><td align="left" valign="top" rowspan="1" colspan="1">IgG2a</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10697718</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10673030</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Ankyrin-R</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883873" class="tag_hotlink" href="/protein/P16157" ref="/protein/P16157"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P16157","term_id":"116241246","term_text":"P16157"}}</span>P16157</a></td><td align="left" valign="top" rowspan="1" colspan="1">N388A/10</td><td align="left" valign="top" rowspan="1" colspan="1">IgG2b</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2336901</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2491109</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">GABA(A)R, Alpha2</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883881" class="tag_hotlink" href="/protein/P23576" ref="/protein/P23576"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P23576","term_id":"120758","term_text":"P23576"}}</span>P23576</a></td><td align="left" valign="top" rowspan="1" colspan="1">N399/19</td><td align="left" valign="top" rowspan="1" colspan="1">IgG1</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2336904</td><td align="left" valign="top" rowspan="1" colspan="1">NA<sup>*</sup></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">GABA(A)R, Alpha5</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883874" class="tag_hotlink" href="/protein/P31644" ref="/protein/P31644"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P31644","term_id":"399519","term_text":"P31644"}}</span>P31644</a></td><td align="left" valign="top" rowspan="1" colspan="1">N415/24</td><td align="left" valign="top" rowspan="1" colspan="1">IgG1</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2491075</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2491187</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">GABA(A)R, Pi</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883872" class="tag_hotlink" href="/protein/O00591" ref="/protein/O00591"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"O00591","term_id":"6016089","term_text":"O00591"}}</span>O00591</a></td><td align="left" valign="top" rowspan="1" colspan="1">N431/64</td><td align="left" valign="top" rowspan="1" colspan="1">IgG2a</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2493092</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2532051</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">GluR1 glutamate receptor</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883876" class="tag_hotlink" href="/protein/P19490" ref="/protein/P19490"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P19490","term_id":"97536283","term_text":"P19490"}}</span>P19490</a></td><td align="left" valign="top" rowspan="1" colspan="1">N355/1</td><td align="left" valign="top" rowspan="1" colspan="1">IgG1</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2315839</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2315840</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">HCN2</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883879" class="tag_hotlink" href="/protein/Q9JKA9" ref="/protein/Q9JKA9"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q9JKA9","term_id":"83303515","term_text":"Q9JKA9"}}</span>Q9JKA9</a></td><td align="left" valign="top" rowspan="1" colspan="1">N71/37</td><td align="left" valign="top" rowspan="1" colspan="1">IgG1</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10672304</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2279449</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">KCC2</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883877" class="tag_hotlink" href="/protein/NP_599190" ref="/protein/NP_599190"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"NP_599190","term_id":"19705463","term_text":"NP_599190"}}</span>NP_599190</a></td><td align="left" valign="top" rowspan="1" colspan="1">N1/12</td><td align="left" valign="top" rowspan="1" colspan="1">IgG2a</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10697875</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10672851</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">KChIP4 K<sup>+</sup> channel</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883880" class="tag_hotlink" href="/protein/Q3YAB7" ref="/protein/Q3YAB7"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q3YAB7","term_id":"121946917","term_text":"Q3YAB7"}}</span>Q3YAB7</a></td><td align="left" valign="top" rowspan="1" colspan="1">N423/75</td><td align="left" valign="top" rowspan="1" colspan="1">IgG2a</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2491080</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2493100</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Mortalin/GRP75</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883875" class="tag_hotlink" href="/protein/P38647" ref="/protein/P38647"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"P38647","term_id":"408360268","term_text":"P38647"}}</span>P38647</a></td><td align="left" valign="top" rowspan="1" colspan="1">N52A/42</td><td align="left" valign="top" rowspan="1" colspan="1">IgG1</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10674108</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2120479</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Navbeta2 Na<sup>+</sup> channel</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883882" class="tag_hotlink" href="/protein/Q56A07" ref="/protein/Q56A07"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q56A07","term_id":"81909358","term_text":"Q56A07"}}</span>Q56A07</a></td><td align="left" valign="top" rowspan="1" colspan="1">N395/68</td><td align="left" valign="top" rowspan="1" colspan="1">IgG2b</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2315895</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2315896</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Navbeta4 Na<sup>+</sup> channel</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883870" class="tag_hotlink" href="/protein/Q7M730" ref="/protein/Q7M730"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q7M730","term_id":"57012630","term_text":"Q7M730"}}</span>Q7M730</a></td><td align="left" valign="top" rowspan="1" colspan="1">N168/6</td><td align="left" valign="top" rowspan="1" colspan="1">IgG1</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10673578</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2301402</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Pan-KChIP K<sup>+</sup> channel</td><td align="left" valign="top" rowspan="1" colspan="1">-</td><td align="left" valign="top" rowspan="1" colspan="1">K55/82</td><td align="left" valign="top" rowspan="1" colspan="1">IgG2a</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2132595</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10673406</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Pan-MAGUK</td><td align="left" valign="top" rowspan="1" colspan="1">-</td><td align="left" valign="top" rowspan="1" colspan="1">K28/86</td><td align="left" valign="top" rowspan="1" colspan="1">IgG1</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10698179</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10673115</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Pan-Nav1 Na<sup>+</sup> channel</td><td align="left" valign="top" rowspan="1" colspan="1">-</td><td align="left" valign="top" rowspan="1" colspan="1">N419/40</td><td align="left" valign="top" rowspan="1" colspan="1">IgG2a</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2491079</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2491098</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">RGS14</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883878" class="tag_hotlink" href="/protein/O08773" ref="/protein/O08773"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"O08773","term_id":"2500170","term_text":"O08773"}}</span>O08773</a></td><td align="left" valign="top" rowspan="1" colspan="1">N133/21</td><td align="left" valign="top" rowspan="1" colspan="1">IgG2a</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10698026</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2179931</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">SynCAM4</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883871" class="tag_hotlink" href="/protein/Q8R464" ref="/protein/Q8R464"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q8R464","term_id":"81915047","term_text":"Q8R464"}}</span>Q8R464</a></td><td align="left" valign="top" rowspan="1" colspan="1">N244/5</td><td align="left" valign="top" rowspan="1" colspan="1">IgG1</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10673109</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_10676101</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">VAChT</td><td align="left" valign="top" rowspan="1" colspan="1"><a id="__tag_832883869" class="tag_hotlink" href="/protein/Q62666" ref="/protein/Q62666"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q62666","term_id":"34924861","term_text":"Q62666"}}</span>Q62666</a></td><td align="left" valign="top" rowspan="1" colspan="1">N425/45</td><td align="left" valign="top" rowspan="1" colspan="1">IgG1</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:AB_2532047</td><td align="left" valign="top" rowspan="1" colspan="1">NA<sup>*</sup></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139909227832320"><a target="object" rel="noopener" href="/pmc/articles/PMC4884554/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>*</sup>Currently not available</div></div></div></div><div id="S12" class="tsec sec"><h2 class="head no_bottom_margin" id="S12title">NeuroMab: an emphasis on &#x0201c;back-end&#x0201d; validation for neuroscience research</h2><p id="P28" class="p p-first">Any process for developing and screening Abs or other binders can be thought of in terms of &#x0201c;front-end&#x0201d; and &#x0201c;back-end&#x0201d; stages, using terminology associated with the production of semiconductors. The front-end comprises the fairly generic phases of immunogen design, production and purification; immunizations and Ab generation (or in the case of some binders, library generation); and primary screening and Ab selection. The back-end includes screens for efficacy and specificity in &#x0201c;real world&#x0201d; applications, as most end users want to use Abs for more than ELISAs. Almost all front-end tasks in Ab production conclude with a high-throughput binding assay to identify or select a manageable number of candidates, in our case ELISAs. This is similar to what is offered to clients at Ab core facilities and custom Ab producers, and presumably what is standard in commercial Ab generation. It is the back-end processes that can differ dramatically in their extent and nature, and what defines NeuroMab as distinct. Most core facilities and custom Ab producers, and presumably most commercial Ab developers, deem this to be the task of the client or end user, understandable as the diverse array of applications, nuances of the target protein behavior and the myriad sample preparation and binding conditions a client might require are more than what generic Ab production systems can handle. As is clear from the process detailed above, the NeuroMab Facility spends much time and effort on back-end validation, with over two-thirds of our effort and project costs devoted to taking the 96 candidate TCS through the NeuroMab-critical validation steps, which we have tailored to the most standard applications and associated methodologies of our end user community. A substantial portion of the front-end costs would also be eliminated by reducing the number of ELISA positives carried forward through expansion and cryopreservation, as typical for many core facilities and custom suppliers who provide clients with 5&#x02013;10 ELISA positive candidates for further consideration. However, our experience, and as reinforced by the data shown in <a href="/pmc/articles/PMC4884554/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, is that the top 5&#x02013;10 ELISA positives are not necessarily those that ultimately prove to be effective and specific in real world applications against endogenous proteins in tissue samples.</p><p id="P29">Given our extensive focus on the back-end validation of NeuroMabs for critical applications in brain samples, the NeuroMab model may not be directly relevant to large-scale efforts to address what many end users feel is a rising tide of substandard Ab-based data across broad areas of biomedical research [<a href="#R4" rid="R4" class=" bibr popnode">4</a>]. On the other hand, as a model for an intensive, rigorous and step-wise approach to Ab validation with a focus on serving end users in a particular research community, NeuroMab may be well placed to offer an example that is relevant to both academic and commercial efforts to provide reliable binders for affinity proteomics. We remain convinced that the overall time, effort and cost devoted to extensive back-end validation of large panels of candidate mAbs, performed with the neuroscientist end user in mind, is justified in providing them with a reliable reagent. Moreover, we argue that the costs associated with the back-end</p><p id="P30" class="p p-last">NeuroMab validation steps will be recouped by the savings of avoiding having these assays done for the first time in numerous independently-funded laboratories, or even worse, should this validation not be done, by the cost to the research system at-large of having incorrect results enter the literature.</p></div><div id="S13" class="tsec sec"><h2 class="head no_bottom_margin" id="S13title">Suggestions for responsible antibody development and use</h2><p id="P31" class="p p-first-last">Numerous questions have been recently raised regarding the transparency and reproducibility of Ab-based research [<a href="#R4" rid="R4" class=" bibr popnode">4</a>; <a href="#R12" rid="R12" class=" bibr popnode">12</a>], and to what extent each link in the chain from Ab developers to distributors, end users, peer reviewers, journal editors and funding agencies are responsible for ensuring adequate Ab validation, use and transparent reporting. Each step in this chain has practical considerations that confound what may be viewed as optimal. Even the largest Ab developers do not have the resources to validate every Ab for every imaginable end user application. However, it seems reasonable to validate Abs in a small number of standard applications (<em>e.g.</em>, IB, IHC) under a set of standard conditions, whose details are made clear to end users, to determine whether the Ab effectively and specifically recognized the full-length target protein endogenously expressed in native tissues, and not merely show data against purified or overexpressed targets. Validation against samples from KO animals would be an added bonus that would set an Ab above most others generated against the same target (as a case in point, there are more than 400 NeuroMabs in the catalog and more than 130 of them have been KO-validated). Antibody distributors (developers and resellers) should have internal quality control systems in place to assay each new Ab production lot relative to established standards, with inferior lots rejected, and should the problem persist, the Ab pulled from the catalog. We have performed such quality control at NeuroMab for all production lots prior to their release. In the case of Ab end users, while &#x0201c;<em>Caveat emptor</em>&#x0201d; always applies, regardless of the reputation of the developer or distributor. However, it does seem that there should be some level of confidence that, to paraphrase Rimm and colleagues &#x0201c;what is on the label corresponds to what is in the tube&#x0201d; [<a href="#R17" rid="R17" class=" bibr popnode">17</a>], as is the case with most other research reagents, especially those obtained at a premium price. It seems unreasonable to pay a high mark-up for a commercial Ab and then need to validate it in-house, especially since extensive Ab validation, as performed in house at the NeuroMab Facility or in certain end user labs [<a href="#R17" rid="R17" class=" bibr popnode">17</a>], is costly in time and effort. That said, any data generated using the Ab will ultimately reflect on the end user and their research program, such that it is in their best interest to perform some basic level of validation, with appropriate positive and negative primary and secondary Ab controls, prior to embarking on key experiments. However, it remains that the optimal balance between these efforts on the part of developers, distributors and end users is not clear, and it remains the choice of scientists at each step in the chain to strive for accuracy and transparency in their efforts. Peer reviewers and journal editors play a unique role in enforcing high standards, for example by requiring transparency in experimental details, and including the use of unambiguous research Ab identifiers such as RRIDs, as well as minimal standards for reporting the extent and nature of validation for each Ab used, as recently proposed for research involving cell lines [<a href="#R55" rid="R55" class=" bibr popnode">55</a>]. Funding agencies can contribute by supporting and/or maintaining curated lists of validated Abs that generate high-quality and reproducible data, and ensure that all funded investigators, study section members and program officials are aware of these lists. Funding agencies should continue to support efforts in the non-profit sector to provide highly validated and renewable Abs to the research community, if for no other reason than for the arguments that have been presented that this is a cost effective approach to the management of research funding [<a href="#R12" rid="R12" class=" bibr popnode">12</a>].</p></div><div id="S14" class="tsec sec"><h2 class="head no_bottom_margin" id="S14title">Conclusions</h2><p id="P32" class="p p-first-last">The generation and validation of high-quality Abs, and the subsequent generation of sound and reproducible data from Ab-based experiments, will continue to play a major role in the future of rigorous and effective proteomics level biomedical research. As such, every link in the Ab chain bears a responsibility for vigilance in experimental design, execution and interpretation, during Ab generation and validation, in evaluating production lots, and when using these reagents in the research laboratory setting, so as not to diminish the potential value that Abs can provide to researchers. An increased focus at each step, and its transparent reporting, will ensure research reproducibility, and avoid many of the problems that have been recently highlighted regarding Ab-based research. The UC Davis/NIH NeuroMab Facility strives to contribute towards this aim, primarily in the neuroscience research arena, but also more broadly, with the goal of serving as a resource, not only for the NeuroMab reagents themselves, but also as a model that may benefit efforts towards generating and validating highly validated Abs and other affinity reagents for biomedical research.</p></div><div id="unreferenced-ft" class="tsec sec"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="caption-9title">Highlights</h3></div><ul class="unordered" style="list-style-type:disc" id="L1"><li><div id="P33">High-quality antibodies are key to proteomic-level neuroscience research.</div></li><li><div id="P34">Irreproducible antibody data cause considerable frustration among neuroscientists.</div></li><li><div id="P35">We make mouse monoclonal antibodies validated via brain-based screens (NeuroMabs).</div></li><li><div id="P36">Determining NeuroMab specificity for staining brain tissue is a unique challenge.</div></li><li><div id="P37">We suggest ideas for responsible and reproducible antibody development and use.</div></li></ul></div></div></div><div id="S16" class="tsec sec"><h2 class="head no_bottom_margin" id="S16title">Acknowledgments</h2><div class="sec"><p>This work was funded by NIH research grants U24 NS050606 and R24 NS092991 to J. S. Trimmer. We thank the current and former staff members of the UC Davis/NIH NeuroMab Facility and the Trimmer laboratory for their contributions and dedicated efforts, Ms. Randi Jenkins for valuable institutional support at UC Davis, and Dr. Randall Stewart at the National Institute of Neurological Disorders and Stroke for support and helpful advice.</p></div></div><div id="GL" class="tsec sec"><h2 class="head no_bottom_margin" id="GLtitle">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">ABA</td><td>Allen Brain Atlas</td></tr><tr><td id="G2">Abs</td><td>antibodies</td></tr><tr><td id="G3">BSA</td><td>bovine serum albumin</td></tr><tr><td id="G4">GST</td><td>glutathione S-transferase</td></tr><tr><td id="G5">ELISA</td><td>enzyme-linked immunosorbent assay</td></tr><tr><td id="G6">FLISA</td><td>fluorescence-linked immunosorbent assay</td></tr><tr><td id="G7">H+L</td><td>heavy-and-light-chain</td></tr><tr><td id="G8">HRP</td><td>horseradish peroxidase</td></tr><tr><td id="G9">IB</td><td>immunoblotting</td></tr><tr><td id="G10">ICC</td><td>immunocytochemistry</td></tr><tr><td id="G11">IF</td><td>immunofluorescence</td></tr><tr><td id="G12">IHC</td><td>immunohistochemistry</td></tr><tr><td id="G13">KLH</td><td>keyhole limpet hemocyanin</td></tr><tr><td id="G14">KO</td><td>knockout</td></tr><tr><td id="G15">mAbs</td><td>monoclonal antibodies</td></tr><tr><td id="G16">MBP</td><td>maltose binding protein</td></tr><tr><td id="G17">MS</td><td>mass spectrometry</td></tr><tr><td id="G18">NiDAB</td><td>nickel enhanced diaminobenzidine</td></tr><tr><td id="G19">pAbs</td><td>polyclonal antibodies</td></tr><tr><td id="G20">PEG</td><td>polyethylene glycol</td></tr><tr><td id="G21">PTM</td><td>posttranslational modification</td></tr><tr><td id="G22">SDS</td><td>sodium dodecyl sulfate</td></tr><tr><td id="G23">sr</td><td>stratum radiatum</td></tr><tr><td id="G24">TCS</td><td>tissue culture supernatants</td></tr><tr><td id="G25">WT</td><td>wild-type</td></tr></tbody></table></div></div></div><div id="fn-group-1" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-1title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"><p class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p></p></div></div><div id="ref-list-1" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-1title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Saper CB. An open letter to our readers on the use of antibodies. <span><span class="ref-journal">J Comp Neurol. </span>2005;<span class="ref-vol">493</span>:477–478.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16304632" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comp+Neurol&amp;title=An+open+letter+to+our+readers+on+the+use+of+antibodies&amp;author=CB+Saper&amp;volume=493&amp;publication_year=2005&amp;pages=477-478&amp;pmid=16304632&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Rhodes KJ, Trimmer JS. Antibodies as valuable neuroscience research tools versus reagents of mass distraction. <span><span class="ref-journal">J Neurosci. </span>2006;<span class="ref-vol">26</span>:8017–8020.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6673789/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16885215" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Antibodies+as+valuable+neuroscience+research+tools+versus+reagents+of+mass+distraction&amp;author=KJ+Rhodes&amp;author=JS+Trimmer&amp;volume=26&amp;publication_year=2006&amp;pages=8017-8020&amp;pmid=16885215&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Saper CB. A guide to the perplexed on the specificity of antibodies. <span><span class="ref-journal">J Histochem Cytochem. </span>2009;<span class="ref-vol">57</span>:1–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2605712/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18854594" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Histochem+Cytochem&amp;title=A+guide+to+the+perplexed+on+the+specificity+of+antibodies&amp;author=CB+Saper&amp;volume=57&amp;publication_year=2009&amp;pages=1-5&amp;pmid=18854594&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Baker M. Reproducibility crisis: Blame it on the antibodies. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">521</span>:274–276.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25993940" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Reproducibility+crisis:+Blame+it+on+the+antibodies&amp;author=M+Baker&amp;volume=521&amp;publication_year=2015&amp;pages=274-276&amp;pmid=25993940&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Moser N, Mechawar N, Jones I, Gochberg-Sarver A, Orr-Urtreger A, et al. Evaluating the suitability of nicotinic acetylcholine receptor antibodies for standard immunodetection procedures. <span><span class="ref-journal">J Neurochem. </span>2007;<span class="ref-vol">102</span>:479–492.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17419810" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurochem&amp;title=Evaluating+the+suitability+of+nicotinic+acetylcholine+receptor+antibodies+for+standard+immunodetection+procedures&amp;author=N+Moser&amp;author=N+Mechawar&amp;author=I+Jones&amp;author=A+Gochberg-Sarver&amp;author=A+Orr-Urtreger&amp;volume=102&amp;publication_year=2007&amp;pages=479-492&amp;pmid=17419810&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Lu X, Bartfai T. Analyzing the validity of GalR1 and GalR2 antibodies using knockout mice. <span><span class="ref-journal">Naunyn Schmiedebergs Arch Pharmacol. </span>2009;<span class="ref-vol">379</span>:417–420.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2725758/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19159918" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Naunyn+Schmiedebergs+Arch+Pharmacol&amp;title=Analyzing+the+validity+of+GalR1+and+GalR2+antibodies+using+knockout+mice&amp;author=X+Lu&amp;author=T+Bartfai&amp;volume=379&amp;publication_year=2009&amp;pages=417-420&amp;pmid=19159918&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Herkenham M, Rathore P, Brown P, Listwak SJ. Cautionary notes on the use of NF-kappaB p65 and p50 antibodies for CNS studies. <span><span class="ref-journal">J Neuroinflammation. </span>2011;<span class="ref-vol">8</span>:141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3210105/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21999414" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neuroinflammation&amp;title=Cautionary+notes+on+the+use+of+NF-kappaB+p65+and+p50+antibodies+for+CNS+studies&amp;author=M+Herkenham&amp;author=P+Rathore&amp;author=P+Brown&amp;author=SJ+Listwak&amp;volume=8&amp;publication_year=2011&amp;pages=141&amp;pmid=21999414&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Ayala R, Kett LR, Leach TL, Young AB, Dunah AW, et al. Metabotropic glutamate receptor 1 (mGluR1): antibody specificity and receptor expression in cultured primary neurons. <span><span class="ref-journal">J Neurosci Methods. </span>2012;<span class="ref-vol">204</span>:221–226.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22155385" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci+Methods&amp;title=Metabotropic+glutamate+receptor+1+(mGluR1):+antibody+specificity+and+receptor+expression+in+cultured+primary+neurons&amp;author=R+Ayala&amp;author=LR+Kett&amp;author=TL+Leach&amp;author=AB+Young&amp;author=AW+Dunah&amp;volume=204&amp;publication_year=2012&amp;pages=221-226&amp;pmid=22155385&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Baek JH, Darlington CL, Smith PF, Ashton JC. Antibody testing for brain immunohistochemistry: brain immunolabeling for the cannabinoid CB(2) receptor. <span><span class="ref-journal">J Neurosci Methods. </span>2013;<span class="ref-vol">216</span>:87–95.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23583232" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci+Methods&amp;title=Antibody+testing+for+brain+immunohistochemistry:+brain+immunolabeling+for+the+cannabinoid+CB(2)+receptor&amp;author=JH+Baek&amp;author=CL+Darlington&amp;author=PF+Smith&amp;author=JC+Ashton&amp;volume=216&amp;publication_year=2013&amp;pages=87-95&amp;pmid=23583232&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Jackson TC, Bayir H, Ikonomovic MD, Janesko-Feldman K, Mi Z, et al. Detection of PHLPP1alpha/beta in human and mouse brain by different anti-PHLPP1 antibodies. <span><span class="ref-journal">Sci Rep. </span>2015;<span class="ref-vol">5</span>:9377.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4649889/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25827478" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Detection+of+PHLPP1alpha/beta+in+human+and+mouse+brain+by+different+anti-PHLPP1+antibodies&amp;author=TC+Jackson&amp;author=H+Bayir&amp;author=MD+Ikonomovic&amp;author=K+Janesko-Feldman&amp;author=Z+Mi&amp;volume=5&amp;publication_year=2015&amp;pages=9377&amp;pmid=25827478&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Schneider EH, Seifert R. The histamine H-receptor and the central and peripheral nervous system: A critical analysis of the literature. <span><span class="ref-journal">Neuropharmacology. </span>2015</span> in press. [<a href="https://pubmed.ncbi.nlm.nih.gov/25986697" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropharmacology&amp;title=The+histamine+H-receptor+and+the+central+and+peripheral+nervous+system:+A+critical+analysis+of+the+literature&amp;author=EH+Schneider&amp;author=R+Seifert&amp;publication_year=2015&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Bradbury A, Pluckthun A. Reproducibility: Standardize antibodies used in research. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">518</span>:27–29.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25652980" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Reproducibility:+Standardize+antibodies+used+in+research&amp;author=A+Bradbury&amp;author=A+Pluckthun&amp;volume=518&amp;publication_year=2015&amp;pages=27-29&amp;pmid=25652980&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Bekele-Arcuri Z, Matos MF, Manganas L, Strassle BW, Monaghan MM, et al. Generation and characterization of subtype-specific monoclonal antibodies to K+ channel alpha- and beta-subunit polypeptides. <span><span class="ref-journal">Neuropharmacology. </span>1996;<span class="ref-vol">35</span>:851–865.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8938716" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropharmacology&amp;title=Generation+and+characterization+of+subtype-specific+monoclonal+antibodies+to+K++channel+alpha-+and+beta-subunit+polypeptides&amp;author=Z+Bekele-Arcuri&amp;author=MF+Matos&amp;author=L+Manganas&amp;author=BW+Strassle&amp;author=MM+Monaghan&amp;volume=35&amp;publication_year=1996&amp;pages=851-865&amp;pmid=8938716&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Rhodes KJ, Strassle BW, Monaghan MM, Bekele-Arcuri Z, Matos MF, et al. Association and colocalization of the Kvbeta1 and Kvbeta2 beta-subunits with Kv1 alpha-subunits in mammalian brain K+ channel complexes. <span><span class="ref-journal">J Neurosci. </span>1997;<span class="ref-vol">17</span>:8246–8258.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6573739/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9334400" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Association+and+colocalization+of+the+Kvbeta1+and+Kvbeta2+beta-subunits+with+Kv1+alpha-subunits+in+mammalian+brain+K++channel+complexes&amp;author=KJ+Rhodes&amp;author=BW+Strassle&amp;author=MM+Monaghan&amp;author=Z+Bekele-Arcuri&amp;author=MF+Matos&amp;volume=17&amp;publication_year=1997&amp;pages=8246-8258&amp;pmid=9334400&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Rasband MN, Peles E, Trimmer JS, Levinson SR, Lux SE, et al. Dependence of nodal sodium channel clustering on paranodal axoglial contact in the developing CNS. <span><span class="ref-journal">J Neurosci. </span>1999;<span class="ref-vol">19</span>:7516–7528.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6782503/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10460258" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Dependence+of+nodal+sodium+channel+clustering+on+paranodal+axoglial+contact+in+the+developing+CNS&amp;author=MN+Rasband&amp;author=E+Peles&amp;author=JS+Trimmer&amp;author=SR+Levinson&amp;author=SE+Lux&amp;volume=19&amp;publication_year=1999&amp;pages=7516-7528&amp;pmid=10460258&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Rhodes KJ, Carroll KI, Sung MA, Doliveira LC, Monaghan MM, et al. KChIPs and Kv4 alpha subunits as integral components of A-type potassium channels in mammalian brain. <span><span class="ref-journal">J Neurosci. </span>2004;<span class="ref-vol">24</span>:7903–7915.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6729940/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15356203" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=KChIPs+and+Kv4+alpha+subunits+as+integral+components+of+A-type+potassium+channels+in+mammalian+brain&amp;author=KJ+Rhodes&amp;author=KI+Carroll&amp;author=MA+Sung&amp;author=LC+Doliveira&amp;author=MM+Monaghan&amp;volume=24&amp;publication_year=2004&amp;pages=7903-7915&amp;pmid=15356203&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Bordeaux J, Welsh A, Agarwal S, Killiam E, Baquero M, et al. Antibody validation. <span><span class="ref-journal">BioTechniques. </span>2010;<span class="ref-vol">48</span>:197–209.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3891910/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20359301" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BioTechniques&amp;title=Antibody+validation&amp;author=J+Bordeaux&amp;author=A+Welsh&amp;author=S+Agarwal&amp;author=E+Killiam&amp;author=M+Baquero&amp;volume=48&amp;publication_year=2010&amp;pages=197-209&amp;pmid=20359301&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Rodrigues AF, Soares HR, Guerreiro MR, Alves PM, Coroadinha AS. Viral vaccines and their manufacturing cell substrates: New trends and designs in modern vaccinology. <span><span class="ref-journal">Biotechnol J. </span>2015;<span class="ref-vol">10</span>:1329–1344.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7161866/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26212697" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol+J&amp;title=Viral+vaccines+and+their+manufacturing+cell+substrates:+New+trends+and+designs+in+modern+vaccinology&amp;author=AF+Rodrigues&amp;author=HR+Soares&amp;author=MR+Guerreiro&amp;author=PM+Alves&amp;author=AS+Coroadinha&amp;volume=10&amp;publication_year=2015&amp;pages=1329-1344&amp;pmid=26212697&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Uhlen M, Bjorling E, Agaton C, Szigyarto CA, Amini B, et al. A human protein atlas for normal and cancer tissues based on antibody proteomics. <span><span class="ref-journal">Mol Cell Proteomics. </span>2005;<span class="ref-vol">4</span>:1920–1932.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16127175" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Proteomics&amp;title=A+human+protein+atlas+for+normal+and+cancer+tissues+based+on+antibody+proteomics&amp;author=M+Uhlen&amp;author=E+Bjorling&amp;author=C+Agaton&amp;author=CA+Szigyarto&amp;author=B+Amini&amp;volume=4&amp;publication_year=2005&amp;pages=1920-1932&amp;pmid=16127175&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Baumgarten H, Schulze M, Hebell T.  Methods of Immunizing Mice. Monoclonal Antibodies. In: Peters JH, Baumgarten H, editors. <span class="ref-journal">Monoclonal Antibodies.</span> Springer-Verlag; 1992. pp. 50–57. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Monoclonal+Antibodies&amp;author=H+Baumgarten&amp;author=M+Schulze&amp;author=T+Hebell&amp;publication_year=1992&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Hu JG, Yokoyama T, Kitagawa T. Studies on the optimal immunization schedule of experimental animals. IV. The optimal age and sex of mice, and the influence of booster injections. <span><span class="ref-journal">Chem Pharm Bull (Tokyo) </span>1990;<span class="ref-vol">38</span>:448–451.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2337959" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Pharm+Bull+(Tokyo)&amp;title=Studies+on+the+optimal+immunization+schedule+of+experimental+animals.+IV.+The+optimal+age+and+sex+of+mice,+and+the+influence+of+booster+injections&amp;author=JG+Hu&amp;author=T+Yokoyama&amp;author=T+Kitagawa&amp;volume=38&amp;publication_year=1990&amp;pages=448-451&amp;pmid=2337959&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Kanduser M, Usaj M. Cell electrofusion: past and future perspectives for antibody production and cancer cell vaccines. <span><span class="ref-journal">Expert Opin Drug Deliv. </span>2014;<span class="ref-vol">11</span>:1885–1898.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25010248" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Opin+Drug+Deliv&amp;title=Cell+electrofusion:+past+and+future+perspectives+for+antibody+production+and+cancer+cell+vaccines&amp;author=M+Kanduser&amp;author=M+Usaj&amp;volume=11&amp;publication_year=2014&amp;pages=1885-1898&amp;pmid=25010248&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Manning CF, Bundros AM, Trimmer JS. Benefits and pitfalls of secondary antibodies: why choosing the right secondary is of primary importance. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e38313.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3365890/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22675541" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Benefits+and+pitfalls+of+secondary+antibodies:+why+choosing+the+right+secondary+is+of+primary+importance&amp;author=CF+Manning&amp;author=AM+Bundros&amp;author=JS+Trimmer&amp;volume=7&amp;publication_year=2012&amp;pages=e38313&amp;pmid=22675541&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Bell MR, Engleka MJ, Malik A, Strickler JE. To fuse or not to fuse: what is your purpose? <span><span class="ref-journal">Protein Sci. </span>2013;<span class="ref-vol">22</span>:1466–1477.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3831663/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24038604" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci&amp;title=To+fuse+or+not+to+fuse:+what+is+your+purpose?&amp;author=MR+Bell&amp;author=MJ+Engleka&amp;author=A+Malik&amp;author=JE+Strickler&amp;volume=22&amp;publication_year=2013&amp;pages=1466-1477&amp;pmid=24038604&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Wogulis M, Chew ER, Donohoue PD, Wilson DK. Identification of formyl kynurenine formamidase and kynurenine aminotransferase from Saccharomyces cerevisiae using crystallographic, bioinformatic and biochemical evidence. <span><span class="ref-journal">Biochemistry. </span>2008;<span class="ref-vol">47</span>:1608–1621.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18205391" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Identification+of+formyl+kynurenine+formamidase+and+kynurenine+aminotransferase+from+Saccharomyces+cerevisiae+using+crystallographic,+bioinformatic+and+biochemical+evidence&amp;author=M+Wogulis&amp;author=ER+Chew&amp;author=PD+Donohoue&amp;author=DK+Wilson&amp;volume=47&amp;publication_year=2008&amp;pages=1608-1621&amp;pmid=18205391&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Chong S, Mersha FB, Comb DG, Scott ME, Landry D, et al. Single-column purification of free recombinant proteins using a self-cleavable affinity tag derived from a protein splicing element. <span><span class="ref-journal">Gene. </span>1997;<span class="ref-vol">192</span>:271–281.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9224900" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Single-column+purification+of+free+recombinant+proteins+using+a+self-cleavable+affinity+tag+derived+from+a+protein+splicing+element&amp;author=S+Chong&amp;author=FB+Mersha&amp;author=DG+Comb&amp;author=ME+Scott&amp;author=D+Landry&amp;volume=192&amp;publication_year=1997&amp;pages=271-281&amp;pmid=9224900&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Lewkowich IP, Campbell JD, HayGlass KT. Comparison of chemiluminescent assays and colorimetric ELISAs for quantification of murine IL-12, human IL-4 and murine IL-4: chemiluminescent substrates provide markedly enhanced sensitivity. <span><span class="ref-journal">J Immunol Methods. </span>2001;<span class="ref-vol">247</span>:111–118.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11150542" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol+Methods&amp;title=Comparison+of+chemiluminescent+assays+and+colorimetric+ELISAs+for+quantification+of+murine+IL-12,+human+IL-4+and+murine+IL-4:+chemiluminescent+substrates+provide+markedly+enhanced+sensitivity&amp;author=IP+Lewkowich&amp;author=JD+Campbell&amp;author=KT+HayGlass&amp;volume=247&amp;publication_year=2001&amp;pages=111-118&amp;pmid=11150542&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Gray EG, Whittaker VP. The isolation of nerve endings from brain: an electron-microscopic study of cell fragments derived by homogenization and centrifugation. <span><span class="ref-journal">J Anat. </span>1962;<span class="ref-vol">96</span>:79–88.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1244174/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13901297" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Anat&amp;title=The+isolation+of+nerve+endings+from+brain:+an+electron-microscopic+study+of+cell+fragments+derived+by+homogenization+and+centrifugation&amp;author=EG+Gray&amp;author=VP+Whittaker&amp;volume=96&amp;publication_year=1962&amp;pages=79-88&amp;pmid=13901297&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Rhodes KJ, Keilbaugh SA, Barrezueta NX, Lopez KL, Trimmer JS. Association and colocalization of K+ channel alpha- and beta-subunit polypeptides in rat brain. <span><span class="ref-journal">J Neurosci. </span>1995;<span class="ref-vol">15</span>:5360–5371.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6577887/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7623158" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Association+and+colocalization+of+K++channel+alpha-+and+beta-subunit+polypeptides+in+rat+brain&amp;author=KJ+Rhodes&amp;author=SA+Keilbaugh&amp;author=NX+Barrezueta&amp;author=KL+Lopez&amp;author=JS+Trimmer&amp;volume=15&amp;publication_year=1995&amp;pages=5360-5371&amp;pmid=7623158&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Berendt FJ, Park KS, Trimmer JS. Multisite phosphorylation of voltage-gated sodium channel alpha subunits from rat brain. <span><span class="ref-journal">J Proteome Res. </span>2010;<span class="ref-vol">9</span>:1976–1984.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2849892/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20131913" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Proteome+Res&amp;title=Multisite+phosphorylation+of+voltage-gated+sodium+channel+alpha+subunits+from+rat+brain&amp;author=FJ+Berendt&amp;author=KS+Park&amp;author=JS+Trimmer&amp;volume=9&amp;publication_year=2010&amp;pages=1976-1984&amp;pmid=20131913&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Holmseth S, Lehre KP, Danbolt NC. Specificity controls for immunocytochemistry. <span><span class="ref-journal">Anat Embryol (Berl) </span>2006;<span class="ref-vol">211</span>:257–266.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16435108" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anat+Embryol+(Berl)&amp;title=Specificity+controls+for+immunocytochemistry&amp;author=S+Holmseth&amp;author=KP+Lehre&amp;author=NC+Danbolt&amp;volume=211&amp;publication_year=2006&amp;pages=257-266&amp;pmid=16435108&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Holmseth S, Zhou Y, Follin-Arbelet VV, Lehre KP, Bergles DE, et al. Specificity controls for immunocytochemistry: the antigen preadsorption test can lead to inaccurate assessment of antibody specificity. <span><span class="ref-journal">J Histochem Cytochem. </span>2012;<span class="ref-vol">60</span>:174–187.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3351124/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22215633" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Histochem+Cytochem&amp;title=Specificity+controls+for+immunocytochemistry:+the+antigen+preadsorption+test+can+lead+to+inaccurate+assessment+of+antibody+specificity&amp;author=S+Holmseth&amp;author=Y+Zhou&amp;author=VV+Follin-Arbelet&amp;author=KP+Lehre&amp;author=DE+Bergles&amp;volume=60&amp;publication_year=2012&amp;pages=174-187&amp;pmid=22215633&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Lorincz A, Nusser Z. Specificity of immunoreactions: the importance of testing specificity in each method. <span><span class="ref-journal">J Neurosci. </span>2008;<span class="ref-vol">28</span>:9083–9086.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2629537/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18784286" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Specificity+of+immunoreactions:+the+importance+of+testing+specificity+in+each+method&amp;author=A+Lorincz&amp;author=Z+Nusser&amp;volume=28&amp;publication_year=2008&amp;pages=9083-9086&amp;pmid=18784286&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Gong B, Rhodes KJ, Bekele-Arcuri Z, Trimmer JS. Type I and type II Na(+) channel alpha-subunit polypeptides exhibit distinct spatial and temporal patterning, and association with auxiliary subunits in rat brain. <span><span class="ref-journal">J Comp Neurol. </span>1999;<span class="ref-vol">412</span>:342–352.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10441760" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comp+Neurol&amp;title=Type+I+and+type+II+Na(+)+channel+alpha-subunit+polypeptides+exhibit+distinct+spatial+and+temporal+patterning,+and+association+with+auxiliary+subunits+in+rat+brain&amp;author=B+Gong&amp;author=KJ+Rhodes&amp;author=Z+Bekele-Arcuri&amp;author=JS+Trimmer&amp;volume=412&amp;publication_year=1999&amp;pages=342-352&amp;pmid=10441760&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Fritschy JM, Weinmann O, Wenzel A, Benke D. Synapse-specific localization of NMDA and GABA(A) receptor subunits revealed by antigen-retrieval immunohistochemistry. <span><span class="ref-journal">J Comp Neurol. </span>1998;<span class="ref-vol">390</span>:194–210.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9453664" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comp+Neurol&amp;title=Synapse-specific+localization+of+NMDA+and+GABA(A)+receptor+subunits+revealed+by+antigen-retrieval+immunohistochemistry&amp;author=JM+Fritschy&amp;author=O+Weinmann&amp;author=A+Wenzel&amp;author=D+Benke&amp;volume=390&amp;publication_year=1998&amp;pages=194-210&amp;pmid=9453664&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Jiao Y, Sun Z, Lee T, Fusco FR, Kimble TD, et al. A simple and sensitive antigen retrieval method for free-floating and slide-mounted tissue sections. <span><span class="ref-journal">J Neurosci Methods. </span>1999;<span class="ref-vol">93</span>:149–162.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10634500" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci+Methods&amp;title=A+simple+and+sensitive+antigen+retrieval+method+for+free-floating+and+slide-mounted+tissue+sections&amp;author=Y+Jiao&amp;author=Z+Sun&amp;author=T+Lee&amp;author=FR+Fusco&amp;author=TD+Kimble&amp;volume=93&amp;publication_year=1999&amp;pages=149-162&amp;pmid=10634500&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Fukaya M, Watanabe M. Improved immunohistochemical detection of postsynaptically located PSD-95/SAP90 protein family by protease section pretreatment: a study in the adult mouse brain. <span><span class="ref-journal">J Comp Neurol. </span>2000;<span class="ref-vol">426</span>:572–586.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11027400" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comp+Neurol&amp;title=Improved+immunohistochemical+detection+of+postsynaptically+located+PSD-95/SAP90+protein+family+by+protease+section+pretreatment:+a+study+in+the+adult+mouse+brain&amp;author=M+Fukaya&amp;author=M+Watanabe&amp;volume=426&amp;publication_year=2000&amp;pages=572-586&amp;pmid=11027400&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Lyck L, Dalmau I, Chemnitz J, Finsen B, Schroder HD. Immunohistochemical markers for quantitative studies of neurons and glia in human neocortex. <span><span class="ref-journal">J Histochem Cytochem. </span>2008;<span class="ref-vol">56</span>:201–221.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2324185/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17998570" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Histochem+Cytochem&amp;title=Immunohistochemical+markers+for+quantitative+studies+of+neurons+and+glia+in+human+neocortex&amp;author=L+Lyck&amp;author=I+Dalmau&amp;author=J+Chemnitz&amp;author=B+Finsen&amp;author=HD+Schroder&amp;volume=56&amp;publication_year=2008&amp;pages=201-221&amp;pmid=17998570&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Schitine C, Nogaroli L, Costa MR, Hedin-Pereira C. Astrocyte heterogeneity in the brain: from development to disease. <span><span class="ref-journal">Front Cell Neurosci. </span>2015;<span class="ref-vol">9</span>:76.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4367182/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25852472" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Cell+Neurosci&amp;title=Astrocyte+heterogeneity+in+the+brain:+from+development+to+disease&amp;author=C+Schitine&amp;author=L+Nogaroli&amp;author=MR+Costa&amp;author=C+Hedin-Pereira&amp;volume=9&amp;publication_year=2015&amp;pages=76&amp;pmid=25852472&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">DeFelipe J, Lopez-Cruz PL, Benavides-Piccione R, Bielza C, Larranaga P, et al. New insights into the classification and nomenclature of cortical GABAergic interneurons. <span><span class="ref-journal">Nat Rev Neurosci. </span>2013;<span class="ref-vol">14</span>:202–216.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3619199/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23385869" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Neurosci&amp;title=New+insights+into+the+classification+and+nomenclature+of+cortical+GABAergic+interneurons&amp;author=J+DeFelipe&amp;author=PL+Lopez-Cruz&amp;author=R+Benavides-Piccione&amp;author=C+Bielza&amp;author=P+Larranaga&amp;volume=14&amp;publication_year=2013&amp;pages=202-216&amp;pmid=23385869&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Lodato S, Shetty AS, Arlotta P. Cerebral cortex assembly: generating and reprogramming projection neuron diversity. <span><span class="ref-journal">Trends Neurosci. </span>2015;<span class="ref-vol">38</span>:117–125.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4334136/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25529141" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Neurosci&amp;title=Cerebral+cortex+assembly:+generating+and+reprogramming+projection+neuron+diversity&amp;author=S+Lodato&amp;author=AS+Shetty&amp;author=P+Arlotta&amp;volume=38&amp;publication_year=2015&amp;pages=117-125&amp;pmid=25529141&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Menshawi K, Mohr JP, Gutierrez J. A Functional Perspective on the Embryology and Anatomy of the Cerebral Blood Supply. <span><span class="ref-journal">J Stroke. </span>2015;<span class="ref-vol">17</span>:144–158.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4460334/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26060802" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Stroke&amp;title=A+Functional+Perspective+on+the+Embryology+and+Anatomy+of+the+Cerebral+Blood+Supply&amp;author=K+Menshawi&amp;author=JP+Mohr&amp;author=J+Gutierrez&amp;volume=17&amp;publication_year=2015&amp;pages=144-158&amp;pmid=26060802&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Kohara K, Pignatelli M, Rivest AJ, Jung HY, Kitamura T, et al. Cell type-specific genetic and optogenetic tools reveal hippocampal CA2 circuits. <span><span class="ref-journal">Nat Neurosci. </span>2014;<span class="ref-vol">17</span>:269–279.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4004172/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24336151" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Cell+type-specific+genetic+and+optogenetic+tools+reveal+hippocampal+CA2+circuits&amp;author=K+Kohara&amp;author=M+Pignatelli&amp;author=AJ+Rivest&amp;author=HY+Jung&amp;author=T+Kitamura&amp;volume=17&amp;publication_year=2014&amp;pages=269-279&amp;pmid=24336151&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Trimmer JS. Subcellular localization of K+ channels in mammalian brain neurons: remarkable precision in the midst of extraordinary complexity. <span><span class="ref-journal">Neuron. </span>2015;<span class="ref-vol">85</span>:238–256.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4303806/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25611506" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Subcellular+localization+of+K++channels+in+mammalian+brain+neurons:+remarkable+precision+in+the+midst+of+extraordinary+complexity&amp;author=JS+Trimmer&amp;volume=85&amp;publication_year=2015&amp;pages=238-256&amp;pmid=25611506&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Weir DM.  <span class="ref-journal">Handbook of Experimental Immunology.</span> 3. Blackwell; 1978.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Handbook+of+Experimental+Immunology&amp;author=DM+Weir&amp;publication_year=1978&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Crosnier C, Staudt N, Wright GJ. A rapid and scalable method for selecting recombinant mouse monoclonal antibodies. <span><span class="ref-journal">BMC Biol. </span>2010;<span class="ref-vol">8</span>:76.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2898661/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20525357" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Biol&amp;title=A+rapid+and+scalable+method+for+selecting+recombinant+mouse+monoclonal+antibodies&amp;author=C+Crosnier&amp;author=N+Staudt&amp;author=GJ+Wright&amp;volume=8&amp;publication_year=2010&amp;pages=76&amp;pmid=20525357&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Muller-Sienerth N, Crosnier C, Wright GJ, Staudt N. Cloning of recombinant monoclonal antibodies from hybridomas in a single mammalian expression plasmid. <span><span class="ref-journal">Methods Mol Biol. </span>2014;<span class="ref-vol">1131</span>:229–240.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24515469" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Cloning+of+recombinant+monoclonal+antibodies+from+hybridomas+in+a+single+mammalian+expression+plasmid&amp;author=N+Muller-Sienerth&amp;author=C+Crosnier&amp;author=GJ+Wright&amp;author=N+Staudt&amp;volume=1131&amp;publication_year=2014&amp;pages=229-240&amp;pmid=24515469&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Anderson CN, Grant SG. High throughput protein expression screening in the nervous system--needs and limitations. <span><span class="ref-journal">J Physiol. </span>2006;<span class="ref-vol">575</span>:367–372.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1819445/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16793899" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Physiol&amp;title=High+throughput+protein+expression+screening+in+the+nervous+system--needs+and+limitations&amp;author=CN+Anderson&amp;author=SG+Grant&amp;volume=575&amp;publication_year=2006&amp;pages=367-372&amp;pmid=16793899&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Landis SC, Amara SG, Asadullah K, Austin CP, Blumenstein R, et al. A call for transparent reporting to optimize the predictive value of preclinical research. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">490</span>:187–191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3511845/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23060188" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+call+for+transparent+reporting+to+optimize+the+predictive+value+of+preclinical+research&amp;author=SC+Landis&amp;author=SG+Amara&amp;author=K+Asadullah&amp;author=CP+Austin&amp;author=R+Blumenstein&amp;volume=490&amp;publication_year=2012&amp;pages=187-191&amp;pmid=23060188&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Saper CB, Sawchenko PE. Magic peptides, magic antibodies: guidelines for appropriate controls for immunohistochemistry. <span><span class="ref-journal">J Comp Neurol. </span>2003;<span class="ref-vol">465</span>:161–163.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12949777" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comp+Neurol&amp;title=Magic+peptides,+magic+antibodies:+guidelines+for+appropriate+controls+for+immunohistochemistry&amp;author=CB+Saper&amp;author=PE+Sawchenko&amp;volume=465&amp;publication_year=2003&amp;pages=161-163&amp;pmid=12949777&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Bandrowski A, Brush M, Grethe JS, Haendel MA, Kennedy DN, et al. The Resource Identification Initiative: A cultural shift in publishing. <span><span class="ref-journal">F1000Research. </span>2015;<span class="ref-vol">4</span>:134.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4648211/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26594330" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=F1000Research&amp;title=The+Resource+Identification+Initiative:+A+cultural+shift+in+publishing&amp;author=A+Bandrowski&amp;author=M+Brush&amp;author=JS+Grethe&amp;author=MA+Haendel&amp;author=DN+Kennedy&amp;volume=4&amp;publication_year=2015&amp;pages=134&amp;pmid=26594330&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Nature Publising Group. Journals unite for reproducibility. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">515</span>:7.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Journals+unite+for+reproducibility&amp;volume=515&amp;publication_year=2014&amp;pages=7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Vasilevsky NA, Brush MH, Paddock H, Ponting L, Tripathy SJ, et al. On the reproducibility of science: unique identification of research resources in the biomedical literature. <span><span class="ref-journal">Peer J. </span>2013;<span class="ref-vol">1</span>:e148.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3771067/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24032093" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Peer+J&amp;title=On+the+reproducibility+of+science:+unique+identification+of+research+resources+in+the+biomedical+literature&amp;author=NA+Vasilevsky&amp;author=MH+Brush&amp;author=H+Paddock&amp;author=L+Ponting&amp;author=SJ+Tripathy&amp;volume=1&amp;publication_year=2013&amp;pages=e148&amp;pmid=24032093&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Collins FS, Tabak LA. Policy: NIH plans to enhance reproducibility. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">505</span>:612–613.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4058759/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24482835" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Policy:+NIH+plans+to+enhance+reproducibility&amp;author=FS+Collins&amp;author=LA+Tabak&amp;volume=505&amp;publication_year=2014&amp;pages=612-613&amp;pmid=24482835&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Lorsch JR, Collins FS, Lippincott-Schwartz J. Cell Biology. Fixing problems with cell lines. <span><span class="ref-journal">Science. </span>2014;<span class="ref-vol">346</span>:1452–1453.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5101941/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25525228" ref="reftype=pubmed&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Cell+Biology.+Fixing+problems+with+cell+lines&amp;author=JR+Lorsch&amp;author=FS+Collins&amp;author=J+Lippincott-Schwartz&amp;volume=346&amp;publication_year=2014&amp;pages=1452-1453&amp;pmid=25525228&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4884554&amp;issue-id=270174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pubreader-link other_item"><a href="/pmc/articles/PMC4884554/?report=reader" class="sidefm-pmclink">PubReader</a></li>
<li class="pdf-link other_item"><a href="/pmc/articles/PMC4884554/pdf/nihms741564.pdf" class="int-view">PDF (1.6M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/4884554/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/4884554/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<div class="collections-button-container" data-article-id="4884554" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4884554%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="7yC9bn4KciKdRXlftnRTsFK0b3cVPKUHpF1K1wESxw4jkZaIGc9Rz0FLewjqH41j">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4884554%2F&amp;text=Developing%20High-Quality%20Mouse%20Monoclonal%20Antibodies%20for%20Neuroscience%20Research%20%02013%20Approaches%2C%20Perspectives%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4884554%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4884554/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC4884554/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/26644354/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC4884554/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/26644354/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC4884554/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/4884554/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.54110350c77632754ed7.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'" /><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css" /><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.586993420cfd.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.bf4d0b9d90eb88f67344.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC4884554/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
