{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "acd721d2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "fatal: destination path 'scrape-google-scholar' already exists and is not an empty directory.\r\n"
     ]
    }
   ],
   "source": [
    "!git clone https://github.com/dimitryzub/scrape-google-scholar.git"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "fc55b57a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: async-generator==1.10 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 1)) (1.10)\n",
      "Requirement already satisfied: attrs==22.2.0 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 2)) (22.2.0)\n",
      "Requirement already satisfied: certifi==2022.12.7 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 3)) (2022.12.7)\n",
      "Requirement already satisfied: charset-normalizer==3.0.1 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 4)) (3.0.1)\n",
      "Requirement already satisfied: Cython==0.29.33 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 5)) (0.29.33)\n",
      "Requirement already satisfied: exceptiongroup==1.1.0 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 6)) (1.1.0)\n",
      "Requirement already satisfied: google-search-results==2.4.1 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 7)) (2.4.1)\n",
      "Requirement already satisfied: h11==0.14.0 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 8)) (0.14.0)\n",
      "Requirement already satisfied: html5lib==1.1 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 9)) (1.1)\n",
      "Requirement already satisfied: idna==3.4 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 10)) (3.4)\n",
      "Requirement already satisfied: lxml==4.9.2 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 11)) (4.9.2)\n",
      "Requirement already satisfied: numpy==1.24.1 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 12)) (1.24.1)\n",
      "Requirement already satisfied: outcome==1.2.0 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 13)) (1.2.0)\n",
      "Requirement already satisfied: pandas==1.5.2 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 14)) (1.5.2)\n",
      "Requirement already satisfied: PySocks==1.7.1 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 15)) (1.7.1)\n",
      "Requirement already satisfied: python-dateutil==2.8.2 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 16)) (2.8.2)\n",
      "Requirement already satisfied: pytz==2022.7.1 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 17)) (2022.7.1)\n",
      "Requirement already satisfied: requests==2.28.2 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 18)) (2.28.2)\n",
      "Requirement already satisfied: selectolax==0.3.12 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 19)) (0.3.12)\n",
      "Requirement already satisfied: selenium==4.7.2 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 20)) (4.7.2)\n",
      "Requirement already satisfied: selenium-stealth==1.0.6 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 21)) (1.0.6)\n",
      "Requirement already satisfied: six==1.16.0 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 22)) (1.16.0)\n",
      "Requirement already satisfied: sniffio==1.3.0 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 23)) (1.3.0)\n",
      "Requirement already satisfied: sortedcontainers==2.4.0 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 24)) (2.4.0)\n",
      "Requirement already satisfied: trio==0.22.0 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 25)) (0.22.0)\n",
      "Requirement already satisfied: trio-websocket==0.9.2 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 26)) (0.9.2)\n",
      "Requirement already satisfied: urllib3==1.26.14 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 27)) (1.26.14)\n",
      "Requirement already satisfied: webencodings==0.5.1 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 28)) (0.5.1)\n",
      "Requirement already satisfied: wsproto==1.2.0 in ./opt/anaconda3/lib/python3.9/site-packages (from -r scrape-google-scholar/requirements.txt (line 29)) (1.2.0)\n"
     ]
    }
   ],
   "source": [
    "!pip install -r scrape-google-scholar/requirements.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ccc13ab1",
   "metadata": {},
   "source": [
    "## Import "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "9ad86166",
   "metadata": {},
   "outputs": [],
   "source": [
    "from selenium import webdriver\n",
    "from selenium_stealth import stealth\n",
    "from selenium.webdriver.chrome.service import Service\n",
    "from selectolax.lexbor import LexborHTMLParser\n",
    "import os, json\n",
    "from typing import List, Dict, Callable\n",
    "import time, random, re\n",
    "import pandas as pd\n",
    "\n",
    "import requests\n",
    "import io\n",
    "from io import BytesIO\n",
    "from io import StringIO\n",
    "\n",
    "import requests\n",
    "\n",
    "import re\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "import time "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "8a98a095",
   "metadata": {},
   "outputs": [],
   "source": [
    "def parse(parser: Callable, organic_results_data: Callable):\n",
    "    '''\n",
    "    Arugments:\n",
    "    - parser:  Lexbor parser from scrape_google_scholar_organic_results() function.\n",
    "    - organic_results_data: List to append data to. List origin location is scrape_google_scholar_organic_results() function. Line 104.\n",
    "    \n",
    "    This function parses data from Google Scholar Organic results and appends data to a List.\n",
    "    \n",
    "    It's used by scrape_google_scholar_organic_results().\n",
    "    \n",
    "    It returns nothing as it appends data to `organic_results_data`, \n",
    "    which appends it to `organic_results_data` List in the scrape_google_scholar_organic_results() function.\n",
    "    '''\n",
    "    \n",
    "    for result in parser.css('.gs_r.gs_or.gs_scl'):\n",
    "        try:\n",
    "            title: str = result.css_first('.gs_rt').text()\n",
    "        except: title = None\n",
    "\n",
    "        try:\n",
    "            title_link: str = result.css_first('.gs_rt a').attrs['href']\n",
    "        except: title_link = None\n",
    "\n",
    "        try:\n",
    "            publication_info: str = result.css_first('.gs_a').text()\n",
    "        except: publication_info = None\n",
    "\n",
    "        try:\n",
    "            snippet: str = result.css_first('.gs_rs').text()\n",
    "        except: snippet = None\n",
    "\n",
    "        try:\n",
    "            # if Cited by is present in inline links, it will be extracted\n",
    "            cited_by_link = ''.join([link.attrs['href'] for link in result.css('.gs_ri .gs_fl a') if 'Cited by' in link.text()])\n",
    "        except: cited_by_link = None\n",
    "        \n",
    "        try:\n",
    "            # if Cited by is present in inline links, it will be extracted and type cast it to integer\n",
    "            cited_by_count = int(''.join([re.search(r'\\d+', link.text()).group() for link in result.css('.gs_ri .gs_fl a') if 'Cited by' in link.text()]))\n",
    "        except: cited_by_count = None\n",
    "        \n",
    "        try:\n",
    "            pdf_file: str = result.css_first('.gs_or_ggsm a').attrs['href']\n",
    "        except: pdf_file = None\n",
    "\n",
    "        organic_results_data.append({\n",
    "            'title': title,\n",
    "            'title_link': title_link,\n",
    "            'publication_info': publication_info,\n",
    "            'snippet': snippet if snippet else None,\n",
    "            'cited_by_link': f'https://scholar.google.com{cited_by_link}' if cited_by_link else None,\n",
    "            'cited_by_count': cited_by_count if cited_by_count else None,\n",
    "            'pdf_file': pdf_file\n",
    "        })\n",
    "\n",
    "def scrape_google_scholar_organic_results(\n",
    "                                        query: str,\n",
    "                                        pagination: bool = False,\n",
    "                                        operating_system: str = 'Windows' or 'Linux',\n",
    "                                        year_start: int = None,\n",
    "                                        year_end: int = None,\n",
    "                                        save_to_csv: bool = False, \n",
    "                                        save_to_json: bool = False\n",
    "                                        ) -> List[Dict[str, str]]:\n",
    "    '''\n",
    "    Extracts data from Google Scholar Organic resutls page:\n",
    "    - title: str\n",
    "    - title_link: str\n",
    "    - publication_info: str \n",
    "    - snippet: str\n",
    "    - cited_by_link: str \n",
    "    - cited_by_count: int\n",
    "    - pdf_file: str\n",
    "    \n",
    "    Arguments:\n",
    "    - query: str. Search query. \n",
    "    - pagination: bool. Enables or disables pagination.\n",
    "    - operating_system: str. 'Windows' or 'Linux', Checks for operating system to either run Windows or Linux verson of chromedriver\n",
    "    \n",
    "    Usage:\n",
    "    data = scrape_google_scholar_organic_results(query='blizzard', pagination=False, operating_system='win') # pagination defaults to False \n",
    "    \n",
    "    for organic_result in data:\n",
    "        print(organic_result['title'])\n",
    "        print(organic_result['pdf_file'])\n",
    "    '''\n",
    "    assert year_start and year_end\n",
    "    if year_start or year_end:\n",
    "        assert year_start and year_end\n",
    "        assert year_start <= year_end\n",
    "\n",
    "    # selenium stealth\n",
    "    options = webdriver.ChromeOptions()\n",
    "    options.add_argument('--headless')\n",
    "    options.add_argument('--no-sandbox')\n",
    "    options.add_argument('--disable-dev-shm-usage')\n",
    "    \n",
    "    options.add_experimental_option('excludeSwitches', ['enable-automation'])\n",
    "    options.add_experimental_option('useAutomationExtension', False)\n",
    "    \n",
    "    # checks for operating system to either run Windows or Linux verson of chromedriver\n",
    "    # expects to have chromedriver near the runnable file\n",
    "    if operating_system is None:\n",
    "        raise Exception('Please provide your OS to `operating_system` argument: \"Windows\" or \"Linux\" for script to operate.')\n",
    "    \n",
    "    if operating_system.lower() == 'windows' or 'win':\n",
    "        driver = webdriver.Chrome(options=options, service=Service(executable_path='chromedriver.exe'))\n",
    "    \n",
    "    if operating_system.lower() == 'linux': \n",
    "        driver = webdriver.Chrome(options=options, service=Service(executable_path='chromedriver'))\n",
    "    \n",
    "    stealth(driver,\n",
    "        languages=['en-US', 'en'],\n",
    "        vendor='Google Inc.',\n",
    "        platform='Win32',\n",
    "        webgl_vendor='Intel Inc.',\n",
    "        renderer='Intel Iris OpenGL Engine',\n",
    "        fix_hairline=True,\n",
    "    )\n",
    "    \n",
    "    page_num: int = 0\n",
    "    organic_results_data: list = []\n",
    "    \n",
    "    if pagination:\n",
    "        while True:\n",
    "            # parse all pages\n",
    "            driver.get(f'https://scholar.google.com/scholar?q={query}&hl=en&gl=us&start={page_num}&as_ylo={year_start}&as_yhi={year_end}')\n",
    "            parser = LexborHTMLParser(driver.page_source)\n",
    "            \n",
    "            parse(parser=parser, organic_results_data=organic_results_data)\n",
    "            \n",
    "            # pagination\n",
    "            if parser.css_first('.gs_ico_nav_next'):  # checks for the \"Next\" page button\n",
    "                page_num += 10                        # paginate to the next page\n",
    "                time.sleep(random.randint(1, 2))      # sleep between paginations\n",
    "            else:\n",
    "                break\n",
    "    else:\n",
    "        # parse single, first page\n",
    "        driver.get(f'https://scholar.google.com/scholar?q={query}&hl=en&gl=us&start={page_num}')\n",
    "        parser = LexborHTMLParser(driver.page_source)\n",
    "    \n",
    "        parse(parser=parser, organic_results_data=organic_results_data)\n",
    "        \n",
    "    if save_to_csv:\n",
    "        pd.DataFrame(data=organic_results_data).to_csv('google_scholar_organic_results_data.csv', \n",
    "                                                        index=False, encoding='utf-8')\n",
    "    if save_to_json:\n",
    "        pd.DataFrame(data=organic_results_data).to_json('google_scholar_organic_results_data.json', \n",
    "                                                        index=False, orient='records')\n",
    "    driver.quit()\n",
    "    return organic_results_data"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4f269e9c",
   "metadata": {},
   "source": [
    "## Add target column from neuromab database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "0c992473",
   "metadata": {},
   "outputs": [],
   "source": [
    "def AddTargetCol(df,final_df):\n",
    "            \n",
    "    merged_df = final_df.merge(df[['Clone', 'Target']], left_on=['Antibody'],right_on=['Clone'], how='left')\n",
    "    merged_df = merged_df.drop('Clone', axis=1)\n",
    "   \n",
    "    return merged_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "53f2f8dd",
   "metadata": {},
   "source": [
    "## Grab Neuromab DataFrame from site"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "1e84ce60",
   "metadata": {},
   "outputs": [],
   "source": [
    "data = requests.get(\"https://neuromab.ucdavis.edu/catalog-download.cfm\").content\n",
    "df = pd.read_csv(BytesIO(data))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e0158f10",
   "metadata": {},
   "source": [
    "## Grab Catalog df for query creation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "b5c86f36",
   "metadata": {},
   "outputs": [],
   "source": [
    "df2 = pd.read_excel('/Users/HMans_MacBook_Pro/Library/Mobile Documents/com~apple~CloudDocs/Desktop/TrimmerLab/runresults.xlsx')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0afa0ce0",
   "metadata": {},
   "source": [
    "## Create URL"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "25550b97",
   "metadata": {},
   "outputs": [],
   "source": [
    "def CreatURL(query):\n",
    "    \n",
    "    query1 = query[0:8]\n",
    "    end_pos = query.find('/', 9)\n",
    "    query2= query[9:end_pos]\n",
    "    query3= query[end_pos+1:]\n",
    "    \n",
    "    url = 'https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=' + (query1 + '+' + query2 + '%' + '2F' + query3 + '&oq=neuromab+')\n",
    "\n",
    "    return url"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f56f4bb8",
   "metadata": {},
   "source": [
    "## Create Dataframe function "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "d813f07a",
   "metadata": {},
   "outputs": [],
   "source": [
    "def CreateDF(query,result_value,data):\n",
    "    \n",
    "    # Record the difference between search result # and data length \n",
    "    count = (int(result_value)) - (int({len(data)}.pop()))\n",
    "    final_df = pd.DataFrame(data)\n",
    "    \n",
    "    # Get rid of extraneous strings from title \n",
    "    final_df['title'] = final_df['title'].str.replace('\\[HTML\\]\\[HTML\\]|\\[PDF\\]\\[PDF\\]|\\[CITATION\\]|\\[C\\]|\\[BOOK\\]\\[BOOK\\]|\\[B\\]|\\[BOOK\\]\\[B\\]', '')\n",
    "\n",
    "    # Creat file for data \n",
    "    file_query = query.replace(\"/\",\"_\")\n",
    "    file_query = file_query[0:8]+file_query[9:]\n",
    "    filename = file_query + \".csv\"\n",
    "    directory = '/Users/HMans_MacBook_Pro/Library/Mobile Documents/com~apple~CloudDocs/Desktop/TrimmerLab/picklequeries'\n",
    "\n",
    "    # Input Antibody name into table \n",
    "    final_df[\"Antibody\"] = [query[9:]] * len(final_df)\n",
    "    final_df = AddTargetCol(df,final_df)\n",
    "    final_df.to_csv(os.path.join(directory, filename), index=False)\n",
    "    \n",
    "    # Input different result string based on difference in count \n",
    "    if count == 0:\n",
    "        print(query + ' ' + 'All accounted for : DF saved to file')\n",
    "        df2.loc[df2['Clone'] == query_name, 'Result'] = 'All accounted for : DF saved to file'\n",
    "\n",
    "    elif (count < 4) and (count > 0) :\n",
    "        print(query + ' ' +'There seems to be'+\" \"+str(count)+' '+\"paper(s) w/o years\")\n",
    "        df2.loc[df2['Clone'] == query_name, 'Result'] = 'There seems to be'+\" \"+str(count)+' '+\"paper(s) w/o years\"\n",
    "\n",
    "    else:\n",
    "        print(query + ' ' +'Something went wrong with Search, too many papers not found')\n",
    "        df2.loc[df2['Clone'] == query_name, 'Result'] = 'Something went wrong with Search, too many papers not found'\n",
    "\n",
    "    return final_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d1805612",
   "metadata": {},
   "source": [
    "## Scholar Query : search into google scholar and grab data from query "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "86de5f79",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "ename": "KeyError",
     "evalue": "'title'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "File \u001b[0;32m~/opt/anaconda3/lib/python3.9/site-packages/pandas/core/indexes/base.py:3803\u001b[0m, in \u001b[0;36mIndex.get_loc\u001b[0;34m(self, key, method, tolerance)\u001b[0m\n\u001b[1;32m   3802\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 3803\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_engine\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget_loc\u001b[49m\u001b[43m(\u001b[49m\u001b[43mcasted_key\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   3804\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mKeyError\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m err:\n",
      "File \u001b[0;32m~/opt/anaconda3/lib/python3.9/site-packages/pandas/_libs/index.pyx:138\u001b[0m, in \u001b[0;36mpandas._libs.index.IndexEngine.get_loc\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/opt/anaconda3/lib/python3.9/site-packages/pandas/_libs/index.pyx:165\u001b[0m, in \u001b[0;36mpandas._libs.index.IndexEngine.get_loc\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32mpandas/_libs/hashtable_class_helper.pxi:5745\u001b[0m, in \u001b[0;36mpandas._libs.hashtable.PyObjectHashTable.get_item\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32mpandas/_libs/hashtable_class_helper.pxi:5753\u001b[0m, in \u001b[0;36mpandas._libs.hashtable.PyObjectHashTable.get_item\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;31mKeyError\u001b[0m: 'title'",
      "\nThe above exception was the direct cause of the following exception:\n",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "Input \u001b[0;32mIn [9]\u001b[0m, in \u001b[0;36m<cell line: 4>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     28\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m result_match:\n\u001b[1;32m     29\u001b[0m     result_value \u001b[38;5;241m=\u001b[39m result_match\u001b[38;5;241m.\u001b[39mgroup(\u001b[38;5;241m1\u001b[39m)\n\u001b[0;32m---> 30\u001b[0m     final_df \u001b[38;5;241m=\u001b[39m \u001b[43mCreateDF\u001b[49m\u001b[43m(\u001b[49m\u001b[43mquery\u001b[49m\u001b[43m,\u001b[49m\u001b[43mresult_value\u001b[49m\u001b[43m,\u001b[49m\u001b[43mdata\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     31\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m     32\u001b[0m     \u001b[38;5;28mprint\u001b[39m(query \u001b[38;5;241m+\u001b[39m \u001b[38;5;124m'\u001b[39m\u001b[38;5;124m \u001b[39m\u001b[38;5;124m'\u001b[39m \u001b[38;5;241m+\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mSearch - No papers Available\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "Input \u001b[0;32mIn [7]\u001b[0m, in \u001b[0;36mCreateDF\u001b[0;34m(query, result_value, data)\u001b[0m\n\u001b[1;32m      3\u001b[0m count \u001b[38;5;241m=\u001b[39m (\u001b[38;5;28mint\u001b[39m(result_value)) \u001b[38;5;241m-\u001b[39m (\u001b[38;5;28mint\u001b[39m({\u001b[38;5;28mlen\u001b[39m(data)}\u001b[38;5;241m.\u001b[39mpop()))\n\u001b[1;32m      4\u001b[0m final_df \u001b[38;5;241m=\u001b[39m pd\u001b[38;5;241m.\u001b[39mDataFrame(data)\n\u001b[0;32m----> 5\u001b[0m final_df[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mtitle\u001b[39m\u001b[38;5;124m'\u001b[39m] \u001b[38;5;241m=\u001b[39m \u001b[43mfinal_df\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mtitle\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[38;5;241m.\u001b[39mstr\u001b[38;5;241m.\u001b[39mreplace(\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[HTML\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[HTML\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]|\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[PDF\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[PDF\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]|\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[CITATION\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]|\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[C\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]|\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[BOOK\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[BOOK\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]|\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[B\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]|\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[BOOK\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m[B\u001b[39m\u001b[38;5;124m\\\u001b[39m\u001b[38;5;124m]\u001b[39m\u001b[38;5;124m'\u001b[39m, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124m'\u001b[39m)\n\u001b[1;32m      7\u001b[0m file_query \u001b[38;5;241m=\u001b[39m query\u001b[38;5;241m.\u001b[39mreplace(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m/\u001b[39m\u001b[38;5;124m\"\u001b[39m,\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m_\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m      8\u001b[0m file_query \u001b[38;5;241m=\u001b[39m file_query[\u001b[38;5;241m0\u001b[39m:\u001b[38;5;241m8\u001b[39m]\u001b[38;5;241m+\u001b[39mfile_query[\u001b[38;5;241m9\u001b[39m:]\n",
      "File \u001b[0;32m~/opt/anaconda3/lib/python3.9/site-packages/pandas/core/frame.py:3805\u001b[0m, in \u001b[0;36mDataFrame.__getitem__\u001b[0;34m(self, key)\u001b[0m\n\u001b[1;32m   3803\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mcolumns\u001b[38;5;241m.\u001b[39mnlevels \u001b[38;5;241m>\u001b[39m \u001b[38;5;241m1\u001b[39m:\n\u001b[1;32m   3804\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_getitem_multilevel(key)\n\u001b[0;32m-> 3805\u001b[0m indexer \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mcolumns\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget_loc\u001b[49m\u001b[43m(\u001b[49m\u001b[43mkey\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   3806\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m is_integer(indexer):\n\u001b[1;32m   3807\u001b[0m     indexer \u001b[38;5;241m=\u001b[39m [indexer]\n",
      "File \u001b[0;32m~/opt/anaconda3/lib/python3.9/site-packages/pandas/core/indexes/base.py:3805\u001b[0m, in \u001b[0;36mIndex.get_loc\u001b[0;34m(self, key, method, tolerance)\u001b[0m\n\u001b[1;32m   3803\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_engine\u001b[38;5;241m.\u001b[39mget_loc(casted_key)\n\u001b[1;32m   3804\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mKeyError\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m err:\n\u001b[0;32m-> 3805\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mKeyError\u001b[39;00m(key) \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01merr\u001b[39;00m\n\u001b[1;32m   3806\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mTypeError\u001b[39;00m:\n\u001b[1;32m   3807\u001b[0m     \u001b[38;5;66;03m# If we have a listlike key, _check_indexing_error will raise\u001b[39;00m\n\u001b[1;32m   3808\u001b[0m     \u001b[38;5;66;03m#  InvalidIndexError. Otherwise we fall through and re-raise\u001b[39;00m\n\u001b[1;32m   3809\u001b[0m     \u001b[38;5;66;03m#  the TypeError.\u001b[39;00m\n\u001b[1;32m   3810\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_check_indexing_error(key)\n",
      "\u001b[0;31mKeyError\u001b[0m: 'title'"
     ]
    }
   ],
   "source": [
    "# Initialize a value to count 0 result pages \n",
    "zero_result_counter = 0\n",
    "\n",
    "for query_name in df2.iloc[109:]['Clone']:\n",
    "    data = {}\n",
    " \n",
    "    # Set query for google scholar \n",
    "    query = 'neuromab' + ' ' + query_name\n",
    "\n",
    "    # Grab data from google scholar \n",
    "    data = scrape_google_scholar_organic_results(query=query, pagination=True, operating_system=\"Windows\", year_start=2005, year_end=2024)\n",
    "\n",
    "    # create URL to grab results \n",
    "    url = CreatURL(query)\n",
    "    url_info = requests.get(url)\n",
    "    html = url_info.text\n",
    "\n",
    "    # Create a BeautifulSoup object from the HTML string\n",
    "    soup = BeautifulSoup(html, 'html.parser')\n",
    "    result_elem = soup.select_one('#gs_ab_md .gs_ab_mdw')\n",
    "\n",
    "    if result_elem:\n",
    "        result_text = result_elem.get_text(strip=True)\n",
    "        result_match = re.search(r'(\\d+[\\d,]*)\\s+results', result_text)\n",
    "        if result_match == None:\n",
    "            result_match = re.search(r'(\\d+[\\d,]*)\\s+result', result_text)\n",
    "\n",
    "        if result_match:\n",
    "            result_value = result_match.group(1)\n",
    "            final_df = CreateDF(query,result_value,data)\n",
    "        else:\n",
    "            print(query + ' ' +\"Search - No papers Available\")\n",
    "            df2.loc[df2['Clone'] == query_name, 'Result'] = \"Search - No papers Available\"\n",
    "            \n",
    "    else:\n",
    "        print(query + ' ' +\"No result element , we got dropped\")\n",
    "        df2.loc[df2['Clone'] == query_name, 'Result'] = \"No result element , we got dropped\"\n",
    "        zero_result_counter = zero_result_counter + 1\n",
    "        if zero_result_counter > 2:\n",
    "            print('Going to bed on' + ' ' + query)\n",
    "            time.sleep(3600)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "6cf10345",
   "metadata": {},
   "outputs": [],
   "source": [
    "df2.to_excel('/Users/HMans_MacBook_Pro/Library/Mobile Documents/com~apple~CloudDocs/Desktop/TrimmerLab/runresults.xlsx')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "658ff054",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>title_link</th>\n",
       "      <th>publication_info</th>\n",
       "      <th>snippet</th>\n",
       "      <th>cited_by_link</th>\n",
       "      <th>cited_by_count</th>\n",
       "      <th>pdf_file</th>\n",
       "      <th>Antibody</th>\n",
       "      <th>Target</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Metabolic regulation of Kv channels and cardia...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>PJ Kilfoil, KC Chapalamadugu, X Hu, D Zhang… -...</td>\n",
       "      <td>… (NeuroMab; K17/70) primary antibodies in 200...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=27357...</td>\n",
       "      <td>17.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Distribution of language‐related Cntnap2 prote...</td>\n",
       "      <td>https://onlinelibrary.wiley.com/doi/abs/10.100...</td>\n",
       "      <td>MC Condro, SA White - Journal of Comparative N...</td>\n",
       "      <td>… NP_034728), conserved in zebra finch Neuroma...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=16321...</td>\n",
       "      <td>36.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Stress-evoked tyrosine phosphorylation of sign...</td>\n",
       "      <td>https://www.jneurosci.org/content/30/31/10472....</td>\n",
       "      <td>H Ohnishi, T Murata, S Kusakari, Y Hayashi… - ...</td>\n",
       "      <td>… Mouse mAbs to Kvβ2 (K17/70) and to Kv1.4 (K1...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=10583...</td>\n",
       "      <td>37.0</td>\n",
       "      <td>https://www.jneurosci.org/content/jneuro/30/31...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Selective loss of presynaptic potassium channe...</td>\n",
       "      <td>https://www.jneurosci.org/content/35/32/11433....</td>\n",
       "      <td>MJ Kole, J Qian, MP Waase, TL Klassen… - Journ...</td>\n",
       "      <td>A specialized axonal ending, the basket cell “...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=18110...</td>\n",
       "      <td>22.0</td>\n",
       "      <td>https://www.jneurosci.org/content/jneuro/35/32...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>K+ channel alterations in the progression of e...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>PI Jukkola, AE Lovett-Racke, SS Zamvil, C Gu -...</td>\n",
       "      <td>… (clone #: K14/16, K17/70, and K13/31, respec...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=98209...</td>\n",
       "      <td>45.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>An αII spectrin-based cytoskeleton protects la...</td>\n",
       "      <td>https://www.jneurosci.org/content/37/47/11323....</td>\n",
       "      <td>CYM Huang, C Zhang, DR Zollinger… - Journal of...</td>\n",
       "      <td>Axons must withstand mechanical forces, includ...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=14284...</td>\n",
       "      <td>53.0</td>\n",
       "      <td>https://www.jneurosci.org/content/jneuro/37/47...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Identification of voltage-gated K+ channel be...</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>C Bavassano, L Marvaldi, M Langeslag, B Sarg… ...</td>\n",
       "      <td>… Monoclonal anti-Kvβ2 antibody (clone K17/70)...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=12225...</td>\n",
       "      <td>23.0</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>A toolbox of IgG subclass-switched recombinant...</td>\n",
       "      <td>https://elifesciences.org/articles/43322</td>\n",
       "      <td>NP Andrews, JX Boeckman, CF Manning, JT Nguyen...</td>\n",
       "      <td>… We thank the current and former staff member...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=17050...</td>\n",
       "      <td>23.0</td>\n",
       "      <td>https://elifesciences.org/articles/43322.pdf</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>mTOR-dependent alterations of Kv1. 1 subunit ...</td>\n",
       "      <td>https://link.springer.com/content/pdf/10.1038/...</td>\n",
       "      <td>LH Nguyen, AE Anderson - Scientific Reports, 2...</td>\n",
       "      <td>Cortical dysplasia (CD) is a common cause for ...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=68005...</td>\n",
       "      <td>25.0</td>\n",
       "      <td>https://link.springer.com/content/pdf/10.1038/...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Characterization of the axon initial segment ...</td>\n",
       "      <td>https://bmcbiol.biomedcentral.com/articles/10....</td>\n",
       "      <td>A Duflocq, F Chareyre, M Giovannini… - BMC …, ...</td>\n",
       "      <td>The axon initial segment (AIS) plays a crucial...</td>\n",
       "      <td>https://scholar.google.com/scholar?cites=34662...</td>\n",
       "      <td>95.0</td>\n",
       "      <td>https://bmcbiol.biomedcentral.com/articles/10....</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Interaction proteomics analysis of the pre-sy...</td>\n",
       "      <td>https://research.vumc.nl/files/472719/chapter%...</td>\n",
       "      <td>N Chen, F Koopmans, RC van der Schors, P Klemm...</td>\n",
       "      <td>… Neuromab antibodies were provided by UC Davi...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://research.vumc.nl/files/472719/chapter%...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Modulation of cardiac Kv currents by Kvbeta2 a...</td>\n",
       "      <td>https://ir.library.louisville.edu/etd/2400/</td>\n",
       "      <td>PJ Kilfoil - 2016 - ir.library.louisville.edu</td>\n",
       "      <td>Myocardial voltage-gated potassium (Kv) channe...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://ir.library.louisville.edu/cgi/viewcont...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Secretin-Modulated Potassium Channel Trafficki...</td>\n",
       "      <td>https://scholarworks.uvm.edu/graddis/241/</td>\n",
       "      <td>M Williams - 2013 - scholarworks.uvm.edu</td>\n",
       "      <td>The voltage-gated potassium channel Kv1. 2 is ...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://scholarworks.uvm.edu/cgi/viewcontent.c...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>The Role of Potassium Ion and Water Channels ...</td>\n",
       "      <td>https://search.proquest.com/openview/e61a02ffd...</td>\n",
       "      <td>PI Jukkola - 2014 - search.proquest.com</td>\n",
       "      <td>Neuroinflammation and demyelination in multipl...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://etd.ohiolink.edu/apexprod/rws_etd/send...</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Role of Autism Susceptibility Gene, CNTNAP2, i...</td>\n",
       "      <td>https://apps.dtic.mil/sti/citations/ADA604001</td>\n",
       "      <td>SA White - 2013 - apps.dtic.mil</td>\n",
       "      <td>Variants of the contactin associated protein-l...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://apps.dtic.mil/sti/pdfs/ADA604001.pdf</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>A Songbird Model of Genetically Based Speech ...</td>\n",
       "      <td>https://search.proquest.com/openview/ab2c1a9e2...</td>\n",
       "      <td>MC Condro - 2013 - search.proquest.com</td>\n",
       "      <td>Abstract Language is a complex communicative b...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>K17/70</td>\n",
       "      <td>Kvbeta2/KCNAB2 K+ channel</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                title  \\\n",
       "0   Metabolic regulation of Kv channels and cardia...   \n",
       "1   Distribution of language‐related Cntnap2 prote...   \n",
       "2   Stress-evoked tyrosine phosphorylation of sign...   \n",
       "3   Selective loss of presynaptic potassium channe...   \n",
       "4   K+ channel alterations in the progression of e...   \n",
       "5   An αII spectrin-based cytoskeleton protects la...   \n",
       "6    Identification of voltage-gated K+ channel be...   \n",
       "7   A toolbox of IgG subclass-switched recombinant...   \n",
       "8    mTOR-dependent alterations of Kv1. 1 subunit ...   \n",
       "9    Characterization of the axon initial segment ...   \n",
       "10   Interaction proteomics analysis of the pre-sy...   \n",
       "11  Modulation of cardiac Kv currents by Kvbeta2 a...   \n",
       "12  Secretin-Modulated Potassium Channel Trafficki...   \n",
       "13   The Role of Potassium Ion and Water Channels ...   \n",
       "14  Role of Autism Susceptibility Gene, CNTNAP2, i...   \n",
       "15   A Songbird Model of Genetically Based Speech ...   \n",
       "\n",
       "                                           title_link  \\\n",
       "0   https://www.sciencedirect.com/science/article/...   \n",
       "1   https://onlinelibrary.wiley.com/doi/abs/10.100...   \n",
       "2   https://www.jneurosci.org/content/30/31/10472....   \n",
       "3   https://www.jneurosci.org/content/35/32/11433....   \n",
       "4   https://www.sciencedirect.com/science/article/...   \n",
       "5   https://www.jneurosci.org/content/37/47/11323....   \n",
       "6   https://www.sciencedirect.com/science/article/...   \n",
       "7            https://elifesciences.org/articles/43322   \n",
       "8   https://link.springer.com/content/pdf/10.1038/...   \n",
       "9   https://bmcbiol.biomedcentral.com/articles/10....   \n",
       "10  https://research.vumc.nl/files/472719/chapter%...   \n",
       "11        https://ir.library.louisville.edu/etd/2400/   \n",
       "12          https://scholarworks.uvm.edu/graddis/241/   \n",
       "13  https://search.proquest.com/openview/e61a02ffd...   \n",
       "14      https://apps.dtic.mil/sti/citations/ADA604001   \n",
       "15  https://search.proquest.com/openview/ab2c1a9e2...   \n",
       "\n",
       "                                     publication_info  \\\n",
       "0   PJ Kilfoil, KC Chapalamadugu, X Hu, D Zhang… -...   \n",
       "1   MC Condro, SA White - Journal of Comparative N...   \n",
       "2   H Ohnishi, T Murata, S Kusakari, Y Hayashi… - ...   \n",
       "3   MJ Kole, J Qian, MP Waase, TL Klassen… - Journ...   \n",
       "4   PI Jukkola, AE Lovett-Racke, SS Zamvil, C Gu -...   \n",
       "5   CYM Huang, C Zhang, DR Zollinger… - Journal of...   \n",
       "6   C Bavassano, L Marvaldi, M Langeslag, B Sarg… ...   \n",
       "7   NP Andrews, JX Boeckman, CF Manning, JT Nguyen...   \n",
       "8   LH Nguyen, AE Anderson - Scientific Reports, 2...   \n",
       "9   A Duflocq, F Chareyre, M Giovannini… - BMC …, ...   \n",
       "10  N Chen, F Koopmans, RC van der Schors, P Klemm...   \n",
       "11      PJ Kilfoil - 2016 - ir.library.louisville.edu   \n",
       "12           M Williams - 2013 - scholarworks.uvm.edu   \n",
       "13            PI Jukkola - 2014 - search.proquest.com   \n",
       "14                    SA White - 2013 - apps.dtic.mil   \n",
       "15             MC Condro - 2013 - search.proquest.com   \n",
       "\n",
       "                                              snippet  \\\n",
       "0   … (NeuroMab; K17/70) primary antibodies in 200...   \n",
       "1   … NP_034728), conserved in zebra finch Neuroma...   \n",
       "2   … Mouse mAbs to Kvβ2 (K17/70) and to Kv1.4 (K1...   \n",
       "3   A specialized axonal ending, the basket cell “...   \n",
       "4   … (clone #: K14/16, K17/70, and K13/31, respec...   \n",
       "5   Axons must withstand mechanical forces, includ...   \n",
       "6   … Monoclonal anti-Kvβ2 antibody (clone K17/70)...   \n",
       "7   … We thank the current and former staff member...   \n",
       "8   Cortical dysplasia (CD) is a common cause for ...   \n",
       "9   The axon initial segment (AIS) plays a crucial...   \n",
       "10  … Neuromab antibodies were provided by UC Davi...   \n",
       "11  Myocardial voltage-gated potassium (Kv) channe...   \n",
       "12  The voltage-gated potassium channel Kv1. 2 is ...   \n",
       "13  Neuroinflammation and demyelination in multipl...   \n",
       "14  Variants of the contactin associated protein-l...   \n",
       "15  Abstract Language is a complex communicative b...   \n",
       "\n",
       "                                        cited_by_link  cited_by_count  \\\n",
       "0   https://scholar.google.com/scholar?cites=27357...            17.0   \n",
       "1   https://scholar.google.com/scholar?cites=16321...            36.0   \n",
       "2   https://scholar.google.com/scholar?cites=10583...            37.0   \n",
       "3   https://scholar.google.com/scholar?cites=18110...            22.0   \n",
       "4   https://scholar.google.com/scholar?cites=98209...            45.0   \n",
       "5   https://scholar.google.com/scholar?cites=14284...            53.0   \n",
       "6   https://scholar.google.com/scholar?cites=12225...            23.0   \n",
       "7   https://scholar.google.com/scholar?cites=17050...            23.0   \n",
       "8   https://scholar.google.com/scholar?cites=68005...            25.0   \n",
       "9   https://scholar.google.com/scholar?cites=34662...            95.0   \n",
       "10                                               None             NaN   \n",
       "11                                               None             NaN   \n",
       "12                                               None             NaN   \n",
       "13                                               None             NaN   \n",
       "14                                               None             NaN   \n",
       "15                                               None             NaN   \n",
       "\n",
       "                                             pdf_file Antibody  \\\n",
       "0   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7...   K17/70   \n",
       "1   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3...   K17/70   \n",
       "2   https://www.jneurosci.org/content/jneuro/30/31...   K17/70   \n",
       "3   https://www.jneurosci.org/content/jneuro/35/32...   K17/70   \n",
       "4   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3...   K17/70   \n",
       "5   https://www.jneurosci.org/content/jneuro/37/47...   K17/70   \n",
       "6   https://www.sciencedirect.com/science/article/...   K17/70   \n",
       "7        https://elifesciences.org/articles/43322.pdf   K17/70   \n",
       "8   https://link.springer.com/content/pdf/10.1038/...   K17/70   \n",
       "9   https://bmcbiol.biomedcentral.com/articles/10....   K17/70   \n",
       "10  https://research.vumc.nl/files/472719/chapter%...   K17/70   \n",
       "11  https://ir.library.louisville.edu/cgi/viewcont...   K17/70   \n",
       "12  https://scholarworks.uvm.edu/cgi/viewcontent.c...   K17/70   \n",
       "13  https://etd.ohiolink.edu/apexprod/rws_etd/send...   K17/70   \n",
       "14       https://apps.dtic.mil/sti/pdfs/ADA604001.pdf   K17/70   \n",
       "15                                               None   K17/70   \n",
       "\n",
       "                       Target  \n",
       "0   Kvbeta2/KCNAB2 K+ channel  \n",
       "1   Kvbeta2/KCNAB2 K+ channel  \n",
       "2   Kvbeta2/KCNAB2 K+ channel  \n",
       "3   Kvbeta2/KCNAB2 K+ channel  \n",
       "4   Kvbeta2/KCNAB2 K+ channel  \n",
       "5   Kvbeta2/KCNAB2 K+ channel  \n",
       "6   Kvbeta2/KCNAB2 K+ channel  \n",
       "7   Kvbeta2/KCNAB2 K+ channel  \n",
       "8   Kvbeta2/KCNAB2 K+ channel  \n",
       "9   Kvbeta2/KCNAB2 K+ channel  \n",
       "10  Kvbeta2/KCNAB2 K+ channel  \n",
       "11  Kvbeta2/KCNAB2 K+ channel  \n",
       "12  Kvbeta2/KCNAB2 K+ channel  \n",
       "13  Kvbeta2/KCNAB2 K+ channel  \n",
       "14  Kvbeta2/KCNAB2 K+ channel  \n",
       "15  Kvbeta2/KCNAB2 K+ channel  "
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "final_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f6720274",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "df_list = []\n",
    "path_to_files = \"\"\n",
    "for file in os.listdir(path_to_files):\n",
    "    query = pd.read_csv(f\"{path_to_files}/{file})\n",
    "    query[\"query\"] = file.split(\".\")[0]\n",
    "    df_list.append(pd.read_csv(\"file\"))\n",
    "final_df = pd.concat(df_list)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
