---
title: Cleaning Report `r tail(unlist(strsplit(getwd(), '/')), 2)[1]`
output:
  html_document:
    df_print: paged
date: "`r format(Sys.time(), '%d %B, %Y')`"
---

```{r setup, echo=FALSE, results="hide"}
library('jsonlite')
library('stringr')
library('knitr')
library('readr')
library(kableExtra)

#setwd('/bio/CoreWork/2019.11.18-Trimmer-Hybridoma-Seq/2019-12-20-new-191213_M02034-data/Report')
#opts_knit$set(root.dir = '/bio/CoreWork/2019.11.18-Trimmer-Hybridoma-Seq/2020-02-07-run-rerun_of_27-samples-SMARTPCR/Report')


################## Functions #############
# Extract specific pieces of data from a set of log files:
parse_set = function(pattern, path){
  df = NA
  samples = c()
  for(f in dir(path=path, pattern=pattern)){
    s = str_replace(f, '.log', '')
    samples = c(samples, s)
    results = fromJSON(file.path(path,f))
    rownames(results$Fragment) = results$Program_details$program

    #Set up primerData 
    primerData = results$Fragment[1,'primers_counts'][[1]]
    primerCounts = list(
        Ga12HC = primerData[primerData[,1] == 'p5primer1' & primerData[,2] == 'Ga12HC.10_REV_CHI',3],
        Ga3HC = primerData[,3][primerData[,1] == 'p5primer1' & primerData[,2] == 'Ga3.10for_mIGH3'],
        LaLC = primerData[,3][primerData[,1] == 'p5primer1' & primerData[,2] == 'LaLC.R-mIGL'],
        KaLC = primerData[,3][primerData[,1] == 'p5primer1' & primerData[,2] == 'KaLC.6_REV_LC'])
    primerCounts = unlist(lapply(primerCounts, function(x) if(length(x) == 0) 0 else as.numeric(x)))

    sdata = c(
        RawReads = as.numeric(results$Fragment[1,'in']), 
        ReadsWithPrimer = as.numeric(results$Fragment[1,'out']),
        primerCounts,
        ReadsToFilter = as.numeric(results$Fragment[3,"in"]),
        PassedAberrantFilter = as.numeric(results$Fragment[3,"out"]),
        Aberrant_LC_reads = as.numeric(results$Paired_end[3,'hits']),
        ReadsToOverlap = as.numeric(results$Fragment[5,'in']),
        ShortInsert = as.numeric(results$Fragment[5,'inserts'][['short']]),
        MediumInserts = as.numeric(results$Fragment[5, 'inserts'][['medium']]),
        LongInserts = as.numeric(results$Fragment[5, 'inserts'][['long']]),
        Overlapped = as.numeric(results$Single_end[5,'out']),
        LengthFiltered = as.numeric(results$Single_end[6,'out']))

    if(is.null(dim(df))){ 
        df = data.frame(sdata) } else {
            df = cbind(df, sdata)
        }
  }
  colnames(df) = samples
  df = t(df)
  return(df)
}


##################

```

# SMART-PCR strategy
Analysis for samples prepared with SMART-PCR + (10-REV-HC1 or 6-REV-LC or mIGHG or mIGK) primers:

1. Demultiplexed with bcl2fastq using Illumina barcodes into pools.
1. Samples pooled by plate where then processed through HTStream:
    +  Demultiplex and orient paired reads based on SMART-PCR primer + oligo sequence and trim primers (perfect match, anchored primers). 
    +  Discard any read containing an "N"
    +  Count reads matching the aberrant LC sequence
    +  Trim bases from reads with -q 10
    +  Overlap reads
    +  Filter any reads less than 385bp in length


## Heavy Chain:

```{r Heavy Chain Results, echo=FALSE}
path = "../01-PrimerTrim/"
pattern = ".*_HCprimers.log"
hcReadStats = parse_set(pattern, path)
#colnames(p10RevHc1) = sapply(strsplit(gsub("1_Short_primers_", '', colnames(p10RevHc1)), '_'), '[', 1)
rownames(hcReadStats) = gsub("_HCprimers", '', rownames(hcReadStats))

kable(hcReadStats) %>%
  kable_styling(bootstrap_options = c("hover", "striped"), full_width = F, 
        position="left", fixed_thead=T) %>%
  row_spec(0, angle = 0)
```




## Light Chain:

```{r Light Chain Results, echo=FALSE}
path = "../01-PrimerTrim/"
pattern = ".*_LCprimers.log"
lcReadStats = parse_set(pattern, path)
rownames(lcReadStats) = gsub("_LCprimers", '', rownames(lcReadStats))

kable(lcReadStats) %>%
  kable_styling(bootstrap_options = c("hover", "striped"), full_width = F, 
        position="left", fixed_thead=T) %>%
  row_spec(0, angle = 0)

```

```{r writecsv, echo=FALSE}
df = data.frame(rbind(data.frame(plate=plate, Chain='H', SMARTindex=rownames(hcReadStats), hcReadStats), 
                data.frame(plate=plate, Chain='L', SMARTindex=rownames(lcReadStats), lcReadStats)))
write.table(file='demux_stats.tsv', df,  row.names=F, col.names=T, sep='\t')

```